{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from PubMed.PmcScraper import PmcScraper\n",
    "import os\n",
    "\n",
    "# need .env file to store ncbi api key\n",
    "pmc_scraper = PmcScraper(os.environ['NCBI_API_KEY'])\n",
    "\n",
    "# search based on keyword and retrieve the desired amount of articles\n",
    "search_results = pmc_scraper.search_pmc(search_term=\"gut microbiota\", number_of_articles=100)\n",
    "\n",
    "# extract the full text version of articles using pmcids\n",
    "full_text_articles = pmc_scraper.fetch_full_text(search_results['esearchresult']['idlist'])\n",
    "\n",
    "# extract the desired section of articles\n",
    "method_sections = pmc_scraper.extract_section(full_text_articles, 'method')\n",
    "\n",
    "# print as dictionary\n",
    "method_dict = dict(zip(search_results['esearchresult']['idlist'], method_sections))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'11193528': 'MATERIALS AND METHODS The detailed protocols are available in Supplemental Materials (11–15); all Supplemental material is available at https://doi.org/10.6084/m9.figshare.24556174.',\n",
       " '11193507': 'Materials and Methods:NAFLD was induced in male Wistar rats (aged 6–8 weeks) by feeding them a high-fat diet (HFD) for 6 weeks. Subsequently, the rats were divided into four groups, with Group 1 continuing on HFD, while groups 2, 3, and 4 received HFD supplemented with saroglitazar, curcumin, and both saroglitazar and curcumin, respectively. We evaluated the expression of Nrf2, ERK1/2, NOX1,2,4, antioxidant enzymes, PPARα, γ, and genes regulating lipid metabolism in the liver. Histopathology of liver tissue was also examined. Furthermore, we analyzed serum levels of lipid profiles and hepatic enzymes.',\n",
       " '11193499': 'Materials and Methods To induce NAFLD, we prepared a high-fat emulsion following the methodology outlined in the studies conducted by Zou et al. (10). This emulsion, containing 4243 kcal/L of energy, comprised 77% fat, 9% carbohydrates, and 14% protein, and was fortified by vitamin/mineral supplements. It was kept at 4 °C and warmed to 42 °C prior to administration. Medicines Saroglitazar was acquired from Cadila Healthcare Limited, located in Ahmedabad, India. Additionally, curcumin powder with 98% purity (98% curcuminoids) was obtained from Abidi Pharmaceutical Company in Tehran, Iran. These medications were then dissolved in a solution of 0.5% sodium carboxymethyl cellulose (Na-CMC). Study design 46 adult male Wistar rats, aged between 6 and 8 weeks and weighing 180–200 grams, were acquired from Ahvaz Jundishapur University of Medical Sciences’ Experimental Animal Center. These rats underwent a 7-day period of adaptation in a controlled environment with a steady temperature of 25.3 °C, relative humidity of 58%, and a light/dark cycle lasting 12 hr each. Then, in order to induce NAFLD, 36 rats were randomly selected and fed HFD (10 ml/kg oral emulsion) for 6 weeks as the high-fat group and received 30 mg/kg streptozotocin (STZ) by IP injection at the end of the 6th week; Another 10 rats, as the normal control group (NC), received a standard food diet. Biopsies were taken from two rats in the NC group and four rats in the high-fat group to evaluate the development of NAFLD or NASH after a 6-week period. Group 1 (or NC) was given a solution containing 0.5% Na-CMC. The second group was fed HFD at a daily dose of 10 mg/kg. The third group received HFD along with saroglitazar at a dose of 3 mg/kg per day (24). The fourth group was also fed HFD and administered curcumin at a dose of 200 mg/kg daily (25). The fifth group, similarly fed HFD, was treated with a combination of saroglitazar and curcumin. After a six-week treatment period, rats were sacrificed by intraperitoneal administration of a high dose of a ketamine and xylazine cocktail while fasting for 12 hr. Blood samples were then collected from the aorta, and serum was obtained by centrifugation at 4500 rpm for 15 min. This serum was preserved at -20 °C until biochemical tests were performed. The liver index was calculated by removing the liver tissues, washing them in chilled saline, drying them, and then weighing them. These liver tissues were then divided into two parts: one was fixed in 10% formalin for histopathological studies, and the other was stored at -80 °C for subsequent gene expression studies. Biochemical measurements All measurements were performed on fresh frozen serum. Serum lipoprotein levels (mmol/l) (Low-density lipoprotein (LDL-C) and high-density lipoprotein (HDL)) and liver enzymes (IU/L) (alanine aminotransferase (ALT) and aspartate aminotransferase (AST)) were evaluated using routine laboratory methods. Gene expression analysis Frozen liver tissue was used for RNA extraction. The RNA kit (Yekta Tajhiz, Iran) was used to isolate total RNA. cDNA was produced using the the PrimeScript RT reagent kit (Amplicon, USA) based on reverse transcription. The produced cDNA was then used to analyze a quantitative real-time polymerase chain reaction (RT-PCR) by an ABI StepOnePlus RT-PCR system. According to the planned cycle schedule, the real-time PCR reaction was carried out. The sequence of designed primers is mentioned in Table 1. The ΔCt method was used to compare the expression of each gene to that of the glyceraldehyde-3-phosphate dehydrogenase (GAPDH) as the internal control gene. The fold-change value was calculated using the 2^(-ΔΔCt) formula. This value represents the quantity changes of the target group compared to the corresponding control group. ROS production assay ROS levels in liver tissue were evaluated using a commercially available kit (KiaZist, Iran). Briefly, ROSs in the sample oxidize the Fe2+-o-dianisidine complex to a Fe3+ which forms a colored complex with xylenol orange that can be detected spectrophotometrically at a wavelength of 560 nm, and its intensity is proportional to ROS levels. The assay is calibrated with H2O2, and the results are reported as nmol H2O2 equivalent/ml. Histopathological evaluations Liver tissue samples were fixed in 10% buffered formalin for histological analysis for 24 hr. Samples were embedded in paraffin wax after dehydration using gradient alcohol. Slices were cut from the paraffin blocks (6–7 μm) and placed on poly-L-lysine-coated slides. Next, using xylene and graded ethanol, the slices were deparaffinized and hydrated. Hematoxylin-eosin (H&E) staining solutions were used to stain these sections, and then the slides were observed and histologically evaluated using a light microscope. Western blot The ERK1/2 and Nrf2 protein levels were measured using western blot analysis. Liver tissue was homogenized, and proteins were extracted using RIPA lysis buffer and a protease inhibitor cocktail. The total protein content was then measured with the Bradford assay, and an equivalent amount of protein was loaded onto a 10% sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) gel, which was then transferred onto a PVDF membrane. The membranes were blocked with 5% skim milk for 1 hr at (20–25 °C). Overnight incubation at 4 °C was performed with primary antibodies targeting phosphorylated ERK1/2 and Nrf2 proteins (1:1000, Santa Cruz). After three washes with PBS-Tween20, the membranes were incubated with an HRP-conjugated secondary antibody for 1 hr. After additional washes, the bands were detected using the Enhanced Chemiluminescence (ECL) Detection Kit (Amersham Corp., Arlington Heights, IL, USA). Protein amounts were normalized against GAPDH levels and semi-quantified using ImageJ software. Statistical analysis The SPSS software (v.21, IBM Corporation) was used to perform statistical analysis. T-tests were employed to assess the results of variable experiments between the HFD and control groups. For comparisons involving more than two groups, one-way analysis of variance (ANOVA) with a Tukey-Kramer post hoc test was utilized. The data were shown as mean ± standard deviation (SD), and statistical significance was set at P<0.05. Figure 1 Assessment of body weight, liver index, TG levels, and morphology The high-fat diet (HFD) fed rats showed significant increases in body weight (A), liver index (liver weight/body weight ratio) (B), and liver Triglyceride levels (C) compared to the control group. After drug administration, these variables decreased significantly compared to the HFD group. Histopathological evaluations of liver tissue were shown by hematoxylin-eosin staining (D). SARO = Saroglitazar; CUR= curcumin, CON= control, (C: congestion; I: inflammation; S: steatosis). Statistical significance between groups is shown by the * and # symbols. (*** P<0.001), (# P<0.05), (## P<0.01), and (### P<0.001). Figure 2 Assessment of the serum levels of lipid profiles and liver enzymes AST, ALT, and LDL-C levels show a significant increase, and HDL-C levels show a significant decrease in high-fat diet (HFD) fed rats. These variables changed significantly in the drug-treated groups. SARO= Saroglitazar; CUR= curcumin, CON= control. Statistical significance between groups is shown by the * and # symbols. (*(P<0.05), (** P<0.01), (# P<0.05), (## P<0.01), and (### P<0.001). Figure 3 Assessment of the level of lipid metabolism genes in liver tissue of rats fed a high-fat diet (HFD) The relative expression levels of SREBP-1c, FAS, ACC, PPARγ, PPARα, and CPT-1α mRNA were assessed using real-time PCR. The mRNA expression each gene was normalized to GAPDH expression (internal control). In each sample pair, the fold change was obtained utilizing 2-ΔΔCT formula. SARO= Saroglitazar; CUR= curcumin, CON= control. Statistical significance between groups is shown by the * and # symbols. (**P<0.01), (*** P<0.001), (#P<0.05), (##P<0.01), and (###P<0.001). Figure 4 Assessment of the level of pro-inflammatory cytokines in liver tissue of rats fed a high-fat diet (HFD) The relative expression of IL-1, IL-6, TNF-α, TGF- β1, COX-2, and MCP-1 mRNA were assessed using real-time PCR. The mRNA expression each gene was normalized to GAPDH expression (internal control). In each sample pair, the fold change was obtained utilizing 2-ΔΔCT formula. SARO= Saroglitazar; CUR= curcumin, CON= control. Statistical significance between groups is shown by the * and # symbols. (* P<0.05), (**P<0.01), (*** P<0.001), (# P<0.05), (## P<0.01), and (### P<0.001). Figure 5 Assessment of antioxidant defense in liver tissue of rats fed a high-fat diet (HFD) The expression levels of Nrf2, SOD-1, HO-1, GPx, and NOX1, 2, 4, mRNA were assessed using real-time PCR. The mRNA each gene was normalized to GAPDH expression (internal control). In each sample pair, the fold change was obtained utilizing 2-ΔΔCT formula. (D) shows the levels of reactive oxygen species (ROS) before and after drug treatment. SARO= Saroglitazar; CUR= curcumin, CON= control. Statistical significance between groups is shown by the * and # symbols. (* P<0.05), (** P<0.01), (*** P<0.001), (# (P<0.05), (## P<0.01), and (### P<0.001). Figure 6 Western blotting detection for ERK-1/2 and Nrf2 protein expression (A) bands of western blotting in different groups of rats fed a high-fat diet (HFD) compared to control group. (B, C) the proteins expression of ERK-1/2 and Nrf2 were estimated by determining intensities of ERK-1/2 and Nrf2 bands relative to GAPDH bands using ImageJ software. SARO= Saroglitazar; CUR= curcumin, CON= control, Nrf2= nuclear factor erythroid 2-related factor 2, ERK1/2= extracellular signal-regulated kinase 1/2.. Statistical significance between groups is shown by the * and # symbols. (*** P<0.001), (## P<0.01), and (### P<0.001). Figure 7 Mechanism of action of saroglitazar and curcumin in improving NAFLD is that both components regulate lipid metabolism and inflammation through nuclear factor erythroid 2-related factor 2 (Nrf2) and extracellular signal-regulated kinase 1/2 (ERK1/2)',\n",
       " '11193497': 'Materials and Methods:Eighty immature female mice were divided into 4 groups: Control (C), PCOS (P), Lupeol (L), and Flutamide (F). PCOS was induced in test groups by injection of Dehydroepiandrosterone (60 mg/kg/day, IP) for twenty days. Following the PCOS induction, the two groups of L and F were treated with lupeol (40 mg/kg/day) and/or flutamide (10 mg/kg/day) respectively and the two groups of C and P received sesame oil (0.1 ml/mouse/day) for 15 days. After the treatment period, ten animals in each group were selected for collecting blood and ovary samples. In vitro fertilization assessment was carried out on 10 remaining mice in each group. The hormonal assays and oxidative stress biomarker determination were performed on serum and tissue samples. Moreover, histopathological analyses were conducted on the ovaries.',\n",
       " '11193432': 'Materials and Methods Chemicals Dehydroepiandrosterone (D4000) and Lupeol (L5632) were procured from Sigma, USA. Flutamide (Flutan) was purchased from Medochemie, Cyprus. A common type of edible sesame oil was obtained from a local grocery. PMSG and hCG hormones were purchased from Folligon, the Netherlands. The hematoxylin and eosin (H&E) and AZAN staining kits were supplied by Asia Pajohesh, Iran. Drugs were dissolved in sesame oil on a daily basis and used freshly. Animal grouping and treatments Eighty immature 20-day-old weaned female NMRI mice weighing approximately 14–17 g were obtained from the Laboratory Animal Center of the Faculty of Veterinary Medicine, Urmia University, Iran. Mice were housed in standard polycarbonate cages in a room with controlled temperature at 23–24 °C and continuous air renovation on a 10:14 hr light-dark cycle and access to regular chow diet and fresh tap water ad libitum. Animals were randomly divided into 4 groups of twenty mice in each group. The groups were nominated as Control (C), PCOS (P), Lupeol (L), and Flutamide (F) and treatment of animals started at the age of 22 days after 2 2-day adaptation periods to the environmental conditions. For induction of PCOS, test groups including the P, L, and F received DHEA (60 mg/kg/day, IP), and the control group received 0.1 ml/mouse/day sesame oil (SO) for twenty consecutive days (36). Afterward, the two groups of L and F were treated with lupeol (40 mg/kg/day) (35) and/or Flutamide (10 mg/kg/day) respectively, and the two groups of C and P received SO (0.1 ml/mouse/day) for 15 consecutive days. All medications were administered intraperitoneally and sesame oil was used as drug solvent and vehicle. Finally, the estrus cycle was determined by preparing vaginal smears and cytological staining of smears using crystal violet. Ten animals in each group in the metestrus/diestrus phase were selected for collecting blood and ovary tissue samples. To evaluate the fertility potential of the animals, an in vitro fertilization experiment was carried out on 10 remaining mice in each group. The Animal Care and Use Committee at Urmia University approved the experimental procedures on animals (IR-UU-AEC/1395/89/2d). Sampling After induction of deep general anesthesia by IP injection of 0.1 ml/20 g B.W. of a Ketamine and Xylazine cocktail (17.5 mg/ml Ketamine/2.5 mg/ml Xylazine), blood samples were collected via cardiac puncture and were centrifuged at 1500 ×g for 10 min to separate blood serum. Collected serum from each animal was aliquoted into 100 μl volumes and stored at -20 °C until further analyses were performed. The ovaries were dissected out and cleaned with chilled saline normal solution. Thereafter, for all animals, the right ovary was snap frozen in liquid nitrogen and stored at -80 °C and the left ovary was fixed in 10% formalin solution and stored at room temperature until being processed for histology analyses. Sex hormone levels in serum The serum level of LH, FSH, Estradiol (E2), and Testosterone (T) hormones were determined in serum samples using commercially available ELISA kits (Monobind Inc, Lake Forest, CA, USA) according to the provided instructions by the manufacturers. Histological analyses of ovaries Formalin-fixed ovarian tissues were dehydrated by passing a standard graded series of ethanol and cleared with xylene, embedded in paraffin, and finally serially sectioned into 5 µm sections. To stain the prepared sections, H&E and/or Azocarmine-Aniline blue (AZAN) methods were performed. Histomorphometric study of the ovaries was performed with an optical microscope (Olympus Bx50) to determine the growth and atresia of ovarian follicles and counting follicles in different developmental stages in H & E stained ovarian serial sections (37, 38). The qualitative study of collagen changes and tissue fibrosis in the ovaries (39) was performed on AZAN-stained sections (40). Hence, the blue areas were considered as collagen fiber sedimentation and quantitative measurements of collagen sedimentation and fibrosis were performed by image analysis using Matlab software (V. 8.1.0) in 40 microscopic images from each group. The average ratio of blue pixels to the entire image pixels was determined as the mean percentage of tissue collagen (41). Oxidative stress assessment Total anti-oxidant capacity (TAC) of ovarian tissue and serum The total anti-oxidant capacity of ovarian tissue and serum samples was measured using the FRAP method. In this method, in acidic pH which has been created by acetate buffer, the blue color produced by the reduction of Fe3+ ions of the Fe (III)-TPTZ complex and converting them into ferrous (Fe2+) ions is measured at 593 nm by a spectrophotometer (42). Malondialdehyde (MDA) levels in ovarian tissue and serum To determine the rate of lipid peroxidation, the concentration of malondialdehyde in ovarian tissue and serum samples was measured by thiobarbituric acid (TBA) reaction. Each single ovarian specimen was separately homogenized in a cold potassium chloride solution (150 mM), and the resulting mixture was centrifuged at 3000 rpm for 10 min. For each ovary specimen, 500 μl of supernatant and for serum samples, 200 μl of serum were mixed with 3 ml phosphoric acid (1% v/v), vortexed, and then 1 ml of TBA (6/7 g/l) was added. The samples were placed at 100 °C for 45 min and then cooled in ice. Finally, after adding 3 ml of n-butanol the samples were centrifuged again at 3000 rpm for 10 min. The absorbance of supernatant was measured at 532 nm by a spectrophotometer. The standard calibration curve of malondialdehyde was used for the calculation of the produced MDA level and expressed as nanomoles of malondialdehyde per mg of protein (43). The protein content of the samples was also measured according to the Lowry method (44). Protein carbonylation (CO) in ovarian tissue and serum The level of protein oxidation in ovarian tissue as well as serum samples was determined by measuring the amount of carbonyl content. The reaction of 2, 4-dinitrophenylhydrazine (DNPH), and carbonyl groups of proteins was measured based on a previously published method (45). Briefly, each ovary specimen was first homogenized in a cold phosphate buffer (50 mM, pH 7.6, containing EDTA 1 mM), and the resulting mixture was centrifuged at 10,000 × g and 4 °C for 10 min. For each ovary specimen, 200 μl of supernatant, and for serum samples, 200 μl of serum were used; one sample as test (T) and one sample as control (C) were prepared. Thereafter 800 μl of DNPH and hydrochloric acid solution (2 M) were added to the test and control samples, respectively. Samples were kept in a dark place at room temperature for one hour and vortexed every 15 min. Then, 0.5 ml of trichloroacetic acid (30%) was added to each sample and vortexed for 30 sec. All samples were centrifuged for 10 min at 10,000 ×g for 3 min, the supernatant was discarded and residual sediment was suspended in 1 ml ethanol/ethyl acetate solution (1:1) for 15 min. After centrifugation at 10,000 ×g for 3 min and dispersing the supernatant, the above step was repeated. After the last washing, the precipitate was dissolved in 0.6 ml of guanidine hydrochloride solution (6 M) at 37 °C for 15 min and then the samples were centrifuged for 10 min at 10,000 ×g for isolating and depositing any residues. For each sample, the optical density (OD) of the control and test was measured against a solution of guanidine hydrochloride (6 M) at a wavelength of 370 nm. The carbonyl content was determined as follows: Carbonyl (nmol/ml) = [(CA)/ (0.011 mM-1)] (600 µl/200 µl) Where CA is the corrected absorbance and computed as the average OD for each control sample was subtracted from the average OD of the test sample at 370 nm. The extinction coefficient for DNPH at 370 nm is 22,000 M-1cm-1. In vitro fertilization (IVF) Ten remaining animals in each group were subjected to hormonal treatment according to a well-established super-ovulation hormonal regimen (45, 46). At the final hr of the 35th day of the treatment period, animals received 10 IU PMSG (Folligon, The Netherlands) per capita, and 46–48 hr later, were injected with 10 IU of hCG (Folligon, The Netherlands) for induction of ovulation. The next morning (10–12 hr after injection of hCG), one healthy young male mouse for three female mice were selected and after euthanizing by cervical dislocation, the abdominal region was completely sterilized and both epididymides were pulled out through a small cut on the scrotum. Epididymis was placed in a petri dish containing 1 ml HTF medium (CO2 pre-incubated and containing 4 mg/ml BSA) and a few cuts were made by surgical blade to let the sperms swim out. The sperms obtained their capacitation after about one hour of incubation. To obtain oocytes, female animals were euthanized, their abdominal region sterilized, and the oviducts were isolated through an incision on the abdomen and placed in a petri dish containing pre-incubated (37 °C) and pH-equilibrated HTF medium. Using fine point gauge 30 needles and a dissecting technique, the oocytes were pulled out of the ampulla. Using a mouth pipette, the oocytes were washed in a series of medium droplets underneath the mineral oil and finally transferred to the droplets of the fertilization medium. Thereafter capacitated sperms (one million/ml in fertilization medium) were introduced to the oocytes and incubated until completion of fertilization. After about 12 hr, pair pronuclei were seen and then fertilized zygotes were picked up and washed a few times and cultured for 120 hr in 100 μl droplets under mineral oil (47). To evaluate embryonic development, the number of two-cell embryos after 24 hr, the number of blastocysts, and the arrested embryos in each group after 120 hr of incubation were recorded. The type of embryos that failed to develop was determined based on the following criteria: Type I: Embryos with lysis, fragmentation, and complete necrosis; Type II: Embryos with lysis and fragmentation in a number of blastomeres Type III: Embryos with a small number of lysed, fragmented blastomeres, and cytoplasmic vesicles. Statistical analysis Data were analyzed using SPSS software v.20 and the results were expressed as mean ± SD. In order to compare the results, one-way ANOVA and Tukey’s post hoc analysis were performed. The results of IVF tests were analyzed using Minitab software v.16 and 2 proportion statistical tests. The P-value (P<0.05) was considered to be statistically significant.',\n",
       " '11193411': 'MethodsInformation related to patients diagnosed with ovarian cancer on the electronic medical record data platform of three tertiary hospitals in Guangdong Province from May 2018 to September 2023 was collected as the case group. Patients with non-ovarian cancer who attended the clinic during the same period were included in the control group. Logistic regression analysis was used to screen the independent variables and explore the factors associated with the development of ovarian cancer. An ovarian cancer risk prediction model was constructed using a decision tree C4.5 algorithm. The ROC and calibration curves were plotted, and the model was validated.',\n",
       " '11193387': 'Materials and Methods Objectives of the Study This was a retrospective study, and data were obtained from the electronic medical record data platforms of three tertiary hospitals in Guangzhou City. Data were collected from 67 hospitalized ovarian cancer patients on these platforms from May 2018 to September 2023, forming the ovarian cancer case group and 567 patients in the non-ovarian cancer control group. The inclusion criteria for the case group were: 1) age ≥18 years; 2) hospitalized surgical patients diagnosed with ovarian cancer by histopathology, without preoperative chemotherapy, radiation therapy, and immunotherapy; and 3) complete case information. The exclusion criteria were concurrent malignant tumors of other organs or tissues and other serious systemic diseases (abnormalities of liver and kidney functions, blood diseases, infections, and inflammation). The inclusion criteria for the control group were members of the general population who had undergone physical examinations or consultations during the same period. The research department collected the data used in this study; it has standardized protocols and principles to follow to prevent the leakage of patient personal information. The study complied with medical ethics standards and was reviewed by the Ethics Committee of Guangdong Provincial Hospital of Chinese Medicine (review approval number: ZE2023-291). When the patient agreed to be hospitalized, the doctor signed an admission notice after informing them of their condition. There is an entry in the patient’s bill of rights, which states that the patient’s disease-related information will be entered into an electronic data information platform. When medical research requires it, relevant data may be retrieved for statistical analysis while ensuring patient confidentiality. Owing to the retrospective nature of this study, an ethical review was conducted based on informed consent. Research Methodology In accordance with previous publications and other related studies, each patient’s dataset included general information (age, education, height, weight, BMI, occupation, blood type, marital status, ethnicity, family history, history of previous illnesses, and medication use), menstrual status (age at menarche, age at menopause, regular menstruation in the past, dysmenorrhea, and others), pregnancy (number of pregnancies and births, age at first pregnancy, and age at first childbirth), and lifestyle (contraceptive methods, history of smoking, alcohol consumption, coffee consumption, bowel movements, and experience of adverse life events), excluding more than 30% of the missing important medical history data. Quality Control Study participants were strictly selected according to the inclusion and exclusion criteria to reduce selection bias. Two experienced researchers extracted and output data from the medical history records of the hospital’s HIS medical record information management system. We further supplemented the partial data for each medical record using statistical software to ensure data integrity. Subsequently, we entered the data using the Epidta tool to double-check, and correct logic errors. Ensure the reliability of the sample in the end. Statistical Methods SPSS 23.0 and R 4.2.3 statistical packages were used for statistical analysis. Logistic regression analysis was used to screen the independent variables and statistically analyze the relevant factors, with P<0.05 considered statistically significant. Seventy percent of the cases in the dataset were randomly selected as the training set for modeling, and the remaining 30% were used as the test set. The ovarian cancer risk prediction model was constructed based on the decision tree C4.5 algorithm, and receiver operating characteristic (ROC) and calibration curves were plotted for both the training and test sets to assess the differentiation and calibration of the model.',\n",
       " '11193365': 'Research Methodology In accordance with previous publications and other related studies, each patient’s dataset included general information (age, education, height, weight, BMI, occupation, blood type, marital status, ethnicity, family history, history of previous illnesses, and medication use), menstrual status (age at menarche, age at menopause, regular menstruation in the past, dysmenorrhea, and others), pregnancy (number of pregnancies and births, age at first pregnancy, and age at first childbirth), and lifestyle (contraceptive methods, history of smoking, alcohol consumption, coffee consumption, bowel movements, and experience of adverse life events), excluding more than 30% of the missing important medical history data.',\n",
       " '11193241': 'Statistical Methods SPSS 23.0 and R 4.2.3 statistical packages were used for statistical analysis. Logistic regression analysis was used to screen the independent variables and statistically analyze the relevant factors, with P<0.05 considered statistically significant. Seventy percent of the cases in the dataset were randomly selected as the training set for modeling, and the remaining 30% were used as the test set. The ovarian cancer risk prediction model was constructed based on the decision tree C4.5 algorithm, and receiver operating characteristic (ROC) and calibration curves were plotted for both the training and test sets to assess the differentiation and calibration of the model.',\n",
       " '11193236': \"METHODS Mice C57BL/6 (stock no: 000664), Nur77-GFP reporter mice, (stock no: 016617), CD45.1 C57BL/6 mice (stock no: 002014), Irf8+32−/− mice (stock no: 032744), UBC-GFP mice (stock no: 004353), OT-I mice (stock no:003831), and IFN-γ−/− (stock no: 002287) were purchased from Jackson Laboratories and then maintained in-house. IFN-γ-Thy1.1 knock-in reporter mice were provided by Dr. Phillip Scott but originated in the laboratory of Dr. Casey Weaver33. Mice used in this study were males or females ranging from 5–12 weeks, and all mice were age- and sex-matched within individual experiments. All protocols for animal care were approved by the Institutional Animal Care and Use Committee of the University of Pennsylvania (protocols #805405 and #806292). Plasmid construction To see the full list of primers used for plasmid construction see Supplementary Table 1. To generate the M2-OVA and ROP1-OVA lines, the SIINFEKL fragment was inserted into p3XHA_nanoluciferase-neomycin (Nluc-Neo)26,27 by Gibson assembly. Repair templates were amplified from the resulting plasmid with primers containing 30 bp overhangs either side of a Cas9 guide-induced double-strand break at the C-terminus of either M2 (Cgd5_4590) or ROP1 (Cgd3_1770). Guide sequences were introduced into a Cas9 expressing vector at a BbsI restriction site as previously described27. To generate the M2-OVA tdTomato plasmid, tdTomato was inserted by Gibson assembly into p3XHA-SIINFEKL_Nluc-Neo, linked to Nluc-Neo with a T2A skip peptide. To generate the NS-M2-OVA plasmid the SIINFEKL coding sequence was introduced into pΔSP25 that expresses a copy of M2 lacking the 22 amino acid signal peptide coding region. The repair template was amplified with 30 bp overhangs on either side of a Cas9 guide-induced double-strand break in the thymidine kinase (Cgd5_4440) locus27. Isolation of transgenic parasites Transgenic parasites were generated as previously described59. Briefly, to excyst parasites, oocysts were bleached, washed in phosphate-buffered saline (PBS), and incubated in sodium taurodeoxycholate. Then sporozoites were resuspended in transfection buffer supplemented with 100 μg deoxyribonucleic acid (comprising 50 μg of Cas9/genomic ribonucleic acid plasmid and 50 μg of repair template) and electroporated using an Amaxa 4D nucleofector (Lonza, Basel, Switzerland). Parasites carrying a stable transgene were selected using paromomycin added to the drinking water of infected mice. Transgenic parasites were propagated by orally infecting IFN-γ−/− mice and oocysts were purified from their feces using sucrose flotation followed by a cesium chloride gradient, as previously described8. Mouse infection and measurement of parasite burden Mice were infected with 104−5 × 104 oocysts by oral gavage. Infected WT C57BL/6J mice were treated with 1 mg αIFN-γ antibody or 1 mg rat IgG1 isotype control, anti-horseradish peroxidase (BioXCell, Lebanon, NH, USA) before and after infection on various days, depending on the experiment, which is explained in the Figure legends. To quantify fecal oocyst shedding, 20 mg of pooled cage feces was suspended in 1 mL lysis buffer. Samples were shaken with glass beads for 5 minutes, then combined in a 1:1 ratio with Nano-Glo® Luciferase solution (Promega, Madison, WI, USA, Ref N1150). A Promega GloMax plate reader was used to measure luminescence. Pooled samples were used because previous studies have demonstrated that mice within each cage are equally infected60. T cell transfers For T cell transfers, OT-I mice were interbred with CD45.1/Nur77-GFP reporter mice. To isolate OT-I CD8+ T cells, lymph nodes, and spleens were harvested, and leukocytes were obtained by dissociation over a 70-um filter. Red blood cells were lysed by incubation for 5 minutes at room temperature in 1 mL of lysis buffer (0.864% ammonium chloride (Sigma-Aldrich) diluted in sterile deionized H2O) and then washed with complete Roswell Park Memorial Institute medium (cRPMI) (10% fetal calf serum, 0.1% beta-2-mercaptoethanol, 1% non-essential amino acids, 1% sodium pyruvate, and 1% pen-strep). OT-I T cells were enriched by magnetic activated cell sorting using the CD8a+ T Cell Isolation Kit (Miltenyi Biotec, Bergisch Gladbach, Germany) or using the EasySep™ Mouse CD8+ T Cell Isolation Kit (Stem Cell Technologies, Vancouver, BC, Canada). OT-I purity was verified (~80%–95%) using flow cytometry for TCR Vα2 and Vβ5.1 expression. 104−106 OT-I T cells were transferred by intraperitoneal injection into recipient mice. Flow cytometry Single-cell suspensions were prepared from intestinal sections by shaking diced tissue at 37°C for 20–30 minutes in Hank’s Balanced Salt Solution with 5 mM ethylenediaminetetraacetic acid and 1 mM DTT(dithiothreitol). Cell pellets were then passed through 70 um and 40 um filters. Ileal-draining mesenteric lymph nodes (mLN) and Peyer's Patches (PP) from the ileum as well as spleens were harvested and dissociated through 70 um filters, then washed with cRPMI. Cells were washed in fluorescence-activated cell sorting (FACS) buffer (1x PBS, 0.2% bovine serum antigen, 1 mM ethylenediaminetetraacetic acid), and incubated in Fc block (99.5% FACS Buffer, 0.5% normal rat IgG, 1 μg/mL 2.4G2) at 4°C for 15 minutes before staining. Cells were stained for cell death using either LIVE/DEAD Fixable Aqua Dead Cell marker (Invitrogen) or GhostDye Violet 510 Viability Dye (TONBO Biosciences, San Diego, CA, USA) in 1x PBS at 4°C for 15 minutes. Cells were washed after cell death staining and surface antibodies were added and stained at 4°C for 20–30 minutes. If intracellular staining for transcription factors was performed, cells were fixed using the eBioscience Foxp3 Transcription Factor Fixation/Permeabilization Concentrate and Diluent (ThermoFisher Scientific, Waltham, MA, USA) for 20 minutes at 4°C and then washed with FACS buffer. Cells were then stained for transcription factors in 1x eBioscience Permeabilization Buffer (ThermoFisher Scientific) at 4°C for 30 minutes. Cells were then washed in FACS buffer before acquisition. Cells were stained using the following fluorochrome-conjugated antibodies: CD19 PerCP-Cy5.5 (clone ID3), natural killer (NK)1.1 PerCP-Cy5.5 (clone PK136), CD4 BV650 (clone RM4-5), CD3 BV785 (clone 17A2), CD45.2 BV711 (clone 104), TCR Vb5.1,5.2 APC (clone MR9-4), TCR Va2 PE (clone B20.1), B220 PerCP-Cy5.5 (clone RA3-6B2), CD8a BV650 (53-6.7), CD4 BV711 (clone GK1.5), CD45.1 PE-Cy7 (clone A20), CD45.1 BV711 (clone A20), CD45.2 APC (clone 104), CD90.2 AF700 (clone 30-H12), CD8b fluorescein isothiocyanate (FITC) (clone YTS156.7.7), CD8a BV711 (clone 53-6.7), CD44 BV785 (clone IM7), CD90.2 BV785 (clone 30-H12), CD4 AF700 (clone RM4-5), PD-1BV605 (clone 29F.1A12), CXCR3 BV650 (clone CXCR3-173), KLRG1 BV711 (clone 2F1), IL-21R APC (clone 4A9), CD45.1 PE-Cy7 (clone A20), IL-21R PE-Cy5 (clone 4A9), CD45.2 PerCP-Cy5.5 (clone 104), CD45.1 BV510 (clone A20), CD11a PE (clone M17/4), CD8a BV421 (53-6.7) from Biolegend; CD90.1 APC-ef780 (clone HIS51), CD8a PE-Cy7 (clone 53-6.7), CD45.1 APC-ef780 (clone A20), CD8a APC-ef780 (clone 53-6.7), TCR Va2 Superbright 780 (clone B20.1), CD45.1 APC (clone A20), Gzm B ef450 (clone NGZB), LPAM-1 PE (clone DATK32), Tbet PE-Cy5 (clone 4B10), CD3e PE-Cy7 (clone 145-2C11), CD8b APC-ef780 (clone eBioH35-17.2), CD4 ef450 (clone GK1.5), Tbet PE-Cy7 (clone eBio4B10), CD45.2 ef450 (clone 104) from eBioscience; CD8a PE-cf594 (clone 53-6.7), CD8a BUV563 (clone 53-6.7), CD19 BUV395 (clone ID3), NK1.1 BUV395 (clone PK136), CD4 BUV805 (clone GK1.5), CD69 BUV737 (clone H1.2F3), CD11a BUV805 (clone 2D7), Ki-67 AF700 (clone B56), EpCAM BUV395 (clone G8.8), CD4 BUV496 (clone GK1.5), CD8a BUV805 (clone 53-6.7), Va2 BV605 (clone B20.1), CD44 PE-Cy5 (clone IM7), CD3 FITC (clone 145-2c11) from BD Biosciences; TCF1 AF488 (clone C63D9) from Cell Signaling. Endogenous and OT-I responses were measured by H2-Kb:SIINFEKL tetramer conjugated to PE or APC (NIH Tetramer Core) staining at room temperature for 20–30 minutes. Data were collected on a FACSCanto, LSRFortessa, or FACSymphony A3 Lite (BD Biosciences) and analyzed with FlowJo v10 software (Treestar, Ashland, OR, USA). Cells were downsampled to a consistent number depending on the tissue using Downsample plugin in FlowJo before UMAP. UMAP and X-Shift plugins were utilized in FlowJo for dimensionality reduction and unsupervised clustering, and X-Shift was visualized by Cluster Explorer. Immunofluorescence imaging Human ileocecal adenocarcinoma cells (HCT-8) (American Type Culture Collection, Manassas, VA, USA) were grown on coverslips in Dulbecco's modified eagle's medium (DMEM) supplemented with 10% cosmic calf serum (Thermo Fisher). A total of 200,000 purified oocysts were excysted and seeded on coverslips when HCT-8 cells were 80% confluent. After excystation at indicated time points after infection, cells were washed with PBS and successively fixed for 10 minutes with 4% paraformaldehyde followed by permeabilization for 10 minutes with 0.1% Triton X-100 (Sigma). Coverslips were blocked with 4% bovine serum albumin (Sigma) and antibodies were diluted in 1% bovine serum albumin solution. Rat monoclonal anti-HA (clone: 3F10; Sigma) was used as primary antibody and goat anti-rat polyclonal Alexa Fluor 594 (Thermo Fisher) as secondary along with VVL conjugated to FITC (Vector Labs, Newark, CA, USA). Host and parasite nuclei were stained with Hoechst 33342 (Thermo Fisher). Slides were imaged using a Leica Wide-field microscope. Multiphoton imaging GFP-expressing OT-I T cells were isolated from UBC-GFP mice that were interbred with OT-I mice and purified as previously described. IFN-γ−/− mice were injected intraperitoneally with 106 GFP+ OT-I cells 24 hours before infection with 5 × 104 C. parvum M2-OVA tdTomato parasites. At 5–7 dpi mice were orally gavaged with 200 μl 2 mg/mL loperamide hydrochloride (Sigma-Aldrich, Burlington, MA, USA) to minimize peristaltic movement for imaging. Mice were subsequently anesthetized by intraperitoneal injection with a xylazine/ketamine cocktail followed by retroorbital injection with 0.5 mg Hoechst 33342 (Invitrogen, Carlsbad, CA, USA) and 30 μl Qtracker 655 vascular label (Invitrogen). Mice were maintained under anesthesia at 37°C with vaporized isoflurane. The ileal loop was extracted through a small incision made in the abdomen of the mouse and a c.1 cm section dissected longitudinally to expose the luminal surface. The cut edges were cauterized to minimize blood loss. 20 μg/mL loperamide hydrochloride (Sigma-Aldrich) and 20 μg/mL indomethacin (Sigma-Aldrich) were applied topically to the luminal intestinal surface to further minimize peristaltic movement. Imaging was carried out on a Leica SP8 multiphoton microscope (Leica Microsystems, Wetzlar, Germany) with a 20X 1.0 NA water-dipping objective and equipped with a resonant scanner (8,000 kHz) and four external HyD detectors. The excitation wavelength of Chameleon Vision II Ti:Sapphire laser (Coherent, Saxonburg, PA, USA) was tuned to 900 nm. After imaging, mice were euthanized by CO2 asphyxiation. Images were analyzed with Imaris imaging software (Oxford Instruments, Abingdon, Oxfordshire, England). Statistics Statistical significance was calculated using unpaired t test with Welch’s correction for comparing groups of two, or analysis of variance followed by multiple comparisons for comparing groups of three or more. Statistical significance was calculated for area under the curve analyses using paired t test. Analyses were performed using GraphPad Prism v9 and 10.\",\n",
       " '11193216': 'Methods Establishment of clinical cohort Longitudinal cohort: We enrolled newly diagnosed TB patients at West China Hospital, Sichuan University. These patients received a standard treatment regimen comprising rifampin (RIF, 450 mg/day) or rifapentine (RFT, 600 mg/twice a week), isoniazid (INH, 300 mg/day), ethambutol (EMB, 750 mg/day), and pyrazinamide (PZA, 1500 mg/day) during the intensive phase, followed by RIF or RFT, INH, and EMB during the continuation phase. At enrollment, all patients were confirmed by experienced clinicians, based on clinical symptoms, microbiological or radiological evidence [16]. The exclusion criteria were as follows: diabetes mellitus, chronic liver diseases, chronic renal diseases, malignant tumor, autoimmune diseases, human immunodeficiency virus infection, or long-term use of traditional medicine. Since the start of antituberculosis treatment, all participants underwent weekly visits in the first month (1 W, 2 W, 3 W, 4 W) and biweekly visits in the second month (6 W, 8 W). According to whether develops ATB-DILI, patients were divided into ATB-DILI group and antituberculosis control (ATB-Ctrl) group. All ATB-DILI subjects were confirmed based on the following criteria: (1) ALT ≥ 3×upper limit of normal (ULN) with hepatitis symptoms such as nausea, vomiting, and jaundice, or (2) ALT ≥ 5×ULN with or without clinical symptoms, or (3) aspartate aminotransferase (AST), alkaline phosphatase (ALP), and total bilirubin (TBIL) elevated simultaneously and at least one ≥ 2×ULN [17, 18]. Cross-sectional cohort A total of 67 cases with ATB-DILI and 61 controls without any adverse events were enrolled in the cross-sectional cohort from October 2020 to December 2020, that were cross-sectional drug-induced liver injury (CS-DILI) group and cross-sectional control (CS-Ctrl) group. The exclusion criteria were the same as the longitudinal cohort. Plasma samples were collected within 7 days after ATB-DILI onset. Healthy control cohort A total of 50 healthy controls were randomly recruited from Physical Examination Center of West China Hospital of Sichuan University from January 2021 to June 2021. Ethics approval and consent to participate This study was approved by the Ethnics Committee of West China Hospital of Sichuan University (reference no. 829, 2019) following the World Medical Association’s Declaration of Helsinki. Written informed content was obtained from patients involved in the cohort. Metabolomic profiling Untargeted metabolomics analysis The EDTA anticoagulated plasma samples were collected from all participants for metabolomic analysis at Discovery HD4™ Platform (supplementary methods) [19]. Targeted metabolomics analysis Quantitation of 5 fatty acids (FAs) and 6 bile acids (BAs) were performed with ultra performance liquid chromatography/tandem mass spectrometry (UPLC-MS/MS) (supplementary methods, supplementary Tables 1, 2, 3, 4, 5, 6). Statistical analysis Data analysis was performed using SPSS 20.0 or GraphPad Prism 8.0 or R (version 4.1.0). Two-way repeated ANOVA was performed for repeated measurement data at multiple timepoints of the longitudinal cohort and adjusted p values were calculated using False Discovery Rate (FDR). Volcano plot, principal component analysis (PCA), partial least squares discriminant analysis (PLS-DA), weighted co-expression network analysis, dynamic feature cluster analysis and random forest classification model were performed by using R packages (ggplot2, mixOmics, WGCNA, Mfuzz, and Random Forest). Enrichment and pathway analysis of different metabolites were performed on the web of MetaboAnalyst 5.0 (https://www.metaboanalyst.ca). Multiple group comparisons of cross-sectional cohorts were performed by Kruskal Wallis test.',\n",
       " '11193124': \"Methods Populations We enrolled a total of 46 female patients: 14 prepubertal controls (mean age 7.78 ± 1.01); 13 pubertal controls (mean age 13.49 ± 1.48) and 19 CPPs with a diagnosis of idiopathic CPP (mean age 7.36 ± 0.85) who were referred to Paediatric Endocrinology Units of the University of Campania “Luigi Vanvitelli,” Naples. Prepubertal and pubertal controls were selected from patients who came to the auxology outpatient clinic on suspicion of poor growth, which was subsequently ruled out, and were therefore considered healthy. Central hypothalamic–pituitary–gonadal activation was defined by measuring luteinizing hormone (LH) levels at baseline (> 0.3 mUI/mL) or a peak-LH > 5mUI/mL after GnRH stimulation test (0.1 mg Relefact LH-Releasing Hormone, Sanofi-Aventis, Frankfurt am Main, Germany). In all CPP patients, MKRN3 mutations were excluded. All patients underwent brain MRI showing normal hypothalamic-pituitary anatomy. Samples collection and genetic analysis were performed in the Department of Woman, Child, General, and Specialized Surgery of the University of Campania “Luigi Vanvitelli” Naples. Laboratory studies All blood samples were drawn at 8:00 a.m. from an antecubital vein, clotted, centrifuged, and serum was stored at − 20 °C until analyses were performed. FSH and LH concentrations were determined by immunochemiluminometric assay (ICMA) with inter-assay coefficient of variation less than 5%. Serum 17-β-estradiol was measured by using competitive chemiluminescent immunoassay using coated magnetic particles with inter-assay coefficient of variation less than 5%. Genetic studies DNA was extracted from peripheral blood samples using the Wizard Genomic DNA Purification Kit (Promega, Madison, WI, USA) following the manufacturer's instructions. By Sanger sequencing, all patients with precocious puberty were screened for the MKRN3 gene to exclude significant mutations in the coding region. Samples preparation and genome-wide DNA methylation analysis Genomic DNA was extracted from peripheral blood leukocytes using standard procedures. DNA quality and quantity were assessed by NanoDrop (Thermo Fisher Scientific) and electrophoresis on 1% agarose gel. Microarray analyses were performed by Genomix4life S.R.L. (Baronissi, Salerno, Italy). The DNA concentration in each sample was assayed with a Qubit fluorometer. Bisulfite converted DNA (250 ng) was used for the analysis of whole-genome methylation using the MethylationEPIC BeadChip (Illumina, San Diego, CA, USA). In brief, bisulfite converted DNA was whole-genome amplified for 20 h followed by end-point fragmentation. Fragmented DNA was precipitated, denatured and hybridized to the BeadChips for 20 h at 48 °C. The BeadChips were washed and the hybridized primers were extended and labeled before scanning the BeadChips using the Illumina iScan system. Data analysis was performed as described in Casarotto et al. 2022 [72]. CpGs beta-values were obtained using ChAMP [73] on R (v. 4.2.0). Values of blood cell composition were obtained from the patients and missing values were imputed with a lasso linear regression model. Singular value decomposition was not able to detect the blood cell composition as factor that accounts for a substantial fraction of variation, for this reason, no corrections were performed. Only CpGs with a detection p-value < 0.01 have been considered for further analysis. CpGs were considered differentially methylated if the beta difference between the groups (Δβ) was more than 10% (|Δβ|> 0.1) and FDR values < 0.05. Further analyses were performed on the CpGs located within promoter regions, which are defined as CpGs annotated in the TSS1500, TSS200, 1 Exone and 5’UTR. DMR analysis was performed with ChAMP using the BumpHunter model with default parameters. DMR with FDR values > 0.05 and |Δβ|< 0.05 were excluded. The canonical pathways were generated through the use of QIAGEN IPA (QIAGEN Inc., https://digitalinsights.qiagen.com/IPA) and only the ones with -log(p-value) > 1.3 were considered significant. A list of regions with high-affinity EREs in the human genome was downloaded from the “MOUSE AND HUMAN ERE DATABASES” (http://mapageweb.umontreal.ca/maders/eredatabase/) [74] and intersected with the genomic coordinates of the differentially methylated positions (DMPs). Hypergeometric test with p-value < 0.05 was used to assess the enrichment of ERE in our DMPs. Transcription factors were found using “epigenetic Landscape In Silico deletion Analysis” (LISA—http://lisa.cistrome.org/) [75] with default parameters. All the statistics, pie charts and heatmaps were obtained using R.\",\n",
       " '11193042': 'MethodsWe utilized univariable and multivariable Mendelian randomization (MR) analyses to investigate associations and mediation effects. Additionally, the multivariable MR analyses incorporated cardiometabolic risk factors, allowing us to account for potential confounders.',\n",
       " '11193000': 'Methods Study design Figure 1 illustrates the study design diagram, which comprises three stages of analyses aimed at thoroughly examining the causal relationships between DM, glycemic traits, and vascular and valvular calcifications.Fig. 1The study\\'s overall workflow. Three stages of analyses designed to meticulously explore the mediating role of calcium homeostasis in the association between diabetes mellitus, glycemic traits, and vascular and valvular calcifications. The detailed description can be found in the main text Stage 1: We conducted two-sample MR analyses to assess the total effects of DM and glycemic traits on both types of calcifications, denoted as β0. Additionally, we performed reverse MR analyses to investigate potential reverse causality from these calcifications back to DM and glycemic traits. Stage 2: Two-sample two-step MR analyses were carried out to identify potential intermediaries in the pathway linking DM and glycemic traits to vascular and valvular calcifications. We quantified the proportion of the mediating effect of DM and glycemic traits on these calcifications through calcium homeostasis, calculated as (β1 * β2)/β0 [13]. Stage 3: We performed multivariable MR analyses to evaluate the independent associations between DM, glycemic traits, and both types of calcifications, accounting for the potential influence of cardiometabolic risk factors such as BMI, smoking initiation, systolic blood pressure, and diastolic blood pressure. Briefly, our study adopted a comprehensive MR approach, utilizing two-sample, two-step, and multivariable analyses. We used SNPs from genome-wide association studies (GWAS) as instrumental variables (IVs) to estimate causal relationships between DM, glycemic traits, and the risk of vascular and valvular calcifications. For MR analyses to yield valid results, three critical assumptions must be satisfied: (1) the SNPs are associated with the exposure; (2) the SNPs are independent of confounders of the exposure-outcome relationship (the independence assumption); and (3) the SNPs affect the outcome only through the exposure. Informed consent and ethics approval were obtained for each of the original studies [13]. Exposure GWAS Initially, we selected type 2 diabetes mellitus (T2DM) from a GWAS dataset comprising 62,892 diabetic patients and 596,424 controls of European ancestry. The diagnosis of T2DM was established using various methods, as described in the primary GWAS publications [14]. Additionally, we selected variants associated with key glycemic traits, including HbA1c levels [15], FI [16], and FG [16], from corresponding GWAS publications. Mediator GWAS Summary statistics for calcium homeostasis, including serum calcium, 25-hydroxyvitamin D (25OHD), and fibroblast growth factor 23 (FGF23), were extracted from three distinct GWASs. For serum calcium, the data were obtained from 400,792 samples within the UK Biobank [17]. The genetic predictors for serum 25OHD were sourced from the largest available GWAS, involving 496,946 participants [18]. Additionally, the summary statistics for FGF23 were obtained from a study comprising 21,758 samples [19]. Outcome GWAS Data on CAC were obtained from an open GWAS published by Kavousi M et al. (2023), encompassing a total of 28,655 individuals [20]. The source for this data is available at CVD HugeAmp (https://cvd.hugeamp.org). Additionally, data on CAVS were extracted from FinnGen (https://www.finngen.fi/en), which included 9153 cases and 402,311 controls. Further details can be found on the FinnGen website. Cardiometabolic risk factors GWAS Summary statistics for cardiometabolic risk factors were extracted from three distinct GWASs. These factors include BMI, with a sample size of 532,396 [21]; smoking initiation, with 607,291 participants [22]; and both systolic and diastolic blood pressure, each with 810,865 participants [23]. All summary data utilized in this study are publicly accessible, with detailed information presented in Supplementary Tables S1. Selection of IVs We identified SNPs as IVs using three criteria: significant genome-wide association (p < 5e−8), confirmed independent heritability (r2 < 0.001), and the absence of linkage disequilibrium, assessed over a dense 10,000 kb window [24]. The evaluation of weak associations between SNPs and exposures was based on the formula F = R2 × (N − 2)/(1 − R2). R2 was estimated via employing the equation: R2 = 2 × MAF × (1 − MAF) × β2, where β is the allele effect value and MAF is the effect allele frequency. SNPs with an F-statistic greater than 10 were considered reliable IVs due to their ability to reduce bias associated with weak instruments in MR analyses [25]. Statistical analysis All analyses were conducted using R (version 4.3.2). For MR analyses, we employed the \"TwoSampleMR\" package and identified outliers with the \"MR-PRESSO\" package [26]. We applied four different MR methods in the univariable MR analyses: inverse variance weighted (IVW), MR-Egger regression, weighted median, and weighted mode. The majority of our statistical evaluations utilized the IVW method, which is recognized for its robustness in identifying causal relationships in univariable MR analyses. For multivariable MR analyses, we used the \"MVMR\" and \"MendelianRandomization\" packages [27]. Multivariable MR was employed to investigate the direct causal relationships between T2DM, glycemic traits, and vascular and valvular calcifications. We also applied several multivariable MR techniques—including multivariable MR-IVW, multivariable MR-median, and the least absolute shrinkage and selection operator (LASSO)—to assess direct causality, adjusting for cardiometabolic risk factors. If at least one of these three methods yields a significant result, it is considered that the causal relationship still exists even after multivariable adjustment [28]. To enhance the reliability of our experimental results and comply with the assumptions of MR, we executed an extensive range of sensitivity analyses. These analyses included Cochran’s Q test, MR Egger regression, and the leave-one-out method. Cochran’s Q test, our primary tool for identifying heterogeneity, indicates significant heterogeneity when its p value is below 0.05. Upon detecting significant heterogeneity, we employed a random-effects IVW model; otherwise, a fixed-effects IVW model was used. Additionally, the intercept from the MR-Egger regression provides an estimate of the extent of directional pleiotropy, with a p value less than 0.05 suggesting horizontal pleiotropy. The leave-one-out analysis was performed to evaluate whether the significant results were driven by a single SNP [29]. We followed the guidelines set forth by the Strengthening the Reporting of Observational Studies in Epidemiology using Mendelian Randomization (STROBE-MR) checklist [30]. p values reaching nominal significance (p < 0.05) were considered to have nominal potential causal effects. To account for multiple exposures, we adjusted the significance threshold using the Bonferroni correction, resulting in a p-value of < 0.0125 (= 0.05/4 exposures) considered statistically significant [31].',\n",
       " '11192909': 'Methods Study Design and Participants The EARLY (Exploration of Treatment of Newly Diagnosed Overweight/Obese Type 2 Diabetes Mellitus) study is a randomized, open-label, active parallel-controlled clinical trial. The study protocol (Supplement 1) was approved by the ethics committees of all participating centers (Beijing Hospital; the Third Affiliated Hospital of Jinzhou Medical University; Nanyang Central Hospital; Henan Provincial People’s Hospital; the First Affiliated Hospital of Zhengzhou University; Hebei Provincial People’s Hospital; the Second Hospital of Hebei Medical University; Sir Run Run Hospital, Nanjing Medical University; and the First Affiliated Hospital of Soochow University). The trial followed the International Conference on Harmonization Guidelines for Good Clinical Practice and the Declaration of Helsinki.21 All patients provided written informed consent. This report adhered to the Consolidated Standards of Reporting Trials (CONSORT) reporting guideline. Eligibility Criteria and Recruitment We recruited adults with newly diagnosed (within 1 year) type 2 diabetes who had not used antidiabetic agents in the past 3 months, aged 18 to 65 years, with a BMI of 24 or more and an HbA1c level of 7% to 9%. The recruitment was conducted concurrently at 9 hospitals across China (eTable 1 in Supplement 2) from November 13, 2020, to December 29, 2022. We excluded participants who had used weight-loss drugs or products within the past 3 months before enrollment, as well as pregnant or breastfeeding women (Supplement 1). Randomization and Masking Randomization was conducted using an interactive web response system. The randomization list of participants was generated by the stratified blocked randomization method using SAS software, version 9.4 (SAS Institute Inc), in which stratification was based on the center (block size of 9). Within each stratum, participants were randomized using a block randomization method, with a block size of 9, in a ratio of 1:1:1 to receive either metformin, empagliflozin, or 5:2 MR. Both the lists for participant and treatment allocation were inputted into the interactive web response system. At the study site, participants were administered treatment based on the randomization code and the corresponding treatment group obtained from the interactive web response system. Due to the nature of the intervention, blinding of participants and investigators was not feasible in this study. However, during the data analysis, the statisticians remained blinded to the study groupings. Interventions The treatment period lasted for 16 weeks, followed by an 8-week follow-up (eFigure 1 in Supplement 2). All participants received dietary and exercise guidance as well as general diabetes education from nutritionists and research physicians in accordance with China Guideline22 (eTable 2 in Supplement 2) every 4 weeks. 5:2 MR Approach Patients in the 5:2 MR group consumed low-energy MR product A (Kang zhijun, Beijing MetabolicControl Technology Co Ltd; eTable 3 in Supplement 2). The 5:2 MR approach (eFigure 2 in Supplement 2) means that, within 1 week, there were 2 nonconsecutive days on which meals are replaced. On these 2 days, participants were required to consume 1 serving of Kang zhijun A instead of all 3 regular meals, with a daily energy intake of 500 kcal for women and 600 kcal for men. On the remaining 5 days, participants chose their own breakfast and lunch but had 1 serving of Kang zhijun B for dinner and were encouraged to monitor their calorie intake. Throughout the 16 weeks, dietary intake was recorded in a diary. Administration of Metformin or Empagliflozin Patients took metformin (Shanghai Bristol-Myers Squibb), 0.5 g, twice a day. If the initial drug dosage was well tolerated, it was escalated to 2 g per day. Empagliflozin (Shanghai Boehringer Ingelheim), 10 mg, was administered once a day. During the study, patients were instructed to promptly contact the research center’s physician in case of severe hypoglycemia. Outcomes The primary outcome was the change in HbA1c level from baseline to 16 weeks. Secondary outcomes included changes in weight (measured by InBody 770 [InBody]), BMI, waist circumference, hip circumference, waist to hip ratio, systolic and diastolic blood pressure, fasting plasma glucose (FPG) level, fasting insulin level, fasting C-peptide level, homeostasis model assessment of insulin resistance (HOMA-IR = FPG [mmol/L] × fasting insulin [μU/mL]/22.5), lipid profiles (total cholesterol, triglycerides, high-density lipoprotein cholesterol [HDL-C], and low-density lipoprotein cholesterollevels), and uric acid levels. The primary and secondary outcomes were reevaluated at the end of 8-week follow-up (week 24). Adverse events were assessed throughout the study. Adverse events of particular interest included gastrointestinal reactions, urinary tract and reproductive system infections, hypoglycemia, and hyperglycemia. Laboratory testing was conducted at a central laboratory. Statistical Analysis The sample size calculation was based on the SD of the change in HbA1c level from a previous study,23 with a 2-sided α of .05, β of 0.2, a minimum detectable between-group difference of 0.1%, and an anticipated SD of 0.2% based on pilot data analysis and a multiple pairwise comparison test using the Tukey-Kramer test. It was computed that each group required 108 participants using PASS 15 software (NCSS). Accounting for an expected 20% dropout rate, each group required 135 patients. The primary outcome was analyzed following the intention-to-treat principle in the full analysis set, which included all randomized participants who received at least 1 dose of drugs or 5:2 MR. The safety outcome was analyzed in the safety analysis set, defined as participants randomized who received at least 1 dose of drugs or 5:2 MR and had safety assessment data collected at least once after the baseline. The primary outcome was analyzed using the analysis of covariance model, which calculated the least-squares mean (LSM) and 95% CI to compare changes in HbA1c level and key secondary outcomes among the 3 groups. The model adjusted for sex, age, height, weight, family history of diabetes and hypertension, physical activity, smoking, alcohol consumption, and baseline HbA1c. Multiple imputation was used for missing values in the primary and key secondary outcomes (eMethods in Supplement 2). Post hoc subgroup analyses were conducted to explore the potential effect of baseline differences on HbA1c and weight loss. Statistical analyses were performed using SPSS, version 24.0 software (SPSS Inc). All P values were from 2-sided tests and results were deemed statistically significant at P < .05.',\n",
       " '11192648': 'MethodsUntargeted metabolomic and transcriptomic analyses were performed in LGG monocolonized germ-free mice fed trp-free or -sufficient diets. LGG-derived metabolites were profiled in vitro under anaerobic and aerobic conditions. Multiomic correlations using a newly developed algorithm discovered novel metabolites tightly linked to tight junction and cell differentiation genes whose abundances were regulated by LGG and dietary trp. Barrier-modulation by these metabolites were functionally tested in Caco2 cells, mouse enteroids, and dextran sulfate sodium experimental colitis. The contribution of these metabolites to barrier protection is delineated at specific tight junction proteins and enterocyte-promoting factors with gain and loss of function approaches.',\n",
       " '11192395': \"Materials and Methods Animal procedures and protocols were in accordance with the Institutional Animal Care and Use Committee, Rutgers New Jersey Medical School. Experimental Designs Gnotobiotic study GF mice (C57BL/6, Charles River Laboratories) were fed either trp-free or trp-sufficient diets.30 The trp-sufficient diet, containing 0.2% trp, had otherwise exactly the same composition as the trp-free. Mice were acclimatized for 2 weeks then divided into 4 groups, 2 of which were fed trp-sufficient, and the other 2 fed trp-free diets for another 2 weeks (Figure 1A). At Week 0, mice in trp-free and trp-sufficient conditions were divided into cohorts that were gavaged with either LGG (108 CFU/mL in 200 μL PBS) or sterile PBS. Mice were continuously fed with trp-free and trp-sufficient diets until sacrifice 21 days after inoculation. Fecal pellets were collected before and after inoculation. Feces, intestine, serum, and liver were collected at sacrifice. LGG administration and GF monitoring LGG was cultured (37°C) in MRS broth and centrifuged (1200 × g, 10 minutes) to pellet before gavaging to mice. Body weight and feeding rate were monitored. GF status before inoculation was validated by a commercial vendor (IDEXX) and constantly monitored in-house by fecal DNA analysis of bacterial 16S rRNA. After inoculation, qPCR for fecal DNA with 16S rRNA and LGG-specific primers were used to confirm colonization. Same engraftment efficiency of LGG was found in mice fed with trp+ or trp- diets. LGG qPCR primers are: forward, 5’-CGCCCTTAACAGCAGTCTTC-3’ and reverse, 5’-GCCCTCCGTATGCTTAAACC-3’. The 16S rRNA qPCR primers are forward 5'-AGAGTTTGATCCTGGCTCAG-3’, and reverse, 5'- CTTGTGCGGGCCCCCGTCAATTC-3'. Intestinal perfusion Procedures followed previous work.64 Mice were perfused with live LGG (3.3X108 CFU/mL) + Inulin (3.3 mg/mL), or with Krebs-Ringer Buffer + inulin. After 4 hours of perfusion, the intestines were collected for RNA sequencing. DSS-induced colitis and metabolite treatment C57BL/6 mice were intraperitoneally injected with PBS or 30 mg/kg MNA every other day for 2 weeks then gavaged with 100 μg/kg chicken ovalbumin (Figure 7A). About 24 hours postgavage, plasma ovalbumin levels were quantified using an enzyme-linked immunosorbent assay kit (abx555335, Abbexa, UK) to determine gut permeability in homeostasis. For DSS experiments, the mice were grouped into PBS, MNA, PBS+DSS, and MNA+DSS cohorts and subjected to 2.5% DSS treatment for 11 days, followed by a 5-day recovery period. MNA was injected every other day after initiation of DSS treatment. After sacrifice, tissues were collected for immunohistochemistry and immunofluorescence staining, qPCR, and Western blot analysis, whereas fecal samples from Day 11 were analyzed for lipocalin 2 levels using enzyme-linked immunosorbent assay (ab199083). Caco-2 transepithelial electrical resistance and paracellular flux Cells (20,000 per well) were suspended in Dulbecco's modified eagle medium media (20% fetal bovine serum, 1% Primocin) in 96-well transplates (MilliporeSigma PSHT004R5) and polarized for 21 days. TEER was measured with an EVOM system using an STX100 electrode (World Precision Instruments). Indole and NAM-derived metabolites were added apically or basolaterally (Table 1). To induce barrier disruption and assess efficacy of metabolites to repair disruption, Caco-2 monolayers were treated with 0.15% vol/vol C difficile cell-free supernatant. Paracellular flux was assessed by introducing FITC-dextran (5 mg/mL) to apical chambers, and FITC fluorescence in 10 μL samples from the basolateral chamber was measured in a black 96-well plate (Costar 3915) using a Glomax (Promega, WI) plate reader (excitation: 490 nm, emission: 520 nm).Table 1Metabolite Dosing InformationMetabolitesDoseApplicationKynurenine10 mMCaco-2 luminal/basolateral treatmentNicotinamide riboside10 mMNicotinamide mononucleotide10 mMNAD+10 mMNicotinamide10 mMMethylnicotinamide10 mMIndole-3-acetamide500 μMCaco2 basolateral treatmentIndole-3-carboxyaldehyde500 μMIndole-3-lactic acid500 μMIndole-3-propionic acid500 μMIndole-3-acrylic acid250 μMIndole-3-carbinol250 μMIndole-3-acetamide250 μMCaco2 luminal treatmentIndole-3-acetic acid250 μMIndole-3-acetonitrile250 μMIndole-3-carboxylic acid250 μMIndole-3-carboxyaldehyde100 μMIndole-3-lactic acid50 μMIndole-3-propionic acid100 μMIndole-3-acrylic acid250 μMIndole-3-carbinol100 μMIndole-3-propionic acid0.1 mMEnteroidsIndole-3- acetamide200 μMMethylnicotinamide87 nMCarnosine20 mM Intestinal organoid treatment with LPS and metabolites Enteroids were isolated as previously described.48 To grow wild-type and Ocln KO enteroids, Oclnfl/fl and Oclnfl/fl-Vil/CREERT2 mice were injected (intraperitoneally) with tamoxifen and sacrificed 7 days after the injection to establish organoids. Day 0 was when enteroids were seeded. On Day 3, media were replaced with LPS (1 μg/mL) with or without metabolites (indole-3-propionic acid [0.1 mM],65 indole-3-acetamide [200 μM], MNA [12 ng/mL], carnosine [20 mM]), incubated for 48 hours, and analyzed. Bulk RNA sequencing Before analyses, the high quality of extracted ileal RNA from all samples was confirmed (RIN, 8.6 ± 0.56). Total RNA was subjected to poly(A) selection using oligo(dT)25 magnetic beads (New England Biolabs). Libraries were constructed using Next (New England Biolabs) Ultra II RNA library preparation with sample purification beads and Next Multiplex Oligos. Poly(A) selection and library quality were assessed using TapeStation 2200 (Agilent) and libraries quantified (Qubit 4.0 fluorometer, ThermoFisher). The prepared libraries were sequenced (Illumina NextSeq 500) then analyzed by CLC Genomics Workbench 20.0.4. Metabolite extraction and liquid chromatography–mass spectrometry analysis Feces and liver metabolites were extracted using the reported procedures.29 Serum metabolites were extracted in methanol (ratio of serum:methanol, 1:4) at –20°C for 20 minutes, and centrifuged at 17,000 × g for 2 minutes. Supernatants were stored on ice. The pellet was further extracted in acetonitrile:methanol:water (400:400:200), for 10 minutes on ice, centrifuged at 17,000 × g for 10 minutes, and the supernatant combined with the first extraction. Liquid chromatography–mass spectrometry analysis was performed as previously described.29 The metabolite data were analyzed in MAVEN (Metabolic Analysis and Visualization Engine). Compound identification used the accurate mass and the retention time learned from an in-house chemical library. MetaboAnalyst 5.0 was then used for further statistical and functional interpretation of metabolomics data. Finally, principal component analysis and variables important in the projection (cutoff 1.229) analyses were used to identify those metabolites substantially driving differences between 2 treatments. RT-qPCR analysis Primers used include: β-actin (Actb), forward 5’-TTG TTA CCA ACT GGG ACG ACA TGG-3’, reverse 5’- CTG GGG TGT TGA AGG TCT CAA ACA-3’; ZO-1 (Tjp1), forward 5’-GGG AGG GTC AAA TGA AGA CA-3’, reverse 5’-GGC ATT CCT GCT GGT TAC AT-3’; Ocln, forward 5’-ATT CCA TCA GTT TCC TAT CT-3’, reverse 5’-ACC AGG ACC TTT CTT GAC-3’; Lyz1, forward 5’-ATG GCT ACC GTG CT TCA AG-3’, reverse 5’- CGG TCT CCA CGG TTG TAG TT-3’; Tff3, forward 5’- TAA TGC TGT TGG TGF TCC TG-3’, reverse 5’- CAG CCA CGG TTG TTA CAC TG-3’, Dll4, forward 5’-AGC TGG AAG TGG ACT GTG GT-3’, reverse 5’-TAG AGT CCC TGG GAG AGC AA-3’; Rnf43, forward 5’-CCG GGT CAT TTC GTG CCT C-3’, reverse 5’-CCT GGT TCC TGG TAA GAT GGA-3’; Lgr5, forward 5’- TGA GCG GGA CCT TGA AGA TT-3’, reverse 5’ AGG TGC TCA CAG GGC TTG AA-3; Notch1, forward 5’- ATG TCG ATG TTC GAG GAC CAG-3’, reverse 5’ TCA CTG TTG CCT GTC TCA AG-3’; IL-6 forward 5’- ACC ACG AGG ATC AGT ACG AA-3’, reverse 5’- TGT TCFG CAT AAG GGC TCT GT-3’;Tnf- α forward 5’- GAT CTC AAA GAC AAC CAA CAT GTG-3’, reverse 5’- CTC CAG CTG GAA GAC TCC AAB CAG-3’. All samples were normalized to β actin. Permeability and immunohistochemistry Permeability and immunohistochemistry of enteroids were described previously.48 For colon and ileal tissues, the sections (5 μm) underwent antigen retrieval in 0.1 M citrate acid buffer (pH 6) at subboiling temperature for 20 minutes. Following blocking with PBS containing 0.1% Triton X-100, 2% normal serum, and 2% bovine serum albumin for 1 hour at room temperature, slides were incubated with primary antibodies overnight at 4°C. After 3 PBS washes, slides were incubated with fluorescent dye-conjugated secondary antibodies for 1 hour at room temperature, followed by DAPI nuclear counterstaining for 10 minutes. Mounted (Prolong Gold Antifade Mountant) slides were imaged at 10X magnification with 3 representative sections per mouse using Nikon TE2000. Quantitative analysis was performed using Image J. Primary antibodies were diluted in blocking buffer. Primary antibodies were rabbit monoclonal OCLN, mouse monoclonal ZO-1 and TFF3 (AB216327, 1A12 and 14475882, respectively, ThermoFisher) and rabbit polyclonal LYZ1 (PU024-5UP, Biogenex). METRCA Correlation analysis between metabolome and transcriptome used our newly developed R script.30 Transcriptome data were filtered and only differentially expressed genes (fold change >1.5, FDR-adjusted P < .05) were considered. Pairwise Pearson correlation coefficients were calculated between genes and metabolites, and their statistical significance was obtained via the cor.test function in R. Metabolite-gene pairs with FDR-adjusted P < .05 were significantly associated. Heatmap was generated by pheatmap package in R. Genes were clustered by hierarchical clustering. Statistical analysis All results are expressed as mean ± standard error of the mean. Significance was determined by 1-way or 2-way analysis of variance, Student t test, and Tukey post hoc test. Animal numbers were described in each figure panel with individual data points plotted in the graphs.\",\n",
       " '11192373': 'Materials and MethodsPatients with non‐small cell lung cancer who received nivolumab, pembrolizumab, or atezolizumab monotherapy were eligible. Blood samples were collected before and 7–21, and 22–42 days after treatment, and changes in the number and characteristics of neutrophils and lymphocytes in the peripheral blood were analyzed. The efficacy of ICIs was evaluated based on the best overall response up to 3 months after treatment. The patients were divided into the following groups: patients with non‐progressive disease (non‐PD) and those with progressive disease (PD). Blood immunological factors related to treatment response were analyzed.',\n",
       " '11192175': \"2 METHODS 2.1 Patient selection, sample collection, and measurements This single‐center prospective study at Mie University Hospital involved patients with unresectable advanced or recurrent NSCLC at stage III or IV. The patients were 20 years or older, had a performance status (PS) of 0–1, and were scheduled to receive one of the following ICIs as a monotherapy: nivolumab (OPDIVO I.V. infusion; Ono Pharmaceutical Co., Ltd., Osaka, Japan), pembrolizumab (KEYTRUDA injection; Merck & Co., Inc., Rahway, NJ), or atezolizumab (TECENTRIQ for intravenous infusion; Chugai Pharmaceutical Co., Ltd., Tokyo, Japan). Patients were excluded if they required systemic treatment with steroids (prednisolone equivalent >10 mg/day) or other immunosuppressive agents within 14 days of enrollment or had concomitant autoimmune diseases. Cases in which treatment was discontinued within 2 weeks of ICI administration were also excluded. The PD‐L1 expression level was evaluated using the tumor proportion score, measured using PD‐L1 IHC 22C3 pharmDx, Dako (Agilent Technologies Japan, Ltd., Tokyo, Japan). Residual blood samples from routine medical examinations were used in the assays. The samples were collected before ICI administration and 7–21 and 22–42 days after administration. Before the administration of ICIs, lymphocyte, neutrophil, CD3− CD56+ cell, CD3− CD56+ IFN‐γ+ cell, CD3+ CD8+ CD28+ T cell, and CD3+ CD8+ IFN‐γ+ T cell counts were measured. On Days 7–21 and 22–42, after treatment with ICIs, lymphocyte, neutrophil, CD3− CD56+ cell, CD3− CD56+ IFN‐γ+ cell, CD3+ CD8+ CD28+ T cell, and CD3+CD8+IFN‐γ+T cell counts, and drug binding to CD3+ CD8+ T cells were determined. The primary endpoint was non‐progression disease (non‐PD) for 3 months. This study was approved by the Ethics Committee of Mie University (approval no. 3062), and written informed consent was obtained from each participant between November 2017 and August 2020. The target number of participants was set at 20 for each of the non‐PD and PD groups, with a ratio of 1:1 for each group and 10% would meet the exclusion criteria, for a total of 50 participants. 2.2 Assessment of activated T cells To activate lymphocytes, PBMCs isolated from patient blood samples were incubated with phorbol 12‐myristate 13‐acetate and a calcium ionophore, A23187 (Sigma Chemicals, St. Louis, MO), for 5 h as described previously. 22 Cell pellets were incubated for 15 min with PE‐Cy7‐labeled anti‐CD3 antibodies (Beckman Coulter, Tokyo, Japan); PE‐labeled anti‐CD4, anti‐CD8, or anti‐CD56 antibodies (BioLegend, Tokyo, Japan); and APC‐labeled anti‐CD28 (IL‐2R) antibodies (BioLegend). FITC‐labeled nivolumab, pembrolizumab, and atezolizumab were prepared using the Mix‐n‐Stain™ CFTM488A antibody labeling kit (FUJIFILM Wako Pure Chemical Corporation, Osaka, Japan), and FTIC‐labeled rituximab was also prepared using the kit and used as the isotype control. The cell pellets were treated with a cell permeation reagent for 20 min and reacted with Pacific blue‐labeled anti‐IFN‐γ antibodies (BioLegend). After rinsing, NK+ and CD8+ T cells were fractionated from CD3− and CD56+ gates and CD3+ and CD8+ gates, respectively. The expression level of CD28+ cells in CD8+ T cells and IFN‐γ in CD8+ T cells was also measured. Flow cytometry was performed using a FACS Canto II flow cytometer (BD Biosciences) and FlowJo 7.6.5 software (BD Biosciences). Flow cytometric quantification of lymphocyte subsets was performed. 2.3 Pharmacodynamic analysis and statistical analysis This study evaluated the pharmacodynamic changes of immune biomarkers in blood NK and CD8+ T cells and their association with response to therapy with ICs at 3 months (Response Evaluation Criteria in Solid Tumors criteria version 1.1). Patients were divided into PD and non‐PD groups 3 months after initiating ICIs. Hematological immune biomarkers were compared between the groups using the non‐parametric Mann–Whitney U‐test for numerical variables and Fisher's exact test for categorical variables. All tests of comparison between the two groups in this study showed the results of the exact p‐value, and the results of the approximate p‐value were not used. Receiver operating characteristic (ROC) curve analysis was performed to estimate the sensitivity, specificity, accuracy, and cutoff values of several hematological immune biomarkers and their association with the primary endpoint. Statistical analyses were performed using GraphPad Prism (version 9.3.1; GraphPad Software, San Diego, CA). Two‐sided p‐values were reported, and the results were considered significant at p < 0.05.\",\n",
       " '11192171': 'Materials and methods Chemicals and reagents Notoginsenoside R1 (NGR1, BP1010, C47H80O18, purity ≥98%, CAS No 80418-24-2, MW: 933.13 Da) was purchased from Chengdu Purifa Technology Development Co. Ltd (Chengdu, China). Dextran sulfate sodium salt (DSS, 0216011010, MW: 36 kDa–50 kDa) was purchased from MP Biomedicals (Shanghai, China). Salicylazosulfapyridine (SASP, S0883, C18H14N4O5S, CAS No 599-79-1, MW: 398.39 Da) and FITC-dextran (FD4, CAS No 60842-46-8) was purchased from Sigma-Aldrich (Darmstadt, Germany). ICG-001 (T6113, C33H32N4O4, purity ≥ 98%, CAS No 780757-88-2, MW: 548.64 Da) was acquired from TOPSCIENCE (Shanghai, China). Water-DEPC treated (693520) and DMSO (D8418) were obtained from MilliporeSigma (Burlington, MA, USA). Cell culture NCM460 human intestinal epithelial cells and CT26 murine colon carcinoma cells were obtained from the American Type Culture Collection (ATCC, Manassas, VA, USA). NCM460 and CT26 cells were cultured in Roswell Park Memorial Institute (RPMI)-1640 culture medium (11875085, Gibco, NY, USA) supplemented with 10% fetal bovine serum (10099158, Gibco, NY, USA). The culture conditions included a humidified atmosphere containing 5% CO2, with a constant temperature maintained at 37 °C. Animals The Laboratory Animal Center of Shanghai University of Traditional Chinese Medicine provided female C57BL/6 mice weighing 20 ± 2 g. These mice were housed in a specific pathogen-free facility under meticulously controlled conditions, including a temperature range of 23–25 °C, humidity maintained at 60%–70%, and a well-regulated 12-h light-dark cycle. The Animal Experimentation Ethics Committee of Shanghai University of Traditional Chinese Medicine granted approval (PZSHUTCM2307310004) for experimental procedures conducted on the animals. All experiments were conducted in accordance with institutional animal care guidelines and protocols approved by the committee. Establishment of acute colitis mouse model According to the method reported by Yue [26], we established the acute colitis mouse model. Briefly, female C57BL/6 mice were divided randomly into four groups: Control, DSS, DSS + SASP, and DSS + NGR1. Acute colitis was induced by administering 3% DSS in the drinking water of mice for a period of 8 days. Mice in the DSS + SASP group were treated orally with SASP (260 mg/kg) once per day for the same duration. The DSS + NGR1 group received NGR1 (25, 50, 125 mg/kg) by oral gavage once per day for 10 days. Mice in the Control and DSS groups were administered the same volume of Control. Daily monitoring of body weight and rectal bleeding was conducted throughout the 10-day period. At the end of the experiment, mice were euthanized, and the colon was collected for further analysis. Female C57BL/6 mice were randomly divided into four groups: DSS, DSS + ICG-001, DSS + NGR1 and DSS + ICG-001 + NGR1. To establish an acute enteritis model, mice were subjected to the protocol described above. Mice in the DSS + NGR1 and DSS + ICG-001 + NGR1 group were given NGR1 (25 mg/kg) orally once daily for 10 consecutive days. Meanwhile, mice in the DSS + ICG-001 and DSS + ICG-001 + NGR1 groups were given ICG-001 (20 mg/kg) via intraperitoneal injection three times per week. The DSS and DSS + NGR1 groups received the same volume of Control. Xenograft tumor transplantation model Male BALB/c mice were acclimated for 1 week in a specific pathogen-free environment. Subsequently, CT26 cells (2 × 105 cells/mouse) were subcutaneously transplanted into the left axillary region of each mouse. Once the tumor size reached 200 mm3, the mice were randomly assigned to the vehicle group or the NGR1 group based on tumor size. Throughout the 18-day experiment, mice in the vehicle group received 0.5% CMC-Na, while those in the NGR1 group were administered 25 mg/kg NGR1. Tumor volume = 0.5 × length (mm) × width (mm)2. Measurement of intestinal permeability C57BL/6 mice were fasted for 4 h before execution. Mice were then orally administered 60 mg/100 g body weight of FITC-dextran in 200 µL of sterile saline. After 4 h, blood samples were collected via retro-orbital bleeding, and serum was separated by centrifugation. The serum FITC-dextran levels were measured at an excitation wavelength of 485 nm and an emission wavelength of 528 nm using a fluorometer (VARIOSKAN FLASH, Thermo Fisher, MA, USA). H&E staining Colonic tissues were collected from mice and fixed in 4% paraformaldehyde. Tissues were then dehydrated, embedded in paraffin, and sectioned into 4 μm thick slices. The sections were then stained with hematoxylin and eosin (H&E) using standard protocols. Stained sections were analyzed under a light microscope (BX61VS, Olympus, Tokyo, Japan), and images were captured for further analysis. Enzyme-linked immunosorbent assay (ELISA) The concentrations of DAO (CSB-E10090m) and LPS (CSB-E13066m) in mouse serum samples were determined using the respective ELISA kit (Wuhan Huamei Biological Engineering Co., Ltd, Wuhan, China). Specifically, serum samples were added to a 96-well plate coated with DAO or LPS-specific antibodies, followed by incubation with detection reagents and substrate solution. Absorbance was measured at 450 nm, and concentrations were calculated using standard curves. Immunofluorescent staining Colonic tissues were fixed in 4% paraformaldehyde, embedded in OCT compound, and sectioned into 5-μm slices. After permeabilization and blocking, sections were incubated with primary antibodies against ZO-1 (#13663, Cell Signaling Technology, CST, MA, USA) and Occludin (#91131, CST, MA, USA), followed by secondary antibodies conjugated to fluorophores (9300039001, ABclonal, Wuhan, China). Nuclei were counterstained with DAPI (#4083, CST, MA, USA), and images were obtained using a fluorescence microscope (BX61VS, Olympus, Tokyo, Japan). Quantification of ZO-1 and Occludin expression was performed using ImageJ software (NIH, Bethesda, MD, USA). Alcian blue staining Colonic tissue samples were obtained from mice, fixed, dehydrated, embedded in paraffin blocks, sectioned, and stained with Alcian blue using a commercial kit. Under a light microscope (BX61VS, Olympus, Tokyo, Japan), the stained sections were examined and images were captured for subsequent analysis. Real-time fluorescence quantitative polymerase chain reaction (RT-qPCR) RNA was extracted using the TRIzol method, and RNA quantity and purity were measured by NanoDrop spectrophotometer (Thermo Fisher Scientific). The RNA was then reverse-transcribed using an Evo M-MLV RT Premix for qPCR kit (AG11706, Accurate Biotechnology Co., Ltd., Chengdu, China), and qPCR was performed using a SYBR® Green Premix × Pro Taq HS qPCR Kit (AG11718, Accurate Biotechnology Co., Ltd., Chengdu, China) (Table 1). The amplification was carried out using an ABI Prism 7900HT Sequence Detection System (Life Technologies, CA, USA), and data were analyzed using the 2−ΔΔCt method.Table 1Genes and associated primer sequences used for RT-qPCR analysis.PrimersForward primer (5′-3′)Reverse primer (5′-3′)m Muc2AGGGCTCGGAACTCCAGAAACCAGGGAATCGGTAGACATCGm ZO-1GCCGCTAAGAGCACAGCAATCCCCACTCTGAAAATGAGGAm OccludinATGTCC GGCCGATGCTCTCTTTGCTGCTCTTGGGTCTGTATm GAPDHGGCCGAGAATGGGAAGCTTGTACATACTCAGCACCGGCCTCAm c-MycATGCCCCTCAACGTGAACTTCCGCAACATAGGATGGAGAGCAm Cyclin DGCGTACCCTGACACCAATCTCCTCCTCTTCGCACTTCTGCTCm c-JunCCTTCTACGACGATGCCCTCGGTTCAAGGTCATGCTCTGTTTm Cdx4TGACATGACCTCCCCAGTTTTGCCGGAGTCAAGAGAAACCAm Lgr5CCTACTCGAAGACTTACCCAGTGCATTGGGGTGAATGATAGCAm Olfm4CAGCCACTTTCCAATTTCACTGGCTGGACATACTCCTTCACCTTAm Bmi1ATCCCCACTTAATGTGTGTCCTCTTGCTGGTCTCCAAGTAACGm Ascl2AAGCACACCTTGACTGGTACGAAGTGGACGTTTGCACCTTCAm Sox9GAGCCGGATCTGAAGAGGGAGCTTGACGTGTGGCTTGTTCm TertGCACTTTGGTTGCCCAATGGCACGTTTCTCTCGTTGCGm HopxAGACGCAGAAATGGTTTAAGCTCCAAGAGCAAGCTCAAGGGm Lyz1GAGACCGAAGCACCGACTATGCGGTTTTGACATTGTGTTCGCm GipTGAGTTCCGATCCCATGCTAACCAGTTCACGAAGTCTTGTTGTCm Sglt1ATGCGGCTGACATCTCAGTCACCAAGGCGTTCCATTCAAAGm VillACCTTTACCGGGTGGTTTGTCGGAGTTAGCTGGAAGTTATGGACm SisGCAAATGGTGCCGAATATCAAACTCTGGGTACACTGTTAATCCTGGm GSK-3βTGGCAGCAAGGTAACCACAGCGGTTCTTAAATCGCTTGTCCTGm β-CateninATGGAGCCGGACAGAAAAGCCTTGCCACTCAGGGAAGGA Western blot Colonic tissues were extracted and homogenized, and protein was obtained using RIPA lysis buffer with phosphatase and protease inhibitors. Protein concentration was measured using a BCA assay kit (20201ES76, Yeasen Biotech Co., Ltd, Shanghai, China). Equal amounts of protein were loaded onto SDS-PAGE (sodium dodecyl sulfate polyacrylamide gel electrophoresis) gels and separated by electrophoresis. Subsequently, the separated proteins were transferred onto PVDF membranes (000025736, Milipore, MA, USA). The membrane was then blocked with 5% BSA solution for 2 h. After blocking, the membrane was incubated with primary antibodies overnight at 4 °C, followed by incubation with HRP-conjugated secondary antibodies for 1 h at room temperature. Protein bands were visualized using ECL reagents (WBKLS0500, Millipore) and imaged with a GS-700 imaging densitometer (Bio-Rad, CA, USA). Protein expression levels were quantified using ImageJ software (NIH, Bethesda, MD, USA). The following primary antibodies were used: rabbit anti-β-Catenin (1:1000, #8480, CST, MA, USA), rabbit anti-p-GSK-3β (1:1000, #5558, CST, MA, USA), rabbit anti-GSK-3β (1:1000, #12456, CST, MA, USA), rabbit anti-Cyclin D1 (1:1000, #2922, CST, MA, USA), rabbit anti-c-Myc (1:1000, #5605, CST, MA, USA) and rabbit anti-β-actin (1:1000, #4970, CST, MA, USA). Immunohistochemistry (IHC) Colonic tissue sections were fixed in 4% paraformaldehyde, embedded in paraffin, and sliced into 5 μm thick sections. Antigen retrieval was performed using citrate buffer solution (pH = 6.0) and heating in a microwave oven. Non-specific binding was blocked with 5% goat serum for 30 min. Sections were incubated overnight at 4 °C with primary antibodies, followed by incubation with a secondary antibody and staining with DAB (3,3′-diaminobenzidine). Hematoxylin was used for counterstaining before the sections were examined microscopically and images were captured. RNA-sequencing Total RNA was extracted from mouse intestinal tissues using TRIzol reagent according to the manufacturer’s instructions. The extracted RNA was evaluated for quality using a NanoDrop spectrophotometer (Thermo Fisher). RNA sequencing libraries were then constructed with the NEBNext Ultra RNA Library Prep Kit for Illumina, and sequencing was performed on an Illumina HiSeq platform. The differential gene was carried out on the cloud platform of majorbio (https://www.majorbio.com/). Transepithelial electrical resistance (TEER) Caco-2 cells were seeded in Millicell inserts of 24-well plates at a density of 5 × 104 cells/400 μL per well. The outer chamber was filled with 600 μL DMEM medium (2323012, Gibco, NY, USA) and replaced every other day. TEER values were measured using a MERS00002 volt-ohm meter system (Milipore), and the electrode was sterilized with 70% ethanol and rinsed with sterile phosphate-buffered saline (PBS) before each measurement. Monolayer formation was assumed at TEER values of 400 Ω/cm2. Measurements were taken at regular intervals using the same electrode and recorded. Organoid extraction, culture and treatment The intestinal crypts were isolated from the small intestine of C57BL/6 mice (6- to 8-week-old). The small intestine was removed and flushed with ice-cold PBS. The intestine was opened longitudinally and cut into 2- to 3-mm pieces. The pieces were then washed with ice-cold PBS and incubated in 3 mM EDTA solution at 4 °C for 20 min with gentle shaking. After incubation, the crypts were released by vigorously shaking the tubes. The supernatant containing the crypts was collected and filtered through a 70-μm cell strainer. The crypts were then centrifuged at 1200 r/min for 5 min and resuspended in Matrigel (Corning, NA, USA). The crypt-Matrigel mixture was plated in 24-well plates and incubated at 37 °C for 30 min to allow the Matrigel to solidify. The IntestCultTM OGM Mouse Basal Medium (#06005, STEMCELL, Vancouver, Canada) was then added to the wells and changed every other day. After cultured 2 days in a 24 well plate, the intestinal crypts were randomly divided into control, DSS model group and DSS + NGR1 group. Then, the organoids were administered DSS (20 μg/mL), DSS (20 μg/mL) plus NGR1 (100 μM) for 4 days. The organoid growth conditions were recorded by the microscope (Olympus CKX4, Tokyo, Japan). IHC assay was conducted to examine the fluorescent protein expression of Lgr5 and β-Catenin (refer to the above method of IHC). Molecular docking The molecular docking was performed using AutoDock Vina software. The 3D crystal structure of β-Catenin protein (PDB: 1JDH) was obtained from the Protein Data Bank (PDB) database. The structure of NGR1 was drawn and optimized using ChemDraw software and converted to a PDB file using Open Babel software. The protein and ligand files were prepared using AutoDock Tools. Docking simulations were performed and the conformation with the lowest binding energy was selected as the final docking result. The docking results were analyzed using PyMOL software. TOPFlash The TOPFlash assay was performed as previously described with slight modifications [27]. HEK293T cells were seeded in 24-well plates and cultured overnight. The cells were transfected with the 500 ng TOPFlash luciferase reporter plasmids (Beyotime Biotechnology, Shanghai, China) and 50 ng Renilla luciferase (Promega GmbH, Mannheim, Germany) using Lipofectamine 3000 (Thermo Fisher). After 24 h, the cells were treated with NGR1 (50 μM) and BIO (0.5 μM) for 24 h, separately. Subsequently, cells were lysed in 150 μL/well passive phenylbenzothiazole (PPBT) buffer, and the luciferase activity was measured using a Dual-LuciferaseTM Reporter Assay System (Promega Corporation, WI, USA). The firefly luciferase activity was normalized to Renilla luciferase activity. Wound healing assay A scratch wound was created using a plastic pipette (10 μL) tip. NCM460 cells were then washed with PBS to remove any debris and treated with either DSS (20 μg/mL) or DSS (20 μg/mL) + NGR1 (100 μM) for 24 h. The width of the scratch was measured using microscopy at 0 and 24 h post-dosing, and the percentage of wound closure was calculated by comparing the scratch width at 24 h to the initial scratch width. Flow cytometry NCM460 cells were treated with either DSS (20 μg/mL) or DSS (20 μg/mL) + NGR1 (100 μM) for 24 h. Then, NCM460 cells were harvested and washed with PBS after experimental treatment. Cells were then suspended in a binding buffer containing Annexin V-fluorescein isothiocyanate (FITC) and propidium iodide (PI), and incubated in the dark at room temperature for 15 min. Flow cytometry analysis was performed to detect apoptotic cells. The data were analyzed using Guava software, and the percentage of apoptotic cells was expressed. Statistical analysis Statistical analysis was performed using GraphPad Prism 9.0 software. Data were presented as mean ± standard deviation (SD). Differences between groups were analyzed using one-way analysis of variance (ANOVA). P < 0.05 was considered statistically significant. All experiments were repeated at least three times.',\n",
       " '11192043': 'MethodKidney neoplasm and NSS related publications before July 3th 2023 were obtained from the Web of Science Core Collection database. We then used bibliometric analysis to conduct performance analysis, citation analysis and co‐citation network of publications, together with keyword co‐occurrence analysis.',\n",
       " '11192023': '2 METHOD 2.1 Data sources and retrieval strategies The Web of Science (WOS) database, widely deemed as a high‐quality global literature search engine and the most appropriate database for bibliometrics analysis, 19 , 20 , 21 was used for publication retrieval on 3rd July 2023. The retrieval formula was demonstrated as follows: ((TS = ((renal OR kidney) NEAR/2 (cancer* OR tumor* OR tumor* OR neoplasm* OR carcinoma* OR oncology))) AND ((TS = nephron sparing) OR (TS = partial nephrectomy))), and altogether 13,314 publications were retrieved. Next, only publication types including reviews and research articles were included, resulting in the exclusion of 3772 documents. Finally, we excluded the publications that were not in the WOS core collection database, and altogether 7564 publications were selected for the following bibliometric analysis. All the publications were downloaded into a text file, and we also uploaded them into Appendix S1. 2.2 Methodology of bibliometric analysis With the help of R language (version 4.2.2, Institute for Statistics and Mathematics, Vienna, Austria; www.r‐project.org) and R studio, we used a R package called Bibliometrix (version 4.1.3) for bibliometric analysis. We put the previous text file (Appendix S1) input into Biblioshiny, which was an open‐source tool and a web‐based operator, and could provide a web operating interface for subsequent quantitative research. 2.3 Data analysis processes of bibliometric analysis In our study, we conducted the performance analysis to explore the most influential authors, countries, affiliations, journals, and keywords in the area of kidney neoplasm and NSS field. We also analyzed the number of publications, citations of the publications, h‐index of authors and journals, as well as their production over time. Besides, we conducted an analysis of collaboration network, to explore the collaborative relationship between different countries. In addition, to figure out the publications that were the most influential in this area, we attached great importance to the most cited documents and references. Considering the keywords, we identified the most relevant keywords as well as trend topics over time. Then we evaluated thematic evolution, and portrayed a thematic map, hoping to catch the latest hotspots and have a general view of the development process. In addition, arrows in the historiograph indicated that several previous influential publications were cited by the relatively newer publications, possibly demonstrating the process of development in a certain field. Furthermore, co‐citation and word co‐occurrence analyses were performed, which respectively could discover articles and words that were usually cited together by other articles, assisting us to make sense of the internal relationship and degree of betweenness about the researching hotspots of this field. The dimensionality reduction technique was utilized to better visualize the conceptual structure of keywords.',\n",
       " '11191908': '2.2 Methodology of bibliometric analysis With the help of R language (version 4.2.2, Institute for Statistics and Mathematics, Vienna, Austria; www.r‐project.org) and R studio, we used a R package called Bibliometrix (version 4.1.3) for bibliometric analysis. We put the previous text file (Appendix S1) input into Biblioshiny, which was an open‐source tool and a web‐based operator, and could provide a web operating interface for subsequent quantitative research.',\n",
       " '11191895': '4.3 The blue cluster: more conservative NSS methods Apart from PN, there are other more conservative NSS methods available for treating clinically localized renal masses suspected to be cancer, including thermal ablation (TA) and active surveillance (AS), which has been widely used in clinical practice. Commonly, minimally invasive thermal ablative techniques referred to cryoablation, percutaneous RFA, stereotactic radiosurgery, laser ablation, high‐intensity focused US ablation, and microwave ablation. 6 TA is usually indicated for elderly patients with SRMs (≤4 cm lesions) who are considered unfit for surgery due to comorbidities, and patients with a genetic predisposition to develop multiple tumors, experiencing recurrences after previous surgery, or having bilateral tumors or a solitary kidney. 157 Moreover, TA is also appropriate for individuals at high risk of experiencing complete loss of RF after undergoing PN. Regarding to the therapeutic effects, TA could provide more favorable perioperative outcomes and possible long‐term RF preservation compared with PN, 83 , 158 and previous research also revealed similar short‐ and medium‐term cancer‐specific results. 159 It should be noted that this seems true even for transplant patients carrying a tumor on their kidney graft despite the few studies available. 160 However, a contemporary meta‐analysis demonstrated that TA conferred lower local recurrence‐free survival with one treatment, while reaching therapeutic equivalences with PN only after multiple treatments. 161 , 162 Moreover, the therapeutic effect for cT1a patients was compared between different TA methods (heat‐based TA and cryoablation). With equivalent outcomes for the patients with tumor size ≤3 cm, cryoablation was proved to be superior in CSM for the patients with tumor size 3.1–4 cm. 163 AS, usually involves the initial monitoring of tumor size through regular abdominal imaging, frequent check‐ups, and physical treatment. While for those who demonstrated clinical progression during follow‐up, delayed intervention was considered. Besides, before initiating surveillance, it was advised to conduct a renal biopsy. 6 As the nonsurgical method, AS resulted in statistically significant lower CSS compared to RN and PN in patients with cT1a tumors. 68 , 164 , 165 Nevertheless, in patients aged 75 years or higher, or those with a high cardiovascular risk, the advantage of surgical intervention in terms of CSS could possibly be reduced. 164 , 165 Moreover, several studies found that the CSS rates of AS were excellent and the metastatic rate was low within short follow up. 166 , 167 , 168 , 169 , 170 , 171 , 172 , 173 , 174 , 175 Consequently, in could be inferred that for the elderly and comorbid patients, especially who had incidentally detected SRMs that had relatively lower mortality compared with other competing causes, AS with delayed intervention was non‐inferior to the primary surgical interventions and could serve as a safe and cost‐effective choice for their treatment. 174 In future, the therapeutic efficacy of these more conservative NSS methods should be more compared with other surgical treatment, to figure out the most suitable patients for these methods. Besides, the therapeutic effects among different TA methods and AS also remains elusive. In the end, we summarized all the major viewpoints of our study xin Figure 5, which could intuitively demonstrate the contents discussed above. FIGURE 5 The schematic diagram vividly demonstrated the main topics in the three major clusters of our discussion section. Red cluster mainly discussed the therapeutic effects of PN versus RN on patients with localized RCC. From past to now, PN gradually becomes the standard of care for cT1 RCC, and we reviewed the general process. Then, we found that whether the indication of PN could be expanded remains unclear nowadays. Besides, the superiority of PN versus RN in small localized RCC is still being challenged by some researchers, and we summarized the perspectives of both sides. In blue cluster, more conservative NSS methods including TA and AS were mainly focused. Considering the low‐quality of the existing researches, the therapeutic efficacy of TA and AS should be more compared with each other as well as other surgical treatment. Moreover, how to figure out the most suitable patients for TA and AS was the most important issue in this field. Last but not the least, we discussed different operative approaches of PN in the green cluster. Firstly, the minimally invasive surgical modalities of PN (laparoscopic, robot‐assisted) were compared together with open PN. Secondly, we talked about the intraoperative renal ischemia (influence on renal function) and clamping techniques of PN (reduce ischemic injury). AS, active surveillance; cT1, clinical T1; NSS, nephron sparing surgery; PN, partial nephrectomy; RCC, renal cell carcinoma; RN, radical nephrectomy; TA, thermal ablation. In our study, although our paper could serve as a seminal work employing bibliometric analysis in the kidney neoplasm and NSS field, it is essential to acknowledge some inherent limitations stemming from the data acquisition process. Firstly, our retrieval only encompassed the WOS core collection database and excluded publications released after the retrieval date, potentially leading to incompleteness and outdatedness of relevant works. Secondly, even though we have analyzed the publications with the highest citations per year, some of the most recent influential studies might be overlooked. Finally, a retrieval formula is often difficult to balance between comprehensive and precise searches, inevitably leading to either an expanded search scope or the omission of some relevant literature that should have been included in the search.',\n",
       " '11191834': 'MethodsThe diet of 103,060 pregnant women at >4 months of gestation who were enrolled in the Japan Environment and Children’s Study was analyzed. Their intake levels of fermented soybeans (miso and natto), yogurt, and cheese were investigated. The developmental status of the offspring at 3 years of age was assessed using the Ages and Stages Questionnaires (ASQ-3). Multivariable logistic regression analysis was performed to determine the risk of maternal intake levels of the fermented foods associated with subsequent developmental delay in the offspring.',\n",
       " '11191688': 'Methods The JECS protocol was reviewed and approved by the Ministry of the Environment’s Institutional Review Board on Epidemiological Studies (Ethical Number: No. 100910001) and the ethics committees of all participating institutions. Participants The present study was based on data from the Japan Environment and Children’s Study (JECS), a nationwide birth cohort study that focuses on the relationship of environmental factors with child health and development. The JECS enrolled pregnant women from study areas throughout Japan between January 2011 and March 2014. Details of the JECS design have been reported previously [12–14]. We used data from the jecs-an-20190930 dataset released in October 2019 in the present study. The dataset includes 103,060 pregnancies. We excluded 5,647 due to multiple enrollment, 948 due to multiple pregnancies, 3,520 due to miscarriage or stillbirth, and 32,305 due to lack of information on maternal diet or developmental delay or due to inability to respond to the survey. Thus, a total of 60,910 mother–infant pairs were analyzed in this study (Fig 1). All maternal dietary data were obtained via a self-administered questionnaire on the consumption of fermented foods during pregnancy. Written informed consent for the study was obtained from all participants. 10.1371/journal.pone.0305535.g001 Fig 1 Flow diagram of participant selection. This study included 103,060 pregnant women, 5,647 of whom were excluded due to multiple enrollment, 948 due to multiple pregnancies, 3,520 due to miscarriage or stillbirth, and 32,305 due to lack of information on diet or developmental delay or due to inability to respond to the survey. A total of 60,910 mother–infant pairs were analyzed in the study. Fermented foods The intake of four types of fermented foods (miso, natto, yogurt, and cheese) by the mothers at >4 months of gestation using the Food Frequency Questionnaire [15]. This semi-quantitative questionnaire contains the following items: name of food, frequency of consumption, and estimated amount consumed in a single meal (weight, nutritional values, and portion size) [16]. These four foods are commonly consumed in Japan. Neurodevelopment The primary outcome was neurodevelopmental status of the offspring at 3 years of age. Neurodevelopment was assessed using the Ages and Stages Questionnaires (ASQ-3). The ASQ-3 is a modified version of the parent-completed child monitoring system, which is a UNICEF-recommended and validated neurodevelopment assessment tool [17,18]. The ASQ consists of 21 age-specific questionnaires administered over time and assesses development in five domains: communication skills, gross motor skills, fine motor skills, problem-solving skills, and social skills. The questionnaire consists of 30 questions: six for each of the five major domains. Each can be answered with “yes,” “no,” “sometimes,” or “not yet.” A score of 10 is given for “yes,” indicating that the child has achieved the item; a score of 5 is given for “sometimes,” indicating that the child is still developing; and a score of 0 is given for “not yet” and “no,” indicating that the child has not yet acquired the ability to achieve the item. If one or two of the six questions were unanswered, a correlation coefficient of 1.2–1.5 was multiplied by the actual number of points earned, and the scores were adjusted to be in the range of 0–60 points; if three or more of the six questions were unanswered, the scores were excluded from the analysis. Participants with scores equal to or below the cut-off values were considered positive cases [19]. The ASQ-3 was completed by mothers for offspring between 35 and 37 months of age. Statistical analysis Data are presented as the mean ± standard deviation or median, unless otherwise stated. To estimate the risk of neurodevelopmental delay with intake of each fermented food, the participants were divided into four groups according to intake level. Odds ratios and 95% confidence intervals were obtained from a multivariate logistic regression analysis. Potential confounders from previous reports included maternal age (<25, 25–<30, 30–<35, ≥35 years), body mass index (<18.5 kg/m2, 18.5–25 kg/m2, ≥25 kg/m2), parity (primipara, multipara), smoking status (never, previously smoked, current smoker), passive smoking (almost never or never, >1 per week), alcohol intake (never, previously drank, current drinker), quartile of physical activity (MET min/week) [20], quartile of folic acid intake (μg), quartile of energy intake (cal), marital status (married [including common-law marriage], single [never married], divorced or widowed), highest education level (≤12, 12–<16, ≥16 years), highest education level of partner (≤12, 12–<16, ≥16 years), employed (yes, no), annual household income (<4 million yen, 4–6 million yen, ˃6 million yen), administration of anti-bacteria medicine (yes, no). Potential mediators were not used as covariates.',\n",
       " '11191580': '2. Materials and methods 2.1. Cell culture Human liver astrocytes (LX-2, 1 vial, model T25) were purchased from Wuhan Procell Life Sciences Co. The cells were cultured in a humidified incubator at 37°C with 5% CO2 for 48 h. When the cell confluence was greater than 80%, the cells were cultured in a humidified incubator at 37°C with 5% CO2 and rinsed twice with 2 ml of 1x PBS (Thermo Fisher Scientific, China). Then, 1 ml of trypsin-EDTA (Thermo Fisher Scientific, China) was added to digest the adherent cells, and the cells were placed in a 37°C and 5% CO2 incubator for 3 min. Then, 4 ml of LX-2 medium was added to the T25 cell culture flasks (Wuhan Procell Life Sciences Co., Ltd., China) to terminate tryptic digestion and mixed with a sterile pasteurized pipette to resuspend the cells. Then, the cells were transferred to 15 ml sterile and enzyme-free centrifuge tubes (Thermo Fisher Scientific, Ltd., China), centrifuged at 1200 × g for 3 min, and resuspended in an appropriate volume of medium for transfer to new culture flasks incubated at 37°C and 5% CO2 to continue the cultivation of the cell lines. The present study used cells from the 3rd–7th passages. 2.2. Sodium fluoride for cell treatments Sodium fluoride (10.5 g, Tianjin Yong da Chemical Reagent Co., Ltd., China) was dissolved in 500 ml of 1x PBS solution and filtered through a membrane to remove any bacteria, and the concentration was adjusted to 500 mmol/l. Complete media with different NaF concentrations were prepared according to each group, as shown in Table 1. After LX-2 cells were cultured at 37°C in a 5% CO2 incubator for 24 h, the cells were treated with fluorine (n = 4). 10.1371/journal.pone.0302530.t001 Table 1 Table of configurations for different NaF concentrations. LX-2 special medium content (ml) Amount of 500 mmol/l NaF added (μl) Total volume (ml) 0.5 9.99 10 10 1 9.98 20 10 2 9.96 40 10 4 9.92 80 10 8 9.84 160 10 2.3. CCK-8 assay for cell viability Cells (Nanjing Novozymes Bioscience and Technology Co., Ltd., China) were aliquoted into 96-well plates containing 2,000 cells/ml and incubated for 24 h, after which the culture medium was removed when the cell wall density was greater than 80%. The cells were rinsed twice with PBS, and a blank group, a control group (0 mmol/l NaF), and experimental groups with different concentrations of NaF (0.5, 1, 2, 4, and 8 mmol/l NaF) were prepared. For each well, except for those for the blank group, 200 μl of cell culture medium was added, and the cells were incubated for 12 h, 24 h or 48 h. Then, 10 μl of CCK-8 reagent was added, and the cells were incubated for 1 h in the dark. A fully automated enzyme labeling instrument (Thermo Fisher Scientific, China) was used to determine the OD value of each group at 450 nm, and cell viability (%) was calculated as (OD of the experimental wells-OD of the blank wells/OD of the control wells) × 100% (n = 4). 2.4. HE staining LX-2 cell cultures were fixed with 4% paraformaldehyde for 5–10 min, washed with water for 1 min, treated with HD Constant Staining Pretreatment Solution for 1 min, and sequentially stained with appropriate amounts of hematoxylin and eosin staining solution. Finally, the cells were fixed with neutral resin, and the morphology of LX-2 hepatocytes was observed under a light microscope. 2.5. Enzyme-Linked Immunosorbent Assays (ELISAs) of the expression of each target protein in the cell supernatant and cell extract of each group of cells CXCR4, SDF-1, IL-1β, TNF-α, IL-6 and NF-κB ELISA kits were obtained from Nanjing Boyan Biotechnology Co., Ltd. (China). Cell supernatant collection: Cell supernatants were collected by centrifugation of each test group at 20 min (3000 r/min) and stored at -20°C to avoid repeated freezing and thawing. Cell extract collection: adherent cells were washed with PBS at 4°C and digested with trypsin, and then, the cells were collected after 5 min (1000 r/min) and washed with PBS 3 times. Then, 200 μl of 1x PBS was added to resuspend the cells, the cells were ruptured by repeated freezing and thawing, and the supernatant was collected for evaluation. The expression level of each target protein in the cell supernatant was detected by an enzyme labeling instrument (450 nm). The kits were left at room temperature for 30 min before use. For the wash buffer, the concentrate in the kit was diluted with distilled water at 1:20. All reagents and samples were used at room temperature. Blank wells (no sample or standard added), standard wells (50 μl of standards at different concentrations) and sample wells (50 μl of samples to be tested) were prepared, and 2 replicate wells were prepared for each group. Horseradish peroxidase (HRP)-labeled antibody (100 μl) was added to each of the wells. The plate was sealed using plate sealing film and incubated for 60 min at 37°C in a water bath or thermostat. The liquid was discarded, 300 μl of wash solution was added to each well, and the plate was washed for 20 s and patted dry on absorbent paper; the washes were repeated 5 times. Then, 50 μl each of chromatography substrates A and B was added to each well and incubated at 37°C for 15 min. Finally, 50 μl of termination solution was added to each well, the color changed from blue to yellow, and readings were taken within 15 min. 2.6. RNA extraction Total RNA was extracted with a Total RNA Extraction Kit (All Specialty Gold Biotechnology Co., Ltd., Beijing, China). Then, cDNA was synthesized with a Reverse Transcription Kit (All Specialty Gold Biotechnology Co., Ltd., Beijing, China) for gene fragment detection; gene fragments were amplified, and the optimal primers as determined by the HIFI RCR enzyme (All Specialty Gold Biotechnology Co., Ltd., Beijing, China) and HIFI RCR enzyme (Beijing Total Gold Biotechnology Co., Ltd., Beijing, China) were used to amplify the gene fragments at the optimal primer annealing temperature. A fully automated enzyme labeling instrument (Thermo Fisher Scientific) was used to determine the CXCR4, SDF-1, IL-1β, TNF-α, IL-6, and NF-κB mRNA levels as well as the A260/280 ratio. Agarose electrophoresis buffer (LABJIC Biotechnology Co., Ltd., Beijing, China) was used to prepare a 1% agarose gel to determine the RNA concentration. 2.7. Real-time quantitative Polymerase Chain Reaction (qPCR) Each reaction contained 0.4 μl of upstream primer, 0.4 μl of related primer, 0.8 μl of template, 10 μl of 2× PerfectStart Green qPCR SuperMix (Beijing Quan Shi Jin Biotechnology Co., Ltd., Beijing, China), 0.4 μl of Passive Reference Dye (50×), and 8 μl of nuclease-free water for a total volume of 20 μl. Seven pairs of primers were designed by Sangyo Bioengineering Co., Ltd. (Shanghai, China); the primer names, primer sequences, gene IDs, and primer lengths are shown in Table 2. The experimental conditions were 94°C for 30 s, 59.6°C for 15 s, and 72°C for 10 s (44 cycles). The experimental results were obtained by repeating the experiment three times and taking the average value. The mRNA expression of the target genes CXCR4, SDF-1, IL-1β, TNF-α, IL-6 and NF-κB was calculated by the 2-ΔΔCT method using glyceraldehyde dehydrogenase 3-phosphate (GAPDH) as the internal reference gene (n = 6). 10.1371/journal.pone.0302530.t002 Table 2 Primer sequences, gene numbers and product amplification lengths. Gene Primer sequences Product length (bp) Gene ID GAPDH F: GAGGAGGCATTGCTGATGAT 20 NM_002046.7 R: GAAGGCTGGGGCTCATTT 18 CXCR4 F: CTCCTCTTTGTCATCACGCTTCC 23 NM_001008540 R: GGATGAGGACACTGCTGTAGAG 22 SDF-1 F: GGGAAGACCCGTGTTACCAG 20 NM_199168.4 R: AGTCCAGCCTGCTATCCTCA 20 NF-κB F: TGTGTTTGTCCAGCTTCG 18 NM_001165412.2 R: GCTTCTGACGTTTCCTCTG 19 IL-1β F: TCGCCAGTGAAATGATGGCT 20 NM_000576 R: TGAAGCCCTTGCTGTAGTGG 20 IL-6 F: GAGGAGACTTGCCTGGTGAA 20 NM_000600.5 R: CAGCTCTGGCTTGTTCCTCA 20 TNF-α F: ATGAGCACTGAAAGCATGATCC 22 NM_000594.4 R: AGGAGAAGAGGCTGAGGAACAAG 23 Gene numbers and product amplification lengths 2.8. Statistical analysis Excel was used for data entry and the development of statistical tables, GraphPad Prism 8.0.2 was used to generate statistical graphs, and SPSS 22.0 was used to statistically analyze the experimental and control groups; a t test was used for data that were normally distributed and exhibited homogeneity of variance, and the Mann‒Whitney test was used for data that did not satisfy these conditions. Measurement data are expressed as the mean ± standard deviation (± s), test level α = 0.05, and P < 0.05 indicated significance.',\n",
       " '11191499': 'Materials and Methods Clinical samples. The study was conducted from January 1st, 2018-December 31st, 2022. All consecutive samples delivered for STEC routine testing in the Microbiology Laboratory of the University Medical Center of the Medical University of Warsaw were included in the study. A total of 180 stool samples from children suspected of HUS were included in the study. The samples came from five pediatric medical centers from five voivodeships: Mazovian, Podlaskie, Greater Poland, Lublin, and Warmian-Masurian. Detection of STEC strains Detection of the presence of stx1/stx2 genes in stool samples was performed using the BD MAX™ Enteric Bacterial Panel and the Viasure Real Time PCR Detection Kit (Becton, Dickinson and Company, Australia) according to the manufacturer’s instruction. All examined samples were cultured on MacConkey agar to select E. coli isolates and on STEC-enriched broth (Axonbiotech GmbH, Germany). After overnight incubation at 37°C, the presence of E. coli was confirmed based on colony morphology and VITEK® 2 identification (bioMérieux, France). Total DNA was extracted from overnight cultures on Columbia Agar using the Genomic Micro AX Bacteria+ Gravity kit (A&A Biotechnology, Poland) according to the manufacturer’s protocol. Quality control and concentration of DNA samples were measured using a Thermo Scientific™ NanoDrop™ Spectrophotometer (Thermo Fisher Scientific, Inc., USA). The purity and concentration of the obtained DNA were determined by A260/A280 and A260/A230 absorbance measurements. Samples with a ratio between 1.8 ± 0.2 and a DNA concentration range between 25–50 ng were used for PCR analysis. Primer and probe sequences applied in this study were previously reported and described by Perelle et al. (2004; 2005) (stx1 and stx2 genes) and Nielsen and Andersen (2003) (the eae gene) (Table I). Primer pair specificity was assessed by Kagkli et al. (2011). Additionally, a comparison of target sequences was performed with sequences found in the GenBank sequence database provided by NCBI, aligning them to all available bacterial, human, bovine, mouse, and rat genomes. RT-PCR reactions were performed according to the protocol described by Kagkli et al. (2011) with a slight modification. Applied GoTaq® Green Master Mix (Promega, USA) with Taq DNA polymerase was used. Applied Biosystems™ TaqMan® probes (Thermo Fisher Scientific, Inc., USA) were labeled with 6-carboxyfluorescein-6-carboxytetramethylrhodamine (FAM-TAMRA), the concentration of each primer was 0.6 μM and the probe 200 nM. The reaction mixtures were heated at 95°C for two minutes, of amplification at 95°C for 15 seconds and 60°C for a minute, followed by 40 cycles conducted in the CFX96 Touch™ Real-Time PCR System (Bio-Rad, USA). Table I Characteristics of primers and probes used in the study. Gene name GenBank Access No. Nucleotide sequences (5’→3’)* Amplicon size (bp) sxt1 M16625 Fw**: TTTGTYACTGTSACAGCWGAAGCYTTACG 132 Rv**: CCCCAGTTCARWGTRAGRTCMACRTC Probe: CTGGATGATCTCAGTGGGCGTTCTTATGTAA sxt2 X07865 Fw: TTTGTYACTGTSACAGCWGAAGCYTTACG 128 Rv: CCCCAGTTCARWGTRAGRTCMACRTC Probe: TCGTCAGGCACTGTCTGAAACTGCTCC uh Z11541 Fw: CATTGATCAGGATTTTTCTGGTGATA 102 Rv: CTCATGCGGAAATAGCCGTTA Poll: ATAGTCTCGCCAGTATTCGCCACCAATACC 1 * – in the sequences, Y is (C/T), S is (C/G), W is (A/T), R is (A/G), and M is (A/C) ** – Fw – forward primer, Rev – reverse primer Antibiotic susceptibility testing Antimicrobial susceptibility testing (AST) was performed using a VITEK® 2 instrument (bioMérieux, France) for all antibiotics except azithromycin. For azithromycin susceptibility testing an MTS™ MIC Test Strip (Liofilchem S.r.l., Italy) was used and plated on Mueller-Hinton agar (Oxoid, Thermo Fisher Scientific, Inc., USA) according to the manufacturer’s instructions. Results were interpreted according to EUCAST v. 12 breakpoints (EUCAST 2022). Breakpoints for azithromycin were taken from the Salmonella/Shigella criteria. Pulsed-field gel electrophoresis (PFGE) The analysis was performed using the CHEF-DR® II instrument (Bio-Rad, USA) according to the procedure described by Kawamori et al. (2008) with minor modifications. E. coli from 24-hour cultures were suspended in cell suspension buffer (CBS) (100 mM Tris, 100 mM EDTA, pH 8.0) to a McFarland density of 2.0. Bacterial suspensions were mixed (1:1) with 1% agarose or agarose SeaKem® Gold Agarose (Lonza Rockland, Inc., USA) in TE buffer (10 mM Tris, 1 mM EDTA, pH 8.0). The agarose plugs with the bacterial suspension were cooled to 4°C to solidify. Lysis was performed for two selected plugs using dissolved CSB (50 mM Tris, 50 mM EDTA, pH 8.0) and 1%N-lauroylsarcosine sodium (Sigma Aldrich®, Merck KGaA, Germany) for 2 hours of incubation at 54–55°C under constant and vigorous agitation. To prepare the genomic DNA, two plugs were digested for 2 hours at 37°C with the enzyme restrictive XbaI (50 units on a sample) (Thermo Scientific™, Thermo Fisher Scientific, Inc., USA) and 100 μg/ml bovine serum albumin (BSA) after washing 4 times with 10 ml TE buffer (10 mM Tris, 1 mM EDTA, pH 8.0). PFGE electrophoresis was performed using 1% PFGE agarose (Bio-Rad, USA) in 0.5 × Tris-Boran-EDTA (TBE) buffer under the following conditions: total duration 19 h, switch block included start time 2.2 s, end time 54.2 s, voltage 6 V/cm, switching angle 120° and temperature 14°C. After electrophoresis, the gel was stained with ethidium bromide solution and photographed using a Bio-Rad Gel Doc™ XR + Imaging System (Bio-Rad, USA). Agglutination with serotype O157 antiserum E. coli O157 antigens were serotyped using the agglutination method on microtiter plates. A total of 3–5 colonies from Columbia agar with 5% sheep blood were selected for inoculation in the infusion broth and incubated overnight at 35–37°C. After incubation, 80 μl of O antiserum (SSI Diagnostica, Denmark) was mixed with 80 μl of boiled culture on a microtiter plate and incubated at 50–52°C overnight. The reaction was read under artificial light on a black background. In each run, a negative control was performed by mixing 80 μl of physiological saline at pH 7.4 with 80 μl of boiled culture. Approval from the Ethics Committee of the Health Centers Not applicable. Medical information was collected from medical records after the discharge of patients. The patient’s informed consent has not been requested to collect information Medical information was collected from medical records after the discharge of patients. Inclusion and exclusion criteria A total of 180 stool samples from children suspected of HUS were included in the study. The samples came from five pediatric medical centers from five voivodeships: Mazovian, Podlaskie, Greater Poland, Lublin, and Warmian-Masurian. Epidemiological analysis – risk factors for STEC infection Medical documentation was analyzed in terms of contact with animals (e.g. domestic, farm, zoo), eating unwashed fruit, and vegetables, visiting a mini zoo, participating in scout camps. Limitations of the study The study was limited to children with HUS suspected who were hospitalized in tertiary hospitals. Analysis of medical parameters of STEC-infected patients Medical documentation was analyzed regarding diarrhea onset, platelet concentration, treatment results, recovery, and HUS complications. Statistical analysis Statistical analyses were performed in Statistica 13 (StatSoft, Inc., USA). The relationships between variables were evaluated using the following statistical tests: Shapiro-Wilk test for normality, Levene test for equality of variances, independent t-test for two samples, Mann-Whitney U test, chi-square goodness of fit test and chi-square test of independence, test of significance of Spearman’s rankorder correlation coefficient. For all analyses, a p-value of < 0,05 was considered statistically significant.',\n",
       " '11191362': 'MethodsTwo hundred and twenty-six patients who underwent cardiac valve surgery were included. They were categorized based on scoring into non-PND group (group non-P) and PND group (group P’). The risk factors associated with PND were analyzed. These patients were further divided into group C and group P by propensity score matching (PSM) to investigate the serum proteome related to the PND by serum proteomics.',\n",
       " '11191355': 'Methods Participants We conducted a single-center prospective cohort study that was approved by the Biomedical Research Ethics Committee of the Affiliated Hospital of Zunyi Medical University on October, 2022 (KLL-2022-700) and was registered in the Chinese Clinical Trials Registry on October, 2022 (ChiCTR2200064929). Patients who underwent heart valve replacement at Zunyi Medical University Hospital from October 2022 to April 2023 under CPB were included (the first patient in this study was included on October 11th, 2022, and the last visit was on November 24th, 2022). Written informed consent was obtained prior the enrollment. The inclusion criteria were as follows: Patients between the ages of 45 and 74 who underwent heart valve replacement surgery under CPB and are classified as ASA grade I- III. The exclusion criteria: surgical history, cerebrovascular disease, epilepsy, other central nervous system diseases, metabolic disease, history of endocrine disorders, patients with neurological or psychiatric illnesses, preoperative Mini-Mental State Examination (MMSE) score ≤ 23, and emergency patients. Anesthesia and surgery No preoperative medication was given. Anesthesia was induced with midazolam (2 mg), sufentanil (0.4∼0.6 μg/kg), atracurium cisbenesulfonic acid (0.2∼0.3 mg/kg) and etomidate (0.3 mg/kg). Anesthesia maintenance involved the intravenous pumping of sufentanil (0.5∼1.0 μg/kg/h), cisatracurium (0.2 mg/kg/h), propofol (0.5∼2 mg/kg/h). Ventilator parameters were adjusted according to airway pressure and end-expiratory CO2 (ETCO2) was maintained at 35∼45 mmHg. Invasive arterial blood pressure (mmHg), body temperature (T), heart rate (HR), saturation of pulse oxygen (SPO2), electrocardiogram (ECG), bis-frequency index (BIS) and ETCO2 were monitored. Intraoperative mean arterial pressure (MAP) and HR fluctuations were maintained within ±20% of baseline values. Patients were returned to the Intensive Care Unit (ICU) on the vascular surgery ward with a tracheal catheter after surgery. Postoperative analgesia was managed in a uniform manner in the ward. The use of opioids and vasoactive drugs was recorded. Two milliliters peripheral venous blood was collected on the second day post-surgery after neuropsychiatric scale assessment. Blood samples were centrifuged at 4 °C for 3,000 rpm for 15 min. Serum was taken and stored in a refrigerator at −80 °C. Neuropsychological test MMSE (provided by The Hartford Institute for Geriatric Nursing, Division of Nursing, New York University) and MoCA-B evaluation scales were performed on pre-surgery and post-surgery, and scores for each test were recorded by trained investigators. Z-score was calculated for each patient based on the scores from pre-surgery and post-surgery cognitive function tests. Cognitive impairment testing was performed with consideration of potential learning effect. The means change from the health control group (spouse of patients) was subtracted to correct for this learning effect. This means that the preoperative score and learning effect were deducted from each postoperative test score. The difference was then divided by the preoperative standard deviation (SD) of the healthy control group. The composite Z-score was calculated as the average Z-score of all tests in a single patient. PND was determined by a Z-score of <−1.96 in two or more test items (Greaves et al., 2020; Wiberg et al., 2022). \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{upgreek} \\\\usepackage{mathrsfs} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document} \\\\begin{eqnarray*}Z= \\\\frac{\\\\text{Postoperative Score}-\\\\text{Preoperative Score}-\\\\text{Learning Effect}}{\\\\text{Preoperative Standard Deviation}} . \\\\end{eqnarray*}\\\\end{document}Z=Postoperative Score−Preoperative Score−Learning EffectPreoperative Standard Deviation. Grouping Based on Z-scores, patients were categorized into PND group (group P’) and non-PND group (group non-P). Using PSM (with four risk factors as covariates and a caliper width of 0.2) to match 1:1 from the group non-PND to create a matched group C that has similar baseline characteristics to the group P’ (Wang, 2021). The resulting matched groups are the group P and the group C. Sample size The postoperative patients were divided into two groups: PND (group P’) and non-PND (group non-P). The incidence rate of PND was evaluated as approximately 24% based on preliminary experiment. The reported incidence of neurocognitive disorders after cardiac valve surgery in Chinese individuals was approximately 33% according to relevant literature (Rappold et al., 2016; Xu et al., 2013). Using the PASS 2021 software (Kaysville, Utah, USA) and setting the parameters for one proportion, two sides to P0 0.33, P1 0.24, with a significance level of 0.05 and a test power of 0.80, we estimated a drop-out rate of 10% and included 226 subjects in the study. Proteomics After matching, serum samples from groups P (n = 25) and C (n = 25) were pooled every five samples for protein detection. The extracted protein was subjected to BCA protein concentration assay by BCA kit (Thermo Scientific, Waltham, MA, USA). Standard curves were plotted based on the optical density values of standards, which allowed the determination of protein concentration. SDS-PAGE was used for protein separation, followed by staining with Coomassie Brilliant Blue, then a fully automated digital gel image analysis system was applied for scanning. The remaining samples were protease-digested, desalted, and then analyzed by mass spectrometry under the following conditions: capillary voltage of 1.5 KV, desiccant temperature of 180 °C, drying gas flow rate of 3.0 L/min, mass spectrometry scan range of 100–1700 m/z, ion mobility range of 0.75−1.4 Vs/cm2, collision energy range of 20–59 eV, followed by evaluation of raw data, database search, and screening for differentially expressed proteins. Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) databases were used to explore the associated functions and/or pathways of target proteins. Statistical analysis Data analysis was performed using SPSS 29.0 (IBM Corp., Armonk, NY, USA, Version 29.0). Quantitative data were expressed as means ± SD, while categorical variables were presented as percentages. Two independent samples were compared using Student’s t-test or the Mann–Whitney U-test (when variances were homogeneous and non-normal), and chi-square tests or Fisher’s exact probability tests were applied for count data. P <0.05 was considered statistically significant. Proteins with a fold change >1.2 or <0.83 and P <0.05 were considered differentially expressed proteins (DEPs). The DEPs were further subjected to GO and KEGG enrichment analysis using the R package. The results of the enrichment were visualized the ggplot2 package in R. Protein interaction analysis was performed using String.',\n",
       " '11191305': 'Methods:This retrospective study conducted in the Kaiser Permanente Northern California region included 7698 patients who underwent Roux-en-Y gastric bypass (RYGB) or sleeve gastrectomy (SG) between January 2009 and March 2015. Trajectory analyses were conducted from 5-year follow-up data to assign patients to “low,” “average,” or “high” postoperative %TBWL groups. We then evaluated whether age, sex, race/ethnicity, neighborhood deprivation index and preoperative body mass index (BMI)/weight loss, diabetes, hypertension, and sleep apnea contributed to postoperative %TBWL using logistic regression models.',\n",
       " '11191266': 'METHODS Study Population All patients were members of the Kaiser Permanente Medical Care Plan, Northern California region (KPNC), an integrated healthcare delivery system that includes ongoing longitudinal electronic health records (EHR). Patients who entered the BS program were part of the KPNC BS cohort, which comprises over 18,000 adult men and women patients, as of 2023, whose primary operation was performed between 2009 and 2023. Patients from the KPNC BS cohort were recruited at their preoperative BS clinic appointments, and health information, including preoperative comorbidities, operative records, and weight loss metrics were collected in a similar manner for each patient. For this study, we identified 7698 patients among the KPNC BS cohort who underwent either RYGB or SG between January 5, 2009 and March 1, 2015. RYGB and SG patients were identified by current procedural terminology (CPT) procedure codes (eg, 43644, 43645, 43846, 43847, 43775, and 43999) which were recorded in the EHR system. As the SG procedure received a unique CPT code in 2010, we identified patients who underwent SG before that based on having a BMI > 30 kg/m2 and the partial gastrectomy CPT code. Further, these cases were done at KPNC medical centers performing BS and by surgeons identified as bariatric surgeons. This date range was chosen to ensure a follow-up period of 5 years and to avoid the potential influence of the COVID-19 pandemic on patient’s lifestyle changes in weight. The Institutional Review Board of the Kaiser Foundation Research Institute approved all study procedures. Written informed consent was obtained from all participants. Primary Outcome Postoperative %TBWL is the gold standard for measuring the extent of weight loss after BS.18–20 Postoperative %TBWL is calculated as follows: [(initial weight)–(postop weight)]/[(initial weight)] × 100. Socio-Demographic Factors Data on age at surgery, sex, and self-reported race/ethnicity were captured from the EHR for each patient. As an indicator of socio-economic status, we used a neighborhood deprivation index (NDI) based on the home address available in the EHR. Preoperative Clinical Factors Health information included preoperative BMI (assessed between the surgery date and 2 months prior), preoperative comorbidities (ie, diabetes, hypertension, and sleep apnea), and preoperative weight loss. Preoperative BMI and weight loss were obtained from the patients’ EHR. Diagnoses for preoperative diabetes (type 1 or 2), hypertension, and sleep apnea were based on the International Classification of Disease, Ninth (ICD-9) or Tenth Revision (ICD-10) diagnosis codes recorded in the EHR system. Postoperative Follow-Up of Weight Loss and Type of Surgery Patients were scheduled for follow-up visits at 6 months and 12 months after BS and then yearly thereafter. Information on the type of surgery (ie, RYGB or SG) and weight loss after surgery for the selected period of 5 years (including 6 postoperative follow-up time points: 6, 12, 24, 36, 48, and 60 months) were captured in a database internal to the KPNC BS cohort. Postoperative %TBWL Trajectory Group Analysis According to postoperative %TBWL based on the 6 postoperative follow-up time points, patients were assigned to 3 trajectory groups [ie, low weight loss (LWL), average weight loss (AWL), and high weight loss (HWL)] by semiparametric trajectory analysis. This semiparametric trajectory-based methodology has been shown to be an accurate method to estimate weight loss over time and has been largely used to analyze longitudinal weight loss data.2,6,9,21 Postoperative %TBWL trajectories were estimated for each patient by latent class growth analysis implemented using Proc Traj in SAS 9.4 (SAS Institute Inc.).22,23 Proc Traj can handle data that are missing completely at random, yet at least 3 time points per patient was set as inclusion criteria. Furthermore, to minimize classification error, follow-up time points with less than 50% completeness were excluded from weight loss trajectory analysis, and patients showing a posterior probability of class assignment less than 0.7 were excluded, as previously done.2,24 Consequently, we excluded 1469 patients from trajectory analyses due to follow-up attrition or posterior probability of class assignment <0.7. A total of 6229 patients were included in the trajectory analyses (Fig. 1), and the 6 follow-up time points (ie, 6, 12, 24, 36, 48, and 60 months) were further analyzed. Spaghetti plots showing the percentage of postoperative TBWL of the included patients over the 6 postoperative follow-up time points were generated in R (version 4.1.2) (https://www.R-project.org) using the “ggplot2” package. A Wald statistical test was used to determine whether the trajectory groups were significantly different. Patients from the AWL and HWL groups were reassigned into a combined (HWL+AWL) group to compare them with patients from the LWL group. The different analyses conducted in this study are summarized in an overview diagram (Fig. 1). FIGURE 1. Flow diagram of the study design and study participants included in the analyses. %TBWL indicates percentage total body weight loss; AWL, average weight loss; HWL, high weight loss; KPNC, Kaiser Permanente Northern California; LWL, low weight loss. Postoperative %TBWL Logistic Models Based on postoperative %TBWL trajectory groups, a predictive model of weight loss response to the BS, including socio-demographic and clinical variables was assessed by multivariate logistic regression. Socio-demographic (ie, age, sex, race/ethnicity, and NDI), preoperative clinical variables (BMI, weight loss, diabetes, hypertension, and sleep apnea), and type of BS were tested independently and added to a multivariate model when showing statistical significance (P < 0.05). Statistical Analyses In addition to the overall analyses, with RYGB and SG data combined, we also conducted stratified analyses by BS type (ie, RYGB or SG analyzed separately). Sex-stratified and race/ethnicity-stratified analyses were also conducted. All tests were 2-sided, and P < 0.05 was considered significant. Both binary logistic and multivariate models were performed using R (version 4.1.2) glm function.',\n",
       " '11191255': '2. Methods 2.1. Study design This MR study is divided into 2 phases (Fig. 1). In the first phase, we preliminarily analyzed the causal relationship between the gut microbiome, inflammatory proteins, and CD or UC, and then screened for candidate gut microbiomes and potential intermediary inflammatory proteins. In the second phase, we conducted multivariable MR analysis to examine whether, after adjusting for the mutual regulation between the gut microbiome and inflammatory proteins, a causal relationship with CD or UC still exists, and to determine whether the candidate inflammatory proteins serve as mediators between the gut microbiome and CD or UC, and calculate the proportion of mediation effect. Our MR analysis is based on the following 3 assumptions: The instrumental variables (IVs) are associated with the exposure; IVs are not related to any confounding factors; IVs only affect the outcome through exposure. Since genetic variations are formed randomly at conception according to Mendel laws, the outcomes of MR analysis tend to be minimally influenced by confounding factors. This study uses summary data from GWAS studies, mainly targeting research on Europeans by professional organizations. Figure 1. Mendelian randomization (MR) process diagram. CD = Crohn disease, IIBDGC = International Inflammatory Bowel Disease Genetics Consortium, IVW = inverse variance weighted, LD = linkage disequilibrium, MR-PRESSO = mendelian randomization pleiotropy residual sum and outlier, MVMR = multivariate mendelian randomization, SNPs = single nucleotide polymorphisms, UC = ulcerative colitis, WM = weighted median. 2.2. Data sources The gut microbiome data comes from the largest Genome-wide association studies (GWAS) dataset currently available from the MiBioGen consortium. This study encompasses 24 cohorts, totaling 18,340 individuals, and includes data on 131 genera. Our research will primarily investigate these genera.[26] Data for 91 inflammatory protein plasma pQTLs comes from a recent study by Zhao et al, which conducted a genome-wide pQTL mapping and meta-analysis of 91 inflammatory proteins across 11 cohorts of 14,824 Europeans.[27] Data on CD and UC comes from the International Inflammatory Bowel Disease Genetics Consortium (IIBDGC), incorporating 6968 UC patients and 20,464 controls, as well as 5956 CD patients and 14,927 controls for this study.[28] Meta-data comes from the study by de Lange KM, which included 12,194 CD patients and 28,072 controls, 12,366 UC patients and 33,609 controls.[29] 2.3. IVs selection We selected single nucleotide polymorphisms (SNPs) of the gut microbiome as IVs based on a P value threshold (<1 × 10−5), a commonly used threshold in previous gut microbiome studies. For selecting IVs for inflammatory proteins, we employed a P value threshold (<5 × 10−6). Linkage disequilibrium (LD) analysis was conducted using the European reference data from the 1000 Genomes Project. All these IVs were clustered within a ± 10000 kb distance with an R2 threshold of < 0.001 for LD. For palindromic SNPs, allele frequency information was used to identify the forward-strand alleles. Ultimately, F-statistics were calculated for all analyzed SNPs, with those having an F-statistic under 10 deemed weak instruments.[30] 2.4. MR analysis To preliminarily explore the potential causal links between gut microbiota, inflammatory proteins, and inflammatory bowel disease initially, we utilized the inverse variance weighted (IVW) method. This approach combines individual SNP Wald ratios to evaluate outcomes. We also determined the P value for Cochrane Q to evaluate result heterogeneity. In cases of heterogeneity, we applied the IVW random effects model; otherwise, we opted for the fixed effects model. Sensitivity and supplementary analyses were performed using the MR-Egger and weighted median (WM) methods, respectively. Additionally, the Mendelian randomization pleiotropy residual sum and outlier (MR-PRESSO) method was used to identify and adjust for potential outliers, thereby refining the causal effect estimations.[31] 2.5. Replication and meta-analysis To further validate the stability of the relationship between potential gut microbiota and inflammatory proteins discovered through the forward MR method, we replicated the IVW analysis on another dataset. Specifically, we initially conducted a preliminary analysis using data from the IIBDGC and replicated the analysis based on the study data of de Lange KM. By integrating the results of these 2 MR analyses, we ultimately confirmed specific gut microbiota and inflammatory proteins that have a causal relationship with CD and UC. During the meta-analysis, we selected the appropriate model based on the results of the IVW: a random-effects model in the face of result heterogeneity, and a fixed-effects model when the results were consistent. 2.6. MVMR and mediation analysis We first calculated the β1 values from the gut microbiota to the inflammatory proteins using 2-sample MR, then determined the β2 values from the inflammatory proteins to CD, and UC, we finally computed the β3 values from the gut microbiota to both CD and UC. Subsequently, we used multivariable MR (MVMR) to ascertain the causal relationship between the remaining exposures and CD, UC after adjusting for either gut microbiota or inflammatory proteins. After mutual adjustment, the β values for inflammatory proteins to CD, UC were designated as β4, and those for gut microbiota to CD, UC were designated as β5. The mediation proportion of inflammatory proteins was calculated using the formula “mediation effect proportion = β1β4/ (β1β4 + β5),” where β1β4 represents the mediating effect in the mechanism, β5 represents the direct effect in the mechanism, and β1β4 + β5 represents the total effect of the exposure in the mechanism (Fig. 2). Figure 2. MVMR and mediation effect calculation mediation effect proportion = β1β4/ (β1β4 + β5). MVMR = multivariable Medelian randomization.',\n",
       " '11191217': 'Materials and Methods Chemicals Lipopolysaccharide (LPS) from Escherichia coli O111:B4 (impurities ≤3.00% protein) (Merck Lifesciences, Madrid, Spain; Cat No:4357765). Standard Teklad diet (Envigo++, Barcelona, Spain; Cat No: Teklad 2014). Tenebrio molitor flour (Iberinsect, S.L; Reus, Spain), Alphitobius diaperinus flour (Protifarm NV, Ermelo, Gelderland, The Netherlands). The nutritional compositions of these two insect flours are described in Table 1. TRIzol reagent (Thermo Fisher Scientific, Waltham, MA, USA). Capacity cDNA Reverse Transcription kit (Applied Biosystems, Madrid, Spain), Specific TaqMan probes (Thermo Fisher Scientific, Madrid, Spain). Table 1 Nutritional Composition of the Administered Treatments Measured on Dry Matter (Values per 100 g of Insect Flour) composition A. diaperinus T. molitor energy (kJ) 2550 2604 protein (g) 56.31 56.1 total lipids (g) 18.82 26.31 starch (g) 1.30 3.34 fiber (g) 7.44 7.78 Experimental Design Forty female rats were acclimated for 14 days under standard conditions (22 °C with a 12 h light-dark cycle). During this period, they had ad libitum access to water and were fed a standard Teklad diet (Envigo++, Barcelona, Spain; Cat No: Teklad 2014). After this period of adaptation, the rats were divided into five experimental groups (8 rats per group), categorized into healthy and inflammatory conditions (Figure 1). The healthy condition included a control group (Control) fed a standard diet, a group supplemented with Tenebrio molitor flour (Tenebrio) (300 mg/kg bw/day), and a group supplemented with Alphitobius diaperinus flour (Buffalo) (300 mg/kg bw/day). On the other hand, the inflammatory condition included a control group receiving 5 days of intraperitoneal lipopolysaccharide (LPS) injections at 0.5 mg/kg of body weight (LPS group), and a group receiving both Tenebrio molitor flour and LPS (Tenebrio + LPS group). The intervention lasted for 26 days, and additional details of the experimental design have been described in a previous article.33 Figure 1 Experimental design. After sacrifice, blood was centrifugated to obtain plasma, while tissue samples, including various intestinal segments (duodenum, jejunum, ileum, ascending colon, and descending colon), were rapidly removed, weighed, and frozen in liquid nitrogen before storage at −80 °C until further analysis. After thorough cleaning and removal of fat, samples from the duodenum, jejunum, ileum, ascending colon, and descending colon were demarcated according to the intestinal segment division described by Vdoviaková et al.(41) All procedures were approved by the GENCAT Animal Experimentation Committee (number 11701). Gene Expression of Intestinal Taste Receptors Total RNA was extracted from the entire tubular tissue from each segment of the intestine using TRIzol reagent (Thermo Fisher Scientific, Waltham, MA, USA) and following the manufacturer’s instructions. Complementary DNA (cDNA) was obtained using a High-Capacity cDNA Reverse Transcription kit (Applied Biosystems, Madrid, Spain), as described in a previous study.42 Quantitative PCR amplification was performed from a total of 40 ng/μL of cDNA using TaqMan Universal PCR Master Mix (Applied Biosystems, Madrid, Spain) and specific TaqMan probes for PPIA (cyclophilin A) (Rn00690933_m1), Tas1r1 (Rn01516038_m1), Tas1r2 (Rn01515494_m1), Tas1r3 (Rn00590759_g1), Tas2r108 (Rn02396427_s1), Tas2r119 (Rn00576950_s1), Tas2r138 (Rn02396417_s1), Tas2r139 (Rn04218919_s1) and Tas2r143 (Rn02585801_s1) genes from Thermo Fisher Scientific. The relative expression of each gene was compared with the control group using the 2–ΔΔCt method43 and with the cyclophilin gene as the endogenous control gene.',\n",
       " '11191211': 'New methods of removing MPs Numerous policies and initiatives exist at both the domestic and global levels to mitigate pollution caused by MPs. On a worldwide scale, the United Nations has initiated efforts to combat MP pollution by launching the Clean Seas campaign, which aims to eliminate primary sources of plastic and MP pollution in the oceans (303). The campaign is centered on advocating for the decrease and eradication of disposable plastics, enhancing waste disposal practices, and raising public consciousness (303, 304). In Europe, the European Union has enforced a prohibition on using MPs in personal care items, including facial cleansers and toothpaste (305, 306). The EU has also suggested implementing a prohibition on disposable plastics, encompassing items like utensils and drinking straws (307). Various strategies are used in MP removal and elimination procedures to address the problem of MP contamination (308). Studies on the degradation of MPs have advanced with a specific emphasis on biological and non-biological methodologies. The utilization of microorganisms such as algae, bacteria, and fungi for the degradation of MPs is viewed as a promising method for cost-efficient and environmentally friendly treatment strategies (309). The process of wastewater treatment is essential for the effective capture and removal of MPs from wastewater before its release into aquatic environments. Sophisticated treatment methods such as membrane filtration and activated sludge systems are utilized for this objective (308). Various filtration systems such as sand filters, mesh screens, and activated carbon filters are employed to capture larger MP particles from water sources. In regions characterized by elevated levels of MP accumulation, floating boom systems may be utilized to confine and retrieve floating plastic waste, which encompasses MPs. One commonly employed approach involves manually extracting visible plastic waste from rivers, shorelines, and beaches as part of clean-up efforts. Moreover, novel technologies such as electrocoagulation, magnetic nano adsorbents, and ultrasonic treatment are currently under investigation for their potential to improve the efficiency of removing MPs (310). The process of biological decomposition of MPs involves the presence of numerous enzymes (311, 312). Various extracellular enzymes such as lipases, esterases, laccases, lignin peroxidases, and manganese peroxidases are crucial in the degradation of MPs. These enzymes enhance the hydrophilicity of MPs and transform them into carbonyl or alcohol residues (313). Hydrolase enzymes, including esterases, lipases, and cutinases, facilitate the degradation of MPs on plastic surfaces by promoting chain cleavage reactions. These enzymes are unable to penetrate the polymer matrix. However, they exert their catalytic activity on the surface, leading to the development of fissures. The resulting monomers are absorbed into the cytoplasm of microorganisms and subsequently participate in various metabolic pathways (309). In the following, we will discuss novel methods for MP removal by algae, fungi, and bacteria.',\n",
       " '11191200': 'METHODS: The data set of colorectal cancer was downloaded from the GEO database (GSE21510, GSE110224 and GSE74602) for differential expression analysis using the GEO2R tool. Hub genes were screened by protein-protein interaction (PPI) comprehensive analysis. GEPIA was used to verify the expression of Hub genes and evaluate its prognostic value. The protein expression of Hub gene in CRC was analyzed using the Human Protein Atlas database. The cBioPortal was used to analyze the type and frequency of Hub gene mutations, and the effects of mutation on the patients’ prognosis. The TIMER database was used to study the correlation between Hub genes and immune infiltration in CRC. Gene set enrichment analysis (GSEA) was used to explore the biological function and signal pathway of the Hub genes and corresponding co-expressed genes.',\n",
       " '11191164': '2. Materials and methods 2.1 Data source As one of the biggest collections of gene chips in the world, the GEO database is a comprehensive gene expression library at the National Center of Biotechnology Information (NCBI) (https://www.ncbi.nlm.nih.gov/ geo/). Expression profiles of GSE21510 and GSE110224 based on the GPL570 platform and GSE74602 based on the GPL6104 platform were obtained from the GEO database. GSE21510 contained 123 CRC samples and 25 normal tissues; GSE110224 included 17 CRC samples and 17 normal tissues, and GSE74602 contained 30 CRC samples and 30 normal tissue. Figure 1 shows the overall flowchart of the study. Figure 1.The flow diagram of study. 2.2 Differential expressed gene analysis The differentially expressed genes (DEGs) were filtered using a threshold of adjusted P values < 0.05 and an absolute log2FC (fold change) > 1 by GEO2R online tools (http://www.ncbi.nlm.nih.gov/geo/geo2r) [22]. Venn analysis was used to select overlapping DEGs among the three datasets mentioned above. 2.3 GO and KEGG pathway analysis To further understand the biological functions and related signaling pathways of the DEGs, we performed Gene Ontology (GO) enrichment analysis and Kyoto Encylopaedia of Genes and Genomes (KEGG) pathway enrichment analysis using the Metascape (http:// metascape.org/) [23, 24]. An adjusted P value of < 0.01 was considered to identify the enriched terms, and the results were visualized. 2.4 Protein-protein interaction (PPI) comprehensive analysis PPI comprehensive analysis was performed using the online tool Search Tool for the Retrieval of Interacting Genes (STRING, https://string-db.org/) [25, 26]. The selected Hub genes were imported into the STRING, and the confidence score of > 0.4 was considered significant. Then, the PPI network information was obtained. Next, the data downloading from STRING was further analyzed and visualized using Cytoscape (version 3.8.2). The core function of Cytoscape is to provide basic functional layout and query network, and build a PPI network based on the combination of basic data into a visual network. The app MCODE plugged into Cytoscape was used to identify the paramount modules in the PPI network (MCODE score > 5, degree cutoff = 2, maximum depth = 100, K-core = 2, node cutoff = 0.2). Finally, the visual network graph was presented in a circle layout according to the score. 2.5 Hub gene selection and analysis of survival and differential expression The PPI analysis of DEGs was performed using the online tool STRING, and results with a minimum interaction score of 0.4 were visualized in Cytoscape software (version 3.8.2). The obtained results were used to select the Hub genes through the MCC algorithm in the Cytoscape cytoHubba application, and the top 30 genes were selected as potential Hub genes. Gene Expression Profiling Interactive Analysis (GEPIA, http://gepia.cancer-pku.cn/) is a web server that analyzes cancerous and normal gene expression profiles and interactions in The Cancer Genome Atlas (TCGA, https://genomecancer.ucsc.edu/) and the Genotype Tissue Expression (GTEx, https://gtexportal.org/ home/) projects [27]. The overall survival of colorectal cancer patients and the expression level validation of Hub genes were evaluated using GEPIA. P< 0.05 was defined as the significant threshold to indicate candidate Hub genes. Genes not statistically significant were removed (P> 0.05). We also analyzed the impact of the expression levels of Hub genes on the disease free survival (DFS) of CRC patients using GEPIA. In addition, the expression levels in CRC from the GEO database were analyzed by Graphpad Prism 8 and scatter plots were drawn. The Tumor Immune Estimation Resource (TIMER) database was also used to explore the gene expression in tumors and adjacent normal tissues [28]. The differences of gene expression among different subtypes of CRC (POLE, MSI, CIN and GS) were investigated using R software. 2.6 Protein expression and receiver operating characteristic (ROC) curves The protein expression levels of Hub genes in human normal and cancer tissues were determined using the Human Protein Atlas (HPA) database (https://www.prot einatlas.org/), which respectively displays the expression of proteins in cells, normal tissues, and cancerous tissues. Receiver operating characteristic (ROC) curves were obtained from the online tool Xiantao Academy (https://www.xiantao.love/). 2.7 Gene mutation status and survival analysis cBioPortal (http://cbioportal.org) is an open-access resource for exploring, visualizing and analyzing multi-dimensional cancer genome data [29]. The cBioPortal was used to analyze the type and frequency of Hub gene mutations, and the effects of mutation on the patients’ prognosis. 2.8 Immune infiltration analysis The TIMER database contains 32 cancers and 10,897 tissue sample information from the TCGA database. It can realize systematic analysis of the correlation between immune infiltrates and other wide spectrum of factors, including related gene expression in tissues and prognosis, gene mutations and copy number of cancer patients [28]. In this study, we evaluated the infiltration of immune cells (CD8+ T cells, CD4+ T cells, B cells, dendritic cells, macrophages, and neutrophils) in CRC patients through TIMER database. The relationship between gene expression and tumor purity was also explored and visualized. 2.9 Co-expression analysis The Linked Omics database (http://www.linkes.org/) is a web-based platform for analyzing 32 TCGA cancer-related cubes [30]. The Link Finder module of Linked Omics was used to study the differentially expressed genes related to selected Hub genes in the TCGA-COAD, and the Pearson correlation coefficient was used for statistical analysis. All results are presented graphically in a volcano map, heat map or scatter plot. The Link-Interpreter module of Linked Omics performs pathway and network analysis of differentially expressed genes. Use the comprehensive functional classification database in the Web-based Web Gestalt to sign and sort the data in the Link Finder results, and use Gene set enrichment analysis (GSEA) to analyze the GO (CC, BP, MF) and KEGG channels. The GSEA program was run with 500 simulations, and the significance level was top 25. P value and false discovery rate (FDR) were both less than 0.05, the gene set was considered significantly enriched. 2.10 Statistical analyses Data are represented as the mean ± standard deviation (SD), and the t-test was used for comparisons between the two groups. GraphPad Prism 8.3.1 (GraphPad Software, Inc., San Diego, CA, United States) was utilized for statistical analysis and generating graphs. P< 0.05 was considered statistically significant.',\n",
       " '11191077': 'Material and methods Melyridae species collection Melyridae beetles were collected during the summers of 2011–22 from various locations (Table 1). Individuals were stored either in ethanol or dry at −20°C or −80°C, depending on future use. Species from the USA and Malachius bipustulatus were identified based on morphology and species from Europe by barcoding of the CO1 gene (see below for details). Table 1 List of Melyridae (and one Lophocateridae) species screened for the Dasytinae-specific symbiont Candidatus Dasytiphilus stammeri. Taxa are ordered by clade in the host phylogeny, separated by lines, and then alphabetically. The presence of Dasytiphilus was assessed by PCR, Sanger sequencing, FISH, and Illumina 16S rRNA gene amplicon profiling; empty cells indicate that this method was not used to assess symbiont presence in the respective host species. “Genome information” gives information about the results of Illumina shotgun sequencing of the symbiont genome, with length in base pairs and the GC content. Host Collection Symbiont presence confirmation method Genome information Genus Species Country Symbiont present PCR Sanger FISH Amplicon Genome available Genome length (bp) GC% Eronyxa pallida USA, CA No No No No Eronyxa pallida Malachius bipustulatus Germany No No Malachius viridulus USA, CA No No Malachius viridulus Dasytastes bicolor USA, CA Yes Yes Yes Yes 589 892 33.0 “Dasytes” lineellus USA, CA Yes Yes Yes Yes Yes 585 061 32.6 “Dasytes” seminudus USA, CA Yes Yes Yes Yes Yes (draft) ~586 000 32.8 Dasytastes sp. 01 USA, CA Yes Yes Yes Yes Yes 600 652 33.8 Dasytastes sp. 02 USA, CA Yes Yes Yes Yes Yes 592 705 33.1 Enallonyx sculptilis USA, CA No No No No Eschatocrepis constrictus USA, CA No No No No No Gracilivectura pygidialis USA, CA No No No No No Listrimorpha pallipes USA, CA No No No No Vectura longiceps USA, CA No No No No Vecturoides sp. USA, CA No No Vecturoides sp. Danacea nigritarsis GER Germany Yes Yes Yes Yes Yes 497 568 28.8 Danacea nigritarsis ITA Italy Yes Yes Yes Yes Yes Yes 497 621 28.8 Danacea pallipes Germany Yes Yes Yes Danacea pallipes Asydates grandiceps USA, CA No No No No Asydates ruficauda USA, CA No No No No Byturosomus fuscus USA, CA No No No No Cradytes serricollis USA, NM No No No No Cradytes serrulatus USA, AZ No No No No Eudasytes grandicollis USA, NV No No No No Listropsis sp. USA, CA No No No No Listrus sp. 00 USA, CA Yes Yes Yes Yes Yes (draft) ~511 000 32.5 Listrus sp. 01 USA, AZ Yes Yes Yes Yes 440 755 28.3 Listrus sp. 02 USA, CA Yes Yes Yes 513 178 33.3 Listrus sp. 03 USA, NV Yes Yes Yes Listrus sp. 04 USA, UT Yes Yes Yes (draft) ~515 000 32.9 Listrus sp. 05 USA, CA Yes Yes Yes Listrus sp. 06 USA, CA Yes Yes Yes 509 826 32.6 Listrus sp. 07 USA, CA Yes Yes Yes (draft) ~461 000 33.3 Listrus sp. 08 USA, CA Yes Yes Yes Listrus sp. 09 USA, OR Yes Yes Yes 518 020 32.9 Listrus sp. 10 USA, OR Yes Yes Yes Yes Microasydates santabarbara USA, CA No No No No Microasydates umbratus USA, CA No No No No Pseudasydates explanatus USA, CA No No No No Pseudasydates sp.n. USA, NM No No No No Trichochrous aenescens USA, CA No No No No Trichochrous brevicornis USA, CA No No No No Trichochrous convergens USA, AZ No No No No Trichochrous sp.n. USA, CA No No No No Trichochrous egenus USA, CA No No No No Trichochrous fulvotarsis USA, CA No No No No Trichochrous fulvovestitus USA, CA No No No No Trichochrous pallescens USA, CA No No No No No Trichochrous quadrinotatus USA, CA No No No No Trichochrous seriellus USA, UT No No No No Trichochrous sordidus USA, CA No No No No Trichochrous sp.n. Trich076 USA, CA No No No No Trichochrous sp.n.Trich076 Dasytes aeratus Germany Yes Yes Yes Yes Yes 455 894 30.9 Dasytes alpigradus Italy Yes Yes Yes Yes 476 890 30.0 Dasytes niger Germany Yes Yes Yes Yes Yes Yes 486 045 30.9 Dasytes plumbeus Germany Yes Yes Yes Yes Yes Yes 456 731 30.4 Dasytes subaeneus Germany Yes Yes Yes Dasytes tristiculus Italy yes yes yes yes yes Dasytes virens Germany yes yes yes yes yes yes 456 754 30.4 Dolichosoma lineare Germany yes yes yes yes Psilothrix viridicoerulea Germany yes yes yes yes yes yes 464 075 28.7 DNA extraction Head and leg tissue samples taken from the insects were extracted for the identification of host species. For diagnostic PCRs for the presence of Dasytiphilus, bacterial community profiling, and genome sequencing, either dissected bacteriomes or, if not possible, entire host abdomens were extracted in order to obtain bacterial symbiont DNA (Fig. S1). Extractions were done using the Epicentre MasterPure™ Complete DNA and RNA Purification Kit (Illumina Inc., Madison, WI, USA) following the manufacturer’s instructions, including RNase digestion. DNA of samples intended for genome sequencing had its purity assessed using a NanoPhotometer P330 (Implen, München, Germany) and DNA quantity was measured using a Qubit dsDNA BR assay kit on a Qubit 2.0 Fluorometer (Invitrogen by Thermo Fisher Scientific, MA, USA). Diagnostic PCR and Sanger sequencing to assess the presence of Dasytiphilus General eubacterial and diagnostic PCRs were performed to assess Dasytiphilus presence and were done on a T-Professional-Gradient Thermocycler (Biometra, Göttingen, Germany) using reaction mixes of 6.9 μl ultrapure H2O, 1.25 μl of 10× reaction buffer, 0.25 μl 25 mM MgCl2, 1.5 μl 2 mM dNTPs, 1 μl of both forward and reverse primer (each 10 pmol/μl), 0.1 μl of 5 U/μl Taq DNA polymerase and 1 μl template. Alternatively, a Mastercycler EP Gradient S Thermocycler (Eppendorf AG, Hamburg, Germany) was used with a reaction mix containing 9.5 μl ultrapure H2O, 12.5 μl of Q5® High-Fidelity 2X Master Mix (NEB), 1 μl of both forward and reverse primer (each 10 pmol/μl), and 1 μl template. Primers used for Dasytiphilus identification were the eubacterial primer pairs fD1 and rP2 as well as slightly modified versions fD1_Mely and rP2_Mely. Moreover, the symbiont-specific primer pair Dasy_Sym_fwd3 and Dasy_Sym_rev3 was utilized (Table S1). To guarantee that Dasytiphilus sequences were amplified, Sanger sequencing was conducted following general eubacterial and diagnostic PCRs. Products were purified using the innuPREP PCRpure Kit (Analytik Jena, Jena, Germany) according to the manufacturer’s instructions and were subsequently sequenced by a commercial service (StarSeq, Mainz, Germany). Alternatively, products were purified with the Zymo Research DNA Clean & Concentrator-5 kit (Zymo Research, Irvine, CA, USA) following the manufacturer’s protocol and were sequenced in house with an AB Hitachi 3730XL DNA Analyzer (Applied Biosystems by Thermo Fisher Scientific, MA, USA). Microbial community profiling by 16S rRNA gene amplicon sequencing Bacterial community composition was evaluated via high-throughput amplicon sequencing of the 16S rRNA gene. Samples were sequenced in a paired end approach with read lengths of 300 nt by a commercial provider (StarSeq, Mainz, Germany) on an Illumina MiSeq platform (Illumina Inc., San Diego, CA, USA) using V3 reagent and 25% PhiX to balance the composition of bases. Amplified regions were either V3–V4 region with primers 341f and 806bR, V4 region with primers 515F and 806bR or V4–V5 region with primers 515F and 909R (Table S1). This was due to temporal differences in sample availability and constraints in assay design due to concurrent sequencing projects. A comprehensive list of all sequenced Melyridae samples with the amplified region can be found in Table S2. No-template DNA extractions were sequenced as negative controls. Based on the sequenced Illumina reads, amplicon sequence variants (ASVs) were obtained after read trimming depending on the amplified region, quality filtering, dereplicating, and chimera removal in R utilizing the package DADA2 [38]. Subsequently, the pretrained classifier Silva 138.1 was used to assign taxonomy [39, 40]. Finally, reads identified as chloroplast or mitochondria were removed prior to analysis. Symbiont localization via fluorescence in situ hybridization To localize the symbionts in multiple species of Melyridae beetles, FISH was performed on specimens of 13 different Dasytinae species and M. bipustulatus as an outgroup (Table 1). Whole beetles or abdomens were individually fixed in Carnoy’s solution (67% ethanol, 25% chloroform, and 8% glacial acetic acid), dehydrated in a series of ascending concentration (30%, 50%, 70%, 80%, 90%, and 96%) of n-butanol and subsequently embedded in Technovit 8100 (Heraeus Kulzer) following the manufacturer’s instructions. Transversal or sagittal histological sections of 8 μm thickness were prepared utilizing a glass knife on a Leica RM 2245 microtome and transferred to microscope slides. To each slide, 100–150 μl hybridization mix was applied and then covered with a glass cover slip. The hybridization mix contained hybridization buffer (0.9 M NaCl, 0.02 M Tris/HCl (pH = 8), 0.01% SDS), 0.5 μM of the fluorescently labeled oligonucleotide probes EUB338-Cy3 and Dasy_ent_Cy5 (Table S1), and 0.5-mg/ml DAPI for host cell counterstaining. Slides were hybridized overnight at 50°C in a humid chamber. Afterwards, the glass cover slip was removed and the slides were washed at 50°C for 2 h submerged in wash buffer, followed by an additional washing step in distilled water at 50°C for 20 min. The wash buffer consisted of 0.1-M NaCl, 0.02-M Tris/HCl (pH = 8), 5 mM EDTA, and 0.01% SDS. After washing, 30 μl of VectaShield was applied to each slide upon which a glass cover slip was then placed. Samples were observed with a Thunder Imaging System (Leica, Wetzlar, Germany). The signals for Cy3, Cy5, background, and DAPI were acquired with the 555-, 635-, 475-, and 390-nm EFW LED8 respectively, at 50% (Cy3, Cy5, background) power or 5% power (DAPI), the DFT51010 filter cube, and a 590-, 642-, 535-, and 460-nm fast emission filter, respectively. Finally, images were processed in the Leica Application Suite X software (Leica, Wetzlar, Germany) with the instant and small volume computational clearing algorithms. Symbiont genome sequencing and assembly For symbiont genome reconstruction, short-read shotgun library preparation and sequencing was performed by the Max Planck Genome Center (Cologne, Germany) utilizing a HiSeq 3000 or NextSeq 2000 system (Illumina Inc., San Diego, CA, USA) or by a commercial service (CeGaT GmbH, Tübingen, Germany) using a NovaSeq 6000 platform (Illumina Inc., San Diego, CA, USA) (Table S3). Obtained paired sequence reads of 150 bp each were uploaded to KBase [41], and read quality was assessed with FastQC v0.11.5-v0.11.9. The reads were trimmed using Trimmomatic v0.36 [42], and trimmed reads were assembled with metaSPAdes v3.13.0-v3.15.3 [43] and MEGAHIT v1.2.9 [44]. For Listrus sp. 02, long reads were used in addition to the Illumina reads to assemble the symbiont genome. For this, high-molecular-weight (HMW) genomic DNA was selectively extracted with the Short Read Eliminator Kit XS (Circulomics, MD, USA), to enrich for fragments longer than 10 kb. The size-selected HMW DNA was used as the starting material for the preparation of an ONT library, following the manufacturer’s guidelines for the Ligation Sequencing Kit SQK-LSK110 (Oxford Nanopore Technologies, Oxford, UK), and sequenced on R9.4.1 (FLO-MIN106) flow cells using the MinION sequencing device with MinKNOW Software version 22.10.7. Super high-accuracy (SUP) base-calling was performed using GUPPY version 5.0.14. Flye (v2.9.1) assembler [45, 46] in meta-mode was used to generate a de novo genome assembly from the ONT data, followed by four iterations of polishing using Racon and one round of error correction using Medaka. For final error correction of the ONT-based assembly, we used ntEdit with paired-end (2× 150 bp) Illumina data generated from the same HMW DNA used for ONT sequencing. To remove duplications (heterozygous regions) and generate haploid genome(s) for further downstream analysis, we used purge_haplotigs. Symbiont genome analysis After assembly, the received contigs were annotated with Prokka v1.14.5 [47] and Rast v1.073 [48]. Symbiont contigs were identified in Geneious (Geneious Prime various versions between 2019 and 2023) based on length, GC content, read coverage, and gene synteny. Contigs from both assembly methods were de novo-assembled, resulting in either closed genomes or high-quality draft genomes. Minor polishing was done on selected genomes by read mapping and pilon genome improvement [49]. Genomes were analyzed with the help of KEGG: Kyoto Encyclopedia of Genes and Genomes [50–52]. Gene synteny was visualized with Clinker [53], showing only the highest similarity links between genes. When analyzing specific symbiont capabilities, we focused on pathways that could be directly relevant to the interaction with the host. Symbiont phylogenetic reconstruction The symbiont phylogeny was reconstructed in KBase [41] with the plugin “Insert Set of Genomes Into SpeciesTree – v2.2.0.” The plugin took user-provided genomes and constructed alignments of 49 core, universal genes defined by COG (Clusters of Orthologous Groups) gene families with publicly available genomes of closely related bacteria. Subsequently, it used an approximately-maximum-likelihood algorithm to create a phylogenetic tree [54]. dN/dS of lysine pathway genes We evaluated the selective forces acting on lysine pathway genes of Listrus spp. more closely by examining the ratio of synonymous vs nonsynonymous substitutions in order to assess whether substitution patterns differed from those of lysine pathway genes of Dasytastes clade symbionts. To infer orthologous genes, orthogroups were reconstructed using OrthoFinder v2.5.4 [55] with the MSA option enabled using muscle v5.1 [56] and FastTree v2.11.1 [54]. Orthologs for each of the candidate genes were extracted from the phylogenetic hierarchical orthogroups results and used for downstream analysis. For each candidate gene, orthologous amino acid sequences were aligned with muscle v5.1 with default parameters. Then, the corresponding nucleotide sequences were codon-aligned based on the protein alignments using the pal2nal v14 script [57]. Phylogenetic trees were reconstructed using FastTree v2.11.1 based on codon alignments. Both tree and alignment were used to detect changes in the selective pressures acting on Listrus spp. genes compared to the outgroup. Branches corresponding to Listrus spp. were marked using the vision.hyphy.org tool and the “select descendant branches” enable. Eventual relaxation or intensification of the selection strength was tested using RELAX from HYPHY package v2.5.33 [58]. Additionally, a test for positive selection was conducted using aBSREL [59] from HYPHY package. The P-value threshold for considering tests as significant was set at 0.05. Host identification and phylogenetic reconstruction Host species identification To robustly link host species with symbiont presence/absence, we attempted to identify each analyzed host taxon. Host species that could not be identified based on morphology were identified via DNA barcoding, i.e. sequencing of the cytochrome c oxidase subunit I gene (CO1). In this gene, >2% sequence divergence is usually recorded between closely related animal species, so the mutation rate is high enough to distinguish between species [60]. The CO1 gene was amplified using PCR with primers LepF1 and LepR1 (Table S1), and the product was purified and subjected to Sanger sequencing (see above for details). However, several species could not be identified to a described species because no conspecific reference sequence was available and the necessary modern taxonomic revisions have not been completed. For the genus Listrus, in particular, species are poorly characterized and undescribed species certainly exist; as a consequence, only morphospecies designations were given for members of this genus. Host phylogenetic reconstruction To infer the evolutionary history of the symbiosis with Dasytiphilus, we reconstructed a phylogeny of Dasytinae beetles and outgroup taxa based on the sequences of up to four marker genes: 18S rRNA gene, 28S rRNA gene, CO1, and Cytochrome b gene (cytB). Where possible, sequences were obtained from the assembled shotgun sequencing data; otherwise, the respective regions were amplified by PCR and Sanger-sequenced. The partial 18S rRNA gene (~1400 bp) was amplified with primer pair 18S_ai and 18S_3’I. Three individual pieces were sequenced with primers 18S_ai, 18S_a1.0, and 18S_3’I, respectively, and subsequently assembled to cover the complete amplified region. The partial 28S rRNA gene (~600 bp) was amplified with primer pair 28Sff and 28Srr and sequenced with 28Sff. For the CO1 gene, either one long sequence (~1400 bp) was amplified with primer pair LepF1 and Pat or two smaller overlapping fragments were amplified, the first one with primer pair LepF1 and LepR1 and the second one with primer pair Melyridae_CO1_F1 and Pat. Following this, three pieces were sequenced with primers LepR1, Melyridae_CO1_F1 and Pat and assembled to obtain a contiguous sequence. The partial cytB gene (~300 bp) was amplified with primer pair Sytb_F and Sytb_R and sequenced with Sytb_F. Additionally, previously published sequences for some species were used [28]. For each gene, an alignment was created using Geneious Prime 2023.1.2. The four alignments were concatenated and used for phylogenetic reconstruction via Bayesian analysis [61] utilizing the MrBayes plugin v3.2.6 in Geneious Prime 2023.1.2. Through the custom command block, the alignment was partitioned by gene, and for the protein-coding genes CO1 and cytB, also by codon position (codon positions 1 and 2 in one partition, position 3 in another partition). From a random starting tree, two runs with four chains were run over 20 million generations, until the standard deviation of split frequencies was consistently below 0.01, using a 4by4 GTR model and gamma-shaped rate variation. Sampling was done every 2000 generations, and a consensus tree was created after 25% burn-in.',\n",
       " '11190884': 'Methods Animals UOX−/− rats were obtained by intercrossing UOX+/− rats (Nanjing Biomedical Research Institute of Nanjing University, China). Six-week-old C57BL/6J mice were obtained from the Guangdong Medical Laboratory Animal Center (Guangzhou, China). NLRP3−/− mice were purchased from Cyagen Biosciences (Suzhou, China). To reduce bias based on genetic background, littermate controls were used for all experiments with genetically altered lines. Animals were kept under an automated 12-h light-dark cycle at a controlled temperature of 22 ± 2 °C, relative humidity of 50–60%, and had ad libitum access to a standard dry diet and tap water. The study was approved by the Standards for Animal Ethics in the Southern Medical University and performed in accordance with the relevant experimental animal guidelines and regulations. Serum UA and renal function parameters measurement Serum UA was determined by a commercial kit (BioAssay Systems, CA, USA). A urine sample from each rat was collected over 24 h in a metabolic cage. SCR, BUN, and urine protein levels were determined using the respective analysis kits (Nanjing Jiancheng Bioengineering Institute, Nanjing, China). Urinary NAGL, KIM-1, RBP, and β2-MG levels were measured using respective rat ELISA quantitation kits, according to the manufacturer’s protocol (Boshen, Nanjing, China). Measurement of serum cytokines, LPS, and IS ELISA kits were used to measure MCP1, IL-6, IL-1β (Boshen, Nanjing, China), LPS (Thermo Scientific, MA, USA,) and IS (Mlbio, Shanghai, China) in serum according to the manufacturer’s instructions. Histology and immunohistochemistry Kidneys and colons were embedded in paraffin and sliced into 4-μm-thick sections by a routine procedure. H&E, PAS, Masson trichrome, and AB staining were conducted using standard protocols. Images were captured using a microscope and quantifies using the ImageJ software. Tubular injury (denudation of tubular cells, loss of brush border, flattening of tubular cells, formation of intratubular casts) was scored from 0 to 4 (0, no changes; 1, changes affecting < 25%; 2, changes affecting 25 to 50%; 3, changes affecting 50 to 75%; 4, changes affecting 75 to 100% of the section). Kidney sections were also obtained from absolute ethanol-fixed kidneys to detect uric acid crystals under polarized light. Immunohistochemical detection of inflammation was performed using primary antibodies against macrophage marker CD68 (1:200, GB113109, Servicebio). Renal fibrosis was detected by collagen III (1:200, GB111629, Servicebio) staining.',\n",
       " '11190797': 'MethodsWe applied two-sample MR to identify the causal relationship between peripheral blood immune cells and CRC. GWAS data were obtained from the IEU OPEN GWAS project. Based on the implications from the MR results, we conducted a comprehensive database search and genetic analysis to explore potential underlying mechanisms. We predicted miRNAs for each gene and employed extensive research for potential therapeutic applications.',\n",
       " '11190764': \"Materials and methods Study design and ethics permit The overall study design is presented in Fig. 1. The study was based on publicly available data from large-scale genome-wide association study on immune cell and colorectal cancer. Included studies had been approved by corresponding ethical review committees.Fig. 1Study design. NEK7, NIMA related kinase 7; LHX9, LIM homeobox 9; GWAS, genome-wide association study Immune cell data sources Immune traits statistics of Treg cells and DN cells were extracted from a large-scale genome-wide association study [19], which are publicly available from the GWAS Catalog (https://www.ebi.ac.uk/gwas/). After filtering for the cell of interest, 91 traits were saved for further study, including absolute cell (AC) counts (n = 17), median fluorescence intensities (MFI) counts (n = 40) and relative cell (RC) counts (n = 34). These traits include Treg, Maturation stages of T cell and TBNK panel and relevant information are presented in Supplementary Table S2. It is worth noting that we only included Treg cells as defined by conventional criteria, thereby excluding other cell populations that may exhibit high expression of CD25 or other characteristics but are not classified as Treg cells. Furthermore, the dataset does not include double-negative B cells, hence a more detailed analysis of this subtype was not included. The original study was based on a population cohort of 3,757 Sardinians that tests approximately 20 million SNPs in 731 immunophenotypes genotyped with high-density arrays or specific sequence-based reference panel. Genome-wide association study (GWAS) data sources for CRC We extracted genome-wide association study(GWAS) summary data for CRC from two different study on the UK Biobank [20, 21] and a cross-population analysis [22] through the IEU Open GWAS project (https://gwas.mrcieu.ac.uk/). Detailed information was documented in Supplementary Table S1. The UK Biobank is a population-based multicenter cohort study consisting of approximately 500,000 participants between ages 40 and 70. One study performed GWAS for pan-cancer and site-specific cancers identified by linkage to the UK CancerRegistry (updated to April 2019). After association statistics, a total number of 377,673 European individuals (Ncase = 5,657, Ncontrol = 372,016) were involved. Another UKB study introduced a novel and efficient machine-learning method named REGENIE and demonstrated the accuracy on 407,746 individuals (case–control ratio = 1:97). The cross-population summary data published in 2021 were performed on 470,002 European (Ncase = 6,581, Ncontrol = 463,421) and 167,691 East Asian (Ncase = 8,305, Ncontrol = 159,386) individuals with approximately 24 million variants detected. Mendelian randomization analysis Mendelian randomization is an epidemiological approach that employs genetic variants as instrumental variables to infer causality between exposures and outcomes. This method relies on the principle that alleles are randomly assorted during gamete formation, mirroring a randomized trial's structure. It assumes that the genetic variants are associated with the exposure of interest, affect the outcome only through the exposure, and are not confounded by other factors. We performed two-sample MR analysis to capture the causal association between immune cells and colorectal cancer based on TwoSampleMR R package (github.com/MRCIEU/TwoSampleMR, version 0.5.7). MR analyses were conducted on each dataset acquired for CRC, and the results were recorded in Supplementary Table S3 ~ S6. We employed inverse variance weighting (IVW) as the primary approach and supplemented with other robust tests, including MR-Egger, weighted median and weighted mode methods, which rely on different assumptions than IVW. To avoid the effect of horizontal pleiotropy, MR-egger, leave-one-out and MR-PRESSO were adopted while Cochran’s Q statistic and p values were used to test the heterogeneity of all eligible instrumental variables (IVs). After excluding all SNPs with potential pleiotropy indicated by MR-PRESSO, we conducted a comprehensive analysis to ensure that the results were not influenced by pleiotropy. In accordance with recent studies, a significance level of 1 × 10^-5 was applied to the instrumental variables for each immune trait, along with a stringent clumping threshold (linkage disequilibrium [LD] r^2 threshold < 0.001 within 10,000 kb distance). For CRC, the selected SNPs were required to have a significance level below 5 × 10^-8. We employed Steiger's test and calculated the F-statistics to ensure all SNPs possess strong efficacy. Additionally, we utilized forest plots and scatter plots to assess and visualize the correlation of SNPs (Supplementary Figure S1-S2). SNP mapping and analysis We employed several database to obtain relevant information for candidate SNPs, including the Functional Mapping and Annotation (FUMA) [23] and GWAS Catalog. By incorporating a multitude of biological resources and integrating associated data, FUMA enhances the richness and significance of results for GWAS studies. We conducted our analysis using the default settings of FUMA's SNP2GENE module, such as employing the 1000 Genomes Project Phase 3 reference panel for LD calculations with a threshold of 250 kb for LD blocks. The GWAS Catalog contains a vast array of data from genome-wide association studies, including the localization information for many SNPs. The functionality of these SNPs has been thoroughly validated by other research, lending them greater credibility. Therefore, we retained the genes identified by both FUMA and the GWAS Catalog for subsequent analysis. We also conducted an extensive document retrieval on the identified gene to verify and explore a potential role in tumorigenesis. Peripheral blood gene expression data from CRC patients and healthy individuals were obtained from the Gene Expression Omnibus (GEO) database (https://www.ncbi.nlm.nih.gov/geo/), and relevant information were documented in Supplementary Table S1. We also utilized GEO2R for differential gene expression analysis and performed visualization in R. xCell is a robust gene signature method that has capability to identify up to 64 immune and stromal cell types in humans across various data sources. The correlation between immune cell types and gene expression levels was confirmed by combining xCell analysis with gene expression profiles, and the results were visualized in figures. xCell [24] analysis was conducted using the xCell R package (v1.2.0), and all relevant R code is publicly available on GitHub. miRNA targeting The miRNet platform is a comprehensive network visualization tool that integrates 14 different databases, including TarBase, miRTarBase, miRecords, miRanda, miR2Disease, HMDD, PhenomiR, SM2miR, PharmacomiR, EpimiR, starBase, TransmiR, ADmiRE, and TAM 2.0. We use the miRNet database (https://www.mirnet.ca/) to identify corresponding miRNAs in peripheral blood and visualized the result in a circular chart. To assess their clinical relevance and expression in blood, we referred to the OncomiR [25] and dbDEMC database [26]. In the OncomiR database, we set the significance threshold at 0.05 for each section. The analysis was performed on both the tumor stage & grade part and the expression part, including data from both COAD and READ. In the dbDEMC database, the focus was on examining the miRNA expression profile in blood. To be considered qualified, the miRNAs had to meet specific criteria, including experimental validation of their close functional relationship with target genes, high expression levels in peripheral blood, and association with at least one clinical trait. Drug discovery We conducted an extensive search across a range of prestigious databases, including the Connectivity Map, DrugBank, MedChemExpress (MCE), depmap, GeneCards, PharmGKB, the Drug-Gene Interaction database (DGIdb), The Therapeutic Target Database (TTD), the ChEMBL, and the PatSnap database. Additionally, we utilized the SM2miR database to obtain comprehensive drug information specifically related to the identified miRNAs. To further explore the draggability, we analyzed the interactions between commonly used clinical drugs and their target genes at the protein level. This investigation was carried out using the highly regarded STRING database (https://cn.string-db.org/). All potential effective small molecules were documented and classified, with their names, details and regulatory approval status up to date.\",\n",
       " '11190720': 'MethodsThe Pubmed, Cochrane Library, Web of Science and Embase were searched until Sep 2022. Randomized controlled trials (RCTs) relevant to the probiotics and placebo treatment on ASD children were screened. Quality assessment of the included RCTs was evaluated by the Cochrane collaboration’s tool. The primary outcomes were ASD assessment scales, including ABC (aberrant behavior checklist) and CBCL (child behavior checklist) for evaluating the behavior improvement, SRS (social responsiveness scale) for social assessment, DQ (developmental quotient) for physical and mental development and CGI-I (clinical global impression improvement) for overall improvement. The secondary outcome was total 6-GSI (gastrointestinal severity index).',\n",
       " '11190606': 'Methods Literature search strategy We followed the method proposed by the Preferred Reporting Items for Systematic Reviewsand Meta-Analyses flow diagram (PRISMA) guidelines. Pubmed, Cochrane Library, Web of Science and Embase were used in the retrieval process. Probiotics, autism spectrum disorder were searched as the key words by the combination of medical subject headings (MeSH) and entry term in English, and all literatures were searched in the database until Sep 2022. Search strategy for Pubmed and other databases were described in the supplement document. This meta-analysis carried out to the standards established by the PRISMA recommendation (Preferred Reporting Items for Systematic Reviews and Meta-Analysis). Literature inclusion and exclusion criteria Literature inclusion criteria: [1] patients diagnosed with ASD; [2] patients who were treated with probiotics and placebo; [3] RCT studies. Exclusion criteria: [1] reviews, comments, case report and animal experiment; [2] literature that lack of clinical trial data; [3] literature with duplicate data; [4] not RCT articles. This systematic review was performed by two authors who independently judged whether the retrieved literature could be included in the study, and the third author need to make an independent judgment whether to include it or not in case of disagreement. Outcome measures Primary outcome: the severity of ASD was evaluated by several rating scales, including ABC (aberrant behavior checklist) [12], CGI-I (clinical global imprssion- improvement) [13], SRS (social responsiveness scale) [14] and CBCL (child behavior checklist) [15].The higher the scores, the more serious the condition of autism. DQ (developmental quotient) [16] indicates physical and mental development. Secondary outcome: the severity of gastrointestinal symptoms was evaluated by total 6-GSI (gastrointestinal severity index) [17]. The higher the level of 6-GSI, the more serious the gastrointestinal condition. Literature quality evaluation criteria The quality assessment was evaluated by the Cochrane collaboration’s tool. The result was provided by the Revman 5.3, which was used to evaluate the quality of the RCTs. (Fig. 1). Fig. 1Flowchart of study selection process in the Meta-analyzes Data extraction The extracted data included the essential information of the study included authors, year of publication, age, gender composition, body mass index (BMI), weight, height, types of probiotics and ASD related index. Data that could not be extracted directly could be obtained by data transformation. Statistical methods Statistical analysis of the data was performed using Revman 5.3. Forest plots were used to present the combined effect quantity of the odds ratio (OR) and 95% confidence interval (CI). Funnel map was used to evaluate publication bias. The heterogeneity was evaluated using the I-square (I2) statistic, I2 < 50% indicated no significant heterogeneity among studies, so the effect indicators were combined using the fixed-effect model. I2 > 50% indicated significant heterogeneity, the randomized effect model was then adopted. The software of R was used for sensitivity analysis, and P value < 0.05 was considered a statistically significant difference.',\n",
       " '11190430': 'Statistical methods Statistical analysis of the data was performed using Revman 5.3. Forest plots were used to present the combined effect quantity of the odds ratio (OR) and 95% confidence interval (CI). Funnel map was used to evaluate publication bias. The heterogeneity was evaluated using the I-square (I2) statistic, I2 < 50% indicated no significant heterogeneity among studies, so the effect indicators were combined using the fixed-effect model. I2 > 50% indicated significant heterogeneity, the randomized effect model was then adopted. The software of R was used for sensitivity analysis, and P value < 0.05 was considered a statistically significant difference.',\n",
       " '11190372': 'Materials and methods Chickens Specific pathogen-free (SPF) chicken eggs were purchased from VALO BioMedia GmbH (VALO BioMedia GmbH, Osterholz-Scharmbeck, Germany) and incubated until hatching. Chickens were raised in a cage-free aviary system with woodshavings under confined conditions in the facilities of the Clinic of Poultry, University of Veterinary Medicine Hannover. Birds had ad libitum access to water and feed („all-mash L“, Deutsche Tiernahrung Cremer GmbH & Co. KG, Düsseldorf, Germany). Between five and twelve weeks posthatch chickens were humanely sacrificed according to the welfare regulations of Lower Saxony, Germany, to collect fresh ceca. Three to nine chickens were sacrificed for each experiment (parts 1–3) to isolate primary chicken intestinal epithelial cells (CIECs) The number of sacrificed chickens was approved and subsequently reported to the authorities according to the German welfare regulations. Isolation of primary chicken intestinal epithelial cells (CIECs) For all experimental parts (parts 1–3) chicken intestinal epithelial cells (CIECs) were isolated as described earlier, with some modifications [32]. Briefly, the ceca of 5-12-week-old SPF-layer chickens were collected aseptically during necropsy, washed in Hank´s Balanced Salt Solution (HBSS), pooled, chopped and digested enzymatically in digestion medium (Dulbecco’s Modified Eagle’s medium (DMEM)/Ham´s F12 (1:1; Biochrom GmbH, Germany, Berlin), 1% fetal bovine serum (FBS; Biochrom GmbH, Germany, Berlin), 50 µg/ml gentamicin (Sigma-Aldrich, USA, St. Louis), 100 U/ml penicillin, 100 µg/ml streptomycin (Biochrom GmbH, Germany, Berlin), 1 U/ml dispase II (Sigma-Aldrich, USA, St. Louis) and 75 U/ml collagenase (Biochrom GmbH, Germany, Berlin)) for 2 hours. Afterwards, single cells and bacteria were removed by using sorbitol gradient centrifugation (DMEM/Ham´s F12 (1:1), 2% d-sorbit (Carl Roth GmbH, Germany, Karlsruhe), 2.5% FBS, 50 µg/ml gentamicin) at 100 × g for 3 min at 37 °C. Sorbitol gradient centrifugations were repeated until the supernatant remained clear. The remaining pellet of crypts was resuspended in growth medium (DMEM/Ham´s F12 (1:1), 2.5% FBS, 10 µg/ml insulin (Sigma-Aldrich, USA, St. Louis), 1.4 µg/ml hydrocortisone (Sigma-Aldrich, USA, St. Louis), 5 µg/ml transferrin (Sigma-Aldrich, USA, St. Louis), 1 µg/ml fibronectin (Biochrom GmbH, Germany, Berlin), 100 U/ml penicillin, 100 µg/ml streptomycin; 50 µg/ml gentamicin). Crypt numbers were identified by counting 50 µl of the suspension on Tissue Culture Dishes with Grid (SARSTEDT AG & Co. KG, Nümbrecht, Germany) with an inverted light microscope. The suspension was adjusted to a concentration of 6000 crypts/ml based on preliminary experiments, seeded on collagen-coated 24-well plates (500 µl/well; Greiner Bio-One GmbH, Germany, Frickenhausen) and incubated at 37 °C in a 5% CO2 atmosphere. After 24 h, the medium was replaced with fresh growth medium (500 µl/well). Cells were used for further investigations after 24–48 h of incubation, when the cell density of CEIC monolayers was approximately 5 × 105 cells/cm2. Culture of the permanent human cell line Caco-2 The permanent human colon cell line Caco-2 was cultured as described previously [56]. Caco-2 cells were routinely cultured in growth media (DMEM, 20% FBS, 1% 100 U/ml penicillin, 100 µg/ml streptomycin and 1% nonessential amino acids [all from Biochrom GmbH, Germany, Berlin]) at 37 °C in a 5% CO2 atmosphere. The cells were passaged every 2–3 days. For use in the assays, the cells were seeded in 24-well plates and were grown for 2 days before further treatment. Bacterial strains Different bacterial strains were used as probiotic candidates in this study. Escherichia coli NISSLE (EcN) was kindly provided by Ardeypharm GmbH, Herdecke, Germany. Bacillus subtilis DSM 17299 (BS) and Bacillus licheniformis DSM 17236 (BL) was kindly provided by BioChem, Lohne, Germany. Clostridium butyricum DSM 10702 (CB), Enterococcus faecium DSM 7134 (EF), and Lactobacillus rhamnosus DSM 7133 (LR) were kindly provided by Lohmann Animal Health GmbH, Cuxhaven, Germany. Nearly all the probiotic strains were cultured on Columbia Sheep Blood Agar (CSBA) at 37 °C under aerobic conditions for 24 h. Lactobacillus rhamnosus DSM 7133 was cultured on MRS agar, and Clostridium butyricum DSM 10,702 was cultured on CSBA at 37 °C but under anaerobic conditions for 48 h. Three different C. jejuni strains were used in this study. The C. jejuni reference strain NCTC 11168, which was isolated from a human patient, was made available by the Institute for Microbiology and Hygiene at the Charité, Berlin, Germany. C. jejuni strain 0097 was kindly provided by the Friedrich-Loeffler-Institute, Jena, Germany, and was isolated from a laying hen. C. jejuni strain Lior6 was isolated from a chicken and was part of the strain collection of the Clinic of Poultry, University of Veterinary Medicine Hannover, Germany. All strains were stored in a 10% skim milk suspension at -70 °C. Prior to the experiments, 100 µl of a C. jejuni (11168, 0097 or Lior6) suspension in 10% skim milk with 106 CFU/ml was added to 3 ml of sterile Standard-I-Bouillon (Merck, Darmstadt, Germany) supplemented with 1 g/l Deoxycholic acid sodium salt (Carl Roth GmbH + Co. KG, Germany, Karlsruhe), 32 mg/l Cefoperazone and 1 mg/l Amphotericin B (Oxoid, Munich, Germany) and incubated at 37 °C under microaerophilic conditions (CampyGen, Oxoid, Wesel, Germany) for 48 h. After incubation, the suspension was centrifuged for 5 min at 30 × g, and the bacteria were resuspended in the required media. Experimental procedure A total of three experimental approaches were conducted to understand the interaction of C. jejuni with CIEC and to identify possible modes of interference with probiotic candidates. In part 1 and 3, Caco-2 cells were used as reference cells, and possible differences to CIEC with respect to pathogen‒host interactions were investigated. Part 1: investigations of the interaction between C. jejuni and primary CIECs In part 1, we investigated the host-pathogen interaction of C. jejuni with primary chicken-derived intestinal epithelial cells (CIECs). In the first experiment (Experiment 1), we studied the dose-dependent adhesion and invasion of the C. jejuni reference strain 11168 at three different concentrations (104 – 106 CFU/ml). In this experiment, CaCo-2 was used as a reference for comparison. In Experiment 2, we investigated possible strain variations in the adhesion and invasion pattern in CIEC by using three different C. jejuni strains (11168, 0097, Lior6; each 106 CFU/ml). In Experiment 3, the proinflammatory host response of CIECs was further investigated by measuring the expression patterns of the selected cytokines IL-1β and IL-6 after four and eight hours of incubation with the three selected C. jejuni strains (11168, 0097, and Lior6; each with 106 CFU/ml). Part 2: investigation of the soluble factor-mediated probiotic effects of six selected probiotic candidates on C. jejuni Six different probiotic candidates (EcN, BS, BL, CB, EF and LR) were investigated for interference with possible soluble factors, which may be released during propagation in growth media during the replication of C. jejuni strains (11168, 0097, Lior6). The Agar Well Diffusion Assay was used in this experiment (Experiment 4). Part 3: investigations of the indirect probiotic effects of three selected probiotic candidates We investigated the ability of three selected probiotic candidates (EcN, BS and BL) to reduce the colonization of C. jejuni 11168 in CIECs via interference assays. Caco-2 cells were used as a reference (Experiments 5 + 6). In addition, to identify possible C. jejuni strain variations, EcN and BS were selected and tested for possible interfering effects on three C. jejuni strains, 11168, 0097 and Lior6 (106 CFU/ml each), in an interference assay on CIECs (Experiment 7). BS was further selected to identify possible dose variations at 103-105 CFU/ml upon interference with 106 CFU/ml C. jejuni 11,168 (Experiment 8). Table 1 provides an overview of the type of cells used and the number of trials and replicates per experiment. Adherence and invasion assay The adherence and invasion of C. jejuni were investigated by using the Gentamicin Protection Assay [32, 57]. Wells with CIECs or Caco2 cells were washed twice with DMEM and covered with conservation media (DMEM/Ham´s F12 (1:1), 2.5% FBS, 10 µg/ml insulin), after which confluence was evaluated. Only wells with a confluence above 75% were used for further investigations. Bacterial suspensions of C. jejuni were adjusted to the required concentration by the use of the McFarland turbidity standard following standard procedures, and CFU were confirmed retrospectively by 10-fold serial dilution and plating [35]. The cells were covered with 500 µl of bacterial suspension and incubated for 3 h at 37 °C in a 5% CO2 atmosphere. Afterwards, the wells were washed three times, after which the cells were lysed with 500 µl of 0.5% Triton X-100, and serial dilutions of the lysates were prepared and subsequently plated on Campylobacter-selective charcoal-cefoperazone-deoxycholate agar (Campylobacter CCDA Selective Medium; Oxoid, Wesel, Germany) to determine the presence of adherent C. jejuni. For determination of invading cells, the wells were washed three times with DMEM and incubated with conservation media supplemented with 100 µg/ml gentamicin. After 1 h, the wells were washed three times with DMEM and lysed with 500 µl of 0.5% Triton-X 100. The number of invaded C. jejuni cells was determined after 10-fold serial dilution of the lysed samples and plating on CCDA plates. The CCDA plates were incubated for 48 h at 37 °C under microaerophilic conditions prior to counting. The enumeration of adherent C. jejuni included the total number of C. jejuni associated with cells prior to lysis (for extra and intracellular bacteria) [57]. The results are expressed as the percentage of CFU of C. jejuni in the inoculum that adhered or were internalized (relative adhesion and invasion) or as the percentage of total cell-associated C. jejuni that was internalized (InvasionIndex [57]). Agar well diffusion assay The Agar Well Diffusion Assay was conducted as described previously by Campana et al. with slight modifications [58]. One colony of Escherichia coli NISSLE, Clostridium butyricum DSM 10702, Bacillus subtilis DSM 17299, Bacillus licheniformis DSM 17236, Enterococcus faecium DSM 7134 or Lactobacillus rhamnosus DSM 7133 was suspended in five ml of sterile Standard I Bouillon and incubated at 37 °C under aerobic conditions. CB was incubated under anaerobic conditions. Noninoculated standard I Bouillon plants were treated in the same way and used as a negative control. After 48 h of incubation, the bacterial suspensions were centrifuged at 30 × g for 10 min. The supernatants were filtered (VWR Syringe Filters, VWR International, Radnor, USA; 0.22 mm pore size) to remove the remaining bacteria. Samples of all cell-free supernatants (CFS) were spread out on Columbia Sheep Blood Agar and incubated under aerobic, anaerobic and microaerophilic conditions at 37 °C for 48 h to confirm the absence of any remaining bacteria. The supernatants were stored at -20 °C until use. Standard I Bouillon (Merck KGaA, Germany, Darmstadt) with 1% Agar Agar (Carl Roth GmbH + Co. KG, Germany, Karlsruhe) was autoclaved and cooled. At a temperature of 45 °C, 100 µl of a C. jejuni (11168, 0097 or Lior6) suspension in 10% skim milk with 106 CFU/ml was added to 200 ml of the bouillon with Agar Agar. Subsequently, 22 ml of this suspension was added to each Petri dish, and after solidifying, five wells each five mm in diameter were punched into the agar under sterile conditions. Three wells were filled with 48 µl of CSF of the same source, one well with the negative control and one well with a Gentamicin solution (positive control; 10 mg/ml; Sigma-Aldrich, USA, St. Louis). Plates were incubated for 48 h at 37 °C under microaerophilic conditions (CampyGen, Oxoid, Wesel, Germany). Antimicrobial activity led to a clear inhibition zone around the subsequent well, while the remaining agar got turbid due to C. jejuni replication. Interference assay One possible mechanism of probiotic action is competitive exclusion. According to this principle, one bacterial species competes more vigorously for receptor sites in the intestinal tract than does another species. Additionally, other mechanisms, such as competition for nutrients, creation of a hostile microecology or secretion of antimicrobial substances, have been described [59]. If a probiotic species has the ability to occupy a particular ecological niche before the pathogen, this process is termed exclusion. An effect based on simultaneous colonization is named competition, and displacement describes a probiotic effect, which leads to the reduction of an already established colonization of another bacterial species. The interference assay was conducted as described previously [14, 56, 58], with slight modifications, to investigate Competitive Exclusion effects. CIECs and Caco-2 cells were prepared and treated in the same way as for the adherence and invasion assays. The bacterial concentrations were adjusted to 106 CFU/ml for C. jejuni, EcN and BL and 105 CFU/ml for BS in the final mixture of bacteria in conservation media. For the investigation. After exclusion, the cells were washed with DMEM and pretreated with one of the probiotic candidates. After 1 h of preincubation, the C. jejuni strain was added.After competition, the cells were incubated with a mixture of one C. jejuni strain and one of the probiotic candidates.After displacement, the cells were preincubated for 1 h with the C. jejuni strain prior to the addition of one probiotic candidates. For adherence and invasion assays, the incubation time was stopped three hours after the addition of C. jejuni to the cells, and the cells were further processed for adherence and invasion analysis as described above. qRT‒PCR detection of the mRNA expression of selected cytokines Four and eight hours after inoculation with one of the C. jejuni strains (11168, 0097, Lior6; each 106 CFU/ml), the CIECs were washed and detached with 250 µl of trypsin/EDTA (0.05%/0.02%; Biochrom GmbH, Germany, Berlin). After detachment, the cells were stored at -80 °C until RNA isolation. Total RNA was extracted from cell samples by using the MasterPure RNA Purification Kit (Epicentre, USA) according to the manufacturer´s instructions. The isolated RNA was stored at -80 °C until qRT‒PCR analysis. qRT‒PCR was performed by using a Stratagene MX 3005P RT-qPCR cycler (Stratagene, USA) and an AgPath-ID One-Step RT‒PCR Kit (Applied Biosystems, USA) according to the manufacturer´s instructions as described previously [33]. The primers and probes used for the detection of the mRNA expression of IL-1β and IL-6 as well as the constantly expressed housekeeping gene 28 S were previously published [33, 60, 61]. Three µl of total RNA in 25 µl of reaction mix were used with the following cycle profile: one cycle at 45 °C for 10 min and 95 °C for 10 min and 40 cycles of 95 °C for 15 s and 57 °C for 45 s. The cycle threshold (Ct) values of the expressed mRNAs of the investigated genes were normalized against those of the expressed housekeeping gene 28 S rRNA of the same sample (ΔCt) as described by Powell et al. [62]. The overall 28 S rRNA expression was comparable between samples independent of the treatment. The ΔCt values of the samples are presented as fold changes and were related to the ΔCt values from negative control groups at the same sampling time point. Statistical analysis Statistical analyses were performed with Statistix version 10.0 (Analytical Software, Tallahassee, FL, USA). p < 0.05 was considered to indicate statistical significance. In Experiments 1 and 5, two sample t tests and Wilcoxon rank sum tests were used; in Experiments 2 and 7, one-way analysis of variance was performed with the Tukey HSD All-Pairwise Comparisons Test. In Experiment 3, one-way analysis of variance with Tukey’s honestly significant difference (HSD) All-Pairwise Comparisons Test and Two-sample T test were applied. Experiment 6 was statistically verified by a two-sample t test. For the number of trials and replicates, see Table 1. Table 1Number of sacrified chickens, trials and total replicationsExperimentalPartExperimentDescriptionUsed cellsSacrificedChickens1Number oftrialsNumber ofreplicates2CIECCaco-211Colonisation pattern after infection with different concentrations of C. jejuni 11168++92-36-912Colonisation pattern after infection of CIEC with three different C. jejuni strains+83913IL-1β and IL-6 mRNA expression in CIEC after C. jejuni infection+315-724Agar Well Diffusion Assay01335Influence of incubation time point of probiotic candidates on colonisation of CIEC by C. jejuni 11168+52636Influence of incubation time point of probiotic candidates on colonisation of Caco2 cells by C. jejuni 11168+02637Influence of probiotic candidates on colonisation of CIEC by different C. jejuni strains+62638Influence of probiotic concentration on colonisation of CIEC by C. jejuni 11168+7261: Number of total sacrificed SPF-chickens per experiment used to prepare CIEC. Isolated cells were pooled within a trial.2: Number of total replicates per treatment group and time point investigated.',\n",
       " '11190321': 'Methods Study participants Participants were recruited into a previously described phase 2b randomized, placebo-controlled clinical trial of the L. crispatus-based live biotherapeutic (LACTIN-V) to prevent BV recurrence following standard antibiotic treatment [4]. Two hundred and twenty-eight women were enrolled in the larger clinical trial. Of the 151 participants who received LACTIN-V in the larger clinical trial, we randomly selected a subset of 32 participants who attended all clinical trial visits for the current sub-study (Fig. 1). Sociodemographic factors did not significantly differ between participants included in this sub-analysis and all other participants randomized to receive LACTIN-V (Table S1). Clinical trial details have been outlined in previous publications [4, 7, 8]. Briefly, women with BV (as defined by ≥ 3 Amsel criteria and a Nugent score ≥ 4) at the screening visit, henceforth referred to as the baseline visit, were given a 5-day course of topical metronidazole and then randomized 2:1 to either LACTIN-V or placebo which were administered intravaginally once daily for 5 days followed by twice weekly for 10 weeks. Vaginal swabs were collected at baseline and within 48 h of completing metronidazole treatment (i.e., at enrollment) and at 4 weeks, 8 weeks, 12 weeks (i.e., 1 week after the last dose of LACTIN-V), and 24 weeks (i.e., 13 weeks after the last dose of LACTIN-V) after enrollment. For simplicity, we will refer to the 4-week, 8-week, and 12-week visits as “during” LACTIN-V administration. Immediately following collection, vaginal swabs were plunged into 2 mL of Starplex transport medium, immediately frozen at − 20 °C, and then transferred to a − 80 °C freezer at the end of the day. Behavioral characteristics were also recorded at each study visit.Fig. 1Summary of study design Quantification of soluble immune factors Cervicovaginal swabs were thawed and the remaining eluent (i.e., the transport medium that the swab was plunged into) was centrifuged at 4500 rpm for 30 min. Supernatant was then removed for immune factor analysis and the bacterial pellet was left intact for qPCR analyses. The soluble immune factors IL-1α, IFN-α2A, IL-17A, IL-6, IP-10, IL-8, MIP-1β, MIP-3α, MIG, sE-cad, and MMP-9 were measured in duplicate on the MSD platform according to manufacturer’s instructions as previously described [9] (Meso Scale Discovery, Rockville, MD). DNA extraction and quantitative polymerase chain reaction DNA was extracted from 175µL of the bacterial pellet in vaginal swab samples using the Qiagen DNEasy PowerSoil kit (Qiagen) according to manufacturer’s instructions. Targeted quantitative polymerase chain reaction was used to estimate total bacterial abundance by targeting the 16S region of the rRNA gene and the absolute abundances of Lactobacillus species (L. crispatus, L. iners, L. jensenii, and L. gasseri) and four common BV-associated bacterial taxa (G. vaginalis, Atopobium vaginae, Megasphaera species, and Prevotella species) using previously validated primer and probe sequences described below. All quantitative polymerase chain reaction assays were Taqman-based and performed on the QuantStudio 6 Flex Real-Time PCR System (Thermofisher). Total bacterial load was quantified with qPCR adopted from Nadkarni and colleagues [10]. Protocol for quantification of L. crispatus, L. iners, L. jensenii, and L. gasseri absolute abundances with multiplex quantitative polymerase chain reaction was adopted from Balashov and colleagues [11]. Absolute abundances of G. vaginalis, A. vaginae, and Megasphaera spp. were quantified with multiplex quantitative polymerase chain reaction according to Kusters and colleagues [12]. Total Prevotella absolute abundance was quantified with quantitative polymerase chain reaction adopted from Martin and colleagues [13]. Primer and probe sequences are presented in Table S2. Total reaction volume for assays was 10µL. Assays for total Prevotella, L. crispatus, L. iners, L. jensenii, and L. gasseri were performed at 95 °C for 10 min, 45 cycles at 95 °C for 15 s, then 60 °C for 1 min. Assays for G. vaginalis, A. vaginae, and Megasphaera spp. were performed at 95 °C for 10 min, 45 cycles at 95 °C for 15 s, then 55 °C for 1 min. Data analysis was performed with QuantStudio Real-Time PCR Software version 1.3 (Applied Biosystems). Copy numbers were quantified using the following equation, where ΔCt represents the difference in Ct between a sample and negative control: \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{mathrsfs} \\\\usepackage{upgreek} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document}$$Copy number={2}^{\\\\Delta Ct}$$\\\\end{document}Copynumber=2ΔCt. L. crispatus CTV-05 polymerase chain reaction Polymerase chain reaction (PCR) reactions performed at the University of California San Francisco were set up in duplicate with 25µL QuantiTect SYBR Green RT-PCR Master Mix (Qiagen), 2.5µL forward primer (10uM), 2.5 µL reverse primer (10uM), 10µL extract, and 10µL H2O for a total volume of 50µL. Following initial denaturation at 95 °C for 15 min, PCR cycling consisted of 40 cycles at 95 °C for 30 s, 58 °C for 60 s, and 72 °C for 60 s. Gene targets for strain and species-specific PCR were selected using genes identified in L. crispatus CTV-05-specific regions which were absent in other sequenced bacterial strains and with low numbers of homologs in vaginal metagenome datasets. Primer sequences are presented in Table S3. Bacterial concentration was calculated from mean sample Ct values using a standard curve based on serial dilutions of CTV-05 strain of L. crispatus. Limits of detection determined at the 95% detection threshold were 6.0 × 102 CFU/mL. Detection L. crispatus CTV-05 was defined as above the lower limit of detection as defined above. A concentration of L. crispatus CTV-05 greater than 106 CFU/mL was pre-defined as the threshold for clinically relevant colonization, based on published evidence that this concentration of L. crispatus CTV-05 was associated with reduced recurrence of BV [4] and UTIs [14, 15]. DNA extraction and metagenomic sequencing Samples were plated from swab collection tubes into ZymoBIOMICS lysis solution for DNA extraction. DNA was extracted and processed with high-throughput automation liquid handlers (Agilent Bravo system and Labcyte ECHO instrument) to maintain constancy in sample experimentation and decrease laboratory processing time. The ZymoBIOMICS 96 MagBead DNA kit was followed as instructed to extract DNA from swab samples, water controls, and storage medium controls. Illumina library preparation was performed using a miniaturized protocol of NEBNext Ultra II FS DNA Library Prep kit for DNA for the Labcyte ECHO instrument [16]. More than 25 million paired-end 150 bp reads per patient sample were collected on an Illumina NovaSeq instrument. The CZ ID platform was used to process raw sequencing reads and remove host reads [17]. The VIRGO bioinformatic pipeline was used to align processed reads with established databases to identify microbes and inferred microbial functions [18]. Samples with more than 100,000 microbial reads were included in the present analyses. Inferred microbial functions were quantified by calculating the relative abundance of genes mapping to individual KEGG orthologs [19]. Statistical analysis Soluble immune factor levels and gene copy numbers were normalized through log10-transformation. Comparison of L. crispatus CTV-05 absolute abundance between colonization was performed with the Mann–Whitney U test. The Pearson Chi-Square test or Mann–Whitney U test was used to compare the frequency (if dichotomous) or value (if continuous) of sociodemographic factors and sexual behaviors between colonization groups. Clustering on pre-LACTIN-V immune data only included participants with complete immune data at the pre-LACTIN-V visit and was performed with k means clustering using the kmeans function in the “stats” package in R. Optimal number of clusters for k means clustering was determined using silhouette analysis with the fviz_nbclust function in the “factoextra” package in R. Principal component analysis (PCA) plots were generated using the prcomp function and visualized with the fviz_pca_biplot function in the “factoextra” and “FactoMineR” packages in R. Principal coordinate analysis (PCoA) plots were generated using a Bray–Curtis dissimilarity matrix that was constructed using the relative abundance of all bacteria species identified with metagenomic sequencing prior to LACTIN-V administration. Comparison of overall vaginal microbiota composition prior to LACTIN-V administration was performed with ANOSIM. PCoA and ANOSIM analyses were performed using the “vegan” and “dplyr” R packages and PCoA visualization was performed with the “ggplot2” R package. Shannon diversity of the vaginal microbiota was determined at the species level using the “diversity” function in the “vegan” package in R with default parameters. Differential abundance analysis of the relative abundance of bacterial species prior to LACTIN-V administration was performed with linear discriminant analysis effect size (LEfSe) [20]. Since we did not include a subclass comparison in our LEfSe analysis, we adjusted p values with the false discovery rate [21]. Differential abundance analysis of KEGG orthologs was performed with MaAsLin2 and p values were adjusted for multiple comparisons with the false discovery rate [22]. Comparisons of levels of immune factors and bacterial absolute abundances and Shannon diversity of the vaginal microbiota prior to LACTIN-V administration between colonization groups were performed with the Mann–Whitney U test. Linear mixed models were generated to evaluate the association between immune factors and bacterial abundances at each visit during LACTIN-V administration. Participants IDs were included as random effects in each model to allow for the inclusion of repeated measurements without violating the assumption of independence. The association between immune factors and bacterial absolute abundances during LACTIN-V administration was performed with linear mixed models that included participant IDs as random effects. Comparisons of immune factors and bacterial abundances at 24 weeks (i.e., 13 weeks after the last dose of LACTIN-V) between sustained colonizers, transient colonizers, and colonization-resistant women were performed with ANOVAs and pairwise comparisons were performed with the Tukey post hoc test. All statistical tests were performed with RStudio (version 2022.02.3) or GraphPad Prism (version 9.0.2).',\n",
       " '11190290': '4 Methods 4.1 Animals and Transcriptome Library Building Brain tissue samples from C57BL/6J background rd1 and wt control mice were used for all of the experiments. There were two groups of mice, including the rd1 mice group with retinitis pigmentosa (rd1, Pde6bem1 Cin purchased from Gem Pharmatech Co., Ltd., Nanjing, China) and the control mice group (C57BL/6J, purchased from Gem Pharmatech Co., Ltd.). The mice were euthanized by cervical subluxation after anesthesia. All animal procedures were performed in accordance with the ethical standards of the Ethics Committee of CUH, Southeast University. The animal ethics approval number for this experiment is 20230206002 and the approval date is February 6, 2023. The age of rd1 mice was consistent with those of PD mice and normal control mice, which were all 8 weeks old. The hippocampal regions of both rd1 mice and normal mice were first isolated. Then, RNA was extracted from crushed tissue blocks using an RNA Extraction Kit FT (Vazyme Biotech Co., Ltd., item no. RM203-01). The RNA was then reverse-transcribed into cDNA. Next, 5 ng of cDNA was treated with a One-step DNA Lib Prep Kit (ABclonal, RK20239) for Illumina sequencing. The library was successfully sequenced on an Illumina HiSeq 2500 (Illumina Inc., San Diego, CA, USA) sequencer with a paired-end pattern and inset size of 350 bp. 4.2 Analysis of RNA-seq Data and Statistical Methods The fastq format raw data obtained after Illumina sequencing was screened to obtain high-quality clean data. Each library was quality-controlled by FastQC with the Q30 value in Table S1. Then, the clean data were compared with the mouse reference sequence using Hisat2 software. The number of counts obtained for each sample was measured using FeatureCounts in the subread Linux package. DEGs were analyzed using the DESeq2 R package (1.36.0) with Log2|Fold change|>1 and P-value <0.05. Gene ontology (GO) enrichment and KEGG analysis of DEGs were performed using the Cluster Profiler R package. Comparisons between the two groups were made by t tests (paired two-sample mean analysis). Statistical significance was defined as P < 0.05. 4.3 Quantitative Real-Time Reverse Transcription Polymerase Chain Reaction RT-qPCR was performed using TB Green dye and specific primer pairs. The primer sequences are listed in Table 1. Three replicates were performed for each sample. The cycle threshold (Ct value) was the number of cycles of fluorescent signaling in each reaction when the set threshold was reached. The samples were quantified using an internal reference method. The internal reference was glyceraldehyde-3-phosphate dehydrogenase (GAPDH) of the mouse. The PCR amplification system consisted of the DNA amplification enzyme TB Green Premix Ex TaqII (Takara, RR420A, 2×), upstream and downstream primers (10 μm), a DNA template, and enzyme-free water to make up the volume to 20 μL. The ratios of the expression levels of DEGs between rd1 and normal control mice were calculated by the 2–ΔΔCt method after normalization. The formula was as follows.12 4.4 Western Blotting Brain tissues from both the rd1 group and the normal control group were lysed and homogenized in RIPA lysis buffer (50 mM Tris–HCl, 150 mM NaCl, 1% Triton X-100, 0.5% sodium deoxycholate, 0.1% SDS, pH 7.4) and 50× Cocktail protease inhibitor (Servicebio, G2006-250UL). Equal amounts of proteins were separated on SDS polyacrylamide gel and transferred to a PVDF membrane (0.45 μm, Servicebio, WGPVDF45). The proteins were standardized prior to SDS-PAGE separation and then transferred to nitrocellulose membranes. The membranes were blocked with 5% skim milk powder in TBST for 2 h at room temperature. The membranes were then incubated with the primary antibody in a blocking solution overnight at 4 °C. The primary and secondary antibodies used for Western blotting are shown in Table 2. The signals were developed with an Ultrasensitive ECL Chemiluminescence Kit (Servicebio, G2020-25 ML). The relative intensities of WB immunoreactive bands were quantified using the gel analysis tool provided by ImageJ software (1.8.0). At least three replicate Western blotting experiments were performed.',\n",
       " '11190269': '4.2 Analysis of RNA-seq Data and Statistical Methods The fastq format raw data obtained after Illumina sequencing was screened to obtain high-quality clean data. Each library was quality-controlled by FastQC with the Q30 value in Table S1. Then, the clean data were compared with the mouse reference sequence using Hisat2 software. The number of counts obtained for each sample was measured using FeatureCounts in the subread Linux package. DEGs were analyzed using the DESeq2 R package (1.36.0) with Log2|Fold change|>1 and P-value <0.05. Gene ontology (GO) enrichment and KEGG analysis of DEGs were performed using the Cluster Profiler R package. Comparisons between the two groups were made by t tests (paired two-sample mean analysis). Statistical significance was defined as P < 0.05.',\n",
       " '11190267': 'Methods Harvest for Health was a 2-arm, single-blinded RCT that used a waitlist-controlled, crossover design to test the effects of a home-based gardening intervention on vegetable and fruit intake, physical activity, physical functioning, quality of life, sleep, adiposity, gut microbiome, and inflammatory biomarkers. Individuals assessing outcomes and conducting analyses were blinded to arm assignment. This study was approved by the University of Alabama at Birmingham (UAB) institutional review board (NCT02985411; the trial protocol is given in Supplement 1). All study participants provided written informed consent. We followed the Consolidated Standards of Reporting Trials (CONSORT) reporting guideline. The study period spanned May 11, 2016, to May 2, 2022, with detailed methods and recruitment results published previously.22 Participants Medicare-eligible cancer survivors who were diagnosed with and had completed curative therapy for cancers with 5-year survival of 60% or more22 were ascertained through UAB or Alabama state cancer registries and approached via direct mail23 followed by telephone contact. Referrals responding to media also were enrolled after verification of diagnoses by health care practitioners. Interested survivors were screened for suboptimal vegetable and fruit consumption (<5 servings per day), physical activity (<150 MVPA minutes per week), physical functioning (36-Item Short Form Health Survey [SF-36] subscale score ≤90 [score range, 0-100, with higher scores indicating better physical functioning]), and living independently at a residence that could accommodate 1 raised bed (1.2 × 2.4 m) or 4 grow boxes (62.2 × 52.1 cm) with 6 or more hours per day of sunshine and ready access to water. Other criteria were ability to speak, read, and write English and willingness to participate for 2 years with randomization to either study arm. Exclusion criteria included vegetable gardening within the past 2 years or medical conditions that precluded gardening. All study participants were followed up from initial contact until study completion or termination according to the CONSORT guideline.24 Assessments Assessments occurred at baseline (before randomization) and 12 months and included anthropometric measures (height [baseline only], weight, and waist circumference),25 physical performance testing (Senior Fitness Battery8), accelerometry (7-day collection via GTPX3 accelerometer [ActiGraph]), the Community Healthy Activities Model Program for Seniors (CHAMPS) Physical Activity Questionnaire for Older Adults,26 blood collection via venipuncture (after ≥4 hours of fasting), and collection of toenail clippings and a fecal wipe. Plasma and serum samples and fecal wipes were stored at −80 °C and toenails at room temperature27 and then batch-analyzed following published methods for α-carotene,28 cortisol,29 fecal microbiome,30,31,32,33 and inflammatory biomarkers.22 A remote protocol was implemented in April 2020 due to the COVID-19 pandemic. It omitted grip strength, arm curl, and gait speed measures from the performance battery because of cost or failure to prove reliability,22,34 and dried blood spots (found valid and reliable for interleukin-6 [IL-6]) and tumor necrosis factor–α [TNF-α]) supplanted venipuncture. Assessment of plasma α-carotene levels (an objective measure of vegetable and fruit intake) was not possible. Surveys administered online (REDCap) or by mail captured self-reported race, ethnicity, comorbidities, and symptoms35; falls36; perceived stress37; reassurance of worth38; sleep quality39; vegetable and fruit consumption40; physical activity26; quality of life41; and changes in health. Race and ethnicity were included in the analysis as potential effect modifiers; categories were African American or Black, non-Hispanic White, and other (included American Indian, Asian, Pacific Islander, Hispanic, and multiracial; combined into 1 category because <2% of participants identified as a race other than African American or Black or White and/or as Hispanic ethnicity). Randomization and Intervention After conclusion of the baseline assessment, a staff member opened a sealed envelope (created by a statistician) to reveal the randomization assignment. Participants were evenly allocated within each county to receive the intervention immediately or to be waitlisted using computer-generated permuted block randomization. In the intervention group, the EMGs were paired with cancer survivors within counties of residence and visited survivors’ homes monthly to help establish and then maintain a spring, summer, and fall garden. Telephone, email, or text messages were scheduled between visits, with social cognitive theory undergirding all communications.42 The EMGs logged all encounters and provided time-stamped garden photographs. Each participant received a 1.2 × 2.4-m raised bed kit (or 4 grow boxes), soil, fertilizer, mulch, frost cloths, tomato cages, trellises, gardening hose, watering can, trowel, cultivator, soft chemistry insect control products, seeds, transplants, and a gardening journal. Participants and EMGs received hats, sunscreen, and binders with contact information, interaction tracking logs, and publications on gardening, safety, health, and vegetables and fruits (including recipes). Supplies were distributed at EMG-participant meet-and-greets (informational sessions and social events) at local community centers before initial planting. During the COVID-19 pandemic, meetings were conducted via videoconferencing and supplies were distributed in parking lots; all EMG-participant interactions were restricted to masked, outdoor encounters. Waitlisted participants received the identical intervention after 12 months. Statistical Analysis A composite dichotomous score (yes or no) served as the primary outcome based on baseline-to-12-month attainment of the following benchmarks: increased consumption of 1 or more vegetable and fruit servings per day, corroborated by an increase of at least 10% in plasma α-carotene level43; 30 or more minutes of MVPA per week, corroborated by accelerometry; and increase of at least 5 SF-36 physical functioning subscale points, corroborated by improved scores in at least 60% of physical performance tests. Power calculations indicated that 185 participants per arm would yield more than 90% power to detect between-arm differences using χ2 testing (assumptions: 15% attrition and a moderate response difference ≥20%).44 Descriptive statistics and normality checks were performed on continuous variables (results from transformed analyses were similar). All statistical tests were 2-sided (α < .05) and conducted using SAS, version 9.4 (SAS Institute Inc); an intent-to-treat approach was used for all analyses. Continuous outcomes were explored using linear mixed-effects models, including fixed effects for group, time, and group × time interaction, with a random intercept for participants estimated using restricted maximum likelihood. The mean change at 12 months vs baseline within each group and the difference in mean change between groups, with corresponding 95% CIs, were reported. Categorical secondary outcomes (weight, waist circumference, health-related quality of life, reassurance of worth [Revised Social Provision Scale], self-efficacy to garden, social support to garden, sleep quality, perceived stress, cortisol level, IL-6 level, TNF-α level, and microbiome alpha diversity) were explored using logistic mixed-effects models with the same aforementioned model terms and assumed a binary or multinomial response distribution. Odds ratios (ORs) and their corresponding 95% CIs were reported. Linear mixed-effects models are robust against missing data, as they can accommodate unbalanced data patterns45,46; thus, all available observations and participants were included in the analysis. For each outcome, participants with at least 1 nonmissing outcome measure were included in the analysis. Logistic regression analyses were performed to determine whether garden type (raised beds or grow boxes) and EMG contact of 90% or more vs less than 90% modified the effects. Other potential effect modifiers (cancer type [breast and gynecologic cancer vs others], time elapsed from diagnosis [<5 years vs ≥5 years], age [<70 years vs ≥70 years], sex, race and ethnicity [non-Hispanic White as the reference group], educational attainment [high school graduate or less vs above], household size [1 vs ≥2 members], annual income [<$60 000 vs ≥$60 000], rural or urban county of residence, BMI [<30 vs ≥30; calculated as weight in kilograms divided by height in meters squared], vitality [<70% vs ≥70% of the SF-36 subscale score],41 garden start season [spring vs others], and trial completion before vs during the COVID-19 pandemic) were explored similarly.',\n",
       " '11190265': \"Materials and Methods Materials Colon-specific carcinogens, dextran sodium sulfate (DSS; MW = 36,000-50,000), and azoxymethane (AOM) 19, 20 were purchased from ICN Biomedicals Inc. (Aurora, Ohio) and Sigma-Aldrich Company (Steinheim, Germany), respectively. Na+/K+-ATPase Mouse Monoclonal Antibody [1G1] (RT1412) was bought from huaBIO company (Zhejiang province, China). Prohibitin antibody (2426) was from Cell Signaling Technology, Inc (Massachusetts (MA), USA). Anti-ALDH2 antibody (ab227021) was sourced from Abcam company (Cambridgeshire, UK), and β-actin antibody (66009-1-Ig) and GAPDH (60004-1-Ig) were obtained from PROTEINTECH company (Chicago, USA). Anti-Rabbit IgG (H+L) Antibody, Peroxidase-Labeled (074-1506), anti-Mouse IgA + IgG + IgM (H+L) Antibody, Human Serum-Adsorbed and Peroxidase-Labeled (074-1807) were provided by KPL Inc. (Massachusetts, USA). The iTRAQ Reagent-8Plex Multiplex Kit and RevertAid Reverse Transcriptase (EP0441) were obtained from Thermo Fisher Scientific (California, USA), and the ultra-high sensitivity ECL Kit was from Millipore company (Now Merck company, Massachusetts, USA). The PerfectStart Green qPCR SuperMix (AQ601-04) was purchased from TransGen Biotech (Beijing, China). Other analytical reagents were obtained from Sinopharm Chemical Reagent Co., Ltd. (Shanghai, China). Patients and clinical sample collection This study received approval from Shanghai Public Health Clinical Center (Approval No. 2019-S035-02), and a written informed consent was obtained from each participating patient. A total of twenty-one patients who underwent surgical resection at Shanghai Public Health Clinical Center were randomly selected from October 2020 to August 2022. The patients were pathologically diagnosed according to the standard of the 8th edition of the American Joint Committee on Cancer/International Union against cancer classification. All patients were Chinese. We enrolled a total of ten males and eleven females with mean age of 62.8±12.1 (range, 42-84) years and differentiation of II to IV. Eight patients with lymph nodes lymph node metastasis and four with distal metastasis were included. Patients with age less than 18, pregnancy, breastfeeding, HIV, and TB infection were excluded. Colon tumor tissues and adjacent normal colon tissues were collected during the surgical resection for pathological examination. Residual tissues from these 21 pairs were utilized for subsequent real-time PCR (n = 14 pairs), western blot (WB) analysis (n = 5 pairs) and metabolomics study (n = 7 pairs). Clinical data, including gender, age, tumor size, and metastasis information, were meticulously documented for each patient (Supplement table 1 (Table S1)). The fresh colon tissues were cut into 1-2 cm sizes, washed with normal saline solution for three times, and stored into -80°C for further use. Colon cancer mouse model developed by AOM/DSS treatment The development of a colon cancer model through AOM/DSS treatment was approved by the Ethics Committee of Shanghai Public Health Clinic Center (2014-A032-02) and followed established procedures 6. Briefly, male C57BL/6N mice, aged 6-8 weeks, were obtained from Shanghai Public Health Clinical Center and divided into two groups: the AOM/DSS-treatment group and the control group, each consisting of 10 mice. After one week of free access to water and food, mice in the AOM/DSS-treatment group were intraperitoneally injected with one dose of AOM (12.5 mg/kg body weight in saline) and were provided 2.5% DSS (in drinking water) for seven days, followed by two weeks of drinking water. This cycle of one week of water containing DSS followed by two weeks of normal drinking water was repeated over a 9-week period. Subsequently, these mice were maintained without receiving any drugs until the 14th week, and are referred to as the “AOM/DSS group.” Control animals, denoted as the “saline group”, were intraperitoneally injected with a single dose of saline (0.2 mL), corresponding to the AOM injection, and had continuous access to drinking water. At the end of 14th week, all mice were anesthetized by intraperitoneal injection of pentobarbital sodium (10 mg/mL in saline) at a dose of 50 mg/kg (50 mg drug per kg mouse body weight), and tumor development was monitored using colon capsule endoscopy. After endoscopy detection, the mice were fed normally for two days, subsequently fasted for 10 h with normal water drinking, and then euthanized by intraperitoneal injection of pentobarbital sodium (50 mg/mL in saline) at the volume of 1.0 mL, which corresponds to a dose of approximately 1850 mg/kg 21. After making sure the mouse death through observing the pain response disappearance, and cardiac and respiratory arrest, the mouse was dissected. Tumor sizes were measured using a vernier caliper. The fresh colon was longitudinally dissected and thoroughly rinsed multiple times with normal saline solution till the intestinal wall clean. A quarter of the colon tissues was fixed in formaldehyde for pathological examination, another quarter was frozen for future use, and the remainder was collected for mucosal separation, membrane enrichment, and western blot (WB) analysis. Colon mucosa separation and purity detection The mucosal layer was carefully peeled under anatomical microscopy guidance. In the AOM/DSS group, tumor tissues with intact mucosa were dissected for mucosal separation (referred to as Part 1). If the mucosal layer was disrupted in the tumor tissues, the tumors were directly extracted (referred to as Part 2). Part 1 and Part 2 were combined for membrane enrichment (referred to as the “tumor” sample). The adjacent normal colon tissues in the AOM/DSS model and the normal colon tissues in the saline-treated group were subjected to mucosal separation and labeled as “adjacent” and “normal”, respectively. Finally, a portion of the separated mucosa was preserved in formalin for subsequent pathological examination using hematoxylin and eosin (HE) staining, and the remainder was designated for cell membrane enrichment. The mucosa of every 3 mice were mixed together, and a total of 3 mixed samples from 10 mice were prepared. The three samples (triple biological repetitions) prepared in this way were used for following experimental processes, including membrane enrichment, protein digestion, iTRAQ labeling, mass spectrometry detection and western blot analysis. Cell membrane enrichment The separated mucosal tissues were homogenized in phosphate buffer saline (PBS) at a ratio of 1:2 (tissue weight to buffer volume). After centrifugation at 700 × g for 10 minutes to remove unbroken cells and nuclei, the supernatant was collected. Subsequently, another centrifugation step was performed for 30 min at 10,000 g to isolate the cell membrane pellet. All these procedures were conducted at a temperature of 4°C to preserve membrane integrity. The effectiveness of membrane enrichment was assessed through WB analysis, utilizing specific markers associated with the plasma membrane and mitochondria. iTRAQ-based proteomic study (1) Protein extraction, quantification, and digestion: The membrane-enriched samples underwent pretreatment with a lysis buffer composed of 5 M Urea, 2 M Thiourea, 4% CHAPS, 1% NP-40, 65 mM dithiothreitol, and 0.5 mM phenylmethylsulfonyl fluoride (PMSF). Protein quantification was determined using the Bradford method (Bio-Rad, Hercules, CA, USA). Subsequently, 100 µg of proteins were subjected to reduction, alkylation, and trypsin (Promega (Madison, USA)) digestion at a protein-to-trypsin ratio of 20:1 at 37°C overnight. (2) iTRAQ labeling: Peptide mixtures were labeled using the 4-plex iTRAQ labeling kit, following the iTRAQ™ Reagent protocol and our prior descriptions 22, 23, with 117 and 116 for tumor tissues and adjacent colon tissues from AOM/DSS-treated mice, while 114 for the saline-treated group, respectively. After incubated at room temperature for one hour, all iTRAQ-labeled tryptic peptide solutions were mixed. (3) Peptide separation and mass spectrometry detection: The labeled peptides were fractionated into 14 segments by strong cation-exchange (SCX) chromatography on an UltiMate high-performance liquid chromatography (HPLC) system (LC Packings, The Netherlands) using a polysulfoethyl A column (2.1µm [diameter] x 150 mm [length], 5 µm [particle size] (Agilent Technologies, USA). The elution was carried out through utilizing two mobile phases: (A) 10 mmol/L KH2P04 and 25% ACN (pH 2.6), and (B) 10 mmol/L KH2P04, 350 mmol/L KCl, and 25% ACN (pH 2.6). A gradient elution started from 0% B, increasing to 80% B, and returning to 0% B over one hour at a flow rate of 0.2 mL/min. Subsequently, the 14 fractions were dried in a vacuum concentrator, resolved in 50 µL water with 0.1% formic acid, and used for Nano LC-MS/MS analysis. For the Nano LC-MS/MS analysis, each fraction (2 µL) was loaded onto a trap column (Thermo Scientific Acclaim PepMap C18, 100 μm×2 cm) for a 3-minute duration at a flow rate of 10 μL/min. The sample was then separated using the analytical column (Acclaim PepMap C18, 75 μm × 25 cm) with a linear gradient, progressing from 5% to 30% phase B over 95 minutes. Phase A consisted of water with 0.1% formic acid, and phase B was composed of ACN with 0.1% formic acid. The flow rate of the analytical column was maintained at 300 nL/min, and the column temperature was held at 45°C. The Q-Exactive mass spectrometer was operated in a data-dependent mode with full-scan MS spectra from 350-1600 m/z at a resolution of 70 K, followed by fifteen sequential high-energy collisional dissociation (HCD) MS/MS scans with a resolution of 17.5 K. For each case, a single microscan was recorded, with dynamic exclusion set at 30 seconds 24, 25. Three technical replications were conducted for Nano LC-MS/MS analysis. (4) Database search: The raw mass spectrometry data files were processed using Proteome Discoverer software (Thermo Fisher Scientific, version 1.4.0.288), and the MS/MS results were analyzed using Mascot software (Matrix Science, London, UK; version 2.3). Search parameters were as follows: Mus musculus as the taxonomy in the Uniprot-SwissProt database, trypsin enzyme specificity, a fragment ion tolerance of 0.050 Da, and a parent ion mass tolerance of 10 PPM. Carbamidomethyl modification of cysteine, iTRAQ 8-plex labeling of lysine, and the N-terminus were considered as fixed modifications, while oxidation of methionine and iTRAQ 8-plex labeling of tyrosine were designated as variable modifications. Peptide-level false discovery rates (FDR) were controlled lower than 1% by the percolator algorithm. Protein groups considered for quantification required at least one peptide with 95% confidence. Protein quantification relied on unique peptides as well as shared peptides. The experimental bias was corrected via normalization based on the protein median, with a minimum of 100 proteins required for normalization. Quantification of metabolites using mass spectrometry Overall, seven pairs of colon cancer tissue samples and their corresponding adjacent colon tissues (refer to Table S1) were used. Colon tissues were dissected into 1-2 mm3 fragments, washed with PBS to remove blood, and ground under liquid nitrogen. Metabolites were extracted at a tissue weight-to-extract volume ratio of 1:2.5 using acetonitrile: methanol (1:1) containing internal standards (200 ng/mL L-valine-d8 and 200 ng/mL L-isoleucine-d10). After thorough mixing, samples were frozen at -80°C overnight, thawed at room temperature (approximately 25°C), subjected to ultrasonic crushing for 10 minutes at 4°C, and centrifuged for 10 minutes at 12,000 rpm at 4°C. The supernatant was collected and the pellet was re-extracted using an equal volume of extract by repeating the above-mentioned procedure. The newly obtained supernatant was combined with the previous supernatant. Finally, 20 µL of the supernatant was diluted with 380 µL of water. The diluted solution (10 µL) was loaded onto an ultra-high-performance liquid chromatography/tandem mass spectrometry (UHPLC-MS) system (Waters Acquity UPLC system and AB Sciex Triple Quad 5500 mass spectrometer) for metabolite quantification. Compounds were separated using an Agela Venusil ASB C18 2.1*50 mm (Agela Technologies, California, USA) column using the mobile phase of 5 mM ammonium formate (A) and acetonitrile with 0.6% formic acid (B). Mass spectrometry detection was performed using multiple reaction monitoring (MRM) in the positive-ion mode. The analytes were detected three times in MRM mode using the MS conditions described in Table S2. Bioinformatics analysis The set of differentially expressed proteins (n = 74) was analyzed using the STRING software (version 11.5, https://string-db.org) 26. The analysis encompassed the examination of cellular components (CC), molecular functions (MF), biological processes (BP), KEGG pathways, and protein-protein interactions. To enhance the accuracy of the biological analysis, a minimum required interaction score of 0.7 (indicating high confidence) and an FDR lower than 0.05 were employed as selection criteria. In cases where the number of description items for the CC, MF, BP, and KEGG pathways (as exported from the STRING software) exceeded 10, only the top 10 were extracted. The protein network derived from the STRING software was subsequently imported into Cytoscape software (version 3.7.1) 27, and an interaction map was drawn using this software. Furthermore, the 74 differentially expressed proteins were entered into the Clinical Proteomic Tumor Analysis Consortium (CPTAC) database (cProSite-Cancer Proteogenomic Data Analysis Site) (https://cprosite.ccr.cancer.gov/) 28 to compare the relative protein abundance in tumor and normal tissue from patients with colon cancer. Moreover, proteins that were observed to have an abundance similar to that reported in the CPTAC data, were input to the TCGA database (http://gepia2.cancer-pku.cn/#survival) to analyze the relationship between protein expression and survival rate in the dataset of colon adenocarcinoma (COAD). Finally, the eight proteins (enriched in the leucine, isoleucine, and valine degradation pathway) were used to analyze the relationship between their protein expression and the survival rate of patients in any of the 33 cancer datasets from the TCGA database. Western blot analysis Protein extraction (25 μg) was initially separated by SDS-PAGE, followed by transferring onto a PVDF membrane. Subsequently, the membrane was incubated with primary antibodies: Na+/K+-ATPase (a plasma membrane biomarker) at a 1:2000 dilutions, prohibit (a mitochondrial apparatus biomarker) at a 1:4000 dilutions, β-actin at a 1:5000 dilutions, or ALDH2 at a 1:500 dilutions. This incubation process occurred overnight at 4°C. Afterwards, the PVDF membrane was washed three times with TBS/Tween 20 and then exposed to a secondary antibody at a 1:5000 dilutions for one hour. The protein bands were visualized using an ultra-high-sensitive ECL Kit. This experimental procedure was performed in triplicate for technical or biological replications and consistency. Real-time PCR Total RNA was extracted from frozen human colon tissues, including 14 pairs of tumor tissues and their adjacent colon tissues (refer to Table S1). The extraction process was performed in accordance with the manufacturer's instructions, including tissue cutting, cryogenic tissue disruption, and RNA extraction using a Trizol Kit (Thermo Fisher Scientific, USA). Subsequently, the RNA was reversely transcribed into cDNA using RevertAid Reverse Transcriptase (EP0442, Thermo Fisher Scientific, USA). Primer pairs for the differentially expressed genes were designed using Primer 5.0 software. The RT-qPCR reactions were conducted on an ABI 7500 real-time fluorescence quantitative PCR system (Thermo, USA) using PerfectStart Green qPCR SuperMix (AQ601-04, TransGen Biotech, Beijing, China), following the manufacturer's protocol. The reaction conditions consisted of an initial denaturation step at 95°C for 3 minutes, followed by 40 cycles of 95°C for 15 seconds, 59°C for 20 seconds, and 72°C for 30 seconds. To confirm primer specificity, a melting curve was generated by incrementally heating the amplicon from 60°C to 95°C. Each qPCR analysis was performed in triplicate. The forward and reverse primers of Aldh2, Hadh and GAPDH were ACAAAAGGGGACGAACACAG and GTGAGCCTGCTGTGAATCAA for Aldh2; CACACAGTAGTGTTGGTAGACC and TGCCACTTTCCTAAGGCTTTC for Hadh, and CCAGGTGGTCTCCTCTGA and GCTGTAGCCAATTCGTTGT for GAPDH, respectively. GAPDH was used as a reference gene. Comparative quantification was performed using the 2-ΔΔCq method 29. Statistical analysis For the proteomic study, quantitative data were statistically analyzed using Microsoft Excel 2010 (Microsoft, Washington, USA). The differentially expressed proteins were defined as a fold change of ≥ 1.50 or ≤ 0.666, with p < 0.05, for cancer-to-normal (117:114) and cancer-to-adjacent (117:116) comparisons, respectively, while, adjacent-to-normal (116:114) and normal-to-adjacent (114:116) comparisons were between 0.667 and 1.5 in all three replicates. R package software (version 4.0.2) (RStudio Inc., MINNESOTA, USA) was used to analyze the volcano plot, Venn diagram, and clustering data 30. For real-time PCR quantification and western blotting, statistical analyses were performed using GraphPad Prism software (version 8.3.0) (GraphPad Company, California, USA). Student's unpaired t-test was used to compare differences between two groups. Significant differences among multiple groups were determined using one-way analysis of variance (ANOVA) with multiple comparisons and adjusted using Tukey's post-hoc test. Differences were considered statistically significant at p < 0.05.\",\n",
       " '11190199': 'MethodsOver one year we measured the microbiome, peripheral counts of CD4+ T populations and cytokines in 67 patients shortly after a first ACS. We compared baseline measurements to those of a matched population of 40 chronic patients. A subgroup of 20 ACS patients underwent repeated assessment of fibrous cap thickness (FCT) of a non-culprit lesion.',\n",
       " '11190077': '2 Methods 2.1 Study design and patient populations Adult admitted between October 2018 and February 2020 with a first established diagnosis of ACS were invited to enter the MIGATER observational longitudinal study (Fig. 1). Exclusion criteria are summarized in the Online Supplemental Table 1. At admission, and days 7, 30, 90, 180 and 365, patients underwent clinical revisions as well as stool and blood sampling. Following standard of care, Mediterranean diet was recommended to all patients. The 14-item dietary screener of Mediterranean diet (MEDAS)[12] was completed at inclusion and clinical visits. Data from repeated samples were available in 67 patients. For comparison, we selected a group of 40 age and sex matched patients with angiographically documented stable chronic coronary artery disease (CAD) following the following inclusion criteria: > 1-year free of cardiac events or admissions and following identical exclusion criteria than for the ACS group. In addition, a coronary imaging substudy was designed for ACS patients in whom an intermediate lesion could be angiographically identified in a non-culprit coronary vessel. In these patients, we performed an optical coherence tomography (OCT) examination of this intermediate lesion (<50 % stenosis) and repeated it at day 365.Fig. 1MIGATER Study Cohort. 67 ACS patients and 40 matched chronic controls are the basis of this report. The study protocol conforms to the ethical guidelines of the 1975 Declaration of Helsinki as reflected in the approval of the Ethics Committee of the Hospital General Universitario Gregorio Marañón, Madrid, Spain (CEIm 237/17) and was registered as NCT03434483. All patients signed informed consent for inclusion in the study. 2.2 Microbiota analysis Total DNA was extracted from whole fecal samples. The hypervariable V4 region of the 16 s rRNA gene was analyzed by NGS (see Online Supplemental Material for details). We calculated alpha diversity using the Shannon index, which accounts for abundance and evenness of the species. Beta diversity, which captures changes in community composition, was calculated by the Bray-Curtis dissimilarity index in a non-msuetric multidimensional scaling (NMDS) using the phyloseq R package. Changes in microbiota abundance were analysed using ANCOMBC-2 [13]. 2.3 Peripheral CD4 + T cell counts. Fresh whole blood samples were employed for quantification of absolute counts (cells per μl) of CD4+ T cells. Absolute counts for Th0, Th1, Th2, Th17 and Treg populations were calculated by flow cytometry (Gallios cytometer, Beckman Coulter) using a combination of specific antibodies (Online Supplemental Figure 1) with flow-count fluorospheres (Beckman Coulter).[14]. 2.4 Cytokine determination We used a predesigned panel of 48 cytokines to measure in plasma samples using Olink Target 48 cytokine Plex (https://olink.com/products-services/target/48-cytokine-panel/) in a KTH-Scilifelab facility (Affinity Proteomics). We quantified C-reactive protein (CRP) from plasma samples using simple plex human CRP cartridge in an ELLA automated immunoassay system (Bio-Techne). 2.5 Coronary imaging In those cases, we could angiographically identify an intermediate (<50 % stenosis) lesion in a non-culprit coronary vessel, we performed an automatic pullback of high-resolution OCT using a frequency-domain catheter (Dragonfly Optis, Abbott, MN, USA; see Online Supplemental Material for details). The same segment was re-imaged 1-year later during an elective catheterization procedure (Fig. 2). OCT analysis was blindly performed by an independent core-laboratory (BARCICORE).Fig. 2Change in Fibrous Cap Thickness. Baseline (A) and 1-year (B) evolution of fibrous cap thickness (FCT) measured by OCT in a non-culprit lesion in a patient with a low CD4+ T cell trajectory. 2.6 Statistical analysis Variables are expressed either as median [interquartile range] or estimated marginal means (95 % confidence interval), as indicated. All statistical models were adjusted for age, sex, and body mass index (BMI) as well as for the first six scores resulting from a multiple correspondence analysis integrating all cardiovascular risk factors and relevant ongoing medications at each visit (Online Supplemental Figure 2). We used multivariate analysis of variance as the global test to compare differences in cell populations and microbiota in ACS vs chronic controls, followed by one-way ANOVAs. Longitudinal changes were analyzed using mixed effects models (see Online Supplemental Material for details). We performed an unsupervised multidimensional clustering of the longitudinal counts of the five CD4+ T cell subsets and cytokines using the kml3d algorithm for joint trajectories (see Online Supplemental Material for details). Baseline predictors of a high vs. low CD4+ T cell trajectory were identified using logistic regression and the c-index. Finally, we used a generalized estimating equation model to address the impact of a high vs. low CD4+ T cell trajectory on FCT and laboratory determinations. All statistical analyses were performed in R (v. 4.1.3); p-values < 0.05 and fold-changes in microbiome abundance > 1.5 were considered significant.',\n",
       " '11190063': \"2 Materials and methods 2.1 Materials and preparation The novel biodegradable high-N Fe-based alloy, referred to as HN–Fe, was prepared under a high-pressure N2 atmosphere of 1.5 MPa. Then, the alloy bars with diameter of 8 mm were obtained by forging at 1473 K with holding time of 2 h and hot rolling at 1423 K with holding time of 1.5 h. Finally, the HN–Fe wires with diameter of 0.23 mm were produced by multi-passes hot and cold drawings combined with annealing treatment. Inductively coupled plasma-optical emission spectrometer (ICP-OES, iCAP6300), carbon-sulfur analyzer (CS600) and oxygen-nitrogen-hydrogen analyzer (ONH836) were used to determine the alloy compositions. Commercial pure Fe wire with a purity of 99.95 % and a diameter of 0.20 mm was chosen as the control, referred to as Fe wire. The chemical composition was shown in Table 1.Table 1Chemical compositions of HN–Fe and Fe wires, wt.%.Table 1HN–Fe wireMnNCOSPSiFe29.320.58000.01500.00960.01600.0070<0.1Bal.Fe wireSnCuCSbAsBiPbFe0.0120.0090.0020.0060.0060.0020.001Bal. All wires were ultrasonically cleaned in the acid wash solution (HCl 500 mL, methenamine 20 g, and H2O 500 mL) to remove the oxide layer on the surfaces before the tests, since HN–Fe and Fe wires were easily oxidized in air. Subsequently, electrolytic polishing was performed in an electrolyte of 10 % perchloric acid and 90 % alcohol at 253 K and 39 V to make the surface smooth. The anastomotic staples were made using HN–Fe wires by polishing, cutting and bending, which were further sterilized by ethylene oxide and sealed in vacuum for gastrointestinal anastomosis experiments in animals. The pure Ti anastomotic staples commonly used in clinic were used as the control. 2.2 Tensile test The static uniaxial tensile tests were conducted on a universal tensile machine (Instron-5569) at room temperature for the wires before and after immersion, according to GB/T 228.1–2010 with a strain rate of 6.7 × 10−4 s−1 and a gauge length of 25 mm. The immersion parameters were as described in section 2.2.1. After 1, 3, 7, 14, 21 and 28 days of immersion, the specimens were removed from the Hank's solution (Table 2), rinsed with deionized water and anhydrous ethanol, and dried at room temperature. At least three specimens for each set were tested to ensure that the results obtained were reliable.Table 2Compositions of Hank's solution, g/L.Table 2NaClKClKH2PO4MgSO4NaHCO3CaCl2Na2HPO4·12H2OGlucose8.00.40.060.10.350.1850.121.0 2.3 Degradation test in vitro 2.3.1 Immersion test To evaluate the degradation properties of wires, weight loss analysis was used. The wire was cut into the length of 12 mm and each set of immersed specimens consisted of seven. Before immersion, the specimens were weighed and labeled. Based on ASTM-G31-72, the wires were all completely immersed with a ratio of 20 mL/cm2 at 37 °C in Hank's solution. Changing the solution every two days. Considering that bacteria may affect the corrosion process of Fe alloys, additional antibiotics were added to the solution, including penicillin 100 U/mL and streptomycin 100 μg/mL [28]. For the removal of degradation products, the immersed specimens were removed from the solution at 7, 14, and 28 days and ultrasonically cleaned in an acid wash for 2 min. The specimens were then weighed again. An average result was obtained by measuring at least three specimens for each time. Calculation of the corrosion rate (CR, mm/year) was given by Eq. (1) [25]:(1)CR = (K × W) / (A × T × D)where K is 87,600, W is the weight loss (g), A is the exposed area of the specimen (cm2), T is the immersion time (h), and D is the density of alloys (g/cm3). 2.3.2 Electrochemical measurement The electrochemical measurements were performed on a Gamry Reference 600+ electrochemical workstation by a standard three-electrode system, where the saturated calomel electrode (SCE) was the reference electrode, the platinum electrode was the counter electrode, and the wire was the working electrode. At least three tests were performed on all specimens at 37 °C in Hank's solution. A 1 h open circuit potential (OCP) test was conducted to determine the stability of the electrochemical system. The potentiodynamic polarization was tested from the corrosion potential (Ecorr) to the cathode (−0.5 V) and anode (+0.5 V), respectively. The scan rate was 0.33 mV/s. The CR of wire was calculated by Eq. (2):(2)CR = 3.27 × 103× Icorr× Ew/ ρwhere Icorr is the corrosion current density, Ew is the weight equivalent, taken as 27.92 g eq−1, and ρ is the density of the wire (g/cm3). The electrochemical impedance spectroscopy (EIS) was tested in the frequency range from 100 kHz to 10 mHz. The excitation signal was the sinusoidal voltage with an amplitude of 10 mV. Gamry Echem Analyst software was used to fit and analyze the data. 2.4 Microstructural characterization X-ray diffraction (XRD, D/max 2400 X) was used to characterize the phase structure of the initial wires with a scanning speed of 10°/min. The radiation source was Cu-Kα with a wavelength of 0.154056 nm. Scanning electron microscope (SEM, FEI Inspect F50) was used to observe the fracture morphology after tensile test, the surface morphology before degradation, and the corrosion morphology after degradation of the specimens. Energy dispersive spectrometer (EDS) was used to semi-quantitatively analyze the compositions of the wires and their degradation products. 2.5 Degradation test in vivo Sixteen male New Zealand white rabbits (2.5–3.0 kg) were randomly divided into HN–Fe-linear-cut staple group and Ti-linear-cut staple group, with eight rabbits in each group. The rabbits were anesthetized by an intramuscular injection of ketamine (3.0 mg/kg) and 2 mL 3 % sodium pentobarbital. The animals were placed in supine position, and immobilized. Epigastrium were shaved and sterilized with povidone-iodine. Cut along the linea alba below xiphoid process to expose the left side of the greater curvature. Partial gastrectomy was performed for greater curvature with HN–Fe-linear-cut staple and Ti-linear-cut staple, respectively. Then, returning gastric stump and layer sutured and incision closed. Postoperative intramuscular injection of 8 × 105 U penicillin sodium/each rabbit for three days to prevent infection. Abdominal X-ray fluoroscopy was performed at 1, 2 and 6 months postoperatively to clarify the morphology and degradation of the staples in vivo. After the removal of the anastomosis staples, weight loss analysis, surface and cross-section corrosion morphology of the staples were carried out as described in the previous section. 2.6 Biocompatibility test 2.6.1 Histology evaluation Six months after partial gastrectomy, all experimental animals were euthanized with sodium pentobarbital injection. The gastric wall tissue with staples were embedded by methyl methacrylate, and sliced (50 μm) with a hard tissue microtome (LEICA, Germany). The heart, kidney, liver, lung and spleen were collected and fixed with 10 % formaldehyde solution for 48 h, embedded in paraffin, and cut into sections with a tissue microtome. All samples were stained with hematoxylin-eosin (HE). And then sections were transferred to the two pathologists for pathology evaluation. 2.6.2 Blood routine and biochemistry After 2 and 6 months, all the experimental animals were anesthetized with 3 % sodium pentobarbital, the blood (2 mL) was drawn from their peripheral ear vein. Routine blood counts and liver and kidney function were examined using the ProCyte Dx and Catalyst One kits (IDEXX, USA). 2.6.3 Reverse transcription-polymerase chain reaction (RT-PCR) The gastric wall tissue at the insertion of the staples was collected and cleaned with phosphate buffered Saline. Total RNA from 50 mg tissue was isolated using RNAiso Plus (TAKARA Biotechnology, China). The concentration of RNA was determined by spectrophotometry at 260 nm. Total RNA (1 μg) was reverse-transcribed using a PrimeScript RT reagent Kit (Perfect Real Time) (TAKARA Biotechnology, China) in a total volume of 20 μL. Quantitative real-time PCR was performed on cDNA samples using a LightCycler480 II instrument (Roche, Basel, Switzerland) with the SYBR Green Premix Ex Taq II (TAKARA Biotechnology, China) in a total volume of 20 μL. The primer sequences for gastric wall tissues were as follows: IL-10-F, 5′-TTGTTAACCGAGTCCCTGCTT-3′; IL-10-R, 5′- CCACTGCCTTGCTCTTGTTT-3′; TNF-α-F, 5′-ACCCTCAGATCAGCTTCTCG-3′; TNF-α-R, 5′- TTGACCGCTGAAGAGAACCT-3′; TGF-β1-F, 5′-ACGGGCTCAACATCTACACA-3′; TGF-β1-R, 5′-GTGTCCAGGCTCCAGATGTA-3′; IL-8-F, 5′-TACAGAGCTTCGATGCCAGT-3′; IL-8-R, 5′-CCTTCTGCACCCACTTTTCC-3′; and GAPDH-F, 5′-GCTGAACGGGAAACTCACTG-3′; GAPDH-R, 5′-CCGAAGGTAGAGGAGTGGGTG-3′. The expression level of glyceraldehyde-3-phosphate dehydrogenase (GAPDH) was measured and used as an internal control. The data were analyzed using 2−ΔΔCt relative expression quantity as previously described [29]. 2.6.4 Enzyme-linked immunosorbent assay (ELISA) ELISA was used to measure the inflammatory factor secretion of gastric wall tissue at the insertion of the anastomotic nail. The gastric wall tissue was collected and cleaned with phosphate buffered saline, and the tissue were dissected out and homogenized in lysis buffer (tissue weight: lysis solution volume = 1:5). The homogenized samples were centrifuged at 5000 g for 10 min, and the supernatants were collected for further analysis. The concentrations of Interleukin 10 (IL-10), Tumor Necrosis Factor Alpha (TNF-α), Transforming Growth Factor Beta 1(TGF-β1) and Interleukin 8 (IL-8) were determined using a Rabbit Interleukin 10 ELISA Kit (Cloud-Clone Corp., USA), a Rabbit Tumor Necrosis Factor Alpha ELISA Kit (Cloud-Clone Corp., USA), a Rabbit Transforming Growth Factor Beta 1 ELISA Kit (Cloud-Clone Corp., USA), a Rabbit Interleukin 8 ELISA Kit (Cloud-Clone Corp., USA) according to the manufacturers' instructions. The results were reported in picograms per mL of proteins and compiled using Prism 9.0 GraphPad Software (GraphPad Software Inc., San Diego, CA).\",\n",
       " '11189975': 'MethodsThis prospective study included adults ≥60 years with any newly diagnosed non-small-cell lung cancer. Participants were recruited from central Ohio (2018–2020). Study assessments included the Cancer and Aging Research Group CGA (CARG-CGA), short physical performance battery (SPPB), and the blessed orientation-memory concentration (BOMC) test at baseline, 3, 6, and 12 months. Activities of daily living (ADLs) and instrumental ADLs (IADLs), quality of life (QoL, PROMIS 10), and treatment toxicity were captured monthly. Stool and blood were collected to characterize the gut microbiome and age-related blood biomarkers.',\n",
       " '11189645': '2 Methods 2.1 Study design This was a pilot prospective longitudinal cohort study of 50 adults ≥60 years of age with any stage of untreated or recurrent NSCLC. Eligible participants underwent a baseline assessment using the CARG CGA (Hurria et al., 2005; Nie et al., 2013). Briefly, this assessment included the CARG chemotherapy toxicity calculator, the Patient-Reported Outcome Measurement Information System (PROMIS 10 (Hays et al., 2009) Global Health Scale Short Form v1.1, and sociodemographic information. The CARG CGA includes self-reported functional status measures, comorbidity, medications, nutrition, psychological state, and social support. A clinical research coordinator administered the cognitive assessment, the Blessed Orientation-Memory-Concentration test (BOMC test (Tuch et al., 2021)), and the short physical performance battery (SPPB) (Guralnik et al., 1994; Kawas et al., 1995). Participants also underwent longitudinal peripheral blood draws and stool sample collection. Participants then received standard-of-care lung cancer treatment or participated in a clinical trial as decided by their treating physician. 2.2 Eligibility criteria Participants were recruited, consented, and enrolled in the FITNESS study at The Ohio State University Comprehensive Cancer Center (OSUCCC)–James Thoracic Oncology clinic. Eligible patients were ≥60 years of age and intended to receive systemic treatment at the OSUCCC for either a new or recurrent NSCLC diagnosis. Participants were required to have decision-making capacity and willingness to sign informed consent. Patients with any medical conditions, including but not limited to unstable angina pectoris, cardiac arrhythmia, psychiatric illness, or cognitive impairment, without a legally authorized representative (LAR) that would verify compliance with study procedures were excluded. 2.3 Study procedures Participants underwent a baseline assessment before initiating lung cancer treatment. This process included sociodemographic and disease assessment; CARG toxicity tool calculation (Hurria et al., 2011); stool and blood sample collection; and completion of the CARG CGA, the BOMC, the SPPB (Almugbel et al., 2022), and the timed up-and-go (TUG) (Nakano et al., 2021). The type of treatment and drug doses (planned and actual) were recorded along with any supportive care medications given throughout treatment. Grade 1–5 toxicities (National Cancer Institute Common Terminology Criteria for Adverse Events, NCI CTCAE V5) (Kluetz et al., 2016), hospitalizations, dose delays and reductions, and drug discontinuation were captured. The toxicity assessment and chemotherapy drug and dose information were collected by a study coordinator at the beginning of every treatment cycle, reviewed and attributed by two certified oncologists. Vital signs, labs, and health-related quality of life questionnaires, including the PROMIS 1028 and European Organization for Research and Treatment of Cancer (EORTC) Quality of Life Questionnaire for Lung Cancer (QLQ-LC-13) (Bergman et al., 1994), were collected at visit 2 (day 90) and visit 3 (day 180). Patients underwent additional monthly functional status assessments (FSA), monthly biospecimen banking, and an additional CGA at visit 3. Baseline assessments were repeated at the end-of-treatment (EOT) visit. These occurred up to 45 days after completing treatment or 365 days after treatment initiation—whichever occurred first. Due to the varying length of treatment plans and different staging at the time of diagnosis, participants prescribed only four cycles of chemotherapy (adjuvant treatment) were only eligible to participate in the baseline and EOT visits. Visit 1 (baseline), visit 2 (day 90), and visit 3 (day 180) allowed a ±30-day window of time, letting patients enroll up to 30 days after treatment initiation. Disease response was calculated using Response Evaluation Criteria in Solid Tumors (RECIST) 1.1 criteria by two medical oncology fellows using chest cat scan (CT) imaging (Eisenhauer et al., 2009). Target lesions were followed throughout the study at each visit in which imaging was available. Tumor response was categorized into four groups: complete response, partial response, stable disease, and progressive disease. Stool samples were collected at baseline and during participants’ oncology visits, roughly every 3–4 weeks. Patients were provided with a stool collection tub and collection instructions. Patients were asked to collect a sample within 24 h of returning to the clinic. Once the biospecimen sample reached the lab, it was aliquoted into cryovials and stored at −80°C. Samples were then processed in batches. DNA was extracted using the QIAGEN Power Fecal Pro kit. Sequencing was performed on the Illumina NovaSeq 6,000 platform to a depth of eight million reads per sample. Demultiplexed fastq files were cleaned, paired, and taxonomically assigned following the MetaPhlAn HUMAnN 3 Pipeline (Beghini et al., 2021). Peripheral blood (40 mL) was collected at baseline, 6 months, and at either 1 year or the end of the patient’s treatment plan. Blood samples were only collected from participants who had not initiated treatment before visit one to control the purity of data. The samples were processed by The Ohio State University Leukemia Tissue Bank (LTB). Peripheral blood T cells were isolated via negative selection using the RosetteSep human T cell isolation cocktail (Stemcell Technologies). Tcell RNA was isolated using a Zymo Research quick-DNA/RNA miniprep kit. The quality and concentration of the resulting RNA was quantified on an Agilent TapeStation and analyzed using a custom NanoString codeset panel, OSU Senescence. The OSU Senescence panel is comprised of 74- T-cell markers with five loading controls. Gene expression markers were selected to measure subset-specific T-cell mRNA markers of cellular senescence and T-cell differentiation proliferation, and exhaustion as previously published (Rosko et al., 2019; Lucas et al., 2020; Fell et al., 2022). Additionally, certain transcription profiles in T cells such as LAG3 and CD8 transcription expression has been shown to be associated with inferior clinical outcomes in patients with myeloma (Lucas et al., 2020). T cells are exhausted when cells lose cytotoxic capacity, show sustained expression of inhibitory checkpoint molecules such as LAG3 and have distinct altered transcription profiles in CD8 subsets (Lucas et al., 2020). Therefore, LAG3 and CD8 transcription profiles were examined for associations with longitudinal IADL scores. 2.4 Statistical analysis The primary aim of this study was to assess the feasibility of a longitudinal geriatric assessment collected concurrently with blood and stool biomarkers. Metrics included the number of blood and stool samples collected, the completion of PRO assessments, and patient withdrawal. Feasibility was defined a priori as the proportion of missing information at each time point, with success defined as at least 50% of participants able to complete study visits 2, 3, and four or EOT. Reasons for missed collections and study withdrawals were captured. Descriptive statistics were used to summarize demographic information, with totals and percentages used for categorical data and medians and interquartile ratio (IQR) used for continuous data. When determining the immune-related adverse event (irAE) grade for a given time point, the grade of the most extreme irAE for that time point was used. For association with baseline CARG, the maximum-grade irAE experienced by the patient was considered. Microbe and gene expression associations with the CGA were performed using longitudinal mixed-effects models using the {lme4} and {lmerTest} packages in R (Bates et al., 2015). A model was run for each combination of GA tool (i.e., SPPB or IADL), microbe, and gene. For associations between microbes and the selected GA tool, each GA tool was considered an independent variable and all other variables were considered dependent variables. Repeated measures in patients were handled using random effects for each patient. All other effects were fixed. The longitudinal nature of the data was accounted for by including a term for the number of days on the study associated with each measure. The models can be described by the following equation, in which the term A i is the score (A) for each geriatric assessment(i), M j is the relative abundance (M) for each microbe (j), D is days on study, C is immune checkpoint inhibitor, (1|P k ) is the intercept (1|P) for each patient (k): Ai,k=Mj,k+D+C+1|Pk For transcript and geriatric assessment tools, variables were centered and scaled before modeling. Geriatric assessments remained the independent variable and patient intercepts remain random effects. The effects of change over time are still controlled for by including days on study in the model. These models can be described with the following equation, where A i is the score (A) geriatric assessment of interest (i), G j is the expression (G) of the gene of interest (i), D is days on study, (1|P k ) is the intercept (1|P) for each patient (k): Ai,k=Gj,k+D+1|Pk Analyses were performed in R, version 4.1.',\n",
       " '11189522': 'Materials and methods Screening of active compounds in SZLBD The active compounds of 7 TCM in SZLBD were searched in the Traditional Chinese Medicine Systems Pharmacology (TCMSP) database (http://tcmspw.com/), and each compounds was screened according to the criteria of by the oral bioavailability (OB) ≥ 30% and drug likeness (DL) ≥ 0.18. Screening of common targets of SZLBD and UC The active compounds of SZLBD and their target information were retrieved from the TCMSP database, and the targets were converted into the corresponding human gene names using the STRING database (https://string-db.org/). UC-associated targets were obtained from the GeneCards database (https://www.genecards.org/) and the NCBI database by entering the keyword “ulcerative colitis”. All targets were limited to Homo sapiens. Next, the common targets of GeneCards and NCBI databases were retained as UC-related targets. Finally, the intersection targets of SZLBD-related targets and UC-related targets were considered as potential therapeutic target for SZLBD in the treatment of UC. Protein–protein interaction (PPI) networks construction The common targets of SZLBD and UC were entered into the STRING database (https://string-db.org/) for PPI network construction. The nodes and edges of the network represent proteins and protein–protein associations, respectively. PPI networks were visualized using Cytoscape 3.7.2 software, and topological features were analyzed to screen core targets. GEO verification We validated the expression levels of these core targets in UC using an external dataset from the Gene Expression Omnibus (GEO). We utilized the \"limma\" package in R language to obtain the expression levels of core genes in both the normal and UC groups. We then generated box plots using the Sangerbox online platform (http://sangerbox.com/home.html). GO functional annotation and KEGG signaling pathway enrichment analysis In order to understand the biological functions and related signaling pathways involved in the common targets of SZLBD and UC, GO functional annotation (including biological process (BP), cell ingredient (CC) and molecular function (MF)) and KEGG signaling pathway enrichment analysis were performed for the common targets using Metascape database (https://metascape.org/). In the end, there are only p-values < 0.01, count > 3, enrichment factor (enrichment factor is the ratio between observed counts and counts expected by chance) > 1.5 is considered a significant enrichment result12,13. Molecular docking The molecular structure of the core active compounds were downloaded from the PubChem database (https://pubchem.ncbi.nlm.nih.gov/). The protein crystal structure of the key targets were obtained from the RCSB PDB database (PDB, https://www.rcsb.org/). The structure of compounds and proteins were processed by AutoDockTools (version 1.5.6). Optimal combinations of docking score were visualized by PyMOL.',\n",
       " '11189458': 'MethodsSix-week-old db/db mice were fed a diet containing 12.5% resistant starch or a control diet matched for equivalent regular starch for 10 weeks. db/m mice receiving the control diet were utilised as non-diabetic controls. Freshly collected kidneys were digested for flow cytometry analysis of immune cell populations. Kidney injury was determined by measuring albuminuria, histology, and immunohistochemistry. Portal vein plasma was collected for targeted analysis of microbially-derived metabolites. Intestinal histology and tight junction protein expression were assessed.',\n",
       " '11189444': 'Methods Animals 4-week old male db/m and db/db mice either heterozygous or homozygous for a mutation in the leptin receptor gene (Lepr) gene (BKS. Cg-Dock7m + /+ Leprdb/J) respectively, were purchased from The Jackson Laboratory (Bar Harbor, ME). The db/db mouse exhibits chronic hyperglycemia, hyperphagia, obesity, and insulin resistance and is widely utilized as a model of type 2 diabetes mellitus, whilst the db/m mouse is a suitable non-diabetic control mouse [7]. The number of mice per group was selected based on pilot data, to detect a change in the urinary albumin creatinine ratio of at least 100 g/mol (α = 0.05, 1 - β = 0.8) between the control and RS-supplemented diabetic mice. Mice were acclimatized for a period of 2 weeks, prior to the commencement of diet intervention at 6 weeks of age. Mice were housed in a climate-controlled animal facility with a 12:12 h light-dark cycle and received ad libitum access to mouse chow and water. Each week, mice were weighed and had random blood glucose levels measured using an Accu-Chek® Performa glucometer (Roche Diagnostics, USA). All study protocols were conducted in accordance with the principles and guidelines devised by the Alfred Medical Research & Education Precinct Animal Ethics Committee (AMREP AEC) under the guidelines laid down by the National Health and Medical Research Council of Australia and had been approved by the AMREP AEC (E1487/2014/B). Diets At 6 weeks of age, mice commenced a specialty formulated semi-pure resistant starch supplemented diet (SF15-015) containing 25% g/g Hi-maize 1043 (Ingredion, Westchester, IL), equivalent to 12.5% g/g resistant starch type 2 [8] or a custom-made matched control diet (SF15-021). Both diets were prepared by Specialty Feeds (Perth, Western Australia, Australia) and were matched in terms of total caloric content, protein and fat (Supplementary Table 1). The control diet was received by the non-diabetic mice (designated “db/m”). The diabetic mice were randomly allocated by cage to receive either the control diet (designated “db/db”) or the resistant starch supplemented diet (designated “db/db RS”). Mice received these diets ad libitum for a period of 10 weeks. Two cohorts of mice, referred to in this paper as Cohort 1 and Cohort 2, were utilized for different experimental endpoints. Both cohorts received the same diets. The experimental overview for each cohort is demonstrated in Fig. 1A and Fig. 4A, respectively.Fig. 1Resistant starch reduces kidney injury in diabetic mice.A Schematic of study design for Cohort 1. Created with biorender. B Urinary Albumin Creatinine Ratio, C Plasma Creatinine Levels, D 24-h urine output. * = P < 0.05, *** = P < 0.001, **** = P < 0.0001. One-way ANOVA with Tukey’s post hoc test. Data = mean ± SD. n = 10–12. Metabolic caging, plasma and tissue collection At experimental week 9 (15 weeks of age), mice in Cohort 1 were housed individually in metabolic cages (Iffa Credo, L’Arbresle, France) for 24 h for urine collection and measurement of water and food intake. Commercially available ELISA tests were utilized for the measurement of urinary albumin (Kit: E90-134, Bethyl Laboratories Inc., USA) and C5a (Kit: DY-1250, R&D Systems, USA). Urinary creatinine was determined via a commercially available assay kit (Kit: 03263991190, Roche Diagnostics Corporation, USA) by a Cobas Integra 400 Plus autoanalyzer (Roche Diagnostics Corporation, USA). At the end of the experimental intervention, mice were anesthetized by intraperitoneal injection of sodium pentobarbitone (100 mg/kg body weight; Euthatal; Sigma-Aldrich, Castle Hill, Australia). Following euthanasia, blood was drawn from the portal vein, treated with sodium citrate (3.2% v/v), centrifuged at 6000 rpm for 6 mins, and plasma was collected and stored at −80 °C. Both kidneys were removed for tissue digestion and flow cytometry. As both kidneys were utilized in Cohort 1 for flow cytometry to investigate renal immune cell populations, a second cohort of mice was run to investigate the effects of these diets on renal and intestinal histology. At the endpoint, mice were anesthetized by intraperitoneal injection of sodium pentobarbitone (100 mg/kg body weight; Euthatal; Sigma-Aldrich, Castle Hill, Australia) followed by cardiac exsanguination. Cardiac blood was centrifuged at 6000 rpm for 6 mins, and plasma was collected and stored at −80 °C. Kidney sections were fixed in neutral buffered formalin (10% v/v) before being embedded in paraffin. The gastrointestinal tract was dissected, and the mesentery was removed. The total gastrointestinal tract, and then the dissected cecum and colon, were weighed and length was measured. The gastrointestinal tract was flushed with chilled phosphate-buffered saline. Jejunum and ileum sections were fixed in paraformaldehyde (4% v/v) for 24 h before being transferred to 4% sucrose solution and embedded in paraffin. Jejunum, ileum, and colon sections were snap-frozen in liquid nitrogen and stored at −80 °C. Flow cytometry Both kidneys were collected from mice in Cohort 1. A single-cell suspension was prepared as follows. Briefly, both kidneys from each mouse were mechanically dissociated (sliced into cubes ~1–2 mm, by scissors) followed by orbital incubation at 37 °C with Collagenase IV (1 mg/ml, Sigma-Aldrich) for 30 min in RPMI 1640 medium with 5% fetal bovine serum. Kidney homogenates were filtered through a 70-μm cell strainer, and then subjected to a Percoll gradient centrifugation in 36% Percoll (Sigma-Aldrich), overlaid onto a 72% Percoll solution and centrifuged at 1000 g with no brake for 20 min at 4 °C. Cells were isolated from the Percoll interface and washed in cold media (RPMI 1640 medium with 5% fetal bovine serum). Erythrocytes were lysed using RBC lysis buffer (BD Biosciences). The resulting single-cell suspension was then treated with Fc Block (anti-CD16/CD32, BD Biosciences, clone 2.4G2) to block non-specific binding of antibodies. Cells were analysed with a panel of antibodies including Brilliant Violet 786–conjugated (BV786) anti-CD45 antibody (BD Biosciences, clone 30-F11, 1/2000), AF700-conjugated anti-CD11b antibody (Biolegend, clone M1/70, 1/1000), BV605-conjugated anti-Ly6C antibody (BD Biosciences, clone AL-21, 1/1000), Fluorescein isothiocyanate-conjugated (FITC) anti-C5aR1 antibody (Cedarlane, clone 20/70, 1/300), PE anti-Ly6G antibody (Tonbo biosciences, clone 1A8, 1/1000). To distinguish between live and dead cells, cells were stained with a Fixable Aqua live dead cell stain kit (Invitrogen, L34966). All antibody and live dead stain concentrations were used according to the manufacturer’s recommendations. Cells were counted with a Tali Image-Based Cytometer (Life Technologies). Analysis was performed on a BD FACSCanto II system using BD FACSDiva software (BD Biosciences). Plasma analyses and targeted metabolomics At cull, portal vein blood was collected from mice in Cohort 1. Plasma lipopolysaccharide-binding protein (LBP) was measured using an ELISA (ab269542, Abcam) as per the manufacturer’s instructions. The intra-assay coefficient of variation was 5.2%. Plasma endotoxin was measured using a commercially available assay based on Limulus amoebocyte lysate (A39552, Pierce Thermo Scientific, Rockford, IL, USA). Portal vein blood that had not undergone any prior freeze-thaw cycles was used for targeted metabolomics as previously described [9]. Renal histology Kidney sections with a thickness of 3 micrometers were subjected to periodic-acid Schiff (PAS) staining. The extent of sclerosis in each glomerulus was evaluated using a subjective 0–4 scale, where 0 represented normal conditions, and higher grades indicated increased sclerotic areas: minimal (grade 1), sclerotic area up to 25%; moderate (grade 2), sclerotic area 26 to 50%; grade 3, sclerotic area 51 to 75%; severe (grade 4), sclerotic area 76 to 100% (severe). The Glomerulosclerosis Index (GSI) was computed using a formula that factored in the number of glomeruli in each grade as previously described [10]. Digital images of the renal cortex were taken with an Eclipse Ci brightfield microscope (Nikon, Melville, NY, USA) at 200x magnification. Evaluations were performed using Image-Pro Plus (version 7.0; Media Cybernetics, Bethesda, MD, USA) and analysis was conducted in a blinded fashion. Renal immunohistochemistry Paraffin sections of mouse kidney (4 μm) were immunostained with rabbit anti-fibronectin (AO245, DAKO) at a 1:400 dilution. In brief, endogenous peroxidases were blocked by 3% hydrogen peroxide for 15 min, followed by a pepsin antigen retrieval for 10 min, after which samples were blocked in Dako Superblock (1:10 in TBS) for 20 min. A primary antibody was applied and left overnight at 4 °C. The following day, slides were incubated with biotinylated secondary antibody for 10 min at room temperature. Sections then underwent incubation with Vectastain ABC reagent (Vector Laboratories, CA, USA). Peroxidase activity was identified by reaction with 3,3’-diaminobenzidine tetrahydrochloride (Sigma-Aldrich Pty. Ltd, NSW, Australia). Counterstaining with hematoxylin was done to identify nuclei. Sections were examined under light microscopy (Olympus BX-50; Olympus Optical). All digital quantitation (Image-Pro Plus, v7.0) and assessments were performed in a blinded manner. Intestinal Histology Paraffin-embedded ileum and jejunum sections, with a thickness of 5 micrometers, were prepared for analysis. To assess villi length and crypt depth, the sections were stained with hematoxylin and eosin. For the enumeration of goblet cells, the sections were initially stained with 1% Alcian blue for 15 min, followed by periodic acid Schiff staining. Images were captured using an Eclipse Ci brightfield microscope (Nikon, Melville, NY, USA) at a 100x magnification. Image-Pro Plus software (version 7.0, Media Cybernetics, Bethesda, MD, USA) was employed to view the images and measure villi length and crypt depth utilizing the built-in measurement tool. The number of goblet cells per villus was manually counted for each villus. All analyses were performed in a blinded manner. Quantitative RT-PCR RNA was extracted from snap-frozen colon, jejunum and ileum samples using TRIzol Reagent (Life Technologies) as previously described [10]. cDNA was synthesized from RNA using M-MuLV Reverse Transcriptase (Thermo Fischer). Expression of zonula occludens-1 (ZO-1; Tjp1) and junctional adhesion molecule A (JAM-A; F11r) was determined using TaqMan reagents (Life Technologies, Carlsbad, CA). For occludin (Ocln), claudin-1 (Cldn1), claudin-2 (Cldn2), claudin-3 (Cldn3), claudin-4 (Cldn4), claudin-5 (Cldn5) and claudin-7 (Cldn7), SYBR Green reagents (Applied Biosystems, California, USA) were utilized. RT-PCR was conducted using either a QuantStudio 3 or 5 Real-Time PCR System (Thermo Fisher). Gene expression was normalized to β-actin (Applied Biosystems) and fold change was calculated relative to db/m CON mice using the ΔΔCT method. Statistical analysis Statistical analyses were performed using Graphpad Prism (Version 9.3.1, Graphpad Software, USA). Outliers were assessed and removed using robust regression and outlier removal (ROUT) [11] using a false discovery rate (FDR) value of 0.01. Data were assessed for normality using Kolmogorov-Smirnov test. Normally distributed data were analyzed using one-way ANOVA with Tukey’s post hoc test for multiple comparisons. Non-normally distributed data were assessed using Kruskal-Wallis test with Dunn’s multiple comparisons test. Targeted metabolomics data were log-transformed and analyzed using Metabolanalyst (version 5.0) [12], with a fold change threshold of 2 and an FDR threshold of 0.05.',\n",
       " '11189354': 'Methods Ethics statement This study was approved by the Public Institutional Review Board Designated by Ministry of Health and Welfare IRB (IRB no.: P01-202112-11-001). Each experiment was approved by the necessary Institutional Review Board, and informed consent was obtained from all participants. All methods in this study were performed in conjunction with the relevant guidelines and regulations. Subjects and sample collection 40 healthy volunteers of ages 20–59 years with comparable numbers of males and females were recruited. All volunteers self-reported to be in good general health with no underlying diseases. The volunteers were asked to wear a facemask for 8 h and continue their daily lives without any restriction of their behavior. Micro-organisms from the skin were sampled from the volunteers’ cheeks (4 cm2) with a sterile CultureSwab 220116 (BD, USA). A total of 80 skin swabs from 40 volunteers were obtained at two time points, before and after wearing a facemask. Saliva and facemask samples were also collected after wearing a facemask. The volunteers were asked to refrain from eating and drinking for 1 h prior to saliva sampling. Saliva samples (1 mL) were collected from volunteers in a 50 mL sterile Falcon tube (Becton, Dickinson and Company, New Jersey, USA). Micro-organisms from facemasks were sampled with a sterile CultureSwab 220116 (BD, USA) after removal of the facemask. All samples were stored at −80 °C before nucleic acid extraction for 16S rRNA analysis. DNA extraction Genomic DNA was extracted from skin swabs, facemask swabs, and saliva samples using the DNeasy PowerSoil Kit (QIAGEN, Hilden, Germany) according to the manufacturer’s instructions. Bacterial DNA was extracted from swabs as previously described (Meisel et al., 2016). Secure PowerBead Tubes were horizontally pulsed using a Vortex Adapter for 24 (1.5–2.0 mL) tubes at maximum speed for 20 min. Tubes were placed on ice for 1 min every 10 min to prevent excess heating. Saliva samples (200 µl) were used in a single extraction, including a wash step with 1 mL TEN buffer before lysis to increase yield. Post-extraction DNA quantity and quality were measured using a Thermo NanoDrop 1000. PCR amplification and Illumina sequencing PCR amplification was performed using primers targeting the V3 to V4 regions of 16S rRNA from extracted DNA. For amplification, we used primers 341F (5’-TCGTCGGCAGCGTC-AGATGTGTATAAGAGACAG-CCTACGGGNGGCWGCAG-3’; underlining sequence indicates the target region primer) and 805R (5’-GTCTCGTGGGCTCGG-AGATGTGTATAAGAGACAG-GACTACHVGGGTATCTAATCC-3’). PCR amplifications were carried out using the following conditions: initial denaturation at 95 °C for 3 min, followed by 30 cycles of denaturation at 95 °C for 30 s, primer annealing at 55 °C for 30 s, and extension at 72 °C for 30 s, with a final elongation at 72 °C for 5 min. A second amplification step for attaching the Illumina NexTera barcode was performed with a i5 forward primer (5’-AATGATACGGCGACCACCGAGATCTACAC-XXXXXXXX-TCGTCGGCAGCGTC-3’; X indicates the barcode region) and i7 reverse primer (5’-CAAGCAGAAGACGGCATACGAGAT-XXXXXXXX-GTCTCGTGGGCTCGG-3’). Conditions for secondary amplification were the same as above but with eight amplification cycles. Successful amplification was confirmed using 1% agarose gel electrophoresis and visualization with a Gel Doc system (BioRad, Hercules, CA, USA). The amplified products were purified with the CleanPCR kit (CleanNA). Equal concentrations of purified products were pooled together, and short fragments were removed using the CleanPCR kit (CleanNA). The quality and product size were assessed using a Bioanalyzer 2100 (Agilent, Palo Alto, CA, USA) with a DNA 7500 chip. Mixed amplicons were pooled, and sequencing was carried out, at ChunLab, Inc. (Seoul, Korea) with the Illumina MiSeq Sequencing system (Illumina, USA) according to the manufacturer’s instructions. Processing of sequencing data Raw sequencing data were filtering to remove low quality (<Q25) reads using Trimmomatic ver. 0.32. After QC, paired-end sequence data were merged using the fastq_mergepairs command from VSEARCH version 2.13.4 with default parameters. Primers were then trimmed with the alignment algorithm of Myers & Miller at a similarity cutoff of 0.8. Nonspecific amplicons not encoding 16S rRNA were detected using nhmmerin HMMER software package ver. 3.2.1 with hmm profiles. Unique reads were extracted, and redundant reads were clustered using the derep_fulllength command of VSEARCH. The EzBioCloud 16 S rRNA gene sequence database was used for taxonomic assignment using the usearch_global command from VSEARCH, followed by pairwise alignment. Chimeric reads were filtered for those with <97% similarity by reference based chimeric detection using the UCHIME algorithm and the nonchimeric 16S rRNA database from EzBioCloud. After chimeric filtering, reads classified to the species level (with <97% similarity) in the EzBioCloud database were compiled, and cluster_fast command was used to perform de novo clustering to generate additional OTUs. Finally, OTUs consisting of single reads were omitted from further analysis. The secondary analysis consisting of diversity calculation and biomarker discovery was conducted by in-house programs from ChunLab, Inc (Seoul, South Korea). The alpha diversity indices (ACE, Chao, Jackknife, Shannon, NPShannon, Simpson, and Phylogenetic diversity), rarefaction curves, and rank abundance curves were estimated. To visualize sample differences, beta diversity distances were calculated by several algorithms (Jensen-Shannon, Bray-Curtis, Generalized UniFrac, and Fast UniFrac). With functional profiles predicted by PICRUSt and MinPath algorithms, taxonomic and functional biomarkers were discovered by statistical comparison algorithms (LDA Effect Size - LEfSe and Kruskal-Wallis H Test). All analyses were performed in EzBioCloud 16S-based MTP, ChunLab’s bioinformatics cloud platform. Anaerobic culture and bacterial isolation All culture procedures were conducted in an anaerobic chamber. The inside of the facemask swab samples were resuspended in 1 mL DPBS by vortexing. 100 µL of the facemask sample solutions was spread on an RCM agar plate and incubated for 2 days. Five anaerobic bacterial colonies were isolated per facemask sample plate. Inside facemask (IFM) strains were characterized by 16S rRNA sequencing, using the following primers: 27F 5’-AGAGTTTGATCCTGGCTCAG-3’, 1492R 5’-GGTTACCTTGTTACGACTT-3’ and 518F 5’-CCAGCAGCCGCGGTAATACG-3’ 800R 5’-TACCAGGGTATCTAATCC-3’. These two primer sets are the most basic primers used for 16S microbial identification, forming a product using the outermost primers 9F/27F or 1492R/1512R. We used these two primer sets for accurate 16S microbial identification. PCR amplifications were carried out using the following conditions: initial denaturation at 95 °C for 5 min, followed by 30 cycles of denaturation at 95 °C for 45 s, primer annealing at 55 °C for 45 s, and extension at 72 °C for 60 s, with a final elongation at 72 °C for 10 min. Intradermal murine infection model Female 7–10-week-old C57BL/6J mice were purchased from Doo-Yeol Biotech and housed in randomly allocated groups of five under specific pathogen free conditions in the ABSL-2 animal facility at the Research Institute of Bioscience and Biotechnology. The experiments were conducted following the approved protocols by the Institute Animal Care and Use Committee of Korea Research Institute of Bioscience and Biotechnology (AEC-22118). All mice were provided with nesting material for enrichment. At experimental endpoints, mice were euthanized by inhalation of CO2 according to local guidelines. Skin infections were conducted according to previously described protocols62,63. Briefly, the back of each mouse was shaved with a microtome blade and wiped with alcohol pads. 30 µl of midlogarithmic bacterial suspension (OD600nm of 0.1) was injected intradermally using an insulin syringe. All procedures were conducted under anesthesia via intraperitoneal injection with 200 μl of 2.5% Avertin (Sigma-Aldrich). Lesions were photographed at 3 days post-infection (DPI), and lesion areas were measured using ImageJ. At 3 DPI, mice were euthanized, and skin tissue was sampled with a 5 mm diameter biopsy and weighed. For histopathological analysis, mouse skin was flattened and fixed using 4% paraformaldehyde (PFA; Thermo Scientific, J19943-K2) for 48 h. After fixation, biopsies were embedded in paraffin and sectioned at 4 μm thickness. Biopsy sections were mounted onto slides and stained with hematoxylin and eosin (H&E) prior to analysis. Tissue slides were examined blind by a veterinary pathologist and scored from 0 to 5 as previously described. In this scale, a score of 1 indicates minimal to mild focal dermal inflammatory cell infiltrate, a score of 2 indicates mild multifocal dermal inflammatory cell infiltrate, a score of 3 indicates multifocal to diffuse moderate dermal inflammatory cell infiltrates extending to the skeletal muscle, a score of 4 indicates marked dermal inflammatory cell infiltrates extending to the skeletal muscle with intraepithelial microabscesses, and a score of 5 indicates full-thickness massive inflammation with myositis and epithelial erosion. Analysis of saliva anaerobic microbiota by whole-metagenome shotgun sequencing For whole metagenome analysis, we used shotgun sequencing of saliva DNA to assess taxonomy. A total of 100 µL of saliva from the respective subject (subjects were tracked based on the presence or absence of IFM strain detection) was spread onto RCM agar plates, followed by incubation under anaerobic conditions for 48 h. Subsequently, colonies were harvested by gently spreading 100 µL of 1X PBS onto the agar surface with a spreader. Five microliters of bacterial suspension were transferred to a fresh nuclease-free tube before addition of an equal volume of 1X PBS. Ninety microliters of 40 mg/mL Lysozyme (Sigma-Aldrich) and 90 µL of 0.2 mg/mL Lysostaphin (Sigma Aldrich) were subsequently added to the tube, before incubation at 37 °C for 30 min. Subsequent steps were performed as described in the QIAamp® DNA Mini and Blood Mini Handbook under Appendix D: Protocols for Bacteria. DNA was sequenced on an Illumina NovaSeq 6000 Sequencer by Accugene, Incheon, South Korea. Samples were processed with the Illumina NovaSeq 6000 S4 Reagent Kit v1.5 (300 cycles) according to manufacturer’s protocol. Analysis of IFM strains by whole genome sequencing For whole genome analysis, we used shotgun sequencing of IFM strain DNA to assess taxonomy. DNA extraction was performed on the selected pathogenic strain, IFM, which was used for intradermal infection. Five microliters of bacterial suspension were transferred to a fresh nuclease-free tube as above before addition of an equal volume of 1X PBS. Ninety microliters of 40 mg/mL Lysozyme (Sigma Aldrich) and 90 µL of 0.2 mg/mL Lysostaphin (Sigma Aldrich) were subsequently added to the tube, before incubation at 37 °C for 30 min. Subsequent steps were performed as described in the QIAamp® DNA Mini and Blood Mini Handbook under Appendix D: Protocols for Bacteria. DNA was sequenced on an Illumina NovaSeq 6000 Sequencer by Accugene, Incheon, South Korea. Samples were processed with the Illumina NovaSeq 6000 S4 Reagent Kit v1.5 (300 cycles) according to the manufacturer’s protocol. In vitro pathogen growth assays We tested the direct effects of 200 anaerobic bacteria isolated from facemasks on the growth of IFM12 in vitro using conditioned supernatants. After 48 h of growth in RCM under anaerobic conditions at 37 °C, all bacterial cultures were centrifuged at 3000 × g for 5 min. Supernatants were subsequently filter sterilized with a 0.2 μm filter to remove planktonic bacteria. 20 μL of supernatant from each culture and 2 μL of OD600 = 0.005 pathogen culture were added to 180 μL of fresh 5x diluted RCM medium. All samples were incubated for 18 h at 37 °C with shaking before measuring the OD600nm with a TECAN instrument. Statistical analysis of IFM12 growth modulation by conditioned supernatants was performed with a Bonferroni t-test comparing differences between each supernatant treatment and the control. When pathogen growth was enhanced or suppressed by a particular supernatant, the derivative strains were termed ‘pathogen helpers’ or ‘pathogen inhibitors,’ respectively. In addition, we determined whether supernatants of pathogen helper and pathogen inhibitor, obtained using the method described above, affect S. aureus strains other than IFM12, respectively. Two reference strains, including USA300 and ATCC25923, as well as six S. aureus isolates from facemasks, were incubated in RCM under aerobic conditions at 37 °C for 14 h. Subsequently, their optical density was adjusted to an OD600nm of 0.05. In 96-well plates, 20 μL of supernatant from the helper and inhibitor, along with 2 μL of the pathogen culture, were added to 180 μL of freshly diluted RCM medium (5x dilution). All samples were incubated at 37 °C for 6 h with shaking, before the OD600nm was measured using a TECAN instrument. For a second round of screening, we assessed the capacity of 200 anaerobic isolates from facemasks to inhibit pathogen helper strains. We first selected strong pathogen helper strains (50M5_1), which greatly enhanced pathogen growth in supernatant assays. We then assessed whether supernatants could suppress or promote pathogen helper strain growth using the same experimental set up described above. Multi-locus sequence typing for C. acnes isolated from the facemask Multi-locus sequence typing (MLST) analysis was conducted on pathogen helper C. acnes isolated from facemasks using methods outlined on the pubMLST web server (https://pubmlst.org/organisms/cutibacterium-acnes). Briefly, partial sequences of the housekeeping loci, including aroE (424 bp), atpD (453 bp), gmk (400 bp), guaA (493 bp), lepA (452 bp), sodA (450 bp), tly (777 bp), and camp2 (807 bp), were PCR amplified as previously described50,51. PCR was carried out in a final volume of 50 μl using an AllInOneCycler™ 384-well PCR system (Bioneer, South Korea). Sequencing was performed with the BigDye Terminator v3.1 sequencing kit (Applied Biosystems, USA) on an ABI3730XL sequencer (Applied Biosystems, USA). In vivo assessment of pathogen virulence and skin disease development in the presence of either live pathogen helper strains or conditioned supernatants To study the indirect effects on pathogen infection, we selected eight supernatants of isolated anaerobic bacteria from the facemask. Mouse skin was intradermally infected with the pathogenic strain in addition to the pathogen helper strain alone or together with pathogen helper supernatant. Each supernatant from the eight selected bacteria was used in combination with the strong pathogen helper strain and pathogen strain (Table S2), in addition to a PBS only control, a pathogen strain only group, and a group administered the pathogenic strain and the strong pathogen helper strain with no supernatant. The experiment was performed five times. Six days after infection, 5 mm diameter skin samples were obtained by biopsy. Quantification of pathogen burden To determine pathogen infection burdens, we homogenized skin samples before serially diluting them and plating them on 1.5% agar-RCM plates containing rifampicin (50 µg/ml). Plates were subsequently incubated, and the number of colony forming units (CFU) was determined. Immunohistochemistry for IL-1β expression The skin histological sections underwent a standard deparaffinization process in xylene, followed by rehydration in ethanol and blocking with 3% hydrogen peroxide. Subsequently, the sections were incubated with a polyclonal rabbit anti-IL1β antibody (NB600-633, Novus Biologicals). Following washing, the sections were subjected to a secondary antibody conjugated to horseradish peroxidase (HRP) and visualized using 3,3’-diaminobenzidine (DAB). Hematoxylin counterstaining was performed, and the signal intensity was quantified utilizing image analysis software (ImageJ 1.54e). Quantification of virulence gene expression of S. aureus Strain IFM12 was inoculated into 15 mL of RCM broth at 37 °C in 200 mL flat bottom flasks to a starting OD600 of 0.05 before incubating for 6 h at 250 rpm shaking in the presence or absence of cell free supernatant (CFS). After incubation, RNase inhibitors (RNAlater, Ambion, TX, USA) were added before harvesting cells and immediately freezing on dry ice. To harvest cells, cultures were centrifuged at 15,000 × g for 1 min. TRIzol™ Reagent (0.75 mL) was added to bacterial pellets that had been stored at −80 °C. RNA extraction was subsequently performed according to the manufacturer’s instructions with the following modifications. To obtain RNA from S. aureus, we added glass beads and vortexed for 1 min, before placing the sample on ice for 1 min and repeating this process 10 times. The RNA was dissolved in either RNase-free water for short-term usage or deionized formamide for long-term usage, and stored at −80 °C. qRT-PCR with gene-specific primers was used to determine expression of nine virulence genes (Accessory gene regulator protein A (agrA), Clumping factor A (clfA), Fibronectin binding protein A (FnBPA), Endoproteinase Glu-C (V8), Serine protease SplA(splA), Zinc metalloproteinase aureolysin (aur), Alpha-hemolysin (hla), Transcriptional regulator (sarA), and Immunoglobulin G-binding protein A (spa)) in IFM12 (Supplementary Table 4). qRT-PCR was performed using a SYBR Green PCR master mix (Thermo Fisher Scientific, Waltham, MA). Metabolite extraction To identify IPH strain-specific compounds, we extracted metabolites from RCM medium and CFS samples from IPH and IFM15 strains. Aliquots of 200 μL were mixed with 800 μL of ice-cold methanol and vortexed for 1 min. For each sample, 800 μL was dried in a SpeedVac and was reconstituted with 1000 μL of 80% methanol. The resuspended samples were filtered through a 0.22 μm syringe filter (Hyundai Micro Co., Ltd., Seoul, Korea) for Ultrahigh performance liquid chromatography-Orbitrap-mass spectrometry (UHPLC-Orbitrap-MS) and gas chromatography-time-of-flight mass spectrometry (GC-TOF-MS) analysis. Mass spectrometry analysis For GC-MS analysis, resuspended samples were dried and subjected to derivatization. First, 50 μL of methoxamine hydrochloride in pyridine (20 mg/mL) was added to the dried samples and incubated at 30 °C for 90 min for oximation. Next, silylation was performed by adding 50 μL of MSTFA to the resulting mixture, followed by incubation at 37 °C for 30 min. The GC-TOF-MS analysis was performed using an Agilent 7890 B GC system with An Rtx-5MS capillary column (30 m × 0.25 mm × 0.25 μm particle size; Restek, Bellefonte, PA, USA) for chromatographic separation. The oven temperature was maintained at 75 °C for 2 min, before increasing to 300 °C at a rate of 15 °C/min and held for 3 min. The mass data were collected in 50–600 m/z (in positive ion mode) using Pegasus BT TOF-MS (LECO, St. Joseph, MI, USA). One microliter of the sample was injected into the GC-TOF-MS systems with a split ratio of 15:1. For LC-HRMS analysis, the UHPLC system was equipped with a Vanquish binary pump C system (Thermo Fisher Scientific) and a Waters ACQUITY UPLC HSS T3 column (150 mm × 2.1 mm, 1.8 μm particle size, Waters, Wexford, Ireland) for chromatographic separation. The mobile phases consisted of 0.1% formic acid in water (A) and 0.1% formic acid in acetonitrile (B). The LC eluents were run following the gradient: 5% B, 0–1 min; 5–100% B, 1–10 min; 100% B, 10–11 min; 100–5% B, 11–13 min; 5% B, 13–15 min. MS data were collected in the range of 100–1500 m/z (in negative and positive ion mode) using an Orbitrap Exploris™ 120 system combined with an ion-trap mass spectrometer (Thermo Fisher Scientific). Thirty microliters of resuspended sample was diluted with 60 μL of 80% methanol and transferred to a vial. Five microliters of the sample were injected into the LC-HRMS systems. All samples were analyzed in a random order toward non-targeted metabolite profiling. Data processing for metabolomics data GC-MS raw data files were converted to NetCDF (*.cdf) using LECO Chroma TOF software (version 4.44). LC-MS raw data were converted to mzXML files using the msconvert program from ProteoWizard. The converted files of GC-MS and LC-Ms were processed for retention time correction, peak detection, and alignment using the Metalign software package (http://www.metalign.nl) and XCMS online software (https://xcmsonline.scripps.edu), respectively (HMDB; https://hmdb.ca/). Metabolic differences between IPH strains, IFM15 strains, and RCM medium were determined using SIMCA-P+ software (version 12.0, Umetrics, Umea, Sweden) based on partial least squares discriminant analysis (PLS-DA) modeling. The discriminant metabolites were selected based on variable importance in the projection (VIP) values (>1.0) and were tentatively identified by comparing in house and reference spectral libraries. Antimicrobial susceptibility testing The susceptibility of pathogen helper (PH) C. acnes and pathogen S. aureus strains to phenyllactic acid was assessed using a modified version of the Clinical and Laboratory Standards Institute (CLSI) standard microdilution method. The PH and pathogen strains were cultured on agar for 2 days under anaerobic conditions. Colonies of each strain were suspended in 1 ml of RCM media with an OD of 0.3, and the suspension was further diluted at a 1:100 ratio. Subsequently, 100 µl of this diluted suspension was added to a 96-well plate. Phenyllactic acid was serially two-fold diluted in a range from 0.098 to 12.5 mg/ml and then treated and incubated with the bacterial suspension for 48–72 h under anaerobic conditions. The inhibition rate (%) was calculated by comparing the OD reduction in the treatment well to the OD of the non-treated control well. Determination of differentially expressed genes (DEGs) between inhibitor of helper bacteria (IPH) and non-IPH strain The quality of RNA sequencing raw data was checked using FastQC (v0.11.9) software (2015), after the trimmed using Trimmomatic (v0.39) software. The trimmed data were mapped to the whole genome sequences of each strain using bowtie2 (v2.4.5) software. The mapping data were assigned read counts to each gene using HTSeq (v1.99.2). To compare transcription levels between strains, an orthologous search was performed using OrthoFinder (v2.5.5) software, and the control S. aureus IFM15 gene with the highest score was assigned to each gene of IPH strain M664. Afterwards, filtering and trimmed mean of M-values (TMM) normalization were performed using the edgeR (v3.36.0) package of the R (v4.1.3) language, and Differentially expressed genes (DEGs) were selected based on |log2FC| > 1 and FDR < 0.05. Cluster of orthologous groups (COGs) were assigned to the determined DEGs using a conserved domain search service65. Afterwards, hypergeometric p-values were obtained for the designated functional pathway associated with the COG, and these p-values were adjusted with Bonferroni correction. Significant functional pathways were selected based on these p-value being less than 0.05. Statistical analysis All descriptive data is presented as median with interquartile range. The skin virulence of the facemask isolates and the effect of pathogen helper and inhibitor on pathogen growth and skin lesion were analyzed by Student’s t-test. When comparing mean differences between groups to evaluate indirect interactions between the skin virulent inside facemask strain (IFM) and other IFM strains, we used an analysis of variance (ANOVA) and Duncan’s multiple range test. Differences were considered significant when p-values were less than 0.05. When comparing mean differences between groups for evaluation of indirect interactions, we used an analysis of variance (ANOVA) and Duncan’s multiple range test. Differences were considered significant when p-values were less than 0.05. Statistical analyses and graphs were performed using GraphPad Prism version 10. The relationship between age and direct/indirect effects on pathogen strain growth in vitro was analyzed using linear regression using R 3.6.3 (www.r-project.org) and Sigma Plot (V.12.5). Metabolic difference between IPH strains, IFM15 strains, and RCM medium were considered significant based on ANOVA and Duncan’s multiple range test, and also by Student’s t-test for IPH-specific compounds.',\n",
       " '11189342': 'Methods Lecithin, sucrose fatty acid esters, and CMC dietary emulsifiers-fed C57BL/6JNarl mouse model The animals were handled following the guidelines of the Institutional Animal Care and Use Committee of National Taiwan University (approval number: NTU107-EL-00121). Male C57BL/6JNarl mice were purchased from the National Laboratory Animal Center (Taipei, Taiwan). The mice were housed in an animal room with a 12 h light/dark cycle at 23 ± 2 °C. After acclimation, 15-week-old mice were randomly divided into four experimental groups (n = 3–4/cage): (i) control, (ii) lecithin (7523.3 mg/kg bw/day, 4.625% in drinking water), (iii) sucrose fatty acid esters (1110 mg/kg bw/day, 0.68% in drinking water), and (iv) CMC (1% in drinking water). CMC was used as a positive control, and its dosage was selected based on a previous study10. The dosages of lecithin and sucrose fatty acid esters were selected based on a report on the estimated dietary exposure of humans (lecithin: 61 mg/kg bw/day; sucrose fatty acid esters: 9 mg/kg bw/day)13 and translated to the corresponding mouse dosages59. Emulsifier dosages were calculated based on the average body weight of mice within each experimental group. Emulsifier doses in this experiment were 10 times the respective daily exposure levels in humans. Emulsifiers were purchased from Gemfont Corporation (Taipei, Taiwan): soy lecithin (LASENOR, India; #15682), sucrose fatty acid ester (Mitsubishi Chemical Corporation, Japan; Ryoto Sugar Ester P-1570 (sucrose palmitate)), and sodium carboxymethylcellulose (Nouryon, Finland; #8021514). Mice were fed with a normal chow diet (MFG; Oriental Yeast Co., Ltd., Tokyo, Japan), and all dietary emulsifiers were supplemented in drinking water. Mice in each experimental group were allowed free access to food and water for 17 weeks. Before sacrifice, the mice fasted for 12 h. The mice were subsequently sacrificed using CO2 asphyxiation. Blood was collected via cardiac puncture using a syringe. Organs were collected for subsequent analysis. Cell culture, differentiation, insulin-resistance model, and glucose uptake assay 3T3-L1 pre-adipocytes (ATCC CL-173) were cultured and maintained in a pre-adipocyte expansion medium, which consisted of low-glucose (1 g/L) Dulbecco’s modified Eagle’s medium (DMEM; Invitrogen, USA), supplemented with 10% bovine calf serum (BCS) and 1% penicillin-streptomycin solution, at 37°C in a 5% CO2 incubator. The differentiation procedure of 3T3-L1 cells into adipocytes followed a previous publication with modifications60. Cells were seeded in a 96-well plate at a density of 1×104 cells per well and grown for 2 days until reaching 70% confluency. Subsequently, the cells were treated with 1 µM dexamethasone (Sigma-Aldrich, USA, D4902), 1 µg/ml insulin (Sigma-Aldrich, UAS, I9278), and 0.5 mM 3-isobutyl-1-methylxanthine (IBMX; Sigma-Aldrich, USA, 5879) for 72 h in low-glucose DMEM containing 10% fetal bovine serum (FBS) and 1% penicillin-streptomycin solution. At the endpoint of adipocyte differentiation, adipocytes were washed with phosphate-buffered saline (PBS) and rinsed with an assay medium. The doses of emulsifiers administered to mice were converted to doses appropriate for the insulin resistance (IR) model in mature 3T3-L1 cells. Ratios of 1:100 and 1:1000 were used for each emulsifier, as depicted in Supplementary Fig. 4. For the glucose uptake assay, adipocytes were treated with assay medium supplemented with 10 μg/mL insulin, 1 μM dexamethasone, and emulsifiers (LEC, 0.2 mg/ml and 2 mg/ml; SUC, 0.03 mg/ml and 0.3 mg/ml; CMC, 0.1 mg/ml and 1 mg/ml; all emulsifiers were dissolved with PBS) for 30 and 60 min. Glucose uptake was measured using the Glucose Uptake-Glo Assay (Promega, USA, J1342), following the manufacturer’s instructions. MDG dietary emulsifiers-fed C57BL/6JNarl mouse model The mice were housed in an animal room with the previously described conditions. Six-week-old male C57BL/6JNarl mice were employed for the experiment after an adaptation period of 2 weeks. Eight-week-old male C57BL/6JNarl mice were divided into two groups based on their diet: (i) control diet group, and (ii) MDG diet [5.5%] group. The MDG dosage for this experiment followed a previous study that studied the effect of 5.5% diacylglycerol (DAG), exhibiting no signs of systemic toxicity15. The control mice were fed with an AIN-93M mature rodent diet. The MDG diet was customized from the AIN-93M mature rodent diet (#D10012M; Research Diets, Inc.) by replacing soybean oil with MDG, and MDG composition was 5.5% in the diet. The nutrition facts of the AIN-93M and MDG diet are shown in Supplementary Table 2. Mice were limited to 85 kcal/week16, receiving 12.14 kcal/mouse (~3.20 g/mouse) daily. Calorie control was implemented to minimize the differences in dietary intake of the mice for solely observing the effect of MDG. The mice were sacrificed after 14 weeks of the experiment. Before sacrifice, mice fasted for 12 h. After CO2 asphyxiation, the blood was collected by cardiac puncture using a syringe and the organs were collected for subsequent analysis. Body composition analysis Both lean and fat mass body composition was measured using a Minispec LF50 TD-NMR body composition analyzer (Bruker, Billerica, MA, USA). The relative fat or lean mass was calculated as the fat or lean mass to the body weight, respectively. Mouse blood biochemistry analysis The serum was extracted by centrifuging the blood at 1000 × g for 15 min at 4 °C. Serum biochemical biomarkers, including total cholesterol, total triglyceride, high-density lipoprotein (HDL-c), AST, ALT, and glucose, were measured using commercial test strips (Spotchem II reagent strips; Arkray Inc., Kyoto, Japan) in an automatic blood analyzer (Spotchem EZ). OGTT The mice were fasting for 5 h before OGTT experiments. A blood sample was collected from the submandibular vein. Then the blood glucose was analyzed using a glucometer (ACCU-CHEK® Performa, Roche, Basel, Switzerland) at 0, 15, 30, 60, 90, and 120 min after oral gavage with 2 g/kg glucose. Intestinal permeability analysis Intestinal barrier permeability was investigated by administering the intestinal permeability probe FITC-dextran (MW 4,000; Sigma-Aldrich, 46944). After 4 h of fasting, the mice were gavaged with 0.2 mL of FITC-dextran solution (15 mg of FITC-dextran in phosphate-buffered saline). Blood was collected after 3 h. The serum was extracted by centrifuging the blood at 1000 × g for 15 min at 4 °C. Plasma FITC-dextran fluorescent intensity was measured using a fluorometer in black 96-well plates at excitation and emission wavelengths of 485 and 538 nm (Fluoroskan Ascent FL, Thermo Fisher Scientific, USA). FITC-dextran concentrations were measured against a standard curve produced by serial dilution of FITC-dextran in mice serum. Periodic acid-Schiff staining and histopathological analysis The mouse colons were removed and fixed in Carnoy’s solution for 24 h and subsequently embedded in paraffin. The colon sections were stained with the periodic acid-Schiff staining method. Hepatic histological scoring was conducted by an experienced pathologist at the College of Veterinary Medicine Animal Disease Diagnostic Center (National Chung Hsing University, Taichung, Taiwan). Visualized the bacteria co-localized the mucus layer (i) Paraffin embedding with Carnoy’s fixation The mouse colonic tissues with fecal matter were immersed in Carnoy’s solution (60% methanol, 30% chloroform, 10% glacial acetic acid) for 24 h. For embedding, samples were immersed twice in anhydrous methanol for 30 min each, 100% ethanol for 20 min each, and xylene for 15 min each, and finally immersed in paraffin wax (Leica, Germany) at 56–58 °C for 3 h each. Blocks were hardened at room temperature, sectioned to a 4 μm thickness, floated on a water bath at 40–45 °C, and transferred to slides. (ii) Fluorescent in situ hybridization Slides were deparaffinized by immersing twice in xylene for 15 min each and 100% ethanol for 5 min each, following which the samples were treated with protease K (5 μg/mL) at 37 °C for 15 min and the treated samples were immersed in 0.9 M NaCl, 20 mM Tris pH 7.4 for 10 min in preparation for fluorescence in situ hybridization (FISH). FISH targeting bacterial cells was performed using the oligonucleotide probe EUB338 (5′-GCTGCCTCCCGTAGGAGT-3′, with a 5′ Alexa 647 label) custom synthesized (Li-Tzung Inc., Taiwan). Sections were incubated in a hybridization buffer (0.9 M NaCl, 0.02 M Tris-HCl pH 7.4, 0.01% SDS, 30% formamide) and 2 μM probe at 46 °C for 2.5 h. Samples were washed at 48 °C for 15 min excess wash buffer after hybridization. (0.215 M NaCl, 0.02 M Tris-HCl pH 7.5, 5 mM EDTA). (iii) Fluorescent staining with Hoechst 33258 and wheat germ agglutinin (WGA) Sections were dyed with Hoechst 33258 (10 μg/mL) and WGA (40 μg/mL) Alexa Fluor 488 conjugate (Thermo Fisher Inc., USA) in phosphate-buffered saline for 15 min and incubated twice for 3 min each in wash buffer (0.112 M NaCl, 20 mM Tris-HCl pH 7.4, 5 mM EDTA, 0.01% SDS). Slides were then washed in water and dried or were submerged for 3 min each into 50%, 80%, and 96% (v/v) ethanol and then dried. Slides were subsequently mounted with Prolong anti-fade mounting media (Life Technologies) and placed in a dark environment at room temperature until the mounting medium solidified. Images were acquired with a Leica TCS SP5 II confocal microscope (Joint Center for Instruments and Researches, College of Bioresources and Agriculture, National Taiwan University, Taiwan). (iv) Measurement of mucus thickness The mucus layer was determined by using Fiji to measure the width of the region brightly stained with WGA and on the interior of the host epithelium61. We visualized the border of the host epithelium using differential interference contrast and then merged the channel to measure the distance. Measurement of serum LPS and insulin concentration Pierce LAL Chromogenic Endotoxin Kit (#88282, Thermo Scientific, USA) was used to detect the LPS levels in the serum. Serum insulin levels were measured using commercial kits supplied by Mercodia AB (#10-1247-01, Mouse Insulin ELISA kit, Mercodia, Sweden). Gut microbiota analyses For these analyses, we decided to use cecal samples instead of fecal samples due to the cecum’s pivotal role in microbial fermentation, notable impact on gut microbiota composition, and potential association with metabolic diseases. DNA from cecal samples was extracted according to the manufacturer’s instructions using the QIAamp PowerFecal DNA Kit (Qiagen, Netherlands). PCR amplification of 16S rRNA V4 regions was performed using the 515F/806R bacterial primer pair (515F: 5′-GTGCCAGCMGCCGCGGTAA-3′ and 806R: 5′-GGACTACHVGGGTWTCTAAT-3′), with a 50 µL reaction volume containing 25 µL 2X Taq Master Mix (Thermo Scientific, USA), 0.2 M of each forward and reverse primer, and 20 ng DNA template. Thermal cycling consisted of initial denaturation at 95 °C for 5 min, followed by 25 cycles of denaturation at 98 °C for 20 s, annealing at 57 °C for 15 s, and elongation at 72 °C for 30 s, with a final hold at 72 °C for 10 min. Next, amplified products were subjected to 2% agarose gel electrophoresis. The amplified PCR product was attached with Illumina sequencing adapters with a Nextera XT Index kit and then purified using AMPure XP beads. Library quantification was achieved employing the DNA 1000 kit and 2100 Bioanalyzer instrument (Agilent Technologies, USA), and sequencing (single-end reads) was completed using Illumina NextSeq (Illumina, USA). Raw sequences were processed following the QIIME2 pipeline. Raw sequences were quality filtered, trimmed, de-noised, merged, and chimeric sequences removed using the QIIME2 dada2 plugin. The ASVs were aligned against the SILVA database (version 138). The alpha diversity, including the observed ASVs and Shannon and Simpson indices, was computed using the vegan package in R. A PCoA was conducted using the Bray–Curtis distance. Additionally, an ANOSIM was performed to determine the heterogeneity of gut microbiota among the groups. Finally, a heatmap at the genus level was generated using the heatmap3 package in R. Statistics and reproducibility All statistical analyses were carried out using R Studio (version 1.2.5001), R (version 3.6.1), or GraphPad Prism (version 9.5.1). Data are presented as the mean ± standard deviation (s.d.). For experiments with lecithin, sucrose fatty acid esters, and CMC emulsifier-fed mice, statistical analysis was performed using one-way analysis of variance (ANOVA) with the Tukey’s range test to determine significant differences between groups, while for experiments with MDG emulsifier-fed mice, a two-tailed Student’s t-test was used for the same purpose. Heterogeneity of the cecal microbiota among groups in the PCoA was assessed using ANOSIM. The gut microbiome-associated vector in the PCoA was calculated using the “envfit” function in the vegan package in R. Heatmap analysis of the relative abundances of cecal microbiota was conducted using the Kruskal–Wallis test and Wilcoxon signed-rank test for experiments with hydrophilic and lipophilic emulsifier-fed mice, respectively. Spearman’s correlation coefficient analysis was conducted to assess the relationship between gut microbiome genera and obesogenic and metabolic biomarkers. Reporting summary Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article.',\n",
       " '11189021': 'MethodWe collected bladder urine samples from 68 adult patients with calcium oxalate stones and 54 age-matched healthy controls by transurethral catheterization. 16S rRNA gene and shotgun sequencing were utilized to characterize the urinary microbiota and functionality associated with calcium oxalate stones.',\n",
       " '11188982': 'Materials and methods Participant recruitment The study protocol was approved by Ethical Review Board of Shenzhen People’s Hospital, Second Clinical Medical College, Jinan University (LL-KY-2020337). Informed consents were obtained from all subjects. Between April 2021 and July 2022, a total of 122 adults were recruited at our hospital, including 68 kidney stone formers and 54 age-matched healthy volunteers. Kidney stones were initially evaluated by ultrasonography, abdominal plain film, intravenous pyelography and computed tomography, and confirmed during endoscopic surgery. We initially screened the CaOx stone formers by X-ray findings and value measurement on CT. The chemical composition of surgically removed stones was further confirmed by infrared spectroscopy analysis. Both calcium oxalate monohydrate and calcium oxalate dihydrate were considered as calcium oxalate composition, while other mineralogies such as carbonate apatite and uric acid were not considered. For kidney stone formers, exclusion criteria included stones of calcium oxalate composition <90%, concurrent ureteral calculus, and moderate to severe hydronephrosis. For healthy participants, exclusion criteria included personal history of urinary stones, episodes of renal colic or imaging confirmed urinary stones. All healthy controls underwent ultrasonography or computed tomography to exclude asymptomatic renal calculus. Excluded from both groups were subjects with urinary tract infections or positive urine culture, antibiotic use within 30 days, indwelling catheter or ureteral stent, congenital abnormalities of the urinary tract, history of major urological surgery, malignant tumors, diabetes, chronic kidney disease, autoimmune disease and age (< 20 years or >70 years old). Sample collection and processing Bladder urine samples were collected by transurethral catheterization from all participants in a fasting state. All samples were obtained prior to antibiotic use, and the volume of each urine sample was approximately 50 ml. To remove host cell fragment, the urine samples were immediately centrifuged 300g for 5 minutes at 4°C. Then the supernatant was collected and centrifuged 12,000g for 15 minutes at 4°C. Pellets were re-suspended, mixed with DNA-free phosphate buffered saline, and stored in sterile containers at -80°C within 2 h from collection. The supernatant collection and pellets resuspension were performed on a clean workbench, following strict aseptic procedures. 16S rRNA microbial profiling analyses Genomic DNA was extracted from urine samples using MagPure Stool DNA KF Kit B (MAGEN, China). The DNA concentration was quantified by Qubit® 2.0 Fluorometer (Life Technologies, USA), and DNA integrity was verified with agarose gel electrophoresis. The DNA extractions were stored at −20°C until further processing. Polymerase chain reaction was performed to amplify the V3-V4 region of the 16S rRNA gene, with primers shown as follows: V3-V4-338F: 5′- ACTCCTACGGGAGGCAGCAG- 3′ and 806R: 5′- GGACTACHVGGGTWTCTAAT -3′. The amplicons were purified using the AMPure beads (Axygen, USA), and barcoded libraries were sequenced on Illumina Hiseq2500 platform. From the raw sequencing data, low-quality reads were eliminated and clean reads were merged to tags using FLASH software (v1.2.11) (He et al., 2013). Then the sequences with 100% similarity were assigned to the same operational taxonomic units (OTUs) using Divisive Amplicon Denoising Algorithm 2 in software QIIME2, a newer algorithm compared to traditional Usearch with 97% similarity. RDP Classifier (v2.2) (Wang et al., 2007) was used to taxonomically classify the representative sequences of each OTU, based on the Greengenes database (V201305). For α−diversity, observed species, ACE index, chao 1 index, Simpson index, Shannon index and Good-coverage index were calculated by Mothur (v.1.31.2) (Schloss et al., 2009). For β−diversity, both weighted and unweighted UniFrac distances were conducted using QIIME (v1.80), and shown by the principal coordinate analysis (PCoA) (Caporaso et al., 2010). LEfSe analysis (Segata et al., 2011) was performed to find biomarkers differentially represented between the KS and HC groups. Shotgun metagenomics analysis of urine samples DNA extracted from 30 urine samples selected to include 15 KS subjects and 15 controls, representative of the average composition of KS individuals and controls, was employed to perform shotgun metagenomics sequencing. The DNA extractions were fragmented to 550–650 bp by a Focused-ultrasonicator (Covaris, USA). The sample preparation was performed using BGI Optimal DNA Library Prep Kit (BGI, China), and barcoded libraries were sequenced on a MGI MGISEQ-2000 platform. All raw data were pre-processed by SOAPnuke v.2.2.1 (Chen et al., 2018), and the trimmed reads were mapped to the host genome using SOAP2 software to identify and remove host originated reads. High-quality reads were de novo assembled using MEGAHIT software (Li et al., 2015), and assembled contigs with length less than 300 bp were discarded. Genes were predicted over contigs by using MetaGeneMark (Zhu et al., 2010). Redundant genes were removed using CD-HIT (Fu et al., 2012) with the identity and coverage cutoff 95%, 90%, respectively. To generate the annotation information, the protein sequences of genes were aligned against the functional database (BacMet, CARD, CAZy, COG, KEGG, EggNOG and Swiss-Prot), using DIAMOND (Buchfink et al., 2015) with an E value cutoff of 1e−5. We identified differentially enriched KEGG pathways according to reporter scores (Patil and Nielsen, 2005), and the detection threshold for significance was an absolute value of reporter score of 1.65 or higher. Statistical analyses Statistical analyses were performed using the SPSS (version 21.0) and R software (version 4.1.3), and p values < 0.05 was considered as statistically significant. Clinical continuous variables were compared using a student’s t test, while categorical variables were analyzed via Pearson’s chi-square test or Fisher’s Exact Test. The continuous variables were expressed as mean ± standard deviation. For 16S rRNA sequencing, α-diversity analysis were performed with mothur (v.1.31.2) using Wilcoxon rank-sum test, while β−diversity was calculated by software QIIME (v1.80). For shotgun metagenomics, both α-diversity and β−diversity was calculated by python package of R project, using scikit-bio and scipy, respectively.',\n",
       " '11188961': 'MethodsIn this study, to understand gut microbiota signatures and serum metabolite changes in postmenopausal women with reduced BMD, postmenopausal individuals with normal or reduced BMD were recruited and divided into normal and OS groups. Feces and serum samples were collected for 16S rRNA gene sequencing, liquid chromatography coupled with mass spectrometry (LC-MS)-based metabolomics and integrated analysis.',\n",
       " '11188957': 'Materials and methods Study subjects This study was approved by the Ethics Committee of The Second People’s Hospital of Dalian (No.2022.174X). For gut microbiota analysis, postmenopausal subjects with normal BMD (normal group, n=38) or reduced BMD (OS group, n=25) were recruited. Clinical information was presented in Table 1 . For metabolomics analysis, normal group (n=61) and OS group (n=61) were recruited. All subjects participating in the study were over 50 years of age and had undergone menopause. Detailed clinical information is presented in Table 2 . The study did not include participants with cancer, kidney disease, metabolic or genetic bone disease, digestive system disease, psychiatric illness, or those who had taken antibiotics within 3 months or those taking medications that might affect bone metabolism. Table 1 Clinical information of the participants for gut microbiota analysis. Characteristics Normal (n=38) OS (n=25) P-value BMI (kg/m2) 25.17 ± 3.51 23.35 ± 2.82 NS Age (years) 65.21 ± 7.55 63.76 ± 6.63 NS LS BMD (g/cm2) 1.24 ± 0.18 0.99 ± 0.11 <0.001 LS T-score 1.24 ± 1.54 -1.02 ± 0.94 <0.001 LS Z-score 2.02 ± 1.57 0.46 ± 1.01 <0.001 Neck BMD (mg/cm2) 955.16 ± 172.41 778.04 ± 165.85 <0.001 Neck T-score 0.67 ± 1.45 -0.72 ± 1.37 <0.001 Neck Z-score 1.67 ± 1.70 0.56 ± 1.38 <0.01 Hip BMD (mg/cm2) 1015.63 ± 116.46 806.44 ± 156.85 <0.001 Hip T-score 0.65 ± 0.92 -0.94 ± 1.21 <0.001 Hip Z-score 1.16 ± 1.14 -0.08 ± 1.67 <0.001 ALP (U/L) 74.10 ± 21.13 93.43 ± 25.91 <0.01 VD (ng/mL) 27.74 ± 9.58 21.78 ± 6.18 <0.01 PTH (pg/mL) 33.20 ± 11.67 32.28 ± 10.73 NS T-PINP (ng/mL) 46.22 ± 16.85 71.52 ± 29.51 <0.001 β-CTX (pg/mL) 504.07 ± 192.87 739.78 ± 263.93 <0.001 N-MID (ng/mL) 14.98 ± 5.21 23.38 ± 10.74 <0.001 E2(pmol/L) 52.11 ± 16.59 42.89 ± 10.37 <0.05 Values are mean ± SD for continuous variables and the P value is based on the Wilcoxon rank-sum test. NS, nonsignificance. Table 2 Clinical information of the participants for serum metabolites analysis. Characteristics Normal (n=61) OS (n=61) P-value BMI (kg/m2) 25.52 ± 3.42 23.82 ± 3.15 NS Age (years) 64.25 ± 7.25 67.41 ± 7.94 NS LS BMD (g/cm2) 1.24 ± 0.15 0.96 ± 0.12 <0.001 LS T-score 1.18 ± 1.33 -1.27 ± 0.98 <0.001 LS Z-score 2.07 ± 1.42 0.51 ± 1.04 <0.001 Neck BMD (mg/cm2) 949.30 ± 159.88 714.18 ± 143.89 <0.001 Neck T-score 0.67 ± 1.34 -1.25 ± 1.21 <0.001 Neck Z-score 1.68 ± 1.57 0.23 ± 1.21 <0.001 Hip BMD (mg/cm2) 1007.59 ± 120.66 755.21 ± 135.88 <0.001 Hip T-score 0.60 ± 0.95 -1.33 ± 1.05 <0.001 Hip Z-score 1.17 ± 1.08 -0.24 ± 1.01 <0.001 ALP (U/L) 73.51 ± 23.86 91.32 ± 23.68 <0.001 VD (ng/mL) 20.17 ± 10.68 20.23 ± 8.30 <0.01 PTH (pg/mL) 31.93 ± 12.45 34.72 ± 12.12 NS T-PINP (ng/mL) 49.98 ± 27.34 70.75 ± 30.21 <0.001 β-CTX (pg/mL) 507.54 ± 193.38 725.53 ± 300.86 <0.001 N-MID (ng/mL) 15.62 ± 5.07 22.28 ± 9.51 <0.001 E2(pmol/L) 56.08 ± 19.40 45.61 ± 15.09 <0.01 Values are mean ± SD for continuous variables and the P value is based on the Wilcoxon rank-sum test. NS, nonsignificance. Clinical data and sample collection All subjects were interviewed and measured for age, height, and weight, as well as their body mass index (BMI). Fresh feces and blood samples were collected after more than six hours of fasting in the morning. After finishing serological testing, all collected serum and feces samples were transported to -80°C freezer for further analysis. Serum levels of total 25 (OH) vitamin D (VD), alkaline phosphatase (ALP), osteocalcin (N-MID), C-terminal peptide of type I collagen (β-CTX), N-terminal propeptide of type 1 procollagen (T-P1NP), and parathyroid hormone (PTH) were measured. BMD, T-score and Z-score in the lumbar spine (L1-4, LS), femoral neck (Neck) and total hip joint (femoral neck, trochanteric and intertrochanteric areas, Hip) were measured using a dual-energy X-ray absorptiometry scanner. Microbiota 16S rRNA gene sequencing Total genome DNA from feces samples was extracted using CTAB/SDS method. DNA concentration and purity was monitored on 1% agarose gels. According to the concentration, DNA was diluted to l ug/μL using sterile water. Mix same volume of IX loading buffer (contained SYB green) with PCR products and operate electrophoresis on 2% agarose gel for detection. PCR products was mixed in equidensity ratios. Then, mixture PCR products was purified with Qiagen Gel Extraction Kit (Qiagen, Germany). Fecal DNA was isolated and the V4 hyper variable regions of the bacterial 16S rRNA gene was amplified by primers 515F (GTGCCAGCMGCCGCGGTAA) and 806R (GGACTACHVGGGTWTCTAAT). After denaturation for 1 minute at 98°C, 30 cycles were performed, each lasting 10 seconds, followed by annealing at 50°C for 30 seconds and elongation at 72°C for 30 seconds. As soon as PCR products were purified, TruSeq DNA PCR-free sample preparation kits (Illumina, USA) and index codes were added to create sequencing libraries. At last, the libraries were sequenced on an Illumina NovaSeq platform and 250 bp paired-end reads were generated. The QIIME V.2.0 pipeline was used after barcodes were taken off the sequences (Hall and Beiko, 2018). Random selection of reads was used to reduce ASVs of each sample to 10000 (Valles-Colomer et al., 2022). ASV taxonomy was based on the Silva 138 SSURef NR99 16S rRNA gene reference database (Quast et al., 2013; Zhao et al., 2022). Analysis of gut microbiota profile Alpha diversity of gut microbiota was estimated by R package Vegan. And the beta diversity was estimated via vegdist function of Vegan R package based on the genus levels’ Bray-Curtis distance matrix (Lloyd-Price et al., 2019). The significant difference of the beta diversity was statistic by PERMANOVA test (adonis function). And the bacterial composition was analyzed by phyloseq R package (McMurdie and Holmes, 2013). Further, the LEfSe (Linear Discriminant Analysis Effect Size) analyze was performed via microbiomeMarker R package (Cao et al., 2022). Moreover, the predicted function of microbiota was an estimated by PICRUSt2 (Liu et al., 2016; Douglas et al., 2020). Analysis of serum metabolomics A difference in serum metabolite signatures was detected between normal and OS groups by LC-MS. For the identification of metabolites, processed data, such as m/z, RT, and normalized peak areas, were imported into SIMCA. Metabolites were identified using the HMDB (Human Metabolome Database). The HMDB database was adopted to map and identify the metabolites. By using R package ropls to identify the metabolites changes within groups, partial least squares discriminant analysis (PLS-DA) was used to analyze the abundance of significant metabolites with projection (VIP) > 1 and p value (Wilcoxon test) < 0.05. The enrichment pathway between normal and OS serum metabolites profiles was analyzed by MetaboAnalyst 5.0 (Pang et al., 2021). Correlation analysis Using Spearman’s correlation analysis, more than 20 variables including clinical variables and differences in microbiome were analyzed for the correlation between predicted function and changes in bacteria, microbiota, and metabolites. R package “psych”, version 2.4.1 was used in the correlation analysis. Marker panel for OS individuals In the discovery dataset, linear regression model was built based on the signatures of microbiota or metabolites of OS subjects (Liang et al., 2020). And the possibility of model was estimated by predict function in both discovery and validation sets. Besides, the accuracy of marker panel for discriminate the OS participants from normal cases in both discovery and validation sets by R package pROC (Robin et al., 2011). Co-occurrence network analysis The co-occurrence network of gut microbiota in normal and OS groups was established by R package ggClusterNet based on top 500 ASVs (Wen et al., 2022). And the network vulnerability was estimated via ggClusterNet R package. Besides, the average variation degree (AVD) of the network was analyzed as previous described (Xun et al., 2021). Statistical analyses All statistical analyses and graphical representations of the study were performed via R. Data were expressed in mean ± SEM unless otherwise stated. We utilized Wilcox and chi-squared test to evaluate the difference between two groups for continuous and categorical variables, respectively. Significant difference between two groups at a confidence level of 0.05.',\n",
       " '11188932': 'Materials and methods Study area and sample collection Water samples were collected from an aquaculture zone in Putian city, China, during different months: May to December 2019, January to December 2021, and January to May 2022. All culture ponds in this area shared the same seawater source, entering the system through an inlet (Sun et al. 2019). The samples were taken in the nearshore water area close to the culture zone, at a depth of 0.5 m, using a plexiglass water sampler. Specifically, 1000 ml of water samples underwent filtration with polycarbonate membranes featuring a pore size of 0.2 μm (EMD Millipore, USA). The filtration membranes were then placed in 1.5 ml sterile centrifuge tubes and stored in liquid nitrogen. Samples DNA extraction was performed using the PowerWater DNA Isolation Kit (MoBio, United States). The extraction process adhered to the DNA extraction protocol, conducted under sterile conditions to prevent contamination. Illumina MiSeq Sequencing of bacteria and eukaryote DNA from the above samples underwent amplification using specific primers: 515F (5’–GTG CCA GCM GCC GCGGTA A–3’) and 806R (5’–GGA CTA CHV GGG TWTCTA AT–3’) (Caporaso et al. 2011) for the 16S rRNA gene. For eukaryotes, the amplification of 18S rRNA was conducted with the specific V9 primers 1380F (5’-CCCTGCCHTTTGTACACAC-3’) and 1510R (5’-CCTTCYGCAGGTTCACCTAC-3’) (Amaral-Zettler et al. 2009). The polymerase chain reaction (PCR) used 15 µL of Phusion® High-Fidelity PCR Master Mix (New England Biolabs). PCR conditions were as follows: initial denaturation at 95 °C for 3 min, followed by 30 cycles of denaturation at 95 °C for 30 s, annealing at 55 °C (for bacteria) and 57 °C (for eukaryotes) for 30 s, extension at 72 °C for 45 s, and a final extension at 72 °C for 10 min. After purifying the PCR products, library preparation was carried out using the TruSeq® DNA PCR-Free Sample Preparation Kit (Illumina, USA). These libraries were then loaded onto an Illumina NovaSeq 6000 Sequencer (Illumina, USA) to generate the sequences. Sequences data analysis Paired-end reads obtained from Illumina sequencing underwent removal of barcodes and primers before assembly using FLASH (version 1.2.11). The resulting reads were filtered to remove low-quality reads and chimeric sequences using Chimera Check. High-quality reads were then clustered into OTUs with a sequence similarity of 97% using VSEARCH (version 1.9.6). Representative sequences of the OTUs were classified using the Ribosomal Database Program (RDP) classifier (version 2.2) with a confidence threshold of 0.8, based on the SILVA Database (version 132). The representative sequence was assigned by performing a BLAST search against the Silva_123 16S rRNA database and the V9_PR2 18S rRNA database. Based on taxonomic information derived from species annotations, potential pathogenic bacteria and red tide species associated with aquaculture were identified. In this study, the R project (version 3.3.1) was used to perform PCoA with unweighted unifrac distances. The resulting data were analyzed to identify significant differences in bacterial and eukaryotic communities between groups, using the Wilcoxon rank-sum test with a significance level of P < 0.05. Network visualizations depicting bacterial and eukaryotic interactions were generated using Cytoscape software (version 3.7.0). Bacterial genera with Spearman coefficient absolute values ≥ 0.7 and correction P values ≤ 0.05 were selected. Each node represents a genus, and its color represents a phylum. Edges color indicate significant correlations (negative and positive) between nodes.',\n",
       " '11188902': 'MethodsA review of medical records was completed using an electronic medical record. Inclusion criteria included males and females aged 18 to 95 with a diagnosis of POD, treated with sarecycline with a documented follow-up.',\n",
       " '11188898': 'Materials and methods Materials All chemicals were obtained from Sigma (St Louis, MO, USA) unless otherwise stated. 2’-fucosyllactose (2’FL) and 3-fucosyllactose (3FL) were obtained from Glycom/DSM (Esbjerg, Denmark). Blood group A type II (BgA), Blood group B type II (BgB), Blood group H type II (BgH) and LewisY (LeY) were obtained from Elicityl (Crolles, France). Lewis A trisaccharide (LeA), 3′-sialyl Lewis A (sLeA), Lewis X trisaccharide (LeX), 3’-sialyl Lewis X (sLeX), 2-acetamido-2-deoxy-6-O-(α-l-fucopyranosyl)-d-glucopyranose (6FN), 2-acetamido-2-deoxy-4-O-(α-l-fucopyranosyl)-d-glucopyranose (4FN), 2-acetamido-2-deoxy-3-O-(α-l-fucopyranosyl)-d-glucopyranose (3FN), 4-nitrophenyl α-l-fucopyranoside (pNP-Fuc), 2-Chloro-4-nitrophenyl-αl-fucopyranoside (CNP-Fuc), 2-Chloro-4-nitrophenol (CNP) and N-acetyllactosamine (LacNAc) were obtained from Biosynth Ltd (Compton, UK). FA2G2 N-glycan was from Ludger (Oxford, UK). IgG was purified from human serum using the protein A IgG purification kit from Thermofisher (Carlsbad, US). Purified porcine gastric mucin (pPGM) was obtained as previously described71. PNGase B035DRAFT_0334172 was a kind gift from Dr Lucy Crouch (Newcastle University). Phospholipase A2 (PLA2) from honeybee venom (Apis mellifera) was purchased from Sigma (St Louis, MO, USA). Recombinant fucosidases E1_101251B from R. gnavus E1 and ATCC_038333A from R. gnavus ATCC 29149 were produced in-house as previously reported11. Bioinformatics analyses For sequence similarity networks (SSN) analysis, the sequences encoding GH29 fucosidases were extracted from CAZy database (www.cazy.org). A total of 9,505 GH29 sequences from the CAZy database (last update 2022-10-18) were winnowed down to 2,971 sequences following a sequence identity cut-off at 0.8 via CD-HIT suite73. The amino acid sequences were then used to generate SSN using the Enzyme Function Initiative-Enzyme Similarity Tool (EFI-EST) with an alignment score threshold of 96 (40% sequence identity)33,74. The SSN was visualised using Cytoscape 3.9.1. The GH29 sequences from each cluster are provided in Supplementary Data 5. A semi-supervised training method, termed GH29BERT, was applied to implement the unsupervised protein sequence representation learning and supervised classification for GH29 fucosidases. This pLM training process is composed of two phases (as illustrated in Fig. S7), pre-training, and classification task-training, respectively. The pre-training utilised the BERT-based (Bidirectional Encoder Representations from Transformers) language model75 to extract features from 34,258 non-redundant GH29 sequences derived from the CAZy database (last update: 2023-10-10) and InterPro database (downloaded on 2023-11-02). A random 95%-5% data split was adopted for model pre-training and training-process validation. The original BERT model has 12 repeated blocks of Transformer encoders76, here we tuned this hyperparameter to 5 for best performance. Notably, the pre-training model, including a Masked Language Modelling (MLM) prediction head, enabled hiding a certain percentage of input tokens and training the model to predict them in a self-supervised approach. It implements both the next-token prediction and the previous-token prediction, facilitating bidirectional context understanding, which is critical for protein sequence modelling29. The classification task-training model, composed of two attention layers, three dense connected layers and one softmax classification head, was performed on 2,796 labelled sequences (see Supplementary Data 6) derived from the top 45 clusters of the SSN excluding 14 GH29 sequences which were further used for validation (see list of enzymes in Table 1). We randomly selected 80% of the labelled data for training and 20% for testing. Pre-training and task-training were executed on two NVIDIA A100 40GB GPU for one week and 2 h, respectively. In addition to evaluating the accuracy of cluster predictions, we incorporated Exponential Cross-Entropy (ECE) to assess the uncertainty of each pLM while processing input sequences. For the classification task in this study, ECE was calculated using \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{mathrsfs} \\\\usepackage{upgreek} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document}$${e}^{1/n{\\\\sum }_{i}^{n}{CE}({s}_{i},{y}_{i})}$$\\\\end{document}e1/n∑inCE(si,yi), where \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{mathrsfs} \\\\usepackage{upgreek} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document}$$n$$\\\\end{document}n is number of protein sequences tested, \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{mathrsfs} \\\\usepackage{upgreek} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document}$${s}_{i}$$\\\\end{document}si and \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{mathrsfs} \\\\usepackage{upgreek} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document}$${y}_{i}$$\\\\end{document}yi denote the sequences and their corresponding cluster labels, respectively. ECE is also known as one kind of perplexity, which in our context ranges from 1, indicating deterministic predictions, to 45, equivalent to a completely random selection from the 45 clusters. In addition to adopting the semi-supervised method through training the pLM on GH29 fucosidase sequences from scratch, i.e., with randomly initialised model parameters, we also included two state-of-the-art pLMs, ESM-228 and ProtT5 model29, for validating their efficacy on GH29 sequence cluster prediction. These pLMs are in larger scale than GH29BERT, in terms of training data, number of parameters, and training time, and were pre-trained on the entire known proteins (see detailed configuration comparison in Table S5). We loaded and froze their official pre-trained parameters, then used labelled GH29 sequences to fine-tune the task-training model, which had the identical structure as for GH29BERT. For comparison, we established two baselines using a non-pretrained GH29BERT and a one-hot encoding approach, respectively. Non-pretrained GH29BERT was trained directly with labelled GH29 sequences, while the one-hot method trained the same task-training model using a one-hot encoding of the protein sequence. Dimension reduction was performed using Uniform Manifold Approximation and Projection (UMAP)52. The GH29BERT model is accessible through a friendly user-interface: https://huggingface.co/spaces/Oiliver/GH29BERT. This web tool assigns a corresponding cluster ID to any sequence uploaded. The configuration of running environment, including all dependencies and used packages, as well as the Python version are detailed in supplementary data 7. The source code and instruction for running environment preparation are available on GitHub: https://github.com/ke-xing/GH29BERT. Cloning, expression and purification of fucosidases The GH29-encoding genes were synthesised exempt of the signal peptide sequence and cloned into pET28a with N terminal His6-tag by Prozomix (Haltwhistle, UK). BaGH2926AD218A and BaGH2926AD218N mutants were synthesised by NZYTech (Lisboa, Portugal). Escherichia coli TunerDE3 pLacI cells were transformed with the recombinant plasmids according to manufacturer’s instructions. Expression was carried out in 1 L LB media growing cells at 37°C until OD600 reached 0.3 to 0.6 and then induced at 16°C for 20-22 h. The cells were harvested by centrifugation at 4,000 g for 35 min. The His-tagged proteins were purified by immobilised metal affinity chromatography (IMAC) and further purified by gel filtration on an ÄKTApure (Cytiva, Little Chalfont, UK). Protein purification was assessed by standard SDS–polyacrylamide gel electrophoresis using the NuPAGE Novex 4–12% Bis-Tris gels (Life Technologies, Paisley, UK). Protein concentration was measured with a NanoDrop (Thermo Scientific, Wilmington, USA) and using the extinction coefficient calculated by Protparam77 from the peptide sequence. Enzymatic activity assays For kinetics, all enzymes were incubated with CNP-Fuc in 50 mM citrate buffer at pH 6 and 37°C. The amount of enzyme was determined to fulfil free-ligand approximation, i.e. the enzyme concentration was linear with product formation. The reaction duration was optimised to measure the reaction rates under initial conditions. A standard curve was made with the reaction product CNP and Fuc in 1:1 ratio from 0 to 0.3 mM to better mimic the reaction products. The release of CNP was monitored using a microplate reader (FLUOstar Omega, BMG LABTECH, Ortenberg, Germany) by monitoring absorbance at 405 nm every 2 min for 40 min in 3 technical replicates. The kinetic parameters were calculated based on the Michaelis-Menten equation using a non-linear regression analysis programme, and one-way ANOVA was performed compared to E1_101251B (Prism 5, GraphPad, San Diego, USA). The enzymatic activity of recombinant fucosidases was determined on 2’FL, 3FL, BgA, BgB, BgH, LeA, sLeA, LeX, sLeX, LeY, 6FN, pNP-Fuc and pPGM using 10 μM enzyme, 0.5 mM substrate or 1 mg/mL for pPGM in 50 mM citrate buffer pH 6 and 1 mg/mL bovine serum albumin (BSA). The reactions were incubated at 37°C and stopped by boiling at 95°C for 10 min. The release of Fuc was quantified with the l-fucose assay kit from Megazyme (Wicklow, Ireland) using a microplate reader (FLUOstar Omega, BMG LABTECH, Ortenberg, Germany) by monitoring absorbance at 340 nm every 2 min. To determine the specific activity, the enzymatic reactions were optimised by adjusting enzyme concentration and incubation time (Table S6) to obtain between 6%-25% of substrate hydrolysis which is within detection limit and corresponds to linear range. Specific activity was calculated from 4 technical replicates. One unit of activity was defined as the amount of enzyme needed to release 1 μmol Fuc per min under the conditions described above. Enzymatic reactions were carried out as above but with 0.1 mM substrate and were incubated for 24 h, and the released Fuc were confirmed by HPAEC-PAD using a Dionex ICS 5000 system (Thermo Scientific, Hemel Hempstead, UK). The sugars were separated on a CarboPac PA1 analytical column protected with a CarboPac PA1 guard column using the following gradient conditions at 1 mL/min: 0-20 min, 18 mM NaOH; 20.1-35 min, 100 mM NaOH; 35.1-50 min, 18 mM NaOH. Enzymatic reactions (20 μL) were also performed against complex glycans and glycoproteins using 10 μM of enzyme and 5 μM of oligosaccharides or FA2G2 (5 ng/μL)78, PLA2 (1 mg/mL), IgG (1 mg/mL) untreated or treated with PNGase B035DRAFT_03341 (10 μM)72 or PNGaseF (5000 units/mL), respectively in 50 mM citrate buffer at pH 6, 37°C for 24 h to release N-glycans. The products were analysed by LC-FD-MS/MS as previously described11. The reactions were stopped by heating 95 °C for 5 min and then dried down using Savant SpeedVac centrifugal evaporator (Thermo Fisher, Wilmington, USA), labelled at the reducing end with procainamide using the glycan labelling kit with sodium cyanoborohydride as the reductant (Ludger, Oxford, UK) and purified using a LudgerClean Procainamide Plate (LC-PROC-96, Ludger, Oxford, UK) to remove the excess dye. The samples were dried down using a Thermo Savant SpeedVac centrifugal evaporator and resuspended in 50 µL of 75% acetonitrile: 25% water. The suspensions were then injected onto a Waters BEH amide column (2.1 ×150 mm, 1.7 µm particle size, 130 Å pore size) at 40 °C on a Dionex Ultimate 3000 UHPLC instrument with a fluorescence detector (λex = 310 nm, λem = 370 nm) coupled to a Bruker Amazon Speed ETD. A 50 mM ammonium formate solution pH 4.4 (Ludger, Oxford, UK) was used as mobile phase A and acetonitrile (Romil, UK) was used as mobile phase B. A 70 min gradient was used at 0.2 mL/min unless otherwise specified, 0-53.5 min, 76% to 51%B, 0.4 mL/min; 53.5-55.5 min, 51% to 0% B; 55.5-57.5 min, 0% B; 57.5-59.5 min, 0% to 76% B; 59.5-65.5 min, 76% B; 65.5-70 min, 76% B, 0.4 mL/min. The Heatmap of enzyme specific activities was constructed via Chiplot (https://www.chiplot.online/). Hierarchical clustering was performed based on Euclidean distance calculated with complete linkage as computing method. Transfucosylation reactions For transfucosylation, enzymatic reactions with 1 μM enzyme (1.43 μM for BaGH2926A), 180 mM GlcNAc and 18 mM pNP-Fuc were incubated in 20% (v/v) DMSO to increase the solubility of pNP-Fuc for 1 h at 37°C. The reactions were stopped by addition of ethanol using three times the volume of the reaction. To assay the capacity of the enzymes to carry out further transfucosylation reactions, 1 μM enzyme (1.43 μM was used for BaGH2926A) was incubated with 180 mM 3FN or 6FN and 18 mM pNP-Fuc in 20% (v/v) DMSO. Thin layer chromatography (TLC) and TLC-electrospray ionisation-mass spectrometry (TLC-ESI-MS) analysis To analyse the products of transfucosylation reactions, standards, Fuc (0.01 µmol), pNP-Fuc (0.03 µmol), GlcNAc (0.25 µmol), 4FN (0.005 µmol), 3FN (0.005 µmol), 6FN (0.005 µmol), and reaction samples with GlcNAc (8 ×0.5 µL) or with 3FN or 6FN (4 ×0.5 µL) were loaded on a 12 cm tall plate (TLC Silica gel 60 F254, Sigma-Aldrich, Germany). The plates were developed using an isopropanol-ammonium hydroxide-water 6:3:1 mixture (namely IPA-NH4OH-H2O) for 3 h or until the frontline of the solvent rose to ca. 11.25 cm. The plate was then dried using a hair dryer and stained using a 5% ethanolic solution of sulphuric acid. Gently heating of the plate allowed the identification of the TLC spots corresponding to controls and reaction products. TLC-ESI-MS of the enzymatic reactions was performed using an Expression Compact Mass Spectrometer (Advion, UK) coupled with a Plate Express reader (Advion, UK) in positive mode to identify the fucosylated reaction products. The enzymatic reactions were analysed through TLC as described above. The analysis was performed in duplicates to stain one TLC plate and use it as a guide to perform the TLC-ESI-MS on the non-stained plate. By comparison with the stained plate, the laser of the Plate Express reader was aimed at the right retention factor (Rf) and the data obtained was analysed using Advion Mass Express software. Comparison of retention factors, in combination with TLC-ESI-MS analysis and NMR allowed identification of reaction products. Nuclear Magnetic Resonance (NMR) spectroscopy An aliquot of the enzymatic reactions (600 μL) was evaporated to dryness and reconstituted in 600 μL of NMR buffer (100 mL D2O containing 0.26 g NaH2PO4, 1.41 g K2HPO4, and 1 mM deuterated trimethylsilyl propionate (TSP) as a reference compound) before 1H-NMR spectroscopic analysis. 1H-NMR spectra were recorded using a 600-MHz Bruker Avance spectrometer fitted with a 5-mm TCI proton-optimised triple resonance NMR inverse cryoprobe and autosampler (Bruker, Bremen, Germany). Sample temperature was controlled at 300 K. Spectra were acquired with 32 scans, a spectral width of 12500 Hz and an acquisition time of 2.6 s. The “noesypr1d” presaturation sequence was used to suppress the residual water signal with a low-power selective irradiation at the water frequency during the recycle delay. Spectra were then transformed with a 0.3-Hz line broadening and zero filling, manually phased, baseline corrected, and referenced by setting the TSP–d4 signal to 0 ppm. Reaction products were identified by comparison with the spectra of standards (GlcNAc, pNP-Fuc, Fucose, 6FN, 3FN and 4FN). STD NMR All NMR binding experiments were performed at 278 K on a Bruker Avance III 800 MHz spectrometer equipped with a 5-mm TXI 800 MHz H-C/N-D-05 Z BTO probe (Bruker, Bremen, Germany). First, FA2G2 was spectroscopically characterised by standard COSY (cosydfesgpph), TOCSY (mlvevphpp), 1H-13C HSQC (hsqctgpsp) and NOESY (noesygpph) for the purpose of assignment. Then, FA2G2 was recovered and prepared in a Shigemi advanced NMR microtube assembly at the concentration of ~200 μM in the presence of ~20 uM BaGH2926A (protein:ligand ratio 1:20), in D2O buffer solution containing 25 mM Tris-d11 pH 7.8 and 100 mM NaCl. An STD NMR pulse sequence including 2.5 ms and 5 ms trim pulses and a 3 ms spoil gradient was used. Saturation was achieved by applying a train of 50 ms Gaussian pulses (0.40 mW) on the f2 channel, at 6.70 ppm (on-resonance experiments) and 40 ppm (off-resonance experiments). The broad protein signals were removed using a 40 ms spinlock (T1ρ) filter. As a first test for binding, an STD NMR experiment with a saturation time of 2 s and a relaxation delay of 5 s was performed. Then, an STD build up curve was performed, by carrying out STD experiments at different saturation times (0.5, 1, 2, 3, 4 and 5 s) with 2 K scans, in order to obtain the binding epitope mapping. The resulting build-up curves for each proton were fitted mathematically to a mono-exponential equation (y = a*[1-exp(b*x)]), from which the initial slopes (a*b) were obtained. Finally, the binding epitope mapping was obtained by dividing the initial slopes by the strongest signal corresponding to the methyl group of GlcNAc A, to which an arbitrary value of 100% was assigned. X-ray crystallography BaGH2926A was dialysed into 20 mM Tris 150 mM NaCl. Sitting drop vapour diffusion plates were set up with a protein concentration of 20 mg/mL and 5 mM 2’FL. Crystals appeared in many conditions across commercial sparse matrix screens with the best diffracting crystals appearing in the following condition: 0.12 M diethylene glycol, 0.12 M triethylene glycol, 0.12 M tetraethylene glycol, 0.12 M pentaethylene glycol, 100 mM Tris(base)/bicine pH 8.5, 12.5% 2-methyl-2,4-pentanediol, 12.5% PEG 1000, 12.5% w/v PEG 3350. Diffraction datasets were collected at Diamond Light Source on beamline I24 at a wavelength of 0.9686 Å. Attempts were made to soak out the resulting Fuc molecule bound in the active site so that additional complexes could be attained. These attempts were unsuccessful. By seeding with WT BaGH2926A crystals we also grew diffracting crystals of BaGH2926AD218N active site mutant. These crystals were grown in 0.1 M sodium acetate pH 4.6, 8% (w/v) PEG 4000. Data were processed using xia279 and dials80. The phase problem was solved by molecular replacement using the search model 1ODU, prepared using Chainsaw81. Initial model building was performed using ArpWarp82, followed by alternating cycles of model building and refinement using coot83, refmac84, and PDBredo85. The refined WT BaGH2926A structure has 0.3% ramachandran outliers. The final D218N BaGH2926A structure has 0.05% ramachandran outliers. Reporting summary Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article.',\n",
       " '11188897': 'Methods The FORISCA study This study was approved by the National Ethic Committee for Health Research (NECHR) of the Ministry of Health, Phnom Penh, Cambodia, the Ministry of Education, Youth and Sports, Phnom Penh, Cambodia, and the Research Ethics Committee of PATH, Seattle, USA. Written informed consent from participants has been obtained. The trial was registered at ClinicalTrials. gov (Identifier NCT0170641 [I https://classic.clinicaltrials.gov/ct2/show/record/NCT01706419?term=NCT01706419&draw=2&rank=1&view=record]). Written informed consent was obtained from their parents or caretakers, and verbal assent was obtained from the participating children prior to enrolment. This present study focusses on a subset of 380 children who participated in the FORISCA project, which was a large, double-blind, cluster randomized, placebo-controlled trial on the impact of fortified rice on the health and cognitive performance of 9500 schoolchildren. The study was conducted in Cambodia in November 2012 and July 2013. A random selection was performed of children for whom all data were available at both points where stool samples had been collected for assessment of parasitic infection and gut inflammation status and who had not taken any antibiotics for the past three months. Exclusion criteria included being less than 6 years old or more than 14 years old, mental or physical disabilities, or severe anaemia (defined as haemoglobin concentration < 70 g/L). There was no participant compensation but children diagnosed with severe anaemia were provided with multiple micronutrient supplements for two months. Six schools in Kampong Speu province that were participating in the United Nations World Food Programme school meal programme were randomly selected for the present trial and placed in three intervention groups named UltraRice Original formulation (UR-original), UltraRice Improved formulation (UR-improved) and Normal rice (Placebo) (Supplementary Table 2)13. Anthropometric measurements were taken, and blood and faecal samples were collected at baseline and at the end of the 6-month intervention. Stool samples were collected by providing school children with a sample collection container and requesting them to return the next day to school with a fresh stool sample. Samples were transported in cool boxes containing cool packs (4 °C) to the capital Phnom Penh within 4 h of collection, and further prepared and stored at −20 °C until analysis in Phnom Penh, or transported on dry ice to site of the analysis. After collection of baseline data, all the children received a single 500 mg dose of mebendazole for deworming. Anthropometry, blood collection and laboratory analysis The weight and height of the children were measured without footwear and wearing minimum clothing using standard procedures. Weight was measured once to the nearest 100 g (model 881U scale; Seca, Hamburg, Germany). The accuracy of the scales was checked every day using a set of two calibration weights. Height was measured twice to the nearest 0.1 cm on a wooden stadiometer (UNICEF-Cambodia) and mean values were used. When differences between two measurements of height in the same child exceeded 0.5 cm, the measurements were repeated. Height-for-age Z-score (HAZ) and BMI-for-age Z-score (BAZ) were calculated according to the WHO 2006 reference49. Underweight and stunting were defined as BAZ < − 2 and HAZ < − 2, respectively. Non-fasting blood samples were taken from the antecubital vein in the morning following a standard protocol. Blood (5 mL) was stored in trace-element free vacutainers with no anticoagulant (Vacuette, Greiner Bio One, Austria) at a temperature of < 5 °C. The blood samples were centrifuged (1,300 g for 10 min) and serum samples were aliquoted and stored at −25 °C until analysis. Iron status (ferritin and soluble transferrin receptor), vitamin A status (retinol-binding protein) and biomarkers of inflammation (C-reactive protein (CRP) and α 1-acid-glycoprotein (AGP)) were determined at VitMin laboratory (Willstaett, Germany). All these proteins were measured using a sandwich enzyme-linked immunosorbent assay (ELISA) technique50. Zinc was measured at the National Institute of Nutrition (Hanoi, Vietnam) using a flame atomic absorption spectrophotometer (GBC, Avanta + ) using trace element-free procedures12. Faecal calprotectin was measured (Calpro AS, Norway) to estimate the gut inflammation17. Faecal parasites were analysed as previously described using the Kato-Katz technique to determine helminth infection by the National Center for Parasitology, Entomology and Malaria control (CNM), Phnom Penh, Cambodia, and recorded as the number of eggs per gram of faeces16. Logistic mixed models were performed to assess the impact of the intervention on child growth and micronutrient status using R 4.1.0. Fixed effects were the intervention group, the time of measurement, their interaction, the age group and the sex. Random terms for child ID and schools were used in the models. The interaction “time of measurement × group of intervention” estimated the impact of the intervention on child growth and micronutrient status. For mixed models, results are expressed as odds ratios (OR) for the interaction “time of measurement × group of intervention” with 95% confidence intervals (CI) and corresponding p-values. Taxonomic profiling of the gut microbiota Faecal samples for taxonomic profiling of the gut microbiota were collected from the three groups at baseline and endline. Faecal samples were stored at −80 °C until DNA extraction. DNA was extracted using the QIAamp DNA Stool Mini kit (ID: 51604, Qiagen, Les Ulis, France) according to the manufacturer’s protocol51. The V3-V5 hypervariable region of bacterial 16SrRNA was sequenced in paired-end mode (2×300 bp) on the MiSeq platform (Illumina, performed by the Research and Testing Laboratory in Lubbock, Texas, US) using primers 357 F (5’-CCTACGGGAGGCAGCAG-3’) and 926 R (5’-CCGTCAATTCMTTTRAGT-3’)52. Different intervention groups as well as time points (baseline and endline) were randomized, pooled in four batches of 190 samples and analysed into four runs to ensure that batches did not contain either all baseline or all intervention group samples. A no template control was included at the PCR step and sequenced along with the samples. The 16 S rRNA gene sequencing data generated in this study have been deposited in the National Centre for Biotechnology Information database under accession number PRJNA882252. Demultiplexed sequences were obtained and the Divisive Amplicon Denoising Algorithm2 (DADA2) R package was used to remove low quality reads, to de-noise and remove chimeric sequences, and the data were then sorted into unique amplicon sequence variants (ASVs)53. Due to the small overlap between forward and reverse reads following DADA2 trimming and quality filtering, only forward reads were used for analysis. The outputs from DADA2 (ASVs) were assigned to taxonomies using the Ribosomal Database Project (RDP) Naïve Bayesian classifier54 using the DADA2 formatted training sets for SILVA 16 S rRNA database version 138.1 (https://benjjneb.github.io/dada2/training.html)53–55. Chloroplast and mitochondria sequences were removed. Sequences were rarefied at 3500 reads per sample to determine bacterial alpha and beta-diversity using QIIME2 (2021.11)56. The core microbiota, represented by taxa occurring in at least 95% of the subjects who participated in the study, was also assessed at the genus level using the microbiome R package57,58. Differences in alpha diversity were statistically assessed using the Kruskal-Wallis test, with significance set at P < 0.05. Additionally, a linear mixed model incorporating covariates such as age and sex was used to further validate the robustness of significant features. Significance was set at P < 0.05. Beta-diversity group significance was assessed using permutational analysis of variance (PERMANOVA) with 999 permutations. Bray-Curtis at the baseline and endline was also compared with PERMANOVA with 999 permutations while accounting for the stratification by subject using adonis2 function from the vegan R package59. Both tests were used on ASVs levels at the baseline for all variables and at endline to explore the effect of intervention on alpha and beta diversity. Additionally, ASVs with a prevalence of < 5% were removed from taxonomic analyses and linear discriminant analysis effect size (LEfSe) was applied (Wilcoxon P-value < 0.05, logarithmic LDA (linear discriminant analysis) score > 2) to identify the biomarkers in different groups of metadata60. Following the LEfSe analysis, taxa showing significant differences were further tested in a linear mixed model with log-transformed values adjusted for covariates such as age and sex. P-values were corrected for multiple comparison using the Benjamini-Hochberg procedure (q-value < 0.1). The effect of multiple-micronutrient fortified rice (UR-original and improved) on gut microbiota composition was investigated using the q2-longitudinal plugin to compare baseline and endline data in terms of alpha and beta diversity group significance using pairwise difference and pairwise distance tests, respectively61. We conducted a multivariate association analysis to assess the impact of the intervention on microbiome composition at the genus level. In this analysis, a linear mixed model effect analysis was performed using the MaAsLin2 R package62. Only bacterial genera present in at least 10% of the participants in the study were included in the analysis. Subsequently, the bacterial genera underwent total sum scale (TSS) transformation without normalization for the analysis. The intervention group and time (baseline and endline) were used as a fixed effect while subjects were used as a random effect. A q-value threshold of 0.25 was applied to determine statistical significance. Data supporting the findings of this study and script used for data analysis are accessible at https://github.com/yoh-s/FORISCA.git. Functional potential of the 16 S rRNA-predicted taxa identified in the samples Phylogenetic Investigation of Communities by Reconstruction of Unobserved States 2 (PICRUSt2) software was used with default parameters to predict the functional potential of the 16 S rRNA22. In PICRUSt2, sequence variants are placed in a phylogenetic tree using Massively Parallel Evolutionary Placement of Genetic Sequences (EPA-NG)63 and processing, analysing and visualising phylogenetic (placement) data (gappa)64. Genomic hidden states were predicted using efficient comparative phylogenetics on large trees (castor)65, and MetaCyc pathway abundances66 were inferred using MinPath67. Statistical analysis of metagenomic profiles (STAMP v2.1.3) was used to test group significance between-groups. Welch’s t-test was performed with Benjamini-Hochberg (BH) False Discovery Rate correction. Adjusted p-value (BH) less than 0.05 were considered statistically significant. Statistics & reproducibility The sample size of FORISCA study was calculated based on the intention to detect changes in anaemia prevalence, haemoglobin concentration and a number of micronutrient statuses. The sample size of 500 children per intervention group was decided on the ability to detect a difference in haemoglobin concentration between placebo and intervention groups of at least 3 g/L, assuming a power of 0.90, a statistical significance of 0.05 and a SD of 20 g/L. Other outcomes all needed smaller sample sizes12. The randomization, blinding and subsample general information are described in the paragraph “The FORISCA study”. No data were excluded but some data are not available for all children due to obtaining not enough sample. The number of schoolchildren included for each analyse is presented in the figures. Statistical analyses of the effect of the intervention on child growth and micronutrient status is described in details in the paragraph “Anthropometry, blood collection and laboratory analysis”. Statistical analysis for the link between faecal microbiota, nutritional, micronutrient, inflammatory and anthropometrical status as well as the effect of intervention on the composition of the faecal microbiota are detailed in the paragraph “Taxonomic profiling of the gut microbiota”. Statistics related to the evaluation of the functional potential of the faecal microbiota and its relation with micronutrient status and the effect of the intervention are described in the paragraph “Functional potential of the 16 S rRNA-predicted taxa identified in the samples”. Reporting summary Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article.',\n",
       " '11188892': 'Materials and methods Participants All the participants or their legal guardians provided written informed consent. The Ethics Committee of the Affiliated Wuxi People’s Hospital of Nanjing Medical University approved the current study (approval number: KY21088). All clinical investigations were conducted in strict adherence to the principles outlined in the Declaration of Helsinki, and all experiments were performed in accordance with relevant guidelines and regulations. A total of 63 LVO patients were recruited from the stroke center of the Affiliated Wuxi People’s Hospital of Nanjing Medical University, and 36 matching NCs were recruited through community health screening in Wuxi City. Moreover, 64 CSVD patients from the Affiliated Wuxi People’s Hospital of Nanjing Medical University, who had been reported in our previous studies, also were included in the present study. The standardized clinical interview was performed for all the participants, involving demographic inventory as well as physical and mental health examination. The brain magnetic resonance imaging (MRI) along with three-dimensional T1-weighted, T2-weighted, fluid-attenuated inversion recovery, susceptibility-weighted images, diffusion-weighted imaging, and magnetic resonance angiography (MRA), were also conducted for each participant. The dietary structure of each participant is similar. The inclusion criteria for all the participants were as follows: (1) > 18 years old and (2) no contraindication in MRI scan. The LVO patients showed first-ever stroke symptoms, and the time of stroke was less than 4.5 h. All LVO patients were anterior circulation single LVO on computed tomography angiography or MRA. The Pullicino formula was used to calculate the BIV of LVO patients based on the cranial MRI scan. Furthermore, the CSVD patients had periventricular and deep white matter hyperintensity, lacunar infarcts, and cerebral microbleeds but not any new brain infarcts. All the NCs did not have any stroke, which was reflected by consistent imaging results. Participants in the NC group were randomly recruited during the same period. The NC group matched with the LVO and CSVD groups for age, sex, BMI, and complications. Meanwhile, all the participants had no habit of taking antiplatelet drugs or statins prior to enrollment. The exclusion criteria for each participant were as follows: (1) hemorrhagic disorder or bleeding tendency (e.g., cerebral hemorrhage and gastrointestinal hemorrhage); (2) any severe psychiatric disorders (e.g., schizophrenia and major depressive disorder); (3) any infectious disease (e.g., pulmonary infection); (4) brain trauma or other neurologic diseases (e.g., Parkinson’s disease and Alzheimer’s disease); (5) any significant medical problems (e.g., tumor, significantly impaired liver or kidney functions, digestive system disease, and metabolic disease); (6) record of antibiotic use in the last six months or during treatment; and (7) usage of prebiotics or probiotics within one month before admission or during treatment. At 24 h after a thrombolytic therapy with alteplase, the LVO patients were arranged for further drug therapies. According to “Chinese Stroke Association Guidelines for Clinical Management of Cerebrovascular Disorders (2019)”39, all the ischemic stroke patients with LVO received dual antiplatelet aggregation therapy, lipid-lowering treatment, and improving cerebral microcirculation. The duration of treatment was 8 ± 1 days for each LVO patient. Only those LVO patients, who met the current treatment protocol, were included in this study. Meanwhile, each LVO patient obtained a standard Mediterranean diet during their duration of hospital stay without the use of a nasogastric tube. Notably, the present study was not a clinical trial but an observational study. Furthermore, NIHSS40 scale was used to evaluate the symptoms of stroke, and the mRS41 scale was used as a measure of post-stroke physical disability. These two assessments were conducted at the time of admission and discharge of the patients. Collection of fecal samples and 16S rRNA gene sequencing After thrombolytic therapy (within 24 h), the fecal sample was collected after LVO participant’s defecation using stool collection tubes with stool DNA stabilizer (Genstone Biotech, Beijing, China), and then stored at − 80 °C. Furthermore, fecal samples were collected at 7:00–8:00 a.m. for other participants. FastDNA Spin Kit For Soil (MP Biomedicals, Santa Ana, CA) was used for the DNA extractions of fecal samples. Subsequently, the compositional analysis of gut microbiota were conducted by Genesky Biotechnologies Inc. (Shanghai, China). Details of sequencing and data analysis could be found in Supplementary Materials and previous studies42,43. Collection of serum samples and detection of blood indexes For LVO patients, peripheral venous blood was collected using a vacutainer tube (without anticoagulant) and an EDTA-coated tube at 6:00–7:00 a.m. on the first day after admission. The serum samples were further obtained by centrifugation at 3500 rpm at 4 °C for 10 min. All blood samples were measured immediately after collection. Due to the main goal of the present study is to investigate specific gut microbiota of LVO and potential association of gut microbiota with the pathologic change in LVO stroke, blood samples were collected only in the LVO group. The total cholesterol, TG, low-density lipoprotein cholesterol, high-density lipoprotein cholesterol, FBG and uric acid were detected in serum using a fully automatic biochemical analyzer (UniCel DxC 600 Synchron, Beckman Coulter, California, United States). Hemoglobin A1c (HbA1c) level was assayed in whole-blood samples and HCY level was assayed in serum samples using the high-performance liquid chromatography (Agilent 1200 device, Agilent Technologies, Waldbronn, Germany). Statistical analyses The data analyses were conducted using SPSS version 23.0 (SPSS Inc. Chicago, IL, USA) and R software package (version 4.2.1). Alpha and beta diversity analyses were performed for the diversity analysis of gut microbiota. In alpha-diversity analysis, Observed species, Chao1, and ACE indices were used for assessing the community richness, and Shannon, Simpson, and Coverage indices were used for assessing the community diversity. Beta diversity was used to analyze differences in the composition of gut microbiota using PCoA and PLS-DA. LEfSe was performed to identify the markers to interpret the difference among groups where the threshold score of LDA was 2. The functions of species in the gut microbiota were predicted using PICRUSt2 analysis tool and Kyoto Encyclopedia of Genes and Genomes database (https://www.genome.jp/kegg/pathway.html). The continuous variables were shown as mean ± standard deviation and were analyzed using the Kruskal–Wallis H test for non-normal distribution, or the One-way ANOVA test for normal distribution. The categorical variables were analyzed using the Chi-square test. Partial correlation analysis was used to determine the correlation between the taxonomies of gut microbiota and clinical data in LVO patients, with adjusting age, sex, BMI, and complications. Furthermore, the mediation analysis in LVO patients could determine whether environmental factors mediated the association between the relative abundance of gut microbiota on the severity of stroke based on a standard three-variable mediation model44,45. Line regression was used to evaluate the interaction between two variables on the functional outcome46,47. Changed value = admission score − discharge score. The statistically significant differences were considered as p-value < 0.05. Institutional review board statement The Ethics Committee of the Affiliated Wuxi People’s Hospital of Nanjing Medical University approved the current study (approval number: KY21088). Informed consent A signed written informed consent form and acceptance to participate in the study were received from all participants in the study.',\n",
       " '11188809': 'Materials and methods Animal grouping and intervention The treatment of experimental animals is in accordance with Chinese ethical requirements (Ethics No. :SYDL20200011). Male C57BL/6 mice between the ages of 4 and 5 weeks were obtained from the Laboratory Animal Center of Shanxi Medical University and maintained under specific pathogen free (SPF) environment with a normal diet (20.3% kcal from protein, 15.8% kcal from fat, and 63.9% kcal from carbohydrates). The mice were arbitrarily assigned into three groups (n = 6 for each group) as follows: (1) Control group, (2) CIH group, and (3) CIH group treated with CB1 receptor antagonist (AM251 group). Specifically, mice were subjected to intraperitoneal injection of the CB1 receptor antagonist AM251(1 mg/kg) before entering the intermittent hypoxic chamber. AM251 is a selective antagonist of the CB1 receptor, which has been widely used in various experiments. The inhibitory effect of AM251 on the CB1 receptor is significantly reliable (Sugawara et al. 2013; Caltana et al. 2015; Higginbotham et al. 2021). CIH-exposed mice were put into the intermittent hypoxic chamber at 9:00 am every day until 5:00 pm. The mice were kept under either a normal air or CIH environment for 6 weeks before the following experiments. Chronic intermittent hypoxia mouse model CIH-exposed mice were placed in the OxyCycler Model A84 hypoxic chamber (Biospherix, Parish, NY, USA), which was set up in a mode of alternating 9–21% oxygen concentration and filled with high-purity compressed nitrogen and high-purity compressed oxygen for 120s per cycle. In the experimental setup, nitrogen was injected to reduce the oxygen concentration from 21% to 9% within 50s, and the hypoxia condition was maintained for 10s. Following this, high-purity oxygen is injected for 50s, rapidly raising the oxygen concentration back to 21%, which was then sustained for an additional 10s. The hypoxia-reoxygenation cycle was repeated 30 times every hour, with the lowest oxygen concentration being ± 9%, as previously described (Dou et al. 2020; Zhao et al. 2021). This protocol was carried out continuously for 8 h each day, from 9:00 am to 5:00 pm. Antibodies CB1 receptor antagonist AM251 was purchased from Topscience (Shanghai, China, No: T1915), and rabbit anti-mouse Occludin (GB111402) and ZO-1 (GB11149) antibodies were purchased from Servicebio (Wuhan, China); rabbit anti-mouse CB1R antibody was purchased from Thermo Fisher (Waltham, MA, USA). Goat anti-rabbit second antibody was purchased from Wuhan Sanying (Wuhan, China). Hematoxylin-eosin (HE) staining Colon tissues of mice were collected and fixed using 4% paraformaldehyde solution. Paraffin embedding of tissue was performed following fixation, then tissues were sectioned (5 µm). The sections were then dewaxed with conventional xylene, dehydrated with a gradient concentration of alcohol, and rinsed with PBS. HE staining was performed for 5 min, followed by dehydration using conventional gradient alcohol. The sections then underwent transparent process using xylene 2 times for 5 min each. Finally, the sections were sealed with neutral gum. An optical microscope was used to view the pathological changes in the colon tissue. Transmission electron microscopy (TEM) Fresh colon tissues were trimmed into the size of 1 mm3 following collection. The tissues were then immediately placed in a centrifuge tube filled with 2.5% glutaraldehyde electron microscope fixative solution and stored at 4 ℃ for preservation. Subsequently, for 2h, the tissues were fixed in 1% osmic acid at room temperature. Dehydration was performed at room temperature, followed by permeation and polymerization. Ultrathin sections were prepared and stained accordingly. The ultrastructural changes within the colon tissue were studied through a transmission electron microscope. Images were captured and subsequently analyzed. Immunofluorescence staining Paraffinized sections of the colon were dewaxed through rehydration, antigen was retrieved using the microwave oven, sealed with 3% bovine serum albumin (BSA) for 30 min, followed by drops of primary antibody CB1R (1:500, Proteintech, Wuhan, China), incubated overnight at 4 ℃, followed by drops of 100 μl anti-rabbit IgG labeled fluorescent antibody, incubated for 50 min at room temperature, nucleated with 4′,6-diamidino-2-phenylindole (DAPI) working solution at 37 ℃ for 10 min, thereafter sealed. Under a laser confocal microscope, the positive signal was red fluorescence and the blue was nuclear staining signal. The average fluorescence intensity of positive expression was calculated. 16S rRNA gene amplification and sequencing At the end of the study (week 6), fecal samples were collected. Each group contains six mice. DNA of fecal samples was extracted and PCR amplification was performed using primers bound to the 16S rRNA highly variable region V3-V4. The community DNA fragments were paired with paired-end sequencing using an Illumina platform (Illumina, San Diego, CA, USA). The raw sequencing data was processed and analyzed on the QIIME2 analysis platform (Sun et al. 2023). Fecal SCFA analysis Mouse feces were collected and stored at −80 ℃. The short-chain fatty acid (SCFA) test includes acetic acid, butyric acid, isovaleric acid, valeric acid, caproic acid, propionic acid, and isobutyric acid. A gas phase mass spectrometer was used for the detection. Conditions for chromatography were as follows: Thermo Trace 1310 gas phase system (Thermo Fisher Scientific, Waltham, MA, USA), Agilent HP-INNOWAX capillary column (Agilent, Santa Clara, CA, USA), shitter injection, injection volume 1 μL, shitter ratio 10:1, inlet temperature 250 ℃, ion source temperature 300 ℃, transmission line temperature 250 ℃. The starting temperature of programmed heating is 90 ℃. Then it was heated at 10 ℃/min to 120 ℃, then at 5 ℃/min to 150 ℃, then lastly at 25 ℃/min to 250 ℃ for 2 min. The carrier gas was helium, and the carrier gas flow rate was 1.0 mL/min. Mass spectrum conditions were as follows: Thermo ISQ LT Mass spectrometer (Thermo Fisher Scientific, Waltham, MA, USA), electron bombardment ionization (EI) source, selected ion monitor (SIM) scanning mode electron energy 70 eV. Western blotting The protein from colon tissue was isolated and the bicinchoninic acid assay (BCA) was run to determine the concentration of the protein. The protein samples were denatured by water bath at 100 ℃ for 10 min prior to sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE). The membrane was transferred to a polyvinylidene fluoride (PVDF) membrane by electric transfer, blocked with skim milk, followed by overnight incubation with the preceding primary antibodies at 4 °C: Rabbit anti-ZO-1 (1:2000, Servicebio, Wuhan, China), rabbit anti-Occludin (1:3000, Servicebio, Wuhan, China), and rabbit anti-CB1 receptor (1:3000, Thermo Fisher, Waltham, MA, USA). Thereafter, membranes were washed with 0.05% Tween-20 (TBST) three times (5 min each). The membranes were then incubated with secondary antibodies at room temperature (Proteintech, Wuhan, China) for 2 h, followed by TBST rinses three times (5 min each). Development was performed using an enhanced chemical luminizer (Millipore, Boston, MA, USA) and the signal was visualized in a comprehensive gel imaging system (Bio-Rad, Hercules, CA, USA). β-Actin was used as an internal reference protein. Real-time polymerase chain reaction (RT-PCR) Total RNA was isolated from frozen colonic tissue by the TRIzol method and reverse transcribed into cDNA (Kidd et al. 2005). Using the gene for GAPDH as an endogenous control, 2-ΔΔCt was used to calculate the relative expression level of the target gene after the reaction. All primers used are listed in Table 1.Table 1The list of primers sequencesGeneForwardReverseGAPDHGGTTGTCTCCTGCGACTTCATGGTCCAGGGTTTCTTACTCCCB1RTCGACAGGTACATATCCATTCACAGAGAGGCAACACAGCGATTACTATTZO-1GGAAACCCGAAACTGATGCTATGGAACTGGCTGGCTGTACTGTGAGOccludinTCTCAGCCGGCATACTCTTTATAGGCTCTGTCCCAAGCAAIL-1βAGCTTCCTTGTGCAAGTGTCTGGACCACTCTCCAGTACCCACTTNF-αCAGGCGGTGCCTATGTCTCCGATCACCCCGAAGTTCAGTAGIL-10CCTCTGGATACAGCTGCGACTAGACACCTTTGTCTTGGAGCTA Enzyme-linked immunosorbent assay (ELISA) Mice whole blood samples were obtained and centrifuged for 2 h at 3000 rpm in a 4 °C low-temperature centrifuge for 15 min. Serum (supernatant) was collected to measure the concentration of LPS according to the kit instructions (Jingmei, Yancheng, China). Statistical analysis Data analysis program utilized was SPSS 26.0 (SPSS, Chicago, IL, USA), and GraphPad Prism 8 (GraphPad, San Diego, CA, USA) was used for plotting. One-way ANOVA was used for samples with normal distribution, LSD test was employed for comparison when variance within groups was homogeneous, and when variance was uneven, Dunnett T3 test was utilized. Samples that did not satisfy the normal distribution were compared between groups using Kruskal-Wallis non-parametric tests. P < 0.05 was judged as statistically significant.',\n",
       " '11188792': 'Materials and methods Culture conditions, media and flask experiments Four individual fungal (Basidiomycetes) strains were purchased from Mycelia®, Deinze, Belgium; they were obtained as slanted media of malt agar with an actively growing mycelia covering the surface. Those strains were as follows: T. versicolor (M9911), L. edodes (M3102), H. erinaceus (M9514) and A. blazei (M7700). A. bisporus (substrain White, ABS14) was obtained from Commercial Mushroom Producers, Monaghan, Ireland. The strains were grown on malt agar plates according to the specifications provided by Mycelia® Deinze, Belgium, and were incubated at 28 °C. The starting flasks cultures were prepared as follows: five individual agar plugs covered with an actively growing mycelia were taken from each plate aseptically. The same were transferred into a 100-mL flask, provided with a stainless-steel spring to facilitate mycelia attachment. The medium composition of the flask was the following: 3 g/L of yeast extract, 5 g/L of soy peptone, 3 g/L malt extract and 10 g/L of glucose; all the reagents were of analytical grade, and they were purchased from Sigma-Aldrich, Dublin, Ireland. Each flask final volume was 100 mL, and they were autoclaved at 121 °C for 15 min before being inoculated. The medium was called YSMG. After a minimal period of 5 days of growth and with a dense attached mycelial aggregate on the stainless-steel spring, the starter culture was used to inoculate the experimental cultures, where the same or a different range of media was used. In order to facilitate a dense and homogenous pelletised fungal population, an IKA® T18 UltraTurrax® submergible homogeniser (Staufen, Germany) provided with a S18 N 10G dispersing tool, able to reach a maximum speed of 25,000 revolutions per minute (RPM), was used. The mycelial aggregate, present in the starting flask, was homogenised for 30 s in aseptic conditions inside a SafeFast laminar cabinet (produced by Faster S.r.l, Ferrara, Italy) by the submergible homogeniser. The flask was inclined at an angle of approximately 45° for the duration of the homogenisation. The experimental flasks cultures and the inoculation cultures for the airlift bioreactor were also prepared in this way. All experimental flasks and airlift bioreactors were kept at 25 °C. Flasks were incubated with a shaking regime of 150 RPM. Long-term storage of fungal cultures The different Basidiomycetes strains were grown in cultivation flasks in the above-mentioned media for 5 days, until a homogenous population of pellets developed an approx. size of 3 mm. Pelletised aggregates of the different Basidiomycetes strains cultivated in flasks were homogenised in sterile conditions using an IKA® T18 UltraTurrax® submergible homogeniser (Staufen, Germany) provided with a S18 N 10G dispersing tool. After homogenization, 100 μL of the culture was transferred into sterile cryovials already containing 450 μL of sterile glycerol (60% v/v solution) and 450 μL of sterile sucrose (30% w/v solution) for a final volume of 1 mL. The cryovials were kept in an ice bath at 0 °C for 5 min before being transferred into a − 70 °C Thermo Scientific Revco® Elite Plus® freezer (Dublin, Ireland). Airlift bioreactor experiments The airlift bioreactor was custom-manufactured regarding the design, software control and continuous module by Bionet®, Fuente Alamo de Murcia, Spain. A twin platform system of two identical airlift bioreactors was used. The bioreactor is shown in the picture diagram (Fig. 1).Fig. 1Diagram picture of the Bionet® airlift bioreactor (Fuente Alamo de Murcia, Spain) equipped with an internal draught tube and a bottom stainless-steel ring sparger The bioreactor was a double jacketed transparent and cylindrical vessel; it had a working volume of max 5 L, and it had an internal draught tube (length of 260 mm with a diameter of ∅ 40 mm) over a stainless-steel ring sparger provided with ∅ 1 mm holes. The advective-like airlift movement was guaranteed by the bioreactor elongated design, and the resulting gradient in hydrodynamic density allows the pelletised biomass and the media nutrients to completely recirculate in 17 s approximately descending inside the annular exterior ring (downcomer) and raising inside the draught tube. The air sparging provided with a filtered mechanism was maintained at a minimal 1 vvm. The continuous process module, a custom-made unit by Bionet® (Fuente Alamo de Murcia, Spain), is equipped with two peristaltic pumps of different heads (one smaller for the addition of the fresh medium and one bigger for the withdraw of the spent medium and the grown biomass). The two pumps worked in a cyclical fashion to match the hourly controlled dilution rate, typically working at a very fast pace for the medium withdrawing pump and at a very fine-tuned and slow flow rate for the medium adding pump. The medium for both the batch and the semi-continuous fermentations was the following: glucose, 10 g/L, yeast extract, 3 g/L (purchased from Sigma-Aldrich, Dublin, Ireland) and corn steep liquor (CSL)15 mL/L (purchased from Santa Cruz Biotechnology, Heidelberg, Germany). Each batch and semi-continuous airlift fermentation was performed in triplicate. Analytical procedures Glucose and other monosaccharide analyses were detected by using a Shimadzu® HPLC equipment (Duisburg, Germany) provided with a RID detector. The mobile phase used was 0.014 N H2SO4 in a Milli-Q aqueous solution that was previously vacuum filtered with 0.2-μm filter pore size; this mobile phase was used in an isocratic fashion. The column used was a BioRad® Aminex HPX-87H (Watford, England) with a solid polymeric matrix composed of polystyrene divinylbenzene. The flow rate of the mobile phase was 0.55 mL/min. α and β-glucans were detected by an enzymatic-based colorimetric method using a specific Megazyme® yeast/fungi glucan detection kit (Wicklow, Ireland) following the company provided protocol.',\n",
       " '11188776': '2. Methodology The studies cited in the present review were published between 1994 and 2024, with the majority published between 2011 and 2024. All of the studies cited in the present review were found on the PubMed database using the following keywords: Lactate, lactate metabolism, lactate metabolism-related genes (LMRGs), glucose metabolism, glycolysis, lactate shuttle, monocarboxylate transporter (MCT), G protein-coupled receptor 81 (GPR81), histone lactylation, non-histone lactylation, NAFLD, liver fibrosis, ALF and HCC.',\n",
       " '11188742': 'MethodsTo determine the value of a non-contact optical fiber mattress for apnea and hypopnea, six healthy people and six OSAHS patients were selected from Tongji Hospital to design a program to identify apnea or hypopnea. A total of 108 patients who received polysomnography for drowsiness, snoring or other suspected OSAHS symptoms. All 108 patients were monitored with both the non-contact optical fiber mattress and PSG were collected.',\n",
       " '11188689': 'Materials and Methods Study subjects To explore the efficacy of the non-contact optical fiber mattress in determining apnea and hypopnea, this study first recruited six healthy controls and six patients with a clear diagnosis of OSAHS by PSG monitoring from November 2021 to December 2021 at Tongji Hospital, Tongji Medical College, Huazhong University of Science and Technology (the Ethics Committee of Tongji Hospital, Tongji Medical College, Huazhong University of Science and Technology granted ethical approval to carry out the study within its facilities, IRB approval number: TJ-IRB20221307). Written informed consent was collected from all study participants. All 12 patients were monitored during one night of sleep using both a non-contact optical fiber mattress (non-contact vital signs monitor AVS-1, Wuhan Kairuipu Medical Technology Co., LTD; Fig. 1) and a PSG polysomnography monitor (SOMNO screen plus PSG+). The respiratory motion waveform signal data from the non-contact optical fiber mattress were compared with the simultaneously-recorded PSG signal data. The respiratory waveform data were also compared between the healthy controls and the OSAHS patients to identify apnea or hypopnea respiratory waveform characteristics, which were then combined with graphic recognition technology and big data processing to design an intelligent program to identify apnea or hypopnea. 10.7717/peerj.17570/fig-1 Figure 1 Non-contact optical fiber mattress AVS-1: (A) optical fiber mattress structure diagram; (B) optical fiber mattress placement diagram; (C) transmission loss diagram of microcurved fiber. To evaluate the diagnostic efficacy of the non-contact optical fiber mattress in adults with OSAHS, 108 patients who received sleep monitoring for drowsiness, snoring, or other suspected OSA symptoms at Tongji Hospital, Tongji Medical College, Huazhong University of Science and Technology from January 2022 to September 2022 were recruited for this study. All participants underwent sleep monitoring using both a non-contact optical fiber mattress and PSG polysomnography. This study was approved by the Ethics Committee of Tongji Hospital, Tongji Medical College, Huazhong University of Science and Technology. Written informed consent was also collected from all study participants. Inclusion criteria: patients with a complete medical history, demographic data, relevant examinations and sleep monitoring data. Exclusion criteria: patients with psychiatric disorders, pregnant women, and patients with episodic sleep disorders, sleep-related motor diseases and other sleep-related diseases. All data were recorded after a standardized medical history and physical examination by a specialist physician. General data collection General patient data, including gender, age, body mass index (BMI), neck circumference, waist circumference, and diagnostic information on hypertension, diabetes mellitus, coronary artery disease and gastroesophageal reflux, were collected. Sleep monitoring related test indicators Total recording time: Total PSG recording time was the total time between when the patient put on the PSG monitoring device and when the patient removed the PSG device; the total non-contact optical fiber mattress recording time was the total time between when the patient lay on the mattress and prepared to sleep and when the patient got up and left the mattress. Total sleep time: For both the PSG and non-contact optical fiber mattress, total sleep time was based on dynamic measurements of heart rate, respiration and movement. When the patient’s heart rate and respiration were stable and body movement decreases lasted for several min, the patient was classified as sleeping. The sleeping period ended when the patient’s body movement increased, their heart rate and respiration fluctuated violently or they got up and left the mattress. Several periods of sleep could occur during the whole night of sleep monitoring; all periods of sleep were added together and recorded as the total sleep time. AHI: AHI was calculated by dividing the total number of apnea and hypopnea events by total recording time in hours. Apnea and hypopnea were both defined using respiratory movement measurements with the non-contact optical fiber mattress: a respiratory wave amplitude 20% of the normal respiratory wave lasting more than 10 s was considered an apnea, and a respiratory wave amplitude 40% of the normal respiratory wave lasting more than 10 s was considered a hypopnea. Average heart rate: The average heart rate of the total recording time. Tachycardia index: The number of times that the average heart rate exceeded 100 beats/min per h during the total recording time (not less than 7 h). Bradycardia index: The number of times that the average heart rate was lower than 60 beats/min per h during the total recording time (not less than 7 h). Average respiratory rate: Average respiratory rate of the total recording time. Body movement index: Average number of times patient turned over or got up per hour during the total recording time. Maximum time of apnea: The longest apnea event during the total recording time. Mean time of apnea: Total apnea time (calculated by adding together the time of each apnea event) divided by the number of apnea events. Maximum time of hypopnea: The longest hypopnea event during the total recording time. Mean time of hypopnea: Total hypopnea time (calculated by adding together the time of each hypopnea event) divided by the number of hypopnea events. Percentage of total apnea time in total sleep time: The total apnea time divided by the total sleep time. Percentage of total hypopnea time in total sleep time: The total hypopnea time divided by the total sleep time. Statistical analysis Prism 8.0 was used for statistical analysis: n (%) was used for qualitative data, mean ± standard deviation was used for quantitative data of normal distribution and median of interquartile interval was used for quantitative data of non-normal distribution. Pearson analysis was used for the correlation analysis, the correlation coefficient was expressed as r and ROC curve was used for evaluating diagnostic performance. P < 0.05 was considered statistically significant.',\n",
       " '11188630': 'Materials and Methods:A total of 20 day-old chicks (DOC) of Cobb broilers were given four treatments with five replicates. The number of chickens used in this research was determined using statistical calculations, and the data obtained was homogeneous so that the population was represented. Treatments included negative control with basal feed (C-), positive control with mycotoxins contamination (C+), treatment 1: Mycotoxins contamination and mycotoxin detoxification 1.1 g/kg (T1), and treatment 2: Mycotoxins contamination and mycotoxin detoxification 1.6 g/kg (T2). Mycotoxin contamination comprised 0.1 mg/kg aflatoxin B1 and 0.1 mg/kg OTA. The treatment period for chickens was 28 days, from 8 to 35 days. A battery cage was used in this study. Chickens were kept in a closed, ventilated room and the room temperature (27°C) was monitored during the treatment period.',\n",
       " '11188585': 'Materials and Methods Ethical approval Chickens were handled humanely throughout the study, and the experimental design and protocol for the use of the chicken for research were approved by the Animal Care Use Committee of the Faculty of Veterinary Medicine Airlangga University (Approval No. 1.KEH.033.02.2023). Study period and location This study was conducted from February 2023 and March 2023 at the Faculty of Veterinary Medicine, Airlangga University, Surabaya, Indonesia. Experimental animals and diets In this study, 20 Cobb broiler day-old chicks were used as experimental animals. Chickens were reared from day old chick until day 35 and obtained from Charoen Pokphand Indonesia Company Limited. It was kept in deep litter with drinkers and standard feeding troughs. A basal diet was started from day-old chickens until 7 days old, and then, the experimental design was continued. Experimental design A total of 20 chickens were divided into four treatment groups with five replications for each treatment. The chickens underwent an adaptation period until they were 7 days old. On the 8th day, they were treated in stages according to a predetermined group. Treatments included negative control with basal feed (C–), positive control with mycotoxin contamination (C+), treatment 1: Mycotoxin contamination and mycotoxin detoxifier 1.1 g/kg (T1), and treatment 2: mycotoxin contamination and mycotoxin detoxifier 1.6 g/kg (T2). Mycotoxin contamination consisted of 0.1 mg/kg aflatoxin B1 and 0.1 mg/kg OTA. The feed used for rearing includes CP511 chicken feed produced by Charoen Pokphand Indonesia Company Limited and drinking water was provided ad libitum. Mycotoxins in the form of aflatoxin B1 (0.1 mg/kg), OTA (0.1 mg/kg), and the mycotoxin detoxifier Mycofix® (Biomin GmbH, A-3130 Herzogenburg, Austria) were used for the treatment. Chickens were fed every morning and evening, with unlimited drinking water. Based on references to previous studies [10, 11], we have considered the use of mycotoxin detoxifier doses. The choice of dose was chosen on the basis of the following considerations: According to Ramandani et al. [11], the level of aflatoxin that is still safe in feed ingredients is no more than 50 pbb or 0.05 mg/kg. Meanwhile, ochratoxin [10] at a dose of 0.1 mg/kg can already cause a decrease in the number of heterophiles. Based on these considerations, aflatoxin and ochratoxin doses of 0.1 mg/kg and 0.1 mg/kg, respectively, were chosen as the dose studied. The cage used is a closed house equipped with a blower and a room thermometer to measure theroom temperature (27°C). Ventilation and lighting in the room were arranged in such a way as to support the health of broiler chickens. To avoid air pollution and vehicle noise, the cage used were placed far from major roads. The feed was stored in a dry and non-humid place to avoid damage to the feed ingredients. Cleaning of the cage was carried out every day by removing chicken droppings and changing the bedding to keep the cage clean. Blood profile with hematological determinations Broiler chickens were to take blood samples on day 35. Chicken blood was collected through the brachial vein using a 1 mL syringe and accommodated in a 1 mL heparin tube [12]. The tube was inverted until it was homogeneous, then stored in a cool box and examined. The blood profile was measured by collecting blood from the wing area in a heparin vacuum tube and then examining it using a hematology analyzer. Statistical analysis Data obtained from the hematology analyzer test were then analyzed using the SPSS Version 20 (IBM Corp., NY, USA) and expressed in mean values and standard deviations [13] using the one-way ANOVA test [14]. If there is a difference between the treatments, it is continued with Ducan’s test with a significance level of 0.05 [15].',\n",
       " '11188542': 'Materials and Methods:A total of 280 Cobb broiler chicks were randomly distributed into four groups: Basal feed without additives (CNTL), basal feed with 0.01% encapsulated A. australis leaf extract (EALE), 0.01% chitosan (CHIT), and 0.01% EALE and 0.01% chitosan (EACH). Sample collection and data measurement were conducted on day 36.',\n",
       " '11188473': 'Materials and Methods Ethical approval The experiment was approved by the Animal Ethical Committee of the Faculty of Animal and Agricultural Sciences under approval number 59-07a/A-16/KEP-FPP. Study period and location The study was conducted from April to July 2023 at the Faculty of Animal and Agricultural Sciences, Universitas Diponegoro, Semarang, Central Java, Indonesia. Preparation of encapsulated A. australis L. leaf extract A. australis leaves were collected from a garden close to the campus of Universitas Diponegoro and air-dried and milled into a fine powder. The leaves are identified and confirmed by the herbarium expert from the Department of Biology, Faculty of Sciences and Mathematics, Universitas Diponegoro. We soaked the leaf powder in 70% ethanol (1:6, g: mL) at room temperature (25°C) for 72 h. A vacuum rotary evaporator was used to evaporate the filtrate. Maltodextrin (1:1.25; g: g) was used to encapsulate the extract in accordance with the freeze-drying-based encapsulation technique. The EALE was kept refrigerated before use in in vivo experiments. Broiler chicken trials A total of 280 mixed-sex Cobb broiler chicks were used in the experiment based on a fully randomized design. The chicks had an average body weight of 42.97 ± 1.64 g after weighing individually. Subsequently, they were divided into four groups of treatments. There were seven replications in each treatment group, with 10 chicks per replicate/pen. The treatment groups were as follows: (1) basal feed without feed additives (CNTL); (2) basal feed added with 0.01% EALE; (3) basal feed added with 0.01% chitosan (CHIT); and (4) basal feed with 0.01% EALE and 0.01% chitosan (EACH). The chicks were offered commercial pre-starter feed from the time of arrival until day 7, which, according to the feed label, contained 22–24% crude protein, less than 5% crude fiber, 5% crude fat, and 7% ash. Chicks were ad libitum fed starter and finisher feeds from days 8 to 36 (Table-1) [12]. The chicks were treated from day 1 to day 36 of the experiments. The chitosan used in this investigation was a 200-nm commercial shrimp shell chitosan (CV. ChiMultiguna, Indonesia). Table 1 Broiler feed composition. Ingredients (%, unless otherwise specified) Starter (day 8–21) Finisher (day 22–36) Yellow maize 54.1 62.7 Palm oil 2.10 2.10 Soybean meal 40.0 31.7 DL-methionine 0.19 0.19 Bentonite 0.75 0.75 Limestone 0.75 0.75 Monocalcium phosphate 1.30 1.05 Premi×1 0.34 0.34 Chlorine chloride 0.07 0.07 Salt 0.40 0.40 Nutritional compositions ME (kcal/kg)2 2,900 3,004 Crude protein, % 22.0 19.0 Crude fiber, % 5.50 5.62 Ca, % 1.04 0.96 P (available), % 0.57 0.54 Met, % 0.52 0.48 Lys, % 1.18 0.98 1Each kg of feed contained 1,100 mg Zn, 1,000 mg Mn, 75 mg Cu, 850 mg Fe, 4 mg Se, 19 mg I, 6 mg Co, 1,225 mg K, 1,225 mg Mg, 1,250,000 IU vit A, 250,000 IU vit D3, 1,350 g pantothenic acid, 1,875 g vit E, 250 g vit K3, 250 g vit B1, 750 g vit B2, 500 g vit B6, 2,500 mg vit B12, 5,000 g niacin, 125 g folic acid and 2,500 mg biotin, 2ME (metabolizable energy) was calculated based on the formula of Bolton [12]: 40.81 {0.87 [crude protein + 2.25 crude fat + nitrogen-free extract] + 2.5} The chicks were reared in open broiler houses lined with rice husks as litter. During the experiment, a continuous lighting program was implemented. The infectious bronchitis virus and Newcastle disease virus (NDV) were sprayed on chicks as soon as they hatch to impart immunity. Using drinking water, the chickens were vaccinated against NDV (Medivac ND La Sota, Indonesia) on day 18 after receiving the Gumboro (infectious bursal disease virus/IBDV) vaccination at age of 11 days (Medivac Gumboro B vaccine). Weekly body weight, feed consumption, and feed conversion ratio (FCR) measurements were made. A 3 mL syringe was used to collect blood sample from wing vein of one male chick (representing the average body weight for each replicate) per replicate (seven chicks per treatment group) on day 36. Blood was placed in a vacutainer containing an anticoagulant (ethylenediaminetetraacetic acid/ethylenediaminetetraacetic acid) for complete blood count analysis. In the absence of anticoagulants, the remaining blood was transferred to another vacutainer. The blood was allowed to clot at 25°C for approximately 2 h before centrifugation for 10 min at 500 × g to produce serum. The serum was stored in a freezer at 10°C until analysis. The intestines were removed from the chicken as soon as it was slaughtered. To measure the morphology of the small intestine (villus height and crypt depth [CD]), segments of the duodenum, jejunum, and ileum (approximately 2 cm) were placed in 10% buffered formalin (Leica Biosystems Richmond, Inc., Richmond, USA). The ileum and cecum digesta were placed in each sterile sample pot and subjected to additional laboratory analysis to determine the population of selected bacteria in the intestines. Laboratory analysis Complete blood counts were measured using the Prima Fully-Auto Hematology Analyzer (PT. Prima Alkesindo Nusantara, Jakarta, Indonesia) according to the manufacturer’s protocols. Enzyme-based colorimetric techniques were used to determine serum lipid profiles (total triglycerides, total cholesterol, low-density lipoprotein, and high-density lipoprotein), as well as serum creatinine and uric acid levels. The levels of serum total protein, albumin, glucose, and aspartate aminotransferase were determined using spectrophotometric and photometric techniques. To determine the globulin concentration, the total protein value was subtracted from the serum albumin value. All biochemical analyses of serum samples (DiaSys Diagnostic System GmbH, Holzheim, Germany) were performed according to the manufacturer’s instructions. Hematoxylin and eosin-stained 5 μm sections of the duodenum, jejunum, or ileum were used to histologically examine the small intestinal segments. The villous height and CD in each segment were measured using an optical microscope fitted with a digital camera (Leica Microsystems GmbH, Wetzlar, Germany). For each sample, the mean CD and villous height were determined using five measurements. The bacterial population in the ileal and cecal contents was determined using the total plate count method. Coliforms and lactose-negative Enterobacteriaceae were counted as red and colorless colonies on MacConkey agar (Merck KGaA, Darmstadt, Germany) after a 24-h aerobic incubation at 38°C. Lactic acid bacteria (LAB) were counted on de Man, Rogosa, and Sharpe (MRS; Merck KGaA) agar after a 48-h anaerobic incubation period at 38°C. Statistical analysis Statistical Package for the Social Sciences version 16.0 (IBM Corp., NY, USA) was used to analyze the data collected during the study. Duncan’s multiple analysis was performed when there was a discernible effect (p= 0.05) from the treatments. Tendency was implemented when 0.05 ≤ p < 0.10.',\n",
       " '11188471': 'Materials and Methods:On day 85 of gestation, 115 multiparous healthy Landrace Yorkshire sows were randomly assigned to two groups with equal parity numbers. The control group (58 sows) was fed with basal diets, and the probiotic group (57 sows) was fed with basal diets +1010 colony-forming unit (CFU) B. subtilis QST 713 from day 85 to parturition. Back fat thickness on days 85 and 110, number of total born, number of born alive, stillbirth and mummy rates, individual birth weight, litter birth weight, within-litter variation of piglet birth weight, and postpartum vaginal discharge duration were recorded and compared between the two groups.',\n",
       " '11188439': 'Materials and Methods Ethical approval The Committee on Animal Research and Ethics of the Faculty of Veterinary Medicine, Vietnam National University of Agriculture (CARE-2023/02) reviewed and approved the study protocol. Study period and location This study was conducted from February to August 2023 on a farm in Quang Ninh province, Vietnam. Animals and housing In total, 115 mixed parity Landrace Yorkshire crossbred sows (parity number 3.55 ± 0.87, 2–5) raised on one farm were used in the present study. During pregnancy, sows were kept in individual gestational crates and moved to farrowing rooms approximately 1 week before farrowing. Sows were allocated in individual farrowing crates measuring 2.2 × 0.6 m in farrowing rooms. During the first 21 days, sows were fed 1.8–2.4 kg of an industrialized gestating feed, which was increased to 2.0–2.8 kg during days 22–84 and 2.2–3.0 kg during days 85–107. Sows were fed 2.2–3.0 kg of industrialized lactating feed from day 108 to farrowing. The ingredients of gestating and lactating feeds consisted of soybean meal, animal protein, rice bran, rice, corn, cassava root, wheat bran, vitamins, amino acids, and minerals. Nutrient compositions of the feeds are presented in Table-1. Table 1 Nutrient compositions of the basal diets. Nutrient compositions Gestation diet Lactation diet Crude protein (g/kg) 136 162 Crude fiber (g/kg) 62.2 60.2 Crude fat (g/kg) 78.2 66.7 Ash (g/kg) 59.6 60.6 Dry matter (g/kg) 894.2 895.3 Metabolizable energy (kcal/kg) 3585 3581 Total phosphorus (g/kg) 7.7 8 Calcium (g/kg) 9.6 9.1 Total amino acids (g/kg) 141.9 174 Selenium (mg/kg) 0.66 0.69 Copper (mg/kg) 47 53 Iron (mg/kg) 236 234 Zinc (mg/kg) 279 267 Manganese (mg/kg) 236 244 The industrialized gestation and lactation feeds comprised soybean meal, animal protein, rice bran, rice, corn, cassava root, wheat bran, vitamins, amino acids, and minerals. The gestational diet was fed to sows during the first 107 days of gestation. The lactational diet was fed to sows from day 108 of gestation to farrowing Sows were vaccinated against classical swine fever, foot and mouth disease, and Escherichia coli at weeks 9, 12, and 14 of gestation, respectively. Vaccination against porcine reproductive and respiratory syndrome and Aujeszky’s disease was conducted every 4 months, and vaccination against porcine parvovirus was conducted on postpartum day 14. Sows were dewormed twice per year. Study design Sows were randomly allocated into two groups on day 85 of pregnancy using a random number table. Randomization was conducted so that each group had a similar number of sows in each parity. According to this method, the treatment group had 6, 28, 15, and 8 sows in parities 5, 4, 3, and 2, respectively, and the control group had 6, 28, 15, and 9 sows in parities 5, 4, 3, and 2, respectively. In addition to the basal diet, sows in the treatment group were supplemented with 1010 colony-forming unit B. subtilis QST 713 (1 g, Baymix, GROBIG®, BS, Bayer de Mexico, S.A. de C.V., Mexico) per day. Supplementation was performed once per day in the morning. B. subtilis QST 713 was added to the feed of the sows so that it could be eaten. Sows in the control group consumed basal diets without B. subtilis QST 713 supplement. Data collection Back fat thickness of the sows at both sites (site B) was measured on days 85 and 110 of pregnancy (Renco LEAN-MEATER, Renco, USA). The last rib of the sow was located by the middle finger, and site B, approximately 65 mm below the spine was identified. Cooking oil was applied to site B, and two measurements were conducted at each side. The back fat thickness of the sow was calculated as the average of four measurements. The parity number, day of insemination, and day of farrowing were recorded at farrowing. Litter size was the total number of piglets born, including those born alive and those born dead. Individual birth weight (g) of born alive piglets was measured before colostrum intake using a 5 g accurate portable digital hook weighing scale (Weihang, China). Litter birth weight was the sum of the birth weights of all piglets born alive. The mean birth weight of a given litter was calculated on the basis of the weight of individual live-born piglets in that litter. Within-litter variation in piglet birth weight (BWV) was expressed as either the standard deviation of birth weight or the coefficient of variation of birth weight. Standard deviation of piglets’ birth weight (g) was calculated from birth weight of individual piglets in a given litter. The coefficient of variation (%) of piglets’ birth weight was calculated by dividing the standard deviation by the mean birth weight of a given litter. After farrowing, postpartum vaginal discharge was observed twice per day in the morning and afternoon until no discharge was detected [10]. The postpartum vaginal discharge duration (PVDD) was calculated as the interval between farrowing and the first observation with no discharge. Statistical analysis Independent Student’s t-test was used to compare back fat thickness, number of total piglets born per litter, number of piglets born alive per litter, standard deviation of birth weight, coefficient of variation of birth weight, and duration of postpartum vaginal discharge between the treatment and control groups. Chi-square test was used to compare the incidence of stillbirths, mummies, and born dead piglets in litter. Generalized linear mixed models were used to compare the stillbirth rate, mummy rate, and dead birth rate. The linear mixed effect model was used to compare individual birth weights. Student’s t-test and Chi-square tests were performed using the Statistical Package for the Social Sciences version 22.0 (IBM SPSS, Armonk, NY, USA). Generalized linear mixed models and linear mixed effect models were conducted in RStudio Desktop 1.3.1093 (RStudio Team: Integrated Development for R, Boston, MA, USA). A p < 0.05 was considered statistically significant.',\n",
       " '11188429': 'MethodsA retrospective analysis of 2016 to 2020 National Inpatient Survey data identified adults with CDI, categorized by the presence of colon cancer. Hospitalization characteristics, comorbidities, and outcomes were compared between groups. Primary outcomes included in-hospital mortality, length of stay, and total hospital charges. The secondary outcomes were CDI complications. Multivariate logistic regression analysis was performed, with P values ≤0.05 indicating statistical significance.',\n",
       " '11188375': 'Materials and methods Dataset establishment WoSCC database stands as one of the foremost comprehensive, systematic, and authoritative repositories globally, often utilized for visualizing scientific literature.10 To procure data that is both representative and accurate, the retrieval parameters were delineated as follows: TS = (“vaccine*” OR “vaccination” OR “vaccinate*” OR “vaccinating”) AND (“Atherosclerosis” OR “Atheroscleroses” OR “Atherogenesis” OR “Atheromatous Plaque*” OR “Arteriosclerosis” OR “Coronary Disease*” OR “coronary artery disease” OR “Atherosclerotic Plaque*” OR Carotid Artery Disease* OR “Fibroatheroma*” OR “Arterial Fatty Streak*” OR “Atheroma*”). To facilitate subsequent analysis, the citation edition was confined to the Science Citation Index Expanded (SCI-Expanded), and only articles and reviews written in English were included. All bibliographic records, encompassing titles, authors, keywords, institutions, abstracts, references, publication year, etc., were archived in plain text format. Data retrieval was conducted on February 8th, 2024. Two authors (BCJ and RW) autonomously executed data identification and screening of the retrieved literature. In instances of disagreement, a third author (YHH) was consulted for arbitration. Given that the research data was derived from an open database, ethical considerations were not applicable. Database analysis The screened literature data was imported into Biblioshiny and VOSviewer (version 1.6.19) for visual analysis. Biblioshiny, powered by the R software 4.3.2 Bibliometrix package (version 4.1.4), offered an intuitive and well-structured interface to quantitatively analyze annual publications and visualize national cooperation, core journals, author publication timelines, as well as trend and thematic maps of keywords within the realm of AS vaccines. VOSviewer specializes in co-occurrence network visualization of scientific knowledge.11 In this study, VOSviewer was leveraged to construct keyword co-occurrence network and density maps. In the co-occurrence network graph, keywords were represented as nodes, with larger nodes denoting higher frequency of keywords occurrences. Keywords sharing similar features were clustered together and assigned the same color (Figure 1). Figure 1.Flow chart. Furthermore, the Hirsch index (H-index) served as a metric for evaluating the academic output and influence of researchers, indicating that a researcher published h articles, each cited at least h times.12 The multiple country publications (MCP) ratio was employed to gauge national cooperation,13 while the impact factor (IF) and journal citation report (JCR) zoning were utilized to assess the quality of scientific literature.14 Bradford’s law was applied to delineate the core journals in the field of AS vaccines.15',\n",
       " '11188354': 'MethodsA large multi-ethnic sample of 4,155 participants from the West China Health and Aging Trend (WCHAT) study was analyzed. SO was defined according to the newly published consensus of the ESPEN/EASO. Furthermore, SO was diagnosed as a combination of sarcopenia and obesity. The criteria established by the Asian Working Group for Sarcopenia 2019 (AWGS2019) were used to define sarcopenia. Obesity was defined by four widely used indicators: percent of body fat (PBF), visceral fat area (VFA), waist circumference (WC), and body mass index (BMI). Cohen’s kappa was used to analyze the diagnostic agreement of the above five diagnostic methods.',\n",
       " '11188322': 'Methods Study population This study used the baseline data from the West China Health and Aging Trend (WCHAT) study. Previous studies have published details of the study design and questionnaires used to generate data (13). In this study, 7,536 participants were enrolled at first. Out of these, only 4,500 participants aged 50 years and above finished sarcopenia assessment. Furthermore, 32 subjects were excluded as they did not have information on handgrip strength (HS), PBF, or SMM /W. In addition, 313 subjects were excluded as they did not have information on obesity measurements like BMI, WC, PBF, or VFA. Finally, 4,155 participants were included in the current study (Figure 1). Figure 1 Study flow. Measurements of sarcopenia According to the Asian Working Group for Sarcopenia 2019 (AWGS2019) consensus criteria, sarcopenia was defined as low muscle mass in the presence of either low HS or slow gait speed (14). The Inbody 770 instrument (Biospace, Seoul, Korea) was used to assess the muscle mass. The cut-off values of appendicular skeletal muscle mass index (ASMI) were 7.0 kg/m2 for males and 5.7 kg/m2 for females. The HS was measured with a grip dynamometer (EH101; Camry, Zhongshan, China). The test was repeated twice and the highest value was recorded (15). Low muscle strength was defined as HS <28 kg in males and < 18 kg in females. The four-meter gait speed was tested using an infrared sensor (16). During the test, participants were required to walk at their usual pace. The acceleration and deceleration phases were excluded. The cut-off value of gait speed was 1.0 m/s (17). Measurements of obesity The indicators of obesity included PBF, VFA, WC, and BMI. PBF and VFA were measured using Bioelectric Impedance Analysis. WC was measured with a flexible, non-elastic tape at the midpoint between the ribs and ilium in the standing position. BMI was calculated by dividing weight by the square of height (CSTF-ST, Qinghuatongfang, China). The cutoff values of obesity indicators were as follows: (1) PBF ≥ 41% for females and PBF ≥ 29% for males; (2) VFA > 100 cm2 (18); (3) WC ≥ 80 cm for females and WC ≥ 90 cm for males (19), and (4) BMI ≥ 28 kg/m2 (20). Definitions of SO According to the 2022 ESPEN/EASO consensus, participants with decreased muscle strength (HS < 28 kg for males and < 18 kg for females), low muscle mass (SMM/W < = 37% for males, ≤ 27.6% for females), and high-fat mass (>29% for male, > 41% for female) were defined as SO (12). Further, SO was also diagnosed as a combination of the above four different diagnostic criteria of obesity and sarcopenia. Laboratory examinations Fasting blood samples were obtained from the antecubital vein after an overnight fast. Complete blood count, blood glucose, and lipid profile were tested. Inflammatory biomarkers were further calculated, including neutrophil-to-lymphocyte ratio (NLR), platelet-to-lymphocyte ratio (PLR), and systemic immune-inflammation index (SII). SII was calculated using the following formula: SII = peripheral platelets* neutrophils/ lymphocyte counts (21). Statistical analysis Continuous variables are presented as mean ± standard deviation. Categorical variables are presented as numbers (percentages). Differences between groups were evaluated with the unpaired t-test and Mann–Whitney U test for continuous data with a normal and non-normal distribution, respectively. A comparison of categorical variables was conducted with chi-square tests. We also assessed the sex-stratified and age-stratified prevalence of SO. Binary logistic regression analysis was used to examine the association of age and sex with SO. A diagnostic agreement of SO between different diagnostic methods was evaluated using Cohen’s kappa score. The interpretations of κ value were as follows: poor agreement = 0.00–0.20, fair agreement = 0.21–0.40, moderate agreement = 0.41–0.60, good agreement = 0.61–0.80, and very good agreement = 0.81–1.00 (22). Statistical analyses were performed using Stata v16.0 (Stata Corp, College Station, TX, USA) software programs. Two-sided p-values of <0.05 were considered statistically significant.',\n",
       " '11188286': 'Agreement between different SO diagnostic methods The agreement between different diagnostic methods is shown in Table 3. The agreement between different diagnostic methods for SO varied, with poor agreement observed between 2022 ESPEN/EASO and AWGS+VFA (κ = 0.16), AWGS+WC (κ = 0.06), and AWGS+BMI (κ = 0.09), while fair agreement was found between 2022 ESPEN/EASO and AWGS+PBF (κ = 0.32). Among the other four diagnostic methods, AWGS+VFA and AWGS+WC showed good agreement (κ = 0.67). Meanwhile, AWGS+VFA was moderately consistent with AWGS+VFA (κ = 0.55). Stratified by sex, there was good agreement between AWGS+VFA and AWGS+PBF in males (κ = 0.62) and between AWGS+VFA and AWGS+WC in females (κ = 0.71). Table 3 Agreement between different diagnostic methods of SO. SO definition Men Women Total Cohen’s kappa Magnitude Cohen’s kappa Magnitude Cohen’s kappa Magnitude 2022 ESPEN/EASO AWGS+PBF 0.35 Fair 0.18 Poor 0.32 Fair 2022 ESPEN/EASO AWGS+VFA 0.28 Fair 0.07 Poor 0.16 Poor 2022 ESPEN/EASO AWGS+WC 0.11 Poor 0.05 Poor 0.06 Poor 2022 ESPEN/EASO AWGS+BMI 0.04 Poor 0.20 Poor 0.09 Poor AWGS+PBF AWGS+VFA 0.62 Good 0.51 Moderate 0.55 Moderate AWGS+PBF AWGS+WC 0.39 Fair 0.38 Fair 0.37 Fair AWGS+PBF AWGS+BMI 0.14 Poor 0.22 Fair 0.17 Poor AWGS+VFA AWGS+WC 0.53 Moderate 0.71 Good 0.67 Good AWGS+VFA AWGS+BMI 0.26 Fair 0.08 Poor 0.13 Poor AWGS+WC AWGS+BMI 0.27 Fair 0.09 Poor 0.12 Poor',\n",
       " '11188251': 'Metabolic and inflammatory profiles of SO diagnosed by different methods The fasting plasma insulin was significantly higher in the SO group diagnosed by the 2022 ESPEN/EASO (10.41 ± 14.69) as compared to the non-SO group (8.26 ± 8.88) (Table 4). Fasting glucose was significantly higher in SO groups diagnosed by the 2022 ESPEN/EASO and AWGS+BMI (p < 0.05 in both). Triglyceride and cholesterol were significantly higher in SO groups diagnosed by AWGS+VFA and AWGS+WC (p < 0.05 in both). High-density lipoprotein (HDL) in SO groups diagnosed by ESPEN/EASO and AWGS+BMI was significantly lower than that in control groups (both p < 0.05). Meanwhile, except for the SO group diagnosed by AWGS+BMI, the low-density lipoprotein (LDL) of all SO groups was significantly increased (all p < 0.05). Table 4 Metabolism and inflammation characteristics of study participants (N = 4,155). Characteristics 2022 ESPEN/EASO AWGS + PBF AWGS + VFA AWGS + WC AWGS + BMI C SO p value C SO p value C SO p value C SO p value C SO p value Insulin 0 (uU/mL) 8.26 (8.88) 10.41 (14.69) <0.001 8.43 (9.57) 8.02 (6.29) 0.447 8.43 (9.63) 8.17 (5.72) 0.512 8.44 (9.66) 7.97 (5.20) 0.797 8.40 (9.44) 9.42 (5.40) 0.137 Fasting glucose (mmol/L) 5.56 (1.73) 5.87 (2.17) 0.025 5.56 (1.72) 5.89 (2.37) 0.199 5.57 (1.74) 5.64 (2.07) 0.403 5.57 (1.73) 5.73 (2.18) 0.823 5.57 (1.76) 6.59 (2.69) 0.010 Triglyceride (mmol/L) 1.86 (1.79) 1.68 (1.10) 0.635 1.85 (1.75) 1.85 (1.82) 0.498 1.85 (1.75) 1.91 (1.72) 0.010 1.84 (1.73) 2.01 (2.04) 0.004 1.85 (1.75) 1.79 (1.05) 0.727 Cholesterol (mmol/L) 4.78 (0.92) 4.80 (0.99) 0.579 4.77 (0.91) 4.90 (1.11) 0.139 4.76 (0.91) 5.07 (1.06) <0.001 4.76 (0.91) 5.00 (1.05) <0.001 4.78 (0.93) 4.80 (0.79) 0.786 HDL (mmol/L) 1.28 (0.31) 1.17 (0.25) <0.001 1.27 (0.31) 1.24 (0.28) 0.115 1.27 (0.31) 1.29 (0.29) 0.158 1.27 (0.31) 1.28 (0.29) 0.375 1.27 (0.31) 1.15 (0.25) 0.041 LDL (mmol/L) 2.65 (0.87) 2.87 (0.85) <0.001 2.66 (0.87) 2.82 (0.85) 0.024 2.65 (0.87) 2.91 (0.85) <0.001 2.66 (0.87) 2.80 (0.92) 0.007 2.67 (0.87) 2.84 (0.76) 0.413 WBC (10^9/L) 5.83 (1.65) 6.24 (1.86) <0.001 5.83 (1.64) 6.32 (1.96) <0.001 5.85 (1.67) 5.91 (1.57) 0.303 5.84 (1.67) 6.04 (1.66) 0.023 5.86 (1.67) 5.70 (1.24) 0.920 GPR (%) 61.07 (8.60) 62.47 (8.94) 0.006 61.08 (8.63) 62.66 (8.62) 0.023 61.16 (8.63) 61.39 (8.65) 0.700 61.12 (8.58) 61.82 (9.31) 0.325 61.19 (8.63) 58.42 (8.09) 0.079 LPR (%) 31.58 (7.94) 30.10 (8.03) 0.004 31.55 (7.94) 30.10 (7.98) 0.023 31.48 (7.94) 31.42 (8.08) 0.902 31.50 (7.90) 31.10 (8.61) 0.545 31.46 (7.95) 33.89 (8.32) 0.080 NLR 2.20 (1.08) 2.45 (1.62) 0.006 2.21 (1.09) 2.46 (1.64) 0.025 2.22 (1.14) 2.22 (0.98) 0.763 2.21 (1.11) 2.34 (1.35) 0.424 2.22 (1.13) 1.93 (0.87) 0.083 PLR 103.07 (42.47) 107.05 (49.17) 0.554 103.29 (42.77) 104.36 (46.45) 0.922 103.03 (42.68) 107.79 (46.86) 0.116 103.15 (42.60) 105.87 (47.64) 0.653 103.34 (42.91) 104.58 (53.71) 0.608 SII 371.58 (228.10) 428.01 (315.16) <0.001 372.56 (229.37) 425.73 (320.28) 0.007 374.24 (236.97) 393.31 (215.85) 0.045 373.10 (232.15) 406.90 (275.19) 0.105 375.73 (235.95) 343.32 (178.59) 0.612 Data are indicated as mean (standard deviation). Insulin 0, fasting plasma insulin; HDL, high-density lipoprotein; LDL, low-density lipoprotein; WBC, white blood cells; GPR, neutrophils ratio; LPR, lymphocyte ratio; NLR, neutrophil-to-lymphocyte ratio; PLR, platelet-to-lymphocyte ratio; SII, systemic immune-inflammation index. Indicators related to inflammation, including neutrophils ratio (GPR) and NLR were significantly higher in SO groups diagnosed by the 2022 ESPEN/EASO and AWGS+PBF (both p < 0.05). White blood cells (WBC) were significantly higher in SO groups diagnosed by the 2022 ESPEN/EASO, AWGS+PBF, and AWGS+WC (all p < 0.05). The lymphocyte ratio was significantly lower in SO groups diagnosed by the 2022 ESPEN/EASO and AWGS+PBF (p < 0.05 in both). Furthermore, SII was significantly higher in SO groups diagnosed by the 2022 ESPEN/EASO, AWGS+PBF, and AWGS+VFA (all p < 0.05).',\n",
       " '11188178': 'Materials and methods Strains All strains used in this work were derived from the E. coli K-12 W3110 RpoS+ strain [14] (Supplementary Table S1). Gene deletions were performed by the λ-Red recombination system as described [19], using the pISJ8 plasmid carrying both the λ-Red system and the Flp recombinase [20]. Water sampling The samples of lake water were taken from the small lake (depth ~1.2 m according to the local authorities, surface area ~ 235 m2 measured with Google Earth) near Marburg, Germany (GPS coordinates: 50°49′50″N 8°46′49″E). It is a man-made urban lake that is used for rainwater collection with no known bacterial/chemical contamination. The surrounding land is forested and not used for agriculture. The sampling times, temperature of air and water, and pH were recorded (Supplementary Table S2). Water samples were either used in their original state or filter-sterilized through the Steritop filter unit (Merck Millipore, Darmstadt, Germany) with a pore size of 0.22 μm. Initial assessment of E. coli growth in lake water Pre-cultures of E. coli W3110 RpoS+ [14] were grown to an OD600 ~1 in tryptone broth (TB) medium (10 g tryptone, 5 g NaCl per litre, pH 7) on a rotary shaker at 37°C. To identify E. coli cells during this initial assessment of growth in the presence of the autochthonous microbiota, E. coli was transformed with a pTrc99a plasmid [21] encoding green fluorescent protein (GFP) under the control of a promoter inducible by isopropyl ß-D-1-thiogalactopyranoside (IPTG), and pre-cultures were grown overnight in TB with 100 μg/ml ampicillin and 50 μM IPTG to induce GFP expression. These pre-cultures were then inoculated into the samples of lake water supplemented with 50 μM IPTG at a 1:50 dilution (resulting in ~107 cells per millilitre as assessed with the flow cytometer) and incubated at the room temperature in 24-well plates without shaking. The total number of cells, and, where applicable, the number of GFP-positive cells per millilitre of lake water were measured daily by flow cytometry on the BD LSRFortessa SORP cell analyser (BD Biosciences, Heidelberg, Germany) using the high-throughput sampling system for 96-well plates. For cytometry, the cultures were mixed vigorously by pipetting and then diluted 10-fold in sterile phosphate-buffered saline (PBS; 140 mM NaCl, 2.7 mM KCl, 1.5 mM KH2PO4, 8.1 mM Na2HPO4, pH 7.4). A 488-nm laser was used to identify the cells by measuring the scatter parameters and to excite GFP. Flow cytometry results were analysed using FlowJo software (BD Biosciences) version 10. Construction and characterization of the transposon mutant library The transposon mutant library of E. coli W3110 RpoS+ [14] was constructed by inserting the randomly barcoded Tn5 system (described in detail in Wetmore et al. [11]) into the genome. This transposon delivery system was kindly gifted to us by Dr. Adam Deutschbauer (University of California, Berkeley). It consists of the set of plasmids carrying the transposase, the kanamycin resistance cassette, and the random 20-bp DNA barcodes [11]. Each barcode is flanked by the common binding sites U1 (GATGTCCACGAGGTCTCT) and U2 (CGTACGCTGCAGGTCGAC), enabling the amplification of the barcode sequence. This barcode, together with the kanamycin resistance cassette, is in turn flanked by the transposon Tn5 inverted repeats. When introduced into the cell, the resistance cassette and the barcode are randomly integrated into the genome by the transposase. We transformed this plasmid library into our strain of interest by electroporation, and after 4 h of recovery at 37°C in Luria-Bertrani (LB) medium (10 g tryptone, 5 g yeast extract, 5 g NaCl per litre, pH 7), the cells were plated in LB-agar supplemented with kanamycin (50 μg/ml). We repeated this procedure until we reached 500 000 colony forming units (CFU) of library coverage. The colonies were washed with TB + 20% glycerol. Some aliquots were kept for library characterization, and the rest were stored at −80°C in 1 ml aliquots. To create the comprehensive map of the library, the genomic DNA was extracted using the NucleoSpin Microbial DNA Kit (Macherey-Nagel, Düren, Germany) and sonicated to an average size of ~300 bp. The NEBNext Ultra II DNA Library Prep Kit and NEBNext Multiplex Oligos (New England Biolabs, Ipswich, Massachusetts) were used to prepare the high-throughput sequencing library. To enrich the barcode sequences in the library, the primer for U1 [11] was used in the final step of the protocol. The sequences and positions of the barcodes were then determined by MiniSeq (Illumina, San Diego, California) sequencing, using the custom script written in Python (version 3.9.12, with Biopython version 1.78) [22]. Briefly, the presence of both U1 and U2 sites, and the quality of the sequence between them, was checked (length of exactly 20 bp and Q > 30 for each base). The script name is tnseq_phase1.py. The reads containing the confirmed barcodes were then aligned to the E. coli K-12 W3110 reference genome (NCBI reference sequence: NC_007779.1) using blastn from the ncbi-blast toolkit [23] (script BLASTN_command.sh). The sequence and the position of each barcode were then combined into a single csv file using another custom-written Python script (tnseq_phase2.py). The mapping resulted in 430 849 unique transposon insertions in 3833 genes. Cultivation of transposon mutant library Pre-cultures were inoculated from whole 1 ml glycerol stocks of the library into 50 ml TB medium supplemented with kanamycin (30 μg/ml) and grown at 37°C on a rotary shaker to an OD600 ~1. These pre-cultures were then diluted 1:50 in lake water (~107 cells per millilitre) and incubated without antibiotics at room temperature in 24-well plates without shaking. Samples for further analysis were taken on Days 2, 4, and 8 of cultivation, with the sample of initial culture (Day 0) used as a control. The experiment was performed with three independent lake water samples, each in three technical replicates. The biomass from four wells of a 24-well plate, each containing 2.5 ml, was combined together to form one technical replicate. Genomic DNA was extracted using the NucleoSpin Microbial DNA Kit (Macherey-Nagel). Barcode sequences were amplified by polymerase chain reaction (PCR) using Q5 polymerase (New England Biolabs) supplemented with GC-enhancer and barcoded tru-seq-derived primers to U1 and U2 sequences (200 nM of each, forward primers carried an individual barcode to allow multiplexing and the reverse primer was common) [11]. The complete primer list can be found in Wetmore et al. [11] (Supplementary Table S2, primers Barseq_P1 to Barseq_P2_IT096). The amount of the template genomic DNA per reaction was 150–200 ng. The cycling conditions were 98°C for 4 min followed by 25 cycles of 30 s at 98°C, 30 s at 55°C, and 30 s at 72°C, followed by a final extension at 72°C for 5 min. The PCR reactions were pooled together (10 μl from each reaction), purified with 0.9X of AMPure beads (Beckman Coulter, Brea, California), and sequenced at the GeneCore facility (EMBL, Heidelberg, Germany) using the NextSeq 500 High platform (Illumina). For demultiplexing, we relied on the sequencing facility. The quality of the resulting dataset was assessed using FastQC [24] and MultiQC [25] software (Supplementary Table S3). RB-TnSeq data analysis Barcodes were extracted from the fastq files and counted using the custom Python script (barcode_counting.py). Knowing the position of the barcode in a read, we extracted the barcode sequences from each read and checked whether these sequences were present in the annotation. Mismatches were not allowed, and the unmapped barcodes were discarded from the analysis. On average, 64.28% of the detected barcodes were successfully found in the annotation. The abundance of each barcode in each sample was then counted. Gene fitness was then calculated in R version 4.2.2 as described in Wetmore et al. [11] (script TnSeqW3110.R). Briefly, we first calculated the barcode fitness, defined as the log2 ratio of barcode counts between the treatment sample (after the incubation in the lake water) and the control sample (the initial culture). The values were normalized to the total number of all the barcodes found in the sample. Gene fitness was then calculated as a weighted average of the fitnesses of the barcodes belonging to the gene, with the weight being inversely proportional to the Poisson noise-based strain variance. The gene fitness values were then normalized for the chromosome to avoid the potential bias introduced during DNA extraction (i.e. having more molecules of DNA representing the ori region compared to the ter region). For this, the running median of the gene fitness values (window size 251) across the chromosome and the mode of the gene fitness values were subtracted from the gene fitness values. This normalization removes local biases and ensures that the gene fitness distribution is centred around zero. Corresponding formulas and detailed explanations can be found in Wetmore et al. [11]. Gene fitness values were then averaged across the technical replicates. Enrichment analysis was performed using the STRING database version 12.0 [26]. The same database was also used for network drawing. Measurement of gadB promoter activity Expression of GFP under the control of the gadB promoter was assayed using the reporter construct from the E. coli promoter collection [27] in 24-well plates using a plate reader. Briefly, E. coli cultures were grown overnight in TB supplemented with kanamycin (50 μg/ml) on a rotary shaker at 37°C. The cultures were diluted in fresh medium to an OD600 of 0.02 and added to the well plates in triplicates (1 ml per well). Measurements of OD600 and GFP fluorescence (excitation 485 nm, emission 520 nm) were performed on an Infinite M1000 Pro plate reader (Tecan, Männedorf, Switzerland) at 37°C, every 10 min, with alternating shaking modes: 150 s linear followed by 150 s orbital, repeated twice. All measurements were corrected for a no-GFP control and a no-growth control. The activity of the gadB promoter is then expressed in arbitrary units (AU) as the value of the GFP fluorescence divided by the 1000 × OD600 value. SYTOX Orange live-dead staining For staining with the SYTOX Orange dye (ThermoFisher, Waltham, MA), E. coli cells were grown in filtered lake water for 8 days as described above, then harvested and washed once with PBS. The dye was added to the cell suspension at a final concentration of 0.2 μM, and the mixture was incubated at 37°C for 15 min in the dark with vigorous shaking. After staining, the cells were washed three times with PBS. The percentage of the SYTOX Orange positive cells was quantified using the BD LSRFortessa SORP cell analyser (BD Biosciences) with the high-throughput system for 96-well plates in three biological triplicates (each in a technical triplicate), with a minimum of 30 000 cells measured per technical replicate. Scatter parameters for cell identification were measured using a 488-nm laser, and SYTOX Orange was excited using a 561-nm laser. To analyse the flow cytometry results, FlowJo software (BD Biosciences) version 10 was used. To assess the significance of the differences between the groups, two-sample Welch’s t-test was used. CFU quantification Samples containing bacteria were serially diluted 10-fold in PBS and then 100 μl of three dilutions were plated on LB agar in quadruplicate using glass beads. The dilutions (typically third to fifth) were selected based on flow cytometry results. After the overnight incubation at 37°C, the resulting colonies were counted using ImageJ software [28]. Plates containing <100 or more than 500 colonies were considered unreliable. The number of CFU per millilitre was determined using the formula \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{upgreek} \\\\usepackage{mathrsfs} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document} $CFU=\\\\frac{N\\\\ast{10}^d}{0.1}$\\\\end{document} where N is the number of colonies per plate and d is the dilution.',\n",
       " '11188080': 'Methods.In this study, we further detailed the multifaceted immunomodulatory properties of protolerogenic and proinflammatory bacterial species over time, using our clinically relevant model of allogenic heart transplantation.',\n",
       " '11188027': 'MATERIALS AND METHODS All materials and methods are shown in Supplemental Materials and Methods (SDC, http://links.lww.com/TP/C995).',\n",
       " '11188014': 'MethodsWe divided the mice into six groups: the blank control (PBS), pneumonia-derived sepsis (KP), pneumonia-derived sepsis + antibiotic (KP + PIP), pneumonia-derived sepsis + faecal microbiota transplantation(KP + FMT), antibiotic treatment control (KP+PIP+PBS), and pneumonia-derived sepsis+ antibiotic + faecal microbiota transplantation (KP + PIP + FMT) groups to compare the survival of mice, lung injury, inflammation response, airway barrier function and the intestinal flora, metabolites and drug resistance genes in each group.',\n",
       " '11187940': 'Methods Mice All mouse experiments were conducted in accordance with the guidelines of the Animal Care and Utilization Committee of Soochow University Institutions. C57BL/6 mice were purchased from Soochow University Laboratory Animal Center, Jiangsu Province, and were housed in Soochow University’s P2 Animal Room under standard care at 23–25°C with free access to food and water. All mice were males, 6–8 weeks old, and acclimatized for 1 week prior to the start of the experiments. Mouse models of K. pneumoniae infection In the present study, K. pneumoniae (ATCC 43816) was used to induce pneumonia-induced sepsis in mice via intranasal inhalation according to previous methods (Claushuis et al., 2018; Perlee et al., 2019; Qin et al., 2022). K. pneumoniae was purchased from the American Type Culture Collection (ATCC) official website of the United States and cultured in the etiology laboratory of Soochow University. The solution was stored at -80°C and removed for resuscitation, culture inoculation, and the determination of the bacterial concentration when needed. During the experiment, the mice were anaesthetized with 2% sodium pentobarbital (50 mg/kg), 1×104 CFU of K. pneumoniae was mixed with 50 µL of phosphate-buffered saline (PBS), and after the bacteria were instilled through the nose, the mice were placed vertically for 30 seconds to enhance delivery to the lungs. The control group received 50 µl of PBS, and all mice were euthanized at the end of the experiment. Animal experiment The mice were randomly divided into six groups of six mice each. The mice were divided into the following groups: the PBS group; the blank control group; the KP group, the pneumonia-derived sepsis group; the KP+PIP group; and the pneumonia-derived sepsis + antibiotic group. At 24 hours after nasal inhalation of Klebsiella pneumoniae, piperacillin (300 mg/kg) was injected subcutaneously, and the one-day dose was divided into two injections separated by at least 12 hours for 3 days (Trautmann et al., 1986; Rice et al., 2004). In the KP+FMT group and the pneumonia-derived sepsis + FMT group, 24 hours after nasal inhalation of Klebsiella pneumoniae, 0.3 ml of fecal microbiota was administered by gavage once a day for 3 days ( Supplementary Figure S1 ). In the KP+PIP+PBS group, three consecutive days of antibiotic treatment were administered, followed by three consecutive days of gavage with PBS solution at one-day intervals. In the KP + PIP + FMT group and the pneumonia-derived sepsis+ antibiotic + FMT group, first, a pre-experiment was carried out to observe the microbial structure and the production of drug resistance genes in the intestines of the KP+PIP group. The time at which the most obvious change was observed was selected as the time point for FMT following antibiotic administration; that is, on the second day of the three consecutive days of antibiotic administration (see Supplementary Figures S3 - S5 for the results of the pre-experiment), Klebsiella pneumoniae was injected subcutaneously with piperacillin (300 mg/kg) 24 hours after nasal inhalation modelling, with one day’s dose divided into two injections separated by an interval of at least 12 hours for three days. After an interval of one day, the FMT solution was given once a day at a rate of 0.3 ml for three days ( Supplementary Figure S2 ). The mice in each group were sacrificed at 12, 24 and 48 hours after the experiment to collect relevant samples. Fecal microbiota transplantation solution Fresh feces were collected from healthy C57BL/6 adult mice (6–8 weeks old), and 5 g of fresh feces was homogenized in 50 mL of sterile PBS (1:10 ratio) and centrifuged at 2000 rpm for 5 minutes to collect the fecal supernatant. Glycerol was added to a final concentration of 10%, after which the solution was stored at -80°C and removed when needed. The KP+FMT and KP+PIP+FMT groups were intragastrically administered 0.3 ml of the various substances at the indicated time points once a day for three consecutive days (Liu et al., 2020; Lou et al., 2022). Stool, lung tissue and alveolar lavage fluid At each time point, fresh feces were taken from the mice; the samples were subsequently placed in sterilized 1.5 ml EP tubes, quickly frozen in liquid nitrogen, and stored at -80°C. In accordance with previous experimental methods (Qin et al., 2022), after blood was taken from the eyeball, the mice were sacrificed by cervical dislocation, and the mice were subsequently fixed on the experimental operating table and opened via the abdomen. The thoracic cavity of the mice was opened, and the lungs were fully exposed and separated. After, the left bronchus and blood vessels were ligated by using surgical sutures, the left lungs were removed and preserved in 4% paraformaldehyde, and the right lung was freed. The skin on the neck was cut and bluntly separated to fully expose the trachea. A disposable mouse tracheal tube was inserted into the trachea, and a 1 ml syringe was passed through the tracheal tube. Afterwards, the tissue was injected with 0.5 ml of sterile PBS, and the solution was slowly extracted from the injected PBS in sterile EP tube on ice for preservation; this process was repeated twice, after which the right lungs of the mice were gently squeezed and the tissue was fully gavaged. The sampling was completed, and the samples were stored at -80°C until use. HE staining and pathologic analysis The left lungs of the mice were removed and placed in 4% paraformaldehyde for adequate fixation. The tissues were subsequently dehydrated, embedded, and sectioned in paraffin, and 4 µm sections were obtained. After dewaxing and hydration, HE staining was performed with hematoxylin and eosin, after which the sections were dehydrated and sealed under a microscope. Pathological analysis was performed according to Smith’s semiquantitative lung injury pathological scoring criteria as detailed in previous studies (Smith et al., 1997; Wang et al., 2023). Cytokine measurement The whole blood of mice was centrifuged to obtain serum, combined with alveolar lavage fluid obtained by lung lavage of mice. The concentration of TNF-α, IL-6, IL-10, IL-1β and IFN-γ in the samples was determined by double antibody sandwich ELISA according to the instructions on the 96-well enzyme-linked immunosorbent assay kit (J & L Biological, China). The serum and alveolar lavage fluid were diluted 5-fold, and added to the ELISA plate in order. After incubation, the prepared 1: 100 biotinylated antibody working solution and enzyme conjugate working solution were added in turn. After incubation and washing, the chromogenic substrate was added to the plate until it appeared blue, and then the termination solution was added to become yellow. The OD value was measured at the wavelength of 450 nm by microplate reader. The concentration of inflammatory factors in the corresponding samples was calculated by the standard curve drawn by the standard substance. Finally, the level of inflammatory factors in each group was obtained by multiplying the dilution multiple (5-fold). Immunohistochemistry The 4 µm paraffin-embedded lung tissue sections were dewaxed and hydrated, and antigen retrieval was carried out via the microwave heating repair method. After the samples were washed with PBS, endogenous catalase activity was blocked, the serum was sealed, the primary antibody was added after the sealing solution was removed and the samples were incubated overnight. Then, the secondary antibody corresponding to the primary antibody was added dropwise and the samples were incubated at room temperature. Finally, freshly prepared DAB color development solution was added dropwise, and the duration of color development was controlled under a microscope. After rinsing and restaining with hematoxylin, the slides were dehydrated, cleared and sealed for microscopic examination, after which the number of positive cells was counted. Immunofluorescence After dewaxing and hydration, the paraffin sections were placed in EDTA antigen retrieval solution (pH 9.0) for antigen retrieval via the microwave heating method. The sections were subsequently washed in PBS and incubated with 3% BSA for 30 min. Afterwards, the corresponding primary antibody was added, and the samples were incubated overnight at 4°C. After washing with PBS solution, the slides were incubated with secondary antibody for 50 minutes in the dark and then subjected to nuclear staining with DAPI, after which the autofluorescence quencher was added for 5 minutes. After rinsing with running water, the slices were blocked, and images were collected for analysis. Metagenomic sequencing With the Illumina HiSeq sequencing platform, a small fragment library was constructed for paired-end sequencing. Fecal samples stored at -80°C were subjected to DNA extraction using the Fast DNA Spin Kit for Feces (MPBIO, CA, USA) according to the manufacturer’s instructions. The qualified DNA samples were randomly broken into small fragments of approximately 300 bp using a Covaris M220 ultrasonic crusher, after which library preparation was completed. The constructed library was initially quantified by Qubit and diluted to 2 ng/µl. Then, an Agilent 2100 was used to analyze the inserted fragments in the library. After the insertion fragments met expectations, the effective concentration of the library was accurately determined via qPCR. Finally, different libraries were pooled according to the effective concentration and target downstream data volume and sequenced on the sequencing platform. After obtaining the non-redundant gene set through the metagenomic sequencing, the gene sequence was annotated with the drug resistance gene database SARG to obtain the corresponding drug resistance gene type. Then, combined with the gene abundance table, the abundance of drug resistance genes in each sample was obtained. Finally, using statistical test methods, the types of drug resistance genes between groups can be obtained. Statistical analysis Statistical analyses were performed using QIIME1, R language software and GraphPad Prism 8.0.1. The mean ± standard deviation was used to assess quantitative data, and t tests or two-way analysis of variance (ANOVA) and Bonferroni post hoc correction were used to analyze the data instead of individual comparisons. The Chao1 and Shannon indices were utilized to assess the α diversity using Mothur. Then, based on the Bray−Curtis algorithm, QIIME was used to calculate the distance between each sample to obtain the Bray−Curtis distance matrix. According to the distance matrix, hierarchical clustering analysis was performed, and the unweighted group average algorithm was subsequently used to construct a tree structure for visual analysis. Based on the calculated inter sample distances, principal component analysis (PCOA) was performed using the R package vegan to visualize the differences in intestinal microecological β diversity among the samples. The probability relationships between the grouping categories and the actual distributions of the samples were inferred from the sample distributions to analyze the overall differences between the two groups of samples at the species level. LEfSe was used to analyze the species that differed between the groups, and the Spearman rank correlation coefficient was calculated for correlation analysis. All the results between groups were analyzed using a statistical significance level of P < 0.05.',\n",
       " '11187919': 'Materials and methods Study design and participants We conducted a longitudinal study nested within an open label, two-arm parallel group, randomized controlled trial (RCT). The RCT compared a two-dose (control arm) and three-dose (intervention arm) Rotarix™ vaccination schedule among Zambian infants. The details of the study design have been published elsewhere [21]. Briefly, 214 infants aged 6–12 weeks were enrolled in the parent RCT and followed up until they were 3 years of age between 2018 and 2021. During the first year of follow-up, all infants were given two doses of an ORV (Rotarix, GlaxoSmithKline) with the first dose administered from 6 weeks old and the second dose administered from 10 weeks old, along with polio, BCG, DPT-HepB-Hib and pneumococcal conjugate vaccines as part of the regular Zambia national immunization schedule. When the infants reached 9 months of age, they were randomly assigned to either a control arm (receiving only a measles-rubella (MR) vaccination) or an intervention arm (receiving MR vaccine and a third dose of Rotarix). Plasma samples were collected at specific intervals: at enrollment (baseline, aged 6-12 weeks) before the first Rotarix dose, 1 month after the second Rotarix dose (aged 14-20 weeks, when vaccine seroconversion was determined), at 9 months (before receipt of the third Rotarix dose and/or MR vaccine), and at 12 months (when the immune-boosting effect of third Rotarix dose was assessed; Fig. 1). These plasma samples were tested for rotavirus-specific immunoglobulin A (RV-IgA) antibodies. RV-IgA seropositivity was defined as an RV-IgA titer ≥ 20 units/ml. Vaccine seroconversion was defined as a 4-fold or greater change in RV-IgA antibody titer 1 month after dose two of Rotarix if pre-vaccination titer was less than 20 U/ml [21]. Figure 1. Study design. An illustration of the parent RCT study design and plasma collection timepoints. For this nested study, infants with available RV-IgA results and sufficient plasma sample to test for HCMV-IgM antibodies at baseline and at least one of the three subsequent time points up to 12 months of age were included. The design thus facilitated the examination of the association between HCMV infection and ORV immunogenicity by focusing on multiple factors and time points, enabling a comprehensive analysis of the relationship. Laboratory procedures Determination of HCMV serostatus: HCMV-IgM antibodies were measured using enzyme-linked immunosorbent assay (ELISA) kits from Demeditec Diagnostics GmbH (Germany) and Alpha Diagnostic International (USA). The procedure followed the manufacturer’s guidelines. Infant plasma samples were diluted and added to a 96-well microtiter plate that had been precoated with a purified HCMV antigen. To detect any HCMV-IgM in these samples, they were treated with horse-radish peroxidase (HRP)-conjugated anti-human IgM. Subsequently, the tetramethylbenzidine (TMB) substrate was added, initiating an enzyme-substrate hydrolysis reaction, which resulted in color development. The color’s absorbance was immediately measured at a 450-nm wavelength using an Epoch 2 microplate reader by Agilent (South Africa). Proper quality control of each experiment was ensured using the calibrators and controls provided in the kit. To check consistency in results, selected HCMV seropositive samples were tested with both kits. The test outcomes for HCMV-IgM serostatus in the plasma samples were categorized as either positive or negative based on specific cut-off control values for each experiment. Quantification of rotavirus-specific immunoglobulin A: The measurement of rotavirus-specific immunoglobulin A (RV-IgA) was carried out using a sandwich ELISA method, as outlined in the parent rotavirus vaccine trial [21]. In the procedure, infant plasma samples were placed on a 96-well microtitre plate, which was coated with alternating columns of rotavirus infected and uninfected cell lysate. To detect the RV-IgA, the samples underwent a subsequent treatment with biotinylated anti-human IgA and an avidin-biotin-peroxidase complex. The addition of the o-phenylenediamine dihydrochloride substrate initiated a color change, whose intensity was measured at a 492-nm wavelength with a microplate reader. The concentration of RV-IgA was determined based on these readings, compared against a standard curve created from a known rotavirus IgA plasma standard. Statistical analysis Background characteristics were summarized with mean and standard deviation (SD) or median and interquartile range (IQR) for continuous variables. Categorical variables were summarized using frequency and proportion. Pearson’s chi-square or Fisher’s exact test was used to compare the distribution of categorical background characteristics by HCMV-IgM serostatus at 9 months and 4-fold change in RV-IgA titers between 9 and 12 months. For RV-IgA titers at 12 months, we used Student t-test on log-transformed values. For anthropometric indices, we calculated z-scores using the 2006 World Health Organization child growth standards. The exposure of interest was HCMV-IgM serostatus at 9 months and the primary outcome was RV-IgA titers at 12 months. Secondary outcome was proportion with 4-fold-change in RV-IgA titers between 9 and 12 months. The primary analysis was conducted among infants that had HCMV serostatus result at the 9-month time point and RV-IgA titer result at both the 9- and 12-month timepoints. We used linear regression model of log-transformed RV-IgA titers to estimate the effect of HCMV-IgM serostatus at 9 months or cumulative HCMV-IgM seroconversion by 9 months on RV-IgA GMT at 12 months, adjusting for potential confounders. P-value of <0.05 was considered statistically significant. The RV-IgA titer below the range of the standard curve were imputed as “1” prior to log-transformation. We used generalized linear model, adjusted for potential confounders, to estimate the effect of HCMV-IgM serostatus at 9 months or cumulative HCMV-IgM seroconversion by 9 months on the proportion with a 4-fold or greater change in RV-IgA titers between 9 and 12 months. In a subgroup analysis, we used likelihood ratio test of interaction to investigate whether the effect of HCMV-IgM serostatus at 9 months on RV-IgA titer at 12 months varied by two-dose versus three-dose vaccination or by infant human immunodeficiency virus (HIV) exposure. In exploratory analyses, we examined the proportion of infants testing seropositive for HCMV-IgM for each time point and the relationship between HCMV-IgM point seropositivity or cumulative HCMV-IgM seroconversion and vaccine seroconversion after two dose vaccination. All analyses were performed in Stata 17 (StataCorp, College Station, TX, USA) and GraphPad Prism v9 (GraphPad Software, LLC).',\n",
       " '11187847': 'MethodsData from the National Health and Nutrition Examination Survey 2017–2018 on U.S. adults (≥20 years) were analyzed to explore the association between phthalate metabolites and gallstones by employed survey-weighted logistic regression, restricted cubic spline (RCS), weighted quantile sum (WQS) regression, and Bayesian kernel machine regression (BKMR). Mediation analyses examined the role of oxidative stress markers, inflammatory markers, metabolic syndrome, body composition, diabetes, and insulin.',\n",
       " '11187845': 'Methods Data sources The NHANES is a series of surveys conducted by the National Center for Health Statistics (NCHS), employing a sophisticated multistage probability cluster design to assess the health and nutritional well-being of a representative sample from the non-institutionalized US population. Comprehensive details are accessible in the NHANES survey methods and analytic guidelines (23). Extracted data originated from the 2017–2018 NHANES cycles, specifically focusing on participants aged 20 years and above. The data underwent analysis between September and December 2023. Ethical approval for NHANES procedures and protocols was obtained from the NCHS Research Ethics Review Board, and all participants provided written informed consent. The study followed the reporting guidelines set forth by the Strengthening the Reporting of Observational Studies in Epidemiology. Study design and population The current study included 2,986 participants from the NHANES 2017–2018 cycle who underwent urinary phthalate metabolite measurements. Exclusion criteria were applied post-data collection, removing participants <20 years old (n = 1,221), those with incomplete data on urinary phthalates metabolite and gallstones (n = 65), and individuals lacking demographic information (n = 230), including age, sex, race/ethnicity, marital status, poverty income ratio (PIR), and educational level. Additional exclusions were made for those without data on physical activity (PA), drinking status, smoking status, body mass index (BMI), and urinary creatinine (n = 86). The final dataset for analysis comprised 1,384 participants (Supplementary Figure S1). Assessment of gallstones In NHANES, the presence of gallstones was evaluated through the question by asking people over the age of 20, “Has a doctor or other health professional ever told you that you had gallstones?.” A response of “yes” to this question was considered indicative of gallstones, while an exact response of “no” was categorized as the absence of gallstones, as previously mentioned in the literature (24). The occurrence of gallstones served as the outcome variable for analysis. Phthalates metabolite measurement This study investigated 19 urinary phthalate metabolites, as outlined in Supplementary Table S1, including Mono (3-carboxypropyl) phthalate (MCPP), Mono-ethyl phthalate (MEP), Mono-isobutyl phthalate (MiBP), Mono-2-hydroxy-iso-butyl phthalate (MHiBP), Mono-n-butyl phthalate (MBP), Mono-3-hydroxybutyl phthalate (MHBP), Monobenzyl phthalate (MBzP), Mono(2-ethylhexyl) phthalate (MEHP), Mono(2-ethyl-5-carboxypentyl) phthalate (MECPP), Mono(2-ethyl-5-hydroxyhexyl) phthalate (MEHHP), Mono(2-ethyl-5-oxohexyl) phthalate (MEOHP), Mono-isononyl phthalate (MNP), Monocarboxyisooctyl phthalate (MCOP), Mono-oxo-isononyl phthalate (MONP), Monocarboxy-isononyl phthalate (MCNP), Cyclohexane-1,2-dicarboxylic acid mono(hydroxy-isononyl) ester (MHINCH), Cyclohexane-1,2-dicarboxylic acid mono(carboxyoctyl) ester (MCOCH), Mono(2-ethyl-5-hydroxyhexyl) terephthalate (MEHHTP), and Mono(2-ethyl-5-carboxypentyl) terephthalate (MECPTP). Among them, MNP is excluded from the analysis due to over 80.07% of participants having concentrations below the limit of detection (LOD) to mitigate potential bias. The LOD and molecular weight (MW) of each metabolite and its corresponding parent compound are shown in Supplementary Table S1. The quantification of phthalate metabolites in urine employed a high-performance liquid chromatography-electrospray ionization-tandem mass spectrometry (HPLC-ESI-MS/MS) method (25). This process involves enzymatic deconjugation of glucuronidated phthalate monoesters in urine samples, followed by on-line solid phase extraction (SPE) coupled with reversed phase HPLC-ESI-MS/MS. The use of isotopically-labeled internal standards for the phthalate metabolites improves assay precision. In addition, 4-methyl umbelliferone glucuro- nide is employed to monitor deconjugation efficiency. This method enables the rapid and selective detection of monoester metabolites of commonly used phthalate diesters in human urine, with LOD in the low ng/mL range. Concentrations below the LOD were replaced using the LOD/√2 method (26). Sum phthalate metabolites were calculated to assess the influence of the parent chemical, including Di(2-ethylhexyl) phthalate (ΣDEHP, expressed as MEHP, MW = 278.2 g/mol), Di-isobutyl phthalate (ΣDiBP, expressed as MiBP, MW = 222.2 g/mol), Di-n-butyl phthalate (ΣDBP, expressed as MBP, MW = 222.2 g/mol), Di-isononyl phthalate (ΣDNP, expressed as MONP, MW = 306.4 g/mol), 1,2-Cyclohexane dicarboxylic acid diisononyl ester (ΣDINCH, expressed as MHINCH, MW = 314.4 g/mol), and Di(2-ethylhexyl) terephthalate (ΣDEHTP, expressed as MEHP, MW = 294.3 g/mol). High MW (> 250) phthalate (high-MWP) is the molar sum of MCPP, MBzP, MHiBP, MEHP, MECPP, MEHHP, MEOHP, MCOP, MONP, MCNP, MHINCH, MCOCH, MEHHTP, and MECPTP (expressed as MCPP, MW = 252.2). Low MW (< 250) phthalate (low-MWP) is the molar sum of MEP, MiBP, MHiBP, MBP, and MHBP (expressed as MEP, MW = 194.2) (27). Covariates Several potential confounding variables were considered based on published research and clinical judgment, including age, sex, marital status, race/ethnicity, education level, PIR, BMI, PA, smoking status, drinking status, and creatinine (28). Age, PIR, BMI, PA, and creatinine were included in the adjustment model as continuous variables. In the subgroup analysis, age was categorized as <40, 40–59, and ≥ 60 when utilized as an exposure variable or considered in subgroup analyses. Sex was categorized into female and male. Marital status was grouped into married, never married, living with a partner, and other (e.g., widowed, divorced, or separated). Self-reported race/ethnicity was categorized as Mexican American, non-Hispanic White, non-Hispanic Black, other Hispanic, and others (including multi-racial participants). Educational level was classified as less than high school, high school or equivalent, and above high school. BMI was calculated as the weight (in kilograms) divided by the square of the height (in meters). PA was defined as the time individuals reported spending during the week on activities such as walking or biking, tasks around the home or yard, work, and recreational activity (29). Smoking status was characterized as never (<100 cigarettes smoked in life), former (>100 cigarettes smoked but currently quit), and now (>100 cigarettes smoked and currently smoking) (30). Drinking status was categorized as never (<12 drinks lifetime), former (<12 drinks lifetime or none in the past year), mild (≤1 drink per day for females and ≤ 2 drinks per day for males in the past year), moderate (≤2 drinks per day for females and ≤ 3 drinks per day for males in the past year), and heavy (≥3 drinks per day for females and ≥ 4 drinks per day for males in the past year) (31). Creatinine was measured from the laboratory. Statistical analysis Complex sampling design and environmental subsample weights were considered in our analyses. Participant characteristics were calculated based on the presence or absence of gallstone, with categorical variables presented as numbers and percentages, and continuous variables as means and standard error (SE). Differences across groups were analyzed using the chi-squared test with Rao & Scott’s second-order correction (for categorical variables) and the Wilcoxon rank-sum test (for continuous variables). The concentrations of individual and sums of phthalate metabolite were naturally log-transformed to achieve a normal distribution, and they were divided into three groups (Tertile1-Tertile3) according to percentile. Survey-weighted multivariable logistic regression was performed to assess associations of individuals and sums phthalate metabolite (treated as both a categorical and continuous variable) with gallstone. The crude model did not account for covariates, while the adjusted model accounted for age, sex, race/ethnicity, PIR, marital status, education level, BMI, PA, smoking status, drinking status, and creatinine. Possible nonlinear effects were modeled using restricted cubic spline (RCS) models with 3 knots at 10, 50, and 90%. Further to calculating the sum of phthalate metabolites based on MW, we employed both weighted quantile sum (WQS) regression and Bayesian kernel machine regression (BKMR) to analyze the joint effects of phthalate metabolites. The WQS regression model built a weighted index to estimate the combined effects of all predictive factors related to the outcomes, demonstrating high sensitivity and specificity in identifying significant exposures (32). Bootstrapping with 1,000 iterations was employed to construct the WQS index as a representative measure of mixed exposure levels to phthalates, and the contribution weights of each compound to the overall impact of the mixture were calculated. The dataset was randomly divided, with 40% allocated to the training set and the remaining 60% used as the validation set. The WQS model estimated the incremental risk of gallstones corresponding to a 25% (one-quarter) increase in the WQS index. BKMR is an approach that models the health effects of complex chemical mixtures, using flexible functions and Bayesian methods to identify important mixture components and account for correlations, particularly in high-dimensional settings (33). After adjusting for all covariates, this model underwent 10,000 iterations using the Markov Chain Monte Carlo algorithm. Posterior incorporation probabilities (PIPs), spanning from 0 to 1, were derived from the BKMR model to ascertain the relative significance of each exposure concerning the outcome, with a defined threshold of 0.5. In additionally analysis, we further explored potential mediators of the association between phthalate exposure and gallstone, including oxidative stress markers (including bilirubin, uric acid, and gamma glutamyl transferase) (34), inflammatory markers (including C-reactive protein, alkaline phosphatase, neutrophils, and ferritin) (35), metabolic syndrome (36), body composition [including BMI, waist circumference, visceral adiposity index (37), lean/fat mass measured using dual-energy X-ray absorptiometry (38)], diabetes and insulin resistance (including fasting blood glucose [FBG], fasting serum insulin [FINS], triglyceride glucose index, and insulin resistance [HOMA-IR], insulin sensitivity [HOMA-IS], and β-cell function [HOMA-β] assessed through the homeostasis model assessment [HOMA]) (39, 40). Mediation analyses were conducted by using the Sobel test, Bootstrap, and the quasi-Bayesian Monte Carlo method with 1,000 simulations based on normal approximation (41, 42). Stratified analyses were performed based on age (<40, 40–59, or ≥ 60 years), sex (female or male), race/ethnicity (Mexican American, non-Hispanic Black, non-Hispanic White, other Hispanic, or other), and BMI (<25, 25–29, ≥30). The interaction was assessed by adding a cross-product term to the regression model and calculating the p value using the likelihood ratio test, comparing models with and without the interaction term. All analyses were conducted with R (4.2.3) and Free Software Foundation statistics software (version 1.9.2). Statistical significance was determined by two-sided p values below 0.05.',\n",
       " '11187777': 'Materials and methods Fly stocks Two natural Canton-S lines of D. melanogaster containing or lacking Wolbachia pipientis (strain wMel) were used in all experiments. Flies were reared on ready-made fly food (LabExpress, Ann Arbor, MI, United States) supplemented with yeast (Carolina Biological Supply, Burlington, NC, United States). All vials were maintained in an incubator at 25°C and 12 h light/12 h dark photoperiod cycle. Both D. melanogaster lines were amplified for experimentation by transferring adult flies to fresh vials every third day. Female and male flies were selected from the same generation and randomly assigned to experimental groups. Zika virus stocks Stocks of Zika virus strain MR766 were maintained and amplified as described before (Harsh et al., 2018). Fly longevity experiments Ten male and 10 female Canton-S newly eclosed D. melanogaster adult flies carrying or lacking Wolbachia endosymbionts, were kept in separate vials containing fly food at 25°C and a 12:12-h light:dark photoperiod cycle. Observations were held at 24-h intervals to record fly survival over a 100-day period. All flies were transferred to fresh vials every 3 days. The experiment was repeated five times with three replicates for each experimental treatment. In total, 150 female and 150 male flies were used in the longevity experiments. Fly survival experiments Ten female and 10 male 3–5 days old flies of the two Canton-S lines carrying or lacking Wolbachia endosymbionts were first anesthetized with carbon dioxide for a few seconds and then injected into the thorax with 100 nL of a Zika virus solution (11,000 PFU/fly) in PBS (pH 7.5) using a Nanojector apparatus (Nanoject III Programmable Nanoliter Injector, Drummond Scientific, Broomall, PA, United States). Flies injected with 100 nL of PBS served as negative controls. Virus-infected and control flies were maintained in vials with fly food at 25°C and they were transferred to fresh vials every third day for the duration of the survival experiment. Survival results were estimated at 24-h intervals for up to 25 days. Three independent experiments were conducted, and each experiment included three replicates for each experimental condition. Gene expression analysis Ten female and 10 male Canton-S 3–5 days old flies containing or lacking Wolbachia were injected with either Zika virus or PBS only (negative control), as described above, and at 4 days post-injection were collected and stored at −80°C. A pool of 10 flies was homogenized using plastic pestles and RNA isolation was carried out using TRIzol (Invitrogen, Waltham, MA, United States) according to the manufacturer’s protocol, followed by complementary DNA (cDNA) synthesis using the AB High-Capacity cDNA Reverse Transcription Kit (Fisher Scientific, Hampton, NH). Zika virus load and immune gene expression were estimated using Zika virus gene-specific primers and D. melanogaster gene-specific primers, respectively (Table 1). Gene expression analysis was assessed using quantitative Real Time PCR (qRT-PCR) and two technical replicates per treatment on a CFX96 Real-Time PCR detection system (Bio-Rad Laboratories, Hercules, CA). Cycle conditions were 95°C for 2 min, 40 repetitions of 95°C for 15 s, 61°C for 30 s, 95°C for 15 s, 65°C for 5 s, and 95°C for 5 s. Quantitative RT-PCR experiments were repeated three times with each experiment consisting of three biological replicates and two technical replicates per condition. In total, 360 female and 360 male flies were used in the gene expression analysis experiments. Table 1 Primer sequences used for quantitative PCR. Gene Accession number Pathway Forward primer (5′ to 3′) Reverse primer (3′ to 5′) Ago-2 CG7439 RNAi CCGGAAGTGACTGTGACAGATCG CCTCCACGCACTGCATTGCTCG Basket CG5680 Jnk GACAGCTCAGCACCAACACT GCTTGGCATGGGTTACATTT Defensin CG1385 Toll CGCATAGAAGCGAGCCACATG CGCATAGAAGCGAGCCACATG Drosocin CG10816 Imd TTCACCATCGTTTTCCTGCT AGCTTGAGCCAGGTGATCCT Dicer-2 CG6493 RNAi GTATGGCGATAGTGTGACTGCGAC GCAGCTTGTTCCGCAGCAATATAGC Diptericin CG10794 Imd TGCGCAATCGCTTCTAC GTGGAGTGGGCTTCATG Drosomycin CG10810 Toll TGAGAACCTTTTCCAATATGATG CCAGGACCACCAGCAT Puckered CG7850 Jnk GGCCTACAAGCTGGTGAAAG AGTTCAGATTGGGCGAGATG TotA CG31509 Jak/Stat GAAGATCGTGAGGCTGACAAC GTCCTGGGCGTTTTTGATAA TotM CG14027 Jak/Stat GCTGGGAAAGGTAAATGCTG AGGCGCTGTTTTTCTGTGAC RpL32 CG7939 – GATGACCATCCGCCCAGCA CGGACCGACAGCTGCTTGGC ZIKV NS5 055839 – CCTTGGATTCTTGAACGAGGA AGAGCTTCATTCTCCAGATCAA Statistical analysis All data were processed using the GraphPad Prism5 software. Results from the longevity and survival experiments were statistically analyzed using a log-rank (Mantel-Cox) test and a chi-square test for pairwise comparison between each experimental group and the control group. Results from the gene expression analyses were conducted using a one-way analysis of variance (ANOVA) as well as a Tukey post-hoc test. Fold changes were estimated using the 2-ΔΔCT method and Ribosomal protein L32 (RpL32) as the housekeeping gene (Livak and Schmittgen, 2001; Schmittgen and Livak, 2008). PBS control treatments were used as a baseline for comparison in differential expression analysis. p-values less than 0.05 were considered significantly different. All error bars represent standard error of mean and all figures were generated using the GraphPad Prism5 software.',\n",
       " '11187773': '2 Materials and methods 2.1 Chemicals and reagents Commercial E-Jiao was obtained from Gansu Tianshui Xihuang E-Jiao Co., Ltd. (Batch No. 17101022, SFDA Approval Number: Z62021196, Tianshui, Gansu, China). The E-Jiao was prepared using donkey hides obtained from domesticated donkeys and was in accordance with the quality standard of the Chinese Pharmacopoeia (2020 edition) (Committee Chinese Pharmacopoeia, 2020). Commercial Danggui Buxue Koufuye was purchased from Zhengzhou Union Pharmaceutical Factory (Batch No. 012303161, SFDA Approval Number: Z10970001, Zhengzhou, Henan, China). The Danggui Buxue Koufuye is a liquid preparation obtained by extracting Astragali Radix (Astragalus membranaceus (Fisch.) Bge.var.mongholicus (Bge.) Hsiao or Astragalus membranaceus (Fisch.) Bge.) and Angelicae Sinensis Radix (Angelica sinensis (Oliv.) Diels) according to the method of Chinese Pharmacopoeia (2020 edition) and meets the quality criteria specified in the Chinese Pharmacopoeia (2020 edition) (Committee Chinese Pharmacopoeia, 2020). Commercial Leucogen tablets were supplied by Jebel Pharmaceuticals (Batch No. 221110, SFDA Approval Number: H32025444, Zhenjiang, Jiangsu, China). Porcine pepsin (1:15,000) and porcine trypsin (≥2500 units/mg) were purchased from Aladdin Chemistry Co., Ltd. (Shanghai, China). Vinorelbine tartrate, cyclophosphamide and Sudan Black B were purchased from Macklin, Inc. (Shanghai, China). Doxorubicin was obtained from Meilunbio Biochemical Co., Ltd. (Dalian, China), while o-dianisidine was purchased from TCL (Shanghai, China). Peptides (≥98% pure) were synthesized by China Peptides (Shanghai, China). MS-grade formic acid and acetonitrile were purchased from Thermo Fisher Scientific (Waltham, MA, United States). Ultrapure deionized water was obtained from a Milli-Q water purification system (MA, United States). 2.2 In vitro preparation and characterization of the peptides from E-Jiao 2.2.1 In vitro gastrointestinal digestion The in vitro simulated gastroduodenal digestion of E-Jiao was performed according to a previous method, with slight modifications (Minekus et al., 2014). E-Jiao was dissolved in deionized water at a concentration of 50 mg/mL, and the resulting solution was centrifuged for 5 min at 10,000 × g at 4°C. For the gastric phase, the pH of the E-Jiao solution was adjusted to 2.0 using 1 mol/L HCl, and subsequently, pepsin was added at an enzyme-to-substrate ratio of 1:250 (w/w). The mixture was incubated at 37°C in a shaking incubator for 3 h. The gastric phase was terminated by adjusting the pH to 7.0 with 1 mol/L NaOH. Moving to the duodenal phase, the solution was incubated with trypsin at an enzyme-to-substrate ratio of 1:250 (w/w) at 37°C for 2 h. At the end of the duodenal phase, trypsin was inactivated by heating the digests at 50°C for 10 min. Subsequently, the digests were centrifuged at 10,000 × g for 10 min. The fraction of the digest with a molecular weight less than 10 kDa, representing LMEJ, was obtained by successively centrifuging the supernatants using an Amicon Ultra 15 mL 30 K centrifugal filter (Millipore, Gloucestershire, United Kingdom) and an Amicon Ultra 15 mL 10 K centrifugal filter (Millipore, Gloucestershire, UK), both at 3500 × g for 30 min. The resulting ultrafiltrate was desalted by dialysis using a membrane with a 100 Da molecular weight cutoff (Yuan Ye Bio-Technology Co., Ltd., Shanghai, China). Subsequently, the ultrafiltrate was lyophilized and stored at −20°C. The entire in vitro digestion experiment was performed independently four times. 2.2.2 Peptide analysis LMEJ was characterized by liquid chromatography–tandem mass spectrometry (LC‒MS/MS) analysis, which was carried out on a Q Exactive Plus Orbitrap mass spectrometer equipped with an EASY-nLC 1200 HPLC system (Thermo Fisher Scientific, Waltham, MA, United States). Before analysis, the samples were dissolved in 0.1% formic acid in water at a concentration of 1 μg/μL, loaded onto a trap column (Acclaim PepMap 100 C18, 5 μm, 100 μm × 20 mm; Thermo Fisher Scientific, San Jose, CA, United States) and subsequently separated with an analytical column (Acclaim PepMap 100 C18, 2 μm, 50 μm × 150 mm). The mobile phase consisted of two solutions: eluent A was MS-grade water with 0.1% formic acid (v/v), and eluent B was acetonitrile with 0.1% formic acid (v/v). The flow rate was set at 300 nL/min. The gradient profile was as follows: 0–51 min, 4%–28% B; 51–56 min, 28%–38% B; 56–60 min, 38%–90% B; and 60–70 min, 90% B. The MS settings were as follows: spray voltage of 2.2 kV in positive mode. For full MS data acquisition, the resolution was set to 70,000 with a scan range of 350–1500 m/z, the AGC target was set to 1 × 106, and the maximum IT was 50 ms. For dd-MS2, the AGC target was 5 × 106, and the maximum IT was 120 ms. The 10 most intense precursors were selected for fragmentation with a resolution of 17,500 and a dynamic exclusion of 12 s. Ions with 1, 7, 8 and 9 charges were excluded from MS/MS selection. The raw LC–MS/MS data files were analyzed using Proteome Discoverer software (version 2.2; Thermo Fisher Scientific, United States). Due to the incompleteness of the UniProt E. asinus database, the UniProt Equus caballus database was also used for the search. Considering the enzymes used in digestion, reference sequences of porcine pepsin (Entry number: P00791) and porcine trypsin (Entry number: P00761) were also included in the database search to exclude peptides originating from these enzymes. The search parameters were set as follows: mass analyzer, Orbitrap mass spectrometer; enzyme, not specific; minimum peptide length, 6; precursor mass tolerance, 10 ppm; fragment mass tolerance, 0.02 Da; and dynamic modification, oxidation/+15.995 Da (M); additionally, no static modification was applied. The significance threshold was set at p < 0.05. The data was filtered using Percolator with a false discovery rate of ≤0.01. 2.3 Zebrafish experiments 2.3.1 Zebrafish maintenance Wild-type adult zebrafish and transgenic Tg (lyz:DsRED2) adult zebrafish, which express red fluorescent protein in their neutrophils, were obtained from the China Zebrafish Resource Center (Wuhan, China) and raised in the zebrafish facility at the College of Pharmacy of Jinan University. All experiments were conducted following the guidelines of the Jinan University Animal Care and Use Committee. Embryos obtained from paired mating were collected and maintained in egg water at a temperature of 28.5°C as described in the Zebrafish Book (Westerfield, 2000). 2.3.2 Effect of LMEJ and E-Jiao on zebrafish At 24-hour-postfertilization (hpf), 20 wild-type zebrafish embryos were randomly distributed into each well of 24-well plates after the egg membrane was removed using 0.5 mg/mL chain protease. The groups were assigned as follows: blank control (Control), model (Model), positive controls (Danggui Buxue Koufuye (DGBX) and leucogen tablet (LXS)), E-Jiao (E-Jiao), low-dose LMEJ (LMEJ-L), medium-dose LMEJ (LMEJ-M) and high-dose LMEJ (LMEJ-H). The Control group was treated with embryo medium, while the Model group was treated with 10 μg/mL doxorubicin in embryo medium. The DGBX, LXS, E-Jiao, LMEJ-L, LMEJ-M and LMEJ-H groups were treated with 10 μg/mL doxorubicin combined with 4.5 mL/L Danggui Buxue Koufuye, 15 μg/mL leucogen, 100 μg/mL E-Jiao, 25 μg/mL LMEJ, 50 μg/mL LMEJ and 100 μg/mL LMEJ, respectively. Each group had three replicates. The embryos were exposed for 24 h. To avoid melanin interference, the embryos were treated with 0.045 g/L 1-phenyl-2-thiourea at 12 hpf to inhibit melanin synthesis. The erythrocyte counts in the zebrafish embryos were evaluated with o-dianisidine staining performed according to previous methods (Yang et al., 2015). After staining, the embryos were photographed using a biomicroscope (Olympus, Japan). The erythrocyte count in the yolk sac of zebrafish larvae was quantified based on the staining intensity of the erythrocytes using ImageJ (version 5.1). At 48 hpf, Tg (lyz:DsRED2) zebrafish embryos were grouped in the same way as wild-type zebrafish embryos but treated with 25 μg/mL vinorelbine tartrate instead of doxorubicin. After 24 h of exposure, images of each Tg (lyz:DsRED2) zebrafish embryo were acquired and collected using an MZ10F fluorescence stereomicroscope (Leica, Germany). The neutrophils in the cloaca-to-tail region of zebrafish were also quantified using ImageJ (version 5.1). 2.4 Mouse experiments 2.4.1 Animals and treatments Sixty-four specific pathogen-free (SPF) Balb/c mice (6 weeks old weighing 16–18 g) were obtained from Charles River (Beijing, China). The mice were kept under standard conditions (temperature, 22°C ± 2°C; humidity, 55% ± 5%) at the Laboratory Animal Center of South China University of Technology (SCUT). All animal procedures were conducted following the Guidelines for the Care and Use of Laboratory Animals of SCUT and were approved by the Laboratory Animal Ethics Committee of SCUT (approval number: 2023094). The mice were randomly assigned to eight groups, which were the same as those described for the zebrafish experiments. To establish a myelosuppressed mouse model, the mice in all groups except the Control group were intraperitoneally injected with 100 mg/kg body weight cyclophosphamide once a day on days 1, 2, 3, and 6. According to the ratio of the clinical daily dose in humans to the human-mouse body surface area ratio, the daily intragastric doses of Danggui Buxue Koufuye (20 mL/d for humans) and leucogen tablets (60 mg/d for humans) were determined to be 2.6 mL/kg and 7.8 mg/kg, respectively. The mice in the E-Jiao group were orally administered a solution of E-Jiao (6 g/d for humans) at a dose of 1.56 g/kg, which is equivalent to double the clinical application dose. Considering that LMEJ was obtained in a 50% yield after in vitro simulated gastroduodenal digestion of E-Jiao, the mice in the LMEJ-L group received a daily oral dosage of 0.39 g/kg LMEJ, which is equivalent to the clinical application dose. The LMEJ-M and LMEJ-H groups received LMEJ at dosages of 0.78 g/kg daily and 1.56 g/kg daily, respectively. The Control and Model groups were orally administered the same volume of ultrapure water daily. The body weights of the mice were recorded daily until the termination of the experiment. 2.4.2 Visceral indices and routine blood test The mice were anesthetized via isoflurane inhalation and blood was collected via the eyeball removal method on day 11. After blood collection, the mice were immediately euthanized by cervical dislocation. The thymus and spleen were promptly excised, rinsed with water, dried on filter paper and weighed. Each organ index was calculated as the weight of the spleen (or thymus) relative to the mouse’s body weight (g/kg). Whole blood was collected into a 1.5 mL tube containing EDTA-2Na (Solaibao Biological Technology Co., Ltd., Beijing, China) for hematological tests, which included white blood cell (WBC), neutrophil (NE), lymphocyte (LY), monocyte (MO), eosinophil (EO), basophil (BA), red blood cell (RBC) and platelet (PLT) counts. These tests were carried out using a Hemavet 950 blood analyzer (Drew Scientific Group, United Kingdom) following the manufacturer’s instructions. All blood samples were analyzed within 2 hours after collection. 2.5 Rat experiments 2.5.1 Animals and treatments Ten eight-week-old male Wistar rats (SPF, Charles River, Guangzhou, China; 280–320 g) were purchased from Charles River (Beijing, China) and housed at the Laboratory Animal Center of SCUT under controlled conditions (temperature, 22°C ± 1°C; humidity, 55% ± 5%). After acclimatization for 1 week, the rats were fasted for 24 h before receiving E-Jiao. E-Jiao was dissolved in ultrapure water and administered orally to each rat at a dose of 0.7 g/kg. Blood samples were collected as previously described with minor modifications (Sheng et al., 2021). The rat was anesthetized with isoflurane and approximately 300 μL of blood was collected from the jugular vein before drug administration and at 1 h and 3 h after administration. Blood was collected in a 1.5 mL tube containing EDTA-2Na (Solaibao Biological Technology Co., Ltd., Beijing, China) and immediately centrifuged at 3000 g for 15 min at 4°C to obtain plasma. The collected plasma was stored at −80°C until further use. The entire experiment was repeated three times, with intervals of at least 7 days to allow the rats to recover. All procedures involving animals were approved by the Laboratory Animal Ethics Committee of South China University of Technology (approval number: 2023027). At the end of each experiment, four individual plasma samples were selected for each time point and used for subsequent peptide extraction. 2.5.2 Extraction of peptides for MS analysis Plasma sample preparation for MS analysis was performed according to previous methods, with minor modifications (Sheng et al., 2021). Specially, 100 μL of each plasma sample was mixed with four volumes of extraction buffer containing 20% acetonitrile, 0.1% formic acid and 0.1% sodium chloride. The mixture was subsequently transferred to an Amicon Ultra 0.5 mL 3 K centrifugal filter (Millipore, Gloucestershire, United Kindom) and centrifuged at 14,000 × g for 30 min at 4°C. The filtrate was mixed with 100 μL of 0.1% formic acid and then desalted using a Waters Sep-Pak C18 column (Waters, Massachusetts, United States) following the manufacturer’s guidelines. The eluted fraction was freeze-dried and stored at −80°C before LC–MS/MS analysis. 2.5.3 Peptide identification The same LC–MS/MS method described in Section 2.2.2 was used to identify E-Jiao-derived peptides in blood samples, with some modifications made to the database searching. The raw data was searched against a homemade database created according to a previous study (Zhang et al., 2023) and the Swiss-Prot database of Rattus norvegicus (accessed in February 2023, 10,128 entries) using the parameters described in Section 2.2.2. The homemade database consisted of the 100 most abundant E-Jiao proteins, including type I collagen, type III collagen and others. The identified peptides were manually verified to ensure that each peptide was detected in at least two samples at each time point. Considering that a database search was not conducted for the entire rat proteome, E-Jiao-derived peptides were verified by searching against UniProtKB/Swiss-Prot (Swiss-Prot) using the BLASTP algorithm (https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastp&PAGE_TYPE=BlastSearch&LINK_LOC=blasthome) to ensure their uniqueness to E-Jiao-derived proteins. The identified peptides that matched both the E-Jiao-derived and rat-derived proteins were further refined manually to exclude those detected in any samples collected before E-Jiao administration. 2.6 Selection of peptides for synthesis and evaluation of their blood-supplementing effect E-Jiao-derived peptides that were identified in both rat blood and LMEJ were scored by PeptideRanker (http://distilldeep.ucd.ie/PeptideRanker/). Only peptides with a length of less than 20 amino acids, no missed cleavage sites for trypsin and PeptideRanker scores exceeding 0.7 were considered as bioactive peptide candidates. Subsequently, the toxicity of these peptides was assessed by the ToxIBTL (https://server.wei-group.net/ToxIBTL). The novelty of the peptides was checked by searching the BIOPEP database (https://biochemia.uwm.edu.pl/). Five peptides (PP-1, PP-2, PP-3, PP-4 and PP-5) that satisfied the aforementioned criteria and obtained the highest PeptideRanker scores were chemically synthesized. Wild-type zebrafish embryos were used to evaluate the blood-supplementing effect of these peptides, following the procedures as described in Section 2.3.2, with modifications to the grouping and the quantification methods for neutrophils. The Control, Model, DGBX and LXS groups were treated in the same way as described in Section 2.3.2, while the LMEJ group was treated with 100 μg/mL LMEJ. To determine the administration dose, the PP-1∼PP-5 were firstly administrated to 24 hpf zebrafish embryos. The maximum tolerated concentration of each PP was determined based on mortality and malformation observed 48 h postexposure. The maximum dosing concentration of each PP was chosen according to the maximum tolerated concentration observed in zebrafish embryos. Three dose groups of PP were set as follows: low (10 μg/mL for PP-1-L; 3.75 μg/mL for PP-2-L; and 2.5 μg/mL for PP-3-L, PP-4-L and PP-5-L); medium (20 μg/mL for PP-1-M; 7.5 μg/mL for PP-2-M; and 5.0 μg/mL for PP-3-M, PP-4-M and PP-5-M); and high (40 μg/mL for PP-1-H; 15.0 μg/mL for PP-2-H; and 10.0 μg/mL for PP-3-H, PP-4-H and PP-5-H). Erythrocytes from zebrafish were stained and observed according to the procedure as described in Section 2.3.2. To evaluate the changes in neutrophil counts, zebrafish were subjected to the Sudan Black B staining, which was performed as previously described (Chen et al., 2022). All experiments were independently repeated at least three times. 2.7 Network pharmacology and molecular docking 2.7.1 Target screening for diseases and peptides The mechanisms behind the blood-supplementing effect of E-Jiao-derived peptides were investigated using network pharmacology. The SMILES (Simplified Molecular Input Line Entry System) of peptides were acquired via the PepSMI tool (https://www.novoprolabs.com/tools/convert-peptide-to-smiles-string). These SMILES strings were then subjected to target prediction using the Super PRED database (https://prediction.charite.de/). Disease (blood deficiency) related targets were identified by searching the GeneCards (https://www.genecards.org/) and OMIM-GENE-MAP databases (https://omim.org/search/advanced/geneMap) with keywords including “anemia,” “bone marrow suppression,” and “neutropenia”. The disease-related targets obtained from GeneCards were screened three times. Only those with relevance scores above the median were chosen. Targets from both GeneCards and OMIM-GENE-MAP were combined, and duplicates were removed. Finally, gene targets at the intersection of peptides and diseases were determined using Venny 2.1 (http://bioinfogp.cnb.csic.es/tools/venny/index.html). 2.7.2 Construction of the PPI network The targets that intersected were inputted into the STRING online software (https://cn.string-db.org/), with the research species restricted to “Homo sapiens”. A confidence level of 0.700 was set. Subsequently, a network diagram was generated using the Cytoscape 3.10.1 software, and the key targets were identified using the CentiScape tool, based on betweenness centrality, closeness centrality, and degree centrality (Scardoni et al., 2009). 2.7.3 GO and KEGG pathway enrichment analysis To elucidate the functions and pathways associated with the peptides’ effect on blood deficiency, the key targets were uploaded to Metascape (https://metascape.org/gp/index.html#/main/step1) for enrichment analyses. The enrichment analyses included Gene Ontology (GO) analyses, which cover Biological Process (BP), Cellular Component (CC), and Molecular Function (MF), as well as Kyoto Encyclopedia of Genes and Genomes (KEGG) analyses. 2.7.4 Molecular docking The molecular docking was carried out to confirm the interactions between peptides and their targets. The three core targets with the highest betweenness centrality were selected for docking with two peptides. The 2D structures of the peptides were transformed into 3D structures using ChemDraw 19.0. Subsequently, AutoDockTools 1.5.6 was utilized to eliminate water molecules, add polar hydrogen atoms, and define rotatable bonds. The crystal structures of the core targets were retrieved from the RSCB PDB database (https://www.rcsb.org). The crystal structures of STAT3 (PDB ID: 6NJS), HSP90AA1 (PDB ID: 3WQ9), and NFE2L2 (PDB ID: 7K2A) were chosen for further analysis. Prior to docking, water molecules and ligands in the crystal structure were removed, and polar hydrogen atoms were added. Docking was then performed using AutoDock Vina 1.1.2, and the results were visualized using PyMOL 2.5.7. 2.8 Data analysis Statistical analysis was conducted using GraphPad Prism (version 8.0.2), utilizing one-way ANOVA, the Kruskal‒Wallis test, or the t‒test to assess intergroup or intragroup differences, respectively. A p-value <0.05 was considered to indicate statistical significance. The data are presented as the mean ± standard deviation (SD).',\n",
       " '11187579': '2 Methods 2.1 Materials and chemicals Zn(II)–curcumin (ZnCM) was synthesized from curcumin (Guangdong Zhongda Greenfield Biotech. Co., Guangzhou, China) and zinc acetate, and prepared as solid dispersion using PVP-30 (BASF Chemical Ltd., Atlantic City, NJ, United States), according to our previous descriptions (the mass ratio for ZnCM: PVP-30 = 1 : 6) (Mei et al., 2011). Streptozotocin (STZ; S0130) was purchased from Sigma-Aldrich Corporation (Sigma-Aldrich, St. Louis, MaO, United States). Other reagents were of analytical grade and were purchased from local chemical suppliers. 2.2 Animal experimental design All animal experiments were approved by the Experimental Animal Care and Use Committee of Sun Yat-sen University (number IACUC-20221030002) and strictly conducted in line with the Guidelines for Care and Use of Laboratory Animals of this organization. Healthy adult female Sprague–Dawley rats (body weight 240–260 g) were provided by the Laboratory Animal Center of Sun Yat-sen University. They were housed under specific pathogen-free standard conditions with a 12-h light/dark cycle and free access to diet and water. After acclimation for 1 week, animals were treated in the presence or absence of 1 mg/kg CdCl2 in drinking water in the initial 8 weeks, subsequently, treated with or without a single intraperitoneal injection of STZ (60 mg/kg) (Xu et al., 2020), and orally administered with or without 100 mg/kg/d ZnCM, 700 mg/kg/d PVP, or 135 mg/kg/d Met (metformin, positive drug) for 8 weeks (n = 8 per group). The schematic diagram of the experimental design is shown in Figure 1A. The animals were weighed weekly in the course of the experiments. In addition, the stool samples were collected at week 16 for gene analysis. At the end of the experiment, rats were fasted for approximately 12 h and humanely euthanized by the administration of CO2. Afterward, serum, urine, and kidney samples were collected for biochemical, histopathological, and metabolic analyses, respectively. FIGURE 1 ZnCM attenuated renal pathological deterioration and dysfunction in STZ-induced rats exposed to Cd. (A) Schematic diagram of the experimental design. (B) Body weight throughout the experiment. (C) The ratio of kidney weight to body weight (KW/BW). (D–F) Representative images of H&E staining, AB-PAS staining, and Masson’s trichrome staining of the kidney (scale bar, 50 μm). PT: proximal tubule; proximal tubule epithelial cell edema and hypertrophy (black arrowheads); tubular epithelial damage (blue arrowheads); thickening of the glomerular basement membrane (light purple arrowheads); mesangial regions proliferative and expansion (red arrowheads); podocyte apoptosis and necrosis (green arrowheads); fibroblasts proliferation (yellow arrowheads). (G, H) Levels of urinary volume and albumin of 24 h. (I) Levels of blood urea nitrogen (BUN). (J–L) Levels of creatinine, urea, and uric acid in the serum. Data are represented by mean ± SD (n = 8) and were analyzed by one-way ANOVA followed by the original FDR method of the Benjamini and Hochberg test. *p < 0.05 and **p < 0.01; ns indicates no significance. 2.3 Fecal microbiota transplantation (FMT) Recipient rats were pretreated with a cocktail of antibiotics (ABX containing 1 g/L ampicillin, 0.5 g/L neomycin, 0.5 g/L vancomycin, and 1 g/L metronidazole) in drinking water for 1 week. Fresh feces from donor rats were daily collected, pooled (100 mg feces/mL), and vigorously mixed for 30 s, and then centrifuged at 800 g for 3 min. A measure of 100 μL of fecal supernatants of each rat was orally transferred to STZ-treated recipient rats once a day for 8 weeks. Kidney, blood, and urine samples from the recipient rats were collected for subsequent experiments. 2.4 Biochemical analysis The levels of creatinine, uric acid, urea, and urea nitrogen (BUN) in the serum or urine were tested using commercial kits (Nanjing Jiancheng Bioengineering Institute, Nanjing, China) (Xu et al., 2020). The contents of serum TNF-α, IL-1β, and IL-6 were quantified using ELISA kits (Nanjing Jiancheng Bioengineering Institute, Nanjing, China), according to the instructions of reagents and previous methods (Zhang et al., 2020; Li W. et al., 2021; Zhang et al., 2021). 2.5 Histopathological analysis Renal tissue was analyzed histologically as previous methods with minor modifications (Wang et al., 2021; Hao et al., 2022). Briefly, renal tissues were fixed in 4% paraformaldehyde and embedded in paraffin. A measure of 3-μm sections were stained with hematoxylin and eosin (H&E), Alcian blue-periodic acid–schiff (AB-PAS), or Masson’s trichrome staining by using standard methods (Zhou et al., 2021; Tao et al., 2022; Zhang et al., 2023). Images were captured under a Leica DM6B microscope (Leica, Weltzar, Germany). 2.6 Western blotting The expressions of proteins were determined by Western blot as described before (Ding et al., 2021; Chen et al., 2022; Zhou et al., 2022). In brief, the total protein of renal tissue was extracted by using RIPA extraction buffer, and the concentrations were quantified using BCA protein assay kits. 10% SDS polyacrylamide gel was used to separate proteins, transferred onto PVDF membranes, and then blocked with 5% skimmed milk for 1 h at room temperature. The primary antibodies (showed in Supplementary Table S1) were immunoblotted with the membranes at 4°C overnight and followed by the incubation with the respective secondary antibodies for 1 h at room temperature. The bands were visualized using ECL kits (Applygen, Beijing, China) on the Tanon5200 chemiluminescence imaging system (Shanghai Tianneng Technology Co., Ltd., Shanghai, China). The relative expressions of proteins were calculated against β-actin. 2.7 Zn and Cd measurements For the quantification of Zn and Cd, 0.1–0.5 g of renal tissue was digested by concentrated nitric acid in a microwave digestion system (MARS-6, CEM Co. Ltd., Matthews, NC, United States). The obtention was subjected to determine the contents of Zn and Cd by using inductively coupled plasma-mass spectrometry (ICP-MS; Thermo Fisher Scientific, Waltham, MA, United States) (Wu et al., 2019a; Wu et al., 2019b). The following formula can be used to calculate the Zn or Cd content per gram of tissue: Zn or Cd content (μg/g) = (Zn or Cd concentration of tissue suspension) × liquid volume (mL)/wet weight of tissue. For the imaging of Zn ions in renal tissue, the sections were deparaffinized, rehydrated, and incubated with fluorescent Zn ion probe TSQ (AAT Bioquest, Inc., United States) for 30 min at room temperature (Wu et al., 2019a; Wu et al., 2019b). Images were captured using a confocal laser scanning microscope (LSM880, Zeiss, Germany). 2.8 16S rRNA gene amplicon sequencing analysis The composition of fecal microbiota was analyzed by 16S rRNA-sequencing, as previously described (Li Y. et al., 2021; Wang et al., 2023). In brief, genomic DNA of bacteria was extracted from feces using the QIAamp DNA Stool Mini Kit (QIAGEN, Hilden, Germany) according to the manufacturer’s instructions. DNA concentration and purification were determined by using a NanoDrop instrument (Thermo Fisher Scientific, Waltham, MA, United States). The V3–V4 regions of genes were amplified with the following primers: 338F (5′-ACT\\u200bCCT\\u200bACG\\u200bGGA\\u200bGGC\\u200bAGC\\u200bAG-3′) and 806R (5′-GGACTACHVGGGTWTCTAAT-3′) using the thermocycler PCR system (thermal cycling condition: 95°C for 5 min (1 cycle), 95°C for 30 s, 50°C for 30 s, 72°C for 40 s (25 cycles), and a final extension at 72°C for 7 min). After purification and quantification, the amplicons were subjected to sequence on a HiSeq 2500 platform (Illumina, San Diego, United States) with the standard protocols by Biomarker Technologies Co., Ltd. (Beijing, China) (Li et al., 2023). The alpha diversity, beta diversity, and differential types of bacteria were analyzed with an online platform at BMKCloud (www.biocloud.net). 2.9 Metabolite analysis The serum metabolites were extracted and measured by gas chromatography–mass spectrometry (GC-MS), as previously described with minor modifications (Chen et al., 2019; Jiang et al., 2021). A measure of 80 μL of 20 mg/mL methoxyamine hydrochloride (Sigma-Aldrich, St Louis, MO, United States) in pyridine was added in 100 μL of serum. The mixture was vigorously vortexed for 5 min and incubated at 37°C for 90 min. A measure of 80 μL of N-methyl-N-trimethylsilyltrifluoroacetamide (Sigma-Aldrich, St Louis, MO, United States) was added and derivatized at 37°C for 30 min. The mixture was centrifuged at 30,000 × g at 4°C for 15 min, and the supernatant was filtered through a 0.2-μM filter. The filtrates were lyophilized and then resuspended in ultrapure water. The metabolites were detected by using an Agilent GC7890A-MS5975C gas chromatography–mass spectrometer (Agilent Technologies, Santa Clara, CA, United States) equipped with the experienced instrument settings (described in Supplementary Table S2) and coupled to an LCMS-8050 triple-quadrupole mass spectrometer (Shimadzu Corporation, Kyoto, Japan). The data were analyzed using SIMCA 14.1 software (Umetrics, Umea, Sweden). Spearman’s correlation between differential metabolites and microbiota was analyzed by using GraphPad Prism software 9.0 or MetaboAnalyst 5.0 (https://www.metaboanalyst.ca/). 2.10 Statistics GraphPad Prism software 9.0.0 (GraphPad Software, La Jolla, CA, United States) was used for the statistical analyses. Unpaired two-tailed t-tests were used to determine the significance of the differences between two groups. One-way ANOVA or two-way ANOVA was used to compare the statistical significance among more than two groups followed by the original false discovery rate (FDR) method of Benjamini and Hochberg or Tukey’s multiple comparison tests. Data are expressed as means ± standard deviation (SD). p-values <0.05 were statistically significant.',\n",
       " '11187502': 'Research methods and data analysis of metabolomics The rapid development of metabolomics is inextricably linked to advancements in its technology. Liquid chromatography (LC) and gas chromatography (GC) are the most prevalent methods for metabolite separation, while metabolite detection primarily relies on nuclear magnetic resonance (NMR) and mass spectrometry (MS) [25]. In metabolomics analysis, LC and GC are often coupled with MS, whereas NMR typically functions as a standalone tool. These techniques have become the mainstream platform for identifying and quantifying metabolites [26]. The combination of high-performance separation chromatography with highly specific and sensitive mass spectrometry enables rapid metabolite identification and accurate quantitative analysis. GC-MS excels at analyzing thermally stable and volatile metabolites with minimal matrix effects from complex samples [27]. LC-MS, on the other hand, boasts broader analytical capabilities. It can be combined with diverse chromatographic columns and other conditions for analysis, enabling the separation and identification of a wider range of metabolites within the sample without extensive pre-treatment. Notably, its high sensitivity makes it ideal for analyzing thermally unstable, non-volatile, and higher molecular weight substances [28]. NMR offers several advantages, including fast analysis speed, high reproducibility, suitability for high-throughput analysis, non-destructive nature, minimal bias, and simple sample preparation. It can simultaneously detect multiple organic compounds, making it widely used in metabolomics analysis [29]. The primary analytical methods in metabolomics encompass univariate and multivariate analysis. Univariate analysis, characterized by its simplicity, intuitiveness, and ease of understanding, is commonly employed in metabolomics research to swiftly examine differences between metabolite categories. Due to the challenges of metabolomics data meeting the assumptions of parametric testing, non-parametric methods like the Wilcoxon rank sum test, Kruskal-Wallis test, and t-test are frequently used. Additionally, calculating fold changes in metabolite concentration between groups and the area under the ROC curve (AUC) are common practices. Multivariate analysis encompasses various techniques such as principal component analysis (PCA), partial least squares discriminant analysis (PLS-DA), orthogonal projection to latent structures (OPLS), and cluster analysis (CA). PCA leverages the relationships between original variables, transforming them into a set of independent, comprehensive indicators (principal components) based on maximizing variation. Typically, 2–3 principal components are plotted to visually depict differences in metabolic patterns and clustering between groups. Load plots are then used to identify original variables contributing to group classification as potential biomarkers [30]. PLS-DA, a supervised analysis method, utilizes known sample group information for classification, identification, and prediction of multivariate data. Similar to PCA, PLS-DA is a linear analysis method, offering the advantage of combining load plots to screen for metabolite differences and identify biomarkers [31]. Compared to PLS-DA, OPLS specifically filters out “uninformative” variables in the observation variable matrix that are not relevant to the prediction variable matrix, essentially removing variation factors unrelated to the target variable in the data [32]. CA, a classification method employing multivariate statistical techniques, often utilizes hierarchical clustering to distinguish disease types and evolutionary stages [33]. Common data processing tools for non-targeted metabolomics include XCMS, MZmine 2, and MS Dial, while targeted metabolomics data processing typically utilizes MRMAnalyzer, MRMPROBS, MAVEN, Max Quant, and XCMS-MRM [34]. Additionally, SIMCA software is frequently used for multivariate analysis, while HMDB, MetLine, and BMRB are common metabolite identification platforms. MetaboAnalyst and KEGG are popular biological function mining platforms [35].',\n",
       " '11187400': 'Materials and methods Study area and study period A Cross sectional study was conducted in three woredas each of Addis Ketema and Akaki/Kality sub-cities to isolate V. cholerae and E. coli O157:H7 from drinking water and wastewater samples. Sample size was determined using simple population proportion formula. \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{mathrsfs} \\\\usepackage{upgreek} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document}$$\\\\begin{gathered}n = \\\\frac{{{{(Z\\\\alpha /2)}^2}pq}}{{{d^2}}} \\\\hfill \\\\\\\\\\\\,\\\\,\\\\,\\\\, = \\\\frac{{{{(1.96)}^2} \\\\times (0.14 \\\\times 0.86)}}{{{{0.05}^2}}} = 185 \\\\hfill \\\\\\\\ \\\\end{gathered}$$\\\\end{document} Where: n = sample size; Zα = risk expresses in z-score; p = expected prevalence (14%, based on 2019 Cholera outbreak); q = 1-p; d = absolute precision. Considering a 10% non-responsive rate, the final sample size was taken as 206. A total of 206 environmental water samples were aseptically collected from drinking water at point-of-use and surface sewerage from May to July 2023. These samples were collected from six different woredas within two sub cities in Addis Ababa (Fig. 1). A woreda is the smallest administrative unit in Addis Ababa. The sub cities were Addis Ketema sub city and Akaki Kality sub city. Three woredas each were selected from the two sub-cities: Woreda Three, Seven and Eight from Addis Ketema and Woreda Six, Seven and Eight were from Akaki Kality sub cities. Fig. 1Map showing the study woredas in Addis Ketema and Akaki/Kality sub-cities, Addis Ababa The study sub-cities and woredas were selected purposively based on their high incidence of cholera cases in the 2019 outbreak in Addis Ababa. The simple population proportion formula was used to determine the sample size and a total of 206 samples consisting of drinking water (n = 97) and wastewater (n = 108) were considered for this study. Study households were selected following the systematic random sampling method. Sewage samples were collected from open sewage ditches close to selected households. All drinking water samples were collected from point-of-use taps found in a total of 97 households from all study woredas each of the two study sub-cities. A volume of 250 mL of drinking water samples was aseptically collected using 500-ml sterile narrow-necked screw capped bottle. Similarly, 250 mLof wastewater samples were aseptically collected from 108 open ditch sewer sites found along innner roads in residential areas in both sub-cities. Collected samples were immediately transported to the laboratory in an icebox. Samples were processed within two hours of collection. Isolation of V. cholerae and E. coli O157:H7 A volume of 100 of a well-mixed sample was filtered through a 0.22-µm pore size polycarbonate filter (diameter 45 mm). The filter was placed in a 50 mL falcon tube containing 12 mL of sterile Phosphate-Buffered Saline (PBS) (OXOID) and vortexed vigorously for 5 min to suspend the attached bacteria into the saline solution. To isolate V. cholerae, an aliquot of one mL of PBS suspension was added to an enrichment flask containing 25-ml Alkaline Peptone Water (APW) and incubated at 30-35oC for 18–24 h [34]. Surface growth from APW was streaked on pre-dried plates of Thiosulfate Citrate Bile salts Sucrose (TCBS) agar (OXOID) and incubated at 30-35oC for 18–24 h. All yellow colonies that fermented sucrose in TCBS agar were sub-cultured on Trypton Soy (TSY) Agar (OXOID). For cell morphology, wet mounts were examined microscopically at 40x using oil immersion. Colonies from TSY agar were also subjected to oxidase and string tests for Vibrio cholearae. Curved rods that had positive oxidase and string tests were presumptively considered as V. cholerae [34]. For serological confirmation, colonies were suspended in 0.5 mL physiological saline on a glass slide and gently mixed with a drop of antiserum for somatic O antigens by tilting the glass slide back and forth for one minute. Agglutination indicated a positive serological reaction. Serogroup O1 was similarly serotyped using type antisera. To isolate E. coli O157:H7, 25 mL of the membrane filtered suspension was added to Phosphate Buffered Saline (PBS) and was incubated at 32 ºC for 18–24 h. The growth was directly streaked on pre-dried plates of Sorbitol MacConkey (SMAC) agar [35] and incubated at 32 ºC for 18–24 h. Non-sorbitol fermenting colorless colonies were further subjected for confirmation using E. coli O157:H7 Latex agglutination Test [36]. Antimicrobial susceptibility testing for V. cholerae O1 and E. coli O157:H7 isolates Antimicrobial susceptibility for V. cholerae and E. coli O157:H7 was determined by Kirby-Bauer disc diffusion method on Mueller-Hinton agar plates with commercially available antibiotic discs (OXOID) The 0.5 MacFarland was maintained using a 0.85% saline suspension of fresh colony from TSA agar and by measuring it using an OXOID MacFarland spectrometer. V. cholerae Isolates were tested against 13 antimicrobial drugs consisting of amoxycillin (AML, 30 µg), azithromycin (AZM, 15 µg), ampicillin (AMP, 10 µg), peperacilin pazobactam (TZP, 110 µg), trimethoprim/sulphamethoxazole (STX, 25 µg), meropenem (MEM, 10 µg), cefoxitin (FOX, 10 µg), tetracycline (TE, 30 µg), ciprofloxacin (CIP, 5 µg), nalidixic acid (NA, 30 µg), ceftriaxone (CRO, 30 µg), ceftazidime (CAZ, 30 µg), ceftazidime/clavulanic acid (CAZ/CLA, 30/10 µg). After an overnight incubation of Mueller-Hinton agar, the inhibition zone of each antibiotic was measured [37]. E. coli O157:H7 isolates were also tested against 12 antimicrobial drugs consisting of ampicillin sulbactam (SAM 20 µg), amoxcillin clavulanic acid (AMC 30 µg), azithromycin (AZM, 15 µg), ampicillin (AMP, 10 µg), amikacin (AK, 30 µg), gentamicin (CN, 10 µg), trimethoprim/ sulphamethoxazole (SXT, 25 µg), imipenem (IPM, 10 µg), tetracycline (TET, 30 µg), ciprofloxacin (CIP, 5 µg), ceftriaxone (CRO, 30 µg), nitrofurantoin (F 300, µg). After an overnight incubation of Mueller-Hinton agar, the inhibition zone of each antibiotic was measured [38]. E. coli ATCC 25,922 strain was used as a positive control. In both susceptibility tests a known positive control, and a blank disc were included. After an overnight incubation of Mueller-Hinton agar, the inhibition zone of each antibiotic was measured and recorded. For interpretation, the ‘Intermediate’ values were considered as ‘Sensitive’. The multiple antibiotic resistance index (MARI) was calculated and interpreted according to Krumperman [39] using the formula: a/b, where ‘a’ represented the number of antibiotics to which an isolate was resistant, and ‘b’ represented the total number of antibiotics tested. Determination of extended spectrum β-lactamase (ESBL) production and carbapenem resistance Extended Spectrum Beta Lactamase (ESBL) producing V. cholerae and E. coli O157:H7 isolates was determined by using a double disk synergy test (DDST) [40]. Ceftazidime and Ceftazidime Clavulanic acid discs were employed and an inhibition zone diameter difference of ≥ 5 mm between the two drugs was considered as indication of ESBL production. To detect Carbapenem Resistance among V. cholerae and E. coli O157: H7 resistance against Meropenem and Imipenem were checked, respectively. After an overnight incubation of Mueller-Hinton agar, the inhibition zone of both antibiotics was measured and recorded. The recorded size of inhibition zone was then changed into Sensitive (S), Intermediate (I) and Resistant (R) as per CLSI-M100. Resistance against the drugs were considered ar Carbapenem Resistance.',\n",
       " '11187374': 'MethodsThe BLOOM study is a multicentre, double-blinded, randomised controlled trial that will assess if brewer’s yeast or beta-glucan can increase early breast milk production following preterm birth. Target population are mothers of preterm infants born at less than 34 weeks’ gestation who intend to provide breast milk for their infant, are less than 72 h following birth and able to give informed consent. Participants will be randomly allocated into three parallel groups at 1:1:1 ratio (n = 33 per group) to receive either brewer’s yeast, beta-glucan or placebo capsules for seven days. The primary outcome is total expressed breast milk volume over a 24-hour period on day 7 of intervention. Participants and their infants will be followed until the infant reaches term corrected age or is discharged home from the neonatal unit (whichever occurs first).'}"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "method_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "# nlp = spacy.load(\"en_core_web_lg\")\n",
    "\n",
    "# doc = nlp(\"Mr Tan Ah Kow has a history of medical conditions. He has had hypertension and hyperlipidemia since 1990 and suffered several strokes in 2005. He subsequentlydeveloped heart problems (cardiomyopathy), cardiac failure and chronic renal diseaseand was treated in ABC Hospital.He was last admitted to the ABC Hospital on 1 April 2010 till 15 April 2010, duringwhich he was diagnosed to have suffered from a stroke. This was confirmed by CTand MRI brain scans.Thereafter, he was transferred to XYZ Hospital for stroke rehabilitation on 15 April2010.After that, Mr Tan was referred to Blackacre Hospital for follow-up treatment from inNovember 2010. The clinical impression was that he was manifesting behavioural andpsychological symptoms secondary to Dementia\")\n",
    "\n",
    "# for ent in doc.ents:\n",
    "#     print(ent.text, ent.start_char, ent.end_char, ent.label_)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "import spacy\n",
    "\n",
    "# products are generally followed by company names\n",
    "nlp = spacy.load(\"en_core_web_trf\")\n",
    "\n",
    "entity_dict = {}\n",
    "for key, value in method_dict.items():\n",
    "    doc = nlp(value)\n",
    "    entities = []\n",
    "    for ent in doc.ents:\n",
    "        if ent.label_ == 'PRODUCT':\n",
    "            entities.append(ent.text)\n",
    "    entity_dict[key] = entities"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "≥98% 97 101 PERCENT\n",
      "933.13 Da 126 135 QUANTITY\n",
      "Chengdu Purifa Technology Development Co. Ltd 156 201 ORG\n",
      "Chengdu 203 210 GPE\n",
      "China 212 217 GPE\n",
      "0216011010 254 264 CARDINAL\n",
      "36 kDa–50 kDa 270 283 QUANTITY\n",
      "MP Biomedicals 304 318 ORG\n",
      "Shanghai 320 328 GPE\n",
      "China 330 335 GPE\n",
      "398.39 Da 410 419 QUANTITY\n",
      "FD4 439 442 PRODUCT\n",
      "Sigma-Aldrich 482 495 ORG\n",
      "Darmstadt 497 506 GPE\n",
      "Germany 508 515 GPE\n",
      "T6113 527 532 PRODUCT\n",
      "98% 555 558 PERCENT\n",
      "548.64 Da 584 593 QUANTITY\n",
      "TOPSCIENCE 613 623 ORG\n",
      "Shanghai 625 633 GPE\n",
      "China 635 640 GPE\n",
      "693520 663 669 PRODUCT\n",
      "MilliporeSigma 707 721 ORG\n",
      "Burlington 723 733 GPE\n",
      "MA 735 737 GPE\n",
      "USA 739 742 GPE\n",
      "the American Type Culture Collection 856 892 ORG\n",
      "ATCC 894 898 ORG\n",
      "Manassas 900 908 GPE\n",
      "VA 910 912 GPE\n",
      "USA 914 917 GPE\n",
      "CT26 931 935 PRODUCT\n",
      "Roswell Park Memorial Institute 959 990 ORG\n",
      "11875085 1019 1027 PRODUCT\n",
      "Gibco 1029 1034 GPE\n",
      "NY 1036 1038 GPE\n",
      "USA 1040 1043 GPE\n",
      "10% 1063 1066 PERCENT\n",
      "10099158 1087 1095 PRODUCT\n",
      "Gibco 1097 1102 GPE\n",
      "NY 1104 1106 GPE\n",
      "USA 1108 1111 GPE\n",
      "5% 1181 1183 PERCENT\n",
      "37 ° 1231 1235 QUANTITY\n",
      "The Laboratory Animal Center 1246 1274 ORG\n",
      "Shanghai University of Traditional Chinese Medicine 1278 1329 ORG\n",
      "20 ± 2 g. 1368 1377 QUANTITY\n",
      "12-h 1576 1580 DATE\n",
      "The Animal Experimentation Ethics Committee 1599 1642 ORG\n",
      "Shanghai University of Traditional Chinese Medicine 1646 1697 ORG\n",
      "Yue 1997 2000 PERSON\n",
      "four 2109 2113 CARDINAL\n",
      "3% 2207 2209 PERCENT\n",
      "8 days 2260 2266 DATE\n",
      "260 mg 2328 2334 QUANTITY\n",
      "25, 50, 125 mg 2411 2425 QUANTITY\n",
      "10 days 2462 2469 DATE\n",
      "Control 2543 2550 PRODUCT\n",
      "Daily 2552 2557 DATE\n",
      "10-day 2633 2639 DATE\n",
      "four 2797 2801 CARDINAL\n",
      "25 mg 3030 3035 QUANTITY\n",
      "10 consecutive days 3062 3081 DATE\n",
      "20 mg 3172 3177 QUANTITY\n",
      "three 3212 3217 CARDINAL\n",
      "1 week 3376 3382 DATE\n",
      "2 3450 3451 CARDINAL\n",
      "105 3454 3457 CARDINAL\n",
      "200 mm3 3577 3584 QUANTITY\n",
      "18-day 3693 3699 DATE\n",
      "0.5% 3747 3751 PERCENT\n",
      "25 mg 3808 3813 QUANTITY\n",
      "0.5 3838 3841 CARDINAL\n",
      "4 h 3939 3942 DATE\n",
      "60 mg/100 g 3996 4007 QUANTITY\n",
      "200 µL 4039 4045 QUANTITY\n",
      "4 h 4071 4074 DATE\n",
      "485 nm 4251 4257 QUANTITY\n",
      "528 nm 4288 4294 QUANTITY\n",
      "VARIOSKAN FLASH 4316 4331 PRODUCT\n",
      "Thermo Fisher 4333 4346 ORG\n",
      "MA 4348 4350 GPE\n",
      "USA 4352 4355 GPE\n",
      "4% 4425 4427 PERCENT\n",
      "4 μm 4517 4521 QUANTITY\n",
      "BX61VS 4683 4689 PRODUCT\n",
      "Olympus 4691 4698 ORG\n",
      "Tokyo 4700 4705 GPE\n",
      "Japan 4707 4712 GPE\n",
      "Wuhan Huamei Biological Engineering Co., Ltd 4937 4981 ORG\n",
      "Wuhan 4983 4988 GPE\n",
      "China 4990 4995 GPE\n",
      "96-well 5042 5049 QUANTITY\n",
      "450 nm 5198 5204 QUANTITY\n",
      "4% 5321 5323 PERCENT\n",
      "5-μm 5387 5391 QUANTITY\n",
      "Cell Signaling Technology 5507 5532 ORG\n",
      "CST 5534 5537 ORG\n",
      "MA 5539 5541 GPE\n",
      "USA 5543 5546 GPE\n",
      "91131 5563 5568 DATE\n",
      "CST 5570 5573 ORG\n",
      "MA 5575 5577 GPE\n",
      "USA 5579 5582 GPE\n",
      "9300039001 5646 5656 PRODUCT\n",
      "ABclonal 5658 5666 ORG\n",
      "Wuhan 5668 5673 GPE\n",
      "China 5675 5680 GPE\n",
      "4083 5722 5726 PRODUCT\n",
      "CST 5728 5731 ORG\n",
      "MA 5733 5735 GPE\n",
      "USA 5737 5740 GPE\n",
      "BX61VS 5801 5807 PRODUCT\n",
      "Olympus 5809 5816 ORG\n",
      "Tokyo 5818 5823 GPE\n",
      "Japan 5825 5830 GPE\n",
      "NIH 5917 5920 ORG\n",
      "Bethesda 5922 5930 GPE\n",
      "MD 5932 5934 GPE\n",
      "USA 5936 5939 GPE\n",
      "BX61VS 6149 6155 PRODUCT\n",
      "Olympus 6157 6164 ORG\n",
      "Tokyo 6166 6171 GPE\n",
      "Japan 6173 6178 GPE\n",
      "Thermo Fisher Scientific 6454 6478 ORG\n",
      "Evo M-MLV RT Premix for 6527 6550 PRODUCT\n",
      "AG11706 6561 6568 PRODUCT\n",
      "Accurate Biotechnology Co., Ltd. 6570 6602 ORG\n",
      "Chengdu 6604 6611 GPE\n",
      "China 6613 6618 GPE\n",
      "AG11718 6694 6701 PRODUCT\n",
      "Accurate Biotechnology Co., Ltd. 6703 6735 ORG\n",
      "Chengdu 6737 6744 GPE\n",
      "China 6746 6751 GPE\n",
      "1 6760 6761 CARDINAL\n",
      "an ABI Prism 7900HT Sequence Detection System 6804 6849 PRODUCT\n",
      "Life Technologies 6851 6868 ORG\n",
      "CA 6870 6872 GPE\n",
      "USA 6874 6877 GPE\n",
      "20201ES76 8334 8343 PRODUCT\n",
      "Yeasen Biotech Co., Ltd 8345 8368 ORG\n",
      "Shanghai 8370 8378 GPE\n",
      "China 8380 8385 GPE\n",
      "000025736 8613 8622 FAC\n",
      "Milipore 8624 8632 GPE\n",
      "MA 8634 8636 GPE\n",
      "USA 8638 8641 GPE\n",
      "5% 8679 8681 PERCENT\n",
      "2 h. 8699 8703 DATE\n",
      "4 ° 8784 8787 QUANTITY\n",
      "1 h 8858 8861 DATE\n",
      "WBKLS0500 8933 8942 PRODUCT\n",
      "Millipore 8944 8953 PRODUCT\n",
      "Bio 9002 9005 ORG\n",
      "- 9005 9006 ORG\n",
      "Rad 9006 9009 ORG\n",
      "CA 9011 9013 ORG\n",
      "USA 9015 9018 GPE\n",
      "ImageJ 9069 9075 PRODUCT\n",
      "NIH 9086 9089 ORG\n",
      "Bethesda 9091 9099 GPE\n",
      "MD 9101 9103 GPE\n",
      "USA 9105 9108 GPE\n",
      "8480 9187 9191 CARDINAL\n",
      "MA 9198 9200 GPE\n",
      "USA 9202 9205 GPE\n",
      "1:1000 9230 9236 CARDINAL\n",
      "5558 9239 9243 CARDINAL\n",
      "MA 9250 9252 GPE\n",
      "USA 9254 9257 GPE\n",
      "1:1000 9280 9286 CARDINAL\n",
      "12456 9289 9294 CARDINAL\n",
      "MA 9301 9303 GPE\n",
      "USA 9305 9308 GPE\n",
      "1:1000 9334 9340 CARDINAL\n",
      "2922 9343 9347 CARDINAL\n",
      "MA 9354 9356 GPE\n",
      "USA 9358 9361 GPE\n",
      "1:1000 9383 9389 CARDINAL\n",
      "5605 9392 9396 CARDINAL\n",
      "MA 9403 9405 GPE\n",
      "USA 9407 9410 GPE\n",
      "1:1000 9437 9443 CARDINAL\n",
      "4970 9446 9450 CARDINAL\n",
      "MA 9457 9459 GPE\n",
      "USA 9461 9464 GPE\n",
      "4% 9532 9534 PERCENT\n",
      "5 μm 9591 9595 QUANTITY\n",
      "6.0 9680 9683 CARDINAL\n",
      "5% 9756 9758 PERCENT\n",
      "30 min 9774 9780 TIME\n",
      "4 ° 9819 9822 QUANTITY\n",
      "Thermo Fisher 10278 10291 ORG\n",
      "the NEBNext Ultra RNA Library Prep Kit 10346 10384 PRODUCT\n",
      "Illumina 10389 10397 ORG\n",
      "Illumina 10434 10442 ORG\n",
      "HiSeq 10443 10448 PRODUCT\n",
      "majorbio 10522 10530 ORG\n",
      "Millicell 10633 10642 ORG\n",
      "24 10654 10656 CARDINAL\n",
      "5 × 104 cells/400 μL 10685 10705 QUANTITY\n",
      "600 μL 10750 10756 QUANTITY\n",
      "2323012 10770 10777 PRODUCT\n",
      "Gibco 10779 10784 GPE\n",
      "NY 10786 10788 GPE\n",
      "USA 10790 10793 GPE\n",
      "MERS00002 10859 10868 PRODUCT\n",
      "Milipore 10892 10900 PRODUCT\n",
      "70% 10941 10944 PERCENT\n",
      "400 Ω/cm2 11084 11093 QUANTITY\n",
      "3-mm 11441 11445 QUANTITY\n",
      "3 mM 11517 11521 QUANTITY\n",
      "4 ° 11539 11542 QUANTITY\n",
      "20 min 11548 11554 TIME\n",
      "70-μm 11727 11732 QUANTITY\n",
      "1200 r/min 11784 11794 QUANTITY\n",
      "5 min 11799 11804 TIME\n",
      "Matrigel 11824 11832 PRODUCT\n",
      "Corning 11834 11841 GPE\n",
      "NA 11843 11845 GPE\n",
      "USA 11847 11850 GPE\n",
      "24 11894 11896 CARDINAL\n",
      "37 ° 11926 11930 QUANTITY\n",
      "30 min 11936 11942 TIME\n",
      "Matrigel 11956 11964 PRODUCT\n",
      "06005 12020 12025 PRODUCT\n",
      "STEMCELL 12027 12035 ORG\n",
      "Vancouver 12037 12046 GPE\n",
      "Canada 12048 12054 GPE\n",
      "every other day 12096 12111 DATE\n",
      "2 days 12128 12134 DATE\n",
      "24 12140 12142 CARDINAL\n",
      "20 μg 12294 12299 QUANTITY\n",
      "20 μg 12310 12315 QUANTITY\n",
      "100 μM 12331 12337 QUANTITY\n",
      "4 days 12343 12349 DATE\n",
      "Olympus 12415 12422 ORG\n",
      "CKX4 12423 12427 PRODUCT\n",
      "Tokyo 12429 12434 GPE\n",
      "Japan 12436 12441 GPE\n",
      "AutoDock Vina 12632 12645 PRODUCT\n",
      "ChemDraw 12822 12830 PRODUCT\n",
      "Open Babel 12874 12884 PRODUCT\n",
      "AutoDock Tools 12944 12958 PRODUCT\n",
      "PyMOL 13129 13134 ORG\n",
      "24-well 13272 13279 QUANTITY\n",
      "500 ng 13347 13353 QUANTITY\n",
      "Beyotime Biotechnology 13393 13415 ORG\n",
      "Shanghai 13417 13425 GPE\n",
      "China 13427 13432 GPE\n",
      "50 ng 13438 13443 QUANTITY\n",
      "Promega GmbH 13464 13476 ORG\n",
      "Mannheim 13478 13486 GPE\n",
      "Germany 13488 13495 GPE\n",
      "Lipofectamine 3000 13503 13521 PRODUCT\n",
      "Thermo Fisher 13523 13536 ORG\n",
      "24 h 13545 13549 DATE\n",
      "50 μM 13585 13590 QUANTITY\n",
      "0.5 μM 13601 13607 QUANTITY\n",
      "24 h 13613 13617 DATE\n",
      "150 μL 13665 13671 QUANTITY\n",
      "a Dual-LuciferaseTM Reporter Assay System 13767 13808 PRODUCT\n",
      "Promega Corporation 13810 13829 ORG\n",
      "WI 13831 13833 GPE\n",
      "USA 13835 13838 GPE\n",
      "10 μL 13993 13998 QUANTITY\n",
      "20 μg/mL 14094 14102 QUANTITY\n",
      "20 μg 14112 14117 QUANTITY\n",
      "100 μM 14130 14136 QUANTITY\n",
      "24 h. 14142 14147 DATE\n",
      "0 and 24 h 14206 14216 DATE\n",
      "24 h 14315 14319 DATE\n",
      "NCM460 14365 14371 PRODUCT\n",
      "20 μg/mL 14408 14416 QUANTITY\n",
      "20 μg/mL 14426 14434 QUANTITY\n",
      "100 μM 14444 14450 QUANTITY\n",
      "24 h. 14456 14461 DATE\n",
      "NCM460 14468 14474 PRODUCT\n",
      "15 min 14724 14730 TIME\n",
      "Guava 14826 14831 PRODUCT\n",
      "GraphPad Prism 9.0 14957 14975 PRODUCT\n",
      "one 15088 15091 CARDINAL\n",
      "0.05 15130 15134 CARDINAL\n",
      "at least three 15207 15221 CARDINAL\n"
     ]
    }
   ],
   "source": [
    "\n",
    "doc = nlp(method_dict['11192171'])\n",
    "\n",
    "for ent in doc.ents:\n",
    "    print(ent.text, ent.start_char, ent.end_char, ent.label_)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "# extract the full text version of articles using pmcids\n",
    "full_text_articles = pmc_scraper.fetch_full_text(['11189522'])\n",
    "\n",
    "# extract the desired section of articles\n",
    "method_sections = pmc_scraper.extract_section(full_text_articles, 'method')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Materials and methods Materials All chemicals were obtained from Sigma (St Louis, MO, USA) unless otherwise stated. 2’-fucosyllactose (2’FL) and 3-fucosyllactose (3FL) were obtained from Glycom/DSM (Esbjerg, Denmark). Blood group A type II (BgA), Blood group B type II (BgB), Blood group H type II (BgH) and LewisY (LeY) were obtained from Elicityl (Crolles, France). Lewis A trisaccharide (LeA), 3′-sialyl Lewis A (sLeA), Lewis X trisaccharide (LeX), 3’-sialyl Lewis X (sLeX), 2-acetamido-2-deoxy-6-O-(α-l-fucopyranosyl)-d-glucopyranose (6FN), 2-acetamido-2-deoxy-4-O-(α-l-fucopyranosyl)-d-glucopyranose (4FN), 2-acetamido-2-deoxy-3-O-(α-l-fucopyranosyl)-d-glucopyranose (3FN), 4-nitrophenyl α-l-fucopyranoside (pNP-Fuc), 2-Chloro-4-nitrophenyl-αl-fucopyranoside (CNP-Fuc), 2-Chloro-4-nitrophenol (CNP) and N-acetyllactosamine (LacNAc) were obtained from Biosynth Ltd (Compton, UK). FA2G2 N-glycan was from Ludger (Oxford, UK). IgG was purified from human serum using the protein A IgG purification kit from Thermofisher (Carlsbad, US). Purified porcine gastric mucin (pPGM) was obtained as previously described71. PNGase B035DRAFT_0334172 was a kind gift from Dr Lucy Crouch (Newcastle University). Phospholipase A2 (PLA2) from honeybee venom (Apis mellifera) was purchased from Sigma (St Louis, MO, USA). Recombinant fucosidases E1_101251B from R. gnavus E1 and ATCC_038333A from R. gnavus ATCC 29149 were produced in-house as previously reported11. Bioinformatics analyses For sequence similarity networks (SSN) analysis, the sequences encoding GH29 fucosidases were extracted from CAZy database (www.cazy.org). A total of 9,505 GH29 sequences from the CAZy database (last update 2022-10-18) were winnowed down to 2,971 sequences following a sequence identity cut-off at 0.8 via CD-HIT suite73. The amino acid sequences were then used to generate SSN using the Enzyme Function Initiative-Enzyme Similarity Tool (EFI-EST) with an alignment score threshold of 96 (40% sequence identity)33,74. The SSN was visualised using Cytoscape 3.9.1. The GH29 sequences from each cluster are provided in Supplementary Data 5. A semi-supervised training method, termed GH29BERT, was applied to implement the unsupervised protein sequence representation learning and supervised classification for GH29 fucosidases. This pLM training process is composed of two phases (as illustrated in Fig. S7), pre-training, and classification task-training, respectively. The pre-training utilised the BERT-based (Bidirectional Encoder Representations from Transformers) language model75 to extract features from 34,258 non-redundant GH29 sequences derived from the CAZy database (last update: 2023-10-10) and InterPro database (downloaded on 2023-11-02). A random 95%-5% data split was adopted for model pre-training and training-process validation. The original BERT model has 12 repeated blocks of Transformer encoders76, here we tuned this hyperparameter to 5 for best performance. Notably, the pre-training model, including a Masked Language Modelling (MLM) prediction head, enabled hiding a certain percentage of input tokens and training the model to predict them in a self-supervised approach. It implements both the next-token prediction and the previous-token prediction, facilitating bidirectional context understanding, which is critical for protein sequence modelling29. The classification task-training model, composed of two attention layers, three dense connected layers and one softmax classification head, was performed on 2,796 labelled sequences (see Supplementary Data 6) derived from the top 45 clusters of the SSN excluding 14 GH29 sequences which were further used for validation (see list of enzymes in Table 1). We randomly selected 80% of the labelled data for training and 20% for testing. Pre-training and task-training were executed on two NVIDIA A100 40GB GPU for one week and 2 h, respectively. In addition to evaluating the accuracy of cluster predictions, we incorporated Exponential Cross-Entropy (ECE) to assess the uncertainty of each pLM while processing input sequences. For the classification task in this study, ECE was calculated using \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{mathrsfs} \\\\usepackage{upgreek} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document}$${e}^{1/n{\\\\sum }_{i}^{n}{CE}({s}_{i},{y}_{i})}$$\\\\end{document}e1/n∑inCE(si,yi), where \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{mathrsfs} \\\\usepackage{upgreek} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document}$$n$$\\\\end{document}n is number of protein sequences tested, \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{mathrsfs} \\\\usepackage{upgreek} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document}$${s}_{i}$$\\\\end{document}si and \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{mathrsfs} \\\\usepackage{upgreek} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document}$${y}_{i}$$\\\\end{document}yi denote the sequences and their corresponding cluster labels, respectively. ECE is also known as one kind of perplexity, which in our context ranges from 1, indicating deterministic predictions, to 45, equivalent to a completely random selection from the 45 clusters. In addition to adopting the semi-supervised method through training the pLM on GH29 fucosidase sequences from scratch, i.e., with randomly initialised model parameters, we also included two state-of-the-art pLMs, ESM-228 and ProtT5 model29, for validating their efficacy on GH29 sequence cluster prediction. These pLMs are in larger scale than GH29BERT, in terms of training data, number of parameters, and training time, and were pre-trained on the entire known proteins (see detailed configuration comparison in Table S5). We loaded and froze their official pre-trained parameters, then used labelled GH29 sequences to fine-tune the task-training model, which had the identical structure as for GH29BERT. For comparison, we established two baselines using a non-pretrained GH29BERT and a one-hot encoding approach, respectively. Non-pretrained GH29BERT was trained directly with labelled GH29 sequences, while the one-hot method trained the same task-training model using a one-hot encoding of the protein sequence. Dimension reduction was performed using Uniform Manifold Approximation and Projection (UMAP)52. The GH29BERT model is accessible through a friendly user-interface: https://huggingface.co/spaces/Oiliver/GH29BERT. This web tool assigns a corresponding cluster ID to any sequence uploaded. The configuration of running environment, including all dependencies and used packages, as well as the Python version are detailed in supplementary data 7. The source code and instruction for running environment preparation are available on GitHub: https://github.com/ke-xing/GH29BERT. Cloning, expression and purification of fucosidases The GH29-encoding genes were synthesised exempt of the signal peptide sequence and cloned into pET28a with N terminal His6-tag by Prozomix (Haltwhistle, UK). BaGH2926AD218A and BaGH2926AD218N mutants were synthesised by NZYTech (Lisboa, Portugal). Escherichia coli TunerDE3 pLacI cells were transformed with the recombinant plasmids according to manufacturer’s instructions. Expression was carried out in 1 L LB media growing cells at 37°C until OD600 reached 0.3 to 0.6 and then induced at 16°C for 20-22 h. The cells were harvested by centrifugation at 4,000 g for 35 min. The His-tagged proteins were purified by immobilised metal affinity chromatography (IMAC) and further purified by gel filtration on an ÄKTApure (Cytiva, Little Chalfont, UK). Protein purification was assessed by standard SDS–polyacrylamide gel electrophoresis using the NuPAGE Novex 4–12% Bis-Tris gels (Life Technologies, Paisley, UK). Protein concentration was measured with a NanoDrop (Thermo Scientific, Wilmington, USA) and using the extinction coefficient calculated by Protparam77 from the peptide sequence. Enzymatic activity assays For kinetics, all enzymes were incubated with CNP-Fuc in 50 mM citrate buffer at pH 6 and 37°C. The amount of enzyme was determined to fulfil free-ligand approximation, i.e. the enzyme concentration was linear with product formation. The reaction duration was optimised to measure the reaction rates under initial conditions. A standard curve was made with the reaction product CNP and Fuc in 1:1 ratio from 0 to 0.3 mM to better mimic the reaction products. The release of CNP was monitored using a microplate reader (FLUOstar Omega, BMG LABTECH, Ortenberg, Germany) by monitoring absorbance at 405 nm every 2 min for 40 min in 3 technical replicates. The kinetic parameters were calculated based on the Michaelis-Menten equation using a non-linear regression analysis programme, and one-way ANOVA was performed compared to E1_101251B (Prism 5, GraphPad, San Diego, USA). The enzymatic activity of recombinant fucosidases was determined on 2’FL, 3FL, BgA, BgB, BgH, LeA, sLeA, LeX, sLeX, LeY, 6FN, pNP-Fuc and pPGM using 10 μM enzyme, 0.5 mM substrate or 1 mg/mL for pPGM in 50 mM citrate buffer pH 6 and 1 mg/mL bovine serum albumin (BSA). The reactions were incubated at 37°C and stopped by boiling at 95°C for 10 min. The release of Fuc was quantified with the l-fucose assay kit from Megazyme (Wicklow, Ireland) using a microplate reader (FLUOstar Omega, BMG LABTECH, Ortenberg, Germany) by monitoring absorbance at 340 nm every 2 min. To determine the specific activity, the enzymatic reactions were optimised by adjusting enzyme concentration and incubation time (Table S6) to obtain between 6%-25% of substrate hydrolysis which is within detection limit and corresponds to linear range. Specific activity was calculated from 4 technical replicates. One unit of activity was defined as the amount of enzyme needed to release 1 μmol Fuc per min under the conditions described above. Enzymatic reactions were carried out as above but with 0.1 mM substrate and were incubated for 24 h, and the released Fuc were confirmed by HPAEC-PAD using a Dionex ICS 5000 system (Thermo Scientific, Hemel Hempstead, UK). The sugars were separated on a CarboPac PA1 analytical column protected with a CarboPac PA1 guard column using the following gradient conditions at 1 mL/min: 0-20 min, 18 mM NaOH; 20.1-35 min, 100 mM NaOH; 35.1-50 min, 18 mM NaOH. Enzymatic reactions (20 μL) were also performed against complex glycans and glycoproteins using 10 μM of enzyme and 5 μM of oligosaccharides or FA2G2 (5 ng/μL)78, PLA2 (1 mg/mL), IgG (1 mg/mL) untreated or treated with PNGase B035DRAFT_03341 (10 μM)72 or PNGaseF (5000 units/mL), respectively in 50 mM citrate buffer at pH 6, 37°C for 24 h to release N-glycans. The products were analysed by LC-FD-MS/MS as previously described11. The reactions were stopped by heating 95 °C for 5 min and then dried down using Savant SpeedVac centrifugal evaporator (Thermo Fisher, Wilmington, USA), labelled at the reducing end with procainamide using the glycan labelling kit with sodium cyanoborohydride as the reductant (Ludger, Oxford, UK) and purified using a LudgerClean Procainamide Plate (LC-PROC-96, Ludger, Oxford, UK) to remove the excess dye. The samples were dried down using a Thermo Savant SpeedVac centrifugal evaporator and resuspended in 50 µL of 75% acetonitrile: 25% water. The suspensions were then injected onto a Waters BEH amide column (2.1 ×150 mm, 1.7 µm particle size, 130 Å pore size) at 40 °C on a Dionex Ultimate 3000 UHPLC instrument with a fluorescence detector (λex = 310 nm, λem = 370 nm) coupled to a Bruker Amazon Speed ETD. A 50 mM ammonium formate solution pH 4.4 (Ludger, Oxford, UK) was used as mobile phase A and acetonitrile (Romil, UK) was used as mobile phase B. A 70 min gradient was used at 0.2 mL/min unless otherwise specified, 0-53.5 min, 76% to 51%B, 0.4 mL/min; 53.5-55.5 min, 51% to 0% B; 55.5-57.5 min, 0% B; 57.5-59.5 min, 0% to 76% B; 59.5-65.5 min, 76% B; 65.5-70 min, 76% B, 0.4 mL/min. The Heatmap of enzyme specific activities was constructed via Chiplot (https://www.chiplot.online/). Hierarchical clustering was performed based on Euclidean distance calculated with complete linkage as computing method. Transfucosylation reactions For transfucosylation, enzymatic reactions with 1 μM enzyme (1.43 μM for BaGH2926A), 180 mM GlcNAc and 18 mM pNP-Fuc were incubated in 20% (v/v) DMSO to increase the solubility of pNP-Fuc for 1 h at 37°C. The reactions were stopped by addition of ethanol using three times the volume of the reaction. To assay the capacity of the enzymes to carry out further transfucosylation reactions, 1 μM enzyme (1.43 μM was used for BaGH2926A) was incubated with 180 mM 3FN or 6FN and 18 mM pNP-Fuc in 20% (v/v) DMSO. Thin layer chromatography (TLC) and TLC-electrospray ionisation-mass spectrometry (TLC-ESI-MS) analysis To analyse the products of transfucosylation reactions, standards, Fuc (0.01 µmol), pNP-Fuc (0.03 µmol), GlcNAc (0.25 µmol), 4FN (0.005 µmol), 3FN (0.005 µmol), 6FN (0.005 µmol), and reaction samples with GlcNAc (8 ×0.5 µL) or with 3FN or 6FN (4 ×0.5 µL) were loaded on a 12 cm tall plate (TLC Silica gel 60 F254, Sigma-Aldrich, Germany). The plates were developed using an isopropanol-ammonium hydroxide-water 6:3:1 mixture (namely IPA-NH4OH-H2O) for 3 h or until the frontline of the solvent rose to ca. 11.25 cm. The plate was then dried using a hair dryer and stained using a 5% ethanolic solution of sulphuric acid. Gently heating of the plate allowed the identification of the TLC spots corresponding to controls and reaction products. TLC-ESI-MS of the enzymatic reactions was performed using an Expression Compact Mass Spectrometer (Advion, UK) coupled with a Plate Express reader (Advion, UK) in positive mode to identify the fucosylated reaction products. The enzymatic reactions were analysed through TLC as described above. The analysis was performed in duplicates to stain one TLC plate and use it as a guide to perform the TLC-ESI-MS on the non-stained plate. By comparison with the stained plate, the laser of the Plate Express reader was aimed at the right retention factor (Rf) and the data obtained was analysed using Advion Mass Express software. Comparison of retention factors, in combination with TLC-ESI-MS analysis and NMR allowed identification of reaction products. Nuclear Magnetic Resonance (NMR) spectroscopy An aliquot of the enzymatic reactions (600 μL) was evaporated to dryness and reconstituted in 600 μL of NMR buffer (100 mL D2O containing 0.26 g NaH2PO4, 1.41 g K2HPO4, and 1 mM deuterated trimethylsilyl propionate (TSP) as a reference compound) before 1H-NMR spectroscopic analysis. 1H-NMR spectra were recorded using a 600-MHz Bruker Avance spectrometer fitted with a 5-mm TCI proton-optimised triple resonance NMR inverse cryoprobe and autosampler (Bruker, Bremen, Germany). Sample temperature was controlled at 300 K. Spectra were acquired with 32 scans, a spectral width of 12500 Hz and an acquisition time of 2.6 s. The “noesypr1d” presaturation sequence was used to suppress the residual water signal with a low-power selective irradiation at the water frequency during the recycle delay. Spectra were then transformed with a 0.3-Hz line broadening and zero filling, manually phased, baseline corrected, and referenced by setting the TSP–d4 signal to 0 ppm. Reaction products were identified by comparison with the spectra of standards (GlcNAc, pNP-Fuc, Fucose, 6FN, 3FN and 4FN). STD NMR All NMR binding experiments were performed at 278 K on a Bruker Avance III 800 MHz spectrometer equipped with a 5-mm TXI 800 MHz H-C/N-D-05 Z BTO probe (Bruker, Bremen, Germany). First, FA2G2 was spectroscopically characterised by standard COSY (cosydfesgpph), TOCSY (mlvevphpp), 1H-13C HSQC (hsqctgpsp) and NOESY (noesygpph) for the purpose of assignment. Then, FA2G2 was recovered and prepared in a Shigemi advanced NMR microtube assembly at the concentration of ~200 μM in the presence of ~20 uM BaGH2926A (protein:ligand ratio 1:20), in D2O buffer solution containing 25 mM Tris-d11 pH 7.8 and 100 mM NaCl. An STD NMR pulse sequence including 2.5 ms and 5 ms trim pulses and a 3 ms spoil gradient was used. Saturation was achieved by applying a train of 50 ms Gaussian pulses (0.40 mW) on the f2 channel, at 6.70 ppm (on-resonance experiments) and 40 ppm (off-resonance experiments). The broad protein signals were removed using a 40 ms spinlock (T1ρ) filter. As a first test for binding, an STD NMR experiment with a saturation time of 2 s and a relaxation delay of 5 s was performed. Then, an STD build up curve was performed, by carrying out STD experiments at different saturation times (0.5, 1, 2, 3, 4 and 5 s) with 2 K scans, in order to obtain the binding epitope mapping. The resulting build-up curves for each proton were fitted mathematically to a mono-exponential equation (y = a*[1-exp(b*x)]), from which the initial slopes (a*b) were obtained. Finally, the binding epitope mapping was obtained by dividing the initial slopes by the strongest signal corresponding to the methyl group of GlcNAc A, to which an arbitrary value of 100% was assigned. X-ray crystallography BaGH2926A was dialysed into 20 mM Tris 150 mM NaCl. Sitting drop vapour diffusion plates were set up with a protein concentration of 20 mg/mL and 5 mM 2’FL. Crystals appeared in many conditions across commercial sparse matrix screens with the best diffracting crystals appearing in the following condition: 0.12 M diethylene glycol, 0.12 M triethylene glycol, 0.12 M tetraethylene glycol, 0.12 M pentaethylene glycol, 100 mM Tris(base)/bicine pH 8.5, 12.5% 2-methyl-2,4-pentanediol, 12.5% PEG 1000, 12.5% w/v PEG 3350. Diffraction datasets were collected at Diamond Light Source on beamline I24 at a wavelength of 0.9686 Å. Attempts were made to soak out the resulting Fuc molecule bound in the active site so that additional complexes could be attained. These attempts were unsuccessful. By seeding with WT BaGH2926A crystals we also grew diffracting crystals of BaGH2926AD218N active site mutant. These crystals were grown in 0.1 M sodium acetate pH 4.6, 8% (w/v) PEG 4000. Data were processed using xia279 and dials80. The phase problem was solved by molecular replacement using the search model 1ODU, prepared using Chainsaw81. Initial model building was performed using ArpWarp82, followed by alternating cycles of model building and refinement using coot83, refmac84, and PDBredo85. The refined WT BaGH2926A structure has 0.3% ramachandran outliers. The final D218N BaGH2926A structure has 0.05% ramachandran outliers. Reporting summary Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article.']"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "method_sections"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Sigma 65 70 ORG\n",
      "Glycom/DSM 187 197 ORG\n",
      "Elicityl 340 348 ORG\n",
      "Biosynth Ltd 856 868 ORG\n",
      "FA2G2 N-glycan 884 898 PRODUCT\n",
      "Ludger 908 914 ORG\n",
      "Thermofisher 1009 1021 ORG\n",
      "Newcastle University 1178 1198 ORG\n",
      "Sigma 1281 1286 ORG\n",
      "R. gnavus E1 1348 1360 PRODUCT\n",
      "R. gnavus 1383 1392 PRODUCT\n",
      "CAZy 1586 1590 ORG\n",
      "GH29 1633 1637 PRODUCT\n",
      "CAZy 1657 1661 ORG\n",
      "the Enzyme Function Initiative-Enzyme Similarity Tool 1861 1914 PRODUCT\n",
      "Cytoscape 3.9.1 2024 2039 PRODUCT\n",
      "GH29 2045 2049 PRODUCT\n",
      "GH29BERT 2158 2166 PRODUCT\n",
      "GH29 2608 2612 PRODUCT\n",
      "InterPro 2684 2692 ORG\n",
      "Transformer 2875 2886 PRODUCT\n",
      "NVIDIA 3844 3850 ORG\n",
      "A100 40GB 3851 3860 PRODUCT\n",
      "ProtT5 model29 5763 5777 PRODUCT\n",
      "GH29BERT 5882 5890 PRODUCT\n",
      "GH29 6428 6432 PRODUCT\n",
      "Uniform Manifold Approximation and Projection 6596 6641 PRODUCT\n",
      "Python 6946 6952 PRODUCT\n",
      "Prozomix 7311 7319 ORG\n",
      "NZYTech 7401 7408 ORG\n",
      "Cytiva 7901 7907 ORG\n",
      "NuPAGE Novex 8026 8038 PRODUCT\n",
      "Bis-Tris 8045 8053 PRODUCT\n",
      "Life Technologies 8060 8077 ORG\n",
      "NanoDrop 8135 8143 PRODUCT\n",
      "Thermo Scientific 8145 8162 ORG\n",
      "FLUOstar Omega 8816 8830 PRODUCT\n",
      "BMG LABTECH 8832 8843 ORG\n",
      "GraphPad 9143 9151 ORG\n",
      "Megazyme 9586 9594 ORG\n",
      "FLUOstar Omega 9641 9655 PRODUCT\n",
      "BMG LABTECH 9657 9668 ORG\n",
      "HPAEC-PAD 10326 10335 PRODUCT\n",
      "Dionex 10344 10350 ORG\n",
      "ICS 5000 10351 10359 PRODUCT\n",
      "Thermo Scientific 10368 10385 ORG\n",
      "CarboPac PA1 10440 10452 PRODUCT\n",
      "CarboPac PA1 10488 10500 PRODUCT\n",
      "Savant SpeedVac 11151 11166 PRODUCT\n",
      "Thermo Fisher 11191 11204 ORG\n",
      "Ludger 11349 11355 ORG\n",
      "a LudgerClean Procainamide Plate 11388 11420 PRODUCT\n",
      "LC-PROC-96 11422 11432 PRODUCT\n",
      "Ludger 11434 11440 ORG\n",
      "Thermo Savant SpeedVac 11516 11538 ORG\n",
      "Waters 11661 11667 ORG\n",
      "Dionex 11752 11758 ORG\n",
      "Ultimate 3000 11759 11772 PRODUCT\n",
      "Bruker 11861 11867 ORG\n",
      "Amazon Speed ETD 11868 11884 PRODUCT\n",
      "Ludger 11928 11934 ORG\n",
      "Romil 11993 11998 ORG\n",
      "Chiplot 12331 12338 ORG\n",
      "an Expression Compact Mass Spectrometer 13929 13968 PRODUCT\n",
      "Advion 13970 13976 ORG\n",
      "Plate Express 13997 14010 PRODUCT\n",
      "Advion 14019 14025 ORG\n",
      "TLC 14219 14222 ORG\n",
      "Plate Express 14358 14371 PRODUCT\n",
      "Advion Mass Express 14465 14484 PRODUCT\n",
      "Bruker Avance 14996 15009 PRODUCT\n",
      "Bruker 15119 15125 ORG\n",
      "Fucose 15728 15734 PRODUCT\n",
      "6FN 15736 15739 PRODUCT\n",
      "3FN 15741 15744 PRODUCT\n",
      "4FN 15749 15752 PRODUCT\n",
      "Bruker Avance III 15820 15837 PRODUCT\n",
      "Bruker 15916 15922 ORG\n",
      "FA2G2 15949 15954 PRODUCT\n",
      "Shigemi 16164 16171 ORG\n",
      "xia279 18451 18457 PRODUCT\n",
      "dials80 18462 18469 PRODUCT\n",
      "1ODU 18548 18552 PRODUCT\n",
      "Chainsaw81 18569 18579 PRODUCT\n",
      "ArpWarp82 18624 18633 PRODUCT\n",
      "coot83 18705 18711 PRODUCT\n",
      "refmac84 18713 18721 PRODUCT\n",
      "PDBredo85 18727 18736 PRODUCT\n"
     ]
    }
   ],
   "source": [
    "doc = nlp(method_sections[0])\n",
    "for ent in doc.ents:\n",
    "    if ent.label_ == 'ORG' or ent.label_ == 'PRODUCT':\n",
    "        print(ent.text, ent.start_char, ent.end_char, ent.label_)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "doc = nlp(method_sections[0])\n",
    "products = []\n",
    "for ent in doc.ents:\n",
    "    if ent.label_ == 'PRODUCT' and ent.text not in products:\n",
    "        products.append(ent.text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['FA2G2 N-glycan',\n",
       " 'R. gnavus E1',\n",
       " 'R. gnavus',\n",
       " 'GH29',\n",
       " 'the Enzyme Function Initiative-Enzyme Similarity Tool',\n",
       " 'Cytoscape 3.9.1',\n",
       " 'GH29BERT',\n",
       " 'Transformer',\n",
       " 'A100 40GB',\n",
       " 'ProtT5 model29',\n",
       " 'Uniform Manifold Approximation and Projection',\n",
       " 'Python',\n",
       " 'NuPAGE Novex',\n",
       " 'Bis-Tris',\n",
       " 'NanoDrop',\n",
       " 'FLUOstar Omega',\n",
       " 'HPAEC-PAD',\n",
       " 'ICS 5000',\n",
       " 'CarboPac PA1',\n",
       " 'Savant SpeedVac',\n",
       " 'a LudgerClean Procainamide Plate',\n",
       " 'LC-PROC-96',\n",
       " 'Ultimate 3000',\n",
       " 'Amazon Speed ETD',\n",
       " 'an Expression Compact Mass Spectrometer',\n",
       " 'Plate Express',\n",
       " 'Advion Mass Express',\n",
       " 'Bruker Avance',\n",
       " 'Fucose',\n",
       " '6FN',\n",
       " '3FN',\n",
       " '4FN',\n",
       " 'Bruker Avance III',\n",
       " 'FA2G2',\n",
       " 'xia279',\n",
       " 'dials80',\n",
       " '1ODU',\n",
       " 'Chainsaw81',\n",
       " 'ArpWarp82',\n",
       " 'coot83',\n",
       " 'refmac84',\n",
       " 'PDBredo85']"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "products"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "instruct_string = \"\"\"You are an expert Physical Commercial Product Recognition system.\n",
    "Your task is to accept a list of strings as input and decide which of these are physical commercial products.\n",
    "Here is an example of the output format for a list of strings.\n",
    "\n",
    "List of strings: ['Python', '3M Littmann Stethoscope', 'Omron Blood Pressure Monitor', 'Microsoft Office', 'Braun ThermoScan', 'SPSS']\n",
    "Answer: ['3M Littmann Stethoscope', 'Omron Blood Pressure Monitor', 'Braun ThermoScan']\n",
    "\n",
    "Only use this output format. Do not output anything besides this output format.\n",
    "Output physical commercial products in the order they occur in the input list of strings.\n",
    "\n",
    "Q: Given the list of strings below, identify the physical commercial products.\n",
    "\n",
    "List of strings: {}\n",
    "Answer:\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "You are an expert Physical Commercial Product Recognition system.\n",
      "Your task is to accept a list of strings as input and decide which of these are physical commercial products.\n",
      "Here is an example of the output format for a list of strings.\n",
      "\n",
      "List of strings: ['Python', '3M Littmann Stethoscope', 'Omron Blood Pressure Monitor', 'Microsoft Office', 'Braun ThermoScan', 'SPSS']\n",
      "Answer: ['3M Littmann Stethoscope', 'Omron Blood Pressure Monitor', 'Braun ThermoScan']\n",
      "\n",
      "Only use this output format. Do not output anything besides this output format.\n",
      "Output physical commercial products in the order they occur in the input list of strings.\n",
      "\n",
      "Q: Given the list of strings below, identify the physical commercial products.\n",
      "\n",
      "List of strings: ['FA2G2 N-glycan', 'R. gnavus E1', 'R. gnavus', 'GH29', 'the Enzyme Function Initiative-Enzyme Similarity Tool', 'Cytoscape 3.9.1', 'GH29BERT', 'Transformer', 'A100 40GB', 'ProtT5 model29', 'Uniform Manifold Approximation and Projection', 'Python', 'NuPAGE Novex', 'Bis-Tris', 'NanoDrop', 'FLUOstar Omega', 'HPAEC-PAD', 'ICS 5000', 'CarboPac PA1', 'Savant SpeedVac', 'a LudgerClean Procainamide Plate', 'LC-PROC-96', 'Ultimate 3000', 'Amazon Speed ETD', 'an Expression Compact Mass Spectrometer', 'Plate Express', 'Advion Mass Express', 'Bruker Avance', 'Fucose', '6FN', '3FN', '4FN', 'Bruker Avance III', 'FA2G2', 'xia279', 'dials80', '1ODU', 'Chainsaw81', 'ArpWarp82', 'coot83', 'refmac84', 'PDBredo85']\n",
      "Answer:\n",
      "\n"
     ]
    }
   ],
   "source": [
    "print(instruct_string.format(products))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['NuPAGE Novex', 'Bis-Tris', 'NanoDrop', 'FLUOstar Omega', 'HPAEC-PAD', 'ICS 5000', 'CarboPac PA1', 'Savant SpeedVac', 'Ultimate 3000', 'Bruker Avance III', 'Chainsaw81']"
     ]
    }
   ],
   "source": [
    "import ollama\n",
    "\n",
    "stream = ollama.chat(\n",
    "    model='llama3',\n",
    "    messages=[{'role': 'user', 'content': instruct_string.format(products)}],\n",
    "    stream=True,\n",
    ")\n",
    "\n",
    "for chunk in stream:\n",
    "  print(chunk['message']['content'], end='', flush=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ['NuPAGE Novex', 'NanoDrop', 'FLUOstar Omega', 'ICS 5000', 'CarboPac PA1', 'Savant SpeedVac', 'a LudgerClean Procainamide Plate', 'LC-PROC-96', 'Ultimate 3000', 'Amazon Speed ETD', 'an Expression Compact Mass Spectrometer', 'Plate Express', 'Advion Mass Express', 'Bruker Avance', 'Fucose']"
     ]
    }
   ],
   "source": [
    "stream = ollama.chat(\n",
    "    model='mistral',\n",
    "    messages=[{'role': 'user', 'content': instruct_string.format(products)}],\n",
    "    stream=True,\n",
    ")\n",
    "\n",
    "for chunk in stream:\n",
    "  print(chunk['message']['content'], end='', flush=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "import openai \n",
    "openai.api_key = os.environ['OPENAI_API_KEY']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "system_prompt = \"\"\"You are an expert Physical Commercial Product Recognition system.\n",
    "Your task is to accept a list of strings as input and decide which of these are physical commercial products.\n",
    "Here is an example of the output format for a list of strings.\n",
    "\n",
    "List of strings: ['Python', '3M Littmann Stethoscope', 'Omron Blood Pressure Monitor', 'Microsoft Office', 'Braun ThermoScan', 'SPSS']\n",
    "Answer: ['3M Littmann Stethoscope', 'Omron Blood Pressure Monitor', 'Braun ThermoScan']\n",
    "\n",
    "Only use this output format. Do not output anything besides this output format.\n",
    "Output physical commercial products in the order they occur in the input list of strings.\n",
    "\"\"\"\n",
    "\n",
    "user_prompt = \"\"\"Q: Given the list of strings below, identify the physical commercial products.\n",
    "\n",
    "List of strings: {}\n",
    "Answer:\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "response = openai.chat.completions.create(\n",
    "        model=\"gpt-4\",\n",
    "        messages=[\n",
    "            {\"role\": \"system\", \"content\": system_prompt},\n",
    "            {\"role\": \"user\", \"content\": user_prompt.format(products)}\n",
    "        ]\n",
    "    )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['A100 40GB', 'NuPAGE Novex', 'Bis-Tris', 'NanoDrop', 'FLUOstar Omega', 'ICS 5000', 'CarboPac PA1', 'Savant SpeedVac', 'Ultimate 3000', 'Amazon Speed ETD', 'an Expression Compact Mass Spectrometer', 'Plate Express', 'Advion Mass Express', 'Bruker Avance', 'Bruker Avance III']\n"
     ]
    }
   ],
   "source": [
    "print(response.choices[0].message.content)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
