{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Extract Method Section"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from PubMed.PmcScraper import PmcScraper\n",
    "import os\n",
    "import ast\n",
    "import spacy\n",
    "import openai\n",
    "from mistralai.client import MistralClient\n",
    "from mistralai.models.chat_completion import ChatMessage"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# python -m spacy download en_core_web_trf\n",
    "nlp = spacy.load(\"en_core_web_trf\")\n",
    "\n",
    "# need .env file to store ncbi api key\n",
    "pmc_scraper = PmcScraper(os.environ['NCBI_API_KEY'])\n",
    "\n",
    "# search based on keyword and retrieve the desired amount of articles\n",
    "search_results = pmc_scraper.search_pmc(search_term=\"gut microbiota\", number_of_articles=1)\n",
    "\n",
    "# extract the full text version of articles using pmcids\n",
    "full_text_articles = pmc_scraper.fetch_full_text(search_results['esearchresult']['idlist'])\n",
    "\n",
    "# extract the desired section of articles\n",
    "method_sections = pmc_scraper.extract_section(full_text_articles, 'method')\n",
    "\n",
    "# print as dictionary\n",
    "method_dict = dict(zip(search_results['esearchresult']['idlist'], method_sections))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'11208022': 'METHODS Animals, Diets, and Baseline Liver Biopsy To obtain diet-induced obese NASH mice (DIO-NASH), male C57BL/6JRj mice aged 5 wk old were fed an amylin diet [AMLN, containing 40% fat (18% trans-fat), 40% carbohydrate (20% fructose), and 2% cholesterol; D09100310, Research Diets] (17) ad libitum for 43 wk to induce advanced histopathological features of NASH. A baseline liver biopsy was performed 3 wk before the drug treatment period as described previously (18). Only those mice (DIO-NASH cohort) with histologically confirmed steatosis ≥ 2 by hematoxylin-eosin stain (H&E) and fibrosis stage ≥ 1 by Picro-Sirius Red (PSR) (19) were stratified and randomly assigned to the treatment groups using a biopsy-based evaluation of liver type 1 collagen α1 (Col1a1) levels. Age-matched normal Chow mice were fed a Chow diet (Altromin 1324, Brogaarden, Denmark) for 43 wk. Throughout the study, the treatment group allocation was blinded to individuals who performed the study. Body weights were measured daily throughout the treatment period and food intake was assessed daily over the first 2 wk and then weekly over the next 10 wk. The AMLN diet was continued during the treatment period for a total of 55 wk. The overall study design is outlined in Supplemental Fig. S1. All experiments were approved by The Animal Experimentation Council, Danish Veterinary and Food Administration (license no. 2013-15-2934-00784) and the Gubra ethics committee. Animal experiments were conducted at Gubra (Hørsholm, Denmark) in compliance with internationally accepted principles for the care and use of laboratory animals. Drug Treatment Mice were treated daily for 12 wk with vehicle (once a day, orally), seladelpar (10 mg/kg, once a day, orally) (11), liraglutide (0.2 mg/kg, twice a day, subcutaneous) (20), selonsertib (30 mg/kg, twice a day, orally) (21), obeticholic acid (OCA, 30 mg/kg, once a day, orally) (20), alone or in combinations of seladelpar with liraglutide or selonsertib (n = 12 mice/group). Carboxymethyl cellulose (0.5%) was used as an oral dosing vehicle and phosphate-buffered saline with 0.1% bovine serum albumin was used as a vehicle for liraglutide subcutaneous dosing. Chow-fed mice dosed with oral vehicle (once a day, orally) served as a normal, non-NASH comparison group (Chow Vehicle, n = 10). Plasma and Liver Biochemistry Levels of plasma triglyceride (TG), total cholesterol (TC), alanine aminotransferase (ALT), aspartate aminotransferase (AST), and liver TG and TC were measured in a single run from blood and liver tissue taken at euthanasia as described previously (18). For liver hydroxyproline measurement, ∼100 mg of each liver sample was homogenized and hydrolyzed in 6 M HCl at 95°C for 16–20 h. Hydroxyproline in the supernatants was measured twice by colorimetric determination using a Hydroxyproline Assay Kit (Sigma-Aldrich). Liver Histopathology and Immunohistochemistry for Pre- and Posttreatment Paired Liver Biopsies Pre- and posttreatment liver biopsies were formaldehyde-fixed, paraffin-embedded, and stained with H&E and PSR, and scored in a blinded manner by a pathologist according to the criteria developed by Kleiner et al. (19) The liver slides were evaluated for steatosis (0–3), inflammation (0–3), ballooning (0–2), the nonalcoholic fatty liver disease (NAFLD) activity score (NAS, sum of individual components, 0–8), and fibrosis stage (stage 0–4). Immunohistochemistry (IHC) staining of Col1a1 (Southern Biotech, Cat. No. 1310-01, 1:100 dilution), α-smooth muscle actin (α-SMA, Abcam, Cat. No. Ab124964, 1:800 dilution), collagen IV (Col IV, Southern Biotech, Cat. No. 1340-01, 1:100 dilution), or laminin (Dako, Cat. No. Z0097, 1:800 dilution) for fibrosis, and Galectin-3 for fibrosis/inflammation (Biolegend, Cat. No. 125402, 1:50,000 dilution) was performed according to the manufacturer’s instructions. Stained liver slides were quantitatively assessed using digital imaging software Visiomorph (Visiopharm) and expressed as the fractional area (%) relative to total tissue area. For quantitative assessment of fibrosis and inflammation, PSR- and IHC-stained area was expressed as fractional area (%) relative to total tissue area after subtracting corresponding fat area by steatosis image analysis. Heavy Water Labeling and Determination of Hepatic Collagen Fractional Synthesis Rate Hepatic new collagen synthesis was measured from its fractional synthesis rate (FSR). Mice were given a priming dose of 100% heavy water (D2O) in 0.9% NaCl by intraperitoneal injection to immediately set the body water enrichment level to ∼5%. This level was then maintained by providing the mice with 8% D2O in the daily drinking water for the last 7 days before euthanasia. The detailed methods for heavy water labeling and hepatic collagen synthesis measurement in vivo were described previously (22). RNA Transcriptomics Purified RNA (10 ng–1 μg) from terminal liver (n = 8 per group) was used to generate a single cDNA library for each mouse using the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina (New England BioLabs, Inc., Ipswich, MA). The cDNA libraries were then sequenced on a NextSeq 500 using NextSeq 500/550 High Output Kit V2 (Illumina, San Diego, CA). The sequencing data were aligned to the genome of the GRCm38 v89 Ensembl Mus musculus obtained from the Ensembl database using the Spliced Transcripts Alignment to a Reference (STAR) software with default parameters. For the bioinformatic analysis, the quality of the data was evaluated using the standard RNA-sequencing quality control parameters, the inter- and intragroup variability was evaluated using principal component analysis (PCA) using R package FactoMineR and hierarchical clustering, and the differentially expressed genes were identified using the R-package DESeq2. RNAseq data were presented as Reads Per Kilobase per Million mapped reads (RPKM). Statistical Analyses JMP statistical software (version 17; SAS Institute, Cary, NC) was used for statistical analysis. Graphs were created in PRISM (v. 9; GraphPad, San Diego, CA). Data were presented as means ± SE. One-way analysis of variance (ANOVA) followed by a post hoc Student’s t test was used to evaluate the difference between treatment groups. Statistically significant differences between treatment groups are presented using different letters; whenever treatment groups do not have a common letter (a, b, c, d, e, or f), they are statistically different (P value < 0.05). Pre- and posttreatment NAFLD activity score (NAS) comparison with the seladelpar group was made by analysis of covariance (ANCOVA) using treatment as a factor and prebiopsy NAS as a covariate. Comparison with the seladelpar group for a decrease of NAS by ≥ 2 points was performed using Fisher’s exact test. The RNAseq data in this study have been deposited in the NCBI sequence read archives (SRA) under BioProject accession number PRJNA1032138.',\n",
       " '11207862': '3. Materials and Methods 3.1. Experimental Design and Sample Collection The experimental sites of this study were located in the Tea Tree Germplasm Resource Nursery of Wuyi University, Wuyishan City, Fujian Province, China (117°59′51.88″ E, 27°44′16.17″ N), and the age of all the tea trees was 10 years. The experimental site was located at an altitude of 216 m, with an average annual temperature of 19 °C, an annual rainfall of 1284 mm, and an average humidity of 74%. In October 2022, the rhizosphere soils and leaves of 44 tea tree varieties (Table 1 and Figure 10) were collected, of which soil was used to determine available nutrient content; amplicon sequencing analysis of bacterial diversity and subsequent qRT-PCR analysis of important characteristic microorganisms and leaves were used to determine tea quality indexes. The rhizosphere soil of the tea tree adopted an S sampling method; that is, five tea trees were randomly selected, the fallen leaves on the soil surface were removed, the tea tree was gently dug out, the root was removed, the soil adhering to the root surface was shaken off, and the soil still adhering to the root was collected as the rhizosphere soil of the tea tree, which was then appropriately mixed to be a replicate. Three independent replicates were taken for each tea tree variety. The sampling method for tea tree leaves was to collect the leaves of the corresponding tea trees while collecting the rhizosphere soil of the tea trees, i.e., one bud and two leaves of five randomly selected tea trees, and mixing them thoroughly to make a replicate. Three independent replicates were taken for each tea tree variety. 3.2. 16S rDNA Amplicon Sequencing Analysis DNA extraction: Rhizosphere soil microbial DNA from tea trees was extracted using instructions from the Bio-Fast Soil Genomic DNA Extraction Kit (BioFlux, Hangzhou, China). The extracted DNA was detected using 1% agarose gel electrophoresis, and the DNA was purified using the gel recovery kit of TianGen Biotech Co., Ltd., Beijing, China. DNA concentration was determined using UV spectrophotometry and was used for PCR amplification after quality control. PCR amplification and sequencing: 16S rDNA amplification of soil bacteria referred to the method of Zhang et al. [52]. PCR amplification primers were 338F (ACTCCTACGGGGAGGCAGCAG) and 806R (GGACTACHVGGGTWTCTAAT). The PCR reaction system was 12.5 μL 2xTaq Plus Master Mix, 3 μL BSA (2 ng/μL), 1 μL forward primer (5 μM), 1 μL reverse primer (5 μM), 2 μL DNA (total amount of DNA added was 30 ng), and 5.5 μL dd H2O. The PCR reaction program was pre-denaturation at 94 °C for 5 min; denaturation at 94 °C for 30 s; annealing at 50 °C for 30 s; extension at 72 °C for 60 s, 30 cycles; and extension at 72 °C for 7 min. PCR products were purified using Agencourt AMPure XP Nucleic Acid Purification Kit (VWR, Radnor, PA, USA) and were used to construct microbial diversity sequencing libraries. After library construction, paired-end sequencing was performed using the Illumina Miseq PE300 high-throughput sequencing platform at Beijing Allwegene Technology Co., Ltd., Beijing, China. Bioinformatics analysis: Purified amplicons representing the bacterial 16S rDNA gene sequence reads were performed using Illumina Analysis Pipeline Version 2.6. First, the Fastq data were analyzed for quality control using trimmomatic (v0.36) and pear (v0.9.6), where trimmomatic used a sliding window strategy with the window size set at 50 bp, the average quality value of 20, and the minimum retained sequence length of 120 bp, and pear was used to remove sequences with N. According to the paired-end overlap relationship, Flash (v1.20) and pear software (v0.9.6) were used to splice the two-end sequences, with a minimum overlap setting of 10 bp and a mismatch rate of 0.1, thus obtaining sequences in Fasta format for raw tags. Based on this, the uchime method was further used to compare and remove chimeras from Fasta sequences and non-compliant short sequences as a way to obtain clean tags. The purified sequences were clustered into operational taxonomic units (OTUs) at a 97% similarity level against the SILVA v128 reference database set at a minimum support threshold of 70%. The Ribosomal Database Project (RDP) classifier tool was used to classify all sequences into different taxonomic groups. The rarefaction curve, Shannon–Wiener, rank-abundance, and species accumulation curves were produced for the obtained data. A random sampling method was used to construct a rarefaction curve with the number of sequences selected versus the number of OTUs they could represent. A Shannon–Wiener curve was constructed using the bacterial diversity index at different sequencing depths for each sample, as a reflection of the microbial diversity of each sample at different sequencing quantities. The abundance of OTUs was sorted from large to small rank, and then rank-abundance curves were produced using the OTU rank as the horizontal coordinate and the number of sequences contained in each OTU as the vertical coordinate. Species accumulation curves were produced using the sample size as the horizontal coordinate and the rate of emergence of new OTUs under continuous sampling as the vertical coordinate. A diversity analysis such as Shannon, Simpson, Chao1, and PD whole tree was performed using QIIME1 (v1.8.0) software. 3.3. Construction and Evaluation of Machine Learning Models Machine learning algorithms are widely used to sift and categorize big data, but so far there is no single algorithm that works in all situations. Therefore, using multiple algorithms for modeling leads to better construction of high-quality models. In this research, five classical machine learning algorithms, including K-nearest neighbor (KNN), support vector machine (SVM), back propagation neural network (BPNN), random forest (RF), and extreme gradient boosting (XGboost), were used to analyze and model rhizosphere soil bacteria from 44 tea tree germplasm resources (Figure 11). All model construction was based on R software version 4.2.3, and evaluation parameter tables were produced in Excel. In the process of model construction, this study adopts a stratified sampling technique, that is, data samples are divided into training sets and test sets in the ratio of 80% and 20%. To further ensure the stability of the model, 100 randomized iterations of training were conducted in this study, and the training set and test set were randomly combined proportionally in each training iteration. Finally, the model integration analysis was carried out with the results of 100 iterations of training, which integrated all the models from the training process into a comprehensive result, improving the stability and accuracy of the model predictions. The KNN classification algorithm is one of the simplest machine learning algorithms, based on the principle of determining attributes based on the category of the nearest k points; predicting new values is simple, fast, and insensitive to outliers. The choice of k value can have a significant impact on the results of the algorithm. When the k value is small, the overall complexity of the model decreases and it is easy to overfit. When the k value is large, it selects training set instances that are far from the validation set samples, which affects the prediction and makes the prediction inaccurate [68]. In practice, the k value is usually chosen to be smaller and then the optimal k value is selected using cross-validation. In this study, a tuning grid containing k values, where k values range from 1 to 20, is set up for cross-validation and parameter optimization. The core idea of SVM is to find an optimal hyperplane that maximizes the interval between two categories. By introducing a kernel function, SVM is able to handle nonlinear classification problems [69]. In this study, a radial basis kernel function (RBF)-based SVM is used, and in order to obtain the optimal model parameters, a parameter grid containing C and σ values, where both C and σ values range from 10−3 to 103, increasing by one order of magnitude at a time, is defined for cross-validation and parameter optimization. The BPNN classification algorithm is also an effective machine learning algorithm. In BPNN, information is propagated forward through layer-to-layer connections and error backpropagation and weight updating is performed through a backpropagation algorithm to minimize the error between the predicted output and the actual output [70]. In this study, a neural network model of backpropagation with a single hidden layer is designed and constructed. The model has a hidden layer between the input and output layers, which is designed to capture and process complex features in the data, thus enhancing the model’s learning and representation capabilities. RF is a comprehensive supervised classification algorithm that can be used to solve large errors and overfitting problems that may occur with individual decision trees. Second, RF performs well in classification problems and has a great potential to be the classifier that works best in each case [71]. The model consists of many decision trees that are unrelated to each other. This model, after obtaining the forest, when judging or predicting new samples, judges each decision tree in the forest separately to distinguish which category the sample belongs to and compares which category has the highest number of selections, thus making a judgment on the sample category and deciding how many trees should be in this model is crucial [72]. The number of trees in this study was initially set at 500 and a parameter grid containing mtry (number of variables considered per split), where mtry ranges from 1 to 59, was defined for cross-validation and parameter optimization. In addition, the importance scores of the impact of all variables on model construction were derived from the model constructed using RF in order to evaluate the value of the contribution of different variables to model construction and thus obtain the importance characteristic variables. XGboost is an efficient, flexible, and portable gradient enhancement algorithm that has been widely used in various classification and regression tasks. XGboost uses an additive strategy to incrementally add trees and optimizes the objective function at each step using the gradient boosting principle. This allows the model to attempt to better correct the residuals of all previous trees at each step, resulting in more accurate predictions [73]. In order to find the optimal model parameters, this study defines a parameter grid with nrounds (number of augmentation iterations), max_depth (maximum depth of the tree), eta (learning rate), colsample_bytree (the ratio of subsamples of the features used in the construction of each tree), min_child_weight (the minimum weight of the child nodes), and subsample (the ratio of subsamples of the training data) for cross-validation and parameter optimization. In addition, the models constructed using XGboost were used to derive the importance scores of the impact of all variables on model construction in order to evaluate the value of the contribution of the different variables to model construction and thus obtain the importance characteristic variables. After all the models were constructed, this study made a preliminary evaluation of the performance of the models through the confusion matrix and ROC curves. Confusion matrices are often used to evaluate the classification effect of each group, reflecting the relationship between the true category of sample data and the predicted results, and quantifying classification details more intuitively [74]. In this study, after completing the iteration of each machine learning algorithm, model integration was used to calculate the overall confusion matrix of the model and draw a heat map of the confusion matrix. Model integration can be seen as a strategy that combines the predictions of multiple models to produce a more robust and accurate combined prediction [75]. With this approach, the confusion matrix for the model as a whole can be computed to obtain an overall assessment of the predictive performance of all models. To further deepen the understanding of model performance, this study plots the overall ROC curve after completing the iteration of each machine learning algorithm. ROC curves are a common tool for evaluating the performance of binary classifiers, which provide a comprehensive view of the model’s performance by plotting the relationship between true positive and false positive rates at different classification thresholds. The generalization ability of the model is also evaluated by the area under the curve (AUC) in the ROC curve. The AUC value ranges from 0 to 1.0 and is positively correlated with model quality [76,77]. 3.4. qRT-PCR Analysis of Soil Characteristic Bacteria Based on the analysis of the research data in this study, 23 characteristic bacterial genera were obtained. Bacteria that were unknown and temporarily unsearchable in the NCBI database were removed, and the number of bacteria in 18 of these genera was quantitatively analyzed in this study. Briefly, soil DNA was extracted using the Bio-Fast Soil Genomic DNA Extraction Kit (BioFlux, Hangzhou, China), and DNA was detected using 1% agarose gel electrophoresis and was purified using the gel recovery kit of TianGen Biotech Co., Ltd., Beijing, China, to perform qRT-PCR analysis. Fluorescence quantitative PCR curves were performed with different concentrations of plasmids, and copy numbers of microorganisms amplified in different samples were determined and converted to quantitative data by using a real-time fluorescence quantitative PCR system of ABI7500 (Thermo Fisher, Carlsbad, CA, USA). The qRT-PCR primers were designed according to the conserved 16S rDNA sequences from each of the 18 bacterial genera and were then analyzed using PCR, as shown in Table S4. The PCR programs were all set to 94 °C pre-denaturation for 5 min, 94 °C for 15 s, 60 °C for 30 s, 72 °C for 15 s, and 30 cycles. 3.5. Determination of Available Nutrient Content of Rhizosphere Soils of Tea Trees Rhizosphere soils collected from 44 different tea tree varieties were used to determine the available nitrogen, available phosphorus, and available potassium content of the soils, as described in Wang et al. [78]. The available nitrogen content was determined using alkalolytic diffusion, i.e., the soil was extracted by NaOH solution (1 mol/L), filtered, and the filtrate was then titrated with hydrochloric acid, which was converted to obtain the available nitrogen content of the soil. The available phosphorus content was determined using molybdenum–antimony resistance colorimetry, i.e., the soil was leached with NaHCO3 solution (0.5 mol/L), filtered, the filtrate was added to molybdenum–antimony resistance color reagent, and then the absorbance was measured at 880 nm, which was converted to obtain the available phosphorus content of the soil. The available potassium content was determined using flame photometry, i.e., the soil was leached with neutral ammonium acetate (1 mol/L), filtered, and the filtrate was directly determined using flame photometry and converted to obtain the available potassium content of the soil. 3.6. Determination of Tea Quality Index Content The collected leaves of 44 different varieties of tea trees were bioinactivated at 105 °C for 15 min, dried at 80 °C until constant weight, ground, and passed through a 60-mesh sieve for the determination of quality indexes. The quality indexes of tea leaves were mainly determined for tea polyphenols, theanine, and caffeine, and three independent replicates were determined for each quality index. Among them, the tea polyphenol content was determined using UV spectrophotometry with reference to the National Standard of the People’s Republic of China, GB/T 8313-2018 [79]. Briefly, 1 g of tea sample was weighed and 5 mL of 70% methanol solution was added and kept warm at 70 °C for 10 min. The above solution was centrifuged, 1 mL of supernatant was taken, and 5 mL of Folin–Ciocalteu reagent was added. After 5 min of reaction, 4 mL of 7.5% Na2CO3 solution was added, and the absorbance was measured at 765 nm after 60 min of standing. Tea polyphenols were quantified with reference to gallic acid as a standard curve. Theanine content was determined using high-performance liquid chromatography (HPLC) with reference to the National Standard of the People’s Republic of China, GB/T 23193-2017 [80]. Briefly, 1 g of tea sample was weighed, added to 100 mL of boiling distilled water, subjected to a water bath at 100 °C for 30 min, and was then filtered and fixed. The above filtrate was passed through a 0.45 μm filter membrane and subjected to high-performance liquid chromatography (HPLC) for the determination and quantification of theanine with reference to the standard curve of theanine. Caffeine content was determined using UV spectrophotometry with reference to the National Standard of the People’s Republic of China, GB/T 8312-2013 [81]. Briefly, 3 g of tea sample was weighed, added to 450 mL of boiling distilled water, and kept in a water bath at 100 °C for 45 min, before being filtered and concentrated. A total of 10 mL of filtrate was taken, 4 mL of 0.01 mol/L hydrochloric acid was added, the volume was then fixed to 100 mL, and was filtrated for standing, and the absorbance was measured at 274 nm to quantify caffeine with reference to the standard curve of caffeine. 3.7. Statistical Analysis Excel 2017 software was used to preprocess raw data with routine graphic production, including calculation of mean, variance, etc., as well as production of stacked and bar charts. Rstudio software (R version 4.2.3) was used to produce K-mean clustering maps (R library was vegan 2.6.4 and fpc 2.2.10), Bray–Curtis heat maps (R library was pheatmap 1.0.12), the orthogonal partial least squares-discriminate analysis model (OPLS-DA, package used for this was ropls and mixOmics), box plots (R library was gghalves 0.1.4), scatter plots (R library was ggplot 2 3.4.0), redundancy analysis (R library was vegan 2.6.4), interaction network diagrams (R library was linkET 0.0.7.1), and the partial least square—structural equation model (PLS-SEM, library was plspm version 0.5.0).',\n",
       " '11207762': '3. Materials and Methods 3.1. Extract Preparation J. nigra leaves and fruits were collected during the summer (2021) at Aleksinac locality, in the southeast region of Serbia. The voucher specimen was deposited at the Herbarium of the Department of Botany, University of Belgrade Faculty of Pharmacy, under the number 3906HFF. The leaves and green husk were dried in the air and grounded, thus obtaining plant material particles of an average size of 0.75 mm. The grounded plant material was mixed with 70% (v/v) ethanol for 4 h under reflux at a solvent-to-solid ratio of 4:1. The suspension of plant particles in the solvent was cooled to room temperature, taken from the flask, and filtered under vacuum to separate the liquid extract from the solid residue. 3.2. The Electrochemical Measurements Reagent-grade calf–thymus DNA (DNA) was supplied by Sigma Aldrich Chemical Co., St. Louis, MI, USA. The electrochemical measurements (cyclic voltammetry, CV) were performed using a potentiostat/galvanostat Autolab PGSTAT 302 N (MetrohmAutolab B.V., Utrecht, The Netherlands) controlled by Nova 2.0 software. Electrochemical measurements were conducted in three electrode glass cells (total volume of 20 mL) with a glassy carbon working electrode, Ag/AgCl electrode (3 M KCl) as the reference electrode, and a Pt wire as the counter electrode. Each potential reported in this paper is given against the Ag/AgCl/3 M KCl electrode at ambient temperature (25 + 1 °C). DNA-binding interactions were examined in two ways. First, several extracts (4 mg/mL) were added to the DNA solution at a concentration of 20 mg/mL to confirm the binding of the extracts to DNA. Second, to study the nature and constant for DNA/extract binding, a known amount of DNA was added to the extracts, and the current reduction was monitored. 3.3. Subjects Venous blood samples were obtained using heparinized sterile vacutainers (Becton Dickinson, Franklin Lakes, NJ, USA) from four healthy, nonsmoking male volunteers who had not been exposed to chemicals, drugs, or other substances. A safety protocol concerning a blood-borne pathogen/biohazard was used. The volunteers gave permission to use their blood for the experiment. From each subject, two 5 mL aliquots of blood were obtained. The study complied with the code of ethics of the World Medical Association (Helsinki Declaration of 1964, as revised in 2002). The blood samples were obtained at the Medical Unit in accordance with the current health and ethical regulations in Serbia (Law on Health Care, Serbia, 2019). 3.4. Characterization of Extracts 3.4.1. Total Extracted Substances The solvent obtained from the extract was then evaporated in a rotary vacuum evaporator until a half-solid residue was obtained, which was then dried at 60 °C to a constant weight. The dry residue represents the total extracted substances. 3.4.2. Total Polyphenolic Content Determination The total phenolic (TP) content in the extracts was determined by a VIS spectrophotometer (SPECO1) according to the Folin–Ciocalteu method using gallic acid as a standard. The tested extract (20 × 10−3 cm−3) and Folin–Ciocalteu reagent (1 cm−3) were placed in a 10 cm−3 volumetric flask. Aliquots (0.8 cm−3) of 7.5% aqueous Na2CO3 solution were added to the solution, and the reaction mixture was increased up to 10 cm−3 with distilled water. The absorbance of the mixture was measured after 30 min at 765 nm. For the quantification of total phenolic content, an external standard method was used with a standard of gallic acid at 765 nm (y = 1.7907x + 0.0244, R2 = 0.9983). The total phenolic content of the extract was expressed as gallic acid equivalents in mg per cm−3 extract (mg GA cm−3). 3.4.3. Total Flavonoid Determination The total flavonoid (TF) content in the extracts was determined with a VIS spectrophotometer (SPECO1) according to Al2O3 using rutin as a standard. Aliquots (0.8 cm−3) of 10% aqueous Al2O3 solution and 1 mol dm−3 aqueous solution CH3COOK (0.1 cm−3) were added to the tested extract (2 cm−3). The absorbance of the mixture was measured after 30 min at 430 nm. For the quantification of total flavonoid content, an external standard method was used with a standard of rutin at 430 nm (y = 7.2328x − 0.2286, R2 = 0.9919). The total flavonoid content of the extract was expressed as rutine equivalents in mg per cm−3 extract (mg R cm−3). 3.4.4. DPPH Radical Scavenging Capacity In order to evaluate the antioxidant activity of the extract from J. nigra leaves and green husks, the DPPH radical (1,1-diphenyl-2-picrylhydrazyl free radical) scavenging capacity was measured. Extracted solutions (0.3 cm−3, 20–200 μg cm−3) were incubated with DPPH solution (2.7 cm−3, 90 μmol dm−3) for 30 min in the dark, and afterwards the absorbance was measured at 517 nm using a VIS spectrophotometer (SPECO1). A blank control of the ethanol/water mixture was run in each assay. Inhibition of DPPH radical was calculated as a percentage (%) using the following equation:Scavenging effect (%) =ADPPH−ASADPPH×100 where AS is the absorbance of the DPPH solution when the sample extract was added and ADPPH is the absorbance of the DPPH solution. The extract concentration providing 50% inhibition (IC50) was calculated from the graph representing the dependence of the scavenging effect on the concentration of extracts. The IC50 value was obtained by a linear regression equation for the extract of leaves (y = 669.59x + 0.1, R2 = 0.9999) and green husks (y = 431.95x + 3.619, R2 = 0.9864). IC50 was expressed in µg cm−3. The measurements were performed in triplicate, and the data were presented as average ± standard deviations (SD). 3.4.5. LC/MS Method for Metabolite Identification LC-HRMS/MS (Thermo Scientific™ Vanquish™ Core HPLC system coupled to the Orbitrap Exploris 120 mass spectrometer, San Jose, CA, USA) was used to determine the metabolic profile of the extracts. The liquid chromatography system was equipped with a Hypersil GOLD™ C18 analytical column (50 × 2.1 mm, 1.9 μm particle size), thermostated at 40 °C. The injection volume was 5 μL, and the flow rate was constant at 300 μL/min. The compounds of interest were eluted with ultrapure water + 0.1% formic acid (A) and acetonitrile (MS grade) + 0.1% formic acid (B): 5% B in the first min; 5–95% B from 1 to 10 min; 95% B from 10 to 12 min; 5% B until 15 min. The used characterization technique was previously optimized by the researchers [25]. The Orbitrap Exploris 120 mass spectrometer was equipped with a heated electrospray ionization (HESI-II) source operating in negative ionization modes. Full-scan MS was monitored from 100 to 1500 m/z with an Orbitrap resolution set to 60,000 FWHM, while data-dependent MS2 experiments were conducted at an Orbitrap resolution of 15,000 FWHM. Normalized collision energy was set to 35% with an isolation width of 1.5 m/z. The dynamic exclusion time was set to 10 s with exclusion from a specific scan after two occurrences, and the intensity threshold was set to 1 × 105. LC/MS dates were evaluated using R Studio (version 2023.09.1, build 494) software. Peak picking was performed using the enviPick R package, and peak correspondence across samples was performed using the density method available in the xcms R package [25]. The identification of the metabolites was performed based on their chromatographic behavior and HRMS/MS2 data by comparison with standard compounds, when available, and literature data providing a tentative identification [25,26,27,28,29,30,31,32,33]. Data acquisition was carried out with the Xcalibur® data system (Thermo Finnigan, San Jose, CA, USA). 3.5. In Vitro Cytokinesis-Block Micronucleus (MN) Assay The lymphocyte cultures were treated with investigated ethanol extracts of the J. nigra leaves and husks at concentrations of 2, 4, and 6 μg/mL. Untreated cell culture served as a blank control. One cell culture containing the known clastogenic agent mitomycin C (MMC; Calbiochem, Merck Chemicals, Darmstadt, Germany, ≥95% purity by HPLC) (0.2 μg/mL, in phosphate buffer) alone was used as a negative control. One cell culture containing Amifostine WR-2721 (98%, S-2[3-aminopropylamino]-ethylphosphothioic acid) at 1.0 μg/mL), (Marligen-Biosciences, Ijamsville, MD, USA) was used as a positive control. They were added to the cultures 25 h after phytohaemaglutinin (PHA) stimulation and left until harvest. All cultures were incubated in a thermostat at 37 °C. Treatment with the investigated ethanol extracts lasted for 19 h, after which all cultures were rinsed with a pure medium, transferred into 5 mL of fresh RPMI 1640 medium (RPMI 1640 Medium + GlutaMAX + 25 mM HEPES; Invitrogen-Gibco-BRL, Vienna, Austria), and incubated for an additional 72 h. Approximately 2 × 106 blood lymphocytes were set up in 5 mL of RPMI-1640 medium supplemented with 15% calf serum and 2.4 μg/mL of phytohaemaglutinin (Invitrogen-Gibco-BRL, Waltham, MA, USA). One hour after initiating the cell stimulation, investigated ethanol extracts (three concentrations) were added to the samples. The incidence of spontaneously occurring MN in control samples was scored. For MN preparation, the cytokinesis-block method of Fenech and Morley [3] was used with some modifications, as described by Stankovic et al. [34]. At least 1000 binucleated (BN) cells per sample were scored, registering MN according to the criteria of Countryman and Heddle [35] and Fenech and Morley [3]. The slides were air-dried and stained with alkaline Giemsa 2% (Sigma-Aldrich, Vienna, Austria). At least 1000 binucleated (BN) cells per sample were scored, registering MN according to the criteria of Countryman and Heddle and Fenech and Morley [3,35]. The effects of the investigated complexes on cell proliferation were estimated by the cytokinesis-block proliferation index (CBPI), calculated as suggested by Surralles et al. [36]. 3.6. Statistics and Index Calculations Statistical analysis was performed using the Origin software package, version 7.0. The statistical significance of differences between data pairs was evaluated by analysis of variance (one-way ANOVA), followed by the Tukey test. A difference was considered significant at p < 0.01. The results are presented as the percentage of change compared to the control. 3.4.5. LC/MS Method for Metabolite Identification LC-HRMS/MS (Thermo Scientific™ Vanquish™ Core HPLC system coupled to the Orbitrap Exploris 120 mass spectrometer, San Jose, CA, USA) was used to determine the metabolic profile of the extracts. The liquid chromatography system was equipped with a Hypersil GOLD™ C18 analytical column (50 × 2.1 mm, 1.9 μm particle size), thermostated at 40 °C. The injection volume was 5 μL, and the flow rate was constant at 300 μL/min. The compounds of interest were eluted with ultrapure water + 0.1% formic acid (A) and acetonitrile (MS grade) + 0.1% formic acid (B): 5% B in the first min; 5–95% B from 1 to 10 min; 95% B from 10 to 12 min; 5% B until 15 min. The used characterization technique was previously optimized by the researchers [25]. The Orbitrap Exploris 120 mass spectrometer was equipped with a heated electrospray ionization (HESI-II) source operating in negative ionization modes. Full-scan MS was monitored from 100 to 1500 m/z with an Orbitrap resolution set to 60,000 FWHM, while data-dependent MS2 experiments were conducted at an Orbitrap resolution of 15,000 FWHM. Normalized collision energy was set to 35% with an isolation width of 1.5 m/z. The dynamic exclusion time was set to 10 s with exclusion from a specific scan after two occurrences, and the intensity threshold was set to 1 × 105. LC/MS dates were evaluated using R Studio (version 2023.09.1, build 494) software. Peak picking was performed using the enviPick R package, and peak correspondence across samples was performed using the density method available in the xcms R package [25]. The identification of the metabolites was performed based on their chromatographic behavior and HRMS/MS2 data by comparison with standard compounds, when available, and literature data providing a tentative identification [25,26,27,28,29,30,31,32,33]. Data acquisition was carried out with the Xcalibur® data system (Thermo Finnigan, San Jose, CA, USA).',\n",
       " '11207711': '2. Materials and Methods 2.1. Soil Collection, Microbial Inoculum, Seed Selection, and Incubation Methods Topsoil from the University of Kentucky North Farm, Lexington, Kentucky Spindletop Farm, (GPS coordinates: 38°07.555′ N, 84°30.901′ W) was collected, homogenized and mixed with perlite prior to use. Nutrient profiling of the soil was performed by core University of Kentucky Soil Regulatory Services (Table S1). N. benthamiana, N. glutinosa, N. rustica, and N. tabacum (two commercial lines were used, KY14 and TN90, as well as the Nicotiana tabacum L. cv. Samsun-NN, which is a well-described transgenic tobacco line to study abiotic stress [26,27,28]) seed was sourced from University of Kentucky Tobacco Research and Development Center (KTRDC). A germination cutoff of 90–100% was used. Surface sterilization of seeds was achieved via a 30% bleach rinse for 20 min followed by 70% ethanol rinse for one minute. These rinses were followed by three additional rinse cycles with sterile water. The sterilized material was tested by culturing on YPDA plates to ensure that no bacterial growth occurred. Bacterial library strains were sourced from the Switchgrass and Giant Burpee tomato library [29]. We screened 24 bacterial strains (Table S2) applied directly to seeds for growth promotion, growth suppression, or no influence. Strains were grown in YPD broth in medium flasks overnight (11 ± 2 h) at 28 °C on a rotary shaker. For inoculations, strains were grown at 28 °C on a rotary shaker until OD600 = 0.6. A total of 20 seeds were placed in each bacterial culture for spermosphere inoculation. Seeds were incubated in the culture for 12 h and kept in the rotary shaker/incubator. 2.2. Sample Selection and Processing Morphological analyses followed the methods of Kelemu et al. [30] with some modification. Specifically, at two weeks, seedlings were checked for differences in the root system. After three to four weeks of inoculation, measurements were taken and recorded for traits including height, number of leaves (NL), leaf length (LL), leaf width (LW), and number of flowers. Measurements were repeated once a week for 60 days. Plants from both the treatment and control were harvested at approximately four to five weeks following the methodology developed by Lundberg et al. [9]. All plants were harvested, surface-sterilized, and processed on the same day. 2.3. DNA Extractions and Library Preparation DNA extraction was performed using FASTDNA™-96 Soil Microbe DNA Kit (MP Biomedical, LLC, Santa Ana, CA, USA). Samples were previously separated based on being from the roots, stem, leaves, the rhizosphere, and outside the rhizosphere. All samples were placed in the freeze-dryer before being pulverized. Daisy bb gun beads were used to pre-pulverize the samples allowing stem and root samples to homogenize for optimal DNA isolation. After extraction, libraries were prepared following the protocol established by Lundberg et al., [31], where peptide nucleic acid (PNA) for mitochondrial (5′-GGCAAGTGTTCTTCGGA-3′) and plastid (5′-GGCTCAACCCTGGACAG-3′) rRNA and plastid sequences were used as elongation arrest clamps to prevent amplification of plant ribosomal 16S sequences. 2.4. PCR Quantification and Sequencing The DNA concentrations of the final reactions obtained from the PCR step were measured in a 96-well plate format using PicoGreen fluorescent dye (Invitrogen, Waltham, MA, USA). This dye measures double-stranded DNA quantification and it was measured in a fluorescent plate reader (475 nm to 530 nm). After quantifying the amounts of DNA present, we analyzed a portion of the samples in an 1.5% agarose gel to ensure the presence of bands with a size of approximately 448 bp. The pooling of all samples from the prepared library was performed using equimolar ratios and samples were cleaned using AMPure beads at a 0.7:1 ratio. These samples were later eluted in 20 µL of deionized water to be denatured and loaded in the MiSeq machine by following the Illumina protocol and the standards established previously [31]. 2.5. Demultiplexing and Heatmap Generation After sequencing, the reads obtained from the Illumina platform were demultiplexed utilizing CASAVA software from Illumina (version1.8.2). A FASTA file was generated in which all the consensus sequences obtained per sample were stored. The software utilized was the Molecular Tag Toolbox 4.1.2. (MT-Toolbox, Google sites) [32]. R scripts were generated based on Lundberg et al. [31] to generate a graphical display of the abundance of different microbial organisms present in the samples. Rarefaction values varied based on the type of heatmap that was generated and were sorted into family, phylum, or operational taxonomic unit (OTU). 2.6. Phylum Analysis and Abundance for Genotype and Inoculation Only the non-plant reads were classified at the phylum level. Reads from the same phyla were pooled, normalized and converted to a ratio by dividing the reads from each phylum by the total number of phylum-classifiable reads in that sample. For better visualization, those phyla representing less than 5% of the total in any sample were reclassified as “low abundance”. Data were plotted in R using the “Hist” function of ggplot2 (R Foundation for statistical computing, Vienna, Austria). 2.7. CAPSCALE Analysis A constrained ordination routine analysis was used to determine if samples could be separated based on the treatment to which they were exposed. These analyses used a distance matrix between samples, showing the coordinates of each sample determined by the profile of OTU counts for that sample using the Bray–Curtis distance. The R packages used for the analysis were vegan, for capscale, ordination, and pscl (R foundation for statistical computing, Vienna, Austria). 2.8. Poly(A) Tag Library Preparation and Sequencing Total RNA was isolated from N. benthamiana plants of approximately four to five weeks of age using the RNAeasy kit (Qiagen, Hilden, Germany). Nicotiana poly(A) tags (PATs) were generated with 1 µg of total RNA using Method B1 described in [33]. The resulting PATs were sequenced on an Illumina high-throughput sequencing DNA platform. In all cases, three independent biological replicates were used. The sequenced PAT-seq reads were processed using the pipeline as detailed in [33]. Briefly, sequences were demultiplexed and trimmed to remove the oligo-dT tracts and sequencing adapters. The processed tags were then mapped to the Nicotiana reference genome. The mapping output was saved in bam file format and used with BEDTools to determine the total count of PATs that mapped to individual annotated genes. The gene expression was determined using the empirical analysis of EDGE tool in CLC Genomics Workbench. Genes were considered significantly different using a p-value < 0.01 and a two-fold change. A total of three replicates were used per treatment. 2.9. CARD-FISH in Root Tissue Briefly, roots from plants of 10- and 21-day old seedlings were collected and used for assessments. These were fixed and prepared as per Lebeis et al. [14]. Probes (EUB338 (59-GCTGCCTCCCGTAGGAGT-39, 35% formamide) were provided by the Lebeis lab at the University of Tennessee, Knoxville, which were selected using probeBase38 (http://www.microbial-ecology.net/default.asp, accessed on 4 January 2015) and labelled with enzyme horseradish peroxidase on the 5′ end (Invitrogen). Sample hybridization was performed as per [14]. For samples belonging to the 21-day-old group, we analyzed multiple panels from the transition zone to the tip and through the expansion zone. A similar approach was used for 10-day-old samples, but a full mosaic was built out of multiple sections of images. 2.10. Cold Treatment for Inoculants All vials containing seeds and the inoculum that had reached OD600 = 0.6 were kept at 4 °C for 12 h in a rotary shaker. These were transferred to room temperature and shaken for an additional two hours prior to being placed in soil. All plants were grown in a greenhouse environment with the same set up mentioned in the prior method of inoculation. 2.11. Statistics for Morphological Data Analyses All data collected were analyzed with SAS using GLM to generate means for each trait. We used Tukey’s test to separate means using a p-value of p < 0.05. Boxplots graphs were generated using BoxPlotR: a web-tool for the generation of box plots, created by the Tyers and Rappsilber labs (http://shiny.chemgrid.org/boxplotr/, accessed on 4 January 2015). Sample size was represented by the width of each box, and notches represented a 95% confidence between median differences. Tukey’s test was used to define the whiskers for each group sample. The number of samples was thirty-six total per year/per trait/per treatment (12 samples in each season per treatment). 2.12. Auxin Expression Related to Bacterial Treatments Transgenic N. tobacum plants expressing a stable form of pB-GFP::P87 [34] obtained from the Maiti lab at the Kentucky Tobacco Research and Development Center (KTRDC) were PCR-verified for the presence of GFP. All seeds of N. tobacum pB-GFP::P87 were inoculated with Bs, Bc, Ms, or a mock control (media only) using the same protocol as described for N. benthamiana. IAA and 2,4-D treatment was performed by placing surface-sterilized seeds in plates containing two concentrations of IAA (1 uM and 10 uM) and 2,4-D (100 nM) that were added to the Murashige and Skoog growth medium used. Seeds were plated in Murashige and Skoog medium, and after 3 days post germination, they were visualized using a laser scanning confocal microscope (Olympus IX83). A total of 10 replicates per treatment were analyzed. The data obtained were analyzed using a Dunnett’s multiple comparison test against the control with a p < 0.05. 2.13. Hormone Profile Auxin (IAA Indole-3-acetic acid), IAA-Asp N-(Indole-3-yl-acetyl)-aspartic acid, IAA-Glu N-(Indole-3-yl-acetyl)-glutamic acid, IAA-Ala N-(Indole-3-yl-acetyl)-alanine, IAA-Leu N-(Indole-3-yl-acetyl)-leucine, and IBA (Indole-3-butyric acid) were measured by means of HPLC-ESI-MS/MS following Zaharia et al., [35] and Lulsdorf et al. [36] as a fee-for-service product by the National Research Council of Canada. Conjugated forms of IAA were measured in addition to the free IAA because free IAA can represent around 25% of the total pool and conjugated forms create sinks of available IAA in situ [37]. 2.14. Biochemical Test for Indole Production from Bacteria This method was followed as described in Peterson and McLaughlin [38]. 2.1. Soil Collection, Microbial Inoculum, Seed Selection, and Incubation Methods Topsoil from the University of Kentucky North Farm, Lexington, Kentucky Spindletop Farm, (GPS coordinates: 38°07.555′ N, 84°30.901′ W) was collected, homogenized and mixed with perlite prior to use. Nutrient profiling of the soil was performed by core University of Kentucky Soil Regulatory Services (Table S1). N. benthamiana, N. glutinosa, N. rustica, and N. tabacum (two commercial lines were used, KY14 and TN90, as well as the Nicotiana tabacum L. cv. Samsun-NN, which is a well-described transgenic tobacco line to study abiotic stress [26,27,28]) seed was sourced from University of Kentucky Tobacco Research and Development Center (KTRDC). A germination cutoff of 90–100% was used. Surface sterilization of seeds was achieved via a 30% bleach rinse for 20 min followed by 70% ethanol rinse for one minute. These rinses were followed by three additional rinse cycles with sterile water. The sterilized material was tested by culturing on YPDA plates to ensure that no bacterial growth occurred. Bacterial library strains were sourced from the Switchgrass and Giant Burpee tomato library [29]. We screened 24 bacterial strains (Table S2) applied directly to seeds for growth promotion, growth suppression, or no influence. Strains were grown in YPD broth in medium flasks overnight (11 ± 2 h) at 28 °C on a rotary shaker. For inoculations, strains were grown at 28 °C on a rotary shaker until OD600 = 0.6. A total of 20 seeds were placed in each bacterial culture for spermosphere inoculation. Seeds were incubated in the culture for 12 h and kept in the rotary shaker/incubator.',\n",
       " '11207628': '',\n",
       " '11207537': '2. Materials and Methods 2.1. Experimental Animals and Procedures Involving Animals Male C57BL/6 mice (2.5 months old, 25–30 g) from the Federal State Budgetary Scientific Institution Scientific Research Institute of Neurosciences and Medicine (SRINM; Novosibirsk, Russia) were used. The animals were maintained on a standard laboratory diet under standard conditions (the light–dark cycle: 14 h light and 10 h dark, temperature: 20–22 °C, and relative humidity: 50–60%). Every effort was made to minimize the number of animals used and their suffering. All protocols follow the recommendations outlined in the NIH Guidelines for the Care and Use of Laboratory Animals (National Institutes of Health Publication, 8th edition, revised 2011) and were approved by the Ethics Committee of the Scientific Research Institute of Neurosciences and Medicine (ID 2, date of approval: 21 February 2019). The experiments were carried out on a murine pharmacological model of AD induced by the central administration of the human amyloid beta fragment oligomers Aβ25–35 (Amyloid Beta-Protein Fragment 25–35 (Sigma-Aldrich, Darmstadt, Germany)). Amyloid preparation and the injections into cerebral ventricles were performed according to previously published protocols [27]. A total of 5 μL of the drug or vehicle were administered into each lateral ventricle using Neurostar’s Stereo Drive motorized stereotaxis system (Neurostar, Tübingen, Germany). A total of 10 μL (9.43 nmole) of the solution was administered to each mouse. The following coordinates adapted from the mouse brain atlas [28] were used: AP = −0.46 mm; ML = ±0.9 mm; and DV = 2.5 mm from the bregma, midline, and skull surface, respectively. The treatment with trehalose started 2 days after the stereotaxic surgery. The practice of treating mice with trehalose, using its 1–5% solutions as a drink, is widespread. In our experiments, the amount of liquid drunk was determined. Mice consumed 5.63 ± 0.18 mL of water or 5.14 ± 0.30 mL of trehalose solution daily per mouse. The mice were subdivided into five groups (6 to 9 animals each): (1) control (bilateral injections of sterile water into the lateral ventricles of the brain [hereafter: i.c.v. injection]); (2) Aβ (bilateral i.c.v. injections of Aβ25–35, AD model); (3) “Aβ + 2% T” (AD model + a 2% solution of trehalose in drinking water daily for 2 weeks); (4) “Aβ + 4% T” (AD model + a 4% solution of trehalose in drinking water daily for 2 weeks); (5) “Aβ + 4% T/2” (AD model + a 4% solution of trehalose in drinking water every other day for 2 weeks). One day after the final drug administration, behavioral testing was performed, after which biological samples were collected from the randomly selected six mice of each group. 2.2. Behavioral Testing Each mouse was handled for 5 min/day during two consecutive days before the behavioral testing. The animals were kept in groups of 3–6 mice per cage between tests. Prior to testing, a mouse was placed individually in a clean cage (25 cm × 40 cm × 20 cm) and transported to a dimly lit observation room (28 lux), with sound isolation reinforced by a masking white noise of 70 dB. A test was started 10 min after establishing the habituation of a mouse in line the testing conditions. Animal behavior was recorded using two digital video cameras Panasonic WV-CL930 (Panasonic System Networks Suzhou Co. Ltd., Suzhou, China) from the side and top. The video data were processed in original EthoVision XT 11.5 software (Noldus IT, Wageningen, The Netherlands). The test equipment was cleaned with 20% ethanol and thoroughly dried before each test trial. The open field test: This test was carried out to assess locomotor and exploratory activity and anxiety. The following parameters were measured: general locomotor activity by the total distance traveled (cm); anxiety by the time spent in the center of the arena; vertical locomotor activity and exploratory behavior by the number of rearing bouts. The test was performed in an open-top transparent Plexiglas installation with a square arena (40 × 40 × 37.5 cm) under bright illumination (1000 lux) from above. Mice were placed in the arena near the wall and their behavior was recorded for 10 min. Passive avoidance test: Training on the passive avoidance response was performed by a standard single-session method in an experimental chamber with dark and light compartments and an automated Gemini Avoidance System apparatus (San Diego Instruments, San Diego, CA, USA), as described in detail earlier [12]. The Gemini software Version 1.0.0 (San Diego Instruments, San Diego, CA, USA) automatically recorded the latency when entering the dark compartment, and the data from the test served as a measure of the acquisition of the conditioned passive avoidance response. 2.3. Neuromorphological Analysis Mice were culled by CO2 asphyxiation and transcardially perfused with phosphate-buffered saline (PBS), followed by 4% paraformaldehyde in PBS; then, their brains were rapidly removed and postfixed in PBS containing 30% of sucrose at 4 °C. After dehydration, brain samples were embedded in Tissue-Tek O.C.T. compound (Sakura Finetek, Torrance, CA, USA) and frozen at −70 °C, until sectioning into 30-μm-thick slices on a cryostat HistoSafe MicroCut—SADV (Citotest Scientific Co., Ltd., Nanjing, China). Coronal slices along the frontal cortex [AP = 2.93–2.57 mm] or hippocampus [AP = −1.91–(−2.45) mm] of each mouse brain were carried out. The immunohistochemical analysis (IHC) and Nissl staining were performed according to protocols described in detail previously [12,27]. 2.4. Assay for Leukocyte Elastase (LE) Activity Leukocyte elastase (LE) activity was measured by the rate of cleavage of the chromogenic substrate Boc-l-alanine 4-nitrophenyl ester (BOC-Ala-ONp) (ICN Biomedical Inc., Costa Mesa, CA, USA), as described in detail previously [29]. The incubation mixture included 0.480 mL PBS (pH 6.5), 0.02 mL 0.40% BOC-Ala-ONp substrate solution (3.1 mg in 1 mL acetonitrile), and 0.005 mL of the serum to be tested. Using the SWIFT 1000 Reaction Kinetics software (version 2.03, Biochrom Ltd., Cambridge, UK), optical density (OD) was measured at 347 nm for 3 min. 2.5. Assay for α1-Protease Inhibitor (α1-PI) Activity α1-protease inhibitor (α1-PI) is the main LE inhibitor in the blood. Since α1-PI is a nonspecific inhibitor of serine proteases, its functional activity in the serum was measured by an enzymatic method based on trypsin inhibition using N-α-benzoyl-l-arginine ethyl ether hydrochloride (BAEE) as a substrate (ICN Biomedical Inc., Costa Mesa, CA, USA) [29]. The experimental sample consisted of 1.9 mL Tris-HCl buffer (pH 8.0); 0.1 mL serum diluted with saline (1:50); and 0.01 mL 0.01% trypsin solution. After preincubation for 3 min at 25 °C, 1 mL of 1.5 mM BAEE solution was added to the sample and then the increase in OD at 253 nm was measured for 3 min using the SWIFT 1000 Reaction Kinetics computer program (version 2.03, Biochrom Ltd., Cambridge, UK). 2.6. Statistics STATISTICA 10.0 software (StatSoft, Tulsa, OK, USA) was used to perform all the statistical analyses. The normality of the data distribution was determined by the Shapiro–Wilk W test. The statistical evaluation of data was performed by one-way analysis of variance (ANOVA) or repeated measures ANOVA (results of the passive avoidance test) followed by Fisher’s LSD post hoc test. The level of significance was defined as p < 0.05 in all experiments reported here.',\n",
       " '11207411': '2. Materials and Methods 2.1. Enrolled Subjects For scRNA-seq experiments, we enrolled five PBMC samples from a previously described cohort of Slovenian CD patients (Table 1) [38]. For patients 001 and 002, we obtained peripheral venous blood before (naïve state) and six months after the initiation of ADA (Humira®, Abbott Laboratories, North Chicago, IL, USA) therapy, while in remission. For patient 003, we obtained peripheral venous blood six months after the initiation of ADA therapy, while in remission. For patients 004 and 005, PBMCs were obtained after six months of UST (Stelara®, Janssen Biotech Inc., Horsham, PA, USA) therapy, while in remission. Both 004 and 005 were switched to UST therapy due to non-response to ADA therapy. Patient 004 became a non-responder to ADA after three months, and patient 005 became a non-responder to ADA therapy after two. None of the patients were on any immunomodulatory therapy at the time of sample acquisition. Clinical remission was assessed based on a Harvey–Bradshaw index (HBI) score of ≤4 and the absence of inflammatory lesions during an endoscopy [39]. For genomic profiling, we used data from a previously described cohort of 84 Slovenian CD patients (58 responders and 26 non-responders at week 30). Out of 26 non-responders, there were 9 males and 17 females, and there were 24 males and 34 females in the group of responders (p value = 0.344). Additional baseline demographics are presented in a previous study [30]. All enrolled patients were of Caucasian Central European ethnicity. Inclusion criteria were as follows: adverse effects to corticosteroids, refractoriness to corticosteroids, and previous loss of response to previous anti-TNFα, if applicable [24]. Exclusion criteria consisted of a history of murine allergy, the presence of stenosis, abscesses, total colectomy, active serious infection in the past three months, infection with mycobacteria, pregnancy and lactation, and any malignancy [40]. ADA therapy was initiated with a loading dose of 160 mg and was followed by 80 mg after two weeks. The maintenance dose consisted of 40 mg every other week. Concomitant therapy with azathioprine, 5-aminosalicylates, corticosteroids, or antibiotics was allowed if the dosage of ADA remained stable in the last three months. UST therapy was initiated using a single-dose intravenous infusion of 390 mg, followed by 90 mg subcutaneously every eight weeks. Concomitant therapy was allowed in UST-treated patients. The response to ADA was measured based on the IBD questionnaire (IBDQ) after 30 weeks of therapy and was defined as positive if there was an increase in the IBDQ score of >22 points after the baseline score or if a total score of >170 points was achieved [41,42]. Written informed consent was obtained from all patients. The present research was conducted following the Declaration of Helsinki and approved by the Ethics Committee of the Slovenian National Committee for Medical Ethics (KME 80/10/07, 21p/12/07, 0120-194/2019/4). 2.2. Preparation of Cells Peripheral venous blood was collected into K2EDTA tubes (Vacutainer BD, Franklin Lakes, NJ, USA). The PBMCs were isolated using density gradient centrifugation with Lympholyte H Cell Separation Medium (Cedarlane, Burlington, ON, Canada) and SepMate 50 centrifuge tubes (StemCells Technologies, Vancouver, BC, Canada). The remaining platelets and erythrocytes were removed using low-speed centrifugation and red blood cell lysis, respectively. The isolated PBMCs were stored in liquid nitrogen at a density of 1–10 × 106 cells/mL. A combination of 10% dimethyl sulfoxide (Sigma-Aldrich, Burlington, MA, USA) and 40% heat-inactivated fetal bovine serum (Gibco, ThermoFisher Scientific, Waltham, MA, USA) was used as the cryopreservation medium. The cell concentration was determined using the CytoSmart automated cell counter (Corning Inc., Corning, NY, USA). 2.3. scRNA-Seq Experiment and Library Preparation To perform scRNA-seq, the 10X Chromium System (10X Genomics, Pleasanton, CA, USA) was used. First, the cryopreserved PBMCs were thawed and washed in Dulbecco’s phosphate-buffered saline supplemented with 5% FBS to remove the cryopreservation medium. The quality of the cell samples was confirmed using the trypan blue exclusion, and the viability of all samples was >90%. A total of 2 × 106 cells per sample were used for the library preparation. Samples 001 and 002 were processed as single reactions using the Chromium Next Gem Single Cell 3′ Kit v3.1 (PN-1000269), Chromium Next GEM Chip G Single Cell Kit (PN-1000127), and Dual Index Kit TT Set A (PN-1000215; all 10X Genomics), according to the manufacturer’s instructions (CG000315 Rev E). Other samples were processed as multiplexed reactions, with three samples per reaction, using the Chromium Next Gem Single Cell 3′ Kit v3.1 (PN-1000269), 3′ Feature Barcode Kit (PN-1000262), Dual Index Kit TT Set A (PN-1000215), and Dual Index Kit NN Set A (PN-1000243; all 10X Genomics). For the multiplexed samples, the cells were first CMO-labeled using a 3′CellPlex Kit Set A (PN-1000261, 10X Genomics), according to the manufacturer’s instructions (CG000391 Rev B). All samples were loaded onto the Chromium Controller Instrument (10X Genomics) to generate gel bead-in-emulsions (GEM). Next, complementary DNA (cDNA) libraries were constructed. GEM reverse transcription (RT) was performed in a C1000 deep-well block thermocycler (BioRad, Hercules, CA, USA). The cDNA was purified using Dynabeads MyOne Silane Beads (Thermo Fisher Scientific) and amplified following the recommendations in the 10X Genomics protocol CG000315 Rev E, for single reaction samples, and protocol CG000388 Rev C, for multiplex reaction samples. The product was subsequently purified using the SPRIselect Reagent Kit (B23318, Beckman Coulter, Brea, CA, USA). Next, 3′ Gene Expression Libraries were constructed using the Chromium Single-Cell 3′ Library Construction Kit (PN-1000196). For the multiplexed samples, Cell Multiplexing Libraries were additionally generated using the Dual Index Kit NN Set A (PN-1000243) and 3′Feature Barcode Kit (PN-1000262). Library quality control was performed using the Agilent Bioanalyzer 2100 and a High Sensitivity DNA Kit (Agilent, Santa Clara, CA, USA). Finally, the prepared libraries were sequenced with Illumina NovaSeq pair-end 150bp at GENEWIZ (GENEWIZ Germany GmbH, Leipzig, Germany), with an average sequencing depth of >100 million reads per library yielding an approximate average of 28,600 reads per cell. 2.4. scRNA-Seq Data Analysis The obtained raw sequencing .fastq files were assessed for quality using FastQC 0.11.9 software prior to downstream analysis [43]. Obtained data were pre-processed using CellRanger 7.0.0 software with intron mode (10X Genomics) to demultiplex, align the reads to the GRCh38 reference genome, and perform cell barcode processing and gene counting. The generated cell barcode and count matrices were further analyzed using the Seurat v5 R package [44]. All data were further manipulated and processed using R 4.3.2 (R Core Team 2023, Vienna, Austria), Python 3.8.10, and the bash environment. Matrices were used to construct Seurat objects containing at least 200 expressed genes per three cells. All cells with gene counts above 5000 and below 500 per cell were removed from subsequent analysis. Additionally, all cells expressing more than 10% mitochondrial genes and less than 5% ribosomal genes were also discarded from downstream analysis. Each Seurat object was additionally analyzed for multiplets using DoubletFinder [45]. All cells flagged as potential high-confidence multiplets were discarded from the objects. Data generated during quality control are available in Supplements (Figure S2). Sex chromosome genes and outliers among genes with the highest expression were also removed in order to obtain post-quality control objects. Subsequently, all seven samples were pooled into a combined RNA assay and subjected to SCT normalization [46]. Cell-cycle scores were calculated on obtained SCT assay objects and used to calculate the S-cycle score minus the G2M cycle score difference. The obtained difference and percent of mitochondrial genes were subsequently regressed out on the original RNA assay to obtain a newly constructed SCT assay for integration. At least 5000 features were used for the immune anchor selection to identify cross-dataset cell pairs in a matched biological state and to correct for batch technical differences, allowing for the alignment of the cells between samples for downstream comparative analysis. After integration, principal component analysis was performed to reduce dimensionality using 50 principal components. Uniform manifold approximation and projection (UMAP) and t-distributed stochastic neighbor embedding (tSNE) analyses were subsequently performed to visualize the high-dimensional data. Automatic reference-based mapping of cell types was performed using Seurat’s Azimuth level one descriptors [44]. Obtained annotated clusters with level one descriptors were further evaluated using Seurat’s FindConservedMarkers function and embedded MetaDE R package [47,48]. This allowed for the detection of conserved cell-type markers with the same direction of perturbation in all datasets. Detected conserved markers were manually examined using publicly available datasets RNA HPA and the RNA Monaco immune cell gene data [49,50]. 2.5. Differential Gene Expression Testing Differential gene expression was examined for each annotated cell type in a pseudo-bulk manner to account for between-sample correlation and to avoid the overinflation of statistics caused by treating each cell as an independent sample [51,52,53]. Differential expression was tested using a three-stage approach. The first stage was used to identify differentially expressed genes influenced by ADA therapy, and the second stage was used to identify differentially expressed genes influenced by UST therapy in ADA non-responders. In the third stage, the above therapy-related genes were removed from analysis, and differentially expressed genes in ADA non-responders relative to ADA responders were identified. The following analysis was carried out for each stage separately. Pseudo-counts (average counts for each cell type in each sample) were converted to counts per million (CPM) using edgeR 3.42.4 [54]. Low-expressed genes were considered based on mean CPM value corresponding to raw read counts of 10 and were filtered out from the downstream analysis. Retained genes were first normalized using the trimmed mean of M values method (TMM) followed by mean-variance modeling at the observational level transformation (VOOM) using limma 3.56.2 R package [55,56]. For stages one and two, differential expression was considered for genes with a Q value (false discovery rate correction) of <0.05 and a Log2FC of >0.5 or <−0.5, while for stage three, differential gene expression was considered for genes with a p value of <0.05 and a Log2FC of >1 or <−1. Additionally, a stage one fitted model was adjusted for sex and age, a stage two model was adjusted for age (sex was a constant variable), and a stage three model was adjusted for sex and age. Blocking (accounting for paired samples) was not deployed in the stage one model to avoid the loss of the least squares coefficient estimate. 2.6. Gene Ontology Analysis Gene ontology (GO) analysis was performed using the clusterProfiler 4.8.3 R package for differentially expressed upregulated and downregulated gene modules at each cell type separately [57]. Enrichment analysis of GO terms was performed for molecular functions and biological processes. Thresholds for enrichment were set as p value < 0.01 and Q value < 0.05. Enrichment results were presented as tables and dot plots using ggplot2 3.4.4 R package [58]. Subsequently, a GO analysis was used to stringently identify and sub-select genes by meaningful involvement into enriched terms. 2.7. Genomic Profiling DNA for genetic analyses was extracted from PBMCs using TRI-reagent® (Merck, Darmstadt, Germany) according to the manufacturer’s instructions. The purity and concentration of DNA were subsequently measured and determined using Synergy™ 2 spectrophotometer (Biotek, Winooski, VT, USA). DNA samples from all 84 enrolled CD patients were genotyped using an Infinium Global Screening Array (GSA_24v1) (Illumina, San Diego, CA, USA). Extensive quality control was carried out as previously described [59]. Genotype imputation was performed using a TOPMed imputation server and a TOPMed r3 panel with the Minimac4 imputation algorithm, an Rsq filter of >0.3, and Eagle 2.4 phasing, yielding 8,155,436 variants [60,61,62]. Association analysis was performed between non-responders and responders at week 30 using Wald logistic regression implemented in PLINK 2.0 software and using genotypic dosages [63]. Principal components to account for possible ethical bias were also calculated using PLINK 2.0 and tested using the gap v1.2.2 R package. A regression model for association analysis was fitted and adjusted to age at diagnosis, sex, azathioprine use, the use of aminosalicylates, the use of corticosteroids, and the first four principal components. The association was tested for alternative alleles and the odds ratio was calculated for alternative alleles and non-responders. Targeted genomic profiling and integration to genomics were performed with the extraction of variants ranging ±100 kb from previously identified and enriched differentially expressed genes in scRNA-seq and GO analysis, respectively. A statistically significant signal for variants was considered at a p value of <0.01 before estimating independent variants based on LD pruning using the SNPclip tool from the LDlink software 1.4.0, with R2 set to 0.5 [64]. After the estimation, a Bonferroni correction was applied to correct for multiple testing. If the p value after correction remained <0.05, the signal was considered statistically significant. Variants residing in selected genomic regions are presented in regional Manhattan plots constructed using LocusZoom [65]. Tissue expression quantitative trait loci (eQTLs) were estimated using the eQTL catalog project (https://www.ebi.ac.uk/eqtl/, accessed on 28 December 2023) [66,67,68,69]. The involvement and weight of the identified variants were further evaluated using a recursive feature elimination ranking algorithm (RFE) and support vector machines (SVM) machine learning approach implemented in the e1017 R package. The ranking was performed based on the achieved average rank after 10-fold cross-validation. Identified independent variants with statistically significant signals were grouped into four subsets based on the achieved RFE rank. Variant dosages were extracted and tested using binomial generalized linear models (GLM), with response at week 30 as the outcome. Subsequently, the probabilities of the models were obtained and analyzed using receiver operating characteristic (ROC) analysis using the pROC R package to obtain area under the curve (AUC) values for each subset [70]. 2.8. Replication and Meta-Analysis The replication of gene expression was performed using publicly available raw RNA sequencing .fastq files from BioProject PRJEB32332 deposited in Sequence Read Archive (SRA) (https://www.ncbi.nlm.nih.gov/sra, accessed on 29 December 2023). Sequencing was performed on mRNA-derived and fluorescence-activated cell sorting (FACS) sorted CD4+ T cells obtained from Belgian CD patients’ PBMCs prior to initiation of anti-TNFα therapy with IFX. The replication cohort consisted of fourteen responders and five non-responders to IFX. Therapy response was determined with endoscopy. Endoscopy-proven active disease was determined with a simple endoscopic scale for Crohn’s disease (SES-CD) > 7 and >4 in patients with pure ileal involvement [71]. Sequencing libraries were prepared using TruSeq stranded mRNA protocol according to the manufacturer’s instructions (Illumina). Single-end reads were aligned to the GRCh37 reference genome using Rsubread 2.14.2 R package [72,73]. After alignment, mapped reads were counted and assigned to genomic meta-features using featureCounts [72]. Sex chromosome genes were removed, and subsequent analysis was carried out as aforementioned for pseudo-bulk differential expression. A fitted model was constructed for IFX non-responders relative to responders. Differential gene expression was considered for genes with a p value of <0.05. Genes identified in scRNA-seq analysis were extracted from the results from the replication cohort and subjected to random-effects model meta-analysis using the MetaVolcanoR 1.14.0 R package to identify meta-statistically differentially expressed genes showing the same direction of expression and consistency of perturbation [74]. The above meta-statistically significant genes (p value < 0.05) were additionally subjected to replication and meta-analysis on the genomic level. For that, publicly available summary statistics of nominally significant variants with ADA immunogenicity in 96 Spanish Caucasian European CD patients were used [75]. All 96 patients were enrolled between July 2007 and June 2012, were naïve with respect to anti-TNFα therapy, and had a follow-up history of at least 3 years. Treatment response was assessed using HBI four weeks after ADA initiation and with clinical remission, defined as an HBI score ≤ 4. ADA immunogenicity was considered as the presence of adalimumab antibodies in patients’ plasma. For variants not present in the summary statistics, their respective linkage disequilibrium (LD) proxy variants were explored with LD parameters set as D′ > 0.9 and R2 > 0.5 or at least D′ > 0.5 if the correlation of alleles was confirmed using LDlink [64]. Odds ratios from the present study were inverted if the correlation of effect alleles was different. Subsequently, a meta-analysis of the genomic variants was carried out using random-effects model meta-analysis with the metafor R package [76]. A meta-statistically significant signal was considered at a p value of <0.05. 2.9. Mendelian Randomization and Oligogenic Risk Score Additionally, a two-sample Mendelian randomization analysis was performed using an inverse-variance weighted random-effect model using the MendelianRandomization R package [77]. Mendelian randomization was performed for genes, which proved to be meta-statistically significant in both gene expression and genomic meta-analyses. Respective variants or highest LD variants, in which eQTLs were listed, were used as instrumental variables. The eQTL effect size with the corresponding standard error was used as a variant–exposure association, and betas with respective standard errors from association analyses were used as variant-outcome associations. Exposure’s (Gene’s) causal effect on anti-TNFα therapy non-response was considered at a p value of < 0.05. The oligogenic risk score was calculated for replicated variants or their LD proxies. Imputed dosages of the present study served as a target dataset, and beta coefficients from the Spanish Caucasian European CD patient replication cohort served as the base dataset [75]. Oligogenic risk score analysis was carried out as previously described [78,79]. Since imputed dosages range from 0 to 2 in the direction from reference to alternative allele, odds ratios calculated for reference alleles as A1 were inverted and converted to beta coefficients to correct the direction of risk score calculation. Risk score validation was subsequently performed using binomial GLM with 10-fold cross-validation to obtain mean accuracy and Cohen’s kappa parameters using the caret R package [80].',\n",
       " '11207399': '',\n",
       " '11207358': '3. Materials and Methods 3.1. Preparation of Rowanberry Pomace Samples Rowanberry cvs were harvested at the beginning of September 2019 from Polli Horticultural Research Centre, (South Estonia, 58°7′ N, 25°33′ E) and were subsequently treated as described in [38]. Briefly, immediately after picking, the fruits were frozen and stored at −20 °C in zip lock bags in batches of 4–6 kg. Juice was pressed from the thawed fruit using a low-speed juicer Smeg SJF01CREU (Smeg S.p. A, Guastalla, Italy). The pomace parts of all cvs were lyophilized in a VirTis Advantage Plus Benchtop Freeze Dryer Model XL-70 (SP Industries, Warminster, PA, USA) for 72 h at 30 µbar. The pomace samples were ground in a Retsch cutting mill (Retsch SM 300, Retsch GmbH, Haan, Germany) with sieve hole diameters of 5 mm to obtain homogenous samples for analyses. All the samples per cultivar were from one lot. 3.2. Chemicals and Reagents Sulfuric acid (H2SO4), boric acid (H3BO3), and hydrogen chloride (HCl; 0.561 M) originated from Sigma-Aldrich Chemie (Steinheim, Germany). Sodium hydroxide (NaOH, 50%) was purchased from Ingle AS (Ingliste, Estonia), and Kjeltabs (FOSS Analytical A/S, Hillerød, Denmark) tablets were from Oridor Eesti OÜ (Tartu, Estonia). Ethanol (95% and 78%) was obtained from Estonian Spirit OÜ (Tallinn, Estonia). Heat-stable α-amylase, amyloglucosidase, protease, pectate Lyase (Aspergillus sp.), analytical grade Celite, Tris-HCl, and MES/TRIS buffer were purchased from Megazyme Ltd. (Wicklow, Ireland). 3.3. Evaluation of Proximate Composition and Pectin Content of Rowanberry Pomace The proximate chemical composition of 16 ground rowanberry pomace samples was determined using standard AOAC methods. The moisture content was determined by drying 2–3 g of samples at 105 °C to a constant weight, according to the Association of Official Analytical Collaboration (AOAC) Method 920.39. The constant dry weight of the pomace was subtracted from the initial weight, and the obtained water content was divided by the initial weight of the sample. The ash content was determined by charring 2 g of pomace for 30 min, followed by incineration in a muffle furnace at 525 °C for 4 h, according to the AOAC Method 930.05. The residues obtained after incineration were recorded and divided by the initial weight of the sample. The crude protein content was determined using the Kjeldahl procedure (N × 6.25), according to the AOAC Method 978.04. The crude fat content was analyzed according to the AOAC Method 948.22, by extracting the fat from the sample using petroleum ether as a solvent in a Soxhlet apparatus. After the extraction, the residues were dried in an oven at 105 °C to a constant weight. The lipid content was calculated from the initial dry weight of pomace by subtracting the final weight of the residues, then dividing by the initial weight of the sample. Crude fiber content was determined as a loss on ignition of dried residue remaining after digestion of the sample with 1.25% H2SO4 (w/v) and 1.25% NaOH (w/v), analyzed by AOAC official method 978.10. The total dietary fiber (TDF) was determined according to the enzymatic-gravimetric method using a total dietary fiber kit (Megazyme, Ireland) based on AOAC 991.43 [39]. Briefly, the samples were cooked at ~100 °C with heat-stable α-amylase to achieve gelatinization, hydrolysis, and depolymerization of starch; incubated at 60 °C with protease (to solubilize and depolymerize proteins) and amyloglucosidase (to hydrolyze starch fragments to glucose); and treated with four volumes of ethanol to precipitate soluble fiber and remove depolymerized protein and glucose (from starch). The residue is filtered, washed with 95% ethanol and acetone, then dried and weighed. Pectin content was determined according to the Megazyme Pectin Identification, Assay procedure (K-PECID II/II) (Megazyme, Bray, Ireland) [40]. Briefly, pectin was dissolved in acidified (pH 2.0) deionized water and adjusted to pH 12.0 to catalyze demethylation with the production of polygalacturonic acid regions in the polymer, i.e., conversion of pectin to pectate. The pectate was incubated with pectate lyase at pH 8.0 for 30 min to cleave the polygalacturonic acid, releasing unsaturated oligosaccharides, whose absorbance values were measured at 235 nm. All extractions and technical replicates were performed in triplicate, and the results were expressed as a percentage of the dry weight of the sample. 3.4. Determination of Fatty Acids Fatty acid composition in rowanberry pomace samples was determined according to the method of Sukhija and Palmquist [41] with slight modifications. Briefly, the lipids of the ground samples were extracted and methylated by applying a solvent mixture consisting of methanol, toluene, and acetyl chloride (20:27:3) for 500 mg of sample containing 3 mg internal standard (C17:0 fatty acid). The contents of fatty acid methyl esters were determined on an Agilent 6890A GC (Santa Clara, CA, USA) equipped with a split/splitless injector and a flame ionization detector, and the fatty acid methyl esters were separated on a quartz capillary column with a liquid-phase CP-Sil 88 (100 m × 0.25 mm, liquid-phase layer thickness 0.20 μm). Identification of common fatty acids were accomplished by comparison of sample peak retention times (Figure 3) with those of fatty acid methyl ester standard mixtures (Supelco 37 Component FAMEMix, Bellefonte, PA, USA, Nu-Chek Prep GLC 603, 408, BAME Mix, Linoleic Acid Methyl Ester Isomer Mix and many individual FA methyl ester standards). Theoretical response factors [42] were used to correct peak areas for individual fatty acids. Two different samples from the same brand (different lots) were analyzed in different runs at different times. All individual samples were analyzed in duplicate within the same run, from extraction to detection. Results were expressed as % of total fatty acids, which are calculated as the peak area of each fatty acid divided by the total peak area (of all fatty acids) multiplied by 100%. In order to express the results as % per sample, the peak area of the 3 mg internal standard (C17:0) was used for calculation. 3.5. Statistical Analysis The mean values and standard deviations (SD) of chemical contents (moisture, ash, protein, crude fat, TDF and CF, pectins, micro- and macro-elements and fatty acids) of pomace samples of 16 rowanberry cvs and wild rowanberry were calculated using MS Excel 2016 and one-way analysis of variance (ANOVA) at p value < 0.05.',\n",
       " '11207308': '2. Materials and Methods 2.1. Pot Experiments and Sampling Three-year-old potted tea plants (Longjing 43 variety) were selected for experimental fertilizer trials at the Tea Research Institute, Chinese Academy of Agricultural Sciences, China (latitude 30°17′99.62″ N; longitude 120°09′39.20″ E) at an altitude of 16 m. Five different fertilizer treatments were selected: organic fertilizers (rapeseed cake, chicken manure, and cow manure), a chemical fertilizer (urea), and a control (no fertilizer). Each fertilizer treatment had 3 replicate pots, with a total of 15 pots containing a single tea plant around 60 cm tall. The size of each experimental pot was 20 cm in diameter at the base, 22 cm in diameter at the top, and 20 cm in height and contained around 3 kg of soil in each pot. The initial %N and δ15N values of the tea plants and soil are represented by the control samples. Potted tea plants were fertilized only once at the start of the study (5 October 2022), and cultivation conditions (temperature, irrigation water, humidity, sunshine hours, etc.) were the same as those used by local tea gardens except for the different fertilizer treatments. Fertilizer type, nitrogen content, dosage, and δ15N values used in the experiment are shown in Table 1, with around 1.6 g of nitrogen delivered to each pot at the start of the experiment, apart from the control samples. Eight sampling events from each pot were undertaken during the study, covering the period of autumn through to late spring growth. In Hangzhou, tea leaf samples from the same growth phase collected on 20 October and 18 November 2022 were classified as autumn; 17 March, 21 April, and 10 May 2023 as early spring; and 20 May and 30 May 2023 as late spring. Top leaves were collected from the top 10 cm of the plant, middle leaves were collected between 20 and 40 cm above the soil, and basal leaves were collected between 0 and 20 cm above the soil. Around 2–3 g of soil and tea leaves were collected from each pot during each sampling event. Soil samples were air-dried at 25 °C and then ground into fine powder (<100 mesh). The freshly picked tea leaves were vacuum-dried for 48 h, crushed and finely ground (<80 mesh, with an aperture of 0.180 mm), and stored over silica gel in a desiccator prior to analysis. Experimental design of the tea plants is shown in Figure 1. 2.2. Stable Isotope Analysis Nitrogen content (%N) and isotope values (δ15N) of tea leaves and soil samples were determined using an elemental analyzer (Isotope PYRO cube, Elementar, Langenselbold, Germany) coupled with an isotope ratio mass spectrometer (Isoprime 100, Elementar, Handforth, UK). Multi-point isotope calibration was applied for more accurate measurement using primary reference standard materials provided by IAEA (International Atomic Energy Agency, Vienna, Austria), including B2155 (protein, δ15N = 5.94‰), USGS40 (L-glutamic acid, δ15N = −4.52‰), USGS64 (glycine, δ15N = 1.76‰), and IAEA-N2 ((NH4)2SO4, δ15N = 20.3‰). High-purity N2 gas served as a reference gas, and helium (He) gas was used as the carrier gas during the analytical process. Elemental analyzer oxidation and reduction furnace temperatures were set at 1020 °C and 650 °C, respectively. Tea leaves (4 mg) and soil (10 mg) were weighed in duplicate (Mettler Toledo, XP6, d = 1 μg, Greifensee, Switzerland) and packed into 6 × 4 mm2 tin capsules for δ15N analysis. Stable isotope ratios are expressed by the following Equation (1):δX = (Rsample − Rreference)/Rreference(1) where δX is the d15N value, and Rsample and Rstandard denote the abundance of the ‘heavy’ to ‘light’ isotope ratio of unknown samples and reference materials [29]. The analytical precision and reproducibility are ±0.2‰ for δ15N. The formulas used to calculate the isotope composition differences between leaves from different parts of the tea plant were as follows in Equation (2):Δ15Nbasal-middle = δ15Nmiddle − δ15Nbasal or Δ15Nmiddle-top = δ15Ntop − δ 15Nmiddle(2) where δ15N values are from the basal, middle, and top leaves and Δ15N is the difference between two locations. 2.3. Statistical Analysis One-way analysis of variance (ANOVA) was performed using SPSS 22.0 software (IBM, Armonk, NY, USA) to calculate the mean and standard deviation (SD) of nitrogen isotope values (δ15N) for leaves of different positions and soil of tea plants. Least significant difference method (LSD) was used to evaluate significant differences between different sets of data, where a significant difference is defined as p < 0.05. Origin 2019b (OriginLab, Northampton, MA, USA) software was used for mapping the sample locations.',\n",
       " '11207240': '',\n",
       " '11207110': '2. Materials and Methods 2.1. Study Design This study is a secondary analysis of a larger study evaluating the effects of dietary phosphorus on intestinal phosphorus absorption. The full study design and details are presented elsewhere [22]; here, we present a summary of the parent study with details relevant to the secondary analysis reported herein. The parent study employed a 2 × 3 factorial design (factors = health status and dietary phosphorus load) using n = 72 commercially available Sprague Dawley rats (Charles River, Indianapolis, IN, USA). Eight-week-old male rats underwent either a 5/6th nephrectomy (Nx, n = 36) or sham operation (n = 36) at Charles River prior to shipment to Purdue University ~1 week post-surgery. Rats were received in four shipment cohorts (n = 18/shipment), with each shipment containing n = 9 Nx and n = 9 sham-operated rats. Rats were randomized to one of three dietary treatments within the four shipment blocks and the two disease states, using a complete block design. During a 3 week acclimatization period, investigators handling the rats were blinded to group allocation. Upon arrival, rats were pair-housed in solid-bottom cages with Aspen bedding on a 12 h light/dark cycle in a temperature- and humidity-controlled room. After 2 weeks, rats were transferred to wire-bottom cages and individually housed for 1 week prior to starting diet treatment; rats remained individually housed in wire-bottom cages until the end of the study. During the 3 week acclimatization period, rats were maintained on the Teklad 2018 non-autoclaved rodent diet (Envigo, Indianapolis, IN, USA). This diet contained 0.7% phosphorus (w/w) and was fed ad libitum for 3 weeks to allow for the development of kidney disease. At the end of this period (i.e., 4 weeks post-surgery and ~12 weeks of age), rats were treated for 1 additional week with one of three treatment diets: low phosphorus (low P; 0.1% w/w [Teklad TD.85010]), high phosphorus (high P; 1.2% w/w [Teklad TD.85349]), or low phosphorus followed by acute high phosphorus on the last (7th) day (LPHP). (For this secondary analysis, LPHP rats were excluded from analysis because rats were only on the high P diet for 4 h and euthanized 2 h later, making it unlikely that the lower intestinally generated UT precursors would have reached peak levels. Thus, to prevent potential confounding in this exploratory study, the a priori decision was made to only analyze data from animals that had been on the low P or high P diets consistently for 1 week.) The dietary phosphorus levels and treatment groups were selected based upon a previous study by Giral et al. [23]. The high P and low P diets were similar in nutrient content to the Teklad 2018 diet, though the nutrient sources were different. In Teklad 2018, the primary ingredients were wheat, corn, wheat middlings, soybean meal, corn gluten meal, and soy oil, while the primary ingredients for the high P and low P diets were sucrose, egg white solids, corn starch, corn oil, and cellulose. Non-phytate phosphorus, which has significantly higher bioavailability than phytate phosphorus [24], was present in the diets at 0.1%, 0.4%, and 1.2% (w/w) in the low P, Teklad 2018, and high P diets, respectively. (For a detailed description of the study diets, see Tables S1 and S2). During the 1 week treatment period, access to food was restricted to a daily 4 h feeding window, with water available ad libitum. The restricted feeding protocol was required by the parent study to determine the pharmacokinetics of phosphorus absorption [22]. Upon completion of the 4 h feeding window on the last day of the study (i.e., 7th day), radiolabeled 33P was administered via oral gavage and phosphorus pharmacokinetics assessed via periodic blood draws over 2 h. Upon completion, rats were euthanized via CO2 asphyxiation. Blood draws for assessment of UT and kidney function were taken at baseline (i.e., day 1, before the start of treatment diets; jugular vein draw) and after euthanasia (i.e., day 7; abdominal aortic draw). Blood samples were immediately aliquoted into both uncoated tubes and lithium heparin-coated tubes for serum and plasma collection, respectively (Sarstedt, Numbrecht, Germany). Serum and plasma were stored at −80 °C until analysis. At necropsy, kidneys were inspected to confirm sham or Nx operation: all sham-operated rats had two fully intact kidneys, while Nx rats were missing the right kidney and only had the central ~1/3 of the left kidney present. Rat experiments were conducted in adherence with Purdue University Animal Care and Use Committee (PACUC) guidelines, following an approved protocol (1402001030) that ensured the 3Rs of animal research are followed (i.e., replacement, reduction, and refinement). This study protocol was pre-registered at Animal Study Registry [25]. 2.2. Analytical Methods 2.2.1. Blood Urea Nitrogen (BUN) Plasma samples collected at sacrifice were analyzed for blood urea nitrogen (BUN) by a commercial colorimetric kit (Urea Assay Kit, BioAssay Systems, Hayward, CA, USA) following the manufacturer’s instructions. 2.2.2. Uremic Toxins (UT) Serum samples collected at sacrifice in all rats (n = 7–11/group) and serum samples collected at baseline from a subset of rats (n = 3–4/group) were analyzed for TMAO, IS, and pCS. Collected serum samples were frozen at −80 °C immediately after collection and stored for two months until analysis in this study. Samples were extracted by mixing 50 µL serum with 150 µL ice-cold methanol (MeOH) and 10 µL of a mixture of deuterated UT analogs containing 33 ng/µL IS-d4, 33 ng/µL pCS-d7, and 166 ng/µL TMAO-d9. Samples were vortex mixed for 10 min then separated via centrifugation for 10 min at 16,200× g. The supernatant was separated, dried, and resuspended in 50 µL of a 95:5 mixture of water–acetonitrile with 0.1% formic acid. An Agilent 1290 Infinity II liquid chromatography (LC) system coupled to an Agilent 6470 series QQQ mass spectrometer (MS, Agilent Technologies, Santa Clara, CA, USA) was used to analyze UT metabolites. A Water’s Corporation HSS T3 2.1 mm × 150 mm, 1.8 µm column was used for LC separation (Water’s Corporation, Milford, MA, USA). The buffers were (A) water + 0.1% formic acid and (B) acetonitrile + 0.1% formic acid. The linear LC gradient was as follows: time 0 min, 0% B; time 2 min, 0% B; time 6 min, 100% B; time 7 min, 100% B; time 7.1 min, 0% B; time 10 min, 0% B. The flow rate was 0.3 mL/min. Multiple reaction monitoring was used for MS analysis. The data were acquired in negative and positive electrospray ionization (ESI) mode (see Table S3 for details). The jet stream ESI interface had a gas temperature of 325 °C, gas flow rate of 8 L/min, nebulizer pressure of 45 psi, sheath gas temperature of 250 °C, sheath gas flow rate of 7 L/min, capillary voltage of 3500 V in positive mode and negative mode, and nozzle voltage of 1000 V. The Δ electron multiplier voltage (ΔEMV) was 300 V in both positive and negative modes. All data were analyzed with Agilent Masshunter Quantitative Analysis (Version B.08.00). 2.3. Statistics For this secondary analysis, the subsets of Nx rats (n = 22) and sham rats (n = 18) randomized to the high P or low P diets were included in the analysis. Sample size in the parent study was calculated based on phosphorus absorption data in Nx from Marks et al. [22,26]. Statistics were completed using Statistical Analysis Software version 9.4 (SAS Institute, Raleigh, NC, USA). Two-way ANOVA was performed for all outcomes of this ancillary study, with main effects for health status, diet, and their interaction compared using Tukey post hoc comparisons; statistical significance was set at α < 0.05. Model residuals were normally distributed for most comparisons. When model residuals were not normally distributed, data were transformed using logarithmic transformations prior to statistical analysis. When logarithmic transformations were performed, the data were normalized and the equality of variances assumption was satisfied. Back-transformed values are presented in their original units. All data are presented as mean ± SEM of the original data, unless otherwise specified. p-values are presented for the overall two-way ANOVA model (pmodel), Nx vs. sham (phealth status), high P vs. low P diet (pdiet), and the interaction between the two model variables (i.e., interaction between health status and diet, pinteraction). 2.2. Analytical Methods 2.2.1. Blood Urea Nitrogen (BUN) Plasma samples collected at sacrifice were analyzed for blood urea nitrogen (BUN) by a commercial colorimetric kit (Urea Assay Kit, BioAssay Systems, Hayward, CA, USA) following the manufacturer’s instructions. 2.2.2. Uremic Toxins (UT) Serum samples collected at sacrifice in all rats (n = 7–11/group) and serum samples collected at baseline from a subset of rats (n = 3–4/group) were analyzed for TMAO, IS, and pCS. Collected serum samples were frozen at −80 °C immediately after collection and stored for two months until analysis in this study. Samples were extracted by mixing 50 µL serum with 150 µL ice-cold methanol (MeOH) and 10 µL of a mixture of deuterated UT analogs containing 33 ng/µL IS-d4, 33 ng/µL pCS-d7, and 166 ng/µL TMAO-d9. Samples were vortex mixed for 10 min then separated via centrifugation for 10 min at 16,200× g. The supernatant was separated, dried, and resuspended in 50 µL of a 95:5 mixture of water–acetonitrile with 0.1% formic acid. An Agilent 1290 Infinity II liquid chromatography (LC) system coupled to an Agilent 6470 series QQQ mass spectrometer (MS, Agilent Technologies, Santa Clara, CA, USA) was used to analyze UT metabolites. A Water’s Corporation HSS T3 2.1 mm × 150 mm, 1.8 µm column was used for LC separation (Water’s Corporation, Milford, MA, USA). The buffers were (A) water + 0.1% formic acid and (B) acetonitrile + 0.1% formic acid. The linear LC gradient was as follows: time 0 min, 0% B; time 2 min, 0% B; time 6 min, 100% B; time 7 min, 100% B; time 7.1 min, 0% B; time 10 min, 0% B. The flow rate was 0.3 mL/min. Multiple reaction monitoring was used for MS analysis. The data were acquired in negative and positive electrospray ionization (ESI) mode (see Table S3 for details). The jet stream ESI interface had a gas temperature of 325 °C, gas flow rate of 8 L/min, nebulizer pressure of 45 psi, sheath gas temperature of 250 °C, sheath gas flow rate of 7 L/min, capillary voltage of 3500 V in positive mode and negative mode, and nozzle voltage of 1000 V. The Δ electron multiplier voltage (ΔEMV) was 300 V in both positive and negative modes. All data were analyzed with Agilent Masshunter Quantitative Analysis (Version B.08.00).',\n",
       " '11207098': '',\n",
       " '11207068': '2. Materials and Methods A detailed description of the CLINMIR study protocol has recently been published elsewhere [40]. Below is a summary of the clinical trial. 2.1. Trial Design The clinical trial protocol was approved by the Scientific Research and Ethics Committee of the Hospital University La Paz (HULP), Madrid (Spain) in version 1 in June 2022 and protocolled by the HULP Code 6164. The present protocol clinical trial has also been registered at http://clinicaltrials.gov (first posted on 3 August 2022) with the number NCT05486260. The CLINMIR study is a pilot randomized, parallel, triple-blind, and placebo-controlled clinical trial allocated in three arms according to treatment with a food supplement enriched in the protein miraculin (DMB) in malnourished cancer patients exhibiting TDs because of active chemotherapy and radiotherapy and adjusted by type of cancer. All patients were recruited from medical consultations in the Clinical and Dietary Nutrition Unit (UNC&D) and by referral from the Oncology Service of the HULP to UNC&D. 2.2. Participants The main inclusion criteria were patients 18 years of age and older with cancer, active chemotherapy and/or radiotherapy, and/or immunotherapy treatment who had a weight loss ≥ 5% in the last six months, malnutrition diagnosis assessed by Global Leadership Initiative on Malnutrition (GLIM Criteria) [41], and TDs measured by electrogustometry. Additionally, patients had to have a life expectancy greater than 3 months and be able to feed by oral intake. Patients also had an understanding of the clinical study guidelines. The exclusion criteria included patients participating in another clinical trial, enteral or parenteral nutrition, poorly controlled diabetes mellitus (HbA1c > 8%), uncontrolled hypertension or hyper/hypothyroidism, severe digestive toxicity due to treatment with chemo-radiotherapy, severe kidney or liver disease (chronic renal failure, nephrotic syndrome, cirrhosis, etc.), severe dementia, brain metastases, eating disorders, history of severe neurological or psychiatric pathology that may interfere with treatment, alcoholism or substance abuse, severe gastrointestinal diseases, and unwillingness to consume the miraculin-based food supplement. Intolerance to miraculin was a withdrawal criterion. Any medication that did not interfere with the study formulation was allowed and registered in the clinical research data. 2.3. Interventions Patients who met the selection criteria were randomized to one of three arms of the clinical trial. The first arm had 150 mg of DMB equivalent to 2.8 mg of miraculin + 150 mg of freeze-dried strawberries per orodispersible tablet; the second arm had 300 mg of DMB equivalent to 5.6 mg of miraculin; and the third arm contained 300 mg of freeze-dried strawberries per orodispersible tablet as a placebo. All treatments were isocaloric (Table 1). Those patients who voluntarily agreed to participate signed the informed consent form. Over 3 months, each patient consumed an orodispersible tablet containing DMB or placebo five minutes before each main meal (breakfast, lunch, and dinner). The clinical trial had six face-to-face visits in two phases, one selection visit (vS) in the selection phase and five visits in the experimental phase (Figure 1). On the selection visit, nutritional status was assessed according to the GLIM criteria as well as electrical (electrogustometry) and chemical taste perception (taste strips). The included patients received the questionnaires to complete and hand in at visit 1 (food daily record of 3 days, one holiday (weekend day, a day off, or a day out of the usual routine), quality of life questionnaire (EORTC QLQ-C30) and International Physical Activity Questionnaire (IPAQ) as well as the blood sample extraction appointment (analysis of biochemical parameters and fatty acids from erythrocytes). Experimental phase visits were 4–7 days after their chemotherapy infusion, except visit 1 before it. At visit 1 (v1), patients were randomized and provided with the necessary product (DMB or placebo) until their next visit (v2). Anthropometric measurements, electrical bioimpedance, and the Sniffin’ sticks smell test were carried out. Healthy eating and physical exercise guidelines for cancer patients were explained. As part of the next visit, the following forms were delivered: a product efficacy satisfaction questionnaire, a product consumption control daily sheet, a product consumption tolerance record sheet, and a record sheet of adverse effects. Additionally, individualized nutritional treatment was implemented. If an oral nutritional supplement was needed, a polymeric, hypercaloric, and hyperproteic formula enriched in omega-3 fatty acids was prescribed depending on their energy requirements. Visits 2 (v2, 4–7 days after the chemotherapy session), 3 (v3, ± 1 month after visit 1) and 4 (v4, ± 2 months after visit 1) were similar and they were carried out 4–7 days after the chemotherapy session. During these visits, nutritional status was monitored, and anthropometric measurements and smell and taste tests (electrogustometry, taste strips tests and Sniffin’ sticks smell test) were carried out. In these visits, biochemical parameters were also measured. Completed questionnaires were collected (food daily record, quality of life questionnaire, product efficacy, product consumption control, tolerance record and adverse effects record) and behavioral reinforcement (nutritional treatment and physical activity, consumption and registration of the assigned treatment, and tolerance and adverse effects registry). Patients received the questionnaires to complete and hand in at the next visit. Finally, during visit 5 (v5, ± 3 months after v1 and 3–4 days after the patient’s chemotherapy), nutritional status was assessed and anthropometric measurements and taste and smell tests were carried out (electrogustometry, taste strips tests, and Sniffin’ sticks smell test). A blood sample was extracted (biochemical parameters and fatty acids from erythrocytes) for analysis. Food daily records and completed quality of life questionnaires were collected, as well as product efficacy questionnaires, product consumption control, tolerance records, and adverse effects records. Behavioral reinforcement of nutritional treatment and physical activity were carried out. 2.4. Outcomes Malnourished cancer patients with TDs who consumed DMB were expected to improve their taste perception by reducing the electrical-chemical taste perception threshold from baseline (v0) and throughout the intervention. Moreover, it is expected that DMB consumption improves the chemical and olfactory perception of food. Improvements in dietary intake and nutritional and safety biochemical parameters, as well as improvements in the essential and polyunsaturated fatty acid status assessed through the fatty acid composition of erythrocytes, were expected because of a better perception of food. Tolerance and possible adverse effects were also outcomes studied since several doses were evaluated. All parameters were evaluated from baseline to the end of the intervention and evolution was measured through the different visits carried out (v1, v2, v3, v4, v5). 2.5. Sample Size Because the CLINMIR study was exploratory and there was a lack of previous studies using miraculin-based nutrition supplements in cancer patients, the sample size was established by the researchers. The number established was 10 patients per arm given a sample size of 30 patients. The results obtained will be able to serve to establish the sample size needed to evaluate the efficacy of the intervention product in multicenter studies. 2.6. Randomization and Blinding Randomization was carried out using computer-generated random numbers in blocks of six taking into account treatment and cancer type. This sequence was generated by the Biostatistics Unit (HULP). The allocation sequence was provided in a separate document. To implement the allocation, the sequences were sequentially numbered and sealed in envelopes that were mailed to the nutritionist who enrolled and assigned participants to interventions. When the patient signed the informed consent (v1), the patient’s randomization envelope was opened. Researchers, trial patients, care providers (nutritionists, nurses, physicians), outcome assessors, data analysts, and the promoter were blinded after assignment to interventions. Both miraculin-based food supplements and placebo had similar appearances (pink tablets). They were packaged in white opaque bottles with 30 orodispersible tablets identified by a lot number (L01, L02, L03) and a barcode for tracking. The test product in its powder form (DMB) and the placebo were provided by Baïa Food (Medicinal Gardens SL) to Rioja Nature Pharma. The packaging, in the form of bottles equipped with protective technology for moisture and oxygen-sensitive products (Activ Vial®), was supplied by CSP Aptar Technologies. Rioja Nature Pharma was responsible for the manufacturing, labeling, identification, and supply of the final product, and maintained the blind throughout the study until the statistical analysis was completed. 2.7. Specific Methodology 2.7.1. Malnutrition Criteria Nutritional diagnosis of malnutrition was established through the GLIM criteria based on phenotypic and etiological criteria. It requires at least one phenotypic criterion and one etiologic criterion to diagnose malnutrition. Body composition by bioelectrical impedance analysis (BIA) was used to evaluate reduced muscle mass. Gastrointestinal symptoms as supportive indicators were considered to assess and evaluate reduced food assimilation and major infection. Finally, trauma or acute conditions were associated with inflammation. Malnutrition was classified as moderate or severe malnutrition [41]. Nutritional status was evaluated at all study visits. 2.7.2. Anthropometric Parameters Anthropometric parameters were taken using standard techniques, following the international norms established by the WHO. Body weight was measured using a clinical digital scale (capacity 0–150 kg). The percentage of weight loss was assessed as follows: [(current weight − weight 6 months ago)/weight 6 months ago] * 100. Height was measured with a height meter with an accuracy of 1 mm (range, 80–200 cm). Body mass index (BMI) was determined using weight (kg)/height (m)2. Anthropometric parameters were measured at the main visits (v1, v3, v4 and v5). 2.7.3. Daily Food Record Diet was collected in three different days’ daily food records, one of which had to be a holiday. Patients were instructed to record the weight of the food consumed or, if this was not possible, to record household measurements (spoonfuls, cups, etc.). All records were thoroughly reviewed by a nutritionist in the presence of the patient to ensure that the information collected was complete. Foods, drinks, dietary supplements, and preparations consumed were transformed into energy and nutrients using DIAL software Version 3.15 (Alce Ingeniería, Madrid, Spain). Results were compared with the recommended intakes of the Spanish population. 2.7.4. Electrogustometry The threshold for an electric-induced taste stimulus (taste acuity) was measured using an electrogustometer (SI-03 Model, Sensonics International, Haddon Heights NJ, USA). Patients were instructed not to eat or drink for an hour before electrogustometry. A monopolar electrode applied the electric stimulus. The electrogustometer produces low-amplitude stimuli of a predetermined duration (0.5 s). The methodology used was that recommended by the manufacturer. The electric threshold scores were measured in the area of the fungiform papillae on both sides of the tongue. To detect thresholds, a two-down and one-up forced-choice single staircase procedure and a stimulus-response staircase were used. Threshold differences between the left and right sides greater than 7 dB were considered abnormal [42]. 2.7.5. Taste Strips Test The taste strips test is a validated method to measure chemical taste perception [30]. This tool is based on the chemical perception of taste through taste-impregnated filter paper strips (Burghart Messtechnik GmbH, Holm, Germany). Four different taste strips (sweet, sour, salty, and bitter) were measured at four different concentrations each. For the assessment of whole-mouth gustatory function, strips were placed on the tongue and savored with the closed mouth for 10 s. Once the strip was removed, the participants had to identify the taste within a forced-choice procedure. A maximum score of 16 points (four concentrations of each of the four basic taste qualities) was obtained. Hypogeusia was considered when a score below nine was obtained regardless of age. 2.7.6. Sniffin’ Sticks Smell Test Smell perception was measured based on odor-containing felt-tip pens (“Sniffin’ sticks” Burghart Messtechnik GmbH, Germany). Consuming food, drinks, or cigarettes 15 min before testing was not allowed. A total of 16 odor pens were presented to be identified. For each pen, a flash card with four choices was provided (e.g., pineapple, orange, blackberry, strawberry). Each uncovered odor pen was held 2 cm in front of the nostrils for 3–4 s. Based on the multiple forced-choice paradigm; patients had to choose the best match with their olfactory perception. The score sums all correct answers and was used to differentiate between normosmia and hyposmia depending on the age of the patient. 2.8. Quality of Life This was evaluated using the EORTC QLQ-C30 questionnaire for cancer patients validated in Spanish [43]. The questionnaire is formed of five functional scales (daily activities and physical, emotional, cognitive, and social functioning), three symptomatic scales (fatigue, pain and nausea, and vomiting), one overall health scale, and six questions about dyspnea, insomnia, anorexia, constipation, diarrhea, and economic impact. All questions are about the previous week and are scored with 1 to 4 points. The last two questions have a score from 1 to 7, with 1 being terrible and 7 being excellent. Scores obtained are standardized from 0 to 100 points to determine the disease impact on each scale. High scores on the global health status and functional scales indicate a better quality of life, while low scores on the symptoms scale indicate a decrease in quality of life. 2.9. Tolerance and Adverse Events Gastrointestinal disorders such as abdominal distension, abdominal pain, nausea, regurgitation or gastroesophageal reflux, vomiting, constipation, diarrhea, and flatulence were defined and recorded based on the Common Terminology Criteria for Adverse Events (CTCAE) from the National Cancer Institute [44]. These adverse events were classified as Grade 0 (not described), Grade 1 (mild), Grade 2 (moderate), Grade 3 (severe), Grade 4 (mortality risk), and Grade 5 (death associated with an event). Additionally, the patients were asked if they could be related to product consumption. 2.10. Fatty Acid Profile of Erythrocytes The separation and quantification of fatty acids from erythrocyte lipids have been reported in previous works [45]. Briefly, erythrocyte lipid extraction and fatty acid methylation were performed as described by Lepage and Roy (1988) [46]. Fatty acid methyl esters (FAME) were identified and quantified by comparing their retention times by gas chromatography-mass spectrometry (GC-MS). This analysis was performed by injecting 1 µL into a Bruker (Bremen, Germany) model 456-GC high-resolution gas chromatograph coupled to a Bruker model EVOQ TQ triple quadrupole mass spectrometer as follows: GC conditions (a)ZB-FAME capillary column (30 m × 0.25 mm ID × 0.20 um film).(b)Split mode injector (100:1)(c)Injector temperature: 250 °C(d)Transfer line temperature: 240 °C(e)Carrier gas: He (1 mL/min)(f)Temperature ramp: 100 °C (2 min) up to 210 °C (5 min) at 4°/min. MS conditions:(a)Temperature of the source: 240 °C(b)Full scan from 45 Da to 450 Da(c)Electron impact ionization (EI+) at 70eVFood daily record 2.11. Biochemical Parameters Biochemical analyses were carried out in the Biochemistry Laboratory of the Hospital La Paz, an ISO-certified laboratory, at each visit (v1, v3, v4, v5) using an Olympus AU5400 Automated Chemistry Analyzer (Olympus Corporation, España Barcelona, Spain). 2.12. Miraculin-Based Food Supplement Taste Perception A visual analog scale (VAS) was designed by the researchers to obtain information about the miraculin-based food supplement’s taste perception efficacy. The questionnaire included five questions using 10 cm scales, where 0 means not at all or very bad and 10 means very good or very effective. The questions included were as follows: Do you notice a food taste change after consuming the product? Does food taste better to you? Does it allow you to eat more food? What is your opinion of the product? Are you satisfied with the effectiveness of the product? Does the administration of the product seem adequate to you? 2.13. Statistical Methods Data analysis was carried out by the intention to treat. Quantitative data are presented as the means ± standard deviations (SD), and percentages. Data type distribution was determined using Shapiro-Wilks tests. Levene’s test was used to evaluate the homogeneity of variances. Parametric or nonparametric tests were performed depending on the data distribution. General linear mixed models (GLM) of covariance (ANCOVA) were used to evaluate the differences between means for treatment, time, and treatment x time using covariates as the baseline data. The analysis of the qualitative variables and percentages was carried out through χ2 or Fisher’s F analysis. Double-sided tests were applied when needed, and a p-value < 0.05 was considered statistically significant. Data were analyzed using R Project for Statistical Computing (https://www.r-project.org/ Accessed on 1 March 2024). 2.7. Specific Methodology 2.7.1. Malnutrition Criteria Nutritional diagnosis of malnutrition was established through the GLIM criteria based on phenotypic and etiological criteria. It requires at least one phenotypic criterion and one etiologic criterion to diagnose malnutrition. Body composition by bioelectrical impedance analysis (BIA) was used to evaluate reduced muscle mass. Gastrointestinal symptoms as supportive indicators were considered to assess and evaluate reduced food assimilation and major infection. Finally, trauma or acute conditions were associated with inflammation. Malnutrition was classified as moderate or severe malnutrition [41]. Nutritional status was evaluated at all study visits. 2.7.2. Anthropometric Parameters Anthropometric parameters were taken using standard techniques, following the international norms established by the WHO. Body weight was measured using a clinical digital scale (capacity 0–150 kg). The percentage of weight loss was assessed as follows: [(current weight − weight 6 months ago)/weight 6 months ago] * 100. Height was measured with a height meter with an accuracy of 1 mm (range, 80–200 cm). Body mass index (BMI) was determined using weight (kg)/height (m)2. Anthropometric parameters were measured at the main visits (v1, v3, v4 and v5). 2.7.3. Daily Food Record Diet was collected in three different days’ daily food records, one of which had to be a holiday. Patients were instructed to record the weight of the food consumed or, if this was not possible, to record household measurements (spoonfuls, cups, etc.). All records were thoroughly reviewed by a nutritionist in the presence of the patient to ensure that the information collected was complete. Foods, drinks, dietary supplements, and preparations consumed were transformed into energy and nutrients using DIAL software Version 3.15 (Alce Ingeniería, Madrid, Spain). Results were compared with the recommended intakes of the Spanish population. 2.7.4. Electrogustometry The threshold for an electric-induced taste stimulus (taste acuity) was measured using an electrogustometer (SI-03 Model, Sensonics International, Haddon Heights NJ, USA). Patients were instructed not to eat or drink for an hour before electrogustometry. A monopolar electrode applied the electric stimulus. The electrogustometer produces low-amplitude stimuli of a predetermined duration (0.5 s). The methodology used was that recommended by the manufacturer. The electric threshold scores were measured in the area of the fungiform papillae on both sides of the tongue. To detect thresholds, a two-down and one-up forced-choice single staircase procedure and a stimulus-response staircase were used. Threshold differences between the left and right sides greater than 7 dB were considered abnormal [42]. 2.7.5. Taste Strips Test The taste strips test is a validated method to measure chemical taste perception [30]. This tool is based on the chemical perception of taste through taste-impregnated filter paper strips (Burghart Messtechnik GmbH, Holm, Germany). Four different taste strips (sweet, sour, salty, and bitter) were measured at four different concentrations each. For the assessment of whole-mouth gustatory function, strips were placed on the tongue and savored with the closed mouth for 10 s. Once the strip was removed, the participants had to identify the taste within a forced-choice procedure. A maximum score of 16 points (four concentrations of each of the four basic taste qualities) was obtained. Hypogeusia was considered when a score below nine was obtained regardless of age. 2.7.6. Sniffin’ Sticks Smell Test Smell perception was measured based on odor-containing felt-tip pens (“Sniffin’ sticks” Burghart Messtechnik GmbH, Germany). Consuming food, drinks, or cigarettes 15 min before testing was not allowed. A total of 16 odor pens were presented to be identified. For each pen, a flash card with four choices was provided (e.g., pineapple, orange, blackberry, strawberry). Each uncovered odor pen was held 2 cm in front of the nostrils for 3–4 s. Based on the multiple forced-choice paradigm; patients had to choose the best match with their olfactory perception. The score sums all correct answers and was used to differentiate between normosmia and hyposmia depending on the age of the patient. 2.13. Statistical Methods Data analysis was carried out by the intention to treat. Quantitative data are presented as the means ± standard deviations (SD), and percentages. Data type distribution was determined using Shapiro-Wilks tests. Levene’s test was used to evaluate the homogeneity of variances. Parametric or nonparametric tests were performed depending on the data distribution. General linear mixed models (GLM) of covariance (ANCOVA) were used to evaluate the differences between means for treatment, time, and treatment x time using covariates as the baseline data. The analysis of the qualitative variables and percentages was carried out through χ2 or Fisher’s F analysis. Double-sided tests were applied when needed, and a p-value < 0.05 was considered statistically significant. Data were analyzed using R Project for Statistical Computing (https://www.r-project.org/ Accessed on 1 March 2024).',\n",
       " '11207046': '2. Methods 2.1. Samples In total, eight composite water samples (1 L each) were collected in pairs along the Iskar River from the two locations (42.367698, 23.555463—Dragushinovo Village and 42.431095, 23.531900—villa area “Mechkata”) with an automatic sampler over a period of 24 hrs to avoid day-night bias on the following four dates: 3 November 2022, 17 November 2022, 8 December 2022, and 22 December 2022. This resulted in a total of 8 samples (2 locations × 4 dates). The first site was located about 800 m after the municipal WWTP in Samokov, thus being suitable for monitoring the effluent waters immediately after entering the river, and the second site was about 10 km downstream, just before the river flows into Lake Iskar. The Iskar River is the main source of water for nearly 80% of the households (approx. 1.3 million people) [17] and industrial needs of the capital city, Sofia, and Sofia region. As this is such a key freshwater ecosystem, we aimed to assess the impact of the WWTP effluent waters on the taxonomy, resistome and mobilome of the Iskar River. Samples were transferred to the laboratory within 6 h and processed immediately. A Lafil 400 vacuum filtration system (Rocker, Dist., New Taipei City 244014, Taiwan) was used with 0.2 µm 47 mm diameter nylon membrane filters (Cytiva, Whatmantm,, Little Chalfont, Buckinghamshire, UK) for filtration purposes. pH was recorded for each sample (Table S6). Each sample was homogenized, divided into two 500 mL portions and passed through two filters. Filters were stored at −80 °C until all samples were collected. 2.2. DNA Extraction and Sequencing When the last pair of samples was acquired, all filters were aseptically cut into smaller pieces and subjected to DNA extraction in duplicate with the Environmental DNA & RNA Purification Kit (Cat. No. E3572, EURx, Gdańsk, Poland). Simultaneously, a Microbial Community Standard (ZymoBIOMICS, Zymo Research, Irvine, CA, USA, cat. D6300) was also subjected to DNA extraction and sequencing (Figure S4). DNA from replicates was measured both spectrophotometrically by BioDrop µLite+ (Biochrom Ltd., Harvard Bioscience, Inc. 84 October Hill Road Holliston, MA 01746, United States) and fluorometrically by Qubit 4.0 Fluorometer (Thermo Fisher Scientific, Waltham, MA, USA). The replicates with the highest 260/280 ratio and highest DNA concentration were used for NGS library construction with Collibri™ ES DNA Library Prep Kit (Thermo Fisher Scientific, USA) with 50 ng genomic DNA input. The libraries were pooled and sequenced on NextSeq 550 with V2.5 Mid Output kit (2 × 150 bp) (Illumina, San Diego, CA, USA). The sample information is provided in Table S1. 2.3. Bioinformatic Analysis Reads were automatically trimmed based on adapter content and average Q30+ quality score with the default setting of Trim-galore v0.6.7, ensuring that only high-quality reads were kept. Next, reads aligned to the human genome were removed with bmtagger and the remaining reads (from now on referred to as raw reads) were quality checked with FASTQC v0.11.9. For the purpose of co-assembly, three sets of reads were produced by concatenating all raw reads (C1-all), and reads per location (C2-Dragushinovo, C3-Mechkata). Co-assemblies were produced with megahit v1.2.9 [18] with default settings using kmers of length 21, 29, 39, 59, 79, 119, 141 and contigs shorter than 1000 bp were filtered out. Binning was performed on the 3 co-assemblies with MaxBin2 v2.2.7 [19], metaBAT2 [20], and CONCOCT v1.1.0 [21] separately by using the default parameters for each tool. Consolidation of the resulting metagenome-assembled genomes (MAGs) was performed by the bin refinement module of MetaWrap v1.3 [22]. Further, refined bins were subjected to the reassembly module of MetaWrap by first extracting reads mapping to the bins and feeding them to the SPAdes v.3.15.1 with different parameters. Afterwards, the resulting re-assemblies were compared and the best one by CheckM output were chosen as representative. Next, resulting MAGs were quality checked, annotated and uploaded to Bioproject PRJNA1071831 and were used for inference of host discovery of ARGs. All MAGs were quality checked with QUAST v5.2 and CheckM v1.2.2 [23]. Raw reads were used to identify ARGs with ResFinder software v2023-08-22 [24] and database v2023-04-12, applying 60% coverage and 60% identity thresholds. For ARGs from assemblies, both ResFinder and AMRFinderPlus v3.11.14 [25] were used. MGEs were searched within assemblies with MobileElementFinder v1.1.2 (https://pypi.org/project/MobileElementFinder/, accessed on 12 December 2023). Putative plasmids were identified using three plasmid prediction tools: Deeplasmid [26] with score above 0.75, PlaSquid v1.0.0 [27], and Plasme v1.1 [28]. The contigs identified as plasmids by at least one tool were included in the analysis. Co-occurrence of either ARG or MGE or both on contigs of plasmid origin were reported. MetaCompare2.0 was also used simultaneously for risk assessment of mobilizable ARGs on pathogens [29]. Taxonomic assignment to raw reads was performed with Kraken2 v2.1.2 [30] with the PlusPF (Standard plus Refeq protozoa & fungi), built in January 2023, followed by Bracken v2.8 [31] as previously described [32]. The resulting OTU count tables were imported into the Qiime2 platform [33], filtered from low count taxa (n = 20) as well as taxa found in less than 3 samples, and used for differential abundance analysis with the ANCOM-BC2 plugin [34]. Counts were transformed to relative frequencies, and taxa below 1% were merged into “Others” and used to visualize the taxonomic composition with matplotlib v3.7 and seaborn v0.12.0 Python packages. Alpha and beta diversity, group-significance, and volatility analyses were performed with ResistoXplorer [35] and Qiime2. Microbial community standard was analysed by submitting the raw reads to the miqScoreStandardShotgun tool v1.0.1. Network analysis was performed with the NetworkX Python package v3.2.1 by only including associations with positive Pearson correlation scores of ≥0.8 coupled with p-value ≤ 0.005. A network graph was then drawn in CytoScape 3.9.1. Statistical calculations were performed either automatically for ResistoXplorer and Qiime2 or by Scipy v1.11.1 for the network analysis.',\n",
       " '11207001': '2. Materials and Methods 2.1. Animals Animal experimental procedures were performed following the National Institutes of Health guidelines for the care and use of laboratory animals and were reviewed and approved by the Institutional Animal Care and Ethics Committee (Approval No. RA AM 2016-007). C57BL/6 mice were purchased from the Jackson Laboratory and the animals were bred in the institutional animal facility and fed ad libitum on standard chow diet. Mice were housed in a temperature-controlled room (23 °C) and maintained on a 12-h dark/12-h light cycle. All experiments were performed using 8–10 weeks old male mice. The mice were randomly divided into 2 groups, 5–6 mice per group per cage, and body weight was recorded. Two types of sucrose-free high fat diets (45% kcal fat) were used: i.e., cocoa butter based experimental diet (D18060406, Research Diets Inc., New Brunswick, NJ, USA) and fish oil based diet (D18060407, Research Diets Inc., New Brunswick, NJ, USA). Body weight and food intake of mice were recorded on a weekly basis. Fish oil based HFD cohort is being used as a control for comparison for various studies conducted simultaneously with the various HFDs based on different fat sources. At 21–22 weeks of dietary intervention, oral glucose tolerance test (OGTT) and insulin tolerance test (ITT) were performed. The mice were sacrificed at the end of dietary intervention (24 weeks) and all tissues and organs were collected and flash frozen in liquid nitrogen. Blood was collected for isolation of plasma. Plasma samples were stored at −80 °C until use. Liver tissue was fixed in 10% neutral buffered formalin and preserved by paraffin embedding. The fecal content was also aseptically collected from each mouse and stored at −80 °C until use. 2.2. Oral Glucose Tolerance Test (OGTT) To perform the glucose tolerance test at 21 weeks feeding, mice were fasted for 12 h. Glucose (1 g/kg body weight) was administered orally to each mouse and blood glucose levels at different time points were measured at 0, 10, 20, 30, 60, 90, and 120 min using a portable glucometer. 2.3. Insulin Tolerance Test (ITT) To determine insulin tolerance at 22 weeks feeding, mice were fasted for 4 h and insulin (0.8 U/kg body weight) was administered intraperitonially. Blood glucose levels were determined at 0, 15, 30, 45, 60, 75, 90, 105, and 120 min using a portable glucometer. 2.4. Plasma Measurements Plasma metabolic hormone levels including insulin, c-peptide, glucagon, amylin, leptin, PYY, PP, ghrelin, GLP-1, GIP, and resistin were measured using a MILLIPLEX kit (MILLIPLEX kit MAP mouse Metabolic Magnetic Bead Panel, Millipore, Burlington, MA, USA) in accordance with the manufacturer’s instructions. 2.5. Histological Analysis Histology analysis was performed using standard protocols. Detailed methods of Histological Analysis were described in the Supplementary Materials methodology section. Hematoxylin-Eosin (H&E) staining and for Oil Red O staining for fat content following standard protocols [23,24,25,26,27] as described in the Supplementary Materials methodology section. 2.6. Gene Expression Analysis by qRT-PCR Total RNA was extracted from liver tissue using RNeasy Mini kit (Qiagen, Hilden, Germany). 1 μg of total RNA were reverse transcribed into cDNA using the High Capacity cDNA Reverse Transcription Kit (Applied Biosystems, Foster City, CA, USA). Quantitative real-time PCR was performed in a QuantStudio™ 5 Real-Time PCR System using TaqMan master mix reagents and gene specific TaqMan assays (Applied Biosystems, Foster City, CA, USA). Each reaction was performed in triplicate using standard reaction conditions. Target gene cycle threshold (Ct) values were normalized against GAPDH Ct values and gene expression level relative to control was calculated using 2−ΔΔCT method. The gene specific primers are listed in Supplementary Materials Table S1. 2.7. Microbiome Sequencing DNA extraction from mouse fecal samples was carried out using Qiamp DNA fast stool mini kit Qiagen, Germany and quantified using a Qubit fluorometer (Thermo Fisher Scientific, Waltham, MA, USA), following the manufacturer’s instructions. A total of 5 ng/μL of microbial DNA was amplified using gene-specific primers that target the bacterial 16S rRNA V3 and V4 regions. The full-length primers with overhang adapter sequences used are as follows: 16S Amplicon PCR Forward Primer: 5′ TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGCC TACGGGNGGCWGCAG; 16S Amplicon PCR Reverse Primer: 5′ GTCTCGTGGGCTCGGAGATGTGTAT AAGAGACAGGACTACHVGGGTATCT AATCC. PCR reaction was carried out using KAPA HiFi HotStart Ready Mix PCR mix kit (Roche Diagnostics, Indianapolis, IN, USA), following the manufacture’s protocol. The resulting PCR product was confirmed on a Bioanalyzer using high sensitivity DNA chip (Agilent Technologies, Inc., Santa Clara, CA, USA), purified using AMPure XP beads (Agilent Technologies, Inc., Santa Clara, CA, USA) and bound to dual indices using the Nextera XT Index Kit (Illumina Inc., San Diego, CA, USA), following the manufacturers’ recommendations. Purified and normalized libraries were multiplexed up to 24 samples and paired-end sequencing was carried out using MiSeq platform (Illumina Inc., San Diego, CA, USA). 2.8. 16S rRNA Microbiome Analysis and Bioinformatics Statistics The sequence data obtained were analyzed using QIIME 2 (Quantitative Insights Into Microbial Ecology) version 2022.8 [28,29], https://www.nature.com/articles/s41598-019-44369-y (accessed on 1 April 2023) ref-CR26 and MicrobiomeAanalyst package [30]. Using QIIME 2, the forward and reverse reads of the same sample were first joined. The paired reads were demultiplexed and quality filtered at Q-score of 25. High-quality amplicon sequence variants (ASVs) were obtained by using the Divisive Amplicon Denoising Algorithm-2 (DADA2) algorithm [31]. The taxonomy profile analysis was performed against the Greengenes reference database [32] and transformed into relative abundance at the phylum, class, order, family, genus, and species levels. After this, using Greengenes reference database, open-reference operational taxonomic units (OTUs) were picked out from the non-chimeric sequences at 97% similarity. The most abundant read from each OTU was selected as the representative read for that OTU. The taxonomy associated with Greengenes database to which OTUs matched was assigned to OTUs. To test two-group differences in the percentage of analyzable read numbers between HFF and HFS, we calculated p-values using the Wilcoxon signed-rank test. The OTU table was filtered to remove the OTUs containing <10 counts in all samples and were transformed into relative abundances at the phylum, class, order, family, genus, and species levels. 2.9. Biodiversity Analysis The intra-sample (alpha) and the inter-sample (beta) diversity analyses were performed as described elsewhere [33,34,35]. The alpha diversity was calculated using three different measures: Observed and Fisher statistics and phylogenetic diversity [36], performed as the references cited for each. The bacterial genus/spp. data were provided. Beta diversity was calculated using Bray-Curtis distances matrices, generating the two-dimensional Principal Coordinate Analysis (PCoA) plots. Weighted and un-weighted UniFrac distance matrices were used to derive the beta diversity of the samples [33] which data could be accessed. Non-parametric Mann-Whitney/Kruskal-Wallace tests were used to determine the statistical significance of α-diversity measures and permutational MANOVA was used to determine the differences in β-diversity. 2.10. Identification of Biomarker Microbiome Linear discriminant analysis (LDA) effect size (LEfSe) analysis was performed to identify the bacteria for which the relative abundance was significantly increased or decreased in each phenotypic category. LEfSe algorithm using the Benjamini-Hochberg false discovery rate (FDR) adjusted p-value cutoff of 0.05 and the logarithmic LDA score cutoff of 2.0. The LEfSe bar plots were created using the MicrobiomeAnalyst package [30]. In all analyses, p-values were corrected for Benjamini-Hochberg FDR. Random Forest analysis was applied to the 16S rRNA data using machine learning methods developed by using the MicrobiomeAnalyst package to identify the most significant microbial features. Features with a minimum prevalence of 10% across samples were included, and those with >0.005 accuracy were considered significant. Data were further transformed to cantered log ratio (CLR) before applying the Random Forest classification algorithm. 2.11. Microbial Metagenomic Functional Predictions Phylogenetic investigation of communities by reconstructing unobserved states-2 (PICRUSt2) software to predict the microbial content from each gut microbe sample and functionally annotate the data [37]. The results were further analyzed using the Microbiomeanalyst package [30]. To investigate the metabolic network of the predicted organisms, we used the MetaCyc database [38], which contains data regarding chemical compounds, reactions, enzymes, and metabolic pathways that have been experimentally validated. The correlation between microbial community and their metabolites was performed using a Microbiomeanalyst package [30]. 2.12. Prediction of Biomarker Microbiome-Epigenome Interactions The human microbiome affects the host gene expression as the microbiota-secreted proteins, microbiota-derived components, and microbiota-derived metabolites may regulate the host’s physiology by modifying its gene expression. Therefore, biomarker microbiome-epigenome interactions were predicted by using the Microbiomeanalyst taxon set enrichment analysis [30], Human Microbe-Disease Association Database (HMDAD) [38], and the human microbiome affect the host epigenome (MIAOME) database [39,40].',\n",
       " '11206969': '',\n",
       " '11206959': '2. Materials and Methods The powered red orange extract (ROE) (batch no. 6/21) and the powered olive leaf extract (OLE) (batch no. E21/2691) utilized in this work were prepared by Medinutrex (Catania, Italy). The ROE and OLE were used for the analysis as previously published [12]. The impact of various concentrations (ROE: 0.5–1–3 mg/mL; OLE: 0.5–1–3 mg/mL; COMBO 1: ROE 0.5 + OLE 0.5 mg/mL; COMBO 2: ROE 1 + OLE 1 mg/mL; and COMBO 3: ROE 3 + OLE 3 mg/mL) was assessed on cell viability, and ROE and OLE 1 mg/mL were chosen as the optimal and non-toxic concentrations for the subsequent experiments. 2.1. Assessment of Antioxidant Activity in a Cell-Free System FRAP The potential of the antioxidant activity of the ROE and OLE was determined by employing the ferric reducing antioxidant power assay (FRAP) in a cell-free system, an in vitro model where biochemical reactions occur independently of living cells. The analysis was conducted following the procedure described in the FRAP assay kit (MAK369, Sigma-Aldrich, St. Louis, MO, USA). In this method, antioxidants were used as reductants in a redox-linked colorimetric reaction, wherein Fe3+ is reduced to Fe2+. The absorbance was measured spectrophotometrically at 590 nm, and color intensity results were proportional to FRAP in the sample. Three replicate wells were used for each group. The results are expressed as µM ferrous equivalents and calculated as follows:(B × D/V)/1000, where B = ferrous ammonium sulphate amount from the standard curve (nmol); D = sample dilution factor; and V = sample volume added into the reaction well (μL). 2.2. Cell Culture Experiments 2.2.1. Cell Culture and Differentiation of 3T3-L1 Pre-Adipocytes The murine pre-adipocyte cell line 3T3-L1 was obtained from the American Type Culture Collection (ATCC; Rockville, MD, USA). Once reached confluence, to induce differentiation, The medium was replaced with an adipogenic one, as already reported by Ballistreri et al. [17]. The cultures were maintained at 37 °C and 5% CO2 in an incubator, and the medium was changed every 3 days in the presence or absence of the ROE and OLE, alone and in combination. 2.2.2. Cell Viability Assay 3T3-L1 cells were seeded in a 96-well cell culture plate and were treated for 48 h with different concentrations of the extracts alone or in combination (ROE: 0.5–1–3 mg/mL; OLE: 0.5–1–3 mg/mL; COMBO 1: ROE 0.5 + OLE 0.5 mg/mL; COMBO 2: ROE 1 + OLE 1 mg/mL; and COMBO 3: ROE 3 + OLE 3 mg/mL). Afterward, to establish an in vitro model of ferroptosis, 3T3-L1 pre-adipocytes were cultured in a 96-well cell culture plate, pre-treated with the ROE and OLE, alone and in combination, and incubated for 24 h. Then, the cells were challenged with erastin (1 μM) in the presence or absence of the extracts for 48 h. Cell viability was assessed by the MTT assay, which was carried out following the previously described procedure by Consoli et al. [18]. Eight replicate wells were utilized for each group. 2.2.3. Oil Red O Staining 3T3-L1 pre-adipocytes were plated in a 24-well plate. After 24 h, the medium was replaced with adipogenic medium, and the cells were differentiated for 13 days with ROE–OLE alone or in combination (ROE: 0.5–1 mg/mL; OLE: 0.5–1 mg/mL; COMBO 1: ROE 0.5 + OLE 0.5 mg/mL; and COMBO 2: ROE 1 + OLE 1 mg/mL). For a second analysis, 3T3-L1 cells were seeded in a 24-well cell culture plate and maintained for 5 days with adipogenic medium. Then, the cells were pre-treated with the ROE and OLE, alone and in combination, and incubated for 24 h. Finally, the differentiated adipocytes were treated with PA (0.5 mM) or ER (1, 5, 10 μM) in the presence or absence of the extracts for 48 h. To calculate the quantity of lipid droplets, at the end of differentiation, in both experimental conditions, Oil Red O staining was performed as previously reported by Ballistreri et al. [17]. The analysis was carried out in triplicate. 2.2.4. Palmitic Acid–Bovine Serum Albumin Complex Preparation Palmitic acid (Sigma-Aldrich, St. Louis, MO, USA) was dissolved in NaOH 0.1 M. BSA (Beyotime Biotechnology, Shanghai, China) was dissolved in DMEM to form a 12% BSA solution. For palmitic acid in conjunction with BSA, 12% BSA was added to the palmitic solution and conjugated by ultrasonic treatment. Following an incubation period of 1 h at 50 °C with continuous shaking, the PA–BSA complexes should be clear and were cooled at room temperature, filtrated through a membrane filter (0.22 μm), and stored at +4 °C for further use. 2.2.5. Sircol Collagen Assay The Sircol collagen assay (Biocolor, Belfast, UK) was utilized to measure total soluble collagen in cell culture supernatants, as previously described by Raffaele et al. [19], with some adjustments. For these experiments, confluent cells in 24-well plates were incubated for 13 days under the same abovementioned conditions for Oil Red O Staining, and the analysis was conducted in triplicate. 2.2.6. RNA Extraction and qRT-PCR Analysis Total RNA was isolated from 3T3-L1 differentiated adipocyte cells using TRIzol reagent (Invitrogen, Carlsbad, CA, USA). It was subsequently converted into cDNA through an Applied Biosystem (Foster City, CA, USA) reverse transcription reagent. Quantitative Real-time PCR analysis was conducted, as formerly described by Vanella et al. [20], to evaluate the gene expression of fatty acid synthase (FAS), diacylglycerol acyltransferases 1 and 2 (DGAT-1 and DGAT-2, respectively), sterol regulatory element-binding protein-1c (SREBP-1C), collagen type I alpha 1 (COL1A1), interleukin-6 (IL-6), fatty acid transport proteins 1 and 4 (FATP-1 and FATP-4, respectively), and heme oxygenase-1 (HO-1). The analysis was executed in triplicate. 2.3. Statistical Analysis Each analysis constituted the results of a minimum of three distinct experiments. Fisher’s method was utilized to determine the statistical significance (p < 0.05) of the differences between the experimental groups. For a comparison of treatment groups, the null hypothesis was tested by either a single-factor analysis of variance (ANOVA) for multiple groups or an unpaired t-test for two groups, and the data were presented as the means ± SEMs.',\n",
       " '11206945': '',\n",
       " '11206929': '2. Methods This review aimed to identify and summarize the available evidence on selenium intake during the antenatal and postnatal periods, its impact on maternal mental health, and the risk of developing postnatal depression. A non-systematic literature review was conducted using electronic searches of the Medline (via PubMed) and Google Scholar databases to identify relevant publications. The initial search targeted interventional or observational studies investigating the relationship between selenium intake and postpartum women’s mental status, using keywords such as ‘selenium’, ‘nutrients’, ‘trace elements’, ‘micronutrients’, along with ‘postnatal depression’, ‘postpartum depression’, ‘postpartum depressive symptoms’, and ‘perinatal depression’, until December 2023. Studies reporting nutrient supplementation, nutritional status, or dietary intakes during pregnancy in the context of postnatal depression were further reviewed to identify if selenium-specific outcomes were reported and, if so, were included in this review. Additionally, focused searches were performed to explore the possible underlying mechanisms and biological plausibility of the beneficial effects of selenium on postnatal depression.',\n",
       " '11206924': '2. Materials and Methods To review current knowledge on the impact of choline deficiency and supplementation in patients receiving TPN, we performed a literature search in Pubmed on 23 January 2023 (with an update on 27 April 2024) with the search strategy “choline” and “parenteral nutrition”. Titles were scanned for relevance to the topic by the authors and potentially related publications evaluated as full text. Additionally, reference lists of these publications and forthcoming statements of organizations like EFSA (see below) were searched. Interventional studies involving animals or humans, and other studies that require ethical approval, had to list the authority that provided approval and the corresponding ethical approval code.',\n",
       " '11206916': '2. Materials and Methods 2.1. Sampling and Bacteria Isolation In total, 25 samples were analyzed; 17 of them were represented by T. molitor larvae (10 g) and 8 by frass samples (10 g). T. molitor larvae were reared in laboratory conditions at the Department of Veterinary Sciences of the University of Pisa (Italy). The larvae were kept at a temperature of 25 ± 1 °C and a relative humidity of 50–60%. The insects were reared on unsold food or former foodstuff, such as spent brewery grains, and bread of different types (white, wholemeal, spicy, mixed). As a source of water supply, potato slices were administered once or twice a week depending on the stage of development of the larvae. Frass samples were obtained by fasting the larvae for 48 h in boxes with perforated bottom to avoid contact between the larvae and the frass. Frass was thus collected in a previously sterilized container located under the boxes. No antibiotics were administered during the entire rearing cycle. Each sample was placed inside a sterile tube, identified by a sequential number, and processed as follows: 10 g of larvae were taken and placed inside 50 mL conical Falcon centrifuge tubes into which an aliquot of sterile saline solution was added. Obtained samples were stirred for one minute to reduce external microbial contamination. The washing solution was then removed. Washed larvae were collected into a sterile Stomacher bag with 90 mL of sterile saline solution and then subjected to crushing and homogenization. The homogenate was collected with a sterile loop and transferred onto a Petri dish containing VRBGA medium (Violet Red Bile Glucose Agar, Thermo Fisher Scientific, Milan, Italy) for the isolation of Enterobacteriaceae or KAA (Kanamycin Aesculin Azide Agar, Thermo Fisher Scientific, Milan, Italy) for Enterococcus spp. VRBGA plates were incubated at 37 °C for 24 h, while KAA plates at 42 °C for 48 h. For frass samples, 10 g of substrate were placed in 50 mL conical Falcon centrifuge tubes with 50 mL of sterile saline solution; the sample was then mixed using a vortex mixer. The solution obtained was collected with a sterile loop and inoculated on the same cultural media reported above. After the incubation period, bacterial growth was assessed for both Enterobacteriaceae and Enterococcus spp. One to three typical colonies (according to the culture media manufacturer’s instructions) of Enterobacteriaceae and Enterococcus spp. from positive samples were subcultured to obtain pure isolates. Obtained isolates were inoculated in BHI broth (Brain Heart Infusion, Thermo Fisher Scientific, Milan, Italy) and incubated at 37 °C per 24 h. After incubation, broth cultures were had 15% glycerol (Microbiol Diagnostici, Cagliari, Italy) added and were stored at −20 °C. Isolates were then tested for their antibiotic resistance profile and for identification purposes. 2.2. Determination of Antibiotic Resistance Profile The disk-diffusion antimicrobial susceptibility testing method was applied using a bacterial inoculum prepared in sterile saline adjusted to the density of a McFarland 0.5 standard (Remel™ McFarland Equivalence Turbidity Standards, Thermo Fisher Scientific, Milan, Italy). The inoculum was evenly spread over the entire surface of an MH agar plate (Mueller Hinton, Thermo Fisher Scientific, Milan, Italy) by swabbing. After the application of antimicrobial disks (Thermo Fisher Scientific, Milan, Italy), the plates were incubated at 37 °C for 24 h. After incubation, inhibition zones were read and zone diameters were interpreted and categorized as susceptible, intermediate, or resistant according to the CLSI [23] and EUCAST [24] breakpoint tables for Enterobacterales and enterococci. Escherichia coli ATCC 25922 and Staphylococcus aureus ATCC 25923 were used as control strains for Enterobacteriaceae and enterococci, respectively. The following antimicrobials were tested for Enterobacteriaceae: ampicillin 10 µg, amoxicillin–clavulanic acid 30 µg, cefoxitin 30 µg, cefotaxime 30 µg, chloramphenicol 10 µg, tetracycline 30 µg, trimethoprim–sulfamethoxazole 25 µg, gentamicin 30 µg, streptomycin 10 µg, imipenem 10 µg, and aztreonam 30 µg, while the following antimicrobials were used for enterococci: tetracycline 30 µg, ciprofloxacin 5 µg, vancomycin 5 µg, imipenem 10 µg, erythromycin 15 µg, ampicillin 10 µg, gentamicin 120 µg, linezolid 30 µg, chloramphenicol 30 µg, and quinupristin–dalfopristin 15 µg. Specifically, for Enterobacteriaceae, CLSI breakpoints were applied for tetracycline, streptomycin, and cefotaxime, while for enterococci, CLSI breakpoints were applied for ampicillin, quinupristin–dalfopristin, tetracycline, erythromycin, ampicillin, gentamicin, linezolid, and chloramphenicol. The isolates were then grouped into clusters based on their phenotypic antibiotic susceptibility profile. Hierarchical clustering was performed with R studio version 4.2.1 using packages factoextra, FactoMineR, ggplot2, Rcolorbrewer, ggpubr, and tidyverse. 2.3. 16S rDNA Sequencing for Bacterial Isolates Identification At least one isolate from each profile presenting at least one resistance to the antibiotics tested was identified. Identification at the species level was performed via 16S rDNA gene sequencing according to [25]. The commercial kit “GenEluteTM Bacterial Genomic DNA kit” (Sigma-Aldrich, St. Louis, MO, USA) was used for DNA extraction following the protocol provided by the manufacturer. EconoTaq®® PLUS 2X Master Mix (Lucigen, Madison, WI, USA) was used for the amplification of 16S rDNA genes. The amplicons (1300 bp) were sent to BMR Genomics (Padua, Italy) for sequencing. The GenBank database was employed to attribute the species using the BLASTN local alignment search tool.',\n",
       " '11206901': '2. Materials and Methods 2.1. Test Substance The exogenous NT mixture (AMP/CMP/GMP/UMP = 16:41:19:24) is a white solid powder provided by Dalian Zhenao Biotechnology Co., Ltd. (Dalian, China) [21,24]. 2.2. Main Reagents and Instruments 2.2.1. Main Reagents The main reagents and instruments were as follows: DMEM high-glucose culture medium (GIBCO, Grand Island, NE, USA), fetal bovine serum (GIBCO, Grand Island, NE, USA), streptomycin mix solution (100×) (Coolaber, Beijing, China), palmitic acid/stearic acid (Coolaber, Beijing, China), bovine serum albumin (fatty acid-free) (Yuan Ye, Shanghai, China), recombinant human insulin solution (Pricella, Wuhan, China), phosphate buffer solution (Hyclone, Shanghai, China), CCK-8 cell proliferation assay kit (KeyGEN, Shanghai, China), GSH-PX assay kit (Nanjing Jiancheng Bioengineering Institute, Nanjing, China), LDH assay kit (Nanjing Jiancheng Bioengineering Institute, Nanjing, China), MDA assay kit (Nanjing Jiancheng Bioengineering Institute, Nanjing, China), PEPCK assay kit (Nanjing Jiancheng Bioengineering Institute, Nanjing, China), an HK assay kit (Nanjing Jiancheng Bioengineering Institute, Nanjing, China), glycogen assay kit (Nanjing Jiancheng Bioengineering Institute, Nanjing, China), BCA protein quantitative detection kit (ready to use) (UElandy, Suzhou, China), protein extraction reagent (Tian Derui, Beijing, China), 10×tris-glycine-SDS electrophoresis buffer (Tian Derui, Beijing, China), 10×TBST (pH8.0) (Tian Derui, Beijing, China), Rainbow 180 broad spectrum protein marker (11-180KD) (Tian Derui, Beijing, China), NC membrane (0.45 μm pore size) (Millipore, Burlington, MA, USA), PMSF (Amresco, Wayne, NJ, USA), acrylamide (Amresco, Wayne, NJ, USA), bis-acrylamide (Amresco, Wayne, NJ, USA), APS (Amresco, Wayne, NJ, USA), ECL (Millipore, Burlington, MA, USA). 2.2.2. Main Instruments The main instruments were as follows: a carbon dioxide incubator (MC0-15AC, Sanyo Electric Co., Ltd., Osaka, Japan), multifunctional microplate reader (BMG FLUOstar Omega, BMGLABTECH GmbH, Offenburg, Germany), high-speed refrigerated centrifuge (Eppendorf, Eppendorf AG, Hamburg, Germany), electronic balance (Sartouris, Gottingen, Germany), Mini P-4 electrophoresis tank (Cavoy, Beijing, China), wet transfer electrophoresis tank (Cavoy, Beijing, China), electrophoresis apparatus (Bio-Rad, Hercules, USA), shaking incubator (Thermo Fisher Scientific, Waltham, MA, USA), pH meter (Sartorius, Gottingen, Germany), and homogenizer (Monad, Cambridge, MA, USA ). 2.3. Experimental Cells and Treatmentss HepG2 cells from a human hepatocellular carcinoma cell line were procured from the Cell Bank of the Chinese Academy of Sciences in Shanghai, China. These cells exhibited an adherent phenotype and were routinely cultured in DMEM medium supplemented with 10% fetal bovine serum and 1% penicillin–streptomycin at 37 °C in a humidified atmosphere with 5% CO2. Passage of cells occurred every 3–4 days upon reaching 80–90% confluence through trypsin digestion, followed by subculturing. Experimental procedures were conducted utilizing cells in the logarithmic growth phase, ensuring optimal growth conditions. 2.3.1. Preparation of PA Solution First, a 10% BSA solution was prepared by weighing out 0.8 g and adding it to 8 mL of distilled water, followed by complete dissolution in a water bath at 55 °C. Subsequently, 800 μL of a 300 mM PA solution dissolved in anhydrous ethanol was slowly added to the 10% BSA solution while stirring thoroughly until complete dissolution occurred in the 55 °C water bath. This yielded a 27 mM PA stock solution. The solution was sterilized using a 0.22 micrometer filter membrane and aliquoted and stored frozen. 2.3.2. Establishment of the HepG2-IR Cell Model After reaching confluence in adherent culture, HepG2 cells in the logarithmic growth phase were selected. Following recommendations from the relevant literature, various concentrations of palmitic acid (PA) were employed for intervention to establish an insulin-resistant cell model. The concentrations tested included 0, 0.10, 0.15, 0.20, 0.25, and 0.30 mM. Following a 24 h incubation period, glucose levels in the supernatant of the culture medium were assessed using a glucose assay kit (Acmec, Shanghai, China) to evaluate glucose consumption by the cells in each group. Subsequently, cell viability was evaluated at each concentration. The optimal insulin resistance model for HepG2 cells was determined based on glucose consumption and cell viability assessments. In this study, a PA concentration of 0.25 mM was selected for further experiments. 2.3.3. Cell Groups The cells were divided into six groups as follows: Control Group (Control): HepG2 cells in optimal growth conditions were cultured in complete normal medium, and equivalent BSA solution concentrations were maintained as the control group. Insulin-Resistant Model Group (IR-HepG2): HepG2 cells in optimal growth conditions were cultured in complete medium supplemented with 5% FBS and 0.25 mM PA for 24 h. Subsequently, during the intervention phase, they were maintained in the same medium for an additional 24 h. Low Dose of NTs Group (NTs-L): HepG2 cells in optimal growth conditions were cultured in complete medium supplemented with 5% FBS and 0.25 mM PA for 24 h. Following this, during the intervention phase, they were cultured in complete medium containing 5% FBS, along with 50 μmol/L NTs and 0.25 mM PA, for an additional 24 h. Medium Dose of NTs Group (NTs-M): HepG2 cells in optimal growth conditions were cultured in complete medium supplemented with 5% FBS and 0.25 mM PA for 24 h. During the intervention phase, they were cultured in complete medium containing 5% FBS, along with 100 μmol/L NTs and 0.25 mM PA, for an additional 24 h. High Dose of NTs Group (NTs-H): HepG2 cells in optimal growth conditions were cultured in complete medium supplemented with 5% FBS and 0.25 mM PA for 24 h. Subsequently, during the intervention phase, they were cultured in complete medium containing 5% FBS, along with 200 μmol/L NTs and 0.25 mM PA, for an additional 24 h. Positive Control Group (Met): HepG2 cells in optimal growth conditions were cultured in complete medium supplemented with 5% FBS and 0.25 mM PA for 24 h. During the intervention phase, they were cultured in complete medium containing 5% FBS, along with 0.1 mg/mL metformin and 0.25 mM PA, for an additional 24 h. Following the intervention phase, insulin treatment was administered at a concentration of 1 × 10−6 mol/L for 40 min. Cells and supernatants were then collected for relevant indicator measurements. 2.4. The Experimental Method 2.4.1. Cell Viability Assay Cells were plated at a density of 100 μL/cell per well, corresponding to approximately 1 × 104 cells, in a 96-well cell culture plate. Following treatment according to experimental protocols, 10 μL of CCK-8 detection solution was added to each well and then incubated at 37 °C for 1–4 h. Subsequently, the absorbance at a wavelength of 450 nm was measured using an enzyme-linked immunosorbent assay (ELISA) reader for each well. 2.4.2. Glucose Consumption Measurement Cells (approximately 1 × 104 cells) were cultured at a volume of 100 μL per well in a 96-well cell culture plate. Following treatment with various experimental groups, glucose production in each cell group was assessed using the guidelines provided by the glucose assay kit. A standard curve was generated, and glucose concentration was determined utilizing the formula: Glucose concentration (mmol/L) = Standard concentration × (Sample well OD − Blank well OD)/(Standard well OD − Blank well OD). Glucose consumption (mmol/L) was calculated as the difference between glucose content in the blank well and the measurement well, with normalization to 1 for the control group. 2.4.3. Glycogen Content Determination Cells were seeded at a density of 3 × 105 cells per well in a 6-well plate. Following treatment with various experimental conditions, the cells underwent three washes with PBS. Subsequently, cell collection was conducted for glycogen detection as per the manufacturer’s guidelines for the glycogen assay kit. Additionally, the protein content of each HepG2 cell group was quantified using the BCA method. The glycogen-to-protein ratio (mg/mg) was calculated and normalized to 1 for the control group. 2.4.4. Liver Enzyme Assay The cells were initially seeded at a density of 3 × 105 cells per well in a 6-well plate. Following treatment with different experimental conditions, the cells underwent three washes with PBS. Subsequently, cell samples were collected to assess PEPCK and HK activities using the PEPCK assay kit and hexokinase (HK) activity assay kit, respectively, following the manufacturer’s instructions. Additionally, the protein content of each HepG2 cell group was quantified using the BCA method. The ratio of PEPCK or HK activity to total protein (mg) was then calculated, with normalization to 1 for the control group. 2.4.5. Oxidative Stress Biomarker Detection The cells were seeded at a density of 3 × 105 cells per well in a 6-well plate. Following treatment with various experimental conditions, the cells underwent three washes with PBS. Subsequently, cell collection was conducted for LDH, MDA and GSH-PX detection utilizing assay kits according to the manufacturer’s instructions. Additionally, the protein content of each group of HepG2 cells was quantified using the BCA method. The ratio of MDA or GSH-Px to total protein (mg) was calculated, with normalization to 1 for the control group. 2.4.6. Western Blot The cells were cultured in a 6-well cell culture plate and treated with different experimental groups, each with 3 replicates. Subsequently, the cells were digested using trypsin, and the supernatant was discarded after centrifugation at 1000 rpm for 5 min. The cells were then washed twice with pre-chilled PBS on ice before storing the cell pellet at -80 °C. Western blot analysis was performed to detect various proteins, including AMPK (1:1000), p-AMPK (Thr172) (1:1000), AKT (1:2000), p-AKT (Ser473) (1:1000), IRS-1 (1:1000), p-IRS-1 (Ser307) (1:500), FOXO1 (1:1000), p-FOXO1 (Ser256) (1:1000), GS (1:1000), p-GS (Ser641) (1:1000), GSK3β (1:1000), p-GSK3β (Ser9) (1:1000), G6pase (1:1000), GLUT2 (1:1000), GLUT4 (1:1000), NOX4 (1:500), and NF-κB (1:2000). (1) Tissue protein extraction was performed by employing pre-chilled RIPA protein extraction reagent, augmented with a protease inhibitor cocktail and a phosphatase inhibitor specifically targeting phosphorylated proteins. Prior to extraction, 0.1M PMSF stock solution was added to achieve a final concentration of 1 mM PMSF. Tissue samples were homogenized in lysis buffer at a 1:9 (weight/volume) ratio using a Fluka electric tissue homogenizer at 15,000 rpm for three cycles of 10 s with 10 s intervals, while maintaining low temperature in an ice/water mixture. Following homogenization, the samples underwent a 20 min incubation on ice, followed by centrifugation at 13,000 rpm for 20 min at 4 °C. Subsequently, the resultant supernatant was gathered, divided into aliquots, and stored appropriately. Then, protein concentration was quantified using the BCA method, and adjustments were made by diluting with RIPA buffer and incorporating 5× reducing sample buffer to achieve a final concentration of 2 mg/mL, followed by protein denaturation through boiling for 5 min. (2) Western blot: Either 12% or 8% separation gels were prepared based on the target protein’s molecular weight, incorporating a 5% stacking gel. A total of 20 μg of protein was loaded per well. Electrophoresis conditions entailed applying a constant 90 V for approximately 20 min to the stacking gel and 160 V to the separation gel, with endpoint determination using pre-stained protein markers. Wet transfer was performed at a constant current of 300 mA using a 0.45 μm pore size NC membrane, transferring for 1 h for 12% separation gels or 2 h for 8% separation gels. Post-transfer, the membrane was stained with Ponceau S solution to evaluate transfer efficiency and mark the lanes. The membrane was blocked in 3% BSA-TBST at room temperature for 30 min with gentle shaking. The primary antibody was diluted in 3% BSA-TBST, incubating initially at room temperature for 10 min, followed by overnight incubation at 4 °C. The next day, the membrane was equilibrated to room temperature and incubated for an additional 30 min. The membrane was washed with TBST five times for 3 min each. The secondary antibody, goat anti-rabbit IgG (H+L) HRP, was diluted in 5% skim milk-TBST at a ratio of 1:10,000 and shaken gently at room temperature for 40 min. Subsequently, the membrane was washed with TBST six times for 3 min each. Finally, visualization was carried out using ECL chemiluminescence detection reagent, and grayscale intensity was analyzed using Image-J software (Total Lab Quant V11.5, Newcastle upon, Tyne, UK). 2.5. Statistical Analysis The experimental results were presented as mean ± standard deviation. Data analysis was performed using SPSS 26.0 software. Initially, the data were subjected to homogeneity of variance analysis. If the data met the assumption of homogeneity of variance, one-way ANOVA was applied. For non-normally distributed or heteroscedastic data, appropriate variable transformations were applied to fulfill the assumptions of normality or homogeneity of variance before conducting one-way ANOVA. If the assumptions were still not met post-transformation, non-parametric methods were employed for statistical analysis. The Least Significant Difference (LSD) method was used for post hoc pairwise comparisons between experimental and control groups, with a significance threshold of p < 0.05 indicating significant differences. 2.4. The Experimental Method 2.4.1. Cell Viability Assay Cells were plated at a density of 100 μL/cell per well, corresponding to approximately 1 × 104 cells, in a 96-well cell culture plate. Following treatment according to experimental protocols, 10 μL of CCK-8 detection solution was added to each well and then incubated at 37 °C for 1–4 h. Subsequently, the absorbance at a wavelength of 450 nm was measured using an enzyme-linked immunosorbent assay (ELISA) reader for each well. 2.4.2. Glucose Consumption Measurement Cells (approximately 1 × 104 cells) were cultured at a volume of 100 μL per well in a 96-well cell culture plate. Following treatment with various experimental groups, glucose production in each cell group was assessed using the guidelines provided by the glucose assay kit. A standard curve was generated, and glucose concentration was determined utilizing the formula: Glucose concentration (mmol/L) = Standard concentration × (Sample well OD − Blank well OD)/(Standard well OD − Blank well OD). Glucose consumption (mmol/L) was calculated as the difference between glucose content in the blank well and the measurement well, with normalization to 1 for the control group. 2.4.3. Glycogen Content Determination Cells were seeded at a density of 3 × 105 cells per well in a 6-well plate. Following treatment with various experimental conditions, the cells underwent three washes with PBS. Subsequently, cell collection was conducted for glycogen detection as per the manufacturer’s guidelines for the glycogen assay kit. Additionally, the protein content of each HepG2 cell group was quantified using the BCA method. The glycogen-to-protein ratio (mg/mg) was calculated and normalized to 1 for the control group. 2.4.4. Liver Enzyme Assay The cells were initially seeded at a density of 3 × 105 cells per well in a 6-well plate. Following treatment with different experimental conditions, the cells underwent three washes with PBS. Subsequently, cell samples were collected to assess PEPCK and HK activities using the PEPCK assay kit and hexokinase (HK) activity assay kit, respectively, following the manufacturer’s instructions. Additionally, the protein content of each HepG2 cell group was quantified using the BCA method. The ratio of PEPCK or HK activity to total protein (mg) was then calculated, with normalization to 1 for the control group. 2.4.5. Oxidative Stress Biomarker Detection The cells were seeded at a density of 3 × 105 cells per well in a 6-well plate. Following treatment with various experimental conditions, the cells underwent three washes with PBS. Subsequently, cell collection was conducted for LDH, MDA and GSH-PX detection utilizing assay kits according to the manufacturer’s instructions. Additionally, the protein content of each group of HepG2 cells was quantified using the BCA method. The ratio of MDA or GSH-Px to total protein (mg) was calculated, with normalization to 1 for the control group. 2.4.6. Western Blot The cells were cultured in a 6-well cell culture plate and treated with different experimental groups, each with 3 replicates. Subsequently, the cells were digested using trypsin, and the supernatant was discarded after centrifugation at 1000 rpm for 5 min. The cells were then washed twice with pre-chilled PBS on ice before storing the cell pellet at -80 °C. Western blot analysis was performed to detect various proteins, including AMPK (1:1000), p-AMPK (Thr172) (1:1000), AKT (1:2000), p-AKT (Ser473) (1:1000), IRS-1 (1:1000), p-IRS-1 (Ser307) (1:500), FOXO1 (1:1000), p-FOXO1 (Ser256) (1:1000), GS (1:1000), p-GS (Ser641) (1:1000), GSK3β (1:1000), p-GSK3β (Ser9) (1:1000), G6pase (1:1000), GLUT2 (1:1000), GLUT4 (1:1000), NOX4 (1:500), and NF-κB (1:2000). (1) Tissue protein extraction was performed by employing pre-chilled RIPA protein extraction reagent, augmented with a protease inhibitor cocktail and a phosphatase inhibitor specifically targeting phosphorylated proteins. Prior to extraction, 0.1M PMSF stock solution was added to achieve a final concentration of 1 mM PMSF. Tissue samples were homogenized in lysis buffer at a 1:9 (weight/volume) ratio using a Fluka electric tissue homogenizer at 15,000 rpm for three cycles of 10 s with 10 s intervals, while maintaining low temperature in an ice/water mixture. Following homogenization, the samples underwent a 20 min incubation on ice, followed by centrifugation at 13,000 rpm for 20 min at 4 °C. Subsequently, the resultant supernatant was gathered, divided into aliquots, and stored appropriately. Then, protein concentration was quantified using the BCA method, and adjustments were made by diluting with RIPA buffer and incorporating 5× reducing sample buffer to achieve a final concentration of 2 mg/mL, followed by protein denaturation through boiling for 5 min. (2) Western blot: Either 12% or 8% separation gels were prepared based on the target protein’s molecular weight, incorporating a 5% stacking gel. A total of 20 μg of protein was loaded per well. Electrophoresis conditions entailed applying a constant 90 V for approximately 20 min to the stacking gel and 160 V to the separation gel, with endpoint determination using pre-stained protein markers. Wet transfer was performed at a constant current of 300 mA using a 0.45 μm pore size NC membrane, transferring for 1 h for 12% separation gels or 2 h for 8% separation gels. Post-transfer, the membrane was stained with Ponceau S solution to evaluate transfer efficiency and mark the lanes. The membrane was blocked in 3% BSA-TBST at room temperature for 30 min with gentle shaking. The primary antibody was diluted in 3% BSA-TBST, incubating initially at room temperature for 10 min, followed by overnight incubation at 4 °C. The next day, the membrane was equilibrated to room temperature and incubated for an additional 30 min. The membrane was washed with TBST five times for 3 min each. The secondary antibody, goat anti-rabbit IgG (H+L) HRP, was diluted in 5% skim milk-TBST at a ratio of 1:10,000 and shaken gently at room temperature for 40 min. Subsequently, the membrane was washed with TBST six times for 3 min each. Finally, visualization was carried out using ECL chemiluminescence detection reagent, and grayscale intensity was analyzed using Image-J software (Total Lab Quant V11.5, Newcastle upon, Tyne, UK).',\n",
       " '11206896': '',\n",
       " '11206889': '2. Materials and Methods 2.1. Animals and Feeding The experiments were performed in accordance with the guidelines provided by the National Institute of Health for Animal Research and approved with ethical permits from the University Animal Experiments Local Ethics Committee (Ethical Approval No. 2021-609, approved on 6 April 2021). In the study, 56 female Wistar albino rats, weighing 230–270 g and aged 6–7 months, were allocated to 7 experimental groups. The rats were given ad libitum access to commercial pelleted rodent diet and tap water. Hawthorn vinegar was given to the rats by oral gavage daily throughout the 45 trial days. The groups were set out as below:Group—Control (basal diet);Group—N0.5 (0.5 mL/kg untreated hawthorn vinegar daily);Group—N1 (1 mL/kg untreated hawthorn vinegar daily);Group—P0.5 (0.5 mL/kg thermal pasteurized hawthorn vinegar daily);Group—P1 (1 mL/kg thermal pasteurized hawthorn vinegar daily);Group—U0.5 (0.5 mL/kg hawthorn vinegar ultrasound treated daily);Group—U1 (1 mL/kg hawthorn vinegar ultrasound treated daily). 2.2. Hawthorn Vinegar Hawthorn fruit (Crataegus tanacetifolia) samples were sourced from Bursa/Turkey, and the untreated hawthorn vinegar was produced using the method of conventional vinegar manufacturing. After vinegar production, the vinegars were processed by 3 methods of modification [14]. The first product was the vinegar without any treatment (Group N0.5 and N1). The second group’s method was thermal pasteurization (Groups P0.5 and P1). The third group’s methods were the response surface method and ultrasound treatment (Groups U0.5 and U1). 2.3. Determination Bioactive Compounds of Vinegar The total phenolic content of samples was analyzed by Folin–Ciocalteu method [25]. All analyses were performed in triplicate. The absorbance was measured on a UV–Vis spectrophotometer (SP-UV/VIS-300SRB, Spectrum Instruments, Melbourne, Australia). The total flavonoid contents of samples were determined by the colorimetric technique [26], and the concentrations were calculated colorimetrically using a UV spectrophotometer (Spectrum Instrument, SP-UV/VIS-300SRB, Australia). The ascorbic acid concentration of the vinegars was calculated by the AOAC 961.27 vitamin preparation and ascorbic acid 2.6 dichlorophenol indophenol-titrimetric methods [27]. The antioxidant capacities of vinegars were evaluated using the CUPRAC and DPPH methods previously described by [28,29,30]. 2.4. Measurements The weights of rats were measured at the beginning and the end of the trial, and the average weight gains were determined. The average weight gains, body mass indexes (BMIs) and feed conversation ratios (FCRs) were calculated. Blood samples were obtained by puncturing the heart under isoflurane anesthesia at the end of the study. The blood hematocrit, hemoglobin, counts of erythrocyte and leukocyte, and types of leukocyte (neutrophil, lymphocyte and monocyte) were determined using an automatic blood counter device (Exigo, Istanbul, Türkiye) in the Laboratory of the Experimental Animal Institute at University. Also, the neutrophil lymphocyte ratio (NL ratio) was recorded to evaluate whether the rats were stressed or not. According to the literature, NL ratio may serve as an early warning of the pathological state or processes such as stress, inflammation and psychiatric disorders. The NL ratio was calculated from complete white blood cell counts by dividing the absolute count of neutrophils by that of lymphocytes [31]. The blood samples were centrifuged on the same day at 3000 rpm for 10 min at 4 °C to separate the plasma. The intestinal tissue samples were separated into pieces of appropriate sizes (1 g tissue, 9 mL PBS). Then, samples were placed in PBS (pH 7.4) and adjusted to a ratio of tissue weight (g):PBS (mL) = 1:9 volumes. Then, homogenates were prepared using a homogenizer (Interlab, Istanbul, Türkiye). The tissue homogenates were centrifuged at 5000× g for 5 min at 4 °C, and then supernatants were added into microtubes. All blood and tissue samples were stored at −80 °C until the analysis. The Interleukin-1β (IL-1β) and Tumor Necrosis Factor-α (TNF-α) concentrations of plasma and intestinal tissue supernatants were measured using the commercially available rat ELISA kits Rat TNF-α ELISA Kit (Cat. No. 201-11-0765) and Rat IL-1β ELISA Kit (Cat. No. 201-11-0120, (Shanghai Sunred Biotechnology, Shanghai, China), and the assays were performed using a microplate reader (Biotek, Epoch, Winooski, VT, USA). The duodenum samples were fixed for 24 h in a 10% formalin solution. After that, the tissue samples were dehydrated (ethanol), cleared (xylene), and embedded in paraffin. The paraffin blocks were cut into 5 μm-thick sections on a rotary microtome (LIECA) and stained with Mallory’s modified triple-staining approach to examine the general structure of the duodenum [32]. The streptavidin biotin peroxidase complex (strepABC) method was used to determine the localization of TNF-α and IL-1β immunohistochemically in the duodenum tissues of all groups. Here, 5 μm cross-sections were fixed on the adhesive slides and were subjected to deparaffinization and dehydration. The sections were processed in citrate buffer solution (pH 6.0) for 10 min in a microwave oven at 800 watts. Then, tissues were held in 3% hydrogen peroxide (H2O2) for 15 min. Blocking solution A was dripped to prevent nonspecific binding by the IHC Kit (Invitrogen-Histostatin Plus Bulk Kit, Waltham, MA, USA). TNF-α primary antibody (ab220210, TNF-α/11721, 1/100 dilution) and IL-1β primary antibody (ab205924, 1/300 dilution) were applied on the sections in a humid environment at ambient temperature for 1 h. Then, secondary antibodies and streptavidin (Thermo Scientific, Ultravision Large Volume Detection System Anti-Polyvalent, HRP, TP-125-BN, and TS-125-HR, Deutsch, Germany) were dripped onto the sections for 30 min. To demonstrate the antibody reaction, the DAB (3.3′-Diaminobenzidine) chromogen solution was added to the cross-sections for 10 min. Then, Gill III hematoxylin was used for the counterstaining and covered with entellan. Negative controls were set up without the primary antibody. The sections were evaluated by two independent observers using the semi quantitative method by taking the degree of staining in the cross-sections as a criterion. Immunoreactive cells were categorized as negative, mild, moderate, or intensive [33,34]. The immunohistochemically prepared samples were photographed and assessed under a light microscope (Olympus BX51, Tokyo, Japan). 2.5. Statistical Analyses Data were expressed as mean ± SEM using Graph Pad Prism Graphical, Statistical package version 5 (30 days demo version). Statistical analysis was performed using ANOVA, followed by significant difference tests to derive comparisons between individual groups. The non-parametric Dunnett Comparison Test was applied to determine differences in variables with 5% levels of significance (p < 0.05).',\n",
       " '11206883': '',\n",
       " '11206871': '2. Materials and Methods 2.1. Participants and Study Design A total of 156 Thai patients with MASLD were prospectively enrolled in this cross-sectional cohort between 2022 and 2023 in the outpatient liver clinic at the King Chulalongkorn Memorial Hospital, Thailand. Written informed consent was obtained from each participant, and the study was approved by the Institute Ethics Committee (IRB No. 981-64). The study was conducted following the Declaration of Helsinki and the principles of Good Clinical Practice. The inclusion criteria were individuals aged ≥ 18 years with the diagnosis of liver steatosis based on MRI-PDFF grade ≥ 1 (defined as MRI-PDFF ≥ 5.4%) [14]. Exclusion criteria were (1) concomitant chronic viral hepatitis, autoimmune hepatitis, and other chronic liver diseases; (2) presence of other disorders causing secondary steatosis such as human immunodeficiency virus (HIV) infection; (3) presence of cirrhotic complications (e.g., ascites and variceal bleeding), (4) known malignancies including HCC, and (5) self-reported daily alcohol consumption of >10 g in women and >20 g in men. All participants were advised not to take any herbal or nutritional supplements, probiotics and prebiotics, antibiotics, and proton pump inhibitors at least 4 weeks before enrollment. The clinical data of patients with MASLD at the enrollment were recorded. Anthropometric measurements were performed by research assistant nurses. 2.2. Assessment of Liver Stiffness and Steatosis MRE and MRI-PDFF were performed to assess liver stiffness and steatosis, respectively, by the MR imaging system Philips Ingenia at 3.0 T (Philips Healthcare, Best, the Netherlands). Based on a systematic review and meta-analysis of MASLD, the cut-off values for fibrosis ≥ F1 ≥ F2, ≥ F3, and F4 obtained by MRE measurement were 2.6, 3.0, 3.6, and 4.7 kPa, respectively [7]. For MRI-PDFF, the cut-off values for diagnosing steatosis grades ≥ 1, ≥2, and ≥3 were 5.4%, 15.4%, and 20.4% respectively [14]. Acquired imaging data were interpreted by a radiologist who was blinded to the clinical and laboratory information of the patients. 2.3. Fecal Collection, DNA Extraction and Sequencing Participants in the study were advised to collect approximately 1 g of feces in DNA/RNA Shield Fecal Collection Tubes (Zymo Research, Irvine, CA, USA). The fecal samples were then vigorously shaken and stored at −80 °C until they were further processed. DNA from the fecal samples was extracted using the ZymoBIOMICS DNA Miniprep Kit (Zymo Research, USA), following the manufacturer’s instructions. The concentration and purity of total DNA were assessed using a DeNovix™ UV–Vis spectrophotometer, and the DNA was preserved at −20 °C until further experiments. Subsequently, for amplicon-based 16S rRNA gene sequencing, the V4 hypervariable regions were amplified using the forward primer 515F (5′-GTGCCAGCMGCCGCGGTAA-3′) and the reverse primer 806R (5′-GGACTACHVGGGTWTCTAAT-3′). This procedure was followed by paired-end sequencing on the Illumina MiSeq 2 × 300 bp platform (San Diego, CA, USA) conducted at Mod Gut Co., Ltd. (Bangkok, Thailand). 2.4. Data Processing and Analysis The raw sequencing data from each sample, obtained via the Illumina platform, were processed using the nfcore/ampliseq analysis pipeline (https://zenodo.org/records/6411790, accessed on 6 March 2024). The procedure involved initially merging the paired-end reads to reconstruct the V4 region of the 16S rRNA gene by aligning overlapping sequences and eliminating chimeras. Following this process, the sequences were organized into unique sequence features based on the amplicon sequence variant (ASV) technique, leading to the generation of a table that captures the feature abundance across the samples with DADA2 [15]. Additionally, the microbial features were classified taxonomically with the SILVA 138.1 database. Subsequent analyses, including the examination of relative abundance, alpha-diversity, and beta-diversity indices, were conducted on the MicrobiomeAnalyst web platform (https://www.microbiomeanalyst.ca/, accessed on 6 March 2024). Differential bacterial abundance between groups was identified by linear discriminant analysis (LDA) effect size (LEfSe). Predictive pathway analysis was performed using MetaCyc Metabolic Pathway databases through PICRUSt2 [16]. 2.5. Gut Microbiota Enterotypes and Dietary Patterns The classification of gut microbiota profiles at the genus level into distinct enterotypes was conducted using Biotyper, an R software package (Version 4.2.3) based on the methodology as previously described [17,18]. The “biotyper.data.frame” function from the Biotyper package was employed to categorize the data into 4 distinct clusters. Subsequently, the enterotype for each cluster was identified by examining the top three genera within each cluster. Gut enterotypes and their dietary pattern correlations were determined by previous reference studies [17,18]. The cluster enriched in Bacteroides has been revealed to be associated with a meat-based diet, Prevotella associated with a carbohydrate-based diet, and Firmicutes with a vegetable-based diet. 2.6. Random Forest (RF) Classification The Random Forest (RF) classifier was developed with the sklearn library in Python. Within receiver operating characteristic (ROC) curves, the F0–F1 group was labeled as class zero, and the F2–F4 group as class one. To perform Random Forest classification in our cohort, a 75:25 train/test split was used to evaluate the sensitivity and specificity across all possible thresholds. The results from those random classifiers were compared using a threshold value of 0.5 as a baseline. 2.7. Analysis of Microbial Surrogate Biomarkers and Inflammatory Cytokines Peripheral blood samples from the patients were collected and handled within two hours to separate plasma, which was then preserved at −80 °C for downstream analysis. The levels of plasma lipopolysaccharide-binding protein (LBP) and intestinal fatty acid binding protein (I-FABP) were measured using ELISA kits from Hycult Biotech (Uden, The Netherlands). The samples were diluted at a ratio of 1:1000 for LBP and 1:2 for I-FABP, following the manufacturer’s recommended guidelines. Serum interleukin-6 (IL-6) and tumor necrosis factor-α (TNF-α) levels were measured by flow cytometry using LEGENDplex Human CD8/NK Panel V02 (Biolegend, San Diego, CA, USA) according to the manufacturer’s instructions. Briefly, the antibody-conjugated bead against IL-6 and TNF-α was applied, followed by washing of the capture bead complex. Subsequently, a cocktail of biotinylated detection antibodies was introduced. Streptavidin-phycoerythrin was then added, and the resulting fluorescent intensity signal was measured using the BD FACSAria™ Fusion Cell Sorter (BD Biosciences, San Jose, CA, USA) and analyzed through the LEGENDplexTM Data Analysis Software (Version 15 February 2023). 2.8. DNA Extraction and the SNP Genotyping DNA was extracted from peripheral blood mononuclear cells (PBMCs) using the phenol-chloroform isoamyl alcohol method. The quantity and quality of the DNA were assessed using a DeNovix™ UV–Vis spectrophotometer, and the samples were stored at −80 °C until further analysis. The genotyping of the PNPLA3 rs738409, TM6SF2 rs58542926, and HSD17B13 rs6834314 genes was performed using allelic discrimination with TaqMan Probe Real-Time PCR Assays (ThermoFisher Scientific, Waltham, MA, USA), and fluorescent signals (FAM and VIC) were utilized for the detection according to previously described methods [19]. To ensure the accuracy of data interpretation, both positive and negative controls were included in each experiment. The allelic discrimination plot was analyzed using the QuantStudio™ 3 Real-Time PCR System (ThermoFisher Scientific, USA). 2.9. Statistical Analysis The statistical evaluation of variables was performed by using SPSS (version 22.0.0, SPSS Inc., Chicago, IL, USA) and GraphPad Prism (version 9.5.0, Boston, MA, USA). Categorical data were analyzed using the Chi-square test and one-way ANOVA. The Mann–Whitney test was applied for comparisons between non-paired groups. Hardy–Weinberg equilibrium (HWE) was determined using Pearson’s Chi-square calculated using online software. Correlations between parameters were analyzed by Spearman’s rank test. Univariable and multivariable analyses were performed on SPSS using binary logistic regression to determine the parameters associated with F2–F4 fibrosis. A p-value of less than 0.05 was considered as statistically significant.',\n",
       " '11206853': '2. Materials and Methods 2.1. Volunteer Recruitment The inclusion criteria for the study of elderly volunteers (aged 60 years or older) were as follows: participants were willing to cooperate with the study and provide stool samples. Volunteers were excluded if they had consumed probiotic or prebiotic products within the three months prior to sample collection. Individuals with unhealthy lifestyle habits, such as smoking and excessive alcohol consumption within the past three years, or those with severe drug dependencies, were not eligible. Additionally, individuals with major health issues, such as liver disease, acute or chronic pancreatitis, infectious diseases, or a history of major gastrointestinal surgeries, were also excluded from the study. Twenty-five elderly volunteers were recruited in the Hebei region, we collected the fecal samples from them, and a follow-up questionnaire was completed at the time of sample collection. The questionnaire included information on participants’ weight, sleep patterns, exercise habits, dietary intake, and medication use. No dietary or lifestyle interventions were made during the course of the study. 2.2. Fecal Material Fecal samples collected from the volunteers were delivered to the laboratory within 2 h at 4 °C for tube splitting, a portion of the sample was taken for TMA detection immediately after being thoroughly homogenized, and the remaining portion was frozen at −80 °C for subsequent 16S rRNA sequencing. No toilets were used in the collection of the feces to prevent contamination of the samples. The detailed information of the elderly sample is as follows (Table 1). 2.3. Extraction and Detection of TMA Fecal samples collected from the volunteers were delivered to the laboratory at 4 °C and the pre-treatment was completed within 1 h. TMA was extracted by the following procedure: Homogenize the fecal sample, then take 1 g of the homogenized feces and add 4 mL of 0.06 mol/L sulfuric acid. Mix the sample well by shaking, then centrifuge it at 13,000× g for 10 min and aspirate the supernatant. Repeat this process twice more for the TMA in the feces’ residue and combine the supernatants from the three extractions into a single container. Mix the combined supernatants thoroughly. Add 5 mL of the supernatant to each of the two headspace vials (which are placed on ice), and add 5 mL of 5 mol/L, 20 mol/L sodium hydroxide solutions to the respective vials, then tightly close the vial caps. Finally, place the samples in the HS-GC instrument for detection. The standard solutions were prepared as follows: Weigh 0.09 g of trimethylamine hydrochloride accurately and dissolve it in 0.06 mol/L sulfuric acid solution, then make up the volume to 80 mL to prepare a 696 mg/L TMA standard stock solution. Dilute the trimethylamine standard stock solution with 0.06 mol/L sulfuric acid to obtain a trimethylamine standard reserve solution with a concentration of 69.6 mg/L. Further dilute the trimethylamine standard reserve solution with 0.06 mol/L sulfuric acid to prepare a trimethylamine standard-use solution with a concentration of 6.96 mg/L, 3.48 mg/L, 1.74 mg/L, 0.87 mg/L, 0.435 mg/L, and 0.174 mg/L. For the headspace instrument, the temperature of the heating chamber was 80 °C, the temperature of the quantification loop was 90 °C, the temperature of the transfer line was 110 °C, and the equilibration time was 40 min. Agilent 8890 gas chromatograph was used for the analysis of TMA. A 30 m × 0.32 mm CP-Volamine column with 5 μm thickness (Agilent, Santa Clara, CA, USA) was fitted to GC to separate TMA. The injection temperature was 200 °C in the splitting mode, and the shunt ratio was 10:1. The flow rate of the high-purity nitrogen carrier gas was 1.5 mL/min. The oven temperature was programmed at successive temperature and time points as follows: 50 °C for 6 min, increased to 240 °C at a rate of 3 °C/min, and held at 240 °C for 4 min. The temperature of the detector was 250 °C. The tailing blowing gas (nitrogen) flow rate was 30 mL/min, the hydrogen flow rate was 30 mL/min, and the air flow rate was 300 mL/min. HS-GC injection volume was 1 mL; all the data were calculated automatically by GC8890 chromatography workstation software (Agilent OpenLab version 3.5). 2.4. 16S rRNA Sequencing and Bioinformatic Analysis The genomic DNA of each sample was extracted, and the DNA quality was detected on 1% agarose gels. Variable regions V3-V4 of bacterial 16S rRNA gene were amplified with degenerate PCR primers, F338 (5′-ACTCCTACGGAGGGGGGGCAG-3′) and R806 (5′-GGACTACHVGGGTWTCTAAT-3′). PCR was performed by taking 30 ng of DNA samples of acceptable quality and the corresponding primers. The PCR amplification products were purified using Agencourt AMPure XP beads and dissolved in elution buffer. Fragment ranges and concentrations of the libraries were measured using an Agilent 2100 Bioanalyzer (Agilent, Santa Clara, CA, USA), and sequenced only if they met the qualification criteria. To generate Operational Taxonomic Unit (OTU) sequences, duplicated sequences were removed using QIIME2 software (version 2019.4), and clustering was performed at 100% similarity using the DADA2 method. The resulting unique sequences were collectively identified as OTUs. The relative abundance of each species was calculated using OTUs, describing the gut microbiota composition of the elderly population at the phylum level, genus level, and enterotype level. Genera with an average relative abundance greater than 0.5% in the gut microbiota OTU results were selected as the dominant genera to explore widely distributed gut bacteria in the elderly population. α-diversity and β-diversity analyses were performed, along with species-level differences using the linear discriminant analysis effect size (LEfSe) method to investigate the diversity and differences in the bacterial composition of the gut microbiota among different groups of elderly individuals. This was performed to explore the gut microbiota diversity and differences in bacterial composition among various elderly populations. 2.5. Fecal Sample DNA Extraction and qPCR Extraction of fecal flora DNA: The collected feces were homogenized and mixed, 0.2 g was taken into a 1.5 mL sterile centrifuge tube, and the DNA in the feces was extracted with the TIANamp Stool DNA Kit (TIANGEN BIOTECH, Beijing, China), and the extracted DNA was placed in the refrigerator at −20 °C for storage. The primer sequences were utilized for the quantitative detection of 4 characterized enterobacteria species that produce TMA, including Klebsiella pneumoniae, Clostridium sporogenes, Escherichia fergusonii, and Desulfovibrio desulfuricans (Table 2). The plasmid expression vectors were constructed, and the copy number conversion was performed to establish the real-time fluorescence quantitative PCR standard curves for these four enterobacteria. An ordinary PCR was carried out using a 12 µL reaction system for PCR amplification, which consisted of 6 µL of 2 × ES Taq MasterMix, 0.35 µL each of the upstream and downstream primers (5 µmol/L), 2 µL of DNA template, and deionized water to a final volume of 12 µL. The PCR conditions were as follows: an initial denaturation step at 95 °C for 3 min, followed by 40 cycles of denaturation at 95 °C for 30 s, annealing at 58 °C for 20 s, and extension at 72 °C for 30 s. After the cycling process, a final extension step was performed at 72 °C for 5 min. Subsequently, 10 µL of the PCR products was taken and subjected to electrophoresis in 2% agarose gel and 1× TAE buffer for 25 min (120 V) to identify and visualize the PCR products. The plasmid concentration was formulaically converted to copy number according to the following formula:DNA (copy)=6.02×10^23 (copy/mol)×DNA amount (g)DNA length (bp)×660 g/bp The qPCR was performed using a 20 μL reaction system, which consisted of 2 μL of sample DNA, 0.6 μL of each of the upstream and downstream primers (5 mol/L), and 10 μL of 2 × SYBR Green qPCR Master Mix. The system was then topped up to 20 μL with deionized water. The qPCR procedure is outlined in Table 3. 2.6. Statistical Analysis Statistical analysis and graph creation of the data were conducted using GraphPad. Data that did not follow a normal distribution were analyzed using non-parametric tests, while data that followed a normal distribution were analyzed using parametric tests, and p < 0.05 was considered significant. In addition, the Wilcoxon Test (for 2 groups) or the Kruskal–Wallis Test (for >2 groups) was utilized to analyze differences in microbial diversity among distinct groups. The Chao1 index and Shannon index were used to illustrate the α-diversity differences between heathy elderly and elderly with CVD, and differences in microbial composition between heathy elderly and elderly with CVD (β-diversity) were described using partial least squares discriminant analysis (PLS-DA) based on Unifrac distances. The top 10 species with the highest abundance in the intestinal flora were selected and a group-wise key species difference analysis was performed to further explore whether there were significant differences in the gut microbiota of elderly individuals in different groups at various taxonomic levels. And then the linear discriminant analysis effect size (LEfSe) was used for identifying the dominant bacteria in the gut microbiota of elderly individuals in different groups. The R packages (RStudio version 3.4.1) ggally package(version 2.2.1), corrplot package(version 0.92), and GraphPad Prism (version 9.5) were used to calculate and display the indicators.',\n",
       " '11206826': '2. Materials and Methods 2.1. Study Design A cross-sectional study was conducted to collect information on the eating habits and nutritional quality of mid-afternoon snack of families with schooled children aged 3 to 12 years from schools in three areas of Catalonia (Barcelona, Lleida, and Girona) using an online survey designed ad hoc. In order to evaluate the survey’s validity of content, the initial proposal was discussed with a group of 12 experts made up of 80% doctors (Ph.D.); however, all of them specialized in nutrition. These experts assessed the relevance and appropriateness of the questions, as described elsewhere [25]. Then, the process of administering the survey to a small sample of known families was tested. Participation was entirely voluntary; participants were provided with an information document about the research and an informed consent form in order to formalize their willingness to participate. Personal data were not used to identify the participants, but they were assigned an alphanumeric identification code in order to guarantee their anonymity throughout the study. The study was approved by the Ethics Committee of the Universitat Oberta de Catalunya (CE21PR04) to ensure compliance with the ethical principles relating to research involving human subjects and the processing of personal data set out in the UNESCO Universal Declaration of Bioethics and Human Rights [26], the Declaration of Helsinki [27], and the Belmont Report [28]. 2.2. Study Population The target population were parents or guardians of schooled children aged 3 to 12 years who voluntarily agreed to complete the survey on eating habits and the nutritional quality of children’s mid-afternoon snack. The selection criteria were families with children aged 3 to 12 years, living and attending school in Barcelona, Girona, or Lleida, whose parents or guardians voluntarily agreed to the study, read and signed the informed consent form, and filled in all mandatory fields of the online form. Exclusion criteria included non-completion of the informed consent form, children outside the age range of 3 to 12 years of age, or failure to provide all the mandatory information on the form. Also excluded were those who claimed to suffer from a disease that required a special diet. Figure 1 represents the recruitment process for participants in three areas of Catalonia: Barcelona, Girona, and Lleida. Initially, we recruited 214 participants from Barcelona, 450 from Girona, and 118 from Lleida. After applying inclusion and exclusion criteria, 18 participants were excluded, leaving us with a total of 764 participants. The sample was not representative of children attending school in Barcelona, Girona, or Lleida, as the selection of participating schools and participants was conducted using the convenience sampling method. 2.3. Data Collection and Study Variables This study pooled the data of three independent samples from schools in Catalonia (Spain), specifically in the cities of Barcelona, Girona, and Lleida, collected between March and April 2018 and 2022. The ad hoc questionnaire included an informed consent form and questions collecting information on demographics and mid-afternoon snacking habits during the school days and weekend, as well as a record of mid-afternoon snack intake over three non-consecutive school days. The form also included instructions about how the participants must include the information in the record. The variable age was categorized into three strata (3–5, 6–8, 9–12 years). In order to study the nutritional characteristics of mid-afternoon snacks, a review of the existing scientific evidence and official public health nutrition recommendations on mid-afternoon snacks was initially carried out. In addition, an electronic search using four major health sciences academic databases (Google Academic [Google LLC, Mountain View, California, United States], PubMed® [National Center for Biotechnology Information at the U.S. National Library of Medicine, Bethesda, Maryland, United States], Scopus [Elsevier, Amsterdam, Netherlands], Cochrane [Cochrane Collaboration, London, United Kingdom], and Web of Science [Clarivate Analytics, London, United Kingdom]) was conducted by combining different keywords (mid-afternoon snack, dietary patterns, children, childhood, toddlers). The search included literature published from 2005 to 2021, and only articles in English and Spanish were selected. Specifically, the criteria for assessing the quality of snacks were drawn from reference documents such as ASPCAT’s guidelines for healthy breakfasts and mid-afternoon snacks [20], the healthy eating guide for families of schooled children [29], and various international guides published by official organizations [21,22,23,24]. These sources provide recommendations on the composition of a healthy snack. We defined four categories (healthy, quite healthy, quite unhealthy, and unhealthy) based on the review of the existing scientific evidence on mid-afternoon snacks. This four-category classification (Figure 2) was used to analyze the nutritional quality of each mid-afternoon snack. To analyze the nutritional quality of the mid-afternoon snacks, the study included two distinct analyses. First, the nutritional quality of all the mid-afternoon snacks was assessed. Second, the nutritional quality of the mid-afternoon snacks was evaluated on a per-participant basis. Initially, we analyzed the composition of each mid-afternoon snack recorded by the participants who confirmed that they had had a snack after school days. Of those that included a sandwich, the composition of the sandwich was analyzed according to the type of filling, but it was not possible to discriminate whether the bread was made with wholemeal or refined flour. The beverage choice of participants who reported consuming a drink with their snack was also documented. Next, based on the analysis of the composition of each mid-afternoon snack recorded by the participants, we categorized the nutritional quality of the snacks into four categories (Figure 2). To calculate the average nutritional quality of the mid-afternoon snack per participant, the values of the three mid-afternoon snacks recorded for each participant were summed and then averaged based on the categories outlined in Figure 2. A healthy mid-afternoon snack model adapted to the habits, practices, and customs of the Spanish population was developed by reviewing available and updated dietary recommendations that included aspects of snacking. The design of the healthy mid-afternoon snack model was made with Adobe Illustrator® software (version 24.3). 2.4. Analysis The data were entered into a database and analyzed using SPSS® version 22.0 statistical software. A descriptive analysis of the variables under study was performed. The Kolmogorov–Smirnov test was used to analyze whether the variables followed a normal distribution. Categorical variables are expressed as absolute numbers and percentages, and continuous variables as means and standard deviation. An analysis of the association of categorical variables was carried out using Pearson’s Chi-square test. In the case of the association between a categorical variable and a continuous variable, Student’s t-test was used. p-values of less than 0.05 were considered statistically significant.',\n",
       " '11206810': '4. Materials and Methods 4.1. Materials Five-week-old male specific-pathogen-free grade Sprague–Dawley rats were purchased from the Laboratory Animal Center of Anhui Medical University (Hefei, China) with the production license number SCXK(wan)2017—001. D. huoshanense Fengdou was purchased from Lu’an Lingshang Ecological Agriculture Co., Ltd. (Huoshan County, Anhui Province, China). The original wild provenance of D. huoshanense came from the protection base of D. huoshanense provenance in Taipingfan Township, Huoshan County, Anhui province, China. The D. huoshanense used in this study was obtained from the wild seedlings of the provenance protection base through tissue culture and then by wild imitation cultivation. From the purchased D. huoshanense Fengdou samples, specimens were prepared (20211104) for future reference and were stored at the specimens center of Chinese medicine, College of Biological and Pharmaceutical Engineering, West Anhui University, China. Furthermore, D. huoshanense Fengdou samples (voucher no: 20211110) were authenticated by Professor Naifu Chen, College of Biological and Pharmaceutical Engineering, West Anhui University, Lu’an, China, and were used in this study. The plant name has been checked with “World Flora Online” (www.worldfloraonline.org, accessed on 10 November 2021) with the access site https://www.worldfloraonline.org/taxon/wfo-0000939417 (accessed on 10 November 2021). Streptozotocin was purchased from Sigma-Aldrich (TEMED; St. Louis, MO, USA). Metformin hydrochloride (98%) was purchased from Hefei Jinu Biotechnology Co., Ltd. (Hefei, China). Vitamin K1 was purchased from Shanghai Macklin Biochemical Technology Co., Ltd. (Shanghai, China). D-cysteine hydrochloride monohydrate, hemin chloride, and bovine bile salt were purchased from Dalian Meilun Biotechnology Co., Ltd. (Dalian, China). Yeast powder and Tween 80 were purchased from Shanghai Aladdin Biochemical Technology Co., Ltd. (Shanghai, China). Resazurin was purchased from Shenzhen Ruji Biotechnology Co., Ltd. (Shenzhen, China). Peptone was purchased from Beijing Sulaibao Technology Co., Ltd. (Beijing, China). Medical blood glucose test paper was purchased from Sinocare Biosensor Co., Ltd. (Changsha, China). Ultra-pure water was in-house prepared using a Milli-Q water purification system (Millipore, Bedford, MA, USA). Methanol, acetonitrile, formic acid, and isopropyl alcohol were LC-MS-grade and purchased from ANPEL Laboratory Technologies (Shanghai) Inc. (Shanghai, China). All other chemical reagents used in this work were purchased from Sinopharm Chemical Reagent Co., Ltd. (Shanghai, China) and were analytical-grade. 4.2. Methods 4.2.1. Preparation of DHWE Stems of D. huoshanense were dried at 65 °C until a constant weight was achieved, and then crushed and sifted over a 60-mesh sieve screen. Twenty grams of D. huoshanense powder were extracted separately in 800 mL distilled water three times. The first extraction temperature was 99 °C, and the other two extraction temperatures were both 95 °C, and each extraction time was 2 h. The three extracts were combined, concentrated with a rotatory evaporator (FE-52AA; Shanghai Yarong Biochemical Instrument Factory, Shanghai, China) at 80 °C and then lyophilized with a vacuum freeze dryer (FD-1A-50; Beijing Boyikang Instrument Co., Ltd., Beijing, China) to obtain DHWE. 4.2.2. Preparation of DHP DHP was prepared in accordance with methods as previously described [35], with minor modifications. Twenty grams of D. huoshanense powder were extracted separately in 800 mL of distilled water three times at 85 °C for 2 h. The extracts were combined, centrifuged at 4800× g for 10 min, and the supernatant was concentrated with a rotatory evaporator (FE-52AA; Shanghai Yarong Biochemical Instrument Factory) at 80 °C until the supernatant became a little sticky. DHP was precipitated by adding four-fold volumes of ethanol (100%, v/v) and it was kept at 4 °C overnight. After centrifugation at 4800× g for 15 min, the precipitate was air-dried and then dissolved in distilled water to prepare a 2% solution. The solution was heated in boiling water for 7 min, and then it was incubated at 40 °C for 10 min after cooling. According to the method as previously described [36], with minor modifications, porcine pancreatic amylase (final concentration: 15 IU/mL) was added to the crude polysaccharide solution and hydrolyzed at 40 °C for 90 min. The amylase was then inactivated by heating the solution in a boiling water bath for 5 min. The solution was then deproteinized in accordance with the Sevag method [37] five times. The deproteinized polysaccharide solution was concentrated, dialyzed with a membrane with a molecular weight cut-off of 3000 Da, and then lyophilized to obtain refined DHPs. 4.2.3. Preparation of NDHP NDHP was prepared by collecting the supernatant when ethanol was used to precipitate polysaccharide during the preparation of DHP, and it was concentrated to an appropriate concentration and lyophilized as mentioned above. 4.2.4. Determination of Carbohydrates, Total Polyphenols, and Protein Content in DHWE, DHP, and NDHP The carbohydrates, total polyphenol, and protein content were determined by the phenol–sulfuric acid method [38], Folin–Ciocalteu method [39], and Coomassie brilliant blue method [40], respectively. 4.2.5. Determination of the Homogeneity and Molecular Weight of DHP The DHP samples were dissolved in a 0.1 M NaNO3 aqueous solution containing 0.02% NaN3 at the concentration of 1 mg/mL and filtered through a filter of 0.45 μm pore size. The homogeneity and molecular weight of DHP were measured using SEC-MALLS-RI (Size Exclusion Chromatography coupled with Multi-Angle Light Scattering with differential Refractive Index detector). The weight average molecular weight and number average molecular weight (Mw and Mn) and polydispersity index (Mw/Mn) of DHP in the 0.1 M NaNO3 aqueous solution containing 0.02% NaN3 were measured on a DAWN HELEOS-II laser photometer (Wyatt Technology Co., Goleta, CA, USA), equipped with two tandem columns (300 × 8 mm, Shodex OH-pak SB-805 and 803; Showa Denko K.K., Tokyo, Japan), which was held at 45 °C using a model column heater by Sanshu Biotech. Co., Ltd. (Shanghai, China). The flow rate was 0.6 mL/min. A differential refractive index detector (Optilab T-rEX, Wyatt Technology Co., USA) was simultaneously connected to give the concentration of fractions and the dn/dc value (the ratio of the change in the refractive index (dn) of a solution with respect to a change in its solute concentration (dc)). The dn/dc value of the fractions in the 0.1 M NaNO3 aqueous solution containing 0.02% NaN3 was determined to be 0.141 mL/g. Data were acquired and processed using ASTRA6.1 (Wyatt Technology). Quantified data were output into Excel format. 4.2.6. Determination of the Structure of Characteristic Functional Groups of DHP To know the structure of the functional groups of DHP, Fourier-transform infrared (FT-691 IR) spectroscopy (Thermo Fisher Scientific, Waltham, MA, USA) was used for analysis. Pure KBr powder slides were used to adjust the baseline. DHP was mixed with appropriate amounts of KBr powder, ground evenly using an agate mortar and pestle for making a very thin layer, and then subjected to infrared scanning at 4 Hz and 32 over a wavelength range of 400–4000 cm−1. 4.2.7. Analysis of the Components of DHWE and NDHP by UPLC-Orbitrap-MS (Ultra-Performance Liquid Chromatography–Orbitrap–Mass Spectrometry) DHWE or NDHP samples (50–100 mg) were accurately weighed, mixed with a 1 mL extraction solution (water–acetonitrile–isopropyl alcohol mixed solution, 1:1:1, v/v/v) in a centrifuge tube, homogenized for 60 s, then ultrasonic extraction at low-temperature for 30 min, centrifuged at 12,000 rpm for 10 min at 4 °C, allowed to settle for 1 h to precipitate protein at −20 °C, and then centrifuged at 12,000 rpm for 10 min at 4 °C. The supernatant solution was dried using a vacuum desiccator, mixed with a 0.2 mL 50% acetonitrile solution, and homogenized and centrifuged at 14,000 rpm for 15 min at 4 °C, and then the supernatant was used for component detection. The extracted samples were analyzed using an UPLC–Orbitrap–MS system (UPLC, Vanquish; MS, HFX). The analytical conditions were as follows: UPLC—column, Waters HSS T3 (100 × 2.1 mm, 1.8 μm); column temperature, 40 °C; flow rate, 0.3 mL/min; injection volume, 2 μL; solvent system—phase A were Milli-Q water (0.1% formic acid), and phase B were acetonitrile (0.1% formic acid); gradient program, 0 min phase A/phase B (100:0, v/v), 1 min phase A/phase B (100:0, v/v), 12 min phase A/phase B (5:95, v/v), 13 min phase A/phase B (5:95, v/v), 13.1 min phase A/phase B (100:0, v/v), and 17 min phase A/phase B (100:0, v/v). The data generated by the high-resolution mass spectrometer (HRMS) were recorded on a Q Exactive HFX Hybrid Quadrupole Orbitrap mass spectrometer equipped with a heated ESI source (Thermo Fisher Scientific) utilizing the Full-ms-ddMS2 acquisition methods. The ESI source parameters were set as follows: sheath gas pressure, 40 arb; aux gas pressure, 10 arb; spray voltage, +3000 v/−2800 v; temperature, 350 °C; and ion transport tube temperature, 320 °C. The scanning range of the primary mass spectrometry was (scan m/z range) 70–1050 Da, with a primary resolution of 70,000 and secondary resolution of 17,500. The raw data were first preprocessed using Progenesis QI (Waters Corporation, Milford, CT, USA) software (https://www.waters.com/nextgen/ch/de/products/informatics-and-software/mass-spectrometry-software/progenesis-qi-software/progenesis-qi.html, accessed on 10 November 2021) for baseline filtering, peak recognition, peak matching, retention time correction, peak alignment, etc., to obtain a data matrix containing retention time, mass-to-charge ratio, and peak intensity. Peaks in the secondary mass spectrometry data were identified using the self-built Chinese herbal secondary mass spectrometry database and corresponding fragmentation patterns of San Shu Biotechnology Inc. (Shanghai, China). The matching situation of secondary mass spectrometry (MS2) is mainly reflected in the score of secondary fragments, which has a total score of 1. The higher the score, the more reliable the identification result. It is generally believed that if it is greater than 0.7, the identification result is reliable. 4.2.8. Preparation of Metformin Solution Metformin (1 g) was dissolved in 100 mL of distilled water to prepare a 10 mg/mL solution and stored in 4 °C until later use within 24 h. 4.2.9. Preparation of Rats with T1D The rats were raised in an air-conditioned room with a temperature of 21 °C–23 °C, humidity of 60–70%, and natural light (about 11 h light and 13 h dark per day) for 1 week to achieve familiarity with the environment. The rats had free access to food and drinking water. According to a lot-drawing method, 16 rats (weight = 200 ± 20 g) were randomly assigned to the model preparation group and the control group, with 8 rats in each group. In the T1D model group, T1D was induced by intraperitoneal injection of streptozotocin prepared in a 0.1 M sodium citrate buffer (pH 4.5) at a dose of 60 mg/kg/body weight after fasting for 12 h [41]. Rats in the control group were administered an equal volume of sodium citrate buffer by intraperitoneal injection. Blood was collected from the tail vein (caudal vein) 72 h later to measure blood glucose using a blood glucose meter (Sinocare Biosensor Co., Ltd., Changsha, China). Rats with fasting blood glucose concentrations > 16.7 mmol/L and exhibiting symptoms of polydipsia and polyuria were considered a successful T1D model. 4.2.10. Preparation of Medium for In Vitro Fermentation Five different media were prepared for in vitro fermentation, and these included a basal nutrient medium (BASE group), a DHWE + basal nutrient medium (DHWE group), a DHP + basal nutrient medium (DHP group), an NDHP + basal nutrient medium (NDHP group), and a metformin + basal nutrient medium (MET group). The basal nutrient medium for in vitro fermentation was prepared as previously described [42,43], with minor modifications. The BASE medium contained 2.0 g/L peptone, 2.0 g/L yeast powder, 0.1 g/L sodium chloride, 0.04 g/L dipotassium hydrogen phosphate, 0.04 g/L potassium dihydrogen phosphate, 0.01 g/L magnesium sulfate heptahydrate, 0.01 g/L calcium chloride hexahydrate, 2.0 g/L sodium bicarbonate, 0.02 g/L heme chloride, 0.5 g/mL cysteine hydrochloride, 0.5 g/L bovine bile salt, 8 mL/L 0.25% (m/v) resazurin, 2.0 mL/L Tween 80 (the weighed Tween 80 was slowly added to an appropriate amount of distilled water according to the required concentration and stirred thoroughly until Tween 80 was completely dissolved; then it was added to the culture medium and stirred thoroughly to evenly disperse Tween 80 in the culture medium), and 10 μL/L vitamin K1 (pH: 7.0 ± 0.1). The fermentation media of the other groups had appropriate amounts of DHWE (100 mg/9 mL), DHP (100 mg/9 mL), NDHP (50 mg/9 mL), or metformin (50 mg/9 mL) added, and then, pH was adjusted to 7.0 ± 0.1. All fermentation media were autoclaved at 105 °C for 20 min, and autoclaving was repeated the next morning. All the media were confirmed as sterile before use. 4.2.11. Preparation of a Fecal Slurry for Fermentation Four weeks after establishing the T1D model, four rats with T1D and a mean blood glucose concentration of 28.15 ± 1.85 mmol/L were anesthetized by intraperitoneal injection of pentobarbital sodium (200 mg/kg) and immediately sacrificed. The same amount of feces from the colons of four rats were collected, mixed, and dispersed with sterile normal saline to obtain a 10% (w/v) fecal suspension. The fecal suspension was filtered with two layers of gauze to obtain a fecal slurry. Then, 2 mL of the fecal slurry was added to 18 mL of BASE containing 200 mg of DHWE (DHWE group, 1% DHWE), 200 mg DHP (DHP group, 1% DHP), 100 mg NDHP (NDHP group, 0.5% NDHP), 100 mg metformin (MET group, 0.5% metformin), or none (BASE group). For the control, 2 mL of the fecal slurry was separately added to 18 mL of BASE before fermentation. The fermentation process was carried out in an anaerobic incubator (AL-B; Dalian Suyoubo Biotechnology Co., Ltd., Dalian, China). All groups were incubated at 37 °C, and the partial pressure of oxygen was <0.1%. The conical flasks containing the fermentation broth were gently shaken every 6 h during fermentation. We extracted 5 mL of fermentation samples at 0, 12, and 24 h after the start of fermentation and rapidly cooled them in an ice bath for 20 min to stop fermentation. The samples were then stored at −80 °C for future analysis. All fermentations in each group were carried out in triplicate. 4.2.12. Determination of the pH and Bacterial Count in the Fermentation Broth After thawing samples stored at −80 °C, the pH of the fermentation broth was measured with a pH meter (PHS-3E; Shanghai Yidian Scientific Instrument Co., Ltd., Shanghai, China), and the absorbance value of the fermentation broth at 600 nm was measured with an ultraviolet visible spectrophotometer (TU-1901; Beijing Purkinje General Instrument Co., Ltd., Beijing, China) to represent the number of bacteria in the fermentation broth. It has been reported that the pH of the in vitro fermentation medium will change after autoclaving [43]. Therefore, to avoid the interference of the difference in initial pH and bacteria number in each group, the change in pH (∆pH) and optical density 600 value (∆OD600) were calculated by the difference between the corresponding value at 0 and 24 h after the start of fermentation. 4.2.13. Determination of Short-Chain Fatty Acid Content in Fermentation Broth The short-chain fatty acid (SCFA) content was analyzed by gas chromatography–mass spectrometry. We prepared a mixed SCFA standard solution with 2-ethylbutyric acid as an internal standard, using 0.2 M hydrochloric acid as a solvent. The concentration of 2-ethylbutyric acid, acetic acid, and other fatty acids (propionic acid, butyric acid, isobutyric acid, valeric acid, and isovaleric acid) in the mixed SCFA standard solution was 150, 750, and 150 μg/mL, respectively. The SCFAs were mixed well with magnetic stirring, then filtered through a 0.45 μm filter membrane before use. Fermentation broth preserved at −80 °C was thawed at 4 °C. We mixed 1.0 mL of a 2-ethylbutyric acid internal standard solution at a concentration of 300 µg/mL using 0.2 M HCl as a solvent with 1.0 mL of the fermentation broth and vortexed and centrifuged it at 12,000 rpm at 4 °C for 10 min. The supernatant was collected and filtered through a 0.45 µm filter and then used for analysis with triple quadrupole gas chromatography–mass spectrometry (Agilent 7890B-5977B; Agilent Technology Company, Santa Clara, CA, USA). The chromatographic conditions were as follows. A highly polar capillary chromatographic column (Wondacap wax, 30 m × 250 µm × 0.25 µm; Shimadzu Company, Kyoto, Japan) was used. The carrier gas was high-purity helium (purity up to 99.996%). The temperature was kept at 90 °C for 1 min, raised to 250 °C at 5 °C/min, and maintained at 200 °C for 4 min. The injection volume was 1 µL, and the split ratio was 1:30. The flow rate was 1 mL/min. Mass spectrum conditions were as follows: EI electron source; electronic energy, 70 EV; ion source temperature, 230 °C; and four-stage rod temperature, 150 °C. Data acquisition was performed by scanning. The content of each SCFA in the fermentation broth was calculated according to the following formula. K=(Ws/As)(Wi/Ai); Wx=K(Ax×Wi)Ai Ws is the content of an SCFA in the mixed SCFA standard solution; As is the peak area of an SCFA in the mixed SCFAs in the chromatogram; Wx is the content of a short-chain fatty acid in the sample (mg); Wi is the content of the internal standard added to the sample or in the mixed SCFA solution (mg); Ax is the peak area of a short-chain fatty acid in the chromatogram; Ai is the peak area of the internal standard in the chromatogram; and K is the relative correction factor. 4.2.14. Bioinformatics Analysis of High-Throughput Sequencing Data from Fecal Microbiota We sent 2 mL of frozen pre-fermentation control and 24 h fermentation broth samples from each group to Shenzhen Weikemeng Technology Group Co., Ltd. (Shenzhen, China) for high-throughput sequencing library construction and Illumina MiSeq sequencing. Total genomic DNA was extracted from each sample by using the cetyl trimethyl ammonium bromide method. The V3–V4 region of 16S rRNA of each sample was chosen for amplification and sequenced. Bioinformatics analysis mainly included the Wayne diagram, α-diversity index (including observed operational taxonomic units [OTUs], the Chao1 index, the Simpson index, and the Shannon index), linear discriminant analysis, linear discriminant analysis effect size (LEfSe), and β-diversity index (principal coordinate analysis). The R language Pheatmap package was mainly used to draw the correlation heat map between fecal flora and the pH of 24 h fermentation and ΔOD600. Furthermore, redundancy analysis (RDA) was carried out with the Vegan package of R language and visualized with ggplot2. 4.2.15. Statistical Analysis The relative abundance of bacteria at the phylum and genus level, pH at different times, ΔpH, ΔOD600, and SCFAs content data were expressed as mean ± standard deviation and analyzed by one-way analysis of variance with the DPS data processing system (Hangzhou Ruifeng Technology Co., Ltd., Hangzhou, China), and multiple comparisons were made with Duncan’s test. A p value < 0.05 indicated a significant difference. 4.2. Methods 4.2.1. Preparation of DHWE Stems of D. huoshanense were dried at 65 °C until a constant weight was achieved, and then crushed and sifted over a 60-mesh sieve screen. Twenty grams of D. huoshanense powder were extracted separately in 800 mL distilled water three times. The first extraction temperature was 99 °C, and the other two extraction temperatures were both 95 °C, and each extraction time was 2 h. The three extracts were combined, concentrated with a rotatory evaporator (FE-52AA; Shanghai Yarong Biochemical Instrument Factory, Shanghai, China) at 80 °C and then lyophilized with a vacuum freeze dryer (FD-1A-50; Beijing Boyikang Instrument Co., Ltd., Beijing, China) to obtain DHWE. 4.2.2. Preparation of DHP DHP was prepared in accordance with methods as previously described [35], with minor modifications. Twenty grams of D. huoshanense powder were extracted separately in 800 mL of distilled water three times at 85 °C for 2 h. The extracts were combined, centrifuged at 4800× g for 10 min, and the supernatant was concentrated with a rotatory evaporator (FE-52AA; Shanghai Yarong Biochemical Instrument Factory) at 80 °C until the supernatant became a little sticky. DHP was precipitated by adding four-fold volumes of ethanol (100%, v/v) and it was kept at 4 °C overnight. After centrifugation at 4800× g for 15 min, the precipitate was air-dried and then dissolved in distilled water to prepare a 2% solution. The solution was heated in boiling water for 7 min, and then it was incubated at 40 °C for 10 min after cooling. According to the method as previously described [36], with minor modifications, porcine pancreatic amylase (final concentration: 15 IU/mL) was added to the crude polysaccharide solution and hydrolyzed at 40 °C for 90 min. The amylase was then inactivated by heating the solution in a boiling water bath for 5 min. The solution was then deproteinized in accordance with the Sevag method [37] five times. The deproteinized polysaccharide solution was concentrated, dialyzed with a membrane with a molecular weight cut-off of 3000 Da, and then lyophilized to obtain refined DHPs. 4.2.3. Preparation of NDHP NDHP was prepared by collecting the supernatant when ethanol was used to precipitate polysaccharide during the preparation of DHP, and it was concentrated to an appropriate concentration and lyophilized as mentioned above. 4.2.4. Determination of Carbohydrates, Total Polyphenols, and Protein Content in DHWE, DHP, and NDHP The carbohydrates, total polyphenol, and protein content were determined by the phenol–sulfuric acid method [38], Folin–Ciocalteu method [39], and Coomassie brilliant blue method [40], respectively. 4.2.5. Determination of the Homogeneity and Molecular Weight of DHP The DHP samples were dissolved in a 0.1 M NaNO3 aqueous solution containing 0.02% NaN3 at the concentration of 1 mg/mL and filtered through a filter of 0.45 μm pore size. The homogeneity and molecular weight of DHP were measured using SEC-MALLS-RI (Size Exclusion Chromatography coupled with Multi-Angle Light Scattering with differential Refractive Index detector). The weight average molecular weight and number average molecular weight (Mw and Mn) and polydispersity index (Mw/Mn) of DHP in the 0.1 M NaNO3 aqueous solution containing 0.02% NaN3 were measured on a DAWN HELEOS-II laser photometer (Wyatt Technology Co., Goleta, CA, USA), equipped with two tandem columns (300 × 8 mm, Shodex OH-pak SB-805 and 803; Showa Denko K.K., Tokyo, Japan), which was held at 45 °C using a model column heater by Sanshu Biotech. Co., Ltd. (Shanghai, China). The flow rate was 0.6 mL/min. A differential refractive index detector (Optilab T-rEX, Wyatt Technology Co., USA) was simultaneously connected to give the concentration of fractions and the dn/dc value (the ratio of the change in the refractive index (dn) of a solution with respect to a change in its solute concentration (dc)). The dn/dc value of the fractions in the 0.1 M NaNO3 aqueous solution containing 0.02% NaN3 was determined to be 0.141 mL/g. Data were acquired and processed using ASTRA6.1 (Wyatt Technology). Quantified data were output into Excel format. 4.2.6. Determination of the Structure of Characteristic Functional Groups of DHP To know the structure of the functional groups of DHP, Fourier-transform infrared (FT-691 IR) spectroscopy (Thermo Fisher Scientific, Waltham, MA, USA) was used for analysis. Pure KBr powder slides were used to adjust the baseline. DHP was mixed with appropriate amounts of KBr powder, ground evenly using an agate mortar and pestle for making a very thin layer, and then subjected to infrared scanning at 4 Hz and 32 over a wavelength range of 400–4000 cm−1. 4.2.7. Analysis of the Components of DHWE and NDHP by UPLC-Orbitrap-MS (Ultra-Performance Liquid Chromatography–Orbitrap–Mass Spectrometry) DHWE or NDHP samples (50–100 mg) were accurately weighed, mixed with a 1 mL extraction solution (water–acetonitrile–isopropyl alcohol mixed solution, 1:1:1, v/v/v) in a centrifuge tube, homogenized for 60 s, then ultrasonic extraction at low-temperature for 30 min, centrifuged at 12,000 rpm for 10 min at 4 °C, allowed to settle for 1 h to precipitate protein at −20 °C, and then centrifuged at 12,000 rpm for 10 min at 4 °C. The supernatant solution was dried using a vacuum desiccator, mixed with a 0.2 mL 50% acetonitrile solution, and homogenized and centrifuged at 14,000 rpm for 15 min at 4 °C, and then the supernatant was used for component detection. The extracted samples were analyzed using an UPLC–Orbitrap–MS system (UPLC, Vanquish; MS, HFX). The analytical conditions were as follows: UPLC—column, Waters HSS T3 (100 × 2.1 mm, 1.8 μm); column temperature, 40 °C; flow rate, 0.3 mL/min; injection volume, 2 μL; solvent system—phase A were Milli-Q water (0.1% formic acid), and phase B were acetonitrile (0.1% formic acid); gradient program, 0 min phase A/phase B (100:0, v/v), 1 min phase A/phase B (100:0, v/v), 12 min phase A/phase B (5:95, v/v), 13 min phase A/phase B (5:95, v/v), 13.1 min phase A/phase B (100:0, v/v), and 17 min phase A/phase B (100:0, v/v). The data generated by the high-resolution mass spectrometer (HRMS) were recorded on a Q Exactive HFX Hybrid Quadrupole Orbitrap mass spectrometer equipped with a heated ESI source (Thermo Fisher Scientific) utilizing the Full-ms-ddMS2 acquisition methods. The ESI source parameters were set as follows: sheath gas pressure, 40 arb; aux gas pressure, 10 arb; spray voltage, +3000 v/−2800 v; temperature, 350 °C; and ion transport tube temperature, 320 °C. The scanning range of the primary mass spectrometry was (scan m/z range) 70–1050 Da, with a primary resolution of 70,000 and secondary resolution of 17,500. The raw data were first preprocessed using Progenesis QI (Waters Corporation, Milford, CT, USA) software (https://www.waters.com/nextgen/ch/de/products/informatics-and-software/mass-spectrometry-software/progenesis-qi-software/progenesis-qi.html, accessed on 10 November 2021) for baseline filtering, peak recognition, peak matching, retention time correction, peak alignment, etc., to obtain a data matrix containing retention time, mass-to-charge ratio, and peak intensity. Peaks in the secondary mass spectrometry data were identified using the self-built Chinese herbal secondary mass spectrometry database and corresponding fragmentation patterns of San Shu Biotechnology Inc. (Shanghai, China). The matching situation of secondary mass spectrometry (MS2) is mainly reflected in the score of secondary fragments, which has a total score of 1. The higher the score, the more reliable the identification result. It is generally believed that if it is greater than 0.7, the identification result is reliable. 4.2.8. Preparation of Metformin Solution Metformin (1 g) was dissolved in 100 mL of distilled water to prepare a 10 mg/mL solution and stored in 4 °C until later use within 24 h. 4.2.9. Preparation of Rats with T1D The rats were raised in an air-conditioned room with a temperature of 21 °C–23 °C, humidity of 60–70%, and natural light (about 11 h light and 13 h dark per day) for 1 week to achieve familiarity with the environment. The rats had free access to food and drinking water. According to a lot-drawing method, 16 rats (weight = 200 ± 20 g) were randomly assigned to the model preparation group and the control group, with 8 rats in each group. In the T1D model group, T1D was induced by intraperitoneal injection of streptozotocin prepared in a 0.1 M sodium citrate buffer (pH 4.5) at a dose of 60 mg/kg/body weight after fasting for 12 h [41]. Rats in the control group were administered an equal volume of sodium citrate buffer by intraperitoneal injection. Blood was collected from the tail vein (caudal vein) 72 h later to measure blood glucose using a blood glucose meter (Sinocare Biosensor Co., Ltd., Changsha, China). Rats with fasting blood glucose concentrations > 16.7 mmol/L and exhibiting symptoms of polydipsia and polyuria were considered a successful T1D model. 4.2.10. Preparation of Medium for In Vitro Fermentation Five different media were prepared for in vitro fermentation, and these included a basal nutrient medium (BASE group), a DHWE + basal nutrient medium (DHWE group), a DHP + basal nutrient medium (DHP group), an NDHP + basal nutrient medium (NDHP group), and a metformin + basal nutrient medium (MET group). The basal nutrient medium for in vitro fermentation was prepared as previously described [42,43], with minor modifications. The BASE medium contained 2.0 g/L peptone, 2.0 g/L yeast powder, 0.1 g/L sodium chloride, 0.04 g/L dipotassium hydrogen phosphate, 0.04 g/L potassium dihydrogen phosphate, 0.01 g/L magnesium sulfate heptahydrate, 0.01 g/L calcium chloride hexahydrate, 2.0 g/L sodium bicarbonate, 0.02 g/L heme chloride, 0.5 g/mL cysteine hydrochloride, 0.5 g/L bovine bile salt, 8 mL/L 0.25% (m/v) resazurin, 2.0 mL/L Tween 80 (the weighed Tween 80 was slowly added to an appropriate amount of distilled water according to the required concentration and stirred thoroughly until Tween 80 was completely dissolved; then it was added to the culture medium and stirred thoroughly to evenly disperse Tween 80 in the culture medium), and 10 μL/L vitamin K1 (pH: 7.0 ± 0.1). The fermentation media of the other groups had appropriate amounts of DHWE (100 mg/9 mL), DHP (100 mg/9 mL), NDHP (50 mg/9 mL), or metformin (50 mg/9 mL) added, and then, pH was adjusted to 7.0 ± 0.1. All fermentation media were autoclaved at 105 °C for 20 min, and autoclaving was repeated the next morning. All the media were confirmed as sterile before use. 4.2.11. Preparation of a Fecal Slurry for Fermentation Four weeks after establishing the T1D model, four rats with T1D and a mean blood glucose concentration of 28.15 ± 1.85 mmol/L were anesthetized by intraperitoneal injection of pentobarbital sodium (200 mg/kg) and immediately sacrificed. The same amount of feces from the colons of four rats were collected, mixed, and dispersed with sterile normal saline to obtain a 10% (w/v) fecal suspension. The fecal suspension was filtered with two layers of gauze to obtain a fecal slurry. Then, 2 mL of the fecal slurry was added to 18 mL of BASE containing 200 mg of DHWE (DHWE group, 1% DHWE), 200 mg DHP (DHP group, 1% DHP), 100 mg NDHP (NDHP group, 0.5% NDHP), 100 mg metformin (MET group, 0.5% metformin), or none (BASE group). For the control, 2 mL of the fecal slurry was separately added to 18 mL of BASE before fermentation. The fermentation process was carried out in an anaerobic incubator (AL-B; Dalian Suyoubo Biotechnology Co., Ltd., Dalian, China). All groups were incubated at 37 °C, and the partial pressure of oxygen was <0.1%. The conical flasks containing the fermentation broth were gently shaken every 6 h during fermentation. We extracted 5 mL of fermentation samples at 0, 12, and 24 h after the start of fermentation and rapidly cooled them in an ice bath for 20 min to stop fermentation. The samples were then stored at −80 °C for future analysis. All fermentations in each group were carried out in triplicate. 4.2.12. Determination of the pH and Bacterial Count in the Fermentation Broth After thawing samples stored at −80 °C, the pH of the fermentation broth was measured with a pH meter (PHS-3E; Shanghai Yidian Scientific Instrument Co., Ltd., Shanghai, China), and the absorbance value of the fermentation broth at 600 nm was measured with an ultraviolet visible spectrophotometer (TU-1901; Beijing Purkinje General Instrument Co., Ltd., Beijing, China) to represent the number of bacteria in the fermentation broth. It has been reported that the pH of the in vitro fermentation medium will change after autoclaving [43]. Therefore, to avoid the interference of the difference in initial pH and bacteria number in each group, the change in pH (∆pH) and optical density 600 value (∆OD600) were calculated by the difference between the corresponding value at 0 and 24 h after the start of fermentation. 4.2.13. Determination of Short-Chain Fatty Acid Content in Fermentation Broth The short-chain fatty acid (SCFA) content was analyzed by gas chromatography–mass spectrometry. We prepared a mixed SCFA standard solution with 2-ethylbutyric acid as an internal standard, using 0.2 M hydrochloric acid as a solvent. The concentration of 2-ethylbutyric acid, acetic acid, and other fatty acids (propionic acid, butyric acid, isobutyric acid, valeric acid, and isovaleric acid) in the mixed SCFA standard solution was 150, 750, and 150 μg/mL, respectively. The SCFAs were mixed well with magnetic stirring, then filtered through a 0.45 μm filter membrane before use. Fermentation broth preserved at −80 °C was thawed at 4 °C. We mixed 1.0 mL of a 2-ethylbutyric acid internal standard solution at a concentration of 300 µg/mL using 0.2 M HCl as a solvent with 1.0 mL of the fermentation broth and vortexed and centrifuged it at 12,000 rpm at 4 °C for 10 min. The supernatant was collected and filtered through a 0.45 µm filter and then used for analysis with triple quadrupole gas chromatography–mass spectrometry (Agilent 7890B-5977B; Agilent Technology Company, Santa Clara, CA, USA). The chromatographic conditions were as follows. A highly polar capillary chromatographic column (Wondacap wax, 30 m × 250 µm × 0.25 µm; Shimadzu Company, Kyoto, Japan) was used. The carrier gas was high-purity helium (purity up to 99.996%). The temperature was kept at 90 °C for 1 min, raised to 250 °C at 5 °C/min, and maintained at 200 °C for 4 min. The injection volume was 1 µL, and the split ratio was 1:30. The flow rate was 1 mL/min. Mass spectrum conditions were as follows: EI electron source; electronic energy, 70 EV; ion source temperature, 230 °C; and four-stage rod temperature, 150 °C. Data acquisition was performed by scanning. The content of each SCFA in the fermentation broth was calculated according to the following formula. K=(Ws/As)(Wi/Ai); Wx=K(Ax×Wi)Ai Ws is the content of an SCFA in the mixed SCFA standard solution; As is the peak area of an SCFA in the mixed SCFAs in the chromatogram; Wx is the content of a short-chain fatty acid in the sample (mg); Wi is the content of the internal standard added to the sample or in the mixed SCFA solution (mg); Ax is the peak area of a short-chain fatty acid in the chromatogram; Ai is the peak area of the internal standard in the chromatogram; and K is the relative correction factor. 4.2.14. Bioinformatics Analysis of High-Throughput Sequencing Data from Fecal Microbiota We sent 2 mL of frozen pre-fermentation control and 24 h fermentation broth samples from each group to Shenzhen Weikemeng Technology Group Co., Ltd. (Shenzhen, China) for high-throughput sequencing library construction and Illumina MiSeq sequencing. Total genomic DNA was extracted from each sample by using the cetyl trimethyl ammonium bromide method. The V3–V4 region of 16S rRNA of each sample was chosen for amplification and sequenced. Bioinformatics analysis mainly included the Wayne diagram, α-diversity index (including observed operational taxonomic units [OTUs], the Chao1 index, the Simpson index, and the Shannon index), linear discriminant analysis, linear discriminant analysis effect size (LEfSe), and β-diversity index (principal coordinate analysis). The R language Pheatmap package was mainly used to draw the correlation heat map between fecal flora and the pH of 24 h fermentation and ΔOD600. Furthermore, redundancy analysis (RDA) was carried out with the Vegan package of R language and visualized with ggplot2. 4.2.15. Statistical Analysis The relative abundance of bacteria at the phylum and genus level, pH at different times, ΔpH, ΔOD600, and SCFAs content data were expressed as mean ± standard deviation and analyzed by one-way analysis of variance with the DPS data processing system (Hangzhou Ruifeng Technology Co., Ltd., Hangzhou, China), and multiple comparisons were made with Duncan’s test. A p value < 0.05 indicated a significant difference.'}"
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "method_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "article_ids = [\n",
    "    11198919,\n",
    "    11198832,\n",
    "    11198784,\n",
    "    11198029,\n",
    "    11197919,\n",
    "    11197476,\n",
    "    11197378,\n",
    "    11197282,\n",
    "    11197204,\n",
    "    11197185,\n",
    "    11196966,\n",
    "    11196847,\n",
    "    11196703,\n",
    "    11196440,\n",
    "    11196417,\n",
    "    11196393,\n",
    "    11196377,\n",
    "    11196265,\n",
    "    11196252\n",
    "]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "# extract the full text version of articles using pmcids\n",
    "full_text_articles = pmc_scraper.fetch_full_text(article_ids)\n",
    "\n",
    "# extract the desired section of articles\n",
    "method_sections = pmc_scraper.extract_section(full_text_articles, 'method')\n",
    "\n",
    "# print as dictionary\n",
    "method_dict = dict(zip(article_ids, method_sections))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{11198919: 'Methods:This was a cross-sectional multicenter study enrolling children aged 0–48 months, attending pediatric clinics. Questionnaires evaluated the clinical history, symptoms, and sociodemographic information. FGIDs were defined according to Rome IV criteria. PATIENTS AND METHODS Study design We performed a prospective, cross-sectional, multicenter study to assess the hospital-based prevalence of FGIDs in five regions of Saudi Arabia (Center, North, East, West, and South) including infants and toddlers, aged 0–4 years. Recruitment was conducted in general pediatric clinics located in six cities: Jeddah (King Abdulaziz Medical City and Soliman Fakeeh Hospital), Riyadh (King Abdulaziz Medical City and King Faisal Hospital and Research Center, and King Saud University Medical City), Tabouk (Tabuk Armed Forces Hospital), Al-Madinah Al-Munawarah (Maternal Children Hospital), Khamis Mushait (Maternal Children Hospital), and Dammam (Maternal Children Hospital). We recruited the patients who visited the well baby clinic, vaccination clinic, and primary care general pediatric clinics without organic diseases. These hospitals are government hospitals except Soliman Fakeeh Hospital where local infants and toddlers visit for routine medical check and/or vaccination. Participants Saudi Arabia’s 2020 population is estimated at 34,813,871 people, and children aged 0 to 14 years represent 24.5% of the population (https://www.stats.gov.sa/en). Data were collected from July 2022 to February 2023. Infants and toddlers were recruited if consent was provided and in the absence of any organic disease with these criteria: age: 0–4 years and fulfilled symptom-based Rome IV criteria for FGIDs. Exclusion criteria were age >4 years and any infants and children who have pathological disorders (neurological, metabolic, cardiac, immunology, and surgery). Those children without FIGD who did not fulfill symptom-based Rome IV criteria for FGIDs were considered to be normal children (controls). Sample size The sample size was calculated using an estimated 10% prevalence for infant colic and childhood constipation.[13] Assuming a confidence level of 95%, power of 80%, and a precision found to be the nearest 2%, a sample size of 1200 young children aged 1 month to 4 years was considered adequate. Survey design We designed a brief, user-friendly questionnaire that assesses FGIDs in Saudi infants and toddlers. This questionnaire was pilot tested by three pediatric gastroenterologists. The questionnaire was then revised based on reproducibility, validity, and question value. An initial pool of items was evaluated for content and understandability by three pediatric gastroenterologists and three pediatric practitioners (Cronbach’s alpha = 0.8). Changes and modifications were based on the pilot results. The survey was administered via two mechanisms: direct communication or through phone call. Most of the participants were interviewed face to face using an English-language questionnaire (translated to Arabic during interviews with parents), whereas few participants completed the questionnaire by phone. The participants’ identities were not part of the research record, and no information in the questionnaire was used to ascertain identity. The estimated time to complete the survey was approximately 25 minutes. Questionnaire The questionnaire had two sections: Section 1 contains questions on demographic characteristics, including age, gender, birth weight, growth pattern, and feeding practices. Section 2 contains explicit questions on FGID symptoms, including stool pattern, spitting, and crying habits, for infants and young children based on Rome IV criteria to diagnose FGIDs as follows: functional vomiting or regurgitation, infant colic, aerophagia, infant dyschezia, functional constipation, and functional diarrhea. The definition of each functional GI disease is based on its criteria.[14] All infants and children who fulfilled one to three symptom-based criteria for FGIDs were included in the study. Those children without FIGD were considered normal children and were included as controls. Statistical analysis The associations of sociodemographic characteristics and variables of potential risk factors with FGIDs of children were examined through univariate analysis. Backward stepwise procedures were used to build the multivariate analysis; the final model included only those variables that were found to be statistically significant in the univariate analysis. The associations were expressed as odds ratios (OR) with their confidence intervals (95% CIs). The data were analyzed with the Statistical Package for the Social Sciences (SPSS) software (SPSS for Windows, version 13). Demographic data were compared using Fisher’s exact test or the Wilcoxon rank-sum test when the epidemiology of FGIDs was examined through logistic regression analysis. A 2-tailed test indicated statistical significance at P < .05. Ethics approval This study was approved by the Ethics and Research Committee of the National Guard Health Affairs, King Abdullah International Medical Research Center (NRJ21J/147/05. 16/8/2021). The parents or legal guardians signed a consent for their children to participate in the study. All information collected was kept strictly confidential.',\n",
       " 11198832: 'MethodsAfter treatment with antibiotics and probiotics, plasma NBP concentrations in SD rats were determined by high-performance liquid chromatography-tandem mass spectrometry (HPLC-MS/MS). The effect of intestinal microbiota changes on NBP pharmacokinetics was compared. Intestinal microbiota changes after NBP treatment were analyzed by 16S rRNA sequencing. Expressions of CYP3A1 mRNA and protein in the liver and small intestine tissues under different intestinal flora conditions were determined by qRT-PCR and Western Blot. KEGG analysis was used to analyze the effect of intestinal microbiota changes on metabolic pathways. Materials and methods Instruments The following instruments were used: high-performance liquid chromatography (HPLC) (Shimadzu, LC-20AD), SCIEX mass spectrometer (MS) (AB SCIEX, API 4000+), PCR amplification instrument (ABI, 2720), enzyme labeling instrument (BioTek, FLX800T, electrophoresis apparatus (Beijing Liuyi Instrument Factory, DYY-6C), a gel imaging system (Beijing Bijing Biotechnology Co., LTD., BG-gdsAUTO130), Nanodrop UV quantitative system (Thermo Fisher Scientific, NC2000), Sequencer (Illumina, Novaseq6000), and fluorescence quantitative PCR instrument (Bio-rad Corporation, CFX). Drugs and reagents The following were used: vancomycin hydrochloride for injection (VIANEX S.A.), bifidobacterium quadruplex viable tablets (Hangzhou Yuanda Biological Pharmaceutical), NBP capsules (Shiyao Group Enbipu Pharmaceutical), NBP reference (Shiyao Group Enbipu Pharmaceutical.), glipizide reference (Sichuan Vicchi Biochemical Technology), Soil DNA Kit (Omega Bio-Tek), Quant-iT PicoGreen dsDNA Assay Kit (ABI), First Strand cDNA Synthesis Kit (Servicebio), and SYBR Green qPCR Master Mix (Servicebio). Vancomycin is not absorbed during oral administration and can maintain a large concentration in the intestine, the oral administration of vancomycin was selected in the study of SD rats [18]. The probiotics selected in this study are bifidobacteria tetrad, including Bifidobacteria infantilus, Lactobacillus acidophilus, Enterococcus faecalis and Bacillus cereus. The first three are normal bacteria in healthy gut, and Bacillus cereus can consume oxygen molecules in intestine to create an anaerobic environment and enhance therapeutic effect. The selection of such probiotics had little effect on microflora structure [19]. Animals Male Sprague Dawley (SD) rats (7 weeks old, 200 ± 20 g) were purchased from Beijing Huafukang Bioscience Co. Ltd. The rats were kept in a specific pathogen-free (SPF) facility. Animal experiments were approved by the Research Ethics Committee of the Second Hospital of Hebei Medical University. The experimental animals were euthanized through bloodletting following urethane anesthesia. Anesthesia was induced by intraperitoneal injection of 20% urethane (1mg/kg), administered 20 minutes prior to blood collection and dissection, ensuring deep anesthesia and minimizing animal discomfort during the experiment. Quantification of NBP in plasma by HPLC-MS/MS Chromatography separation was performed on a Symmetry C18 column (4.6 × 150 mm, 3.5 μm), and the column temperature was 40°C. The mobile phase consisted of water (A) and acetonitrile (B, containing 0.1% formic acid) with a flow rate of 0.8 mL·min-1. The elution procedure was as follows: 0–2.0 min, 75%-95% B; 2.0–6.5 min, 95% B; 6.5–7.0 min, 95%- 75% B; and 7.0–8.0 min, 75% B. The injection volume was 5 μL, and the internal standard (IS) was glipizide. Mass spectrum conditions Both NBP and glipizide were monitored in positive ESI mode. The scanning mode was multireaction monitoring (MRM), with the ion transitions of m/z 191.1→45.1 and 446.2→321.0, respectively. The declustering potential (DP) and collision energy (CE) of NBP were 60V and 22V, and those of the glipizide were 100V and 23V, respectively. Plasma sample processing method Acetonitrile (150 μL) containing glipizide (250 ng·mL-1) was added to plasma (50 μL) and centrifuged at 10,900 g for 5 min. The supernatant was transferred to the autosampler vial for HPLC-MS/MS detection. Standard curves A series of NBP solutions with a concentration of 20–2000 ng·mL-1 were added to the blank plasma to prepare the simulated plasma samples, which were tested according to the plasma sample processing method. The standard curve was drawn with the concentration of NBP as the abscissa (X) and the peak area ratio of NBP to the glipizide as the ordinate (Y). The standard curve equation was obtained through regression. Experimental design and sample collection Before the experiment, 30 SD male rats were randomly divided into three groups (n = 10): the antibiotic, probiotic, and control groups. The antibiotic group received 50 mg·kg-1 of vancomycin solution. The probiotic group received 600 mg·kg-1 of live bifidobacterium tetrad bacteria suspension. The control group received physiological saline. All groups received daily gavage for 7 days. Fecal samples were collected on day 0 and day 8 before NBP administration. On day 8, all rats were gavaged using 70 mg·kg-1 of NBP solution. Blood samples (about 300 μL) were collected in heparinized centrifuge tubes at 0.08, 0.16, 0.33, 0.5, 0.75, 1, 1.5, 2, 3, 4, 6, and 8 h after gavage. Plasma samples were centrifuged at 8000 RPM for 10 min at 4°C. The rats were sacrificed after collecting blood and samples from the liver and small intestine tissues. The samples were stored at -80°C. Effects of the intestinal microbiota on the pharmacokinetics of NBP The frozen plasma samples were thawed naturally, processed according to the “plasma sample processing method,” and injected for analysis. NBP concentrations at different time points were calculated according to the standard curve equation. The pharmacokinetic parameters were calculated using the DAS 2.0 software. Statistical analysis was performed with SPSS 20 software to investigate the effect of antibiotic or probiotic interventions on NBP pharmacokinetics. T-test was used for data consistent with normal distribution, and Wilcoxon rank sum test was used for data inconsistent with normal distribution. CYP3A1 mRNA and protein expressions in the liver and small intestinal Quantitative real-time PCR (qRT-PCR) to detect CYP3A1 mRNA expression The frozen small intestine and liver tissue samples were naturally thawed at room temperature. Total RNA was extracted and tested for RNA concentration and purity, followed by reverse transcription and primer amplification (primer sequence shown in Table 1), with three replicates per transcript. CYP3A1 was encoded by Cyp3a1, and its internal reference was β-actin. Data processing was performed using the 2-ΔΔCt method. T-test was used for data consistent with normal distribution, and Wilcoxon rank sum test was used for data inconsistent with normal distribution. 10.1371/journal.pone.0297713.t001 Table 1 The sequence of primers. Name Sequence (5’-3’) 16s RNA V3~V4 F: barcode + ACTCCTACGGGAGGCAGCA R: GGACTACHVGGGTWTCTAAT Cyp3a1 F: ACCATCCTCGTGCTCCTGTATC R: AAGCTCCGACGGTTTGTGAAG β-actin F: TGCTATGTTGCCCTAGACTTCG R: GTTGGCATAGAGGTCTTTACGG Western blot assay for CYP3A1 protein expression The frozen small intestine and liver tissue samples were thawed at room temperature. CYP3A1 total protein was extracted, added to 10% SDS-PAGE gel, and then transferred to the nitrocellulose membranes. Nitrocellulose membranes were incubated with the monoclonal CYP3A4 antibody (1: 2000 dilution) at 4°C for 12 h, followed by the incubation with a secondary antibody (1:5000 dilution) at room temperature for 30 min. The GAPDH was used as a loading control. The Odyssey CLx Imaging System was used to detect target proteins. The protein level was normalized to the control group and expressed as multiple changes relative to the control group. For statistical analysis, t-test was used for data consistent with normal distribution, and Wilcoxon rank sum test was used for data inconsistent with normal distribution. Effects of antibiotics and probiotics on the intestinal microbiota The frozen feces were naturally thawed at room temperature, and DNA was extracted. PCR amplification was performed on specific primers for the V3-V4 region of 16S rRNA bacterial genes according to the Illumina 16S metagenomic sequencing library (Table 1). The amplified products were quantified and mixed for library construction and quality inspection. After the individual quantification step, amplicons were pooled in equal amounts and subjected to paired-end 2 × 250 bp sequencing to understand the effects of antibiotics and probiotics on the intestinal microbiota. Data analysis and statistics 16S rRNA intestinal microbiota data were analyzed using the Genescloud cloud platform online. The relative abundance of the microbiota was plotted as a stacked-bar plot at the phylum and genus levels. α-diversity indices (diversity within a sample) were calculated using filtered data. Among these indices, the Chao1 diversity index estimated total richness, and the Shannon diversity index evaluated the richness and evenness of species at the genus level. β-diversity (sample dissimilarity, the difference between two samples) was calculated using the principal coordinate analysis (PCoA) based on the Bray-Curtis dissimilar matrix. Kruskal-Wallis rank sum test was used to analyze bacteria with significant abundance differences between groups, and linear discriminant analysis effect size (LEfSe) was used to analyze bacteria enrichment. Microbial functional abundances based on marker gene sequences were predicted by Phylogenetic Investigation of Communities by Reconstruction of Unobserved States (PICRUSt2) in the Kyoto Encyclopedia of Genes and Genomes (KEGG). Spearman correlation analysis was performed to explore the relationship between the intestinal microbiota and the metabolic pathways of NBP. P<0.05 was considered statistically significant. Plasma sample processing method Acetonitrile (150 μL) containing glipizide (250 ng·mL-1) was added to plasma (50 μL) and centrifuged at 10,900 g for 5 min. The supernatant was transferred to the autosampler vial for HPLC-MS/MS detection.',\n",
       " 11198784: 'Materials and methods Participant characteristics Participants diagnosed with GWI and HC participants were recruited between 06th June and 30th November 2023 to be included in this study. The GWI group was composed of six Australian males who engaged in combat at the GW, were diagnosed with GWI previously, and met CDC Case Definition [14] and Kansas criteria [15] for GWI. Six HC male participants were selected for this study, all those reported to be in good health, absence of illness and no fatigue episodes. All volunteers were between 18 and 65 years and non-smokers. Participants were excluded from this investigation if they were previously diagnosed with a chronic illness or reported alcohol abuse, use of opioids, medication, or supplements that interfere with TRPM3 ion channels or Ca2+ signalling. However, participants had the option to cease taking conflicting medications in accordance with the half-life of pharmacological compounds and if authorised by their physician. This investigation was approved by the Griffith University Human Research Ethics Committee (GU HREC 2022/666) and all participants provided their written consent. Participant symptoms and disability All participants completed a questionnaire created by the National Centre for Neuroimmunology and Emerging Diseases (NCNED) to provide their medical history, sociodemographic background, illness symptoms, and disability information. To assess disability data the questionnaire includes the World Health Organization Disability Assessment Schedule (WHODAS). In this study, symptoms from people with GWI were classified in ten subtypes: (1) cognitive difficulties (e.g. cognitive overload, confusion, disorientation, impaired concentration, forgetfulness and memory problems); (2) pain (e.g. headaches, muscle aches and multi-joint pain); (3) sleep disturbances (e.g. unrefreshing sleep, frequent awakenings, prolonged sleep, reversed sleep cycle); (4) cardiovascular symptoms (e.g. orthostatic intolerance, cardiac arrhythmias, heart palpitations, light headedness and dizziness); (5) respiratory symptoms (e.g. air hunger, difficulty breathing); (6) thermostatic intolerances (e.g. subnormal body temperature, abnormal sweating episodes, hot flushes and cold extremities); (7) neurosensory or perceptual symptoms (e.g. inability to focus vision, impaired depth perception, sensitivity to touch, light, odour, taste, sound, vibration and poor balance or coordination); (8) urinary changes (e.g. changes to urination frequency and urgency to urinate); (9) immune disturbances (e.g. sore throat, tender lymph nodes, new allergies/sensitivities); and (10) gastrointestinal disturbances (e.g. nausea, abdominal pain, bloating, diarrhoea and irritable bowel syndrome). The WHODAS indicates the level of disability from each participant and data is combined by groups. WHODAS is subdivided into seven domains of life: (1) Communication and understanding; (2) Mobility; (3) Self-care; (4) Interpersonal connections; (5) Life activities; (6) Work or School participation; and (7) Participation in society. WHODAS items were scored on a five-point scale (none, mild, moderate, severe, and extreme or cannot do). The subscale scores were determined in accordance with the WHODAS 2.0 manual, first converting each item score into the corresponding, predefined weighted values [46]. Scores converted from 0% to 100%, disability are inversely proportional to the scale, whereby lower scores indicate less disability and correspond 100% to full disability [47]. Peripheral blood mononuclear cell and natural killer cell isolation Each participant donated between 40 ml and 84 ml of whole blood. All blood collections were conducted by a qualified phlebotomist, via venepuncture, using ethylenediaminetetraacetic acid (EDTA) tubes. A sample of 4 ml of whole blood from each participant was sent to a pathology laboratory for full blood count (FBC). The remaining whole blood samples were used to isolate peripheral blood mononuclear cells (PBMCs) by centrifugation over a density gradient medium (Ficoll-Paque Premium, GE Healthcare, Uppsala, Sweden). PBMCs total cell count, live cell count and viability were assessed using trypan blue dye (Invitrogen, Carlsband, CA, USA) and automatic cell counter (TC20 Automated cell counter, Bio-Rad, Laboratories, Hercules, CA). For NK cell isolations, PBMCs were adjusted for a concentration of 5×107 cells/ml. NK cell isolations were conducted by immunomagnetic selection using an EasySep Negative Human NK Cell Isolation Kit (Stem Cell Technologies, Vancouver, BC, Canada). Flow cytometry was performed to identify the NK cell purification from each NK cell isolation. Immediately after NK cell isolation, cells were incubated with CD56 APC (0.25g/20l) and CD3 PE Cy7 (0.25g/5l) monoclonal antibodies (Becton Dickinson (BD) Bioscience, San Jose, CA, USA) for 20 minutes in the dark at room temperature. NK cells were washed and resuspended in 350 ml of stain buffer (BD Bioscience, New Jersey, USA) and acquired at 10,000 events using the BD LSR- FortessaTM X-20 flow cytometer (BD Biosciences, San Diego, CA, USA). The NK cell population was then identified using phenotypic surface expression as CD3-CD56+. For this study, acceptable NK cells purity was ≥ 90%. S1 Fig shows and compares purity results from HC and GWI groups. Importantly, there was no statistical difference between groups. Electrophysiological experiments The gold standard patch-clamp technique was conducted to determine TRPM3 ion channel activity in NK cells freshly isolated from HC and people with GWI. In this study, borosilicate glass capillaries (Harvard Apparatus, Holliston, MA, USA, GC150F-15, outside diameter = 1.5 mm, inside diameter = 0.86 mm) were pulled to obtain glass pipette (Sutter Instrumental, model P-97) and polished posteriorly (Narishige, Micro Forge MF-900). When filled with pipette solution, membrane resistance was 8 to 12 MΩ. A CV203BU head-stage (Molecular Devices, Sunnyvale, CA, USA) connected to a 3-way coarse manipulator and a micromanipulator (Narishige, Tokyo, Japan) were used in these experiments. To amplify and record electrical signals, an Axopatch 200B amplifier and pClamp 10.7 software (Molecular Devices, Sunnyvale, CA, USA) were used, with data filtered at 5 kHz and sampled digitally at 10 kHz via a Digidata 1440A analogue to digital converter (Molecular Devices, Sunnyvale, CA, USA). The voltage-ramp protocol was a step from a holding potential of +10 mV to -90 mV, followed by a 0.1 s ramp to +110 mV, before returning to +10 mV (repeated every 10 seconds). The liquid junction potential between the pipette and bath solutions (10 mV) was corrected and no leak current component was subtracted. The intracellular pipette solution contained: 30 mM CsCl, 2 mM MgCl2, 110 mM L-Aspartic acid, 1 mM EGTA, 10 mM HEPES, 4 mM ATP disodium hydrate, 0.1 mM GTP sodium salt hydrate (pH = 7.2, adjusted with CsOH; Osmolality = 290 mOsm/L, adjusted with D-mannitol), filtered with 0.22 m membrane filter (Sigma-Aldrich, St. Louise, MO, USA), aliquoted and stored at -20°C. The possibility of chloride current involvement in TRPM3 assessment was minimized by using L-Aspartic acid in the intracellular pipette solution. The extracellular solution contained: 130 mM NaCl, 10 mM CsCl, 1 mM MgCl2, 1.5 mM CaCl2 2H2O, 10 mM HEPES, (pH = 7.4, adjusted with NaOH; Osmolarity = 300 mOsm/L, adjusted with D-glucose) freshly prepared. As previously validated by NCNED, pharmacological agents were included in the extracellular solution to assess TRPM3 ionic currents [26]. Briefly, a gravity perfusion system was used to apply extracellular solution for 50 seconds to establish a baseline current. Subsequently, extracellular solution containing 100 μM of PregS was added to stimulate TRPM3 ion channels for 2.5 minutes. Following on from this addition extracellular solution with 10 μM Ononetin and 100 μM of PregS was applied for another 2.5 minutes to block TRPM3 ion channels. After the conclusion of the drugs application, another cycle of 100 seconds of only extracellular solution was applied to remove the drugs. ATP and GTP were purchased from Sapphire Bioscience Reagents, PregS and Ononetin were purchased from Tocris Bioscience, while all other reagents and chemicals were ordered from Sigma-Aldrich. PregS and Ononetin were resuspended and stored in accordance with the manufacturer’s instructions. Electrophysiological experiments were conducted at room temperature (22–24°C). All recordings were analysed individually by a blinded researcher and posteriorly data was reviewed one by one by another blinded researcher, as detailed in S2 Fig. Additionally, any unstable currents or chloride contamination was excluded from the analysis. Statistical analysis Questionnaire data were analysed through the Statistical Package for the Social Sciences (SPSS) software, version 27 (IBM Corp, Armonk, NY, USA) and purity results with GraphPad Prism v9 (GraphPad Software Inc., La Jolla, CA, USA). For electrophysiological data analysis and data presentation, pCLAMP 10.7 software (Molecular Devices, Sunnyvale, CA, USA), Origin 2021 (OriginLab Corporation, Northampton, MA, USA), and GraphPad Prism version 9 were used. Shapiro-Wilk normality test was performed to identify the distribution of data. ROUT method was conducted to determine outliers and they were removed from analysis. The independent nonparametric Mann-Whitney U test was performed to identify the statistical significance between GWI and HC groups in PregS and Ononetin amplitude. The Fisher’s exact test (applying Bonferroni method) was conducted to determine statistical significance regarding sensitivity to Ononetin in NK cells. Significance was set at p < 0.05 and the data are presented as mean ± standard error of the mean (SEM) unless otherwise stated.',\n",
       " 11198029: 'Methods Literature Search For the purpose of evaluating the evidence regarding the efficacy of metformin therapy for AF, a narrative review of the literature published on Embase was conducted, facilitated by a librarian and co-author (RM). For this review, both peer reviewed original studies and reviews, along with non-peer reviewed (preprint) papers from bioRxiv and medRxiv etc., regarding the effects of metformin on AF were critically evaluated. Publications were selected based on evidence that the manuscripts support their claims with statistical evidence. Their scientific merit was also determined on the basis of their study design. Publications in Embase were identified by using the search strategy described in Table 2. Table 2Search Strategy Used on Embase. The Search Was Based on “Metformin and AF and Diabetes and (Clinical Trials or Reviews)”. Total Number of Studies Used Was = 196 After Removing Duplicates (n = 3)StepSearchResults1(metformin or dimethylbiguanidine or dimethylguanylguanidine or glucophage).ab,ti.47,4802Metformin emtree: exp metformin/86,0233(“Atrial fibrillation” OR AF OR Afib OR AFib OR A-fib OR A-Fib OR “Supraventricular tachycardia” OR SVT OR Arrhythmi* OR palpitat*).ab,ti.346,3314AF emtree: exp atrial fibrillation/126,43851 OR 289,22763 OR 4379,49975 AND 61,1378Diabetes (Emtree term): exp diabetes mellitus/1,224,3809Clinical trials (Emtree term): exp clinical trials/447,028109 OR (randomized controlled trial.ti,ab).574,55511Reviews (Emtree term): exp review/3,175,7991211 OR (literature review or literature).ti,ab.4,159,842137 AND 88621410 OR 124,515,7651513 AND 1420016Limit 15 to yr=”1998–2024”199 The data presented in Figure 1 was generated from the Scopus database and based on the following search: (TITLE-ABS-KEY (metformin OR dimethylbiguanidine OR dimethylguanylguanidine OR glucophage)) AND (TITLE-ABS-KEY (“atrial fibrillation” OR afib OR a-fib OR “Supraventricular tachycardia” OR svt OR arrhythmi* OR palpitat*))',\n",
       " 11197919: 'Materials and methods Study participants This study was conducted at Cornell University in Ithaca, New York, between October and December 2020. Participants were recruited via advertisements using flyers and e-mail listservs in and around the Cornell University campus. All human-related procedures and sample and data collection were approved by the Cornell University Institutional Review Board for Human Participant Research (Protocol Number: 1902008575) prior to recruitment and enrollment of participants. Study participants included healthy males and healthy, non-pregnant or lactating females 18–59 years old. Baseline participant characteristics can be found in Table 1. Exclusion criteria included a history of gastrointestinal diseases or surgeries; type 1 or type 2 diabetes, prediabetes or impaired glucose tolerance, self-reported untreated thyroid condition; use of antibiotics 6 months prior to the start of the study; and chronic alcohol intake (>5 drinks/day). The full list of inclusion and exclusion criteria is provided in Table S1. This trial was registered at clinicaltrials.gov as NCT05743790 on February 24, 2023.Table 1.Baseline characteristics for all participants assigned to a study arm. Group A (n = 35)Group B (n = 33)Totalp valueAge in years28 ± 927 ± 1127 ± 100.376 Sex, n (%) 0.542 Female22 (62.9%)24 (72.7%)46 (67.6%) Male13 (37.1%)9 (27.3%)22 (32.4%) Body fat percentage Female26.2 ± 7.924.9 ± 6.625.50 ± 7.20.531 Male18.0 ± 5.120.40 ± 7.019.00 ± 5.90.351Energy-adjusted dietary fiber intake at baseline in grams12.7 ± 5.511.0 ± 4.311.9 ± 5.00.235There were 35 participants in Group A and 33 in Group B, for a total of 68 who were assigned to a study arm. Data are presented as mean (standard deviation) or n (%). There were no significant differences between Group A and B at baseline for age, sex, body fat percentage for males, body fat percentage for females, and energy-adjusted dietary fiber intake at baseline. T-tests and Wilcoxon rank sum tests were used for numerical variables and chi-squared tests were used for binary variables. Study design We enrolled eligible participants into a 7-week open-label crossover dietary intervention study (CONSORT diagram: Figure S1). Each participant was assigned a study identification (ID) number based on the order in which they were enrolled in the study. All participants with an odd ID number were allocated to Group A, and all participants with an even ID number were allocated to Group B, achieving a 1:1 balanced allocation. A single participant with an odd ID number was allocated to Group B to balance the groups. Participants were instructed to consume crackers containing RS2, RS4, or a digestible control starch (Ctl), in their assigned treatment order, each for 10 d (Figure 1), in addition to their normal dietary intake. The groups received treatments in the following order. Group A: Treatment 1 = RS2, Treatment 2 = Ctl, Treatment 3 = RS4; Group B: Treatment 1 = RS4, Treatment 2 = Ctl, Treatment 3 = RS2. The three treatment periods were separated by 5-d washout periods when no study crackers were consumed. Participants collected fecal samples prior to (“PreRS2,” “PreCtl,” or “PreRS4”) and at the end of (“EndRS2,” “EndCtl,” or “EndRS4”) the treatment periods (Figure 1: see “Pre” and “End” time points). Figure 1.Study design.Participants were enrolled in one of the two arms (Groups A and B) of a 7-week crossover dietary intervention study during which we supplemented their diets with crackers containing one of the three starches: RS2 = resistant starch type 2, RS4 = resistant starch type 4, Control = digestible starch. Each treatment lasted for 10 d, and there were 5-d washouts between treatments. Stool samples were collected at baseline and at the beginning (Pre) and end (End) of each treatment. Please see CONSORT flow diagram (Figure S1). This figure was created using BioRender. Dietary supplementation Participants were provided with crackers that contained RS2 (HI-MAIZE® 260 starch, Ingredion), RS4 (VERSAFIBE™ 1490 starch, Ingredion), or a digestible control starch (AMIOCA™ TF starch, Ingredion). Study cracker formulations are listed in Table S2a. During Treatments 1 and 3, we aimed to provide 30 g of each type of RS in a daily serving of 120 g of crackers for 7 d after a 3-d ramp-up period to minimize gastrointestinal discomfort. The daily serving size for the control cracker after the ramp-up period of 3 d was also 120 g. Thus, during each 10-d treatment period, we followed a dose escalation of study crackers where participants gradually increased the dosage over 3 d to reach a final dose of 120 g of crackers per day for the last 7 d (Day 1: 25%, Day 2: 50%, Day 3: 75%, Day 4–Day 10: 100%). The RS2, RS4, and control crackers were approximately matched for total carbohydrate, fat, and protein content, but the control cracker contained more calories than the RS crackers (Table S2b). Following production, the crackers were analyzed for RS and total dietary fiber content by Medallion Laboratories (Minneapolis, MN) using AOAC 2002.02 and AOAC 991.43 methods, respectively (Table S3). The method AOAC 2002.02 accurately quantifies RS2 but not RS4, so it is not providing an accurate measurement of RS in the RS4 crackers.24,25 However, the method AOAC 991.43 does detect RS4, so by deduction, the AOAC 991.43 measurement is indicating the approximate amount of RS4 (less the 1.92 g of total dietary fiber from the other ingredients, which are also in the control cracker).26 After baking, the amount of RS in RS2 crackers was reduced from 30 g to 21.27 g per serving, possibly due to starch gelatinization, and this may have also occurred in the RS4 crackers. Participants were asked to maintain their habitual dietary intake and physical activity across interventions and to avoid taking prebiotic- or probiotic-added foods, drinks, or supplements throughout the study. Study crackers were provided in individual bags containing a preportioned daily supply by weight. At the end of each treatment period, participants were asked to complete a short questionnaire to indicate the percentage of crackers consumed per day as an indicator of adherence to the protocol. Data from participants who reported a low adherence to cracker consumption were excluded from analysis. Low adherence was defined as having consumed <75% of study crackers for 5 or more days during the treatment or <50% on the day of or day before collecting their End time point fecal sample. Anthropometric measurements At baseline, we measured the participants’ total body fat percentage with a Tanita SC-240 Total Body Composition Analyzer using Bioelectrical Impedance Analysis (BIA) as per the manufacturer’s instructions. Dietary intake data During the baseline and treatment periods, we instructed participants to complete two nonconsecutive 1-d food records of one weekday and one weekend day, excluding study cracker consumption, using the Automated Self-Administered 24-hour (ASA24) Dietary Assessment Tool 2020 developed by the National Cancer Institute, Bethesda, MD (https://epi.grants.cancer.gov/asa24.).27 Data were averaged from the two food records for each participant within the baseline week and each of the three treatments (Figure S2a). Data were excluded if only one of the two records was completed. Saliva collection We obtained 5 ml of saliva at baseline and during each treatment for a total of four saliva samples throughout the study for each participant (Figure 1). Participants were instructed to refrain from brushing their teeth for a minimum of 6 hours and to avoid consuming any food or beverages including water for a minimum of 30 minutes prior to sample collection. They were then instructed to accumulate saliva in their mouth and express it into a 50 ml sterile conical tube. Saliva samples were stored on ice immediately after collection, aliquoted within 3 hours, and stored at −80°C. AMY1 copy number determination by qPCR and ddPCR Genomic DNA was extracted from saliva samples using the QIAamp 96 DNA Blood Kit, QIAamp Blood Mini Kit, and the QIAamp Investigator Kit (Qiagen, cat # 51161, 51104, 56504). We performed qPCR using primers, previously described in Poole et al.,18 to amplify AMY1 paralogs and our reference gene, EIF2B2 (CN = 2). For each gene, each qPCR reaction consisted of 1 μl genomic DNA normalized to 5 ng/μl, 0.5 μl of each forward and reverse primer at 10 μM, 3 μl of PCR grade H2O, and 5 μl iTaq™ Universal SYBR® Green Supermix (BioRad, cat # 1725122), for a total volume of 10 μl per reaction. The qPCR conditions were as follows: initial denaturation at 95°C for 5 minutes and 40 cycles of 95°C for 10 seconds and 60°C for 30 seconds on a Roche LightCycler 480 Real-Time PCR Instrument. We made a standard curve using genomic DNA NA12286 (Coriell Institute; AMY1 CN = 2). The following genomic DNAs were used as positive controls on all qPCR plates, as their AMY1 copy numbers have previously been estimated: NA18972, NA12873, NA10472, NA12890, NA10852, NA12043, NA11992, NA12414, NA12340, NA06994, NA12342, NA12286, NA18522, and NA19138 (Coriell Institute). All reactions including standards and blanks were performed in quadruplicate, and results were averaged for technical replicates with a coefficient of variation <0.05. At least two qPCR runs were performed for all participants, and we calculated the median value of all qPCR results to determine the final qPCR AMY1 CN value. For digital PCR, genomic DNA was digested with the restriction enzyme HaeIII (New England Biolabs, cat # R0108S) and diluted to a final concentration of approximately 15 ng/µl. Digital PCR was performed using Life Technologies TaqMan Copy Number Assay Id Hs07226361_cn for the AMY1 locus and TaqMan Copy Number Reference Assay Hs06006763_cn for the AP3B1 gene to normalize for total DNA. The reactions were run on a QX100 Droplet Digital PCR System in duplicate. We averaged the median of all qPCR results and the two digital PCR results to determine the final AMY1 CN used in our analyses (Figure S2b). AMY1 CN group was defined as that used in a previous publication that inspired this work18 because even though categorization is often used, currently, there is no consensus in the field regarding the threshold for high and low categories.28,29 Perry et al. reported that low starch consuming populations have a median AMY1 CN of 5 and high starch consumers have a median of 7.30 Although the threshold for the effects of this gene on physiology or health is currently unknown, Perry et al.’s conclusion that positive selection occurred at this locus due to dietary adaptation suggests that the low end of phenotypic effect is CN of 5. In our cohort, salivary amylase activity (SAA) is significantly different between our AMY1 CN groups with the mean of the high group (SAA = 145 U/ml) being over twice that of the low group (SAA = 66 U/ml) over time (p = 4.85 × 10−4). Salivary amylase activity assay We measured SAA for each saliva sample in triplicate, from up to four saliva samples donated from each participant throughout the duration of the study, using the Salimetrics Salivary Alpha-Amylase Enzymatic Kit (SALIMETRICS, cat # 1–1902). The manufacturer’s protocol was followed except for the use of 300 µl amylase substrate per reaction instead of 320 µl. We averaged SAA determined for each participant to obtain the mean SAA used in our analyses (Figure S2c). Fecal sample collection Participants collected fecal samples at seven time points throughout the study (Figure 1). Samples were collected from a single bowel movement and stored at −80°C within 24 hours of collection. We lyophilized an aliquot from each sample for 16S rRNA gene sequencing. Samples from study participants who reported collection of fecal samples ≥2 d after completion of the treatment were excluded from analysis. 16S rRNA gene sequencing DNA extraction was performed on 0.030–0.045 g lyophilized fecal samples using the DNeasy PowerSoil 96 HTP Kit (Qiagen, cat # 12888–100), following the manufacturer’s instructions with the specifications: samples were loaded into PowerBead Plates and stored at −20°C in Bead Solution until extraction, and instead of vortexing to mechanically lyse samples, samples were placed in a BioSpec 1001 Mini-Beadbeater-96 for 3 minutes. We amplified the V4 region of the 16S rRNA gene using the universal primers 515F and barcoded 806 R31 and approximately 100 ng of genomic DNA from each sample in duplicate PCR reactions using 25 µl Classic++™ Hot Start Taq DNA Polymerase Master Mix (Tonbo Biosciences, cat # TB-31-5011–1000 R), 22 µl PCR water, 0.5 µl of each 10 nM primer, and 2 µl DNA. We used the PCR program previously described but with 25 cycles of amplification.31 We purified amplicons using Mag-Bind TotalPure (Omega Bio-tek, cat # M1378–01) using a 1.8X bead ratio, pooled 100 ng of amplicons from each sample, and performed 2 × 250 bp sequencing on an Illumina MiSeq instrument. We performed microbiome bioinformatics with QIIME 2.32 We demultiplexed and quality filtered raw sequence data via q2-demux. Then we resolved ASVs using DADA233 via q2-dada2. We aligned ASVs with mafft34 via q2-alignment and constructed a phylogenetic tree using fasttree35 via q2-phylogeny. We used q2-diversity to calculate alpha-diversity (Faith’s Phylogenetic Diversity, Faith’s PD)36 after samples were subsampled without replacement to 23,081 sequences per sample based on the sample with the lowest sequence count. These rarefied data were used exclusively for calculating Faith’s PD. For differential abundance analysis, we utilized the non-rarefied feature table to retain the full complexity of the microbial communities. We used the q2-feature-classifier37 classify-sklearn naive Bayes classifier against the Greengenes 13_8 99% OTUs reference sequences38 to assign taxonomy to ASVs. We identified 6,547 ASVs within our entire dataset (all time points from 59 participants). After we calculated the mean number of ASVs present (number of ASVs with a non-zero read count) in each individual across all time points, we observed that the mean of the means was 159 ASVs per person, the median of the means was 156 ASVs per person, and the standard deviation of the means was 45. Short-chain fatty acid measurements We measured SCFA concentrations in stool samples collected at baseline and at the time points Pre and End of each treatment. SCFAs including acetate, propionate, isobutyrate, butyrate, isovalerate, valeric acid, isocaproic acid, caproic acid, and heptanoic acid were quantified using ultra-performance liquid chromatography (Acquity UPLC system, Waters Corporation, Milford, MA) at the PennCHOP Microbiome Program Microbial Culture & Metabolomics Core. Total SCFA concentration was calculated as the sum of all nine SCFAs quantified. Statistical analysis We performed all statistical analyses using RStudio version 4.2.1.39 We adjusted dietary fiber intake by energy (g/1,000 kcal/day) to account for differences in participants’ overall energy intake. We considered p-values <0.05 to be statistically significant. When adjusting p-values we used the Benjamini–Hochberg false discovery rate correction and considered a q-value (adjusted p-value) of <0.05 to be statistically significant. To calculate fold changes of ASVs, we added a pseudocount of 1 to all ASV counts prior to calculating relative abundances to avoid division by zero. Since the limit of detection of the quantification method for SCFAs was 5.0 µmol/g of stool, a count of 2.5 µmol/g was added to all SCFA measurements in the linear regression and LASSO analyses, in which logarithm or fold change was calculated, to avoid an undefined term (logarithm of zero or division by zero). We used R package ComplexHeatmap40 and circlize41 to generate all heat maps. To assess the response of ASVs to each treatment, we used the R package MaAsLin2 (Microbiome Multivariable Association with Linear Models 2) because this software allows us to include repeat measures in longitudinal sampling (Pre and End) as well as covariates, takes into consideration the nature of ASV data (zero-inflated, high dimensional, non-normal distribution) and controls for the false discovery rate.42 Data from all participants were included, regardless of the order of treatment (Group). For each treatment (RS2, RS4, and Ctl), we used the min_prevalence default = 0.1 parameter to filter out the ASVs present in less than 10% of the samples collected. For example, ASVs had to be present in at least 10% of the combined samples from PreRS2 and EndRS2 to be included in the RS2 linear mixed model, leaving a total of 325 ASVs. After prevalence filtering, there were 330 ASVs included in the linear mixed model for RS4 and 365 ASVs for Ctl. Prior to fitting the models, the ASV data were normalized using total sum scaling. We used the log transformation setting within MaAsLin2’s transform parameter. For each treatment, we fit three types of models. First, we fit main effects models using time point, abbreviated TP (PreRS2 versus EndRS2, PreRS4 versus EndRS4, or PreCtl versus EndCtl), as a fixed effect and participant as a random effect in each model to identify ASVs that, on average, changed in relative abundance from Pre to End of each treatment period.(1)Relative abundance \\\\~ TP Pre, End + 1|Participant Our next goal was to identify variables that affected the change in relative abundances of ASVs during each treatment. Thus, we added interaction terms to each of these separate models (AMY1 CN × time point, AMY1 CN group × time point, mean SAA × time point, baseline dietary fiber intake × time point, dietary fiber intake during treatment × time point, and treatment order × time point). Because this was a screening step, we followed up on all interactions with q < 0.25. Next, we fit main effects models using End time points only, with each candidate predictor as an independent variable in a separate model, to detect whether the candidate predicted the relative abundance of any ASVs at the end of the treatment with q < 0.05. To determine the effect of each treatment on SCFA concentrations, we fit linear mixed models with log SCFA concentration as the response variable, time point (TP; Pre vs End) as a fixed effect, and participant as a random effect.(2)Log SCFA + 2.5 \\\\~ TPPre, End + 1|Participant We fit separate models for each treatment (RS2, RS4, Ctl) and SCFA (acetate, butyrate, propionate, and total). To determine whether treatment order (Group A versus B) affected changes in SCFA concentrations from Pre to End (time point), we used linear regression models with each SCFA as the response variable and included an interaction term between Group and time point. A random effect term was included for participant. Fixed effects were tested using F tests with the Kenward-Roger approximation for the degrees of freedom, and post-hoc pairwise comparisons were performed using Tukey’s HSD method.(3)Log SCFA + 2.5 \\\\~ TPPre, End x GroupA, B + TPPre, End + GroupA, B + 1|Participant To assess whether alpha diversity could predict changes in SCFA concentrations, we fit logistic regression models with SCFA change score as the response variable and Faith’s phylogenetic diversity (PD) as a fixed effect. We defined any increase in SCFA concentration from Pre to End (End – Pre > 0) to have a change score of 1, while no change or a decrease in SCFA concentration (End – Pre ≤ 0) was assigned a change score of 0.(4)SCFA change score \\\\~ Faith′s PD Pre Because our goal is to predict the SCFA change score, we used Faith’s PD at the beginning of the treatment (Pre) as the predictor in each model. We fit separate models for each treatment (RS2, RS4, Ctl). We used the R package glmnet43 to perform 10-fold cross-validated Least Absolute Shrinkage and Selection Operator (LASSO) linear regressions to model the log fold change of acetate, propionate, and butyrate concentrations between Pre and End of each treatment period (log2 [End/Pre]). For variable selection, we included the predictors: 10% prevalence-filtered ASVs at Pre, body fat percentage, sex, treatment order, AMY1 CN, AMY1 CN group, mean SAA, and self-reported physical activity (minutes/week of vigorous, moderate, and low-intensity activity) at baseline. We used a separate model for each SCFA and each treatment, and after 10% prevalence filtering for each treatment, there were 332 ASVs in the RS2 models, 339 ASVs in the RS4 models, and 349 ASVs in the Ctl models. To create the figure depicting the LASSO coefficients, we standardized the regression coefficients and excluded the predictor group, or treatment order, to show all the coefficients on a comparable scale. Standardization was performed by multiplying each β coefficient by the standard deviation of the relative abundance of the corresponding ASV at Pre.',\n",
       " 11197476: 'Materials and Methods Preparation and characterization of CXB@Lipo and C@ECN CXB@Lipo (C@L) was obtained by the thin-film hydration method. Briefly, CXB (1 mg), cholesterol (38 mg), and HSPC (200 mg) were dissolved in chloroform. Then, chloroform was removed from this hybrid solution by a rotary evaporator from the film on the round-bottom flask. Next, the film was retuned by adding 2 ml of H2O and sonicating for 10 min. Following this, the pre-C@L was further extruded with a liposome extrude (LiposoFast, Avestin, Canada) through 200-nm polycarbonate membrane filters (Merck Millipore Ltd., USA). Finally, the prepared C@L was concentrated in an ultrafiltration centrifuge tube (30 kDa at 2,000 rpm for 30 min). C@L was riveted on the surface of the ECN, ECN (1 × 107) was resuspended in PBS buffer, and 1 mg of glycol chitosan was added and stirred for 30 min. Then, 1 mL of C@L was added to this buffer (ECN-GC) and stirred for another 30 min. Finally, C@L was removed by washing the complex 3 times with H2O and the mixture was stored at 4 °C for 1 week. The morphology of the ECN or C@ECN was determined via transmission electron microscopy (TEM; JEM-1400, Japan). The average size and zeta potential of the various groups were measured by a Malvern Nano ZS ZEN3600 (Malvern, UK). The characteristic absorption of CXB, liposome, and C@ECN-PL was also determined by UV-VIS (Thermo Fisher Scientific, USA). ICG was used to replace CXB using the above method to obtain ICG@ECN (I@ECN). Then, the effect of mixed liposomes loaded with I@ECN was tested by flow cytometry (CytoFLEX, Beckman, USA). The viability of C@ECN was determined in LB media supplemented with UV‒VIS. Then, 10 μL of ECN, ECN-GC, or C@ECN was added to 5 mL of LB media and incubated at 37 ℃ and 200 rpm. The number of ECNs was determined by the absorption at OD600 nm using UV‒VIS. Preparation of C@ECN-PLs pGEX-Pvhb-Lysis was constructed through the Gibson self-assembly method, involving cleavage of the pGEX plasmid by Bam HI and Eco RI endonucleases, seamless ligation of the Pvhb-Lysis gene fragment into linearized pGEX using a cloning kit (Beyotime Biotechnology Co. Ltd., Shanghai, China), and subsequent introduction of the ligated plasmid into ECN competent cells. Cell uptake of C@ECN To determine the uptake effect of C@ECN or C@L, ICG was used to replace CXB with ICG@Liposome (I@L) or I@ECN [ICG concentration of 0.1 mg/ml and 108 cfu of ECN], respectively. CT26 tumor cells (1 × 106) were seeded into 30-mm glass plates. After incubating for 12 h, fresh DMEM containing I@ECN or I@L was added to the CT26 tumor cells and incubated for another 4 h. Then, the DMEM was removed, and the cells were washed with PBS 3 times. The sections were incubated with DAPI for another 10 min at room temperature, washed with PBS 3 times and stored at 4 °C. Finally, the cells were imaged by confocal laser scanning microscopy (CLSM) (Nikon, Japan). Cell cytotoxicity assay CT26 and MCF-7 tumor cells were seeded into 96-well plate at a starting concentration of 104 cells per well. After incubation for 12 h, free DMEM containing various concentrations of C@L, ECN, C@ECN-PL, C@L + DOX, ECN + DOX, or C@ECN-PL + DOX was added to the medium (DOX concentration of 1.5 μM) and the cells were incubated for another 24 h. Finally, cell viability was determined with a CCK-8 assay kit (Beyotime Biotechnology Co. Ltd., Shanghai, China). COX-2 expression analysis To determine the expression level of COX-2 after various treatments, CT26 tumor cells (106) were seeded into 6-well plates, respectively. After incubating for 12 h, the cell were incubated with fresh DMEM supplemented with 20 μM CXB for another 12 h. Then, RIPA lysis buffer was used to extract proteins from these cells. The expression levels of COX-2 and β-actin were determined by Western blot (WB) analysis. Additionally, all these images were further analyzed and quantified using ImageJ. CLSM was also used to further test COX-2 expression. Briefly, CT26 or MCF-7 tumor cells (106) were seeded into 3cm glass plates. After 12 h of incubation, 20 or 30 μM CXB was added to the plate, and the cells were cultured for another 6 h. Then, the DMEM was removed, and the cells were washed 3 times with PBS. The cells were stained with a FITC-conjugated anti-COX-2 antibody (Affinity, Jiangsu, China) for 2 h at 4 °C, followed by DAPI staining for another 10 min and 3 washes with PBS. Finally, CLSM was used to test the fluorescence signal. After CT26 cells were treated with the aforementioned method, they were collected in 1.5 mL EP tubes and subjected to flow cytometric analysis. Pharmacokinetics and biodistribution of C@ECN in vivo The biodistribution of C@ECN was investigated using an in vivo images system (IVIS). In total, 107 CT26 cells were subcutaneously injected into the left forelimb under the armpit of 6-week-old BALB/c mice. Once the tumor volume reached approximately 100 mm3, the mice were randomly divided into 2 groups and intravenously administered I@L or I@ECN (ECN, 106). The biodistribution of I@L or I@ECN was observed through imaging at 0, 12, 24, 48, and 72 h after injection. After the 72-h treatment period, the mice were euthanized, and major organs (heart, liver, spleen, lung, kidney, and tumor) were collected for in vitro fluorescence signal testing. Tumor tissues were harvested and subsequently embedded in optimal cutting temperature (OCT) compound. After treatment with I@L or I@ECN for 48 h, the specimens were sectioned into slices of 10 μm thick slices. After incubation with DAPI for a period of 10 min, the slices were imaged via CLSM to capture their fluorescence characteristics. Antitumor efficacy of C@ECN-PLs in CT26 tumors in vivo CT26 tumor model was established by subcutaneous transplantation. First, CT26 tumor cells (108) were injected into the left forelimb under the armpit of BALB/c mice to establish the main tumor model. When the tumor volume reached approximately 100 mm3, the mice were randomly divided into 4 groups: PBS, DOX, CXB + ECN-PL + DOX, and C@ECN-PL + DOX, respectively (DOX, 4 mg/kg; CXB, 5 mg/kg; ECN, 107 CFU). Tumor volume and body weight were measured by a Vernier caliper and electronic scale every 2 d for 14 d. The tumor volume was calculated by the following formula: (major axis) × (minor axis2)/2. After 14 d, the mice were sacrificed, and tumors were collected for weighing. Evaluation of tumor apoptosis and proliferation in vivo The CT26 tumor model was established by the above method. All the mice were sacrificed, and the tumor tissue was collected and stored in OCT gel and cut into 6-μm slices at −20 °C. First, this slices were blocked with PBS containing 5% FBS for 30 min. Then, FITC-conjugated anti-mouse CD3, FITC-conjugated anti-mouse CD4, and PE-conjugated anti-mouse CD8 were used to stain the slices for 4 h (dilution factor 1:500, Biolegend Inc., USA). After staining with DAPI for another 10 min and washing with PBS 3 times, the fluorescence signal was obtained by CLSM. In addition, TUNEL and H&E staining were used to analyze CT26 tumor apoptosis in vivo, and Ki67 was used to assess proliferation after various treatments. Biosafety evaluation of C@ECN-PL The biosafety of C@ECN-PL was evaluated by intravenously injected into BALB/c mice. Briefly, healthy mice were intravenously injected with C@ECN-PL at a concentration of (5 mg/kg) CXB, and the number of ECNs was (107 CFU). Seven days later, the mice were sacrificed and blood and major organs (heart, liver, spleen, lungs, and kidney) were collected. The serum levels of aspartate aminotransferase (AST), alanine aminotransferase (ALT), blood urea nitrogen (BUN), and creatinine (CRE) were evaluated by assay kits purchased from the Jiancheng Bioengineering Institute (Nanjing, China). The integrity of major organs was also evaluated by H&E staining analysis, and images were further quantitatively analyzed by ImageJ 6.0. Statistical analysis Statistical analysis was performed via a one-way analysis of variance (ANOVA) test. Moreover, post hoc analysis was performed using the Wilcoxon rank sum test with a Bonferroni correction when needed. *P < 0.05 was considered to indicate statistical significance; **P < 0.01 and ***P < 0.001 were considered to indicate extreme significance; NS, no significant difference.',\n",
       " 11197378: \"MethodsDaily average mass concentration data of six air pollutants in the urban area of Yan'an City from January 1, 2017 to December 31, 2020 were collected from the monitoring station in Baota District, Yan'an City. Meteorological information was obtained from the Meteorological Bureau of Yan'an City, including temperature,relative humidity and wind speed for the time period. The mass concentration of air pollutants in each exposure window of pregnant women was assessed by the nearest monitoring station method, and conditional logistic regression was used to analyze the relationship between air pollutants and preterm births, as well as the lagged and cumulative effects of air pollutants. Multiple linear regression was used to explore the relationship between air pollutants and blood tests after stepwise linear regression was used to determine confounders for each blood test. Methods Research population In this study, the data of pregnant women in the Department of Obstetrics and Gynecology of two local hospitals in Bota District, Yan'an City, were collected from 2018 to 2020. After collecting data information, we confirmed the inclusion and exclusion criteria. Inclusion criteria: residents of Baota District, Yan'an City, who have lived in the district for one year or more; normal mental status; no communication barriers; no assisted conception; no missing information; no major diseases.Exclusion criteria: people with various missing information; people with abnormal mental status who cannot communicate; people with assisted conception or multiple pregnancies; people with hereditary diseases.All pregnant women have signed an informed consent form, and the study has been approved by the ethics committee of Medical School of Yan'an University (approval number: 2018051). After screening for inclusion criteria, 460 cases of preterm birth(PTB) with complete maternal data were collected as the case group. In order to reduce the influence of confounding factors on the results of the study, we took term births(TB) of the same age and the same gestation as preterm births as the control group and selected 1,840 cases of term births as the control group in a ratio of 1 to 4. The data collected from pregnant women included the following: general data of pregnant women (name, age, gestational address, occupation), pregnancy data (gestational age, Pregnancy times, number of births, number of caesarean sections, regularity of menstrual cycle, season of the last menstrual period, Complications diseases, Comorbidity diseases, and hypertension in pregnancy), neonatal data (birth weight, date of birth) and routine blood data:leukocyte count (WBC), percentage of neutrophils (NEUT), percentage of lymphocytes (LYM), percentage of monocytes (MONO), percentage of eosinophils (EOS), percentage of basophils (BAS), erythrocyte count (RBC), hs-CRP,CRP. Complications diseases: severe vomiting of pregnancy, ectopic pregnancy, placenta previa, placental abruption, excessive or low amniotic fluid, premature rupture of membranes, hyperemesis gravidarum, acute chorioamnionitis. Comorbidity diseases: combined cardiovascular diseases (congenital heart disease, rheumatic heart disease,etc.), combined hematological diseases (chronic aplastic anemia, idiopathic thrombocytopenic purpura,etc.), combined respiratory diseases (tuberculosis, bronchial asthma), combined gastrointestinal system diseases (viral hepatitis, acute appendicitis,etc.), combined urinary system diseases (acute pyelonephritis, chronic glomerulonephritis,etc.), combined endocrine system diseases (hyperthyroidism, hypothyroidism,etc.), combined dermatological disorders (scleroderma, hives, herpes), combined infectious diseases (cytomegalovirus infections, genital herpes,etc.), combined tumors (uterine fibroids, cervical cancer,etc.). Pollutant exposure assessment Data sources for air pollutants: All pollutants information was obtained from the Qingyue Open Environmental Data Centre (http://data.epmap.org/), and data were obtained from the daily average mass values of six pollutants (PM2.5, PM10, SO2, NO2, CO and O3) at four monitoring stations in Baota District, Yan'an City (the mass concentration unit of CO is mg/m3, and the mass concentration of the remaining air pollutants is μg/m3). Meteorological data were obtained from Yan'an Meteorological Monitoring Station. The time span of pollutant data and meteorological data is from 1 January 2017 to 31 December 2020. Exposure assessment method: Based on the address of the pregnant woman's place of residence during pregnancy, the latitude and longitude of her place of residence were obtained from Gaode Map. The distance from the latitude and longitude of the place of residence to the latitude and longitude of the monitoring station was calculated, and the pollutant mass concentration of the nearest monitoring station to the place of residence was selected as the individual's exposure mass concentration. After dividing each exposure window according to the date of the last menstrual period of pregnant women, the mass concentration of pollutants in each exposure window was calculated (Fig. 1).Fig. 1Location of Yan'an city in the surrounding area Calculation of pollutants The exposure window is divided according to the date of the mother's last menstrual period into:E: the entire pregnancy (last menstrual period to the date of birth of the births), T1:early pregnancy (last menstrual period to the twelfth week of gestation), T2:mid-pregnancy (the thirteenth week of gestation to the twenty-seventh week of gestation), and T3:late pregnancy (the twenty-eighth week of gestation to the date of birth of the births). Considering that air pollutants have short-term effects in addition to long-term effects on birth, lagged and cumulative effects were calculated based on the date of admission to hospital. Matching of same-day and lag-day mass concentrations: Short-term same-day lagged mass concentrations of each pollutant on the day of the admission date and before the admission date are matched based on the maternal admission date. This exposure mass concentration is the daily average of the day, where the short-term lagged exposure dates are the day of the admission date (Lag0) and the days 1 (Lag1), 2 (Lag2), 3 (Lag3), 4 (Lag4), 5 (Lag5), 6 (Lag6), and 7 (Lag7) before the admission date. Calculation of mass concentration values for cumulative exposure: Matching the mass concentration of each pollutant prior to the date of admission to the date of maternal admission, the cumulative exposure mass concentration value is the average of all daily averages over the exposure period. The cumulative exposure period is 1 day (Lag-1), 2 days (Lag-2), 3 days (Lag-3), 4 days (Lag-4), 5 days (Lag-5), 6 days (Lag-6), and 7 days (Lag-7) before the day of admission. Statistical methods A database was created by EXCEL and data were analyzed by IBM SPSS 20. Normality test for pollutants was performed and pollutants were described using mean, median, standard deviation(SD), and interquartile range(IQR). The categorical variable information in the general information of pregnant women was statistically described using frequency (n) and composition ratio (%), and the difference between the case and control groups was compared using the chi-square test to determine confounders in turn. The correlation between air pollutants and temperature and relative humidity was analyzed, and pollutant mass concentrations were calculated for each exposure window using the R language. The long-term effects and lagged and cumulative effects of air pollutants and preterm birth were analyzed using conditional logisitic models, and after adjusting for each confounding factor, pollutants were introduced into the conditional logistic model with as a continuous variable with in, and ORs and 95% confidence intervals were calculated. After identifying the association between air pollutants and preterm birth, we adjusted for confounders and investigated the relationship between air pollutants and blood counts using multiple linear regression. Logistic regression was then used to explore the relationship between air pollutants and C-reactive protein at a test level of 0.05 (two-sided test). Correlation analysis Spearman's correlation analysis was used to analyze the correlation between air pollutants, blood markers, and meteorological factors (mean temperature throughout the pregnancy, relative humidity throughout the pregnancy, and mean wind speed throughout the pregnancy). Sensitivity analysis In order to verify the stability of the main model, the effect of each pollutant on preterm birth was analyzed separately. Then other confounding factors were added one by one to analyze the relationship between each pollutant and preterm delivery. Three sensitivity analyses were performed with meteorological factors as confounding factors: (1) Model 1: correlation analysis between a single pollutant and each blood index; (2) Model 2: add occupation, birth order and last menstrual season to analyze the correlation between each pollutant and preterm delivery in Model 1; and (3) Model 3: add menstrual cycle and complications diseases to Model 2. Statistical methods A database was created by EXCEL and data were analyzed by IBM SPSS 20. Normality test for pollutants was performed and pollutants were described using mean, median, standard deviation(SD), and interquartile range(IQR). The categorical variable information in the general information of pregnant women was statistically described using frequency (n) and composition ratio (%), and the difference between the case and control groups was compared using the chi-square test to determine confounders in turn. The correlation between air pollutants and temperature and relative humidity was analyzed, and pollutant mass concentrations were calculated for each exposure window using the R language. The long-term effects and lagged and cumulative effects of air pollutants and preterm birth were analyzed using conditional logisitic models, and after adjusting for each confounding factor, pollutants were introduced into the conditional logistic model with as a continuous variable with in, and ORs and 95% confidence intervals were calculated. After identifying the association between air pollutants and preterm birth, we adjusted for confounders and investigated the relationship between air pollutants and blood counts using multiple linear regression. Logistic regression was then used to explore the relationship between air pollutants and C-reactive protein at a test level of 0.05 (two-sided test). Correlation analysis Spearman's correlation analysis was used to analyze the correlation between air pollutants, blood markers, and meteorological factors (mean temperature throughout the pregnancy, relative humidity throughout the pregnancy, and mean wind speed throughout the pregnancy). Sensitivity analysis In order to verify the stability of the main model, the effect of each pollutant on preterm birth was analyzed separately. Then other confounding factors were added one by one to analyze the relationship between each pollutant and preterm delivery. Three sensitivity analyses were performed with meteorological factors as confounding factors: (1) Model 1: correlation analysis between a single pollutant and each blood index; (2) Model 2: add occupation, birth order and last menstrual season to analyze the correlation between each pollutant and preterm delivery in Model 1; and (3) Model 3: add menstrual cycle and complications diseases to Model 2.\",\n",
       " 11197282: 'MethodsWe included 623 matched case–control pairs from the Cancer Prevention Study (CPS) cohorts. Pre-diagnosis blood samples were collected between 1998 and 2001 in the CPS-II Nutrition cohort and 2006 and 2013 in the CPS-3 cohort and were sent for metabolomics profiling simultaneously. Cancer-free controls at the time of case diagnosis were 1:1 matched to cases on date of birth, blood draw date, sex, and race/ethnicity. Odds ratios (ORs) and 95% confidence intervals (CIs) were estimated using conditional logistic regression, controlling for confounders. The Benjamini–Hochberg method was used to correct for multiple comparisons. Methods Study design and population Lung cancer cases and matched controls included in this analysis are participants from the CPS-II Nutrition cohort and ongoing CPS-3 cohort. At enrollment of the CPS-II Nutrition cohort in 1992–1993, participants completed a self-administered questionnaire that included anthropometric, demographic, dietary, lifestyle, and medical information. Follow-up questionnaires were sent to the cohort participants in 1997 and every other year thereafter to update exposures and to ascertain newly diagnosed cancers. A subset of 39,371 CPS-II Nutrition cohort participants provided a non-fasting blood samples between 1998 and 2001, and the information on demographic characteristics and other covariates in the analysis was assessed from the survey collected at blood draw or the 1999 survey. At enrollment of the CPS-3 cohort between 2006 and 2013, participants provided informed consent, a non-fasting blood sample and completed a brief enrollment survey on demographic characteristics and other covariates. Follow-up questionnaires were sent to active participants in 2015 and every 3 years to update exposures and ascertain newly diagnosed cancer cases. Detailed descriptions of the two cohorts can be found elsewhere [22, 23]. All aspects of the CPS-II Nutrition cohort (IRB00045780) and CPS-3 cohort (IRB00059007) were reviewed and approved by the Emory University Institutional Review Board. A total of 1913 lung cancer cases were identified in the CPS-II Nutrition cohort through June 2015 and 176 lung cancer cases were identified in the CPS-3 Cohort through December 2015. Cases in the CPS-II Nutrition cohort were first identified through self-report and then were verified with medical records, state cancer registry linkage, or linkage with the National Death Index (defined by ICD-10 codes C33 and C34, excluding histology codes ≥ 9590). Cases in the CPS-3 cohort were identified primarily through linkage with state cancer registries, and a small proportion were identified by self-report that were verified by medical records during tumor collection. We applied a series of exclusion criteria to include participants (Additional file 1: Fig. S1). As a result, 500 and 123 lung cancer cases from the CPS-II Nutrition cohort and CPS-3 cohort were included in the analysis, respectively. Controls who were cancer-free at the time of case diagnosis were matched 1:1 to cases on age at blood draw (± 6 months), sex, race/ethnicity, and blood draw date (± 30 days). Metabolomics profiling The pre-diagnosis blood samples collected from both cohorts were sent to Metabolon, Inc. (Durham, NC, USA) for untargeted metabolomics profiling simultaneously, using ultrahigh-performance liquid chromatography-tandem mass spectrometry (UPLC-MS/MS) analysis techniques. A detailed process was described elsewhere [24, 25] and in supplemental materials. A total of 1,401 metabolites were detected. After filtering metabolites that were unknown (n = 238), were missing technical intraclass correlation coefficient (ICC) (n = 34), with ICC < 50% (n = 201), and were undetectable in > 90% of samples (n = 41), 887 known metabolites were included in the statistical analysis with an average ICC of 84% (interquartile range (IQR): 77–94%) and the coefficient of variation (CV)% of 24% (IQR: 12–30%). Statistical analysis As metabolomics assessments were conducted simultaneously for cases and controls in both cohorts, we performed a pooled analysis. Metabolites were naturally log-transformed and auto-scaled to approximate normal distribution before formal analysis. Covariate data obtained in each cohort were harmonized. The characteristics between lung cancer cases and matched controls were compared using Student’s t-test for continuous variables and Pearson’s chi-squared test for categorical variables. For the primary pooled analysis, we applied conditional logistic regression to estimate the odds ratio (OR) and 95% confidence interval (CI) per one standard deviation increase in the naturally log-transformed level of each known metabolite with lung cancer risk. The statistical models were conditioned on the matching variables and controlled for the body mass index (BMI) group (underweight: < 18.5 kg/m2, normal weight: 18.5–25 kg/m2, overweight: 25–30 kg/m2, obese: ≥ 30 kg/m2), hours since last meal (continuous; to account for length of fasting), physical activity (continuous; hours/week), fruits and vegetables consumption (continuous; servings/week), smoking status (categorical: never, former, current, and unknown), and hormone use (categorical: not a current user, current user, not applicable, unknown). Physical activity estimates the average total hours per week of walking or exercise in the CPS-II Nutrition cohort, while it estimates the average hours per day during the past 2 years in the CPS-3 cohort. We harmonized the variables and converted them into hours per week. The covariates were selected based on the literature review and a Directed Acyclic Graph. We removed the observations with any missing data for continuous covariates. We assigned an unknown category for missing data for categorical covariates. If a case was removed, its matching control was removed simultaneously, and vice versa. For the primary pooled analysis, 116 case–control pairs were removed due to missing values in hours since the last meal, physical activity, and fruit and vegetable consumption for either case or its matching control (Additional file 1: Fig. S1). Benjamini–Hochberg approach was used to calculate false discovery rates (FDRs) to correct for multiple comparisons. Metabolites associated with lung cancer risk at FDR < 0.2 were deemed statistically significant. To gain more biological responses of lung cancer, we focused on metabolites associated with lung cancer risk at P < 0.05 (P-value from statistical models before multiple comparison corrections) and further described and summarized the pathways in which these metabolites were involved. We conducted an agglomerative hierarchical clustering analysis to group the lung cancer-associated metabolites (P < 0.05) based on their similarities. Pearson correlation was calculated between each pair of metabolites and then used for distance measure. Euclidean distance was computed between each pair of metabolites and returned the distance matrix. We then used the Ward clustering method to compute the similarity of the two clusters for merging [26]. The R package “pheatmap” was used for this analysis and result visualization. We further examined the associations stratified by sex and by years between blood draw and lung cancer diagnosis (< 3 years, ≥ 3 years) using conditional logistic regression with the same set of covariates but excluding hormone use for males. The goal of the stratified analysis by years since the blood draw was to identify metabolites that may potentially serve as early detection biomarkers of lung cancer. Additionally, we stratified the analysis by smoking status (never, ever), by stage (localized, regional, distant), and by histological subtypes (squamous cell carcinoma, adenocarcinoma) using unconditional logistic regression, adjusting for matching variables as well as BMI group, hours since last meal, physical activity, fruits and vegetables consumption, hormone use, and smoking status (only for stage- and subtype-stratified analyses). For stratified analyses by smoking status, stage, and subtype where the unconditional logistic regression was applied, matching factors were adjusted as covariates in the model. If a case was removed due to missing covariates, its matching control would not be removed if the control has completed the covariates information, and vice versa. The lung cancer stage was examined according to the Surveillance, Epidemiology and End Results (SEER) stage at diagnosis: localized (invasive tumors confined to the lung); regional (tumors that extend to adjacent tissue or regional lymph nodes); distant (tumors are metastasized). The lung cancer histological subtype was categorized using ICD-O-3 morphology codes [27]. The morphology codes for each subtype can be found in supplementary materials. P for interaction was calculated using the likelihood ratio test. To test heterogeneity by stage and by subtype, we used the “eh_test_subtype” function in the R package “riskclustr” [28]. This function is designed for the test of etiologic heterogeneity across disease subtypes in the context of the case–control study. P-heterogeneity < 0.05 was considered statistically significant. To examine the robustness of the results, we conducted a series of sensitivity analyses: (1) we recategorized the smoking variables into 7 categories (never, current smoker for < 50 years, current smoker for ≥ 50 years, former smoker quit < 10 years ago, former smoker quit 10–20 years ago, former smoker quit ≥ 20 years ago, unknown) based on smoking status and duration of time and adjusted it in the main analysis; (2) we further adjusted for cohort (CPS-II Nutrition, CPS-3) in the analysis to evaluate if any differences between cohorts (e.g., age of blood samples) would affect the main analysis results; (3) we also examined the associations stratified by years between blood draw and lung cancer diagnosis (< 5 years, ≥ 5 years). All analyses were conducted using R (version 4.1.0.).',\n",
       " 11197204: 'MethodsThe study utilised 6 publicly available bulk and single-cell transcriptomic datasets from human and mice studies downloaded from Gene Expression Omnibus (GEO). Machine learning models were employed to model and statistically test datasets for conserved gene expression profiles. Identified genes were validated in OA tissues from obese and healthy weight individuals using quantitative PCR method (N = 38). Obese and healthy-weight patients were categorised by BMI > 30 and BMI between 18 and 24.9 respectively. Informed consent was obtained from all study participants who were scheduled to undergo elective arthroplasty. Methods Study recruitment UK National Research Ethics Committee (NRES 16/SS/0172) granted ethical approval for post-operative collection of joint tissue from consenting patients with OA. Study recruitment was coordinated by research nurses at the Royal Orthopaedic Hospital, Birmingham (United Kingdom) and Russell’s Hall Hospital, Dudley (United Kingdom). Differential gene expression analysis Genes are commonly assessed according to their fold change, which represents the extent of expression variations, and by their respective p-values, which measure the statistical significance of such variations. Genes with significant fold changes (fold changes ± 1.5) and low p-values (p < 0.05) are considered statistically significant differentially expressed. Multiple testing correction was performed to control for false positives that may arise due to testing multiple genes simultaneously, and multiple testing correction methods (BH false discovery rate correction) were applied to adjust p-values (adj p < 0.05). Datasets All the datasets were obtained from Gene Expression Omnibus (GEO) on 29th October 2022. Those data sets are from both human and mouse samples. A detailed summary of each dataset can be found in Table 1.Table 1List of the public data sets, organism and tissue sources used in this studyGEO dataOrganismType of datasetNumber of participantsTissue sourceLeanObeseTotalGSE24883HumanArray323264Subcutaneous and visceral adipose tissueGSE59034HumanArray161632Subcutaneous and visceral adipose tissueGSE49195MouseArray6612Lean and obese LiverGSE39375MouseArray5510Lean and obese LiverGSE152815HumanSingle Cell RNA sequencing448Hand, hip, knee, and foot JointsGSE219027HumanBulk RNA sequencing242448Hand, hip, knee, and foot Joints Patients and tissue samples Obese and lean-weight patients, categorised by BMI > 30 and BMI between 18 and 24.9 respectively, were recruited for the study. These patients were scheduled to undergo elective arthroplasty for OA and synovium was collected from hip or knee joints (N = 38). Ethics approval was provided by the UK National Research Ethics Committee (approval no. 14/ES/1044), and informed consent was obtained from all patients. The characteristics of study participants are summarised in Table 2.Table 2Summary of participant characteristics. Data displayed as mean ± standard deviationLeanN = 16ObeseN = 22P-valueAge in years69.1 ± 2.666.7 ± 1.80.4382Gender (Number of female participats, (%))12 (75%)12 (55%)0.2071Hip: Knee Ratio (Number of hip joints, (%))12 (75%)12 (55%)0.0717BMI22.8 ± 0.435.9 ± 1.3 < 0.0001Waist: Hip Ratio0.8 ± 0.020.9 ± 0.010.0007N Number of participants Quantitative real-time PCR validations of identified genes Total RNA was isolated from synovium using TRIzol (Life Technologies). Primers for individual transcripts of interest (see Supplementary Table 1) were either predesigned TaqMan™ Gene Expression Assay (FAM) from Life Technologies Ltd or designed using NCBI’s Primer-BLAST tool and Easy Oligos were ordered from Merck Life Science UK Ltd. PCR was performed from total RNA in a one-step reaction using either iTaq Universal One-Step or iTaq™ Universal SYBR® Green One-Step Kit from BioRad. Relative expression was determined using the ΔΔCt method, followed by normalization of values to those of 18S and GAPDH. Statistical and machine learning methods Multiple statistical and machine learning models were used to identify genes differentiating between obesity and lean samples (DEG summary in Supplementary Table 1). Unsupervised Principal Component Analysis (PCA), a dimensionality reduction technique, was used in machine learning and data analysis. PCA seeks to preserve the underlying structure and relationships in the data while reducing its dimensionality, making it easier to visualize and analyse. Logistic regression was used to evaluate how well different classes could be distinguished. To estimate sample size, an inhouse developed tool known as Tools was used [20]. qPCR data was plotted using GraphPad PRISM with unpaired student’s t-test. Pathway enrichment analysis Pathway enrichment analysis was performed using the Enrichr tool to determine functional mechanisms informed by differentially expressed genes derived from the datasets analysed. This approach identifies biological pathways that show enrichment in a gene list beyond what would be expected by random chance. We have used ShinyGO [21] and Enrichr [22] for this analysis. The analysis workflow is illustrated in Fig. 1. Fig. 1Overview of the gene target identification workflow using public and internal RNA sequencing data Statistical and machine learning methods Multiple statistical and machine learning models were used to identify genes differentiating between obesity and lean samples (DEG summary in Supplementary Table 1). Unsupervised Principal Component Analysis (PCA), a dimensionality reduction technique, was used in machine learning and data analysis. PCA seeks to preserve the underlying structure and relationships in the data while reducing its dimensionality, making it easier to visualize and analyse. Logistic regression was used to evaluate how well different classes could be distinguished. To estimate sample size, an inhouse developed tool known as Tools was used [20]. qPCR data was plotted using GraphPad PRISM with unpaired student’s t-test.',\n",
       " 11197185: 'MethodsThe current study aimed at analyzing and comparing the microbial profile of Aedes albopictus saliva and midgut as well as assessing the impact of Zika virus (ZIKV) infection on the midgut and saliva microbial composition. Colony-reared Ae. albopictus strains were either exposed to ZIKV infectious or noninfectious bloodmeal. At 14 ays postinfection, the 16S V3–V4 hypervariable rRNA region was amplified from midgut and saliva samples and sequenced on an Illumina MiSeq platform. The relative abundance and diversity of midgut and saliva microbial taxa were assessed. Methods Virus The ZIKV strain utilized in this study was HND (2016-19563, GenBank accession no. KX906952). The Zika virus isolate used in this study was obtained from an infected patient serum. In the lab, virus stocks were created by passaging three times on Vero cell culture (ATCC) and once on C636 cell culture (ATCC). Stocks were frozen at −70 °C before use. Insect sampling and sample preparation Ae. albopictus mosquitoes were collected from Suffolk County, New York State (NYS) in 2015 (kindly provided by I. Rochlin) and colonized at the New York State Department of Health (NYSDOH) Arbovirus Laboratory. The F15 generation Ae. albopictus eggs were vacuum hatched. The larvae that emerged from the hatched eggs were maintained in plastic rectangular flat containers (35.6 cm length × 27.9 cm width × 8.3 cm height; Sterilite, catalog no. 1963) at a density of one larva per 5 ml of dechlorinated water and reared at 60% relative humidity and a light–dark (LD) photoperiod of 14:10 h. The larvae were fed 1.25 mg/larvae of Tetra Pond Koi growth feed for first and second instar larvae and 2.5 mg/larvae for third and fourth instar larvae [37]. Upon emergence, the male and female adults were transferred to 3.8-l containers and housed together for 8 days at 28 °C at a relative humidity of 60% while being provided with sugar and water ad libitum [37]. To stimulate blood feeding, females were starved of water and sugar for 12 h before infectious blood meal according to our insectary laboratory standard operating procedures. The blood meal consisted of either 1:1 dilution of defibrinated sheep blood plus 2.5% sucrose, sodium bicarbonate (to adjust pH to 8.0) and virus, or a non-infectious blood meal containing a final concentration of 2.5% sucrose solution. Infectious blood meals contained 8.3 log10 PFU/ml ZIKV HND [38]. The female mosquitoes were exposed to blood meals in a 37 °C preheated Hemotek membrane feeding system (Discovery Workshops, Accrington, UK) with a porcine sausage casing membrane. After an hour, the mosquitoes were anesthetized with CO2 and immobilized on a prechilled tray connected to 100% CO2 Engorged females were separated and placed in separate 0.6 l cardboard cartons (30 individuals per carton). Blood-fed females were maintained on a 10% sucrose solution provided ad libitum. At 14 days postinfection (dpi), the female mosquitoes were immobilized using triethylamine (Sigma Aldrich, St. Louis, MO, USA). The mosquitoes were surface sterilized by dipping twice in 70% ethanol. To examine for ZIKV dissemination, the legs were removed from the mosquitoes and placed in individual tubes. Saliva was collected under sterile conditions by inserting the proboscis of the surface-sterilized female into a capillary tube containing ~ 20 µl filter sterilized fetal bovine serum plus 50% sucrose 1:1 for 30 min before subsequently ejecting the mixture into filter-sterilized 125 µl Mosquito Diluent [MD; 20% heat-inactivated fetal bovine serum in Dulbecco phosphate-buffered saline plus 50 µg/ml penicillin/streptomycin, 50 µg/ml gentamicin and 2 µg/ml Fungizone (Sigma Aldrich, St. Louis, MO, USA)]. As described in [39], briefly, to dissect the salivary glands, the tip of the abdomen was removed with a sharp cut using a dissecting probe. Placing a probe in the thorax, the head was removed gently by pulling away from the thorax. The salivary glands were then detached from the head using the probe, rinsed twice in sterilized phosphate-buffered Saline (PBS) (Sigma Aldrich, St. Louis, MO, USA), and transferred to a sterile microcentrifuge tube and stored at −80 °C. The midgut was pulled from the tip of the abdomen and rinsed twice in sterile PBS before being transferred to sterile microcentrifuge tubes and stored at −80 °C until tested. The individual mosquito carcasses were then transferred into individual tubes sterile microcentrifuge tubes. All samples were held at −80 °C until assayed. Sequencing and analysis of microbiome of Aedes mosquito saliva and midguts A ZIKV-specific quantitative PCR assay that targets the NS1 region [40] was utilized to obtain viral titer from the legs and saliva as described by [38]. The individuals whose midguts and saliva samples were infected with the virus were identified. ZIKV-infected saliva and midgut samples as well as those exposed to a naïve blood meal were used for downstream analysis. We experienced challenges in obtaining enough genomic DNA material from singleton saliva samples and hence to increase the amount of genomic material for microbiome study, we adopted a whole transcriptome amplification approach. RNA was isolated from the saliva and midgut samples using Trizol (Thermo Fisher Scientific, Waltham, MA, USA). The saliva and midgut RNA samples were reverse transcribed, followed by whole transcriptome amplification according to the REPLI-g whole transcriptome amplification (WTA) single-cell kit protocol (Qiagen, Hilden, Germany). The resulting amplified complementary DNA (cDNA) was diluted 1:100. Thereafter, the 16S V3–V4 hypervariable region was amplified using an Illumina barcoded 16S primer set [16S_341F (TCG TCG GCA GCG TCA GAT GTG TAT AAG AGA CAG CCT ACG GGN GGC WGC AG) and 16S_805R(GTC TCG TGG GCT CGG AGA TCT GTA TAA GAG ACA GGA CTA CHV GGG TAT CTA ATC C)] [36]. PCR reactions were carried out in a total volume of 50 µl, 5 µM each of 16S ribosomal RNA (rRNA) primer pair, 2 µl WTA cDNA, 8 µl deionized filter-sterilized water, and 36 µl AccuStart II PCR supermix (Quantabio, Beverly, MA, USA). We included two negative controls at the PCR amplification step: water and nontemplate control. In addition, a positive spike at the sequencing step. A fragment size of ~ 460 bp of each sample was submitted to the Wadsworth Center Applied Genomics Core for sequencing together with the negative controls. Automated cluster generation and paired-end sequencing (250-bp reads) was performed on the Illumina MiSeq 500 cycle. A total of 30 samples [five infected midguts (INF MG), six noninfected midguts (NINF MG), ten infected saliva (INF SAL), six noninfected saliva (NINF SAL), one negative control (NC), one nontemplate control (NTC), and one positive spike (PC)] were included in this study. Analysis of the data was carried out on QIITA (https://qiita.ucsd.edu/) and MicrobiomeAnalyst [41]. In summary, the command split libraries FASTQ was used to convert the de-multiplexed FASTQ files to the format used by QIITA. This was followed by clustering of the sequences into operational taxonomic units (OTUs), which was done by the utilization of a closed-reference OTU picking based on the GreenGenes 16S reference database. The choice of closed reference was due to its fully parallelizable process, which makes it a computationally efficient process. We neither elected to utilize open reference nor de novo OTU picking due to the computationally complex strategies that make them unsuitable for large data sets. Open-reference OTU picking reduces read loss by de novo clustering reads that do not match the reference sequences; some of the steps of the workflow run serially and hence slows down the process. De novo OTU picking, on the other hand, clusters reads against one another without an external reference collection, is computationally complex, results in large data set loss, and can reduce quantification accuracy while reducing the potential to detect rare taxa [42]. A 97% sequence identity threshold was utilized to assign taxonomy to the sequence. A BIOM-formatted OTU table was subsequently generated. To estimate alpha diversity metrics, we adjusted for differences in library sizes across samples using rarefaction and generated rarefaction curves to assess whether the sequencing depth was sufficient. Rarefaction involves picking a specified number of samples equal to or less than the number of samples in the smallest sample followed by a random removal of reads from larger samples until the number of remaining samples is equal to this threshold [43]. The reads have been deposited in NCBI GenBank Short Read Archives. We anticipated the presence of rare taxa in our microbial community. To accommodate the rare taxa while estimating diversity, we utilized Chao1 diversity measure at the family taxonomic level to assess species richness of INF and NINF saliva and INF and NINF MG. Statistical significance was tested using a t-test assuming a normal distribution of values. Chao1 index was used to estimate alpha diversity because in estimating the diversity it makes inferences about rare or difficult-to-detect species [44]. We compared these results with other ecological indices used to test for alpha diversity such as the Shannon diversity and Simpson index. The Shannon diversity index considers both the richness and evenness of the microbial species present. It calculates this by utilizing the proportion of species i relative to the total number of species (pi) and multiplying by the natural logarithm of the proportion (lnpi) [45]. The Simpson Index tests for species dominance by considering the total number of species present as well as the relative abundance of each species. In this diversity index, species richness and evenness are directly proportional to diversity [46]. To compare the microbial community composition between the global INF and NINF saliva and INF and NINF MG, beta diversity was measured utilizing the Bray–Curtis dissimilarity statistic at the family taxonomic level [47]. The beta diversity measurement utilized the permutational analysis of variance (PERMANOVA) test to estimate the differences in microbial communities. PERMANOVA is a nonparametric multivariate statistical permutational test measured through a geometric partitioning of multivariate variation of a dissimilarity measure according to a given ANOVA design [48]. The distance matrix generated after comparing each sample to the other was visualized for the dissimilarity between samples using the principal coordinate analysis ordination method. Biomarker analysis Random forest algorithm [49] within Microbiome Analyst [41] was utilized to identify taxa associated with the different infection statuses of saliva and midgut. Random forest algorithm [49] is a directed classification algorithm of trees generated by bootstrapping samples while training data and random feature selection in tree induction. It is an ensemble of unpruned classification or regression trees trained with the bagging method [50]. The default setting of the number of trees to grow and the number of predictors to try was applied (500 and 7, respectively) with the randomness setting left on. Further, we tried to adjust the parameters to different settings, but we did not observe any improvement in the model performance. We assessed the Out-of-Bag (OOB) error rate of the model.',\n",
       " 11196966: 'MethodsCRC organoids were implanted into the cecal walls of wild type mice, and animals were screened for liver metastasis. At the time of metastasis, 1/3 partial hepatectomy was performed and the tumor burden was assessed longitudinally using MRI. After euthanasia, different tissues were analyzed for immunological and transcriptional changes using FACS, qPCR, RNA sequencing, and immunohistochemistry. 2 Methods 2.1 Organoid culture APTAK organoids were kindly gifted by Florian Greten, Georg Speyer Haus, Frankfurt. These cells are murine colorectal carcinoma cells holding the following mutational features: Deletions in the Apc-, Trp53- and Tgfbr2-gene, a myristoylated Akt-1 with constitutively activated signaling and a Kras G12D-mutation (16). The APTAK organoids were maintained in Advanced DMEM-F12 (Gibco/Thermo Fisher Scientific, Waltham, MA, USA) supplemented with penicillin (100 U/ml)/streptomycin (100 µg/ml) (Gibco/Thermo Fisher Scientific), HEPES 10 mmol/L (Invitrogen/Thermo Fisher Scientific), Glutamax 2 mM (Invitrogen/Thermo Fisher Scientific), N2 1× (Gibco/Thermo Fisher Scientific), B27 1× (Gibco/Thermo Fisher Scientific), and N-acetylcysteine 1 mmol/L (Sigma-Aldrich, St. Louis, MO, USA) and supplemented with hygromycin (200 µg/ml, Invitrogen/Thermo Fisher Scientific) and puromycin (2 µg/ml, InvivoGen, San Diego, CA, USA) (hereafter referred to as basal medium). The organoids were split every 3–5 days at a 1:10 ratio and cryoconserved for long-term storage. 2.2 Animal studies All animal studies were performed using 8–12 weeks old C57Bl6/J-mice (purchased from Charles River, Wilmington, MA, USA) at the pathogen-free barrier facility of the CEMT, University Medical Center Freiburg. The experiments were planned and carried out in accordance with the institutional guidelines and all experiments were approved by the regional board (Regierungspräsidium Freiburg, G21/001). 2.3 Orthotopic organoid transplantation mouse model We adapted the subserosal orthotopic organoid implantation technique described by Fumagalli et al. (17). APTAK organoids cultured in Cultrex RGF basement membrane extract (BME) Type 2 (R&D Systems, Minneapolis, MN, USA) were dissociated into single cells by mechanical disruption and washed with cold PBS. After determining the cell number with the cell counter TC20 (BioRad, Hercules, CA, USA), the appropriate cell number was resuspended in collagen I/5x neutralization buffer (vol/vol-ratio 1:4), and collagen domes of 100.000 cells/10 µl were plated on a 6-well plate. After 30 min polymerization at 37°C, basal medium was added and cells were incubated over night until implantation. Mice analgesia was performed with intraperitoneal (i.p.) injection of 0.1 mg/kg bodyweight (BW) buprenorphine (Temgesic, Schering-Plough, Kenilworth, NJ, USA). Anesthesia was induced by 2.5 Vol% Isoflurane (Isoflurane Piramal, Piramal, Mumbai, India) with the SomnoSuite® Low-Flow Anesthesia System (Kent Scientific Corporation, Torrington, CT, USA). The unconscious mice were positioned on a heating pad with all legs fixed and the abdomen was shaved while anesthesia was maintained by continuous application of isoflurane via mouthpiece. After disinfection of the skin, a median laparotomy of approximately 1 cm was performed on the lower abdomen. The cecum was mobilized using two cotton swabs and placed on a wet gauze. Under the microscope, a 2 mm incision of the serosal layer was made and a deep subserosal pocket was created in which the organoid-collagen-graft was fully placed. After covering the graft with the serosal layer, an anti-adhesion layer (Seprafilm, Baxter, Deerfield, IL, USA) was placed on top. The cecum was then replaced in the abdominal cavity and the abdominal wall and skin were closed in two layers using 6–0 Vicryl (Ethicon/Johnson&Johnson, New Brunswick, NJ, USA) suture. The duration of the surgical procedure was approximately 10–15 minutes. 2.4 Magnetic resonance imaging MR imaging was performed using a 9.4 tesla small bore animal scanner (BioSpec 94/21, Bruker Biospin, Karlsruhe Germany) and a dedicated mouse quadrature-resonator (Bruker). The mice were anesthetized under spontaneous breathing conditions using Isoflurane. Heart and respiration rates were continuously monitored and maintained at a constant level. Gating was used to reduce motion and blood flow artifacts during the scan. The MRI protocol consisted of a localizer and a T2-weighted spin echo RARE (Rapid Acquisition with Relaxation Enhancement) sequence. This sequence was performed to delineate the tumor and eventual metastasis from the surrounding healthy tissue. The RARE sequence in axial orientation featured a field-of-view (FOV) of 28 mm (1), a matrix size of 280 x 280 pixels, and an in-plane resolution of 100 x 100 µm (1). The slice thickness was 0.75 mm (TR/TEeff/FA: 300 ms/36 ms/180°). The number of slices was adjusted to the measured volume (on average 30) to ensure complete coverage of the abdomen. The Bruker ParaVision 6.0.1 software was used for imaging and images were analyzed using the NORA software supplied by the Medical Center of the University of Freiburg. The volumes of all hepatic lobes and metastases were assessed individually by manually marking the respective lobes/LMs as regions of interest (ROIs) in each section. 2.5 Ultrasound During isoflurane anesthesia mice were placed on a heating pad and the abdomen was shaved. Prewarmed ultrasound gel was applied to the abdomen and a VEVO 3100 (Fujifilm/VisualSonics, Toronto, Canada) ultrasound together with a small animal transducer was used for ultrasound imaging. 2.6 Partial hepatectomy For distribution among the control-, SHAM-, and pHx-groups, we assessed the preoperative hepatic tumor load using MRI. Animals displaying an approximately similar number and volume of LM in the imaging were allocated equally among the different protocols to reduce the influence of cage-specific phenotypes. After general analgesia and anesthesia with buprenorphine and isoflurane as described above, mice were shaved and placed on a heating pad. After disinfection, a median laparotomy of approximately 2 cm, starting at the xiphoid, was performed and retractors were placed on both sides of the abdominal wall and under the xiphoid. The falciforme ligament and the membrane between the medial and left hepatic lobes were dissected. A 3–0 Vicryl (Ethicon/Johnson&Johnson) suture was placed around the base of the medial lobe and tightened with 3 knots, after which the ischemic lobe was cut approximately 0.1 cm distal to the suture ( Supplementary Figure 1A ). The resected lobe was placed in cooled PBS Buffer on ice until further analysis. The abdominal cavity was flushed with prewarmed saline, and the abdominal wall and skin were closed using 6–0 Vicryl (Ethicon/Johnson&Johnson) stitches. The pHx procedure took approximately 15 min. SHAM mice underwent laparotomy, including dissection of the falciforme ligament and the membrane between the medial and left lobes without resection of the lobe. Control animals received general anesthesia without any surgical procedures. 2.7 Tumor dissociation Animals were euthanized by cervical dislocation. Subsequently, the abdomen was opened and each lobe of the liver was resected directly at the vena cava. The lobes were weighed and all visible liver metastases were dissected while the organ was sliced. The entire cecum was resected and the tumor tissue was separated from the healthy intestine. We collected samples for RNA isolation (stored in RNAprotect buffer (Qiagen, Netherlands)) and for protein isolation from all tissues mentioned. Healthy liver tissue was minced and dissociated using the Liver Dissociation Kit, mouse (Miltenyi Biotec, Bergisch Gladbach, Germany) according to the manufacturer’s protocol. The primary tumor and LM were minced into <1 mm pieces and dissected using the Tumor Dissociation Kit, mouse (Miltenyi Biotec) according to the manufacturer’s instructions. The single cell suspension was filtered through a 100 µm cell strainer and ACK lysis buffer (Gibco/Thermo Fisher Scientific) was added to lyse erythrocytes during a 1.5 minutes incubation. 2.8 Flow cytometry Cells were transferred to 100 µl FACS buffer (PBS + 0,5% BSA), incubated with TruStain FcX PLUS (anti-mouse CD16/32) antibody (BioLegend, San Diego, CA, USA) for 15 min and then stained with the respective fluorescent antibodies for 60 min. Cells were then washed with FACS buffer (for extracellular staining) and fixed with 1x Fixation/Permeabilization buffer for 60 min. For intracellular staining, we used the eBioscience™ Foxp3/Transcription Factor Staining Buffer Set (Invitrogen/Thermo Fisher Scientific) according to the manufacturer’s instructions. All FACS antibodies used were supplied by BioLegend and used undiluted with 1 µl/100 µl cell suspension. FACS was performed using Beckman Coulter Gallios (Beckman Coulter, Brea, CA, USA) or BD LSRFortessa (Becton Dickinson, Franklin Lakes, NJ, USA). Data were analyzed using Kaluza Software (Beckman Coulter). Cellular aggregates were excluded using SSC-W. 2.9 T cell stimulation Equal amounts of single cells from the tumor/liver dissociation were seeded in RPMI-Medium (+ 1% Pen/Strep + 1% HEPES + 5 µl beta-mercaptoethanol) on a 12-well plate overnight. The cells were then stimulated by adding Cell Activation Cocktail (BioLegend) for 3 hours, followed by addition of Brefeldin A (BioLegend) for 2 hours. The cells were then harvested and stained for FACS analysis, as described above. 2.10 RNA isolation and cDNA synthesis Tissue samples for RNA isolation were stored in RNAprotect Buffer. The tissue was shredded in 350 µl RLT buffer (Qiagen, Venlo, Netherlands) + 1% beta-mercaptoethanol. Further RNA isolation was carried out using the RNeasy Mini Kit (Qiagen) according to the manufacturer’s instructions. RNA concentrations were measured using a NanoDrop™ spectrophotometer (Thermo Fisher Scientific). For cDNA synthesis RNA was transcribed using ProtoScript II First Strand cDNA Synthesis Kit (New England Biolabs, Ipswich, MA USA) according to the manufacturer’s instructions. cDNA was finally diluted to 3 ng/µl for qPCR experiments. 2.11 DNA isolation For quantification of bacterial DNA, we isolated DNA from tissue samples using the DNeasy Blood & Tissue Kit (Qiagen) according to the manufacturer’s instructions, and concentrations were assessed using a NanoDrop™ spectrophotometer (Thermo Fisher Scientific). 2.12 Quantitative real-time PCR Quantitative Real-Time PCR was performed using QuantiTect SYBR Green RT PCR mastermix (Qiagen) with 7.5 ng template cDNA and 5 pmol of each primer (purchased from Eurofins Genomics, Ebersberg, Germany; primer sequences supplied the supplements). qPCR was carried out on a Roche LightCycler 480 (Roche, Basel, Switzerland) with each sample-gene combination in triplicates. The protocol consisted of a 10 min activation step at 95°C followed by 50 cycles of 45 s amplification at 60°C and inactivation at 95°C for 15 s. 18S was used as a housekeeping gene, Cp values were calculated using the LightCycler480 software (Roche), and the relative expression of target genes was calculated by comparative method after normalization to 18S-expression. 2.13 Bacterial 16S qPCR In order to quantify bacterial infiltration, we performed qPCR with amplification of the bacterial V6 region using five different primers (see Supplements). Each sample was measured in triplicates using the QuantiFast SYBR® Green PCR Kit (Qiagen). 100 ng DNA was mixed with 500 nM primers. By diluting the DNA of Bifidobacterium actinocoloniiforme (DSMZ-Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, DSM22766) in 10-fold series we generated a standard curve using the Threshold cycle (Ct) values. To quantify the sample bacterial DNA, we correlated the average Ct-values to the standard curve of log values from the Bifidobacterium actinocoloniforme. 2.14 Bulk RNA sequencing RNA sequencing was performed at Genewiz, Germany, using the standard Illumina protocol to create raw sequence files (.fastq files). Poly(A) mRNA was used to prepare cDNA libraries, which were then sequenced as 2x 150 bp paired end using the Illumina NovaSeq instrument (Illumina, San Diego, CA, USA). 2.15 RNA seq analysis Raw sequencing files were uploaded to the galaxy web platform (https://usegalaxy.eu) (18) and quality was assessed using FASTQC. Owing to good quality and no adaptor content, trimming was not necessary. Data were mapped to the murine reference genome (GRCm39) provided by GENCODE (www.gencodegenes.org, downloaded September 2021) using the STAR aligner (19) in standard settings for paired sequencing data. Reads were assigned to counts using featureCounts software (20) and the gene annotations also provided by GENCODEs (downloaded September 2021). Paired-end settings were used for featureCounts software, and a quality score of >10 was required for counting. Data were imported in R 4.0.2 (https://www.r-project.org), and differentially expressed gene analyses were conducted using DESeq2 (21). A batch effect was removed using the limma::removeBatchEffect() function (22). Differentially expressed genes were defined as padj <0.05 and abs(log2 FC) >1. The Ensembl gene ID was converted to MGI symbols using the biomaRt package (23). The fgsea package (24) conducted Gene Set Enrichment Analysis (GSEA) by ranking all genes with at least >10 counts according to the log2 FC, which were previously shrunken by the apeglm logarithm (25) before. Gene sets were accessed from MsigDB (26) and downloaded on the 26th of May 2023. For visualization, ggplot2 (27), ggraph and ComplexHeatmaps (28) were used. 2.16 Protein isolation Tissue samples were shredded in RIPA buffer containing phosphatase and proteinase inhibitors (Phosstop (Roche), protease inhibitor cocktail, sodium orthovanadate, and PMSF (Santa Cruz, Santa Cruz, CA, USA) on ice. Three cycles of freezing and thawing were applied using liquid nitrogen, and the suspension was centrifuged at maximum speed for 10 min. The supernatant containing the protein lysate was transferred to a new tube. Protein concentration was measured using a Pierce BCA-Kit (Thermo Fisher Scientific) and an Azure Ao Microplate Reader (Azure Biosystems, Dublin, CA; USA). 2.17 Western blot Protein samples were aliquoted with 4x Laemmli Buffer (BioRad) and denatured at 95°C for 3 min. An equal amount of protein was loaded on a 12–20% Mini-Protean TGX precast gel (BioRad) together with a color-coded prestained broad range protein ladder (Cell Signaling Technology, Danvers, MA, USA). After SDS-electrophoresis, the proteins were blotted onto a PVDF membrane using the trans-blot turbo transfer system (BioRad) according to the manufacturer’s instructions. The membrane was then blocked with 5% BSA/TBST for 60 min at RT and incubated with the respective primary (Anti-S100A8, Anti-MMP9, Anti-Claudin-2, Anti-Occludin, and Anti-ZO-1) and secondary (anti-rabbit IgG) antibodies according to the manufacturer’s instructions (a detailed description of the antibodies used is supplied in the Supplements). For detection, the Immobilon Crescendo Western HRP substrate (Merck Millipore, Burlington, MA, USA) was added to the membrane and imaging was carried out using a ChemiDoc BioRad imager (BioRad). 2.18 Immunohistochemistry Tissue samples were fixed in 4% PFA for 24 hours, dehydrated and embedded in paraffin. Sections of 3 µm were cut, rehydrated, and heat induced epitope retrieval was carried out with citrate or TE-Buffer (Zytomed Systems, Berlin, Germany). Sections were then blocked with 2.5% BSA and avidin-biotin-block (Zytomed Systems) according to the manufacturer’s instructions. Sections were then incubated with the primary antibody overnight and biotinylated secondary antibody for 1 h according to the manufacturer’s instructions (see a detailed description of the antibodies used in Supplements). Positive staining was visualized using DAB (Zytomed Systems), and the sections were counterstained with hematoxylin before dehydration and mounting with ROTI® Histokitt (Carl Roth, Karlsruhe, Germany). Slides were digitalized using a Zeiss Axio Scan Z.1 Microscope Slide Scanner (Zeiss, Jena, Germany). 2.19 Immunofluorescence Slides were treated until secondary antibody incubation as described in the Immunohistochemistry section, and then incubated with the Vector® TrueVIEW® Autofluorescence Quenching Kit (Biozol, Eching, Germany) to reduce autofluorescence. After washing, nuclear staining was performed using NucBlue™ Fixed Cell Stain ReadyProbes™ (Thermo Fisher Scientific). Sections were then mounted with VECTASHIELD Vibrance Antifade Mounting Medium (Biozol) and scanned using a Zeiss Axio Scan Z.1 Microscope Slide Scanner (Zeiss). 2.20 Statistics For statistical analysis, we used a one-way analysis of variance (ANOVA) test with Tukey’s post-hoc analysis when comparing multiple samples or a two-tailed Students-t-test for two variables. Statistical significance was set at p-values <0.05, and significance levels are represented in the graphic as follows: *** = p<0.001, ** = p<0.01, * = p<0.05, n.s. = not significant. All statistical analyses were performed using GraphPad Prism.',\n",
       " 11196847: '2 Materials and methods 2.1 Sample collection The macrofungal strain fruiting body was collected from the Neelkantha forest, Rishikesh, Uttarakhand, situated at an altitude of 6,596 meters, latitude 30°43′48′′N, and longitude 79°24′20′′E. 2.2 Establishment of tissue culture and submerged cultivation of Trametes sp. Fruiting bodies were cleaned, and the inner mass of the mushroom was removed with a sterilized knife and cultivated for 14 days at 20°C in a suitable potato dextrose agar (PDA) medium. A pure culture was obtained and sustained after additional tissue culture was established and multiple subcultures were carried out to purify the culture. After generating further sub-cultures from the pure culture medium, the biomass was grown in a liquid medium (potato dextrose broth; PDB) in a 250-mL volumetric flask and incubated for 14 days at 20°C in the dark. This step was done in batches. Following that, biomass was filtered and dried at 50°C for 2 h. Using a pestle and mortar, the dried biomass was ground into a fine powder. The resulting powder was then used and prepared for analysis. The pure culture was stored in a suitable potato dextrose agar medium at 4°C and was subcultured every 30 days onto the fresh medium. 2.3 Amplification of the LSU F/R-rDNA fragment and phylogenetic analysis Genomic DNA was extracted from cultured samples by a simple lysis protocol, as described in Karn et al. (2011). In total, 125 ηg of extracted DNA is used for amplification, along with 10 pM of each primer. LSU F/R-rDNA fragment was amplified using the ABI 3130 Genetic Analyzer with the following set of primers: forward primer: 5’-TCCTGAGGGAAACTTCG-3’ and reverse primer: 5’-ACCCGCTGAACTTAAGC-3’. The reaction mixture of PCR contained 10x Taq DNA polymerase Assay Buffer, 2.5 mM each dNTP, 3.2 mM MgCl2, and 3 U/ml Taq DNA Polymerase in a final volume of 30 ml sterile MQ water. The PCR was performed with an initial denaturation at 94°C for 3 min, followed by 30 cycles of 94°C for 1 min, 50°C for 1 min, and 72°C for 2 min, and a final extension of 72°C for 7 min. The amplification product was gel-purified using ABI 3130xl gel sequencing. The PCR product was sequenced bi-directionally. The sequence data were aligned and compared against the available DNA sequences from type strains in GenBank (http://www.ncbi.nlm.nih.gov/) using the BLAST sequence match tool and analyzed to identify the mushroom and its closest neighbors joining by making a phylogenetic tree using the software Phylogenetic Tree Builder (Bruno et al., 2000). 2.4 Preparation of mycelial biomass extract and broth extract Ethanol (80%, v/v) was used as an extracting solvent for obtaining MBE and BE. The dry biomass (10g) was ground and mixed with 100 ml of 80% ethanol, then left in a laboratory shaker at 25°C and 140 rpm for 24 h. After that, extracts were separated from the mycelium biomass by centrifugation. The process was then carried out once more after the solid had been redissolved in ethanol. Next, Whatman paper No. 4 was used to combine and filter the liquid. In a vacuum evaporator, the filtrate was concentrated, and the thick concentrated extract was collected and used for phytochemical and polysaccharide quantification research. 2.5 Phytochemical analysis 2.5.1 Total protein analysis The total protein content in MBE and BE was determined using Lowry’s assay. An aliquot of 1 mL of extract is treated with 5 mL of Reagent A and kept in the dark for 10 min. Then the mixture was loaded with 0.5 mL of Folin-Ciocalteu reagent and left in a dark place for 30 min. After incubation, the absorbance was measured at 630nm. Protein concentration (mg/mL) was determined using the Bovine Serum Albumin (BSA) standard curve in the extracted samples. Reagent A is made by combining 99 mL of reagent B (2% Na2CO3 and 0.1N NaOH), 0.5 mL of reagent C (2% of copper sulfate pentahydrate), and 0.5 mL of reagent D (2% of potassium sodium tartrate). 2.5.2 Total carbohydrate analysis The total carbohydrate content of extracted samples was determined using the Anthrone technique, with glucose (10–80 mg/L) used as the standard. For the test, 1 mL of extract is treated with 5 mL of the anthrone reagent (dissolve 2g of anthrone in 1000 mL of concentrated sulfuric acid) and incubated at 100°C for 10 min. After incubation, the absorbance was measured at 620nm. Carbohydrate concentration (mg/mL) was determined using the glucose standard curve in the extracted samples. 2.5.3 Total flavonoid analysis The total flavonoid content (TFC) of fungal MBE and BE extracts was evaluated according to the method described by Wandati et al. (2013). An aliquot of 1 mL of the sample was added to 0.3 mL of 5% NaNO2, and 4 mL of distilled water was kept for 5 min. Then the mixture was loaded with 0.3 mL of 10% AlCl3 and allowed to sit for 6 min. Finally, 2 mL of 1 M NaOH was added to these tubes, followed by 10 ml of distilled water. The absorbance was measured at 415nm. Quercetin was used as a standard in a concentration range of 20–100 µg/mL, and the result was expressed as µg QE/g DW. 2.5.4 Total phenol analysis The total phenol content (TPC) of the fungal MBE and BE extracts was analyzed using the Folin-Ciocalteu method of Wandati et al. (2013), with some modifications. Each sample (1 mL) was mixed with 0.5 mL of 10% (v/v) Folin-Ciocalteu phenol reagent and 2 mL of 700 mM Na2CO3 in distilled H2O. The mixture was vortexed well and left for 30 min. After incubation, the absorbance was measured at 765nm. The TPC in the extract was expressed as μg gallic acid equivalent (GAE) per g dry weight (μg GAE/g DW). 2.5.5 Total saponin analysis The total amount of saponin in fungal MBE and BE extracts was analyzed as described by Le et al. (2018). An aliquot of 1 mL of the sample was added to 1 mL of 8% vanillin and 2.5 mL of 72% sulfuric acid. The mixture was incubated for 15 min at 60°C. After incubation, the absorbance was measured at 560nm. Aescin was used as a standard in a range of 20–100 µg/mL, and the result was expressed as mg Aescin/g DW. 2.5.6 Total anthraquinone analysis The total anthraquinone content of fungal MBE and BE extracts was evaluated according to the method described by Sakulpanich and Gritsanapan (2008), with some modifications. In 1 mL of sample, 10 mL of distilled water was added. Then, 20 mL of 10.5% ferric chloride was added, and the mixture was mixed and refluxed for 10 min. Add 1 mL of sulfuric acid and reflux for another 10 min. Transfer the final content into a separating funnel, and extraction was done three times with 25 mL of ether. After that, the ether layer was evaporated to dryness to collect the residues. The residues were dissolved in 1 ml of 0.5% Mg(CH3COO)2 in CH3OH, and absorbance was measured at 515 nm. Rhein was used as a standard in a range of 20–100 µg/ml, and the results were expressed as µg Rhein/g DW. 2.5.7 Total alkaloid analysis The total alkaloid content in macrofungal MBE and BE extracts was determined as described by Nantongo et al. (2018). A total of 50 mL of 10% acetic acid in ethanol was added to 100 mg of the sample; the mixture was covered and incubated for 4 h. After that, the extract was concentrated in a water bath to one-quarter of the original volume, followed by the addition of five drops of concentrated NH4OH to precipitate alkaloids. The precipitate was allowed to rest for 3 h to sediment before being rinsed with 10 mL of 0.1M NH4OH and filtered. The residue on the paper was dried at 60°C for 30 min and then reweighed. Mathematically, the % of the alkaloid is calculated as mentioned in Equation 1: (1) Alkaloid percentage=weight of residue/weight of sample∗100 2.5.8 Terpenoids analysis The terpenoid content in macrofungal MBE and BE extracts was determined as described in Malik (2017). A total of 9 mL of ethanol was added to 100 mg of extract and incubated in the mixture for 24 h. Using the separating funnel, the extract was filtered with 10 mL of petroleum ether. A pre-weighed glass vial was used to collect the ether extract. At room temperature, the solvent was allowed to completely evaporate and then reweighed. The % terpenoids content was measured using the formula given in Equation 2: (2) Terpenoids Percentage= weight of residue/weight of sample ∗100 2.5.9 Total tannin analysis The total tannin content in macrofungal MBE and BE extracts was determined using the Folin-Denis reagent. 10 mL of distilled water was added to 1mg of extract, gently boiled for 1 h in a water bath, and filtered using Whatman filter paper in a 10 mL volumetric flask. To this content, 5 mL of Folin-Denis reagent and 10 mL of saturated Na2CO3 solution were added and incubated at 25°C in a water bath for 30 min. After incubation, the absorbance was measured at 700 nm. The amount of tannin in the extracts was calculated using the equation developed on the standard curve of tannic acid. 2.5.10 Sterols analysis The sterol content in macrofungal MBE and BE extracts was determined using the Liebermann-Burchard reagent, and cholesterol was used as the standard concentration range of 0.2–1 mg/mL. In 1 ml of acetic acid, 1 mg of the extract was dissolved. Then, it was treated with 5 mL of Liebermann-Burchard reagent and incubated at 37°C in a water bath for 10 min. After incubation, the absorbance was measured at 650 nm. 2.5.11 Antioxidant property of the 2,2-diphenyl-1-picrylhydrazyl assay The ability of the MBE and BE extracts to donate electrons and scavenge DPPH radicals was determined by the slightly modified method of Anjana et al. (2016). A freshly prepared 1 mM ethanol solution of DPPH was mixed with the mycelium biomass and mushroom extract at a ratio of 1:0.5 (v/v). The mixture was vortexed properly and incubated in the dark for 10 min. The light absorption was measured at 517 nm. The % scavenging value at the variable concentration was used to plot a regression curve, and the equation generated on the curve was used to estimate the IC50 value for both extract samples. The Equation 3 is used for % scavenging calculation: (3) Scavenging percentage = absorbance of control – absorbance of sample * 100 2.5.12 β-glucan analysis The amount of β-glucan in fungal MBE and BE was measured based on the enzymatic release of glucose, as described in Toledo et al. (2013). This procedure calls for a standard reactive glucose solution with a concentration of 100 mg dL-1, an enzyme reactive mixture with glucose oxidase and peroxidase, color reagent 1 (4-aminophenazone, 25 mM, 920 mM Tris), and color reagent 2 (55 mM phenol). The working reagent was prepared in a 250-mL volumetric flask using 225 mL of distilled water, 12.5 mL of color reagents 1 and 2, and 0.75 mL of enzyme reactive. In total, 20 mL of each sample extract was added to 2 mL of working reagent in the reaction mixture and incubated at 37°C for 15 min. After that, the absorbance at 505 nm was determined using a spectrophotometer. Using 2 mL of the working reagent and 20 mL of standard reactive glucose, the reaction was carried out under the same conditions as above. The results were examined using Equation 4 below: (4) β–glucan (mg/dL)=A * f * 0.9 Where A denotes the absorbance of the sample at 505 nm, which is equal to 100 mg/dL of normal glucose, and 0.9 denotes the proportion of free glucose conversion that was established for anhydrous glucose, which occurs in β-glucan. 2.5.13 Glucan content analysis using the Megazyme kit The contents of the total α- and β-glucans were determined in the MBE and BE using the mushroom and yeast β-glucan assay kit (Megazyme Int., Dublin, Ireland), following the instructions of the manufacturer. Briefly, to estimate the total glucan content in the samples, 2 mL of ice-cold 12M sulfuric acid was used for the hydrolysis of the polysaccharides in the samples at 100°C for 2 h. After neutralization, hydrolysis proceeded to glucose using a mixture of exo-β-(1,3)-D-glucanase + β-glucosidase in sodium acetate buffer (pH 4.5) at 40°C for 1 h. For α-glucan content estimation, enzymatic hydrolysis with amyloglucosidase and invertase was conducted. After that, a glucose oxidase/peroxidase reagent was added to estimate the total glucan and α-glucan content at 510nm. The β-glucan content was calculated by subtracting the α-glucan from the total glucan content. All values of total, α-, and β-glucans in biomass and mushrooms were expressed in g/100g of DW. Determination of β-glucan: The β-glucan content was determined by subtracting the α glucan content from the total glucan content. 2.5.14 Quantification of phytochemical content by HPLC The identification and quantification of phytochemicals from mycelium biomass and mushroom extracts were performed using the high-performance liquid chromatography (PDA) technique. Chromatography was performed on a 250mm x 4.5mm RP C-18 column. Preliminary experiments, in which various ratios of methanol and water were examined, helped to optimize the condition. Quercetin, sesquiterpene glycosides, ergosterol, and RidentinB were the standards employed for the identification and quantification of total polyphenols, terpenoids, sterols, and sesquiterpenes, respectively. The flow rate, column temperature, and detector wavelength were all adjusted to 1 mL/min, 25°C, and 210 to 370 nm, respectively. The formula used for calculation is mentioned in Equation 5: (5) The concentration of compound (mg/mL)=(sample absorbance/mean absorbance of standard) * (standard dilution/sample dilution) * (standard potency/100) * 100. 2.6 Statistical evaluation All the experiments were conducted in triplicate, and the values were expressed as mean ± SD. Statistical significance was detected by the Statistical Package for the Social Sciences (SPSS); a value of p<0.05 indicated a statistical difference.',\n",
       " 11196703: 'Methods This trial was designed as a randomized, double-blind cross-over study. The effect of the sugar substitute ENSO 16 was compared with a 30 g glucose in water solution on glucose metabolism parameters in healthy male or female humans. The trial was approved by the Ethics Committee of the Medical University of Vienna (EK 1126/2022). The study was conducted in accordance with the protocol, the principles of the \"Declaration of Helsinki\" in its current version and the requirements of the Austrian law (AMG). According to the stipulations of the Austrian Data Protection Law, confidentiality and pseudonymity of the volunteers were assured. The informed consent was obtained from all the participants for study participation. The recruiting and data allocation took place between 11. July 2022 and 29. July 2022. Population 18 healthy participants (7 males, 8 females) were included in the study. Participants that dropped out were replaced according to protocol to generate data of 15 different individuals. Inclusion criteria were signed informed consent, age > 18 years, body mass index 18–25 kg*m−2 and a fasting capillary plasma glucose concentration ≤ 100 mg*dL−1. Exclusion criteria were presence of any chronic illness, smoking or drug abuse, food allergies, pregnancy or lactations or participation in another trial 3 weeks before the first study day. Study product The investigational product ENSO 16 (NEOH by Alpha Republic GmbH, Rooseveltplatz 11/1, 1090 Vienna, Austria) is a sugar substitute mixture containing 62% organic fibers like inulin and oligofructose and 38% of other sugar alcohols and sweetener substances. The composition is confidential and not fully disclosed. ENSO 16 was produced, packaged and labeled by STAMAG Stadlauer Malzfabrik GesmbH, Smolagasse 1, 1220 Vienna, Austria according to applicable food law provision and guidelines. The active comparator was glucose powder provided by Special Ingredients Ltd, 4, Foxwood Industrial Park, Chesterfield, S41 9RN, UK. 30 g of each substance were dissolved in 200 mL water for oral administration. A similar amount of sweetener was used in other investigations of sugar substitutes to describe endocrine responses14. 30 g of ENSO 16 contain 198 kJ (49.5 kcal) with an amount of 9.3 g carbohydrates of which 0.72 g is glucose. 30 g of glucose have an equivalent of 486 kJ (116 kcal). The FDA recommends a maximum daily intake of added sugar of 200 kcal (48 g). Study procedure Two study days with a washout period of at least 7 days were planned. Participants were randomized to receive ENSO 16 or glucose on the first study day. The sponsor conducted randomization using a predefined randomization list from the web application at www.randomizer.org. The study medication was provided by the sponsor in identical-looking bottles. Due to their completely identical appearance and smell (white, powdery substance, sweet smell), no difference could be perceived by the investigators or subjects. Preparation of the study product on the study days was performed by blinded investigators. Unblinding was performed after the database lock. Following the wash out period, subjects crossed over to the alternate treatment on the second study day. (Supplement Table 1) Plasma samples for analysis of glucose, C-peptide and insulin were collected under fasted conditions and at pre-defined timepoints (15, 30, 45, 60, 90, 120, 180 min) after single administration of ENSO 16 or glucose. (Supplement Fig. 1) Gastrointestinal symptoms (abdominal pain, nausea, vomiting, diarrhea, abdominal rumbling, bloating, belching and flatulence) were assessed using a standardized questionnaire. Participants were asked to choose between “no symptom” (0 points), “mild symptoms” (1 point) and “severe symptoms” (2 points) for each question at each time point. (Supplement Table 2). Table 1Baseline characteristics of the study population, n = 15.Baseline characteristicsp valuesFemalen, %853.3Malen, %746.7BMI[kg*m−2]Mean ± SD22 ± 2Age[yrs]Mean ± SD28.4 ± 6.1Fasting glucose D1[mg*dL−1]Mean ± SD87 ± 8Fasting glucose D2[mg*dL−1]Mean ± SD89 ± 70.3746 vs. Day 1Figure 1Study flow chart.Table 2Fasting plasma concentrations of glucose, insulin and C-peptide at baseline in participants receiving a single dose of glucose or ENSO 16.GlucoseENSO 16p valuesGlucose [mg*dL−1]79 ± 980 ± 60.5069Insulin [µIU*mL−1]6.36 ± 2.096.59 ± 2.080.676C-peptide [ng*dL−1]1.78 ± 0.691.67 ± 0.350.4593Results are mean ± SD, n = 15 per treatment. Differences in laboratory values were not significant between treatments at baseline (Student’s paired t-test). Methodology Plasma glucose, insulin and C-peptide concentrations were analyzed according to laboratory standards at the Department of Laboratory Medicine, Medical University of Vienna. Venous plasma sampling was performed from an indwelling cannula in an antecubital vein. To assess plasma glucose concentrations, blood was drawn into tubes containing sodium fluoride K3EDTA (Vacuette, Greiner Bio-One, Austria). Insulin and C-peptide concentrations were quantified from blood collected into serum tubes with clot activator (Vacuette, Greiner Bio-One, Austria). Study endpoints Changes in plasma glucose were compared from baseline-adjusted area under the plasma glucose versus time curve until 180 min (AUC0–180 min). Secondary endpoints included plasma glucose, plasma insulin and C-peptide, respectively. Further, the impact of ENSO 16 and glucose on gastrointestinal symptoms were assessed. Statistics Due to a lack of data for the impact of ENSO 16 compared to glucose on plasma, a difference of 10% between groups is assumed to be clinically relevant. With a sample size of 15 subjects, based on other trials and recommendations13 (cross over) a difference of at least 10% between AUCs (with a standard deviation of 10%) can be detected with 80% power using a paired t test at a significance level of 5% (two-sided). All statistical analyses were conducted using the commercially available GraphPad Prism 9 (GraphPad, 225 Franklin Street. Fl. 26 Boston, MA 02110, USA). Continuous variables were summarized as mean ± standard deviation (SD) if their distributions were consistent with normality. If the distributions were not consistent with normality, the variables were summarized by their median and the 25th to 75th percentile range. Between-group comparisons for primary and secondary outcome parameters were assessed using paired t-tests or Wilcoxon signed rank tests as appropriate. Differences were considered significant if the p-value was less than 0.05. For the evaluation of insulin, C-peptide, and glucose profiles, baseline-corrected values were used to calculate the area under the curve (AUC0–180 min) using the trapezoidal rule. Maximum plasma concentrations were used to calculate individual ratios of effects between treatments. Ethical approval The trial was approved by the Ethics Committee of the Medical University of Vienna (EK 1126/2022). Methodology Plasma glucose, insulin and C-peptide concentrations were analyzed according to laboratory standards at the Department of Laboratory Medicine, Medical University of Vienna. Venous plasma sampling was performed from an indwelling cannula in an antecubital vein. To assess plasma glucose concentrations, blood was drawn into tubes containing sodium fluoride K3EDTA (Vacuette, Greiner Bio-One, Austria). Insulin and C-peptide concentrations were quantified from blood collected into serum tubes with clot activator (Vacuette, Greiner Bio-One, Austria).',\n",
       " 11196440: 'MethodPubMed, Embase, Web of Science and Cochrane Library databases were search up to March, 2023 to identify relevant studies, including studies researching beverage consumption, the incidence and mortality of T2DM and incidence of cardiovascular disease, a kind of complication of T2DM. The way to explore the source of heterogeneity is performing subgroup analyses and sensitivity analyses. Funnel plots and Egger’s regression test were performed to assess publication bias. The Hazard ratio (HR) and 95% confidence intervals (95% CIs) were used to analysis the results. Fifteen observational studies were included in our meta-analysis. Methods This systematic review and meta-analysis was written based on the Preferred Reporting Items of Systematic Reviews and Meta-Analysis (PRISMA) statement guidelines [8]. Search methods The databases searched for this article include PubMed, Embase, Web of science and Cochrane Library databases, and the search authors include Zhang Yuqing, Wang Xu and Liu Yuxiang. The search term is ((type 2 diabetes [Title/Abstract]) OR (type 2 diabetes mellitus[MeSH Terms])) AND ((beverage[MeSH Terms]) OR (beverage[Title/Abstract])), which is consist of main Mesh and non-Mesh terms. The search of relevant literature were searched for relevant studies from inception to March 2023. The MeSH/Emtree and abstract/title keyword were combined to identify the eligible English articles. In addition, a manual search was conducted for the references of the included articles (shown in Fig. 1).Fig. 1Flowchart of selecting studies for review Study eligibility Eligibility criteria and exclusion criteria should be identified, which were used to search and screen potential articles. Articles may be eligible, if they fulfill all the following conditions: (1) prospective population-based studies or patients diagnosed with T2DM; (2) the study compared with the use of beverages (including coffee, tea and SBs) with no use of beverages; (3) the study measured and reported the outcomes in terms of mortality of T2DM, incidence of T2DM or incidence of complications; The exclusion criteria were as follows: (1) all criteria were not fulfilled; (2) the study was a case report, an animal study, a review, a conference abstract or an abstract; (3) OR/RR/HR with 95% CIs were not provided or could not be calculated; Date extraction and study quality The selection and searching of potential articles were completed independently by two researchers, based on the inclusion and exclusion criteria, to identify eligible studies. Data extraction was conducted separately by another researchers. Moreover, any disagreements would be decided by the senior researcher to resolve this controversy. Relevant data were collected, which include the year of publication, first author, study title, study type, study period, type of beverage, the efficacy in control and treatment groups, T2DM definition, reported outcomes and so on. The characteristics of the studies included in this meta-analysis included beverage type, follow-up duration range, number of participants, age, sex distribution, beverage intake, and outcome measures (shown in Table 1). Beverage portion sizes and frequency of consumption were different among the included articles, some were not mentioned in the original study, the available data have been extracted into the Table 1. For every abstracted data of articles, two investigators independently assess the risk of bias for this meta-analysis based on The Newcastle–Ottawa Scale (NOS) [9] (shown in Table 2). Articles were thought high quality which received seven or more points. Covariates were composed of eight parts: age and sex, race and socioeconomic status, physical activities, smoking and drinking, body mass index, diet factors, clinical features and other types of beverages. The types of statistical data related to outcomes extracted in these 15 articles were all binary variables, HR values, and 95% confidence intervals. Table 1Characteristics of the included studies about the association between beverages and the risk of incidence or prognosis in T2DM patientsAbbreviationT2DM Type 2 of diabetes mellitus; SSB Sugar-sweetened beverages; ASB Artificially sweetened beverages; TSB Total sweetened beverage (combined ASB and SSB); NJs Natural juices; LCSBs Low-calorie sweetened beverageTable 2NOS scores of the included studies Statistical analysis and publication bias The aim of forest plots was to synthesize HRs and 95%CI across studies. The I2 statistics presented the potential heterogeneity; the pooling outcome was considered to represent non-significant, low, moderate and high heterogeneity respectively, when I2 was 0–25%, 26–50%, 51–75% or > 75%. The source of heterogeneity was explored by performing subgroup analysis. The potential source of heterogeneity includes the definition of exposure, grouping criteria, study sites, number of participants, sex ratio, follow-up period, quality of study, baseline age and adjustment of clinical features. Egger’s linearity regression test and Begg’s funnel plot was performed to evaluate the publication bias. For the asymmetry, funnel plots were visually assessed. The aim of Egger’s lire available to assess, sensitivity analysis was performed to evaluate the influence of a single study on the overall effect estimate by sequential stepwise omitnear regression is to confirm the presence of a small size. If sufficient data weting of one study. All analyses were completed by using Stata12.0(College Station, Texas 77845, USA), random effects model and inverted variance method. P < 0.05 was considered statistically significant. Search methods The databases searched for this article include PubMed, Embase, Web of science and Cochrane Library databases, and the search authors include Zhang Yuqing, Wang Xu and Liu Yuxiang. The search term is ((type 2 diabetes [Title/Abstract]) OR (type 2 diabetes mellitus[MeSH Terms])) AND ((beverage[MeSH Terms]) OR (beverage[Title/Abstract])), which is consist of main Mesh and non-Mesh terms. The search of relevant literature were searched for relevant studies from inception to March 2023. The MeSH/Emtree and abstract/title keyword were combined to identify the eligible English articles. In addition, a manual search was conducted for the references of the included articles (shown in Fig. 1).Fig. 1Flowchart of selecting studies for review',\n",
       " 11196417: 'MethodsThe study first accessed SD datasets from the GEO and identified DEGs. These DEGs were then subjected to gene set enrichment analysis. Several advanced techniques, including the RF, SVM-RFE, PPI networks, and LASSO methodologies, were utilized to identify hub genes closely associated with SD. Additionally, the ssGSEA approach was employed to analyze immune cell infiltration and functional gene set scores in SD. DEGs were also scrutinized in relation to miRNA, and the DGIdb database was used to explore potential pharmacological treatments for SD. Furthermore, in an SD murine model, the expression levels of these hub genes were confirmed through RT-qPCR and Western Blot analyses. 2 Methods 2.1 Data collection and normalization We retrieved datasets related to SD and their corresponding control groups from the Gene Expression Omnibus (GEO) database (http://www.ncbi.nlm.nih.gov/geo) (12). Human RNA expression data from 17 individuals with SD and 25 healthy controls were obtained from the GSE208668 dataset, derived from the GPL10904 platform. For the validation of the subsequently identified hub genes, we utilized the GSE240851, GSE56931, and GSE98582 datasets, which were sourced from the GPL24676, GPL10379, and GPL6244 platforms, respectively. These datasets included varying numbers of SD patient and control samples. Additionally, the GSE165041 dataset, generated on the GPL18573 platform, comprised microRNA expression data from 10 SD patients and an equal number of healthy controls. All datasets were normalized using the “limma” package within the R software environment, version 4.1.2 (13). 2.2 Identification of differentially expressed genes We utilized the limma package in R software to process the normalized datasets GSE240851 and GSE208668, aiming to identify DEGs. To ascertain statistical significance, DEGs were determined based on |log2fold change| ≥ 0.58 and a false discovery rate <0.05. Subsequently, we employed the ggplot2 package to create volcano plots for the DEGs. Moreover, we selected the top 20 genes, ranked by |log2fold change|, to build heatmaps as part of our analysis. 2.3 Gene function enrichment analysis The “c2.all.v2023.1.Hs.entrez” and “c5.all.v2023.1.Hs.entrez” datasets, which serve as reference gene sets, were downloaded from the GSEA official website originating from the MSigDB database (14). Gene set enrichment analysis (GSEA) was then performed using the “clusterProfiler” package in R software (15). Subsequently, the analysis results were visualized utilizing the “enrichplot” package in R software. 2.4 Immune infiltration analysis We employed the ImmuCellAI and ssGSEA methodologies to estimate the abundance of 24 and 28 distinct types of immune cells, respectively, in the tissues of SD patients and a normal population, thereby enabling precise delineation of immune cell profile differences between the two groups (16, 17). Furthermore, we conducted Spearman correlation analysis to elucidate the interrelationships among the distributions of various immune cells. 2.5 ssGSEA We downloaded the H: hallmark gene sets from the GSEA official website to probe the functional disparities between SD patients and the normal cohort. With these gene sets, we applied the ssGSEA algorithm to evaluate 50 gene sets, aiming to discern potential variances between the two groups. Subsequently, we utilized the Mantel algorithm to analyze the correlations among these gene sets (18). This approach allowed us to investigate the differences in gene expression profiles between the two cohorts and to identify potential molecular pathways associated with the disease. 2.6 Protein-protein interaction network construction For protein interaction analysis, we utilized the STRING 4 online platform and specifically selected PPI pairs with a confidence score greater than 0.40. Subsequently, we employed the Cytoscape V3.9.0 software for visualizing the PPI network (19). Within the network, the significance of each node was determined by calculating their Degree values using the CytoHubba plugin. This analysis allowed us to identify the top 20 pivotal genes based on their ranking of importance (20). 2.7 Random forest gene selection For gene selection, we utilized the Random Forest (RF) algorithm, a binary tree-based recursive partitioning method. The “randomForest” package in R was used, with parameters set to ntree=1000, mtry=3, and importance=true (21). Employing the Gini index as the primary assessment criterion, the Random Forest algorithm was used to rank the DEGs, and the top 20 genes with a significance value greater than 3 were earmarked for further analysis. 2.8 Support vector machine gene selection We utilized the SVM-RFE (Recursive Feature Elimination) approach to optimize the predictive model by minimizing the number of feature vectors produced by the SVM. This approach, being an effective binary classification tool, operates by constructing a classification hyperplane to delineate decision boundaries. In order to enhance the algorithm’s precision, we configured parameters to method=repeatedcv and repeats=10 in the R package “1071”, and employed ten-fold cross-validation. This approach aimed at augmenting the algorithm’s precision (22). 2.9 Model construction and evaluation To develop the LASSO model (23), we integrated genes identified through CytoHubba, RF, and SVM algorithms. This method effectively enabled the identification of critical hub genes for diagnosing SD. We then employed the Logistic regression approach to investigate pivotal factors associated with SD, ultimately constructing a simplified model. Subsequently, we evaluated the classification performance of the model using the Receiver Operating Characteristic (ROC) curve and the corresponding Area Under the Curve (AUC). 2.10 Validation of the diagnostic model In order to evaluate the robustness and general applicability of our developed diagnostic model, we computed the AUC of the ROC curve for the model using three distinct datasets: GSE240851, GSE56931, and GSE98582. This procedure aimed to ascertain the model’s performance across diverse datasets. It was essential to assess its efficacy in different contexts and ensure its capability to perform consistently across varied data sources. 2.11 Exploration microRNAs targeting the genes We utilized the Limma package in R to conduct a differential analysis of the expression matrix from GSE165041, aiming to identify miRNAs that were differentially expressed (DEmiRNAs). Our criteria for significance were miRNAs exhibiting a false discovery rate <0.05 and |log2fold change| >0.5. Moreover, we employed the miRNet database to investigate potential miRNAs associated with differentially expressed genes (DEGs), in order to gain further insight into their involvement in SD (24). 2.12 Drug and gene interaction scoring We obtained data on drugs related to core genes from the Drug-Gene Interaction database (DGIdb) (25). We used the “ggplot2” package in R to create bar charts showing interaction scores, visually indicating the intensity of interactions between various drugs and core genes. 2.13 Establishment of animal models The SD model, following the methodology outlined by Alkadhi and Alhaider (26), was established in 10-month-old male mice. The experimental mice were subjected to SD treatment for a duration of 8 weeks by being placed on a small fixed platform encircled by water, with access only to water and food. Meanwhile, for the control group, another set of mice was housed in a comfortable environment with a 12-hour light/dark cycle and unrestricted access to water and food. All mice received standard pellet feed, with the daily quantity of feed being maintained uniformly across all groups. This experimental design aimed to simulate the effects of SD on physiological functions, thus laying the groundwork for subsequent investigations into gene expression and drug treatment efficacy. 2.14 RT-qPCR The cortex in SD mice was used for total RNA extraction, employing the TransZol Up Plus RNA Kit (TransGEN, Beijing, China) (27). The RNA concentration and quality were then assessed using the Nanodrop Spectrophotometer (Termo Scientifc, Waltham, MA, USA). Subsequently, reverse transcription was carried out using the TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix (AT311, TransGEN, Beijing, China). Amplification was monitored with the ChamQ Universal SYBR qPCR Master Mix (Novozymes Q711) and a QuantStudio™ 5 Real-Time PCR System (Thermo Fisher Scientific). The internal reference was β-actin, and the relative gene expression was determined using the 2-ΔΔCT formula. Detailed primer sequences can be found in Table 1 . Table 1 Primer sequences of mRNA for RT-qPCR. Gene Primer sequence, 5’–3’ Forward Reverse IPO9 CAGTGACAGCCTTGGTGAAA TCTCCAGTAGGGCATGGACA RAP2A CAAACTGTACCACGCCCTCT GTTGCTAGGTGGATTGGGCT DDX17 TCTTCAGCCAACAATCCCAATC GGCTCTATCGGTTTCACTACG MBNL2 CCCAAAAGTTGCCAGGTTGAA CTGGGTTTTTAAGTGTGTCGGA PIK3AP1 CTGGACTCTGCTTCTAACCCC TGACACCATTCCTCCGCATC ZNF385A CAGAACCAAGGGAAGGGGAC GAAGGGCAGGATCTGCTTGA β-Actin GCAGGAGTACGATGAGTCCG ACGCAGCTCAGTAACAGTCC 2.15 Immunoblotting for protein evaluation Western blotting was performed for IPO9, RAP2A, DDX17, and GAPDH, as described previously ( Table 2 ). Enhanced chemiluminescence reagents were used to detect protein expression, and quantitative analysis was conducted using Image J software (28). Table 2 List of the primary antibodies. Antibody Catalogue number Brand Application Dilution Species MW(kDa) IPO9 abs134286 absin WB 1:1000 Rabbit 116 RAP2A abs105749 absin WB 1:1000 Rabbit 21 DDX17 abs111924 absin WB 1:1000 Rabbit 72 GAPDH GB15002-100 Servicebio WB 1:2000 Rabbit 36 2.16 Statistical analysis The normality of the data was assessed using the Shapiro–Wilk test. The t-test was utilized to compare the data between two groups, while comparisons among multiple groups were conducted using one-way ANOVA, followed by either the LSD post hoc test or Tukey’s post hoc test. The statistical analyses were conducted using R software version 4.2.1 and SPSS 25, with significance defined as P < 0.05. Furthermore, all experiments were independently repeated at least three times to ensure the validity of the results.',\n",
       " 11196393: '2 Materials and methods 2.1 Human samples Samples of peripheral blood were collected from healthy adults, full-term infants, and preterm infants. Samples were collected from 32 subjects from Guangzhou Women and Children’s Medical Center and the Third Affiliated Hospital of Guangzhou Medical University. The basic characteristics of the samples are included in Supplementary Table S2 . All subjects were collected within 7 days after birth and screened for serum hepatitis B surface antigen (HBsAg), hepatitis C virus (HCV) antibody, hepatitis D virus (HDV) antigen, HDV antibody, and HIV antibody, and positive individuals were excluded from this study. Preterm infants were defined as those with a gestational age of less than 37 weeks. Full-term infants were defined as those with gestational age between 37 and 42 weeks. Informed consent was obtained from all participants. 2.2 Sample preparation for 10× genomic single-cell sequencing Peripheral blood samples were collected from three full-term, three preterm, and three adult controls. Single-cell suspensions of peripheral blood mononuclear cells (PBMCs) were obtained by centrifuging human blood samples on a Ficoll gradient. Briefly, blood was mixed with an equal volume of 2% FBS in PBS and gently layered on the Ficoll gradient. Cells were centrifuged at 1,000×g for 25 min without braking. The cells in the middle layer were then washed once with PBS, and the red cells were removed using ACK lysing buffer and resuspended in 2% FBS in PBS for use. The PBMC suspensions were stained with surface markers at 30 min at 4°C in the dark. The strategy for MDSCs was CD11b+HLA-DR−/lo. Live immature myeloid cells in RPMI1640 supplemented with 20% FBS were sorted in a BD FACSAria III cell sorter (BD Biosciences, USA). 2.3 Flow cytometric analysis Freshly prepared PBMCs were incubated with Fc-block (Miltenyi, Bergisch Gladbach, Nordrhein-Westfalen, Germany) for 10 min, and surface staining was performed at 4°C for 30 min. Cells were stained with surface antibodies against CD11b, HLA-DR, CD14, CD177, and CXCR2. For Arg1 staining, single-cell suspensions were stained with antibodies to surface antigen, then fixed and permeabilized, followed by staining with anti-Arg1 (A1exF5; Thermo Fisher, Waltham, Massachusetts, USA). For ROS staining, single-cell suspensions were stained with antibodies to surface antigen and were evaluated with 20 μM DCFDA (Abcam, Cambridge, UK) staining at 37°C for 30 min. The analysis was performed by BD FACSCanto™ II. The following antibodies from (BioLegend, San Diego, California, United States) were used: APC antihuman CD11b(ICRF44), Brilliant Violet 605™ antihuman HLA-DR(L243), APC/Cyanine7 antihuman CD14(63D3), PerCP/Cyanine5.5 antihuman CXCR2(5E8/CXCR2), and FITC antihuman CD177(MEM-166). 2.4 scRNA-seq data collection and analysis The data about newborns have been deposited in NCBI’s Gene Expression Omnibus and are accessible through GEO Series accession number GSE253963 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE253963). The raw data were preprocessed using Cell Ranger (version 7.0). The R toolkit Seurat (version 4.0.1) was used for downstream analysis (10). Cell subpopulations were annotated based on the expression of canonical marker genes. The function of AddModuleScore was performed to calculate the scores of molecular signatures in individual cells. The gene sets used for each molecular signature are listed in Supplementary Table S4 . Genes associated with the enriched pathways identified through GO analysis are listed in Supplementary Tables S3 , S4 . The scRNA-seq dataset of tumor MDSCs (GSE163834) was downloaded from the Gene Expression Omnibus GEO database (https://www.ncbi.nlm.nih.gov/gds/) (11). 2.5 Pseudo-time developmental trajectory analysis The R package monocle2 (version 2.26.0) was used to infer the differentiation path among distinct cell subpopulations (12). Briefly, a CellDataSet class was created by using a normalized scRNA-seq dataset. Subsequently, the differentialGeneTest function was performed to identify differentially expressed genes (DEGs) with each subpopulation. These DEGs were then used as input genes to perform dimension reduction using the DDRTree method. Next, each cell was assigned a pseudo-time order by executing the orderCell function. Finally, the pseudo-time trajectory was plotted by performing the function of plot_cell_trajectory. 2.6 Statistical analysis All data were derived from at least two independent experiments. Statistical analysis was performed with GraphPad Prism 8.0. Statistical significance was determined by a paired Student’s t-test or an unpaired Student’s t-test for comparing two groups, and a one-way ANOVA with a Tukey–Kramer multiple comparisons test was performed for comparing three groups within the same experiment. A p-value < 0.05 was considered significant, and the results showed the mean ± SEM.',\n",
       " 11196377: \"MethodsA total of 600 23‐day‐old broiler male chicks (Cobb‐500) were assigned to 10 treatments with six replicates and 10 birds each. Experimental treatments were included factorial arrangement of five increment (2.5%) levels of finisher diet ND (92.5%, 95%, 97.5%, 100% and 102.5% as strain recommendation) and slaughtered at 38 or 46 days of age. The relative difference in the energy level of experimental diets was used to increase ND levels at the same ratio. 2 MATERIALS AND METHODS 2.1 Feedstuff analysis Before to the trial, protein, amino acids and digestible amino acids of feed ingredients were measured by using near‐infrared analysis through Evonik Co. (Evonik Nutrition & Care GmbH) agent in Tehran, Iran. These values were used in a least‐cost equation for the formulation experimental diets. 2.2 Birds, housing and diets A total of 840 1‐day‐old male broiler chicks (Cobb‐500) were allocated into 60 pens (14 birds/pen, 100 × 100 cm2) and raised until 22 days on a common condition. The ambient temperature initially was maintained at 32 ± 2°C; after 3 days, it decreased by 0.5°C per day to reach 20–22°C and after that, it was kept constant, the relative humidity was 50%–60%; and a lighting schedule of 18L: 6D was imposed throughout the experimental period. At 23 days of age, with removing the highest and the lowest weight birds, 10 birds were fixed in each pen (average LBW was 695 ±11 g/bird). The pens were randomly assigned to a 5 × 2 factorial arrangement of feeding five levels of finisher diet ND 92.5%, 95%, 97.5%, 100% and 102.5% as strain recommendation (Cobb‐Vantress, 2019), and two SAs (38 vs. 46 days old) with six replicates per treatment. The experimental diets were formulated in a minimum cost equation by user‐friendly feed formulation done again (UFFDA, 1992) software, University of Georgia, Athens, GA, United States (Table 1). The relative difference in the experimental diets’ energy level was used to change protein, indispensable amino acids and minerals inclusion levels at the same ratio. All birds had free access to drinking water and feed throughout the experiment period. Birds and housing facilities were inspected three times daily. TABLE 1 Ingredients and nutrients composition of the experimental diets. Items Finisher diet nutrients density a 92.5% 95% 97.5% 100% 102.5% Ingredients, g/kg as‐fed basis Corn 652.8 636.2 619.6 603.0 586.4 Soya bean meal 236.0 253.5 271.1 288.8 306.3 Wheat bran 68.1 53.6 39.1 24.5 10.0 Soya bean oil 10.0 22.7 35.4 48.1 60.8 Limestone 7.9 8.0 8.0 8.1 8.2 Dicalcium phosphate 11.2 11.9 12.6 13.3 14.0 Common salt 3.9 4.0 4.1 4.1 4.2 Vitamin premix b 2.5 2.5 2.5 2.5 2.5 Mineral premix c 2.5 2.5 2.5 2.5 2.5 dl‐Methionine 2.3 2.4 2.5 2.6 2.7 l‐Lysine‐HCL 1.9 1.8 1.7 1.6 1.5 l‐Threonine 0.9 0.9 0.9 0.9 0.9 Determined nutrients composition d , as‐fed basis Metabolizable energy, kcal/kg 2928 3007 3087 3166 3246 Crude protein, g/kg 175.7 180.5 185.2 190.0 194.7 ME/CP ratio 16.6 16.6 16.6 16.6 16.6 Crude fibre, g/kg 35.2 34.2 33.2 32.2 31.2 Calcium, g/kg 7.0 7.2 7.4 7.6 7.8 Available phosphorus, g/kg 3.5 3.6 3.7 3.8 3.9 Sodium, g/kg 1.8 1.8 1.8 1.9 1.9 Digestible lysine, g/kg 9.2 9.5 9.7 10.0 10.3 Digestible methionine, g/kg 4.7 4.9 5.0 5.2 5.3 Digestible Sulphur amino acids, g/kg 7.2 7.4 7.6 7.8 8.0 Digestible threonine, g/kg 6.3 6.5 6.7 6.8 7.0 Abbreviations: CP, crude protein; ME, metabolizable energy. a Percentage of nutrient recommendations stated by the Cobb‐500 (Cobb‐Vantress, 2019) commercial management guide. b Vitamin premix supplied the following per kilogram of diet: vitamin A (all‐trans‐retinol), 12,000 IU; vitamin D3 (cholecalciferol), 5000 IU; vitamin E (α‐tocopherol), 18 IU; vitamin K3 (menadione), 2.65 mg; vitamin B1 (thiamin), 2.97 mg; vitamin B2 (riboflavin), 8.0 mg; vitamin B3 (niacin), 57.42 mg; vitamin B5 (pantothenic acid), 17.86 mg; vitamin B6 (pyridoxine), 4.45 mg; vitamin B9 (folic acid), 1.9 mg; vitamin B12 (cyanocobalamin), 0.02 mg; vitamin H2 (biotin), 0.18 mg; choline chloride and 487.5 mg, antioxidant 1.0 mg. c Mineral premix supplied the following per kilogram of diet. Zn (zinc sulphate), 110 mg; Mn (manganese sulphate), 120.6 mg; Fe (iron sulphate), 40.5 mg; Cu (copper sulphate), 16.1 mg; I (calcium iodate), 1.26 mg; Se (Sodium Selenite) and 0.31 mg; choline chloride, 474.0 mg. d The determined ingredient analysis data was used to calculate nutrients composition; crude protein, calcium and sodium were measured by the AOAC (2002) methods; metabolizable energy, digestible amino acids, and available phosphorus were measured by using the near‐infrared analysis. John Wiley & Sons, Ltd. 2.3 Data collection and sampling 2.3.1 Growth performance The chicks of each pen were weighed in groups at the beginning (23 days old) and at the end (38 and or 46 days old). In order to minimize the error resulting from the digestive tract contents, the birds were starved for 4 h before weighing. The feed consumption of each pen was calculated by subtracting the amount of feed remaining at the end from the total feed given during the experimental period and adjusted for mortality. The growth performance traits, such as LBW, FI, WG, feed conversion ratio (FCR), yield per unit area (YUA) (Abougabal & Taboosha, 2020), bio‐economical index (BEI) (Santos et al., 2008) and European production efficiency factor (EPEF) (Cobb‐Vantress, 2019), were calculated. The YUA, BEI and EPEF were calculated by the following equations: YUA=Penlivebodyweightkg×Livability%Penaream2 BEI=Weightgainkg−FeedcostProductioncost×Feedintakekg EPEF=Weightgainkg×Viability%Feedconversionratio×Ageday×100 2.3.2 Energy and protein efficiency The following equations were used to calculate fat content of gain (FCG), protein content of gain (PCG), energy and protein efficiency and energy requirements (maintenance + gain) by using the function of initial and final LBW, WG and gain composition (Carré & Méda, 2015). The expected feed intake (EFI) was estimated by dividing the total energy requirements to diet AME. Residual feed intake (RFI) was obtained by subtracting the EFI from the observed FI (Nikbakhtzade et al., 2020). FCG=12.30.025+0.137BWinitial0.418+BWfinal0.4182e−1.082CPdietAMEdiet−2.09PE PCG=0.179+0.00169Ageinitial+Agefinal2×1−FCG Energyefficiency=WG9.13FCG+5.64PCGFI×AMEdiet Proteinefficiency=WG×PCGFI×CPdiet Energyrequirementsformaintenance=0.680BWinitial0.643+BWfinal0.6432 Energyrequirementsforgain=WG0.03288PCG+0.04372FCG where FCG is the fat content of gain, PCG is the protein content of gain, BW initial is the live body weight at the start of the test, BW final is the live body weight at the end of the test, WG is the weight gain, CP diet is the diet crude protein, AME diet is the diet metabolizable energy and FI is the feed intake. 2.3.3 Carcass and cut‐up parts At 38 and 46 days old, 2 birds from each pen (12 birds/treatment), after 6 h of feed withdrawal but had free access to drinking water, were randomly selected and slaughtered by severing the jugular vein, allowed bleeding for the 90 s, scalded for 2 min at 60°C, and picked (Zarghi et al., 2020). The gastrointestinal tract (GIT) was emptied, and abdominal fat was collected carefully. After carcass chilling for 24 h at 4°C (Tabatabaei et al., 2017), carcasses were weighed to determine the carcass yield as a percentage of LBW. The carcass parts – legs (thigh + drumstick), breast and frame (carcass without breast and legs) – were cut according to a standardized procedure (Tabatabaei et al., 2017) and weighed by a digital weighing scale (0.001−g; Model GF 400, A&D Weighing). 2.3.4 Breast meat's physicochemical properties The right side of the breast muscle was used to assess physicochemical properties. Water holding capacity (WHC) was measured as (100% expressed juice), following a modification of the press method (Zarghi et al., 2020). Cooking loss (CL) was measured as proportionate weight loss of a sample after cooking in an open plastic bag in a water bath at 70°C for 40 min, followed by cooling in cold‐running tap water for 15 min (Honikel, 1998). The equation is shown as follows: Cl%=Weightbeforecooking−WeightaftercookingWeightbeforecooking×100 Colour intensity was measured on breast muscle samples with a Chroma‐Meter (CR‐410 Konica Minolta, INC). The optical density of meat, lightness (L*), redness (a*) and yellowness (b*) values. The hue angle (h*) and Chroma (c*) were calculated using the following formula (Zarghi et al., 2020). The equations are shown as follows: h∗=tan−1b∗a∗ c∗=a∗2+b∗20.5 The left side of skinless and boneless breast meat, after freezing, was homogenized in a cutter for approximately 3 min until a homogeneous paste‐like mixture was obtained, then about 100 g of them were removed and freeze‐dried, which was then used to determine the breast meat composition such as moisture, crude protein, fat and ash according to standardized methods (AOAC, 2002). 2.4 Statistical analysis The experimental data were analysed for normality via the univariate procedure and then analysed by ANOVA using general linear model procedure of SAS 9.4 software (SAS Institute, 2014) appropriate for treatments in a randomized completely design with factorial arrangement. Means were compared using the Tukey test at the level of 0.05 probability (Tukey, 1991). Orthogonal polynomials contrast coefficients were used to determine linear and quadratic effects of increasing finisher diet ND levels on all measurements. To estimate the optimized responses, liner broken‐line, quadratic broken‐line and quadratic polynomial equation models were used via the NLIN procedure Gauss–Newton method of SAS 9.4 software. The iterative procedure makes repeated estimates for coefficients and minimizes residual error until the best‐fit lines are achieved (Robbins et al., 2006). The equations are shown as follows: ForlinearbrokenlineY=L+U×R−X×I ForquadraticbrokenlineY=L+U×R−X2×I ForquadraticpolynomialequationY=a+bX+cX2 R2=correctedtotalsumofsquares−sumsofsquaresforerrorcorrectedtotalsumofsquares where Y is the dependent variable, L is the theoretical maximum, R is the optimized dietary ND, X is the independent variable, I is the 1 (if X < R) or I = 0 (if X > R) and U, a, b and c = rate constant.\",\n",
       " 11196265: \"Material and methods Chemicals Polyethylene microplastics (PE-MPs) powder was supplied by Toxemerge Pty Ltd. (Melbourne, Australia). Purified water (Milli-Q) was used to create the base solution from the powder in accordance with the manufacturer's instructions and kept at 4 °C in the dark until characterization15. AK Scientific, Inc., Union City, CA, USA, provided tadalafil (Cilais) substance. Justification for drug dosage selection The dosage of tadalafil employed in our study was chosen based on previous research21. However, a study by Bianco et al. [21] indicated that there is no human health risk for exposure to sildenafil (LC50 = 42.74 for B. calyciflorus) and tadalafil (LC50 = no effect up to 20 for B. calyciflorus). However, they require special care as this form of contamination is found in any aquatic system evaluated. Another study found that the process of chlorination of sildenafil (LC50 = 0.13–0.27) and tadalafil (LC50 = 0.15–0.29) can be hazardous to fish that are exposed to it34. Therefore, we chose to utilize the maximum dosage of 20 mg/l of tadalafil in our research. Conversely, the PEMPs used in our investigation were selected based on earlier research15. Experimental design Fish (Oreochromis niloticus) (mean body weighing 242.7 ± 22.4 g; mean body length 27.3 ± 1.4 cm) were originated from private fish farm, Assiut, Egypt. They were healthy and parasite-free according to35. Oreochromis niloticus were acclimatized for 4 weeks in 96 L glass aquaria (80 × 40 × 30 cm) filled with dechlorinated water in the laboratory conditions: conductivity (261 ± 5.2 mM cm−1),water temperature (26 ± 1 °C), dissolved oxygen (7 ± 0.2 mg L−1), pH (7.4 ± 0.4), and photoperiod 12:12 h. Oreochromis niloticus were fed with a commercial fish pellet feed (containing 30% protein and 12% lipid; El Nasr Company, Egypt) at rate of 3% body biomass daily. One hundred and twenty fish (O. niloticus) were selected and divided into four treatments, each treatment contains 30 fish in triplicates (as 10 fish for each) in 96L glass aquarium water. The fish were distributed into 1st group control group (fish exposed to only water); 2nd group exposed to 10 mg/LPE-MPs15; 3rd group exposed to 20 mg/l tadalafil (Cilais)21; 4th group exposed to 20 mg/l tadalafil (Cilais) + 10 mg/LPE-MPs. Twelve fish from each group were chosen at random after 15 days of exposure and put unconscious with crushed ice36. Blood samples (1.5 ml) were collected from the caudal vein, part for hematological indices, and the other part was centrifuged undercooling for biochemical and antioxidants parameters. Liver, gills, and kidney tissues were used for histological and immunohistochemical studies. Hemato-biochemical parameters Hematological parameters such as the number of red blood cells (RBCs), white blood cells (WBCs), differential WBCs, blood platelets, hematocrit level (Hct) and hemoglobin level (Hb), as well as erythrocyte indices such as mean corpuscular hemoglobin (MCH), mean corpuscular volume (MCV), and mean corpuscular hemoglobin concentration (MCHC) were all measured using an automated technical analyzer (BC-2800 from Mindray). Blood samples for biochemical parameters were obtained from fish without the use of an anticoagulant agent to acquire serum. Colorimetric determinations of the following important biochemical indices: alanine aminotransferase (ALT), aspartate aminotransferase (AST), creatinine, uric acid, albumin, globulin, and glucose were carried out according to Hamed, Soliman15 by using a spectrophotometer in a wavelength range of 340–546 nm (Biodiagonstic Company, Egypt). Antioxidant parameters and Lipid peroxidation Reduced glutathione (GSH) concentration was evaluated according to Beutler et al.37. TAC (total antioxidant capacity) was determined utilizing kits (Sigma-Aldrich, USA). The malondialdehyde (MDA) level was determined using a thiobarbituric acid reaction38. Histopathological analysis Tissue samples of the gills, liver, and kidney were collected from three fish from each group and fixed in neutral buffered formalin before being processed using a standard automated process (dehydrated through graded ethanol concentration, cleared with methyl benzoate), embedded in paraffin wax, and sectioned (5 microns)39. De-waxed slides are rehydrated before being stained with hematoxylin and eosin (H & E)40. Finally, an Olympus CH30 microscope was used to analyze and photograph the specimen. Pathology severity was graded as follows: (−), absent; (+), minor; (++), moderate; (+++), severe; and a maximum score (++++)41. Immunohistochemical analysis An immunohistochemistry recoloring method was utilized to look at Interleukin-6 (IL-6). Four- µm -thick tissue segments were deparaffinized in xylene and rehydrated. To halt endogenous peroxidase action, the areas were submerged in 3% H2O2 for 10 min. After that, they were PBS-washed for 2 min. three times. The segments were at that point treated with standard goat serum per the prescribed procedure for 30 min at 37 °C (Vector Research facilities, Burlingame, CA), and after that for 1 h at room temperature with essential polyclonal IL-6 antibody; 1/50 (E-AB-40021, Elabscience Biotechnology Inc, USA). The expansion of polyperoxidase-anti-Mouse/Rabbit IgG is taken after 20 min. With the help of a streptavidin–biotin-peroxidase pack and Mayer's hematoxylin as a counterstain, the antigen–antibody complex was found. For each explore, positive and negative control areas were utilized. Intense brown nuclear and cytoplasmic staining assisted in identifying active IL-6 immunostaining cells. A light microscope was used to examine the slices in the 20 randomly selected locations (Olympus BX41, New York) at a magnification (50 ×). A blind fashion evaluation was carried out, with comparisons performed using the average score across all groups. The photomicrographs were taken with digital camera (canon digital camera (PowershotA95)). Statistical analysis The SPSS 16.0 program for Windows (SPSS 2007, Inc., IL, USA) was used to analyze the data, and 0.05 was regarded as the point of significance. The Shapiro–Wilk test was used to determine whether the data was normal. ANOVA was used to determine the homogeneity of variances after the one-way analysis of variance (Levene’s test). Fisher’s LSD post hoc analysis was used to compare the treatment groups to the control group in the case of variance equality. Dunnett’s post hoc test was used to compare the treatment groups to the control group where there was variance inequality. Ethical statement The ethics committee of the Faculty of Veterinary Medicine, Assiut University, Assiut, Egypt, authorized the experimental setup and fish handling in accordance with the OIE criteria for the use of animals in research. All methods were performed in accordance with the relevant guidelines and regulations and in accordance with ARRIVE guidelines (No, 06\\\\2023\\\\0037).\",\n",
       " 11196252: 'Materials and methods ITIES studies All electroanalytical studies were conducted using an AUTOLAB-PGSTAT302N instrument (Metrohm Autolab B.V., The Netherlands) controlled via NOVA 1.11.1 software. Measurements were carried out with a four-electrode electrochemical system (Fig. 1B) comprising of two platinum (Pt) wires serving as the counter electrodes and two silver/silver chloride wires (Ag/AgCl) functioning as reference electrodes (as depicted in Scheme 1). The Galvani potential difference of the studied ions was calculated using the data from cyclic voltammograms whereas the potential axis was calibrated using the standard Galvani potential difference of the tetrapropylammonium cation ion transfer (TPrA+, \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{mathrsfs} \\\\usepackage{upgreek} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document}$${\\\\Delta }_{org}^{aq}{\\\\Phi }_{{TPrA}^{+}}^{0}=-0.091 V$$\\\\end{document}ΔorgaqΦTPrA+0=-0.091V)29. Cells from Schemes 1 and 2 were used during this study.Scheme 1Electrochemical cell employed for electroanalytical studies of DANO at ITIES.Scheme 2Electrochemical cell used to evaluate the effect of pH on the DANA behavior at ITIES. Measurements at the GCE This part of measurements were conducted using an EmStat3 potentiostat under the control of PSTrace software supplied by PalmSens B.V., the Netherlands. All experiments were carried out in conjunction with an automated electrode stand (M164, MTM Anko Instruments, Cracow, Poland). The electrochemical three-electrode setup (Fig. 1C) consists of glassy carbon electrode (GCE) with a 3 mm diameter and a geometric area of 7.1 mm2 (Basi®, USA) as the working electrode, silver/silver chloride electrode (Ag/AgCl/3.0 mol L−1 KCl, MTM Anko Instruments, Kraków, Poland) as a reference electrode and a platinum wire (Pt, 99.99%, The Mint of Poland, Warsaw, Poland) as an auxiliary electrode. Square-wave voltammetry (SWV) and cyclic voltammetry (CV) were employed as the electroanalytical techniques to investigate DANO behavior at the GCE. Reagents and solutions All chemicals utilized in this study were of analytical reagent grade. DANO with a purity of ≥ 98% was procured from Merck. Fresh stock solutions of DANO (10 mM) were meticulously prepared in graduated glass flasks by dissolving the calculated quantity of DANO in the appropriate volume of 10 mM hydrochloric acid (HCl) (studies at the ITIES) or deionized water (studies at the GCE). Potassium tetrakis(4-chlorophenyl)borate (KTPBCl, > 98%) and bis(triphenylphosphoranylidene) ammonium chloride (BTPPACl, 97%) were obtained from Merck and served as substrates for synthesizing bis(triphenylphosphoranylidene)ammonium tetrakis(4-chlorophenyl)borate (BTPPATPBCl) which was used as the background electrolyte in the organic phase. Tetrapropylammonium chloride (TPrACl, Alfa Aesar, > 99%,) served as the interfacially active internal reference probe used to calibrate the potential axis to the Galvani potential difference. The chemicals for the preparation of Britton–Robinson buffers (BRBs) were procured from POCH. A BRB matrix with a concentration of 40 mM was formulated by dissolving H3BO3, H3PO4, and CH3COOH in 10 mM NaCl (ITIES) or deionized water (GCE). The BRB matrix was titrated with 0.2 M NaOH to achieve the required pH values spanning through the range 2.0–12.0. 1,2–dichloroethane (1,2–DCE, POCH) and deionized water were employed as the solvents for the preparation of solutions in the organic and aqueous phases, respectively. Analytical standards for potentially interfering agents (citric acid, galactose, lactose, glucose, calcium chloride, potassium chloride, magnesium chloride, iron(III) chloride, sodium lactate and orthophosphoric acid (V)) were procured in their analytical reagent-grade from different suppliers (Sigma–Aldrich, Chempur, Fisher Chemical or Alfa Aesar). Fresh stock solutions of interfering species (10 mM) were prepared in a glass graduated flasks by dissolving the appropriate amount of chosen analyte in 10 mL of 10 mM HCl. Only in the case of orthophosphoric acid (V) deionized water was used as a solvent. Shortly after preparation all solutions were subjected to 10 min of sonification and were stored at 4.0 °C. The real samples were ultra high temperature (UHT) milk (1.5% fat content), which were obtained from a local supermarket. The milk samples were not subjected to any preparation process. The pH of the BRBs and the aqueous phase solutions was adjusted utilizing a combined pH electrode (Polilyte Lab, Hamilton, Switzerland).'}"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "method_dict"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Identify Product Entities with Spacy"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "entity_dict = {}\n",
    "for key, value in method_dict.items():\n",
    "    if value:\n",
    "        doc = nlp(value)\n",
    "        entities = []\n",
    "        for ent in doc.ents:\n",
    "            if ent.label_ == 'PRODUCT' and ent.text not in entities:\n",
    "                entities.append(ent.text)\n",
    "        entity_dict[key] = entities"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{11198919: ['Rome IV',\n",
       "  'the Statistical Package for the Social Sciences',\n",
       "  'SPSS',\n",
       "  'Windows'],\n",
       " 11198832: ['Western Blot',\n",
       "  'SCIEX',\n",
       "  '2720',\n",
       "  'FLX800',\n",
       "  'Nanodrop UV',\n",
       "  'NC2000',\n",
       "  'Sequencer',\n",
       "  'Novaseq6000',\n",
       "  'Quant-iT PicoGreen dsDNA Assay Kit',\n",
       "  'First Strand',\n",
       "  'SYBR Green',\n",
       "  'Servicebio',\n",
       "  'GAPDH',\n",
       "  'The Odyssey CLx Imaging System'],\n",
       " 11198784: ['Ficoll-Paque Premium',\n",
       "  'TC20 Automated cell counter',\n",
       "  'an EasySep Negative Human NK Cell Isolation Kit',\n",
       "  'Micro Forge MF-900',\n",
       "  'Axopatch 200B',\n",
       "  'PregS',\n",
       "  'Ononetin',\n",
       "  'GraphPad Prism v9',\n",
       "  'GraphPad Prism version'],\n",
       " 11198029: ['Embase'],\n",
       " 11197919: ['BioRender',\n",
       "  'RS4',\n",
       "  'a Tanita SC-240 Total Body Composition Analyzer',\n",
       "  'the Automated Self-Administered 24-hour',\n",
       "  'ASA24',\n",
       "  'the QIAamp 96 DNA Blood Kit',\n",
       "  'the QIAamp Investigator Kit',\n",
       "  'iTaq™ Universal SYBR® Green Supermix',\n",
       "  'NA12286',\n",
       "  'AMY1',\n",
       "  'a QX100 Droplet Digital PCR System',\n",
       "  'SALIMETRICS',\n",
       "  'the DNeasy PowerSoil 96 HTP Kit',\n",
       "  'Qiagen',\n",
       "  'a BioSpec 1001 Mini-Beadbeater-96 for 3 minutes',\n",
       "  'Mag-Bind TotalPure',\n",
       "  'Omega Bio',\n",
       "  'MiSeq',\n",
       "  'DADA233',\n",
       "  'RStudio',\n",
       "  'ComplexHeatmap40',\n",
       "  'circlize41',\n",
       "  'RS2',\n",
       "  'Ctl',\n",
       "  'PreRS2',\n",
       "  'EndRS2',\n",
       "  'MaAsLin2',\n",
       "  'PreRS4',\n",
       "  'EndRS4',\n",
       "  'glmnet43',\n",
       "  'ASVs'],\n",
       " 11197476: ['a Malvern Nano ZS ZEN3600',\n",
       "  'Bam HI',\n",
       "  'Eco RI',\n",
       "  'CT26',\n",
       "  'Ki67',\n",
       "  'C@ECN-PL'],\n",
       " 11197378: ['Gaode Map'],\n",
       " 11197282: ['pheatmap'],\n",
       " 11197204: ['TRIzol',\n",
       "  'Easy Oligos',\n",
       "  'iTaq Universal One-Step',\n",
       "  'iTaq™ Universal SYBR® Green One-Step',\n",
       "  'Tools',\n",
       "  'GraphPad PRISM',\n",
       "  'Enrichr',\n",
       "  'ShinyGO'],\n",
       " 11197185: ['MiSeq',\n",
       "  'F15',\n",
       "  'Hemotek',\n",
       "  'Mosquito Diluent',\n",
       "  'Trizol',\n",
       "  'AccuStart II',\n",
       "  'MiSeq 500',\n",
       "  'MicrobiomeAnalyst',\n",
       "  '16S'],\n",
       " 11196966: ['Advanced DMEM-F12',\n",
       "  'C57Bl6/J-mice',\n",
       "  'TC20',\n",
       "  'Isoflurane Piramal',\n",
       "  'the SomnoSuite® Low-Flow Anesthesia System',\n",
       "  'BioSpec 94/21',\n",
       "  'Bruker',\n",
       "  'Isoflurane',\n",
       "  'the Liver Dissociation Kit',\n",
       "  'Miltenyi Biotec',\n",
       "  'the Tumor Dissociation Kit',\n",
       "  'SSC-W.',\n",
       "  'Cell Activation Cocktail',\n",
       "  'BioLegend',\n",
       "  'Brefeldin A',\n",
       "  'the RNeasy Mini Kit',\n",
       "  'ProtoScript II First Strand cDNA Synthesis Kit',\n",
       "  'the DNeasy Blood & Tissue Kit',\n",
       "  'Qiagen',\n",
       "  'QuantiTect SYBR Green RT PCR mastermix',\n",
       "  'LightCycler480',\n",
       "  'Roche',\n",
       "  'the QuantiFast SYBR® Green PCR Kit',\n",
       "  'FASTQC',\n",
       "  'DESeq2',\n",
       "  'ggplot2',\n",
       "  'ggraph',\n",
       "  'ComplexHeatmaps',\n",
       "  'a Pierce BCA-Kit',\n",
       "  'an Azure Ao Microplate Reader',\n",
       "  'BioRad',\n",
       "  'Mini-Protean TGX',\n",
       "  'Supplements',\n",
       "  'Immobilon Crescendo Western HRP substrate',\n",
       "  'ROTI® Histokitt',\n",
       "  'a Zeiss Axio Scan Z.1 Microscope Slide Scanner',\n",
       "  'the Vector® TrueVIEW® Autofluorescence Quenching Kit',\n",
       "  'NucBlue™ Fixed Cell Stain ReadyProbes™',\n",
       "  'VECTASHIELD Vibrance Antifade Mounting Medium',\n",
       "  'GraphPad Prism'],\n",
       " 11196847: ['the ABI 3130 Genetic Analyzer',\n",
       "  'Phylogenetic Tree Builder',\n",
       "  '12M',\n",
       "  'RP C-18 column'],\n",
       " 11196703: ['ENSO 16', 'NEOH', 'GraphPad Prism 9'],\n",
       " 11196440: ['Embase'],\n",
       " 11196417: ['GSE208668',\n",
       "  'GPL10904',\n",
       "  'GSE240851',\n",
       "  'GSE56931',\n",
       "  'GSE98582',\n",
       "  'GPL24676',\n",
       "  'GPL10379',\n",
       "  'GPL6244',\n",
       "  'GSE165041',\n",
       "  'GPL18573',\n",
       "  'ggplot2',\n",
       "  'Cytoscape V3.9.0',\n",
       "  '1071',\n",
       "  'the TransZol Up Plus RNA Kit',\n",
       "  'the Nanodrop Spectrophotometer',\n",
       "  'the TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix',\n",
       "  'AT311',\n",
       "  'the ChamQ Universal SYBR qPCR Master Mix',\n",
       "  'Q711',\n",
       "  'a QuantStudio™ 5',\n",
       "  'Image J'],\n",
       " 11196393: ['RPMI1640',\n",
       "  'BD FACSCanto™ II',\n",
       "  'Brilliant Violet 605™',\n",
       "  'GEO Series',\n",
       "  'Cell Ranger',\n",
       "  'Seurat',\n",
       "  'GraphPad Prism 8.0'],\n",
       " 11196377: ['Model GF 400', 'a Chroma‐Meter (CR‐410 Konica Minolta, INC'],\n",
       " 11196265: ['Milli-Q', 'CH30', 'PowershotA95'],\n",
       " 11196252: ['AUTOLAB-PGSTAT302N', 'NOVA 1.11.1', 'EmStat3', 'PSTrace', 'M164']}"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "entity_dict"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Filtering Products with LLM"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "openai.api_key = os.environ['OPENAI_API_KEY']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "system_prompt = \"\"\"You are an expert Product Recognition system.\n",
    "Your task is to accept a list of strings as input and extract the products.\n",
    "Here is an example of the output format for a list of strings.\n",
    "\n",
    "List of strings: [\"Python\", \"Researchers used 3M Littmann Stethoscope\", \"Omron Blood Pressure Monitor is important\", \"I love travelling\", \"Braun ThermoScan\"]\n",
    "Answer: [\"Python\", \"3M Littmann Stethoscope\", \"Omron Blood Pressure Monitor\", \"Braun ThermoScan\"]\n",
    "\n",
    "Only use this output format. Do not return anything besides this output format.\n",
    "Output products in the order they occur in the input list of strings.\n",
    "\"\"\"\n",
    "\n",
    "user_prompt = \"\"\"Q: Given the list of strings below, identify the products.\n",
    "\n",
    "List of strings: {}\n",
    "Answer:\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "def product_filtering(list_of_products):\n",
    "    response = openai.chat.completions.create(\n",
    "        model=\"gpt-4-turbo\",\n",
    "        presence_penalty=0,\n",
    "        # top_p=1e-16,\n",
    "        temperature=1e-16,\n",
    "        messages=[\n",
    "            {\"role\": \"system\", \"content\": system_prompt},\n",
    "            {\"role\": \"user\", \"content\": user_prompt.format(list_of_products)}\n",
    "        ]\n",
    "    )\n",
    "    return response.choices[0].message.content"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "filtered_entity_dict = {}\n",
    "for key, value in entity_dict.items():\n",
    "    syntax_error = False\n",
    "    if value:\n",
    "        filtered_products = product_filtering(value)\n",
    "        try:\n",
    "            filtered_products = ast.literal_eval(filtered_products)\n",
    "        except SyntaxError:\n",
    "            syntax_error = True\n",
    "            print(f\"Syntax Error with ID {key}\")\n",
    "        if not syntax_error:\n",
    "            filtered_entity_dict[key] = filtered_products"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Cleaning Product Entities"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "def remove_stopwords(input_string):\n",
    "    # can add more stopwords here\n",
    "    # use NLTK if these stopwords are not enough\n",
    "    stopwords = ['a', 'an', 'the', 'of']\n",
    "    tokens = input_string.split()\n",
    "    token_list = []\n",
    "    for token in tokens:\n",
    "        if token not in stopwords:\n",
    "            token_list.append(token)\n",
    "    output_string = \" \".join(token_list)\n",
    "    return output_string"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "cleaned_entity_dict = {}\n",
    "for key, value in filtered_entity_dict.items():\n",
    "    if value:\n",
    "        cleaned_entities = [remove_stopwords(entity) for entity in value]\n",
    "        cleaned_entity_dict[key] = cleaned_entities"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198919: ['Rome IV', 'Statistical Package for Social Sciences', 'SPSS', 'Windows']\n",
      "11198832: ['Western Blot', 'SCIEX', '2720', 'FLX800', 'Nanodrop UV', 'NC2000', 'Sequencer', 'Novaseq6000', 'Quant-iT PicoGreen dsDNA Assay Kit', 'First Strand', 'SYBR Green', 'Servicebio', 'GAPDH', 'The Odyssey CLx Imaging System']\n",
      "11198784: ['Ficoll-Paque Premium', 'TC20 Automated cell counter', 'EasySep Negative Human NK Cell Isolation Kit', 'Micro Forge MF-900', 'Axopatch 200B', 'GraphPad Prism v9', 'GraphPad Prism version']\n",
      "11198029: ['Embase']\n",
      "11197919: ['BioRender', 'RS4', 'Tanita SC-240 Total Body Composition Analyzer', 'Automated Self-Administered 24-hour', 'ASA24', 'QIAamp 96 DNA Blood Kit', 'QIAamp Investigator Kit', 'iTaq™ Universal SYBR® Green Supermix', 'QX100 Droplet Digital PCR System', 'SALIMETRICS', 'DNeasy PowerSoil 96 HTP Kit', 'Qiagen', 'BioSpec 1001 Mini-Beadbeater-96', 'Mag-Bind TotalPure', 'Omega Bio', 'MiSeq', 'RStudio', 'ComplexHeatmap40', 'circlize41', 'MaAsLin2', 'glmnet43']\n",
      "11197476: ['Malvern Nano ZS ZEN3600', 'Bam HI', 'Eco RI', 'CT26', 'Ki67', 'C@ECN-PL']\n",
      "11197378: ['Gaode Map']\n",
      "11197282: ['pheatmap']\n",
      "11197204: ['TRIzol', 'Easy Oligos', 'iTaq Universal One-Step', 'iTaq™ Universal SYBR® Green One-Step', 'GraphPad PRISM', 'Enrichr', 'ShinyGO']\n",
      "11197185: ['MiSeq', 'F15', 'Hemotek', 'Mosquito Diluent', 'Trizol', 'AccuStart II', 'MiSeq 500', 'MicrobiomeAnalyst', '16S']\n",
      "11196966: ['Advanced DMEM-F12', 'Isoflurane Piramal', 'SomnoSuite® Low-Flow Anesthesia System', 'BioSpec 94/21', 'Liver Dissociation Kit', 'Tumor Dissociation Kit', 'Cell Activation Cocktail', 'RNeasy Mini Kit', 'ProtoScript II First Strand cDNA Synthesis Kit', 'DNeasy Blood & Tissue Kit', 'QuantiTect SYBR Green RT PCR mastermix', 'QuantiFast SYBR® Green PCR Kit', 'Pierce BCA-Kit', 'Azure Ao Microplate Reader', 'Mini-Protean TGX', 'Immobilon Crescendo Western HRP substrate', 'Zeiss Axio Scan Z.1 Microscope Slide Scanner', 'Vector® TrueVIEW® Autofluorescence Quenching Kit', 'NucBlue™ Fixed Cell Stain ReadyProbes™', 'VECTASHIELD Vibrance Antifade Mounting Medium']\n",
      "11196847: ['ABI 3130 Genetic Analyzer', 'Phylogenetic Tree Builder', 'RP C-18 column']\n",
      "11196703: ['ENSO 16', 'NEOH', 'GraphPad Prism 9']\n",
      "11196440: ['Embase']\n",
      "11196417: ['GSE208668', 'GPL10904', 'GSE240851', 'GSE56931', 'GSE98582', 'GPL24676', 'GPL10379', 'GPL6244', 'GSE165041', 'GPL18573', 'ggplot2', 'Cytoscape V3.9.0', 'TransZol Up Plus RNA Kit', 'Nanodrop Spectrophotometer', 'TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix', 'ChamQ Universal SYBR qPCR Master Mix', 'QuantStudio™ 5', 'Image J']\n",
      "11196393: ['RPMI1640', 'BD FACSCanto™ II', 'Brilliant Violet 605™', 'GEO Series', 'Cell Ranger', 'Seurat', 'GraphPad Prism 8.0']\n",
      "11196377: ['Model GF 400', 'Chroma‐Meter CR‐410 Konica Minolta']\n",
      "11196265: ['Milli-Q', 'CH30', 'PowershotA95']\n",
      "11196252: ['AUTOLAB-PGSTAT302N', 'NOVA 1.11.1', 'EmStat3', 'PSTrace', 'M164']\n"
     ]
    }
   ],
   "source": [
    "for key, value in cleaned_entity_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Identify Products with LLM"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## OpenAI"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "system_prompt = \"\"\"You are an expert Product Recognition system.\n",
    "Your task is to accept a text as input and extract the products.\n",
    "If the manufacturer of the product is indicated, include it with the product name.\n",
    "Here is an example of the output format for a list of strings.\n",
    "\n",
    "Text: I started my day with a fresh cup of coffee (Folgers) brewed from my coffee maker (Keurig), and then I prepared breakfast using my new KitchenAid stand mixer. After breakfast, I grabbed my running shoes (Nike) and went for a jog in the park. On my way back, I stopped by the store to pick up a pack of Tide laundry detergent and a bottle of Coca-Cola. After getting home, I started working on my laptop, which runs on a Microsoft operating system.\n",
    "Answer: [\"Folgers coffee\", \"Keurig coffee maker\", \"KitchenAid stand mixer\", \"Nike running shoes\", \"Tide laundry detergent\", \"Coca-Cola\", \"Microsoft operating system\"]\n",
    "\n",
    "Only use this output format. Do not return anything besides this output format.\n",
    "Output products in the order they occur in the input text.\n",
    "\"\"\"\n",
    "\n",
    "user_prompt = \"\"\"Q: Given the text below, identify the products.\n",
    "\n",
    "Text: {}\n",
    "Answer:\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "def product_recognition(input_text):\n",
    "    response = openai.chat.completions.create(\n",
    "        model=\"gpt-4-turbo\",\n",
    "        presence_penalty=0,\n",
    "        # top_p=1e-16,\n",
    "        temperature=1e-16,\n",
    "        messages=[\n",
    "            {\"role\": \"system\", \"content\": system_prompt},\n",
    "            {\"role\": \"user\", \"content\": user_prompt.format(input_text)}\n",
    "        ]\n",
    "    )\n",
    "    return response.choices[0].message.content"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "product_dict = {}\n",
    "for key, value in method_dict.items():\n",
    "    syntax_error = False\n",
    "    if value:\n",
    "        identified_products = product_recognition(value)\n",
    "        try:\n",
    "            identified_products = ast.literal_eval(identified_products)\n",
    "        except SyntaxError:\n",
    "            syntax_error = True\n",
    "            print(f\"Syntax Error with ID {key}\")\n",
    "        if not syntax_error:\n",
    "            product_dict[key] = identified_products"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198832: ['Shimadzu high-performance liquid chromatography (HPLC)', 'SCIEX mass spectrometer (MS)', 'ABI PCR amplification instrument', 'BioTek enzyme labeling instrument', 'Beijing Liuyi Instrument Factory electrophoresis apparatus', 'Beijing Bijing Biotechnology Co. gel imaging system', 'Thermo Fisher Scientific Nanodrop UV quantitative system', 'Illumina Sequencer', 'Bio-rad Corporation fluorescence quantitative PCR instrument', 'VIANEX S.A. vancomycin hydrochloride for injection', 'Hangzhou Yuanda Biological Pharmaceutical bifidobacterium quadruplex viable tablets', 'Shiyao Group Enbipu Pharmaceutical NBP capsules', 'Shiyao Group Enbipu Pharmaceutical NBP reference', 'Sichuan Vicchi Biochemical Technology glipizide reference', 'Omega Bio-Tek Soil DNA Kit', 'ABI Quant-iT PicoGreen dsDNA Assay Kit', 'Servicebio First Strand cDNA Synthesis Kit', 'Servicebio SYBR Green qPCR Master Mix']\n",
      "11198784: ['Ficoll-Paque Premium density gradient medium', 'Invitrogen trypan blue dye', 'Bio-Rad TC20 Automated cell counter', 'EasySep Negative Human NK Cell Isolation Kit', 'Becton Dickinson CD56 APC', 'Becton Dickinson CD3 PE Cy7', 'BD Bioscience stain buffer', 'BD LSR-Fortessa X-20 flow cytometer', 'Harvard Apparatus borosilicate glass capillaries', 'Sutter Instrumental P-97 glass pipette', 'Narishige Micro Forge MF-900', 'Molecular Devices CV203BU head-stage', 'Narishige manipulators', 'Molecular Devices Axopatch 200B amplifier', 'Molecular Devices pClamp 10.7 software', 'Molecular Devices Digidata 1440A', 'Sigma-Aldrich membrane filter', 'Sapphire Bioscience Reagents ATP', 'Sapphire Bioscience Reagents GTP', 'Tocris Bioscience PregS', 'Tocris Bioscience Ononetin', 'Sigma-Aldrich chemicals', 'IBM SPSS software', 'GraphPad Prism software', 'OriginLab Origin 2021 software']\n",
      "11197919: ['HI-MAIZE® 260 starch', 'VERSAFIBE™ 1490 starch', 'AMIOCA™ TF starch', 'Salimetrics Salivary Alpha-Amylase Enzymatic Kit']\n",
      "11197476: ['LiposoFast liposome extruder', 'Merck Millipore polycarbonate membrane filters', 'ultrafiltration centrifuge tube', 'JEM-1400 transmission electron microscope', 'Malvern Nano ZS ZEN3600', 'UV-VIS Thermo Fisher Scientific', 'CytoFLEX flow cytometer', 'Beyotime Biotechnology cloning kit', 'Nikon confocal laser scanning microscopy', 'CCK-8 assay kit Beyotime Biotechnology', 'IVIS in vivo imaging system', 'Biolegend Inc. antibodies', 'Jiancheng Bioengineering Institute assay kits']\n",
      "11197378: ['Qingyue Open Environmental Data Centre data', 'Gaode Map', 'EXCEL database', 'IBM SPSS 20 software', 'R language']\n",
      "11197282: ['Metabolon untargeted metabolomics profiling']\n",
      "11197204: ['TRIzol', 'TaqMan Gene Expression Assay', 'iTaq Universal One-Step Kit', 'iTaq Universal SYBR Green One-Step Kit', 'BioRad', 'GraphPad PRISM', 'Enrichr', 'ShinyGO']\n",
      "11197185: ['Illumina MiSeq platform', 'Vero cell culture (ATCC)', 'C636 cell culture (ATCC)', 'Sterilite container', 'Tetra Pond Koi growth feed', 'Hemotek membrane feeding system', 'Sigma Aldrich triethylamine', 'Trizol (Thermo Fisher Scientific)', 'REPLI-g WTA single-cell kit (Qiagen)', 'AccuStart II PCR supermix (Quantabio)']\n",
      "11196966: ['Advanced DMEM-F12 (Gibco/Thermo Fisher Scientific)', 'penicillin/streptomycin (Gibco/Thermo Fisher Scientific)', 'HEPES (Invitrogen/Thermo Fisher Scientific)', 'Glutamax (Invitrogen/Thermo Fisher Scientific)', 'N2 (Gibco/Thermo Fisher Scientific)', 'B27 (Gibco/Thermo Fisher Scientific)', 'N-acetylcysteine (Sigma-Aldrich)', 'hygromycin (Invitrogen/Thermo Fisher Scientific)', 'puromycin (InvivoGen)', 'Cultrex RGF basement membrane extract (R&D Systems)', 'cell counter TC20 (BioRad)', 'buprenorphine (Temgesic, Schering-Plough)', 'Isoflurane (Isoflurane Piramal, Piramal)', 'SomnoSuite® Low-Flow Anesthesia System (Kent Scientific Corporation)', 'Seprafilm (Baxter)', 'Vicryl (Ethicon/Johnson&Johnson)', 'BioSpec 94/21 MRI scanner (Bruker Biospin)', 'VEVO 3100 ultrasound (Fujifilm/VisualSonics)', 'Liver Dissociation Kit, mouse (Miltenyi Biotec)', 'Tumor Dissociation Kit, mouse (Miltenyi Biotec)', 'ACK lysis buffer (Gibco/Thermo Fisher Scientific)', 'TruStain FcX PLUS (BioLegend)', 'eBioscience™ Foxp3/Transcription Factor Staining Buffer Set (Invitrogen/Thermo Fisher Scientific)', 'Beckman Coulter Gallios (Beckman Coulter)', 'BD LSRFortessa (Becton Dickinson)', 'Cell Activation Cocktail (BioLegend)', 'Brefeldin A (BioLegend)', 'RNeasy Mini Kit (Qiagen)', 'ProtoScript II First Strand cDNA Synthesis Kit (New England Biolabs)', 'DNeasy Blood & Tissue Kit (Qiagen)', 'QuantiTect SYBR Green RT PCR mastermix (Qiagen)', 'Roche LightCycler 480 (Roche)', 'Pierce BCA-Kit (Thermo Fisher Scientific)', 'Azure Ao Microplate Reader (Azure Biosystems)', 'Laemmli Buffer (BioRad)', 'Mini-Protean TGX precast gel (BioRad)', 'Immobilon Crescendo Western HRP substrate (Merck Millipore)', 'ChemiDoc BioRad imager (BioRad)', 'Vector® TrueVIEW® Autofluorescence Quenching Kit (Biozol)', 'NucBlue™ Fixed Cell Stain ReadyProbes™ (Thermo Fisher Scientific)', 'VECTASHIELD Vibrance Antifade Mounting Medium (Biozol)', 'Zeiss Axio Scan Z.1 Microscope Slide Scanner (Zeiss)', 'GraphPad Prism']\n",
      "11196847: ['ABI 3130 Genetic Analyzer', 'Megazyme mushroom and yeast β-glucan assay kit']\n",
      "11196703: ['ENSO 16 sugar substitute', 'glucose powder']\n",
      "11196417: ['GEO database', 'limma package', 'ggplot2 package', 'clusterProfiler package', 'enrichplot package', 'ImmuCellAI methodology', 'ssGSEA methodology', 'Mantel algorithm', 'STRING 4 online platform', 'Cytoscape V3.9.0 software', 'CytoHubba plugin', 'Random Forest package', 'SVM-RFE approach', 'LASSO model', 'Logistic regression approach', 'ROC curve', 'AUC', 'Limma package', 'miRNet database', 'DGIdb database', 'TransZol Up Plus RNA Kit', 'Nanodrop Spectrophotometer', 'TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix', 'ChamQ Universal SYBR qPCR Master Mix', 'QuantStudio™ 5 Real-Time PCR System', 'Image J software']\n",
      "11196393: ['Ficoll gradient', 'ACK lysing buffer', 'BD FACSAria III cell sorter', 'Fc-block (Miltenyi)', 'A1exF5 antibody (Thermo Fisher)', 'DCFDA (Abcam)', 'APC antihuman CD11b(ICRF44) antibody (BioLegend)', 'Brilliant Violet 605™ antihuman HLA-DR(L243) antibody (BioLegend)', 'APC/Cyanine7 antihuman CD14(63D3) antibody (BioLegend)', 'PerCP/Cyanine5.5 antihuman CXCR2(5E8/CXCR2) antibody (BioLegend)', 'FITC antihuman CD177(MEM-166) antibody (BioLegend)', 'Cell Ranger software', 'Seurat toolkit', 'monocle2 package', 'GraphPad Prism software']\n",
      "11196377: ['Cobb-500 broiler chicks', 'Evonik Nutrition & Care GmbH near-infrared analysis', 'Cobb-Vantress finisher diet', 'UFFDA software', 'A&D Weighing digital weighing scale', 'Konica Minolta Chroma-Meter CR-410', 'SAS 9.4 software']\n",
      "11196265: ['Polyethylene microplastics (PE-MPs) powder', 'Milli-Q purified water', 'tadalafil (Cilais)', 'commercial fish pellet feed (El Nasr Company)', 'BC-2800 automated technical analyzer (Mindray)', 'spectrophotometer (Biodiagonstic Company)', 'Sigma-Aldrich total antioxidant capacity kits', 'Olympus CH30 microscope', 'Vector Laboratories standard goat serum', 'E-AB-40021 IL-6 antibody (Elabscience Biotechnology Inc)', 'Olympus BX41 microscope', 'Canon digital camera (PowershotA95)', 'SPSS 16.0 program (SPSS 2007, Inc.)']\n",
      "11196252: ['Metrohm Autolab AUTOLAB-PGSTAT302N instrument', 'NOVA 1.11.1 software', 'EmStat3 potentiostat', 'PSTrace software', 'PalmSens B.V.', 'automated electrode stand M164', 'MTM Anko Instruments', 'Basi® glassy carbon electrode', 'Merck DANO', 'Potassium tetrakis(4-chlorophenyl)borate KTPBCl', 'bis(triphenylphosphoranylidene) ammonium chloride BTPPACl', 'Tetrapropylammonium chloride TPrACl', 'Alfa Aesar', 'POCH Britton–Robinson buffers', '1,2–dichloroethane 1,2–DCE', 'Sigma–Aldrich', 'Chempur', 'Fisher Chemical', 'Polilyte Lab pH electrode', 'Hamilton']\n"
     ]
    }
   ],
   "source": [
    "for key, value in product_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Mistral"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "system_prompt = \"\"\"You are an expert Product Recognition system.\n",
    "Your task is to accept a text as input and extract the products.\n",
    "Here is an example of the output format for a list of strings.\n",
    "\n",
    "Text: I started my day with a fresh cup of coffee (Folgers) brewed from my coffee maker (Keurig), and then I prepared breakfast using my new KitchenAid stand mixer. After breakfast, I grabbed my running shoes (Nike) and went for a jog in the park. On my way back, I stopped by the store to pick up a pack of Tide laundry detergent and a bottle of Coca-Cola. After getting home, I started working on my laptop, which runs on a Microsoft operating system.\n",
    "Answer: [\"Folgers coffee\", \"Keurig coffee maker\", \"KitchenAid stand mixer\", \"Nike running shoes\", \"Tide laundry detergent\", \"Coca-Cola\", \"Microsoft operating system\"]\n",
    "\n",
    "Only use this output format. Do not return anything besides this output format.\n",
    "Output products in the order they occur in the input text.\n",
    "\"\"\"\n",
    "\n",
    "user_prompt = \"\"\"Q: Given the text below, identify the products.\n",
    "\n",
    "Text: {}\n",
    "Answer:\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "api_key = os.environ[\"MISTRAL_API_KEY\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "def product_recognition(input_text):\n",
    "    client = MistralClient(api_key=api_key)\n",
    "    response = client.chat(\n",
    "        model=\"mistral-large-latest\",\n",
    "        temperature=1e-16,\n",
    "        messages=[\n",
    "        ChatMessage(role=\"system\", content=system_prompt),\n",
    "        ChatMessage(role=\"user\", content=user_prompt.format(input_text))\n",
    "    ]\n",
    "    )\n",
    "    return response.choices[0].message.content"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198919: []\n",
      "11198832: [\"Shimadzu high-performance liquid chromatography (HPLC)\", \"AB SCIEX mass spectrometer (MS)\", \"ABI PCR amplification instrument\", \"BioTek enzyme labeling instrument\", \"Beijing Liuyi electrophoresis apparatus\", \"Beijing Bijing Biotechnology Co., LTD. gel imaging system\", \"Thermo Fisher Scientific Nanodrop UV quantitative system\", \"Illumina Sequencer\", \"Bio-rad Corporation fluorescence quantitative PCR instrument\", \"vancomycin hydrochloride for injection (VIANEX S.A.)\", \"bifidobacterium quadruplex viable tablets (Hangzhou Yuanda Biological Pharmaceutical)\", \"NBP capsules (Shiyao Group Enbipu Pharmaceutical)\", \"NBP reference (Shiyao Group Enbipu Pharmaceutical)\", \"glipizide reference (Sichuan Vicchi Biochemical Technology)\", \"Omega Bio-Tek Soil DNA Kit\", \"ABI Quant-iT PicoGreen dsDNA Assay Kit\", \"Servicebio First Strand cDNA Synthesis Kit\", \"Servicebio SYBR Green qPCR Master Mix\"]\n",
      "11198784: []\n",
      "11198029: []\n",
      "11197919: [\"HI-MAIZE® 260 starch\", \"VERSAFIBE™ 1490 starch\", \"AMIOCA™ TF starch\", \"Tanita SC-240 Total Body Composition Analyzer\", \"Automated Self-Administered 24-hour (ASA24) Dietary Assessment Tool 2020\", \"QIAamp 96 DNA Blood Kit\", \"QIAamp Blood Mini Kit\", \"QIAamp Investigator Kit\", \"iTaq™ Universal SYBR® Green Supermix\", \"Salimetrics Salivary Alpha-Amylase Enzymatic Kit\", \"DNeasy PowerSoil 96 HTP Kit\", \"Classic++™ Hot Start Taq DNA Polymerase Master Mix\", \"Mag-Bind TotalPure\", \"QIIME 2\", \"DADA2\", \"mafft\", \"fasttree\", \"Greengenes 13_8 99% OTUs reference sequences\", \"RStudio version 4.2.1\", \"ComplexHeatmap\", \"circlize\", \"MaAsLin2 (Microbiome Multivariable Association with Linear Models 2)\", \"glmnet\"]\n",
      "11197476: [\"CXB@Lipo (C@L)\", \"chloroform\", \"HSPC\", \"LiposoFast (Avestin, Canada)\", \"polycarbonate membrane filters (Merck Millipore Ltd., USA)\", \"ultrafiltration centrifuge tube\", \"ECN\", \"glycol chitosan\", \"transmission electron microscopy (TEM; JEM-1400, Japan)\", \"Malvern Nano ZS ZEN3600 (Malvern, UK)\", \"UV-VIS (Thermo Fisher Scientific, USA)\", \"ICG@ECN (I@ECN)\", \"flow cytometry (CytoFLEX, Beckman, USA)\", \"UV‒VIS\", \"pGEX-Pvhb-Lysis\", \"cloning kit (Beyotime Biotechnology Co. Ltd., Shanghai, China)\", \"ICG@Liposome (I@L)\", \"confocal laser scanning microscopy (CLSM) (Nikon, Japan)\", \"CCK-8 assay kit (Beyotime Biotechnology Co. Ltd., Shanghai, China)\", \"Western blot (WB) analysis\", \"FITC-conjugated anti-COX-2 antibody (Affinity, Jiangsu, China)\", \"in vivo images system (IVIS)\", \"optimal cutting temperature (OCT) compound\", \"CT26 tumor model\", \"FITC-conjugated anti-mouse CD3\", \"FITC-conjugated anti-mouse CD4\", \"PE-conjugated anti-mouse CD8 (Biolegend Inc., USA)\", \"TUNEL and H&E staining\", \"Ki67\", \"C@ECN-PL\", \"aspartate aminotransferase (AST)\", \"alanine aminotransferase (ALT)\", \"blood urea nitrogen (BUN)\", \"creatinine (CRE)\", \"assay kits (Jiancheng Bioengineering Institute, Nanjing, China)\", \"H&E staining analysis\", \"ImageJ 6.0\"]\n",
      "11197378: [\"EXCEL\", \"IBM SPSS 20\", \"R language\"]\n",
      "11197282: []\n",
      "11197204: [\"Gene Expression Omnibus (GEO)\", \"quantitative PCR method\", \"UK National Research Ethics Committee (NRES 16/SS/0172)\", \"BH false discovery rate correction\", \"TRIzol (Life Technologies)\", \"TaqMan™ Gene Expression Assay (FAM) from Life Technologies Ltd\", \"Easy Oligos from Merck Life Science UK Ltd\", \"iTaq Universal One-Step or iTaq™ Universal SYBR® Green One-Step Kit from BioRad\", \"GraphPad PRISM\", \"ShinyGO\", \"Enrichr\", \"Tools\"]\n",
      "11197185: [\"ZIKV\", \"Vero cell culture (ATCC)\", \"C636 cell culture (ATCC)\", \"Tetra Pond Koi growth feed\", \"defibrinated sheep blood\", \"sodium bicarbonate\", \"Hemotek membrane feeding system (Discovery Workshops, Accrington, UK)\", \"triethylamine (Sigma Aldrich, St. Louis, MO, USA)\", \"filter sterilized fetal bovine serum\", \"Mosquito Diluent [MD]\", \"penicillin/streptomycin\", \"gentamicin\", \"Fungizone (Sigma Aldrich, St. Louis, MO, USA)\", \"phosphate-buffered Saline (PBS) (Sigma Aldrich, St. Louis, MO, USA)\", \"Illumina MiSeq\", \"REPLI-g whole transcriptome amplification (WTA) single-cell kit (Qiagen, Hilden, Germany)\", \"Illumina barcoded 16S primer set\", \"AccuStart II PCR supermix (Quantabio, Beverly, MA, USA)\", \"QIITA\", \"MicrobiomeAnalyst\", \"GreenGenes 16S reference database\", \"BIOM-formatted OTU table\", \"NCBI GenBank Short Read Archives\"]\n",
      "11196966: [\"Advanced DMEM-F12 (Gibco/Thermo Fisher Scientific)\", \"penicillin (Gibco/Thermo Fisher Scientific)\", \"streptomycin (Gibco/Thermo Fisher Scientific)\", \"HEPES (Invitrogen/Thermo Fisher Scientific)\", \"Glutamax (Invitrogen/Thermo Fisher Scientific)\", \"N2 (Gibco/Thermo Fisher Scientific)\", \"B27 (Gibco/Thermo Fisher Scientific)\", \"N-acetylcysteine (Sigma-Aldrich)\", \"hygromycin (Invitrogen/Thermo Fisher Scientific)\", \"puromycin (InvivoGen)\", \"Cultrex RGF basement membrane extract (BME) Type 2 (R&D Systems)\", \"cell counter TC20 (BioRad)\", \"collagen I/5x neutralization buffer\", \"buprenorphine (Temgesic, Schering-Plough)\", \"Isoflurane Piramal (Piramal)\", \"SomnoSuite® Low-Flow Anesthesia System (Kent Scientific Corporation)\", \"Seprafilm (Baxter)\", \"6–0 Vicryl (Ethicon/Johnson&Johnson) suture\", \"9.4 tesla small bore animal scanner (BioSpec 94/21, Bruker Biospin)\", \"dedicated mouse quadrature-resonator (Bruker)\", \"VEVO 3100 (Fujifilm/VisualSonics) ultrasound\", \"small animal transducer\", \"RNAprotect buffer (Qiagen, Netherlands)\", \"Liver Dissociation Kit, mouse (Miltenyi Biotec)\", \"Tumor Dissociation Kit, mouse (Miltenyi Biotec)\", \"ACK lysis buffer (Gibco/Thermo Fisher Scientific)\", \"TruStain FcX PLUS (anti-mouse CD16/32) antibody (BioLegend)\", \"FACS buffer (PBS + 0,5% BSA)\", \"1x Fixation/Permeabilization buffer\", \"eBioscience™ Foxp3/Transcription Factor Staining Buffer Set (Invitrogen/Thermo Fisher Scientific)\", \"FACS antibodies (BioLegend)\", \"Beckman Coulter Gallios (Beckman Coulter)\", \"BD LSRFortessa (Becton Dickinson)\", \"Kaluza Software (Beckman Coulter)\", \"RPMI-Medium (+ 1% Pen/Strep + 1% HEPES + 5 µl beta-mercaptoethanol)\", \"Cell Activation Cocktail (BioLegend)\", \"Brefeldin A (BioLegend)\", \"RLT buffer (Qiagen, Venlo, Netherlands)\", \"RNeasy Mini Kit (Qiagen)\", \"NanoDrop™ spectrophotometer (Thermo Fisher Scientific)\", \"ProtoScript II First Strand cDNA Synthesis Kit (New England Biolabs)\", \"QuantiTect SYBR Green RT PCR mastermix (Qiagen)\", \"DNeasy Blood & Tissue Kit (Qiagen)\", \"primer sequences (purchased from Eurofins Genomics)\", \"Roche LightCycler 480 (Roche)\", \"QuantiFast SYBR® Green PCR Kit (Qiagen)\", \"Bifidobacterium actinocoloniiforme (DSMZ-Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, DSM22766)\"]\n",
      "11196847: [\"Potato dextrose agar (PDA) medium\", \"Potato dextrose broth (PDB)\", \"ABI 3130 Genetic Analyzer\", \"Taq DNA polymerase Assay Buffer\", \"ABI 3130xl gel sequencing\", \"Phylogenetic Tree Builder\", \"Ethanol (80%, v/v)\", \"Whatman paper No. 4\", \"Lowry's assay\", \"Folin-Ciocalteu reagent\", \"Bovine Serum Albumin (BSA)\", \"Anthrone reagent\", \"Anthrone technique\", \"Quercetin\", \"Folin-Ciocalteu method\", \"Gallic acid equivalent (GAE)\", \"Aescin\", \"Ferric chloride\", \"Mg(CH3COO)2 in CH3OH\", \"Rhein\", \"NH4OH\", \"Liebermann-Burchard reagent\", \"DPPH radicals\", \"Liebermann-Burchard reagent\", \"β-glucan assay kit (Megazyme Int., Dublin, Ireland)\", \"High-performance liquid chromatography (PDA)\", \"RP C-18 column\", \"Quercetin\", \"Sesquiterpene glycosides\", \"Ergosterol\", \"RidentinB\", \"Statistical Package for the Social Sciences (SPSS)\"]\n",
      "11196703: [\"ENSO 16 sugar substitute\", \"glucose in water solution\", \"NEOH by Alpha Republic GmbH\", \"STAMAG Stadlauer Malzfabrik GesmbH\", \"Special Ingredients Ltd glucose powder\", \"GraphPad Prism 9\"]\n",
      "11196440: Based on the text you provided, it doesn't seem to contain any specific brand names or products, unlike the example text where clear brands like \"Folgers coffee\", \"Keurig coffee maker\", etc., were mentioned. The text you provided is primarily about a methodology for a meta-analysis study on the relationship between beverages and Type 2 Diabetes Mellitus (T2DM). Therefore, I'm unable to extract any products from this text in the same way as the example.\n",
      "11196417: [\"GEO database\", \"limma package\", \"R software\", \"ggplot2 package\", \"GSEA official website\", \"MSigDB database\", \"clusterProfiler package\", \"enrichplot package\", \"ImmuCellAI\", \"ssGSEA\", \"STRING 4 online platform\", \"Cytoscape V3.9.0 software\", \"CytoHubba plugin\", \"Random Forest algorithm\", \"SVM-RFE\", \"LASSO model\", \"Logistic regression\", \"Receiver Operating Characteristic (ROC) curve\", \"Area Under the Curve (AUC)\", \"miRNet database\", \"Drug-Gene Interaction database (DGIdb)\", \"ggplot2 package\", \"TransZol Up Plus RNA Kit\", \"Nanodrop Spectrophotometer\", \"TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix\", \"ChamQ Universal SYBR qPCR Master Mix\", \"QuantStudio™ 5 Real-Time PCR System\", \"Image J software\"]\n",
      "11196393: [\"Ficoll gradient\", \"ACK lysing buffer\", \"RPMI1640 supplemented with 20% FBS\", \"BD FACSAria III cell sorter\", \"Fc-block (Miltenyi)\", \"surface antibodies against CD11b, HLA-DR, CD14, CD177, and CXCR2\", \"anti-Arg1 (A1exF5; Thermo Fisher)\", \"DCFDA (Abcam)\", \"BD FACSCanto II\", \"APC antihuman CD11b(ICRF44)\", \"Brilliant Violet 605™ antihuman HLA-DR(L243)\", \"APC/Cyanine7 antihuman CD14(63D3)\", \"PerCP/Cyanine5.5 antihuman CXCR2(5E8/CXCR2)\", \"FITC antihuman CD177(MEM-166)\", \"Cell Ranger (version 7.0)\", \"Seurat (version 4.0.1)\", \"monocle2 (version 2.26.0)\", \"GraphPad Prism 8.0\"]\n",
      "11196377: [\"Cobb-500 chicks\", \"Evonik Co. near-infrared analysis\", \"UFFDA software\", \"Cobb-Vantress commercial management guide\", \"Vitamin premix\", \"Mineral premix\", \"SAS 9.4 software\"]\n",
      "11196265: [\"Polyethylene microplastics (PE-MPs) powder\", \"Purified water (Milli-Q)\", \"Tadalafil (Cilais) substance\", \"Sildenafil\", \"Commercial fish pellet feed (containing 30% protein and 12% lipid; El Nasr Company, Egypt)\", \"Automated technical analyzer (BC-2800 from Mindray)\", \"Spectrophotometer (in a wavelength range of 340–546 nm; Biodiagonstic Company, Egypt)\", \"Kits for TAC (total antioxidant capacity) determination (Sigma-Aldrich, USA)\", \"Olympus CH30 microscope\", \"IL-6 antibody (E-AB-40021, Elabscience Biotechnology Inc, USA)\", \"Streptavidin–biotin-peroxidase pack\", \"Mayer's hematoxylin\", \"SPSS 16.0 program for Windows (SPSS 2007, Inc., IL, USA)\", \"Canon digital camera (PowershotA95)\"]\n",
      "11196252: [\"AUTOLAB-PGSTAT302N instrument\", \"NOVA 1.11.1 software\", \"platinum (Pt) wires\", \"silver/silver chloride wires (Ag/AgCl)\", \"EmStat3 potentiostat\", \"PSTrace software\", \"automated electrode stand (M164, MTM Anko Instruments)\", \"glassy carbon electrode (GCE)\", \"silver/silver chloride electrode (Ag/AgCl/3.0 mol L−1 KCl)\", \"platinum wire (Pt, 99.99%, The Mint of Poland)\", \"DANO (Merck)\", \"hydrochloric acid (HCl)\", \"Potassium tetrakis(4-chlorophenyl)borate (KTPBCl, Merck)\", \"bis(triphenylphosphoranylidene) ammonium chloride (BTPPACl, Merck)\", \"bis(triphenylphosphoranylidene)ammonium tetrakis(4-chlorophenyl)borate (BTPPATPBCl)\", \"Tetrapropylammonium chloride (TPrACl, Alfa Aesar)\", \"Britton–Robinson buffers (BRBs)\", \"1,2–dichloroethane (1,2–DCE, POCH)\", \"citric acid (Sigma–Aldrich)\", \"galactose (Chempur)\", \"lactose (Fisher Chemical)\", \"glucose (Alfa Aesar)\", \"calcium chloride (Sigma–Aldrich)\", \"potassium chloride (Chempur)\", \"magnesium chloride (Fisher Chemical)\", \"iron(III) chloride (Alfa Aesar)\", \"sodium lactate (Sigma–Aldrich)\", \"orthophosphoric acid (V) (Alfa Aesar)\", \"combined pH electrode (Polilyte Lab, Hamilton, Switzerland)\"]\n"
     ]
    }
   ],
   "source": [
    "for key, value in method_dict.items():\n",
    "    identified_products = product_recognition(value)\n",
    "    print(f\"{key}: {identified_products}\")\n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "product_dict = {\n",
    "    \"11198919\": [],\n",
    "    \"11198832\": [\"Shimadzu high-performance liquid chromatography (HPLC)\", \"AB SCIEX mass spectrometer (MS)\", \"ABI PCR amplification instrument\", \"BioTek enzyme labeling instrument\", \"Beijing Liuyi electrophoresis apparatus\", \"Beijing Bijing Biotechnology Co., LTD. gel imaging system\", \"Thermo Fisher Scientific Nanodrop UV quantitative system\", \"Illumina Sequencer\", \"Bio-rad Corporation fluorescence quantitative PCR instrument\", \"vancomycin hydrochloride for injection (VIANEX S.A.)\", \"bifidobacterium quadruplex viable tablets (Hangzhou Yuanda Biological Pharmaceutical)\", \"NBP capsules (Shiyao Group Enbipu Pharmaceutical)\", \"NBP reference (Shiyao Group Enbipu Pharmaceutical)\", \"glipizide reference (Sichuan Vicchi Biochemical Technology)\", \"Omega Bio-Tek Soil DNA Kit\", \"ABI Quant-iT PicoGreen dsDNA Assay Kit\", \"Servicebio First Strand cDNA Synthesis Kit\", \"Servicebio SYBR Green qPCR Master Mix\"],\n",
    "    \"11198784\": [],\n",
    "    \"11198029\": [],\n",
    "    \"11197919\": [\"HI-MAIZE® 260 starch\", \"VERSAFIBE™ 1490 starch\", \"AMIOCA™ TF starch\", \"Tanita SC-240 Total Body Composition Analyzer\", \"Automated Self-Administered 24-hour (ASA24) Dietary Assessment Tool 2020\", \"QIAamp 96 DNA Blood Kit\", \"QIAamp Blood Mini Kit\", \"QIAamp Investigator Kit\", \"iTaq™ Universal SYBR® Green Supermix\", \"Salimetrics Salivary Alpha-Amylase Enzymatic Kit\", \"DNeasy PowerSoil 96 HTP Kit\", \"Classic++™ Hot Start Taq DNA Polymerase Master Mix\", \"Mag-Bind TotalPure\", \"QIIME 2\", \"DADA2\", \"mafft\", \"fasttree\", \"Greengenes 13_8 99% OTUs reference sequences\", \"RStudio version 4.2.1\", \"ComplexHeatmap\", \"circlize\", \"MaAsLin2 (Microbiome Multivariable Association with Linear Models 2)\", \"glmnet\"],\n",
    "    \"11197476\": [\"CXB@Lipo (C@L)\", \"chloroform\", \"HSPC\", \"LiposoFast (Avestin, Canada)\", \"polycarbonate membrane filters (Merck Millipore Ltd., USA)\", \"ultrafiltration centrifuge tube\", \"ECN\", \"glycol chitosan\", \"transmission electron microscopy (TEM; JEM-1400, Japan)\", \"Malvern Nano ZS ZEN3600 (Malvern, UK)\", \"UV-VIS (Thermo Fisher Scientific, USA)\", \"ICG@ECN (I@ECN)\", \"flow cytometry (CytoFLEX, Beckman, USA)\", \"UV‒VIS\", \"pGEX-Pvhb-Lysis\", \"cloning kit (Beyotime Biotechnology Co. Ltd., Shanghai, China)\", \"ICG@Liposome (I@L)\", \"confocal laser scanning microscopy (CLSM) (Nikon, Japan)\", \"CCK-8 assay kit (Beyotime Biotechnology Co. Ltd., Shanghai, China)\", \"Western blot (WB) analysis\", \"FITC-conjugated anti-COX-2 antibody (Affinity, Jiangsu, China)\", \"in vivo images system (IVIS)\", \"optimal cutting temperature (OCT) compound\", \"CT26 tumor model\", \"FITC-conjugated anti-mouse CD3\", \"FITC-conjugated anti-mouse CD4\", \"PE-conjugated anti-mouse CD8 (Biolegend Inc., USA)\", \"TUNEL and H&E staining\", \"Ki67\", \"C@ECN-PL\", \"aspartate aminotransferase (AST)\", \"alanine aminotransferase (ALT)\", \"blood urea nitrogen (BUN)\", \"creatinine (CRE)\", \"assay kits (Jiancheng Bioengineering Institute, Nanjing, China)\", \"H&E staining analysis\", \"ImageJ 6.0\"],\n",
    "    \"11197378\": [\"EXCEL\", \"IBM SPSS 20\", \"R language\"],\n",
    "    \"11197282\": [],\n",
    "    \"11197204\": [\"Gene Expression Omnibus (GEO)\", \"quantitative PCR method\", \"UK National Research Ethics Committee (NRES 16/SS/0172)\", \"BH false discovery rate correction\", \"TRIzol (Life Technologies)\", \"TaqMan™ Gene Expression Assay (FAM) from Life Technologies Ltd\", \"Easy Oligos from Merck Life Science UK Ltd\", \"iTaq Universal One-Step or iTaq™ Universal SYBR® Green One-Step Kit from BioRad\", \"GraphPad PRISM\", \"ShinyGO\", \"Enrichr\", \"Tools\"],\n",
    "    \"11197185\": [\"ZIKV\", \"Vero cell culture (ATCC)\", \"C636 cell culture (ATCC)\", \"Tetra Pond Koi growth feed\", \"defibrinated sheep blood\", \"sodium bicarbonate\", \"Hemotek membrane feeding system (Discovery Workshops, Accrington, UK)\", \"triethylamine (Sigma Aldrich, St. Louis, MO, USA)\", \"filter sterilized fetal bovine serum\", \"Mosquito Diluent [MD]\", \"penicillin/streptomycin\", \"gentamicin\", \"Fungizone (Sigma Aldrich, St. Louis, MO, USA)\", \"phosphate-buffered Saline (PBS) (Sigma Aldrich, St. Louis, MO, USA)\", \"Illumina MiSeq\", \"REPLI-g whole transcriptome amplification (WTA) single-cell kit (Qiagen, Hilden, Germany)\", \"Illumina barcoded 16S primer set\", \"AccuStart II PCR supermix (Quantabio, Beverly, MA, USA)\", \"QIITA\", \"MicrobiomeAnalyst\", \"GreenGenes 16S reference database\", \"BIOM-formatted OTU table\", \"NCBI GenBank Short Read Archives\"],\n",
    "    \"11196966\": [\"Advanced DMEM-F12 (Gibco/Thermo Fisher Scientific)\", \"penicillin (Gibco/Thermo Fisher Scientific)\", \"streptomycin (Gibco/Thermo Fisher Scientific)\", \"HEPES (Invitrogen/Thermo Fisher Scientific)\", \"Glutamax (Invitrogen/Thermo Fisher Scientific)\", \"N2 (Gibco/Thermo Fisher Scientific)\", \"B27 (Gibco/Thermo Fisher Scientific)\", \"N-acetylcysteine (Sigma-Aldrich)\", \"hygromycin (Invitrogen/Thermo Fisher Scientific)\", \"puromycin (InvivoGen)\", \"Cultrex RGF basement membrane extract (BME) Type 2 (R&D Systems)\", \"cell counter TC20 (BioRad)\", \"collagen I/5x neutralization buffer\", \"buprenorphine (Temgesic, Schering-Plough)\", \"Isoflurane Piramal (Piramal)\", \"SomnoSuite® Low-Flow Anesthesia System (Kent Scientific Corporation)\", \"Seprafilm (Baxter)\", \"6-0 Vicryl (Ethicon/Johnson&Johnson) suture\", \"9.4 tesla small bore animal scanner (BioSpec 94/21, Bruker Biospin)\", \"dedicated mouse quadrature-resonator (Bruker)\", \"VEVO 3100 (Fujifilm/VisualSonics) ultrasound\", \"small animal transducer\", \"RNAprotect buffer (Qiagen, Netherlands)\", \"Liver Dissociation Kit, mouse (Miltenyi Biotec)\", \"Tumor Dissociation Kit, mouse (Miltenyi Biotec)\", \"ACK lysis buffer (Gibco/Thermo Fisher Scientific)\", \"TruStain FcX PLUS (anti-mouse CD16/32) antibody (BioLegend)\", \"FACS buffer (PBS + 0,5% BSA)\", \"1x Fixation/Permeabilization buffer\", \"eBioscience™ Foxp3/Transcription Factor Staining Buffer Set (Invitrogen/Thermo Fisher Scientific)\", \"FACS antibodies (BioLegend)\", \"Beckman Coulter Gallios (Beckman Coulter)\", \"BD LSRFortessa (Becton Dickinson)\", \"Kaluza Software (Beckman Coulter)\", \"RPMI-Medium (+ 1% Pen/Strep + 1% HEPES + 5 µl beta-mercaptoethanol)\", \"Cell Activation Cocktail (BioLegend)\", \"Brefeldin A (BioLegend)\", \"RLT buffer (Qiagen, Venlo, Netherlands)\", \"RNeasy Mini Kit (Qiagen)\", \"NanoDrop™ spectrophotometer (Thermo Fisher Scientific)\", \"ProtoScript II First Strand cDNA Synthesis Kit (New England Biolabs)\", \"QuantiTect SYBR Green RT PCR mastermix (Qiagen)\", \"DNeasy Blood & Tissue Kit (Qiagen)\", \"primer sequences (purchased from Eurofins Genomics)\", \"Roche LightCycler 480 (Roche)\", \"QuantiFast SYBR® Green PCR Kit (Qiagen)\", \"Bifidobacterium actinocoloniiforme (DSMZ-Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, DSM22766)\"],\n",
    "    \"11196847\": [\"Potato dextrose agar (PDA) medium\", \"Potato dextrose broth (PDB)\", \"ABI 3130 Genetic Analyzer\", \"Taq DNA polymerase Assay Buffer\", \"ABI 3130xl gel sequencing\", \"Phylogenetic Tree Builder\", \"Ethanol (80%, v/v)\", \"Whatman paper No. 4\", \"Lowry's assay\", \"Folin-Ciocalteu reagent\", \"Bovine Serum Albumin (BSA)\", \"Anthrone reagent\", \"Anthrone technique\", \"Quercetin\", \"Folin-Ciocalteu method\", \"Gallic acid equivalent (GAE)\", \"Aescin\", \"Ferric chloride\", \"Mg(CH3COO)2 in CH3OH\", \"Rhein\", \"NH4OH\", \"Liebermann-Burchard reagent\", \"DPPH radicals\", \"Liebermann-Burchard reagent\", \"β-glucan assay kit (Megazyme Int., Dublin, Ireland)\", \"High-performance liquid chromatography (PDA)\", \"RP C-18 column\", \"Quercetin\", \"Sesquiterpene glycosides\", \"Ergosterol\", \"RidentinB\", \"Statistical Package for the Social Sciences (SPSS)\"],\n",
    "    \"11196703\": [\"ENSO 16 sugar substitute\", \"glucose in water solution\", \"NEOH by Alpha Republic GmbH\", \"STAMAG Stadlauer Malzfabrik GesmbH\", \"Special Ingredients Ltd glucose powder\", \"GraphPad Prism 9\"],\n",
    "    \"11196440\": [],\n",
    "    \"11196417\": [\"GEO database\", \"limma package\", \"R software\", \"ggplot2 package\", \"GSEA official website\", \"MSigDB database\", \"clusterProfiler package\", \"enrichplot package\", \"ImmuCellAI\", \"ssGSEA\", \"STRING 4 online platform\", \"Cytoscape V3.9.0 software\", \"CytoHubba plugin\", \"Random Forest algorithm\", \"SVM-RFE\", \"LASSO model\", \"Logistic regression\", \"Receiver Operating Characteristic (ROC) curve\", \"Area Under the Curve (AUC)\", \"miRNet database\", \"Drug-Gene Interaction database (DGIdb)\", \"ggplot2 package\", \"TransZol Up Plus RNA Kit\", \"Nanodrop Spectrophotometer\", \"TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix\", \"ChamQ Universal SYBR qPCR Master Mix\", \"QuantStudio™ 5 Real-Time PCR System\", \"Image J software\"],\n",
    "    \"11196393\": [\"Ficoll gradient\", \"ACK lysing buffer\", \"RPMI1640 supplemented with 20% FBS\", \"BD FACSAria III cell sorter\", \"Fc-block (Miltenyi)\", \"surface antibodies against CD11b, HLA-DR, CD14, CD177, and CXCR2\", \"anti-Arg1 (A1exF5; Thermo Fisher)\", \"DCFDA (Abcam)\", \"BD FACSCanto II\", \"APC antihuman CD11b(ICRF44)\", \"Brilliant Violet 605™ antihuman HLA-DR(L243)\", \"APC/Cyanine7 antihuman CD14(63D3)\", \"PerCP/Cyanine5.5 antihuman CXCR2(5E8/CXCR2)\", \"FITC antihuman CD177(MEM-166)\", \"Cell Ranger (version 7.0)\", \"Seurat (version 4.0.1)\", \"monocle2 (version 2.26.0)\", \"GraphPad Prism 8.0\"],\n",
    "    \"11196377\": [\"Cobb-500 chicks\", \"Evonik Co. near-infrared analysis\", \"UFFDA software\", \"Cobb-Vantress commercial management guide\", \"Vitamin premix\", \"Mineral premix\", \"SAS 9.4 software\"],\n",
    "    \"11196265\": [\"Polyethylene microplastics (PE-MPs) powder\", \"Purified water (Milli-Q)\", \"Tadalafil (Cilais) substance\", \"Sildenafil\", \"Commercial fish pellet feed (containing 30% protein and 12% lipid; El Nasr Company, Egypt)\", \"Automated technical analyzer (BC-2800 from Mindray)\", \"Spectrophotometer (in a wavelength range of 340-546 nm; Biodiagonstic Company, Egypt)\", \"Kits for TAC (total antioxidant capacity) determination (Sigma-Aldrich, USA)\", \"Olympus CH30 microscope\", \"IL-6 antibody (E-AB-40021, Elabscience Biotechnology Inc, USA)\", \"Streptavidin-biotin-peroxidase pack\", \"Mayer's hematoxylin\", \"SPSS 16.0 program for Windows (SPSS 2007, Inc., IL, USA)\", \"Canon digital camera (PowershotA95)\"],\n",
    "    \"11196252\": [\"AUTOLAB-PGSTAT302N instrument\", \"NOVA 1.11.1 software\", \"platinum (Pt) wires\", \"silver/silver chloride wires (Ag/AgCl)\", \"EmStat3 potentiostat\", \"PSTrace software\", \"automated electrode stand (M164, MTM Anko Instruments)\", \"glassy carbon electrode (GCE)\", \"silver/silver chloride electrode (Ag/AgCl/3.0 mol L−1 KCl)\", \"platinum wire (Pt, 99.99%, The Mint of Poland)\", \"DANO (Merck)\", \"hydrochloric acid (HCl)\", \"Potassium tetrakis(4-chlorophenyl)borate (KTPBCl, Merck)\", \"bis(triphenylphosphoranylidene) ammonium chloride (BTPPACl, Merck)\", \"bis(triphenylphosphoranylidene)ammonium tetrakis(4-chlorophenyl)borate (BTPPATPBCl)\", \"Tetrapropylammonium chloride (TPrACl, Alfa Aesar)\", \"Britton-Robinson buffers (BRBs)\", \"1,2-dichloroethane (1,2-DCE, POCH)\", \"citric acid (Sigma-Aldrich)\", \"galactose (Chempur)\", \"lactose (Fisher Chemical)\", \"glucose (Alfa Aesar)\", \"calcium chloride (Sigma-Aldrich)\", \"potassium chloride (Chempur)\", \"magnesium chloride (Fisher Chemical)\", \"iron(III) chloride (Alfa Aesar)\", \"sodium lactate (Sigma-Aldrich)\", \"orthophosphoric acid (V) (Alfa Aesar)\", \"combined pH electrode (Polilyte Lab, Hamilton, Switzerland)\"]\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198832: ['Shimadzu high-performance liquid chromatography (HPLC)', 'AB SCIEX mass spectrometer (MS)', 'ABI PCR amplification instrument', 'BioTek enzyme labeling instrument', 'Beijing Liuyi electrophoresis apparatus', 'Beijing Bijing Biotechnology Co., LTD. gel imaging system', 'Thermo Fisher Scientific Nanodrop UV quantitative system', 'Illumina Sequencer', 'Bio-rad Corporation fluorescence quantitative PCR instrument', 'vancomycin hydrochloride for injection (VIANEX S.A.)', 'bifidobacterium quadruplex viable tablets (Hangzhou Yuanda Biological Pharmaceutical)', 'NBP capsules (Shiyao Group Enbipu Pharmaceutical)', 'NBP reference (Shiyao Group Enbipu Pharmaceutical)', 'glipizide reference (Sichuan Vicchi Biochemical Technology)', 'Omega Bio-Tek Soil DNA Kit', 'ABI Quant-iT PicoGreen dsDNA Assay Kit', 'Servicebio First Strand cDNA Synthesis Kit', 'Servicebio SYBR Green qPCR Master Mix']\n",
      "11197919: ['HI-MAIZE® 260 starch', 'VERSAFIBE™ 1490 starch', 'AMIOCA™ TF starch', 'Tanita SC-240 Total Body Composition Analyzer', 'Automated Self-Administered 24-hour (ASA24) Dietary Assessment Tool 2020', 'QIAamp 96 DNA Blood Kit', 'QIAamp Blood Mini Kit', 'QIAamp Investigator Kit', 'iTaq™ Universal SYBR® Green Supermix', 'Salimetrics Salivary Alpha-Amylase Enzymatic Kit', 'DNeasy PowerSoil 96 HTP Kit', 'Classic++™ Hot Start Taq DNA Polymerase Master Mix', 'Mag-Bind TotalPure', 'QIIME 2', 'DADA2', 'mafft', 'fasttree', 'Greengenes 13_8 99% OTUs reference sequences', 'RStudio version 4.2.1', 'ComplexHeatmap', 'circlize', 'MaAsLin2 (Microbiome Multivariable Association with Linear Models 2)', 'glmnet']\n",
      "11197476: ['CXB@Lipo (C@L)', 'chloroform', 'HSPC', 'LiposoFast (Avestin, Canada)', 'polycarbonate membrane filters (Merck Millipore Ltd., USA)', 'ultrafiltration centrifuge tube', 'ECN', 'glycol chitosan', 'transmission electron microscopy (TEM; JEM-1400, Japan)', 'Malvern Nano ZS ZEN3600 (Malvern, UK)', 'UV-VIS (Thermo Fisher Scientific, USA)', 'ICG@ECN (I@ECN)', 'flow cytometry (CytoFLEX, Beckman, USA)', 'UV‒VIS', 'pGEX-Pvhb-Lysis', 'cloning kit (Beyotime Biotechnology Co. Ltd., Shanghai, China)', 'ICG@Liposome (I@L)', 'confocal laser scanning microscopy (CLSM) (Nikon, Japan)', 'CCK-8 assay kit (Beyotime Biotechnology Co. Ltd., Shanghai, China)', 'Western blot (WB) analysis', 'FITC-conjugated anti-COX-2 antibody (Affinity, Jiangsu, China)', 'in vivo images system (IVIS)', 'optimal cutting temperature (OCT) compound', 'CT26 tumor model', 'FITC-conjugated anti-mouse CD3', 'FITC-conjugated anti-mouse CD4', 'PE-conjugated anti-mouse CD8 (Biolegend Inc., USA)', 'TUNEL and H&E staining', 'Ki67', 'C@ECN-PL', 'aspartate aminotransferase (AST)', 'alanine aminotransferase (ALT)', 'blood urea nitrogen (BUN)', 'creatinine (CRE)', 'assay kits (Jiancheng Bioengineering Institute, Nanjing, China)', 'H&E staining analysis', 'ImageJ 6.0']\n",
      "11197378: ['EXCEL', 'IBM SPSS 20', 'R language']\n",
      "11197204: ['Gene Expression Omnibus (GEO)', 'quantitative PCR method', 'UK National Research Ethics Committee (NRES 16/SS/0172)', 'BH false discovery rate correction', 'TRIzol (Life Technologies)', 'TaqMan™ Gene Expression Assay (FAM) from Life Technologies Ltd', 'Easy Oligos from Merck Life Science UK Ltd', 'iTaq Universal One-Step or iTaq™ Universal SYBR® Green One-Step Kit from BioRad', 'GraphPad PRISM', 'ShinyGO', 'Enrichr', 'Tools']\n",
      "11197185: ['ZIKV', 'Vero cell culture (ATCC)', 'C636 cell culture (ATCC)', 'Tetra Pond Koi growth feed', 'defibrinated sheep blood', 'sodium bicarbonate', 'Hemotek membrane feeding system (Discovery Workshops, Accrington, UK)', 'triethylamine (Sigma Aldrich, St. Louis, MO, USA)', 'filter sterilized fetal bovine serum', 'Mosquito Diluent [MD]', 'penicillin/streptomycin', 'gentamicin', 'Fungizone (Sigma Aldrich, St. Louis, MO, USA)', 'phosphate-buffered Saline (PBS) (Sigma Aldrich, St. Louis, MO, USA)', 'Illumina MiSeq', 'REPLI-g whole transcriptome amplification (WTA) single-cell kit (Qiagen, Hilden, Germany)', 'Illumina barcoded 16S primer set', 'AccuStart II PCR supermix (Quantabio, Beverly, MA, USA)', 'QIITA', 'MicrobiomeAnalyst', 'GreenGenes 16S reference database', 'BIOM-formatted OTU table', 'NCBI GenBank Short Read Archives']\n",
      "11196966: ['Advanced DMEM-F12 (Gibco/Thermo Fisher Scientific)', 'penicillin (Gibco/Thermo Fisher Scientific)', 'streptomycin (Gibco/Thermo Fisher Scientific)', 'HEPES (Invitrogen/Thermo Fisher Scientific)', 'Glutamax (Invitrogen/Thermo Fisher Scientific)', 'N2 (Gibco/Thermo Fisher Scientific)', 'B27 (Gibco/Thermo Fisher Scientific)', 'N-acetylcysteine (Sigma-Aldrich)', 'hygromycin (Invitrogen/Thermo Fisher Scientific)', 'puromycin (InvivoGen)', 'Cultrex RGF basement membrane extract (BME) Type 2 (R&D Systems)', 'cell counter TC20 (BioRad)', 'collagen I/5x neutralization buffer', 'buprenorphine (Temgesic, Schering-Plough)', 'Isoflurane Piramal (Piramal)', 'SomnoSuite® Low-Flow Anesthesia System (Kent Scientific Corporation)', 'Seprafilm (Baxter)', '6-0 Vicryl (Ethicon/Johnson&Johnson) suture', '9.4 tesla small bore animal scanner (BioSpec 94/21, Bruker Biospin)', 'dedicated mouse quadrature-resonator (Bruker)', 'VEVO 3100 (Fujifilm/VisualSonics) ultrasound', 'small animal transducer', 'RNAprotect buffer (Qiagen, Netherlands)', 'Liver Dissociation Kit, mouse (Miltenyi Biotec)', 'Tumor Dissociation Kit, mouse (Miltenyi Biotec)', 'ACK lysis buffer (Gibco/Thermo Fisher Scientific)', 'TruStain FcX PLUS (anti-mouse CD16/32) antibody (BioLegend)', 'FACS buffer (PBS + 0,5% BSA)', '1x Fixation/Permeabilization buffer', 'eBioscience™ Foxp3/Transcription Factor Staining Buffer Set (Invitrogen/Thermo Fisher Scientific)', 'FACS antibodies (BioLegend)', 'Beckman Coulter Gallios (Beckman Coulter)', 'BD LSRFortessa (Becton Dickinson)', 'Kaluza Software (Beckman Coulter)', 'RPMI-Medium (+ 1% Pen/Strep + 1% HEPES + 5 µl beta-mercaptoethanol)', 'Cell Activation Cocktail (BioLegend)', 'Brefeldin A (BioLegend)', 'RLT buffer (Qiagen, Venlo, Netherlands)', 'RNeasy Mini Kit (Qiagen)', 'NanoDrop™ spectrophotometer (Thermo Fisher Scientific)', 'ProtoScript II First Strand cDNA Synthesis Kit (New England Biolabs)', 'QuantiTect SYBR Green RT PCR mastermix (Qiagen)', 'DNeasy Blood & Tissue Kit (Qiagen)', 'primer sequences (purchased from Eurofins Genomics)', 'Roche LightCycler 480 (Roche)', 'QuantiFast SYBR® Green PCR Kit (Qiagen)', 'Bifidobacterium actinocoloniiforme (DSMZ-Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, DSM22766)']\n",
      "11196847: ['Potato dextrose agar (PDA) medium', 'Potato dextrose broth (PDB)', 'ABI 3130 Genetic Analyzer', 'Taq DNA polymerase Assay Buffer', 'ABI 3130xl gel sequencing', 'Phylogenetic Tree Builder', 'Ethanol (80%, v/v)', 'Whatman paper No. 4', \"Lowry's assay\", 'Folin-Ciocalteu reagent', 'Bovine Serum Albumin (BSA)', 'Anthrone reagent', 'Anthrone technique', 'Quercetin', 'Folin-Ciocalteu method', 'Gallic acid equivalent (GAE)', 'Aescin', 'Ferric chloride', 'Mg(CH3COO)2 in CH3OH', 'Rhein', 'NH4OH', 'Liebermann-Burchard reagent', 'DPPH radicals', 'Liebermann-Burchard reagent', 'β-glucan assay kit (Megazyme Int., Dublin, Ireland)', 'High-performance liquid chromatography (PDA)', 'RP C-18 column', 'Quercetin', 'Sesquiterpene glycosides', 'Ergosterol', 'RidentinB', 'Statistical Package for the Social Sciences (SPSS)']\n",
      "11196703: ['ENSO 16 sugar substitute', 'glucose in water solution', 'NEOH by Alpha Republic GmbH', 'STAMAG Stadlauer Malzfabrik GesmbH', 'Special Ingredients Ltd glucose powder', 'GraphPad Prism 9']\n",
      "11196417: ['GEO database', 'limma package', 'R software', 'ggplot2 package', 'GSEA official website', 'MSigDB database', 'clusterProfiler package', 'enrichplot package', 'ImmuCellAI', 'ssGSEA', 'STRING 4 online platform', 'Cytoscape V3.9.0 software', 'CytoHubba plugin', 'Random Forest algorithm', 'SVM-RFE', 'LASSO model', 'Logistic regression', 'Receiver Operating Characteristic (ROC) curve', 'Area Under the Curve (AUC)', 'miRNet database', 'Drug-Gene Interaction database (DGIdb)', 'ggplot2 package', 'TransZol Up Plus RNA Kit', 'Nanodrop Spectrophotometer', 'TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix', 'ChamQ Universal SYBR qPCR Master Mix', 'QuantStudio™ 5 Real-Time PCR System', 'Image J software']\n",
      "11196393: ['Ficoll gradient', 'ACK lysing buffer', 'RPMI1640 supplemented with 20% FBS', 'BD FACSAria III cell sorter', 'Fc-block (Miltenyi)', 'surface antibodies against CD11b, HLA-DR, CD14, CD177, and CXCR2', 'anti-Arg1 (A1exF5; Thermo Fisher)', 'DCFDA (Abcam)', 'BD FACSCanto II', 'APC antihuman CD11b(ICRF44)', 'Brilliant Violet 605™ antihuman HLA-DR(L243)', 'APC/Cyanine7 antihuman CD14(63D3)', 'PerCP/Cyanine5.5 antihuman CXCR2(5E8/CXCR2)', 'FITC antihuman CD177(MEM-166)', 'Cell Ranger (version 7.0)', 'Seurat (version 4.0.1)', 'monocle2 (version 2.26.0)', 'GraphPad Prism 8.0']\n",
      "11196377: ['Cobb-500 chicks', 'Evonik Co. near-infrared analysis', 'UFFDA software', 'Cobb-Vantress commercial management guide', 'Vitamin premix', 'Mineral premix', 'SAS 9.4 software']\n",
      "11196265: ['Polyethylene microplastics (PE-MPs) powder', 'Purified water (Milli-Q)', 'Tadalafil (Cilais) substance', 'Sildenafil', 'Commercial fish pellet feed (containing 30% protein and 12% lipid; El Nasr Company, Egypt)', 'Automated technical analyzer (BC-2800 from Mindray)', 'Spectrophotometer (in a wavelength range of 340-546 nm; Biodiagonstic Company, Egypt)', 'Kits for TAC (total antioxidant capacity) determination (Sigma-Aldrich, USA)', 'Olympus CH30 microscope', 'IL-6 antibody (E-AB-40021, Elabscience Biotechnology Inc, USA)', 'Streptavidin-biotin-peroxidase pack', \"Mayer's hematoxylin\", 'SPSS 16.0 program for Windows (SPSS 2007, Inc., IL, USA)', 'Canon digital camera (PowershotA95)']\n",
      "11196252: ['AUTOLAB-PGSTAT302N instrument', 'NOVA 1.11.1 software', 'platinum (Pt) wires', 'silver/silver chloride wires (Ag/AgCl)', 'EmStat3 potentiostat', 'PSTrace software', 'automated electrode stand (M164, MTM Anko Instruments)', 'glassy carbon electrode (GCE)', 'silver/silver chloride electrode (Ag/AgCl/3.0 mol L−1 KCl)', 'platinum wire (Pt, 99.99%, The Mint of Poland)', 'DANO (Merck)', 'hydrochloric acid (HCl)', 'Potassium tetrakis(4-chlorophenyl)borate (KTPBCl, Merck)', 'bis(triphenylphosphoranylidene) ammonium chloride (BTPPACl, Merck)', 'bis(triphenylphosphoranylidene)ammonium tetrakis(4-chlorophenyl)borate (BTPPATPBCl)', 'Tetrapropylammonium chloride (TPrACl, Alfa Aesar)', 'Britton-Robinson buffers (BRBs)', '1,2-dichloroethane (1,2-DCE, POCH)', 'citric acid (Sigma-Aldrich)', 'galactose (Chempur)', 'lactose (Fisher Chemical)', 'glucose (Alfa Aesar)', 'calcium chloride (Sigma-Aldrich)', 'potassium chloride (Chempur)', 'magnesium chloride (Fisher Chemical)', 'iron(III) chloride (Alfa Aesar)', 'sodium lactate (Sigma-Aldrich)', 'orthophosphoric acid (V) (Alfa Aesar)', 'combined pH electrode (Polilyte Lab, Hamilton, Switzerland)']\n"
     ]
    }
   ],
   "source": [
    "for key, value in product_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11197919: ['HI-MAIZE® 260 starch', 'VERSAFIBE™ 1490 starch', 'AMIOCA™ TF starch', 'Tanita SC-240 Total Body Composition Analyzer', 'Automated Self-Administered 24-hour (ASA24) Dietary Assessment Tool 2020', 'QIAamp 96 DNA Blood Kit', 'QIAamp Blood Mini Kit', 'QIAamp Investigator Kit', 'iTaq™ Universal SYBR® Green Supermix', 'Salimetrics Salivary Alpha-Amylase Enzymatic Kit', 'DNeasy PowerSoil 96 HTP Kit', 'Classic++™ Hot Start Taq DNA Polymerase Master Mix', 'Mag-Bind TotalPure', 'QIIME 2', 'DADA2', 'mafft', 'fasttree']\n",
      "11197476: ['CXB@Lipo', 'chloroform', 'rotary evaporator', 'liposome extrude (LiposoFast, Avestin, Canada)', 'polycarbonate membrane filters (Merck Millipore Ltd., USA)', 'ultrafiltration centrifuge tube', 'ECN', 'glycol chitosan', 'transmission electron microscopy (TEM; JEM-1400, Japan)', 'Malvern Nano ZS ZEN3600 (Malvern, UK)', 'UV-VIS (Thermo Fisher Scientific, USA)', 'ICG@ECN (I@ECN)', 'flow cytometry (CytoFLEX, Beckman, USA)', 'UV‒VIS', 'pGEX-Pvhb-Lysis', 'cloning kit (Beyotime Biotechnology Co. Ltd., Shanghai, China)', 'ICG@Liposome (I@L)', 'confocal laser scanning microscopy (CLSM) (Nikon, Japan)', 'CCK-8 assay kit (Beyotime Biotechnology Co. Ltd., Shanghai, China)', 'Western blot (WB) analysis', 'FITC-conjugated anti-COX-2 antibody (Affinity, Jiangsu, China)', 'in vivo images system (IVIS)', 'optimal cutting temperature (OCT) compound', 'CT26 tumor model', 'FITC-conjugated anti-mouse CD3', 'FITC-conjugated anti-mouse CD4', 'PE-conjugated anti-mouse CD8 (Biolegend Inc., USA)', 'TUNEL and H&E staining', 'Ki67', 'C@ECN-PL', 'assay kits (Jiancheng Bioengineering Institute, Nanjing, China)', 'H&E staining analysis', 'ImageJ 6.0']\n",
      "11197204: ['TRIzol', 'TaqMan™ Gene Expression Assay', 'Easy Oligos', 'iTaq Universal One-Step', 'iTaq™ Universal SYBR® Green One-Step Kit', 'Tools', 'GraphPad PRISM']\n",
      "11197185: ['Tetra Pond Koi growth feed', 'Hemotek membrane feeding system', 'Trizol', 'REPLI-g whole transcriptome amplification (WTA) single-cell kit', 'Illumina barcoded 16S primer set', 'AccuStart II PCR supermix', 'QIITA', 'MicrobiomeAnalyst', 'GreenGenes 16S reference database', 'BIOM-formatted OTU table', 'NCBI GenBank Short Read Archives', 'Chao1 diversity measure', 'Shannon diversity index', 'Simpson Index', 'Bray–Curtis dissimilarity statistic', 'PERMANOVA test', 'Random forest algorithm', 'Microbiome Analyst']\n",
      "11196966: ['Advanced DMEM-F12', 'penicillin/streptomycin', 'HEPES', 'Glutamax', 'N2', 'B27', 'N-acetylcysteine', 'hygromycin', 'puromycin', 'Cultrex RGF basement membrane extract (BME) Type 2', 'collagen I/5x neutralization buffer', 'buprenorphine', 'Isoflurane', 'Seprafilm', 'Vicryl suture', 'RNAprotect buffer', 'Liver Dissociation Kit, mouse', 'Tumor Dissociation Kit, mouse', 'ACK lysis buffer', 'FACS buffer', 'TruStain FcX PLUS (anti-mouse CD16/32) antibody', 'eBioscience™ Foxp3/Transcription Factor Staining Buffer Set', 'RPMI-Medium', 'Cell Activation Cocktail', 'Brefeldin A', 'RLT buffer', 'RNeasy Mini Kit', 'ProtoScript II First Strand cDNA Synthesis Kit', 'QuantiTect SYBR Green RT PCR mastermix', 'DNeasy Blood & Tissue Kit', 'QuantiFast SYBR® Green PCR Kit', 'Roche LightCycler 480', 'QuantiFast SYBR® Green PCR Kit', 'Bacterial 16S qPCR', 'QuantiFast SYBR® Green PCR Kit', 'Illumina NovaSeq instrument', 'Pierce BCA-Kit', 'Azure Ao Microplate Reader', '4x Laemmli Buffer', 'Mini-Protean TGX precast gel', 'color-coded prestained broad range protein ladder', 'Trans-Blot Turbo Transfer System', 'Immobilon Crescendo Western HRP substrate', 'ChemiDoc BioRad imager', 'PFA', 'TE-Buffer', 'avidin-biotin-block', 'DAB', 'ROTI® Histokitt', 'Zeiss Axio Scan Z.1 Microscope Slide Scanner', 'Vector® TrueVIEW® Autofluorescence Quenching Kit', 'NucBlue™ Fixed Cell Stain ReadyProbes™', 'VECTASHIELD Vibrance Antifade Mounting Medium']\n",
      "11196847: ['potato dextrose agar (PDA) medium', 'potato dextrose broth (PDB)', 'pestle and mortar', 'ABI 3130 Genetic Analyzer', 'Taq DNA polymerase Assay Buffer', 'dNTP', 'MgCl2', 'Taq DNA Polymerase', 'Whatman paper No. 4', 'vacuum evaporator', 'Folin-Ciocalteu reagent', 'Bovine Serum Albumin (BSA)', 'anthrone reagent', 'NaNO2', 'AlCl3', 'NaOH', 'Folin-Ciocalteu phenol reagent', 'Na2CO3', 'vanillin', 'sulfuric acid', 'Aescin', 'ferric chloride', 'Mg(CH3COO)2', 'Rhein', 'NH4OH', 'petroleum ether', 'Folin-Denis reagent', 'Liebermann-Burchard reagent', 'cholesterol', 'DPPH', '4-aminophenazone', 'Tris', 'phenol', 'glucose oxidase and peroxidase', 'amyloglucosidase', 'invertase', 'high-performance liquid chromatography (PDA)']\n",
      "11196703: ['ENSO 16', 'glucose powder', 'Vacuette sodium fluoride K3EDTA tubes', 'Vacuette serum tubes with clot activator']\n",
      "11196393: ['Ficoll gradient', 'ACK lysing buffer', 'RPMI1640', 'FBS', 'BD FACSAria III cell sorter', 'Fc-block', 'CD11b antibodies', 'HLA-DR antibodies', 'CD14 antibodies', 'CD177 antibodies', 'CXCR2 antibodies', 'Arg1 antibodies', 'DCFDA', 'APC antihuman CD11b(ICRF44)', 'Brilliant Violet 605™ antihuman HLA-DR(L243)', 'APC/Cyanine7 antihuman CD14(63D3)', 'PerCP/Cyanine5.5 antihuman CXCR2(5E8/CXCR2)', 'FITC antihuman CD177(MEM-166)', 'Cell Ranger', 'Seurat', 'GraphPad Prism 8.0']\n",
      "11196265: ['Polyethylene microplastics (PE-MPs)', 'Purified water (Milli-Q)', 'tadalafil (Cilais)', 'commercial fish pellet feed (containing 30% protein and 12% lipid; El Nasr Company, Egypt)', 'crushed ice', 'automated technical analyzer (BC-2800 from Mindray)', 'spectrophotometer (Biodiagonstic Company, Egypt)', 'kits (Sigma-Aldrich, USA)', 'neutral buffered formalin', 'hematoxylin and eosin (H & E)', 'Olympus CH30 microscope', 'xylene', 'standard goat serum (Vector Research facilities, Burlingame, CA)', 'essential polyclonal IL-6 antibody; 1/50 (E-AB-40021, Elabscience Biotechnology Inc, USA)', 'streptavidin–biotin-peroxidase pack', \"Mayer's hematoxylin\", 'Olympus BX41 microscope', 'canon digital camera (PowershotA95)', 'SPSS 16.0 program for Windows (SPSS 2007, Inc., IL, USA)']\n",
      "11196252: ['AUTOLAB-PGSTAT302N instrument', 'NOVA 1.11.1 software', 'platinum (Pt) wires', 'silver/silver chloride wires (Ag/AgCl)', 'EmStat3 potentiostat', 'PSTrace software', 'automated electrode stand (M164, MTM Anko Instruments)', 'glassy carbon electrode (GCE)', 'silver/silver chloride electrode (Ag/AgCl/3.0 mol L−1 KCl)', 'platinum wire (Pt, 99.99%, The Mint of Poland)', 'DANO', 'hydrochloric acid (HCl)', 'Potassium tetrakis(4-chlorophenyl)borate (KTPBCl)', 'bis(triphenylphosphoranylidene) ammonium chloride (BTPPACl)', 'bis(triphenylphosphoranylidene)ammonium tetrakis(4-chlorophenyl)borate (BTPPATPBCl)', 'tetrapropylammonium chloride (TPrACl)', 'Britton–Robinson buffers (BRBs)', '1,2–dichloroethane (1,2–DCE)', 'citric acid', 'galactose', 'lactose', 'glucose', 'calcium chloride', 'potassium chloride', 'magnesium chloride', 'iron(III) chloride', 'sodium lactate', 'orthophosphoric acid (V)', 'combined pH electrode (Polilyte Lab, Hamilton, Switzerland)', 'ultra high temperature (UHT) milk (1.5% fat content)']\n"
     ]
    }
   ],
   "source": [
    "for key, value in product_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Company Identification (New)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## OpenAI"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "system_prompt = \"\"\"You are an expert Company Identification system. \n",
    "Your task is to take a list of strings as input and identify the company that manufactures each mentioned product.\n",
    "If the strings do not contain company names, use your internal knowledge to determine the companies.\n",
    "For any product where the manufacturer is unclear, return a null value for that string.\n",
    "Here is an example of the output format for a list of strings.\n",
    "\n",
    "List of strings: [\"Sony PlayStation 5\", \"iPhone 15 Plus\", \"Staples Pencil\", \"Stainless Steel Tumbler\", \"Car\", \"Whopper\", \"Amazon Speed ETD\"]\n",
    "Answer: [\"Sony\", \"Apple\", \"Staples\", None, None, \"Burger King\", \"Bruker\"]\n",
    "\n",
    "Only use this output format. Do not output anything besides this output format.\n",
    "Output the manufacturer names in the order they occur in the input list of strings.\n",
    "\"\"\"\n",
    "\n",
    "user_prompt = \"\"\"Q: Given the list of strings below, identify the companies that manufactures each product.\n",
    "\n",
    "List of strings: {}\n",
    "Answer:\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "product_dict = {\n",
    "    \"11198832\": ['Shimadzu high-performance liquid chromatography (HPLC)', 'SCIEX mass spectrometer (MS)', 'ABI PCR amplification instrument', 'BioTek enzyme labeling instrument', 'Beijing Liuyi Instrument Factory electrophoresis apparatus', 'Beijing Bijing Biotechnology Co. gel imaging system', 'Thermo Fisher Scientific Nanodrop UV quantitative system', 'Illumina Sequencer', 'Bio-rad Corporation fluorescence quantitative PCR instrument', 'VIANEX S.A. vancomycin hydrochloride for injection', 'Hangzhou Yuanda Biological Pharmaceutical bifidobacterium quadruplex viable tablets', 'Shiyao Group Enbipu Pharmaceutical NBP capsules', 'Shiyao Group Enbipu Pharmaceutical NBP reference', 'Sichuan Vicchi Biochemical Technology glipizide reference', 'Omega Bio-Tek Soil DNA Kit', 'ABI Quant-iT PicoGreen dsDNA Assay Kit', 'Servicebio First Strand cDNA Synthesis Kit', 'Servicebio SYBR Green qPCR Master Mix'],\n",
    "    \"11198784\": ['Ficoll-Paque Premium density gradient medium', 'Invitrogen trypan blue dye', 'Bio-Rad TC20 Automated cell counter', 'EasySep Negative Human NK Cell Isolation Kit', 'Becton Dickinson CD56 APC', 'Becton Dickinson CD3 PE Cy7', 'BD Bioscience stain buffer', 'BD LSR-Fortessa X-20 flow cytometer', 'Harvard Apparatus borosilicate glass capillaries', 'Sutter Instrumental P-97 glass pipette', 'Narishige Micro Forge MF-900', 'Molecular Devices CV203BU head-stage', 'Narishige manipulators', 'Molecular Devices Axopatch 200B amplifier', 'Molecular Devices pClamp 10.7 software', 'Molecular Devices Digidata 1440A', 'Sigma-Aldrich membrane filter', 'Sapphire Bioscience Reagents ATP', 'Sapphire Bioscience Reagents GTP', 'Tocris Bioscience PregS', 'Tocris Bioscience Ononetin', 'Sigma-Aldrich chemicals', 'IBM SPSS software', 'GraphPad Prism software', 'OriginLab Origin 2021 software'],\n",
    "    \"11197919\": ['HI-MAIZE® 260 starch', 'VERSAFIBE™ 1490 starch', 'AMIOCA™ TF starch', 'Salimetrics Salivary Alpha-Amylase Enzymatic Kit'],\n",
    "    \"11197476\": ['LiposoFast liposome extruder', 'Merck Millipore polycarbonate membrane filters', 'ultrafiltration centrifuge tube', 'JEM-1400 transmission electron microscope', 'Malvern Nano ZS ZEN3600', 'UV-VIS Thermo Fisher Scientific', 'CytoFLEX flow cytometer', 'Beyotime Biotechnology cloning kit', 'Nikon confocal laser scanning microscopy', 'CCK-8 assay kit Beyotime Biotechnology', 'IVIS in vivo imaging system', 'Biolegend Inc. antibodies', 'Jiancheng Bioengineering Institute assay kits'],\n",
    "    \"11197378\": ['Qingyue Open Environmental Data Centre data', 'Gaode Map', 'EXCEL database', 'IBM SPSS 20 software', 'R language'],\n",
    "    \"11197282\": ['Metabolon untargeted metabolomics profiling'],\n",
    "    \"11197204\": ['TRIzol', 'TaqMan Gene Expression Assay', 'iTaq Universal One-Step Kit', 'iTaq Universal SYBR Green One-Step Kit', 'BioRad', 'GraphPad PRISM', 'Enrichr', 'ShinyGO'],\n",
    "    \"11197185\": ['Illumina MiSeq platform', 'Vero cell culture (ATCC)', 'C636 cell culture (ATCC)', 'Sterilite container', 'Tetra Pond Koi growth feed', 'Hemotek membrane feeding system', 'Sigma Aldrich triethylamine', 'Trizol (Thermo Fisher Scientific)', 'REPLI-g WTA single-cell kit (Qiagen)', 'AccuStart II PCR supermix (Quantabio)'],\n",
    "    \"11196966\": ['Advanced DMEM-F12 (Gibco/Thermo Fisher Scientific)', 'penicillin/streptomycin (Gibco/Thermo Fisher Scientific)', 'HEPES (Invitrogen/Thermo Fisher Scientific)', 'Glutamax (Invitrogen/Thermo Fisher Scientific)', 'N2 (Gibco/Thermo Fisher Scientific)', 'B27 (Gibco/Thermo Fisher Scientific)', 'N-acetylcysteine (Sigma-Aldrich)', 'hygromycin (Invitrogen/Thermo Fisher Scientific)', 'puromycin (InvivoGen)', 'Cultrex RGF basement membrane extract (R&D Systems)', 'cell counter TC20 (BioRad)', 'buprenorphine (Temgesic, Schering-Plough)', 'Isoflurane (Isoflurane Piramal, Piramal)', 'SomnoSuite® Low-Flow Anesthesia System (Kent Scientific Corporation)', 'Seprafilm (Baxter)', 'Vicryl (Ethicon/Johnson&Johnson)', 'BioSpec 94/21 MRI scanner (Bruker Biospin)', 'VEVO 3100 ultrasound (Fujifilm/VisualSonics)', 'Liver Dissociation Kit, mouse (Miltenyi Biotec)', 'Tumor Dissociation Kit, mouse (Miltenyi Biotec)', 'ACK lysis buffer (Gibco/Thermo Fisher Scientific)', 'TruStain FcX PLUS (BioLegend)', 'eBioscience™ Foxp3/Transcription Factor Staining Buffer Set (Invitrogen/Thermo Fisher Scientific)', 'Beckman Coulter Gallios (Beckman Coulter)', 'BD LSRFortessa (Becton Dickinson)', 'Cell Activation Cocktail (BioLegend)', 'Brefeldin A (BioLegend)', 'RNeasy Mini Kit (Qiagen)', 'ProtoScript II First Strand cDNA Synthesis Kit (New England Biolabs)', 'DNeasy Blood & Tissue Kit (Qiagen)', 'QuantiTect SYBR Green RT PCR mastermix (Qiagen)', 'Roche LightCycler 480 (Roche)', 'Pierce BCA-Kit (Thermo Fisher Scientific)', 'Azure Ao Microplate Reader (Azure Biosystems)', 'Laemmli Buffer (BioRad)', 'Mini-Protean TGX precast gel (BioRad)', 'Immobilon Crescendo Western HRP substrate (Merck Millipore)', 'ChemiDoc BioRad imager (BioRad)', 'Vector® TrueVIEW® Autofluorescence Quenching Kit (Biozol)', 'NucBlue™ Fixed Cell Stain ReadyProbes™ (Thermo Fisher Scientific)', 'VECTASHIELD Vibrance Antifade Mounting Medium (Biozol)', 'Zeiss Axio Scan Z.1 Microscope Slide Scanner (Zeiss)', 'GraphPad Prism'],\n",
    "    \"11196847\": ['ABI 3130 Genetic Analyzer', 'Megazyme mushroom and yeast β-glucan assay kit'],\n",
    "    \"11196703\": ['ENSO 16 sugar substitute', 'glucose powder'],\n",
    "    \"11196417\": ['GEO database', 'limma package', 'ggplot2 package', 'clusterProfiler package', 'enrichplot package', 'ImmuCellAI methodology', 'ssGSEA methodology', 'Mantel algorithm', 'STRING 4 online platform', 'Cytoscape V3.9.0 software', 'CytoHubba plugin', 'Random Forest package', 'SVM-RFE approach', 'LASSO model', 'Logistic regression approach', 'ROC curve', 'AUC', 'Limma package', 'miRNet database', 'DGIdb database', 'TransZol Up Plus RNA Kit', 'Nanodrop Spectrophotometer', 'TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix', 'ChamQ Universal SYBR qPCR Master Mix', 'QuantStudio™ 5 Real-Time PCR System', 'Image J software'],\n",
    "    \"11196393\": ['Ficoll gradient', 'ACK lysing buffer', 'BD FACSAria III cell sorter', 'Fc-block (Miltenyi)', 'A1exF5 antibody (Thermo Fisher)', 'DCFDA (Abcam)', 'APC antihuman CD11b(ICRF44) antibody (BioLegend)', 'Brilliant Violet 605™ antihuman HLA-DR(L243) antibody (BioLegend)', 'APC/Cyanine7 antihuman CD14(63D3) antibody (BioLegend)', 'PerCP/Cyanine5.5 antihuman CXCR2(5E8/CXCR2) antibody (BioLegend)', 'FITC antihuman CD177(MEM-166) antibody (BioLegend)', 'Cell Ranger software', 'Seurat toolkit', 'monocle2 package', 'GraphPad Prism software'],\n",
    "    \"11196377\": ['Cobb-500 broiler chicks', 'Evonik Nutrition & Care GmbH near-infrared analysis', 'Cobb-Vantress finisher diet', 'UFFDA software', 'A&D Weighing digital weighing scale', 'Konica Minolta Chroma-Meter CR-410', 'SAS 9.4 software'],\n",
    "    \"11196265\": ['Polyethylene microplastics (PE-MPs) powder', 'Milli-Q purified water', 'tadalafil (Cilais)', 'commercial fish pellet feed (El Nasr Company)', 'BC-2800 automated technical analyzer (Mindray)', 'spectrophotometer (Biodiagonstic Company)', 'Sigma-Aldrich total antioxidant capacity kits', 'Olympus CH30 microscope', 'Vector Laboratories standard goat serum', 'E-AB-40021 IL-6 antibody (Elabscience Biotechnology Inc)', 'Olympus BX41 microscope', 'Canon digital camera (PowershotA95)', 'SPSS 16.0 program (SPSS 2007, Inc.)'],\n",
    "    \"11196252\": ['Metrohm Autolab AUTOLAB-PGSTAT302N instrument', 'NOVA 1.11.1 software', 'EmStat3 potentiostat', 'PSTrace software', 'PalmSens B.V.', 'automated electrode stand M164', 'MTM Anko Instruments', 'Basi® glassy carbon electrode', 'Merck DANO', 'Potassium tetrakis(4-chlorophenyl)borate KTPBCl', 'bis(triphenylphosphoranylidene) ammonium chloride BTPPACl', 'Tetrapropylammonium chloride TPrACl', 'Alfa Aesar', 'POCH Britton-Robinson buffers', '1,2-dichloroethane 1,2-DCE', 'Sigma-Aldrich', 'Chempur', 'Fisher Chemical', 'Polilyte Lab pH electrode', 'Hamilton']\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "def company_identification(list_of_products):\n",
    "    response = openai.chat.completions.create(\n",
    "        model=\"gpt-4-turbo\",\n",
    "        presence_penalty=0,\n",
    "        # top_p=1e-16,\n",
    "        temperature=1e-16,\n",
    "        messages=[\n",
    "            {\"role\": \"system\", \"content\": system_prompt},\n",
    "            {\"role\": \"user\", \"content\": user_prompt.format(list_of_products)}\n",
    "        ]\n",
    "    )\n",
    "    return response.choices[0].message.content"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "company_dict = {}\n",
    "for key, value in product_dict.items():\n",
    "    syntax_error = False\n",
    "    if value:\n",
    "        identified_companies = company_identification(value)\n",
    "        try:\n",
    "            identified_companies = ast.literal_eval(identified_companies)\n",
    "        except SyntaxError:\n",
    "            syntax_error = True\n",
    "            print(f\"Syntax Error with ID {key}\")\n",
    "        if not syntax_error:\n",
    "            company_dict[key] = identified_companies"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198832: ['Shimadzu', 'SCIEX', 'ABI', 'BioTek', 'Beijing Liuyi Instrument Factory', 'Beijing Bijing Biotechnology Co.', 'Thermo Fisher Scientific', 'Illumina', 'Bio-rad Corporation', 'VIANEX S.A.', 'Hangzhou Yuanda Biological Pharmaceutical', 'Shiyao Group', 'Shiyao Group', 'Sichuan Vicchi Biochemical Technology', 'Omega Bio-Tek', 'ABI', 'Servicebio', 'Servicebio']\n",
      "11198784: ['Cytiva', 'Thermo Fisher Scientific', 'Bio-Rad Laboratories', 'STEMCELL Technologies', 'Becton Dickinson', 'Becton Dickinson', 'BD Biosciences', 'BD Biosciences', 'Harvard Apparatus', 'Sutter Instrument', 'Narishige', 'Molecular Devices', 'Narishige', 'Molecular Devices', 'Molecular Devices', 'Molecular Devices', 'Merck (Sigma-Aldrich)', 'Sapphire Bioscience', 'Sapphire Bioscience', 'Tocris Bioscience', 'Tocris Bioscience', 'Merck (Sigma-Aldrich)', 'IBM', 'GraphPad Software', 'OriginLab']\n",
      "11197919: ['Ingredion', 'Ingredion', 'Ingredion', 'Salimetrics']\n",
      "11197476: ['Harvard Apparatus', 'Merck', None, 'JEOL', 'Malvern Instruments', 'Thermo Fisher Scientific', 'Beckman Coulter', 'Beyotime Biotechnology', 'Nikon', 'Beyotime Biotechnology', 'PerkinElmer', 'BioLegend', 'Nanjing Jiancheng Bioengineering Institute']\n",
      "11197378: ['None', 'Alibaba', 'Microsoft', 'IBM', 'None']\n",
      "11197282: ['Metabolon']\n",
      "11197204: ['Thermo Fisher Scientific', 'Thermo Fisher Scientific', 'Bio-Rad Laboratories', 'Bio-Rad Laboratories', 'Bio-Rad Laboratories', None, None, None]\n",
      "11197185: ['Illumina', None, None, 'Sterilite', 'Tetra', None, 'Sigma Aldrich', 'Thermo Fisher Scientific', 'Qiagen', 'Quantabio']\n",
      "11196966: ['Thermo Fisher Scientific', 'Thermo Fisher Scientific', 'Thermo Fisher Scientific', 'Thermo Fisher Scientific', 'Thermo Fisher Scientific', 'Thermo Fisher Scientific', 'Sigma-Aldrich', 'Thermo Fisher Scientific', 'InvivoGen', 'R&D Systems', 'BioRad', 'Schering-Plough', 'Piramal', 'Kent Scientific Corporation', 'Baxter', 'Johnson & Johnson', 'Bruker', 'Fujifilm', 'Miltenyi Biotec', 'Miltenyi Biotec', 'Thermo Fisher Scientific', 'BioLegend', 'Thermo Fisher Scientific', 'Beckman Coulter', 'Becton Dickinson', 'BioLegend', 'BioLegend', 'Qiagen', 'New England Biolabs', 'Qiagen', 'Qiagen', 'Roche', 'Thermo Fisher Scientific', 'Azure Biosystems', 'BioRad', 'BioRad', 'Merck Millipore', 'BioRad', 'Biozol', 'Thermo Fisher Scientific', 'Biozol', 'Zeiss', None]\n",
      "11196847: ['Applied Biosystems', 'Megazyme']\n",
      "11196703: ['Cargill', None]\n",
      "11196417: ['null', 'Bioconductor', 'The R Foundation', 'Bioconductor', 'Bioconductor', 'null', 'null', 'null', 'STRING Consortium', 'Cytoscape Consortium', 'Cytoscape Consortium', 'The R Foundation', 'null', 'null', 'null', 'null', 'null', 'Bioconductor', 'null', 'null', 'TransGen Biotech', 'Thermo Fisher Scientific', 'TransGen Biotech', 'Vazyme', 'Thermo Fisher Scientific', 'National Institutes of Health']\n",
      "11196393: ['GE Healthcare', 'Lonza', 'BD Biosciences', 'Miltenyi Biotec', 'Thermo Fisher Scientific', 'Abcam', 'BioLegend', 'BioLegend', 'BioLegend', 'BioLegend', 'BioLegend', None, None, None, None]\n",
      "11196377: ['Cobb-Vantress', 'Evonik', 'Cobb-Vantress', None, 'A&D Weighing', 'Konica Minolta', 'SAS']\n",
      "11196265: ['None', 'None', 'None', 'El Nasr Company', 'Mindray', 'None', 'Sigma-Aldrich', 'Olympus', 'Vector Laboratories', 'Elabscience Biotechnology Inc', 'Olympus', 'Canon', 'SPSS Inc.']\n",
      "11196252: ['Metrohm', None, 'PalmSens', None, 'PalmSens', None, None, None, 'Merck', None, None, None, 'Alfa Aesar', None, None, 'Sigma-Aldrich', 'Chempur', 'Fisher Scientific', 'Hamilton', 'Hamilton']\n"
     ]
    }
   ],
   "source": [
    "for key, value in company_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198832: ['Shimadzu high-performance liquid chromatography (HPLC)', 'SCIEX mass spectrometer (MS)', 'ABI PCR amplification instrument', 'BioTek enzyme labeling instrument', 'Beijing Liuyi Instrument Factory electrophoresis apparatus', 'Beijing Bijing Biotechnology Co. gel imaging system', 'Thermo Fisher Scientific Nanodrop UV quantitative system', 'Illumina Sequencer', 'Bio-rad Corporation fluorescence quantitative PCR instrument', 'VIANEX S.A. vancomycin hydrochloride for injection', 'Hangzhou Yuanda Biological Pharmaceutical bifidobacterium quadruplex viable tablets', 'Shiyao Group Enbipu Pharmaceutical NBP capsules', 'Shiyao Group Enbipu Pharmaceutical NBP reference', 'Sichuan Vicchi Biochemical Technology glipizide reference', 'Omega Bio-Tek Soil DNA Kit', 'ABI Quant-iT PicoGreen dsDNA Assay Kit', 'Servicebio First Strand cDNA Synthesis Kit', 'Servicebio SYBR Green qPCR Master Mix']\n",
      "11198784: ['Ficoll-Paque Premium density gradient medium', 'Invitrogen trypan blue dye', 'Bio-Rad TC20 Automated cell counter', 'EasySep Negative Human NK Cell Isolation Kit', 'Becton Dickinson CD56 APC', 'Becton Dickinson CD3 PE Cy7', 'BD Bioscience stain buffer', 'BD LSR-Fortessa X-20 flow cytometer', 'Harvard Apparatus borosilicate glass capillaries', 'Sutter Instrumental P-97 glass pipette', 'Narishige Micro Forge MF-900', 'Molecular Devices CV203BU head-stage', 'Narishige manipulators', 'Molecular Devices Axopatch 200B amplifier', 'Molecular Devices pClamp 10.7 software', 'Molecular Devices Digidata 1440A', 'Sigma-Aldrich membrane filter', 'Sapphire Bioscience Reagents ATP', 'Sapphire Bioscience Reagents GTP', 'Tocris Bioscience PregS', 'Tocris Bioscience Ononetin', 'Sigma-Aldrich chemicals', 'IBM SPSS software', 'GraphPad Prism software', 'OriginLab Origin 2021 software']\n",
      "11197919: ['HI-MAIZE® 260 starch', 'VERSAFIBE™ 1490 starch', 'AMIOCA™ TF starch', 'Salimetrics Salivary Alpha-Amylase Enzymatic Kit']\n",
      "11197476: ['LiposoFast liposome extruder', 'Merck Millipore polycarbonate membrane filters', 'ultrafiltration centrifuge tube', 'JEM-1400 transmission electron microscope', 'Malvern Nano ZS ZEN3600', 'UV-VIS Thermo Fisher Scientific', 'CytoFLEX flow cytometer', 'Beyotime Biotechnology cloning kit', 'Nikon confocal laser scanning microscopy', 'CCK-8 assay kit Beyotime Biotechnology', 'IVIS in vivo imaging system', 'Biolegend Inc. antibodies', 'Jiancheng Bioengineering Institute assay kits']\n",
      "11197378: ['Qingyue Open Environmental Data Centre data', 'Gaode Map', 'EXCEL database', 'IBM SPSS 20 software', 'R language']\n",
      "11197282: ['Metabolon untargeted metabolomics profiling']\n",
      "11197204: ['TRIzol', 'TaqMan Gene Expression Assay', 'iTaq Universal One-Step Kit', 'iTaq Universal SYBR Green One-Step Kit', 'BioRad', 'GraphPad PRISM', 'Enrichr', 'ShinyGO']\n",
      "11197185: ['Illumina MiSeq platform', 'Vero cell culture (ATCC)', 'C636 cell culture (ATCC)', 'Sterilite container', 'Tetra Pond Koi growth feed', 'Hemotek membrane feeding system', 'Sigma Aldrich triethylamine', 'Trizol (Thermo Fisher Scientific)', 'REPLI-g WTA single-cell kit (Qiagen)', 'AccuStart II PCR supermix (Quantabio)']\n",
      "11196966: ['Advanced DMEM-F12 (Gibco/Thermo Fisher Scientific)', 'penicillin/streptomycin (Gibco/Thermo Fisher Scientific)', 'HEPES (Invitrogen/Thermo Fisher Scientific)', 'Glutamax (Invitrogen/Thermo Fisher Scientific)', 'N2 (Gibco/Thermo Fisher Scientific)', 'B27 (Gibco/Thermo Fisher Scientific)', 'N-acetylcysteine (Sigma-Aldrich)', 'hygromycin (Invitrogen/Thermo Fisher Scientific)', 'puromycin (InvivoGen)', 'Cultrex RGF basement membrane extract (R&D Systems)', 'cell counter TC20 (BioRad)', 'buprenorphine (Temgesic, Schering-Plough)', 'Isoflurane (Isoflurane Piramal, Piramal)', 'SomnoSuite® Low-Flow Anesthesia System (Kent Scientific Corporation)', 'Seprafilm (Baxter)', 'Vicryl (Ethicon/Johnson&Johnson)', 'BioSpec 94/21 MRI scanner (Bruker Biospin)', 'VEVO 3100 ultrasound (Fujifilm/VisualSonics)', 'Liver Dissociation Kit, mouse (Miltenyi Biotec)', 'Tumor Dissociation Kit, mouse (Miltenyi Biotec)', 'ACK lysis buffer (Gibco/Thermo Fisher Scientific)', 'TruStain FcX PLUS (BioLegend)', 'eBioscience™ Foxp3/Transcription Factor Staining Buffer Set (Invitrogen/Thermo Fisher Scientific)', 'Beckman Coulter Gallios (Beckman Coulter)', 'BD LSRFortessa (Becton Dickinson)', 'Cell Activation Cocktail (BioLegend)', 'Brefeldin A (BioLegend)', 'RNeasy Mini Kit (Qiagen)', 'ProtoScript II First Strand cDNA Synthesis Kit (New England Biolabs)', 'DNeasy Blood & Tissue Kit (Qiagen)', 'QuantiTect SYBR Green RT PCR mastermix (Qiagen)', 'Roche LightCycler 480 (Roche)', 'Pierce BCA-Kit (Thermo Fisher Scientific)', 'Azure Ao Microplate Reader (Azure Biosystems)', 'Laemmli Buffer (BioRad)', 'Mini-Protean TGX precast gel (BioRad)', 'Immobilon Crescendo Western HRP substrate (Merck Millipore)', 'ChemiDoc BioRad imager (BioRad)', 'Vector® TrueVIEW® Autofluorescence Quenching Kit (Biozol)', 'NucBlue™ Fixed Cell Stain ReadyProbes™ (Thermo Fisher Scientific)', 'VECTASHIELD Vibrance Antifade Mounting Medium (Biozol)', 'Zeiss Axio Scan Z.1 Microscope Slide Scanner (Zeiss)', 'GraphPad Prism']\n",
      "11196847: ['ABI 3130 Genetic Analyzer', 'Megazyme mushroom and yeast β-glucan assay kit']\n",
      "11196703: ['ENSO 16 sugar substitute', 'glucose powder']\n",
      "11196417: ['GEO database', 'limma package', 'ggplot2 package', 'clusterProfiler package', 'enrichplot package', 'ImmuCellAI methodology', 'ssGSEA methodology', 'Mantel algorithm', 'STRING 4 online platform', 'Cytoscape V3.9.0 software', 'CytoHubba plugin', 'Random Forest package', 'SVM-RFE approach', 'LASSO model', 'Logistic regression approach', 'ROC curve', 'AUC', 'Limma package', 'miRNet database', 'DGIdb database', 'TransZol Up Plus RNA Kit', 'Nanodrop Spectrophotometer', 'TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix', 'ChamQ Universal SYBR qPCR Master Mix', 'QuantStudio™ 5 Real-Time PCR System', 'Image J software']\n",
      "11196393: ['Ficoll gradient', 'ACK lysing buffer', 'BD FACSAria III cell sorter', 'Fc-block (Miltenyi)', 'A1exF5 antibody (Thermo Fisher)', 'DCFDA (Abcam)', 'APC antihuman CD11b(ICRF44) antibody (BioLegend)', 'Brilliant Violet 605™ antihuman HLA-DR(L243) antibody (BioLegend)', 'APC/Cyanine7 antihuman CD14(63D3) antibody (BioLegend)', 'PerCP/Cyanine5.5 antihuman CXCR2(5E8/CXCR2) antibody (BioLegend)', 'FITC antihuman CD177(MEM-166) antibody (BioLegend)', 'Cell Ranger software', 'Seurat toolkit', 'monocle2 package', 'GraphPad Prism software']\n",
      "11196377: ['Cobb-500 broiler chicks', 'Evonik Nutrition & Care GmbH near-infrared analysis', 'Cobb-Vantress finisher diet', 'UFFDA software', 'A&D Weighing digital weighing scale', 'Konica Minolta Chroma-Meter CR-410', 'SAS 9.4 software']\n",
      "11196265: ['Polyethylene microplastics (PE-MPs) powder', 'Milli-Q purified water', 'tadalafil (Cilais)', 'commercial fish pellet feed (El Nasr Company)', 'BC-2800 automated technical analyzer (Mindray)', 'spectrophotometer (Biodiagonstic Company)', 'Sigma-Aldrich total antioxidant capacity kits', 'Olympus CH30 microscope', 'Vector Laboratories standard goat serum', 'E-AB-40021 IL-6 antibody (Elabscience Biotechnology Inc)', 'Olympus BX41 microscope', 'Canon digital camera (PowershotA95)', 'SPSS 16.0 program (SPSS 2007, Inc.)']\n",
      "11196252: ['Metrohm Autolab AUTOLAB-PGSTAT302N instrument', 'NOVA 1.11.1 software', 'EmStat3 potentiostat', 'PSTrace software', 'PalmSens B.V.', 'automated electrode stand M164', 'MTM Anko Instruments', 'Basi® glassy carbon electrode', 'Merck DANO', 'Potassium tetrakis(4-chlorophenyl)borate KTPBCl', 'bis(triphenylphosphoranylidene) ammonium chloride BTPPACl', 'Tetrapropylammonium chloride TPrACl', 'Alfa Aesar', 'POCH Britton-Robinson buffers', '1,2-dichloroethane 1,2-DCE', 'Sigma-Aldrich', 'Chempur', 'Fisher Chemical', 'Polilyte Lab pH electrode', 'Hamilton']\n"
     ]
    }
   ],
   "source": [
    "for key, value in product_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Mistral"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "system_prompt = \"\"\"You are an expert Company Identification system. \n",
    "Your task is to take a list of strings as input and identify the company that manufactures each product mentioned.\n",
    "Strings provided in the list can be misleading; use your internal knowledge to identify company names.\n",
    "If the manufacturer is unclear for any product, return a null value for that string.\n",
    "Here is an example of the output format for a list of strings.\n",
    "\n",
    "List of strings: [\"PlayStation 5\", \"iPhone 15 Plus\", \"Pencil\", \"Stainless Steel Tumbler\", \"Whopper\", \"Amazon Speed ETD\"]\n",
    "Answer: [\"Sony\", \"Apple\", None, None, \"Burger King\", \"Bruker\"]\n",
    "\n",
    "Only use this output format. Do not output anything besides this output format.\n",
    "Output the manufacturer names in the order they occur in the input list of strings.\n",
    "\"\"\"\n",
    "\n",
    "user_prompt = \"\"\"Q: Given the list of strings below, identify the companies that manufactures each product.\n",
    "\n",
    "List of strings: {}\n",
    "Answer:\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "api_key = os.environ[\"MISTRAL_API_KEY\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "def company_identification(input_text):\n",
    "    client = MistralClient(api_key=api_key)\n",
    "    response = client.chat(\n",
    "        model=\"mistral-large-latest\",\n",
    "        temperature=1e-16,\n",
    "        messages=[\n",
    "        ChatMessage(role=\"system\", content=system_prompt),\n",
    "        ChatMessage(role=\"user\", content=user_prompt.format(input_text))\n",
    "    ]\n",
    "    )\n",
    "    return response.choices[0].message.content"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "product_dict = {\n",
    "    \"11198919\": [],\n",
    "    \"11198832\": [\"Shimadzu high-performance liquid chromatography (HPLC)\", \"AB SCIEX mass spectrometer (MS)\", \"ABI PCR amplification instrument\", \"BioTek enzyme labeling instrument\", \"Beijing Liuyi electrophoresis apparatus\", \"Beijing Bijing Biotechnology Co., LTD. gel imaging system\", \"Thermo Fisher Scientific Nanodrop UV quantitative system\", \"Illumina Sequencer\", \"Bio-rad Corporation fluorescence quantitative PCR instrument\", \"vancomycin hydrochloride for injection (VIANEX S.A.)\", \"bifidobacterium quadruplex viable tablets (Hangzhou Yuanda Biological Pharmaceutical)\", \"NBP capsules (Shiyao Group Enbipu Pharmaceutical)\", \"NBP reference (Shiyao Group Enbipu Pharmaceutical)\", \"glipizide reference (Sichuan Vicchi Biochemical Technology)\", \"Omega Bio-Tek Soil DNA Kit\", \"ABI Quant-iT PicoGreen dsDNA Assay Kit\", \"Servicebio First Strand cDNA Synthesis Kit\", \"Servicebio SYBR Green qPCR Master Mix\"],\n",
    "    \"11198784\": [],\n",
    "    \"11198029\": [],\n",
    "    \"11197919\": [\"HI-MAIZE® 260 starch\", \"VERSAFIBE™ 1490 starch\", \"AMIOCA™ TF starch\", \"Tanita SC-240 Total Body Composition Analyzer\", \"Automated Self-Administered 24-hour (ASA24) Dietary Assessment Tool 2020\", \"QIAamp 96 DNA Blood Kit\", \"QIAamp Blood Mini Kit\", \"QIAamp Investigator Kit\", \"iTaq™ Universal SYBR® Green Supermix\", \"Salimetrics Salivary Alpha-Amylase Enzymatic Kit\", \"DNeasy PowerSoil 96 HTP Kit\", \"Classic++™ Hot Start Taq DNA Polymerase Master Mix\", \"Mag-Bind TotalPure\", \"QIIME 2\", \"DADA2\", \"mafft\", \"fasttree\", \"Greengenes 13_8 99% OTUs reference sequences\", \"RStudio version 4.2.1\", \"ComplexHeatmap\", \"circlize\", \"MaAsLin2 (Microbiome Multivariable Association with Linear Models 2)\", \"glmnet\"],\n",
    "    \"11197476\": [\"CXB@Lipo (C@L)\", \"chloroform\", \"HSPC\", \"LiposoFast (Avestin, Canada)\", \"polycarbonate membrane filters (Merck Millipore Ltd., USA)\", \"ultrafiltration centrifuge tube\", \"ECN\", \"glycol chitosan\", \"transmission electron microscopy (TEM; JEM-1400, Japan)\", \"Malvern Nano ZS ZEN3600 (Malvern, UK)\", \"UV-VIS (Thermo Fisher Scientific, USA)\", \"ICG@ECN (I@ECN)\", \"flow cytometry (CytoFLEX, Beckman, USA)\", \"UV‒VIS\", \"pGEX-Pvhb-Lysis\", \"cloning kit (Beyotime Biotechnology Co. Ltd., Shanghai, China)\", \"ICG@Liposome (I@L)\", \"confocal laser scanning microscopy (CLSM) (Nikon, Japan)\", \"CCK-8 assay kit (Beyotime Biotechnology Co. Ltd., Shanghai, China)\", \"Western blot (WB) analysis\", \"FITC-conjugated anti-COX-2 antibody (Affinity, Jiangsu, China)\", \"in vivo images system (IVIS)\", \"optimal cutting temperature (OCT) compound\", \"CT26 tumor model\", \"FITC-conjugated anti-mouse CD3\", \"FITC-conjugated anti-mouse CD4\", \"PE-conjugated anti-mouse CD8 (Biolegend Inc., USA)\", \"TUNEL and H&E staining\", \"Ki67\", \"C@ECN-PL\", \"aspartate aminotransferase (AST)\", \"alanine aminotransferase (ALT)\", \"blood urea nitrogen (BUN)\", \"creatinine (CRE)\", \"assay kits (Jiancheng Bioengineering Institute, Nanjing, China)\", \"H&E staining analysis\", \"ImageJ 6.0\"],\n",
    "    \"11197378\": [\"EXCEL\", \"IBM SPSS 20\", \"R language\"],\n",
    "    \"11197282\": [],\n",
    "    \"11197204\": [\"Gene Expression Omnibus (GEO)\", \"quantitative PCR method\", \"UK National Research Ethics Committee (NRES 16/SS/0172)\", \"BH false discovery rate correction\", \"TRIzol (Life Technologies)\", \"TaqMan™ Gene Expression Assay (FAM) from Life Technologies Ltd\", \"Easy Oligos from Merck Life Science UK Ltd\", \"iTaq Universal One-Step or iTaq™ Universal SYBR® Green One-Step Kit from BioRad\", \"GraphPad PRISM\", \"ShinyGO\", \"Enrichr\", \"Tools\"],\n",
    "    \"11197185\": [\"ZIKV\", \"Vero cell culture (ATCC)\", \"C636 cell culture (ATCC)\", \"Tetra Pond Koi growth feed\", \"defibrinated sheep blood\", \"sodium bicarbonate\", \"Hemotek membrane feeding system (Discovery Workshops, Accrington, UK)\", \"triethylamine (Sigma Aldrich, St. Louis, MO, USA)\", \"filter sterilized fetal bovine serum\", \"Mosquito Diluent [MD]\", \"penicillin/streptomycin\", \"gentamicin\", \"Fungizone (Sigma Aldrich, St. Louis, MO, USA)\", \"phosphate-buffered Saline (PBS) (Sigma Aldrich, St. Louis, MO, USA)\", \"Illumina MiSeq\", \"REPLI-g whole transcriptome amplification (WTA) single-cell kit (Qiagen, Hilden, Germany)\", \"Illumina barcoded 16S primer set\", \"AccuStart II PCR supermix (Quantabio, Beverly, MA, USA)\", \"QIITA\", \"MicrobiomeAnalyst\", \"GreenGenes 16S reference database\", \"BIOM-formatted OTU table\", \"NCBI GenBank Short Read Archives\"],\n",
    "    \"11196966\": [\"Advanced DMEM-F12 (Gibco/Thermo Fisher Scientific)\", \"penicillin (Gibco/Thermo Fisher Scientific)\", \"streptomycin (Gibco/Thermo Fisher Scientific)\", \"HEPES (Invitrogen/Thermo Fisher Scientific)\", \"Glutamax (Invitrogen/Thermo Fisher Scientific)\", \"N2 (Gibco/Thermo Fisher Scientific)\", \"B27 (Gibco/Thermo Fisher Scientific)\", \"N-acetylcysteine (Sigma-Aldrich)\", \"hygromycin (Invitrogen/Thermo Fisher Scientific)\", \"puromycin (InvivoGen)\", \"Cultrex RGF basement membrane extract (BME) Type 2 (R&D Systems)\", \"cell counter TC20 (BioRad)\", \"collagen I/5x neutralization buffer\", \"buprenorphine (Temgesic, Schering-Plough)\", \"Isoflurane Piramal (Piramal)\", \"SomnoSuite® Low-Flow Anesthesia System (Kent Scientific Corporation)\", \"Seprafilm (Baxter)\", \"6-0 Vicryl (Ethicon/Johnson&Johnson) suture\", \"9.4 tesla small bore animal scanner (BioSpec 94/21, Bruker Biospin)\", \"dedicated mouse quadrature-resonator (Bruker)\", \"VEVO 3100 (Fujifilm/VisualSonics) ultrasound\", \"small animal transducer\", \"RNAprotect buffer (Qiagen, Netherlands)\", \"Liver Dissociation Kit, mouse (Miltenyi Biotec)\", \"Tumor Dissociation Kit, mouse (Miltenyi Biotec)\", \"ACK lysis buffer (Gibco/Thermo Fisher Scientific)\", \"TruStain FcX PLUS (anti-mouse CD16/32) antibody (BioLegend)\", \"FACS buffer (PBS + 0,5% BSA)\", \"1x Fixation/Permeabilization buffer\", \"eBioscience™ Foxp3/Transcription Factor Staining Buffer Set (Invitrogen/Thermo Fisher Scientific)\", \"FACS antibodies (BioLegend)\", \"Beckman Coulter Gallios (Beckman Coulter)\", \"BD LSRFortessa (Becton Dickinson)\", \"Kaluza Software (Beckman Coulter)\", \"RPMI-Medium (+ 1% Pen/Strep + 1% HEPES + 5 µl beta-mercaptoethanol)\", \"Cell Activation Cocktail (BioLegend)\", \"Brefeldin A (BioLegend)\", \"RLT buffer (Qiagen, Venlo, Netherlands)\", \"RNeasy Mini Kit (Qiagen)\", \"NanoDrop™ spectrophotometer (Thermo Fisher Scientific)\", \"ProtoScript II First Strand cDNA Synthesis Kit (New England Biolabs)\", \"QuantiTect SYBR Green RT PCR mastermix (Qiagen)\", \"DNeasy Blood & Tissue Kit (Qiagen)\", \"primer sequences (purchased from Eurofins Genomics)\", \"Roche LightCycler 480 (Roche)\", \"QuantiFast SYBR® Green PCR Kit (Qiagen)\", \"Bifidobacterium actinocoloniiforme (DSMZ-Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, DSM22766)\"],\n",
    "    \"11196847\": [\"Potato dextrose agar (PDA) medium\", \"Potato dextrose broth (PDB)\", \"ABI 3130 Genetic Analyzer\", \"Taq DNA polymerase Assay Buffer\", \"ABI 3130xl gel sequencing\", \"Phylogenetic Tree Builder\", \"Ethanol (80%, v/v)\", \"Whatman paper No. 4\", \"Lowry's assay\", \"Folin-Ciocalteu reagent\", \"Bovine Serum Albumin (BSA)\", \"Anthrone reagent\", \"Anthrone technique\", \"Quercetin\", \"Folin-Ciocalteu method\", \"Gallic acid equivalent (GAE)\", \"Aescin\", \"Ferric chloride\", \"Mg(CH3COO)2 in CH3OH\", \"Rhein\", \"NH4OH\", \"Liebermann-Burchard reagent\", \"DPPH radicals\", \"Liebermann-Burchard reagent\", \"β-glucan assay kit (Megazyme Int., Dublin, Ireland)\", \"High-performance liquid chromatography (PDA)\", \"RP C-18 column\", \"Quercetin\", \"Sesquiterpene glycosides\", \"Ergosterol\", \"RidentinB\", \"Statistical Package for the Social Sciences (SPSS)\"],\n",
    "    \"11196703\": [\"ENSO 16 sugar substitute\", \"glucose in water solution\", \"NEOH by Alpha Republic GmbH\", \"STAMAG Stadlauer Malzfabrik GesmbH\", \"Special Ingredients Ltd glucose powder\", \"GraphPad Prism 9\"],\n",
    "    \"11196440\": [],\n",
    "    \"11196417\": [\"GEO database\", \"limma package\", \"R software\", \"ggplot2 package\", \"GSEA official website\", \"MSigDB database\", \"clusterProfiler package\", \"enrichplot package\", \"ImmuCellAI\", \"ssGSEA\", \"STRING 4 online platform\", \"Cytoscape V3.9.0 software\", \"CytoHubba plugin\", \"Random Forest algorithm\", \"SVM-RFE\", \"LASSO model\", \"Logistic regression\", \"Receiver Operating Characteristic (ROC) curve\", \"Area Under the Curve (AUC)\", \"miRNet database\", \"Drug-Gene Interaction database (DGIdb)\", \"ggplot2 package\", \"TransZol Up Plus RNA Kit\", \"Nanodrop Spectrophotometer\", \"TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix\", \"ChamQ Universal SYBR qPCR Master Mix\", \"QuantStudio™ 5 Real-Time PCR System\", \"Image J software\"],\n",
    "    \"11196393\": [\"Ficoll gradient\", \"ACK lysing buffer\", \"RPMI1640 supplemented with 20% FBS\", \"BD FACSAria III cell sorter\", \"Fc-block (Miltenyi)\", \"surface antibodies against CD11b, HLA-DR, CD14, CD177, and CXCR2\", \"anti-Arg1 (A1exF5; Thermo Fisher)\", \"DCFDA (Abcam)\", \"BD FACSCanto II\", \"APC antihuman CD11b(ICRF44)\", \"Brilliant Violet 605™ antihuman HLA-DR(L243)\", \"APC/Cyanine7 antihuman CD14(63D3)\", \"PerCP/Cyanine5.5 antihuman CXCR2(5E8/CXCR2)\", \"FITC antihuman CD177(MEM-166)\", \"Cell Ranger (version 7.0)\", \"Seurat (version 4.0.1)\", \"monocle2 (version 2.26.0)\", \"GraphPad Prism 8.0\"],\n",
    "    \"11196377\": [\"Cobb-500 chicks\", \"Evonik Co. near-infrared analysis\", \"UFFDA software\", \"Cobb-Vantress commercial management guide\", \"Vitamin premix\", \"Mineral premix\", \"SAS 9.4 software\"],\n",
    "    \"11196265\": [\"Polyethylene microplastics (PE-MPs) powder\", \"Purified water (Milli-Q)\", \"Tadalafil (Cilais) substance\", \"Sildenafil\", \"Commercial fish pellet feed (containing 30% protein and 12% lipid; El Nasr Company, Egypt)\", \"Automated technical analyzer (BC-2800 from Mindray)\", \"Spectrophotometer (in a wavelength range of 340-546 nm; Biodiagonstic Company, Egypt)\", \"Kits for TAC (total antioxidant capacity) determination (Sigma-Aldrich, USA)\", \"Olympus CH30 microscope\", \"IL-6 antibody (E-AB-40021, Elabscience Biotechnology Inc, USA)\", \"Streptavidin-biotin-peroxidase pack\", \"Mayer's hematoxylin\", \"SPSS 16.0 program for Windows (SPSS 2007, Inc., IL, USA)\", \"Canon digital camera (PowershotA95)\"],\n",
    "    \"11196252\": [\"AUTOLAB-PGSTAT302N instrument\", \"NOVA 1.11.1 software\", \"platinum (Pt) wires\", \"silver/silver chloride wires (Ag/AgCl)\", \"EmStat3 potentiostat\", \"PSTrace software\", \"automated electrode stand (M164, MTM Anko Instruments)\", \"glassy carbon electrode (GCE)\", \"silver/silver chloride electrode (Ag/AgCl/3.0 mol L−1 KCl)\", \"platinum wire (Pt, 99.99%, The Mint of Poland)\", \"DANO (Merck)\", \"hydrochloric acid (HCl)\", \"Potassium tetrakis(4-chlorophenyl)borate (KTPBCl, Merck)\", \"bis(triphenylphosphoranylidene) ammonium chloride (BTPPACl, Merck)\", \"bis(triphenylphosphoranylidene)ammonium tetrakis(4-chlorophenyl)borate (BTPPATPBCl)\", \"Tetrapropylammonium chloride (TPrACl, Alfa Aesar)\", \"Britton-Robinson buffers (BRBs)\", \"1,2-dichloroethane (1,2-DCE, POCH)\", \"citric acid (Sigma-Aldrich)\", \"galactose (Chempur)\", \"lactose (Fisher Chemical)\", \"glucose (Alfa Aesar)\", \"calcium chloride (Sigma-Aldrich)\", \"potassium chloride (Chempur)\", \"magnesium chloride (Fisher Chemical)\", \"iron(III) chloride (Alfa Aesar)\", \"sodium lactate (Sigma-Aldrich)\", \"orthophosphoric acid (V) (Alfa Aesar)\", \"combined pH electrode (Polilyte Lab, Hamilton, Switzerland)\"]\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198919: []\n",
      "11198832: [\"Shimadzu\", \"AB SCIEX\", \"Applied Biosystems (ABI)\", \"BioTek\", None, None, \"Thermo Fisher Scientific\", \"Illumina\", \"Bio-Rad\", \"VIANEX S.A.\", \"Hangzhou Yuanda Biological Pharmaceutical\", \"Shiyao Group Enbipu Pharmaceutical\", \"Shiyao Group Enbipu Pharmaceutical\", \"Sichuan Vicchi Biochemical Technology\", \"Omega Bio-Tek\", \"Applied Biosystems (ABI)\", None, \"Servicebio\"]\n",
      "11198784: []\n",
      "11198029: []\n",
      "11197919: [\"Ingredion\", \"Ingredion\", \"Ingredion\", \"Tanita\", None, \"QIAGEN\", \"QIAGEN\", \"QIAGEN\", \"Bio-Rad Laboratories\", \"Salimetrics\", \"QIAGEN\", \"Bio-Rad Laboratories\", \"Omega Bio-tek\", None, None, None, None, None, \"RStudio\", None, None, None, None]\n",
      "11197476: [\"Avestin\", None, None, \"Avestin\", \"Merck Millipore Ltd.\", None, None, None, \"JEM-1400\", \"Malvern\", \"Thermo Fisher Scientific\", None, \"Beckman\", None, None, \"Beyotime Biotechnology Co. Ltd.\", None, \"Nikon\", \"Beyotime Biotechnology Co. Ltd.\", None, \"Affinity\", None, None, \"Biolegend Inc.\", None, None, None, None, \"Biolegend Inc.\", None, None, \"Jiancheng Bioengineering Institute\", None, None, \"ImageJ\"]\n",
      "11197378: [\"Microsoft\", \"IBM\", None]\n",
      "11197282: []\n",
      "11197204: [\"National Center for Biotechnology Information\", None, None, None, \"Thermo Fisher Scientific\", \"Thermo Fisher Scientific\", \"Merck Life Science UK Ltd\", \"BioRad\", \"GraphPad Software\", None, None, None]\n",
      "11197185: [\"None\", \"ATCC\", \"ATCC\", \"Tetra\", \"None\", \"None\", \"Hemotek\", \"Sigma Aldrich\", \"None\", \"None\", \"None\", \"None\", \"Sigma Aldrich\", \"Sigma Aldrich\", \"Illumina\", \"Qiagen\", \"Illumina\", \"Quantabio\", \"QIITA\", \"MicrobiomeAnalyst\", \"GreenGenes\", \"None\", \"NCBI\"]\n",
      "11196966: [\"Thermo Fisher Scientific\", \"Thermo Fisher Scientific\", \"Thermo Fisher Scientific\", \"Thermo Fisher Scientific\", \"Thermo Fisher Scientific\", \"Thermo Fisher Scientific\", \"Thermo Fisher Scientific\", \"Sigma-Aldrich\", \"Thermo Fisher Scientific\", \"InvivoGen\", \"R&D Systems\", \"BioRad\", None, \"Schering-Plough\", \"Piramal\", \"Kent Scientific Corporation\", \"Baxter\", \"Johnson&Johnson\", \"Bruker Biospin\", \"Bruker\", \"Fujifilm/VisualSonics\", None, \"Qiagen\", \"Miltenyi Biotec\", \"Miltenyi Biotec\", \"Thermo Fisher Scientific\", \"BioLegend\", None, \"Thermo Fisher Scientific\", \"BioLegend\", \"BioLegend\", \"Beckman Coulter\", \"Becton Dickinson\", \"Beckman Coulter\", None, \"BioLegend\", \"BioLegend\", \"Qiagen\", \"Qiagen\", \"Thermo Fisher Scientific\", \"New England Biolabs\", \"Qiagen\", None, \"Qiagen\", \"Eurofins Genomics\", \"Roche\", \"Qiagen\", \"DSMZ-Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH\"]\n",
      "11196847: [\"None\", \"None\", \"Applied Biosystems\", \"None\", \"Applied Biosystems\", \"None\", \"None\", \"GE Healthcare Life Sciences\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"IBM\"]\n",
      "11196703: [\"Alpha Republic GmbH\", None, \"Alpha Republic GmbH\", \"STAMAG Stadlauer Malzfabrik GesmbH\", \"Special Ingredients Ltd\", \"GraphPad Software\"]\n",
      "11196440: []\n",
      "11196417: [\"None\", \"None\", \"R Core Team\", \"Hadley Wickham\", \"Broad Institute\", \"Broad Institute\", \"Yu Guan\", \"Yu Guan\", \"None\", \"None\", \"None\", \"Cytoscape Consortium\", \"Chin-Chu Chen\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"None\", \"ggplot2 developers\", \"TransGen Biotech\", \"Thermo Fisher Scientific\", \"TransGen Biotech\", \"Vazyme Biotech\", \"Thermo Fisher Scientific\", \"National Institutes of Health\"]\n",
      "11196393: [\"GE Healthcare\", None, None, \"Becton, Dickinson and Company\", \"Miltenyi Biotec\", None, \"Thermo Fisher Scientific\", \"Abcam\", \"Becton, Dickinson and Company\", \"BioLegend\", \"BioLegend\", \"BioLegend\", \"BioLegend\", \"BioLegend\", \"10x Genomics\", None, None, \"GraphPad Software\"]\n",
      "11196377: [\"Cobb-Vantress\", \"Evonik\", None, \"Cobb-Vantress\", None, None, \"SAS Institute\"]\n",
      "11196265: [None, None, \"Eli Lilly and Company\", \"Pfizer\", \"El Nasr Company\", \"Mindray\", \"Biodiagonstic Company\", \"Sigma-Aldrich\", \"Olympus\", \"Elabscience Biotechnology Inc\", None, None, \"SPSS Inc.\", \"Canon\"]\n",
      "11196252: [\"Metrohm Autolab\", \"Metrohm Autolab\", None, None, \"PalmSens\", \"Metrohm Autolab\", \"MTM Anko Instruments\", None, \"Metrohm Autolab\", \"The Mint of Poland\", \"Merck\", None, \"Merck\", \"Merck\", \"Merck\", None, \"Alfa Aesar\", None, \"POCH\", \"Sigma-Aldrich\", \"Chempur\", \"Fisher Chemical\", \"Alfa Aesar\", \"Sigma-Aldrich\", \"Chempur\", \"Fisher Chemical\", \"Alfa Aesar\", \"Sigma-Aldrich\", \"Alfa Aesar\", \"Hamilton\"]\n"
     ]
    }
   ],
   "source": [
    "for key, value in product_dict.items():\n",
    "    identified_companies = company_identification(value)\n",
    "    print(f\"{key}: {identified_companies}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198919: []\n",
      "11198832: ['Shimadzu high-performance liquid chromatography (HPLC)', 'AB SCIEX mass spectrometer (MS)', 'ABI PCR amplification instrument', 'BioTek enzyme labeling instrument', 'Beijing Liuyi electrophoresis apparatus', 'Beijing Bijing Biotechnology Co., LTD. gel imaging system', 'Thermo Fisher Scientific Nanodrop UV quantitative system', 'Illumina Sequencer', 'Bio-rad Corporation fluorescence quantitative PCR instrument', 'vancomycin hydrochloride for injection (VIANEX S.A.)', 'bifidobacterium quadruplex viable tablets (Hangzhou Yuanda Biological Pharmaceutical)', 'NBP capsules (Shiyao Group Enbipu Pharmaceutical)', 'NBP reference (Shiyao Group Enbipu Pharmaceutical)', 'glipizide reference (Sichuan Vicchi Biochemical Technology)', 'Omega Bio-Tek Soil DNA Kit', 'ABI Quant-iT PicoGreen dsDNA Assay Kit', 'Servicebio First Strand cDNA Synthesis Kit', 'Servicebio SYBR Green qPCR Master Mix']\n",
      "11198784: []\n",
      "11198029: []\n",
      "11197919: ['HI-MAIZE® 260 starch', 'VERSAFIBE™ 1490 starch', 'AMIOCA™ TF starch', 'Tanita SC-240 Total Body Composition Analyzer', 'Automated Self-Administered 24-hour (ASA24) Dietary Assessment Tool 2020', 'QIAamp 96 DNA Blood Kit', 'QIAamp Blood Mini Kit', 'QIAamp Investigator Kit', 'iTaq™ Universal SYBR® Green Supermix', 'Salimetrics Salivary Alpha-Amylase Enzymatic Kit', 'DNeasy PowerSoil 96 HTP Kit', 'Classic++™ Hot Start Taq DNA Polymerase Master Mix', 'Mag-Bind TotalPure', 'QIIME 2', 'DADA2', 'mafft', 'fasttree', 'Greengenes 13_8 99% OTUs reference sequences', 'RStudio version 4.2.1', 'ComplexHeatmap', 'circlize', 'MaAsLin2 (Microbiome Multivariable Association with Linear Models 2)', 'glmnet']\n",
      "11197476: ['CXB@Lipo (C@L)', 'chloroform', 'HSPC', 'LiposoFast (Avestin, Canada)', 'polycarbonate membrane filters (Merck Millipore Ltd., USA)', 'ultrafiltration centrifuge tube', 'ECN', 'glycol chitosan', 'transmission electron microscopy (TEM; JEM-1400, Japan)', 'Malvern Nano ZS ZEN3600 (Malvern, UK)', 'UV-VIS (Thermo Fisher Scientific, USA)', 'ICG@ECN (I@ECN)', 'flow cytometry (CytoFLEX, Beckman, USA)', 'UV‒VIS', 'pGEX-Pvhb-Lysis', 'cloning kit (Beyotime Biotechnology Co. Ltd., Shanghai, China)', 'ICG@Liposome (I@L)', 'confocal laser scanning microscopy (CLSM) (Nikon, Japan)', 'CCK-8 assay kit (Beyotime Biotechnology Co. Ltd., Shanghai, China)', 'Western blot (WB) analysis', 'FITC-conjugated anti-COX-2 antibody (Affinity, Jiangsu, China)', 'in vivo images system (IVIS)', 'optimal cutting temperature (OCT) compound', 'CT26 tumor model', 'FITC-conjugated anti-mouse CD3', 'FITC-conjugated anti-mouse CD4', 'PE-conjugated anti-mouse CD8 (Biolegend Inc., USA)', 'TUNEL and H&E staining', 'Ki67', 'C@ECN-PL', 'aspartate aminotransferase (AST)', 'alanine aminotransferase (ALT)', 'blood urea nitrogen (BUN)', 'creatinine (CRE)', 'assay kits (Jiancheng Bioengineering Institute, Nanjing, China)', 'H&E staining analysis', 'ImageJ 6.0']\n",
      "11197378: ['EXCEL', 'IBM SPSS 20', 'R language']\n",
      "11197282: []\n",
      "11197204: ['Gene Expression Omnibus (GEO)', 'quantitative PCR method', 'UK National Research Ethics Committee (NRES 16/SS/0172)', 'BH false discovery rate correction', 'TRIzol (Life Technologies)', 'TaqMan™ Gene Expression Assay (FAM) from Life Technologies Ltd', 'Easy Oligos from Merck Life Science UK Ltd', 'iTaq Universal One-Step or iTaq™ Universal SYBR® Green One-Step Kit from BioRad', 'GraphPad PRISM', 'ShinyGO', 'Enrichr', 'Tools']\n",
      "11197185: ['ZIKV', 'Vero cell culture (ATCC)', 'C636 cell culture (ATCC)', 'Tetra Pond Koi growth feed', 'defibrinated sheep blood', 'sodium bicarbonate', 'Hemotek membrane feeding system (Discovery Workshops, Accrington, UK)', 'triethylamine (Sigma Aldrich, St. Louis, MO, USA)', 'filter sterilized fetal bovine serum', 'Mosquito Diluent [MD]', 'penicillin/streptomycin', 'gentamicin', 'Fungizone (Sigma Aldrich, St. Louis, MO, USA)', 'phosphate-buffered Saline (PBS) (Sigma Aldrich, St. Louis, MO, USA)', 'Illumina MiSeq', 'REPLI-g whole transcriptome amplification (WTA) single-cell kit (Qiagen, Hilden, Germany)', 'Illumina barcoded 16S primer set', 'AccuStart II PCR supermix (Quantabio, Beverly, MA, USA)', 'QIITA', 'MicrobiomeAnalyst', 'GreenGenes 16S reference database', 'BIOM-formatted OTU table', 'NCBI GenBank Short Read Archives']\n",
      "11196966: ['Advanced DMEM-F12 (Gibco/Thermo Fisher Scientific)', 'penicillin (Gibco/Thermo Fisher Scientific)', 'streptomycin (Gibco/Thermo Fisher Scientific)', 'HEPES (Invitrogen/Thermo Fisher Scientific)', 'Glutamax (Invitrogen/Thermo Fisher Scientific)', 'N2 (Gibco/Thermo Fisher Scientific)', 'B27 (Gibco/Thermo Fisher Scientific)', 'N-acetylcysteine (Sigma-Aldrich)', 'hygromycin (Invitrogen/Thermo Fisher Scientific)', 'puromycin (InvivoGen)', 'Cultrex RGF basement membrane extract (BME) Type 2 (R&D Systems)', 'cell counter TC20 (BioRad)', 'collagen I/5x neutralization buffer', 'buprenorphine (Temgesic, Schering-Plough)', 'Isoflurane Piramal (Piramal)', 'SomnoSuite® Low-Flow Anesthesia System (Kent Scientific Corporation)', 'Seprafilm (Baxter)', '6-0 Vicryl (Ethicon/Johnson&Johnson) suture', '9.4 tesla small bore animal scanner (BioSpec 94/21, Bruker Biospin)', 'dedicated mouse quadrature-resonator (Bruker)', 'VEVO 3100 (Fujifilm/VisualSonics) ultrasound', 'small animal transducer', 'RNAprotect buffer (Qiagen, Netherlands)', 'Liver Dissociation Kit, mouse (Miltenyi Biotec)', 'Tumor Dissociation Kit, mouse (Miltenyi Biotec)', 'ACK lysis buffer (Gibco/Thermo Fisher Scientific)', 'TruStain FcX PLUS (anti-mouse CD16/32) antibody (BioLegend)', 'FACS buffer (PBS + 0,5% BSA)', '1x Fixation/Permeabilization buffer', 'eBioscience™ Foxp3/Transcription Factor Staining Buffer Set (Invitrogen/Thermo Fisher Scientific)', 'FACS antibodies (BioLegend)', 'Beckman Coulter Gallios (Beckman Coulter)', 'BD LSRFortessa (Becton Dickinson)', 'Kaluza Software (Beckman Coulter)', 'RPMI-Medium (+ 1% Pen/Strep + 1% HEPES + 5 µl beta-mercaptoethanol)', 'Cell Activation Cocktail (BioLegend)', 'Brefeldin A (BioLegend)', 'RLT buffer (Qiagen, Venlo, Netherlands)', 'RNeasy Mini Kit (Qiagen)', 'NanoDrop™ spectrophotometer (Thermo Fisher Scientific)', 'ProtoScript II First Strand cDNA Synthesis Kit (New England Biolabs)', 'QuantiTect SYBR Green RT PCR mastermix (Qiagen)', 'DNeasy Blood & Tissue Kit (Qiagen)', 'primer sequences (purchased from Eurofins Genomics)', 'Roche LightCycler 480 (Roche)', 'QuantiFast SYBR® Green PCR Kit (Qiagen)', 'Bifidobacterium actinocoloniiforme (DSMZ-Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, DSM22766)']\n",
      "11196847: ['Potato dextrose agar (PDA) medium', 'Potato dextrose broth (PDB)', 'ABI 3130 Genetic Analyzer', 'Taq DNA polymerase Assay Buffer', 'ABI 3130xl gel sequencing', 'Phylogenetic Tree Builder', 'Ethanol (80%, v/v)', 'Whatman paper No. 4', \"Lowry's assay\", 'Folin-Ciocalteu reagent', 'Bovine Serum Albumin (BSA)', 'Anthrone reagent', 'Anthrone technique', 'Quercetin', 'Folin-Ciocalteu method', 'Gallic acid equivalent (GAE)', 'Aescin', 'Ferric chloride', 'Mg(CH3COO)2 in CH3OH', 'Rhein', 'NH4OH', 'Liebermann-Burchard reagent', 'DPPH radicals', 'Liebermann-Burchard reagent', 'β-glucan assay kit (Megazyme Int., Dublin, Ireland)', 'High-performance liquid chromatography (PDA)', 'RP C-18 column', 'Quercetin', 'Sesquiterpene glycosides', 'Ergosterol', 'RidentinB', 'Statistical Package for the Social Sciences (SPSS)']\n",
      "11196703: ['ENSO 16 sugar substitute', 'glucose in water solution', 'NEOH by Alpha Republic GmbH', 'STAMAG Stadlauer Malzfabrik GesmbH', 'Special Ingredients Ltd glucose powder', 'GraphPad Prism 9']\n",
      "11196440: []\n",
      "11196417: ['GEO database', 'limma package', 'R software', 'ggplot2 package', 'GSEA official website', 'MSigDB database', 'clusterProfiler package', 'enrichplot package', 'ImmuCellAI', 'ssGSEA', 'STRING 4 online platform', 'Cytoscape V3.9.0 software', 'CytoHubba plugin', 'Random Forest algorithm', 'SVM-RFE', 'LASSO model', 'Logistic regression', 'Receiver Operating Characteristic (ROC) curve', 'Area Under the Curve (AUC)', 'miRNet database', 'Drug-Gene Interaction database (DGIdb)', 'ggplot2 package', 'TransZol Up Plus RNA Kit', 'Nanodrop Spectrophotometer', 'TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix', 'ChamQ Universal SYBR qPCR Master Mix', 'QuantStudio™ 5 Real-Time PCR System', 'Image J software']\n",
      "11196393: ['Ficoll gradient', 'ACK lysing buffer', 'RPMI1640 supplemented with 20% FBS', 'BD FACSAria III cell sorter', 'Fc-block (Miltenyi)', 'surface antibodies against CD11b, HLA-DR, CD14, CD177, and CXCR2', 'anti-Arg1 (A1exF5; Thermo Fisher)', 'DCFDA (Abcam)', 'BD FACSCanto II', 'APC antihuman CD11b(ICRF44)', 'Brilliant Violet 605™ antihuman HLA-DR(L243)', 'APC/Cyanine7 antihuman CD14(63D3)', 'PerCP/Cyanine5.5 antihuman CXCR2(5E8/CXCR2)', 'FITC antihuman CD177(MEM-166)', 'Cell Ranger (version 7.0)', 'Seurat (version 4.0.1)', 'monocle2 (version 2.26.0)', 'GraphPad Prism 8.0']\n",
      "11196377: ['Cobb-500 chicks', 'Evonik Co. near-infrared analysis', 'UFFDA software', 'Cobb-Vantress commercial management guide', 'Vitamin premix', 'Mineral premix', 'SAS 9.4 software']\n",
      "11196265: ['Polyethylene microplastics (PE-MPs) powder', 'Purified water (Milli-Q)', 'Tadalafil (Cilais) substance', 'Sildenafil', 'Commercial fish pellet feed (containing 30% protein and 12% lipid; El Nasr Company, Egypt)', 'Automated technical analyzer (BC-2800 from Mindray)', 'Spectrophotometer (in a wavelength range of 340-546 nm; Biodiagonstic Company, Egypt)', 'Kits for TAC (total antioxidant capacity) determination (Sigma-Aldrich, USA)', 'Olympus CH30 microscope', 'IL-6 antibody (E-AB-40021, Elabscience Biotechnology Inc, USA)', 'Streptavidin-biotin-peroxidase pack', \"Mayer's hematoxylin\", 'SPSS 16.0 program for Windows (SPSS 2007, Inc., IL, USA)', 'Canon digital camera (PowershotA95)']\n",
      "11196252: ['AUTOLAB-PGSTAT302N instrument', 'NOVA 1.11.1 software', 'platinum (Pt) wires', 'silver/silver chloride wires (Ag/AgCl)', 'EmStat3 potentiostat', 'PSTrace software', 'automated electrode stand (M164, MTM Anko Instruments)', 'glassy carbon electrode (GCE)', 'silver/silver chloride electrode (Ag/AgCl/3.0 mol L−1 KCl)', 'platinum wire (Pt, 99.99%, The Mint of Poland)', 'DANO (Merck)', 'hydrochloric acid (HCl)', 'Potassium tetrakis(4-chlorophenyl)borate (KTPBCl, Merck)', 'bis(triphenylphosphoranylidene) ammonium chloride (BTPPACl, Merck)', 'bis(triphenylphosphoranylidene)ammonium tetrakis(4-chlorophenyl)borate (BTPPATPBCl)', 'Tetrapropylammonium chloride (TPrACl, Alfa Aesar)', 'Britton-Robinson buffers (BRBs)', '1,2-dichloroethane (1,2-DCE, POCH)', 'citric acid (Sigma-Aldrich)', 'galactose (Chempur)', 'lactose (Fisher Chemical)', 'glucose (Alfa Aesar)', 'calcium chloride (Sigma-Aldrich)', 'potassium chloride (Chempur)', 'magnesium chloride (Fisher Chemical)', 'iron(III) chloride (Alfa Aesar)', 'sodium lactate (Sigma-Aldrich)', 'orthophosphoric acid (V) (Alfa Aesar)', 'combined pH electrode (Polilyte Lab, Hamilton, Switzerland)']\n"
     ]
    }
   ],
   "source": [
    "for key, value in product_dict.items():\n",
    "    print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Extracting Company/Brand Names"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## OpenAI"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "system_prompt = \"\"\"You are an expert Company Identification system. \n",
    "Your task is to take a list of strings as input and identify the company that manufactures each product mentioned.\n",
    "Strings provided in the list can be misleading; use your internal knowledge to identify company names.\n",
    "If the manufacturer is unclear for any product, return a null value for that string.\n",
    "Here is an example of the output format for a list of strings.\n",
    "\n",
    "List of strings: [\"PlayStation 5\", \"iPhone 15 Plus\", \"Pencil\", \"Stainless Steel Tumbler\", \"Whopper\", \"Amazon Speed ETD\"]\n",
    "Answer: [\"Sony\", \"Apple\", None, None, \"Burger King\", \"Bruker\"]\n",
    "\n",
    "Only use this output format. Do not output anything besides this output format.\n",
    "Output the manufacturer names in the order they occur in the input list of strings.\n",
    "\"\"\"\n",
    "\n",
    "user_prompt = \"\"\"Q: Given the list of strings below, identify the companies that manufactures each product.\n",
    "\n",
    "List of strings: {}\n",
    "Answer:\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "def company_identification(list_of_products):\n",
    "    response = openai.chat.completions.create(\n",
    "        model=\"gpt-4-turbo\",\n",
    "        presence_penalty=0,\n",
    "        # top_p=1e-16,\n",
    "        temperature=1e-16,\n",
    "        messages=[\n",
    "            {\"role\": \"system\", \"content\": system_prompt},\n",
    "            {\"role\": \"user\", \"content\": user_prompt.format(list_of_products)}\n",
    "        ]\n",
    "    )\n",
    "    return response.choices[0].message.content"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "company_dict = {}\n",
    "for key, value in product_dict.items():\n",
    "    syntax_error = False\n",
    "    if value:\n",
    "        identified_companies = company_identification(value)\n",
    "        try:\n",
    "            identified_companies = ast.literal_eval(identified_companies)\n",
    "        except SyntaxError:\n",
    "            syntax_error = True\n",
    "            print(f\"Syntax Error with ID {key}\")\n",
    "        if not syntax_error:\n",
    "            company_dict[key] = identified_companies"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198832: ['Shimadzu', 'SCIEX', 'ABI', 'BioTek', 'Beijing Liuyi Instrument Factory', 'Beijing Bijing Biotechnology Co., LTD.', 'Thermo Fisher Scientific', 'Illumina', 'Bio-rad Corporation', 'VIANEX S.A.', 'Hangzhou Yuanda Biological Pharmaceutical', 'Shiyao Group Enbipu Pharmaceutical', 'Shiyao Group Enbipu Pharmaceutical', 'Sichuan Vicchi Biochemical Technology', 'Omega Bio-Tek', 'ABI', 'Servicebio', 'Servicebio']\n",
      "11198784: ['Cytiva', None, 'Bio-Rad', 'STEMCELL Technologies', None, None, None, 'BD Biosciences', None, 'Sutter Instrument', 'Narishige', None, 'Molecular Devices', 'Molecular Devices', 'Molecular Devices', None, None, None]\n",
      "11197919: ['Ingredion', 'Ingredion', 'Ingredion', 'Tanita', None, 'Salimetrics', 'Waters Corporation', 'Qiagen', None, None, 'Qiagen', 'Qiagen', 'Qiagen', None]\n",
      "11197476: ['Bruker', 'Merck', 'Malvern Instruments', 'Thermo Fisher Scientific', 'Beckman Coulter', 'JEOL', 'Nikon', None, None, None, 'BioLegend']\n",
      "11197382: ['Roche']\n",
      "11197204: ['Thermo Fisher Scientific', 'Thermo Fisher Scientific', 'Bio-Rad Laboratories', 'Bio-Rad Laboratories', 'GraphPad Software']\n",
      "11197185: ['Sterilite', 'Tetra', None, None, None, 'Thermo Fisher Scientific', 'Qiagen', 'QuantaBio']\n",
      "11196966: ['Thermo Fisher Scientific', None, 'Kent Scientific', 'Baxter', 'Ethicon', 'FUJIFILM VisualSonics', 'Bruker', 'Miltenyi Biotec', 'Miltenyi Biotec', None, 'Thermo Fisher Scientific', 'QIAGEN', 'New England Biolabs', 'QIAGEN', 'QIAGEN', 'Thermo Fisher Scientific', 'Bio-Rad', 'Bio-Rad', 'Merck Millipore', 'Bio-Rad', 'Vector Laboratories', 'Thermo Fisher Scientific', 'Vector Laboratories']\n",
      "11196847: ['Applied Biosystems', None, 'Megazyme International']\n",
      "11196703: [None, None]\n",
      "11196393: ['BD Biosciences', None, None, 'BD Biosciences', None, None, None, None]\n",
      "11196377: ['Tyson Foods', None, None, 'A&D Company', 'Konica Minolta']\n",
      "11196265: [None]\n",
      "11196252: ['Metrohm', 'PalmSens', None, None, None, None, 'Hamilton']\n"
     ]
    }
   ],
   "source": [
    "for key, value in company_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198832: ['high-performance liquid chromatography (HPLC) (Shimadzu, LC-20AD)', 'SCIEX mass spectrometer (MS) (AB SCIEX, API 4000+)', 'PCR amplification instrument (ABI, 2720)', 'enzyme labeling instrument (BioTek, FLX800T)', 'electrophoresis apparatus (Beijing Liuyi Instrument Factory, DYY-6C)', 'gel imaging system (Beijing Bijing Biotechnology Co., LTD., BG-gdsAUTO130)', 'Nanodrop UV quantitative system (Thermo Fisher Scientific, NC2000)', 'Sequencer (Illumina, Novaseq6000)', 'fluorescence quantitative PCR instrument (Bio-rad Corporation, CFX)', 'vancomycin hydrochloride for injection (VIANEX S.A.)', 'bifidobacterium quadruplex viable tablets (Hangzhou Yuanda Biological Pharmaceutical)', 'NBP capsules (Shiyao Group Enbipu Pharmaceutical)', 'NBP reference (Shiyao Group Enbipu Pharmaceutical)', 'glipizide reference (Sichuan Vicchi Biochemical Technology)', 'Soil DNA Kit (Omega Bio-Tek)', 'Quant-iT PicoGreen dsDNA Assay Kit (ABI)', 'First Strand cDNA Synthesis Kit (Servicebio)', 'SYBR Green qPCR Master Mix (Servicebio)']\n",
      "11198784: ['Ficoll-Paque Premium', 'trypan blue dye', 'TC20 Automated cell counter', 'EasySep Negative Human NK Cell Isolation Kit', 'CD56 APC', 'CD3 PE Cy7', 'stain buffer', 'BD LSR- FortessaTM X-20 flow cytometer', 'borosilicate glass capillaries', 'Sutter Instrumental model P-97', 'Narishige Micro Forge MF-900', 'CV203BU head-stage', 'Axopatch 200B amplifier', 'pClamp 10.7 software', 'Digidata 1440A analogue to digital converter', '0.22 m membrane filter', 'Sapphire Bioscience Reagents', 'Tocris Bioscience']\n",
      "11197919: ['HI-MAIZE® 260 starch', 'VERSAFIBE™ 1490 starch', 'AMIOCA™ TF starch', 'Tanita SC-240 Total Body Composition Analyzer', 'Automated Self-Administered 24-hour (ASA24) Dietary Assessment Tool', 'Salimetrics Salivary Alpha-Amylase Enzymatic Kit', 'Acquity UPLC system', 'DNeasy PowerSoil 96 HTP Kit', 'Classic++™ Hot Start Taq DNA Polymerase Master Mix', 'Mag-Bind TotalPure', 'QIAamp 96 DNA Blood Kit', 'QIAamp Blood Mini Kit', 'QIAamp Investigator Kit', 'iTaq™ Universal SYBR® Green Supermix']\n",
      "11197476: ['LiposoFast', 'Merck Millipore Ltd.', 'Malvern Nano ZS ZEN3600', 'Thermo Fisher Scientific', 'CytoFLEX', 'JEM-1400', 'Nikon', 'CCK-8 assay kit', 'FITC-conjugated anti-COX-2 antibody', 'in vivo images system', 'Biolegend Inc.']\n",
      "11197382: ['COBAS Integra 400']\n",
      "11197204: ['TRIzol', 'TaqMan™ Gene Expression Assay', 'iTaq Universal One-Step', 'iTaq™ Universal SYBR® Green One-Step Kit', 'GraphPad PRISM']\n",
      "11197185: ['Sterilite container', 'Tetra Pond Koi growth feed', 'Hemotek membrane feeding system', 'Triethylamine', 'Mosquito Diluent', 'Trizol', 'REPLI-g whole transcriptome amplification single-cell kit', 'AccuStart II PCR supermix']\n",
      "11196966: ['Advanced DMEM-F12', 'N-acetylcysteine', 'SomnoSuite® Low-Flow Anesthesia System', 'Seprafilm', '6–0 Vicryl', 'VEVO 3100', 'BioSpec 94/21', 'Liver Dissociation Kit', 'Tumor Dissociation Kit', 'ACK lysis buffer', 'eBioscience™ Foxp3/Transcription Factor Staining Buffer Set', 'RNeasy Mini Kit', 'ProtoScript II First Strand cDNA Synthesis Kit', 'DNeasy Blood & Tissue Kit', 'QuantiFast SYBR® Green PCR Kit', 'Pierce BCA-Kit', 'Mini-Protean TGX precast gel', 'trans-blot turbo transfer system', 'Immobilon Crescendo Western HRP substrate', 'ChemiDoc BioRad imager', 'Vector® TrueVIEW® Autofluorescence Quenching Kit', 'NucBlue™ Fixed Cell Stain ReadyProbes™', 'VECTASHIELD Vibrance Antifade Mounting Medium']\n",
      "11196847: ['ABI 3130 Genetic Analyzer', 'Whatman paper No. 4', 'Megazyme kit']\n",
      "11196703: ['ENSO 16', 'glucose powder']\n",
      "11196393: ['BD FACSAria III cell sorter', 'Ficoll gradient', 'ACK lysing buffer', 'BD FACSCanto™ II', 'Cell Ranger', 'Seurat', 'monocle2', 'GraphPad Prism']\n",
      "11196377: ['Cobb-500', 'Evonik Nutrition & Care GmbH', 'UFFDA software', 'Model GF 400 A&D Weighing', 'CR-410 Konica Minolta']\n",
      "11196265: ['commercial fish pellet feed']\n",
      "11196252: ['AUTOLAB-PGSTAT302N instrument', 'EmStat3 potentiostat', 'automated electrode stand (M164)', 'glassy carbon electrode (GCE)', 'silver/silver chloride electrode (Ag/AgCl/3.0 mol L−1 KCl)', 'platinum wire (Pt, 99.99%)', 'combined pH electrode (Polilyte Lab)']\n"
     ]
    }
   ],
   "source": [
    "for key, value in product_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Mistral"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "system_prompt = \"\"\"You are an expert Company Identification system. \n",
    "Your task is to take a list of strings as input and identify the company that manufactures each product mentioned.\n",
    "Strings provided in the list can be misleading; use your internal knowledge to identify company names.\n",
    "If the manufacturer is unclear for any product, return a null value for that string.\n",
    "Here is an example of the output format for a list of strings.\n",
    "\n",
    "List of strings: [\"PlayStation 5\", \"iPhone 15 Plus\", \"Pencil\", \"Stainless Steel Tumbler\", \"Whopper\", \"Amazon Speed ETD\"]\n",
    "Answer: [\"Sony\", \"Apple\", None, None, \"Burger King\", \"Bruker\"]\n",
    "\n",
    "Only use this output format. Do not output anything besides this output format.\n",
    "Output the manufacturer names in the order they occur in the input list of strings.\n",
    "\"\"\"\n",
    "\n",
    "user_prompt = \"\"\"Q: Given the list of strings below, identify the companies that manufactures each product.\n",
    "\n",
    "List of strings: {}\n",
    "Answer:\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "def company_identification(input_text):\n",
    "    client = MistralClient(api_key=api_key)\n",
    "    response = client.chat(\n",
    "        model=\"mistral-large-latest\",\n",
    "        temperature=1e-16,\n",
    "        messages=[\n",
    "        ChatMessage(role=\"system\", content=system_prompt),\n",
    "        ChatMessage(role=\"user\", content=user_prompt.format(input_text))\n",
    "    ]\n",
    "    )\n",
    "    return response.choices[0].message.content"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "company_dict = {}\n",
    "syntax_error = False\n",
    "for key, value in product_dict.items():\n",
    "    if value:\n",
    "        identified_companies = company_identification(value)\n",
    "        try:\n",
    "            identified_companies = ast.literal_eval(identified_companies)\n",
    "        except SyntaxError:\n",
    "            syntax_error = True\n",
    "            print(f\"Syntax Error with ID {key}\")\n",
    "        if not syntax_error:\n",
    "            company_dict[key] = identified_companies"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11197919: ['Ingredion', 'Ingredion', 'Ingredion', 'Tanita', None, 'QIAGEN', 'QIAGEN', 'QIAGEN', 'Bio-Rad Laboratories', 'Salimetrics', 'QIAGEN', 'Bio-Rad Laboratories', None, None, None, None, None, None]\n",
      "11197476: ['None', 'None', 'None', 'Avestin', 'Merck Millipore Ltd.', 'None', 'None', 'None', 'JEOL Ltd.', 'Malvern Instruments Ltd.', 'Thermo Fisher Scientific', 'None', 'Beckman Coulter Inc.', 'None', 'None', 'Beyotime Biotechnology Co. Ltd.', 'None', 'Nikon Corporation', 'Beyotime Biotechnology Co. Ltd.', 'None', 'Affinity Biosciences', 'None', 'PerkinElmer Inc.', 'None', 'Biolegend Inc.', 'None', 'None', 'None', 'None', 'None', 'None', 'None', 'None', 'None', 'None', 'None', 'None', 'None', 'None', 'None', 'None', 'None', 'None', 'None', 'None']\n",
      "11197204: ['Thermo Fisher Scientific', 'Thermo Fisher Scientific', None, 'Bio-Rad Laboratories', 'Bio-Rad Laboratories', None, 'GraphPad Software']\n",
      "11197185: ['Tetra', 'Hemotek', 'Invitrogen', 'QIAGEN', 'Illumina', 'QuantaBio', None, 'MicrobiomeAnalyst', 'GreenGenes', None, 'NCBI', None, None, None, None, None, 'Random Forest', 'MicrobiomeAnalyst']\n",
      "11196966: ['Thermo Fisher Scientific', None, 'Thermo Fisher Scientific', 'Thermo Fisher Scientific', 'Thermo Fisher Scientific', 'Thermo Fisher Scientific', 'Sigma-Aldrich', 'Thermo Fisher Scientific', 'Thermo Fisher Scientific', 'Trevigen', 'Corning', 'Reckitt Benckiser', 'Piramal', 'Baxter', 'Ethicon', 'Qiagen', 'Miltenyi Biotec', 'Miltenyi Biotec', 'Thermo Fisher Scientific', 'BD Biosciences', 'eBioscience', 'Thermo Fisher Scientific', 'BD Biosciences', 'BD Biosciences', 'Qiagen', 'New England Biolabs', 'Qiagen', 'Qiagen', 'Roche', 'Qiagen', 'Roche', 'Illumina', 'Thermo Fisher Scientific', 'Thermo Fisher Scientific', 'Bio-Rad', 'Azure Biosystems', 'Bio-Rad', 'Bio-Rad', 'Trans-Blot Turbo Transfer System', 'MilliporeSigma', 'Bio-Rad', 'Bio-Rad', 'Zeiss', 'Vector Laboratories', 'Thermo Fisher Scientific', 'Vector Laboratories']\n",
      "11196847: [None, None, None, 'Applied Biosystems', None, None, None, 'Thermo Fisher Scientific', None, None, 'Sigma-Aldrich', None, 'Sigma-Aldrich', None, None, None, 'Sigma-Aldrich', None, 'Sigma-Aldrich', None, 'Sigma-Aldrich', None, 'Sigma-Aldrich', None, None, 'Sigma-Aldrich', 'Sigma-Aldrich', None, None, 'Sigma-Aldrich', None, None, None, 'Sigma-Aldrich', None, None, None, None, None, None, None, 'Agilent Technologies']\n",
      "11196703: ['Bruker', None, 'Greiner Bio-One', 'Greiner Bio-One']\n",
      "11196393: ['GE Healthcare', None, None, None, 'Becton, Dickinson and Company', None, None, None, None, None, None, None, 'Thermo Fisher Scientific', 'BioLegend', 'BioLegend', 'BioLegend', 'BioLegend', 'BioLegend', '10x Genomics', 'Satija Lab', 'GraphPad Software']\n",
      "11196265: ['None', 'None', 'Eli Lilly and Company', 'El Nasr Company', 'None', 'Mindray', 'Biodiagonstic Company', 'Sigma-Aldrich', 'None', 'None', 'Olympus Corporation', 'None', 'Vector Research facilities', 'Elabscience Biotechnology Inc', 'None', 'None', 'Olympus Corporation', 'Canon', 'None', 'SPSS Inc.']\n",
      "11196252: ['Metrohm Autolab', 'Metrohm Autolab', None, None, 'PalmSens', 'PalmSens', 'MTM Anko Instruments', None, None, 'The Mint of Poland', None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, 'Hamilton Company']\n"
     ]
    }
   ],
   "source": [
    "for key, value in company_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Mistral with Article 11198832"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198832 [\"high-performance liquid chromatography (HPLC)\" (Shimadzu, LC-20AD), \"SCIEX mass spectrometer (MS)\" (AB SCIEX, API 4000+), \"PCR amplification instrument\" (ABI, 2720), \"enzyme labeling instrument\" (BioTek, FLX800T), \"electrophoresis apparatus\" (Beijing Liuyi Instrument Factory, DYY-6C), \"gel imaging system\" (Beijing Bijing Biotechnology Co., LTD., BG-gdsAUTO130), \"Nanodrop UV quantitative system\" (Thermo Fisher Scientific, NC2000), \"Sequencer\" (Illumina, Novaseq6000), \"fluorescence quantitative PCR instrument\" (Bio-rad Corporation, CFX), \"vancomycin hydrochloride for injection\" (VIANEX S.A.), \"bifidobacterium quadruplex viable tablets\" (Hangzhou Yuanda Biological Pharmaceutical), \"NBP capsules\" (Shiyao Group Enbipu Pharmaceutical), \"NBP reference\" (Shiyao Group Enbipu Pharmaceutical.), \"glipizide reference\" (Sichuan Vicchi Biochemical Technology), \"Soil DNA Kit\" (Omega Bio-Tek), \"Quant-iT PicoGreen dsDNA Assay Kit\" (ABI), \"First Strand cDNA Synthesis Kit\" (Servicebio), \"SYBR Green qPCR Master Mix\" (Servicebio)]\n"
     ]
    }
   ],
   "source": [
    "print('11198832', '[\"high-performance liquid chromatography (HPLC)\" (Shimadzu, LC-20AD), \"SCIEX mass spectrometer (MS)\" (AB SCIEX, API 4000+), \"PCR amplification instrument\" (ABI, 2720), \"enzyme labeling instrument\" (BioTek, FLX800T), \"electrophoresis apparatus\" (Beijing Liuyi Instrument Factory, DYY-6C), \"gel imaging system\" (Beijing Bijing Biotechnology Co., LTD., BG-gdsAUTO130), \"Nanodrop UV quantitative system\" (Thermo Fisher Scientific, NC2000), \"Sequencer\" (Illumina, Novaseq6000), \"fluorescence quantitative PCR instrument\" (Bio-rad Corporation, CFX), \"vancomycin hydrochloride for injection\" (VIANEX S.A.), \"bifidobacterium quadruplex viable tablets\" (Hangzhou Yuanda Biological Pharmaceutical), \"NBP capsules\" (Shiyao Group Enbipu Pharmaceutical), \"NBP reference\" (Shiyao Group Enbipu Pharmaceutical.), \"glipizide reference\" (Sichuan Vicchi Biochemical Technology), \"Soil DNA Kit\" (Omega Bio-Tek), \"Quant-iT PicoGreen dsDNA Assay Kit\" (ABI), \"First Strand cDNA Synthesis Kit\" (Servicebio), \"SYBR Green qPCR Master Mix\" (Servicebio)]')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11197919: ['HI-MAIZE® 260 starch', 'VERSAFIBE™ 1490 starch', 'AMIOCA™ TF starch', 'Tanita SC-240 Total Body Composition Analyzer', 'Automated Self-Administered 24-hour (ASA24) Dietary Assessment Tool 2020', 'QIAamp 96 DNA Blood Kit', 'QIAamp Blood Mini Kit', 'QIAamp Investigator Kit', 'iTaq™ Universal SYBR® Green Supermix', 'Salimetrics Salivary Alpha-Amylase Enzymatic Kit', 'DNeasy PowerSoil 96 HTP Kit', 'Classic++™ Hot Start Taq DNA Polymerase Master Mix', 'Mag-Bind TotalPure', 'QIIME 2', 'DADA2', 'mafft', 'fasttree']\n",
      "11197476: ['CXB@Lipo', 'chloroform', 'rotary evaporator', 'liposome extrude (LiposoFast, Avestin, Canada)', 'polycarbonate membrane filters (Merck Millipore Ltd., USA)', 'ultrafiltration centrifuge tube', 'ECN', 'glycol chitosan', 'transmission electron microscopy (TEM; JEM-1400, Japan)', 'Malvern Nano ZS ZEN3600 (Malvern, UK)', 'UV-VIS (Thermo Fisher Scientific, USA)', 'ICG@ECN (I@ECN)', 'flow cytometry (CytoFLEX, Beckman, USA)', 'UV‒VIS', 'pGEX-Pvhb-Lysis', 'cloning kit (Beyotime Biotechnology Co. Ltd., Shanghai, China)', 'ICG@Liposome (I@L)', 'confocal laser scanning microscopy (CLSM) (Nikon, Japan)', 'CCK-8 assay kit (Beyotime Biotechnology Co. Ltd., Shanghai, China)', 'Western blot (WB) analysis', 'FITC-conjugated anti-COX-2 antibody (Affinity, Jiangsu, China)', 'in vivo images system (IVIS)', 'optimal cutting temperature (OCT) compound', 'CT26 tumor model', 'FITC-conjugated anti-mouse CD3', 'FITC-conjugated anti-mouse CD4', 'PE-conjugated anti-mouse CD8 (Biolegend Inc., USA)', 'TUNEL and H&E staining', 'Ki67', 'C@ECN-PL', 'assay kits (Jiancheng Bioengineering Institute, Nanjing, China)', 'H&E staining analysis', 'ImageJ 6.0']\n",
      "11197204: ['TRIzol', 'TaqMan™ Gene Expression Assay', 'Easy Oligos', 'iTaq Universal One-Step', 'iTaq™ Universal SYBR® Green One-Step Kit', 'Tools', 'GraphPad PRISM']\n",
      "11197185: ['Tetra Pond Koi growth feed', 'Hemotek membrane feeding system', 'Trizol', 'REPLI-g whole transcriptome amplification (WTA) single-cell kit', 'Illumina barcoded 16S primer set', 'AccuStart II PCR supermix', 'QIITA', 'MicrobiomeAnalyst', 'GreenGenes 16S reference database', 'BIOM-formatted OTU table', 'NCBI GenBank Short Read Archives', 'Chao1 diversity measure', 'Shannon diversity index', 'Simpson Index', 'Bray–Curtis dissimilarity statistic', 'PERMANOVA test', 'Random forest algorithm', 'Microbiome Analyst']\n",
      "11196966: ['Advanced DMEM-F12', 'penicillin/streptomycin', 'HEPES', 'Glutamax', 'N2', 'B27', 'N-acetylcysteine', 'hygromycin', 'puromycin', 'Cultrex RGF basement membrane extract (BME) Type 2', 'collagen I/5x neutralization buffer', 'buprenorphine', 'Isoflurane', 'Seprafilm', 'Vicryl suture', 'RNAprotect buffer', 'Liver Dissociation Kit, mouse', 'Tumor Dissociation Kit, mouse', 'ACK lysis buffer', 'FACS buffer', 'TruStain FcX PLUS (anti-mouse CD16/32) antibody', 'eBioscience™ Foxp3/Transcription Factor Staining Buffer Set', 'RPMI-Medium', 'Cell Activation Cocktail', 'Brefeldin A', 'RLT buffer', 'RNeasy Mini Kit', 'ProtoScript II First Strand cDNA Synthesis Kit', 'QuantiTect SYBR Green RT PCR mastermix', 'DNeasy Blood & Tissue Kit', 'QuantiFast SYBR® Green PCR Kit', 'Roche LightCycler 480', 'QuantiFast SYBR® Green PCR Kit', 'Bacterial 16S qPCR', 'QuantiFast SYBR® Green PCR Kit', 'Illumina NovaSeq instrument', 'Pierce BCA-Kit', 'Azure Ao Microplate Reader', '4x Laemmli Buffer', 'Mini-Protean TGX precast gel', 'color-coded prestained broad range protein ladder', 'Trans-Blot Turbo Transfer System', 'Immobilon Crescendo Western HRP substrate', 'ChemiDoc BioRad imager', 'PFA', 'TE-Buffer', 'avidin-biotin-block', 'DAB', 'ROTI® Histokitt', 'Zeiss Axio Scan Z.1 Microscope Slide Scanner', 'Vector® TrueVIEW® Autofluorescence Quenching Kit', 'NucBlue™ Fixed Cell Stain ReadyProbes™', 'VECTASHIELD Vibrance Antifade Mounting Medium']\n",
      "11196847: ['potato dextrose agar (PDA) medium', 'potato dextrose broth (PDB)', 'pestle and mortar', 'ABI 3130 Genetic Analyzer', 'Taq DNA polymerase Assay Buffer', 'dNTP', 'MgCl2', 'Taq DNA Polymerase', 'Whatman paper No. 4', 'vacuum evaporator', 'Folin-Ciocalteu reagent', 'Bovine Serum Albumin (BSA)', 'anthrone reagent', 'NaNO2', 'AlCl3', 'NaOH', 'Folin-Ciocalteu phenol reagent', 'Na2CO3', 'vanillin', 'sulfuric acid', 'Aescin', 'ferric chloride', 'Mg(CH3COO)2', 'Rhein', 'NH4OH', 'petroleum ether', 'Folin-Denis reagent', 'Liebermann-Burchard reagent', 'cholesterol', 'DPPH', '4-aminophenazone', 'Tris', 'phenol', 'glucose oxidase and peroxidase', 'amyloglucosidase', 'invertase', 'high-performance liquid chromatography (PDA)']\n",
      "11196703: ['ENSO 16', 'glucose powder', 'Vacuette sodium fluoride K3EDTA tubes', 'Vacuette serum tubes with clot activator']\n",
      "11196393: ['Ficoll gradient', 'ACK lysing buffer', 'RPMI1640', 'FBS', 'BD FACSAria III cell sorter', 'Fc-block', 'CD11b antibodies', 'HLA-DR antibodies', 'CD14 antibodies', 'CD177 antibodies', 'CXCR2 antibodies', 'Arg1 antibodies', 'DCFDA', 'APC antihuman CD11b(ICRF44)', 'Brilliant Violet 605™ antihuman HLA-DR(L243)', 'APC/Cyanine7 antihuman CD14(63D3)', 'PerCP/Cyanine5.5 antihuman CXCR2(5E8/CXCR2)', 'FITC antihuman CD177(MEM-166)', 'Cell Ranger', 'Seurat', 'GraphPad Prism 8.0']\n",
      "11196265: ['Polyethylene microplastics (PE-MPs)', 'Purified water (Milli-Q)', 'tadalafil (Cilais)', 'commercial fish pellet feed (containing 30% protein and 12% lipid; El Nasr Company, Egypt)', 'crushed ice', 'automated technical analyzer (BC-2800 from Mindray)', 'spectrophotometer (Biodiagonstic Company, Egypt)', 'kits (Sigma-Aldrich, USA)', 'neutral buffered formalin', 'hematoxylin and eosin (H & E)', 'Olympus CH30 microscope', 'xylene', 'standard goat serum (Vector Research facilities, Burlingame, CA)', 'essential polyclonal IL-6 antibody; 1/50 (E-AB-40021, Elabscience Biotechnology Inc, USA)', 'streptavidin–biotin-peroxidase pack', \"Mayer's hematoxylin\", 'Olympus BX41 microscope', 'canon digital camera (PowershotA95)', 'SPSS 16.0 program for Windows (SPSS 2007, Inc., IL, USA)']\n",
      "11196252: ['AUTOLAB-PGSTAT302N instrument', 'NOVA 1.11.1 software', 'platinum (Pt) wires', 'silver/silver chloride wires (Ag/AgCl)', 'EmStat3 potentiostat', 'PSTrace software', 'automated electrode stand (M164, MTM Anko Instruments)', 'glassy carbon electrode (GCE)', 'silver/silver chloride electrode (Ag/AgCl/3.0 mol L−1 KCl)', 'platinum wire (Pt, 99.99%, The Mint of Poland)', 'DANO', 'hydrochloric acid (HCl)', 'Potassium tetrakis(4-chlorophenyl)borate (KTPBCl)', 'bis(triphenylphosphoranylidene) ammonium chloride (BTPPACl)', 'bis(triphenylphosphoranylidene)ammonium tetrakis(4-chlorophenyl)borate (BTPPATPBCl)', 'tetrapropylammonium chloride (TPrACl)', 'Britton–Robinson buffers (BRBs)', '1,2–dichloroethane (1,2–DCE)', 'citric acid', 'galactose', 'lactose', 'glucose', 'calcium chloride', 'potassium chloride', 'magnesium chloride', 'iron(III) chloride', 'sodium lactate', 'orthophosphoric acid (V)', 'combined pH electrode (Polilyte Lab, Hamilton, Switzerland)', 'ultra high temperature (UHT) milk (1.5% fat content)']\n"
     ]
    }
   ],
   "source": [
    "for key, value in product_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Spacy"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198832: ['Thermo Fisher Scientific', 'Illumina', 'Thermo Fisher Scientific', 'LI-COR Biosciences']\n",
      "11198784: ['Cytiva', 'Bio-Rad', 'STEMCELL Technologies', 'Narishige', 'Molecular Devices']\n",
      "11197919: ['Tanita', 'Qiagen', 'Qiagen', 'Bio-Rad', 'Qiagen', 'BioSpec Products']\n",
      "11197476: ['Malvern Panalytical']\n",
      "11197204: ['Thermo Fisher Scientific', None, 'Bio-Rad', 'Bio-Rad']\n",
      "11197185: ['Illumina', None, 'Thermo Fisher Scientific', None, 'Illumina']\n",
      "11196966: ['Piramal', 'Kent Scientific', None, None, 'Qiagen', 'New England Biolabs', 'Qiagen', 'Qiagen', 'Thermo Fisher Scientific', 'Azure Biosystems', 'Zeiss', 'Vector Laboratories', 'Thermo Fisher Scientific', 'Vector Laboratories']\n",
      "11196847: ['Thermo Fisher Scientific', None]\n",
      "11196703: [None, None]\n",
      "11196417: ['TransGen Biotech', 'Thermo Fisher Scientific', 'TransGen Biotech', 'Vazyme', 'Thermo Fisher Scientific']\n",
      "11196393: ['Becton Dickinson', None, 'GraphPad Software']\n",
      "11196377: ['Konica Minolta']\n",
      "11196265: ['Canon']\n",
      "11196252: ['Metrohm', 'PalmSens', None]\n"
     ]
    }
   ],
   "source": [
    "for key, value in company_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198832: ['Nanodrop UV', 'Novaseq6000', 'Quant-iT PicoGreen dsDNA Assay Kit', 'The Odyssey CLx Imaging System']\n",
      "11198784: ['Ficoll-Paque Premium', 'TC20 Automated cell counter', 'EasySep Negative Human NK Cell Isolation Kit', 'Micro Forge MF-900', 'Axopatch 200B']\n",
      "11197919: ['Tanita SC-240 Total Body Composition Analyzer', 'QIAamp 96 DNA Blood Kit', 'QIAamp Investigator Kit', 'QX100 Droplet Digital PCR System', 'DNeasy PowerSoil 96 HTP Kit', 'BioSpec 1001 Mini-Beadbeater-96']\n",
      "11197476: ['Malvern Nano ZS ZEN3600']\n",
      "11197204: ['TRIzol', 'Easy Oligos', 'iTaq Universal One-Step', 'iTaq™ Universal SYBR® Green One-Step']\n",
      "11197185: ['MiSeq', 'Hemotek', 'Trizol', 'AccuStart II', 'MiSeq 500']\n",
      "11196966: ['Isoflurane Piramal', 'SomnoSuite® Low-Flow Anesthesia System', 'Liver Dissociation Kit', 'Tumor Dissociation Kit', 'RNeasy Mini Kit', 'ProtoScript II First Strand cDNA Synthesis Kit', 'DNeasy Blood & Tissue Kit', 'QuantiFast SYBR® Green PCR Kit', 'Pierce BCA-Kit', 'Azure Ao Microplate Reader', 'Zeiss Axio Scan Z.1 Microscope Slide Scanner', 'Vector® TrueVIEW® Autofluorescence Quenching Kit', 'NucBlue™ Fixed Cell Stain ReadyProbes™', 'VECTASHIELD Vibrance Antifade Mounting Medium']\n",
      "11196847: ['ABI 3130 Genetic Analyzer', 'RP C-18 column']\n",
      "11196703: ['ENSO 16', 'NEOH']\n",
      "11196417: ['TransZol Up Plus RNA Kit', 'Nanodrop Spectrophotometer', 'TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix', 'ChamQ Universal SYBR qPCR Master Mix', 'QuantStudio™ 5']\n",
      "11196393: ['BD FACSCanto™ II', 'Brilliant Violet 605™', 'GraphPad Prism 8.0']\n",
      "11196377: ['Chroma‐Meter (CR‐410 Konica Minolta, INC']\n",
      "11196265: ['PowershotA95']\n",
      "11196252: ['AUTOLAB-PGSTAT302N', 'EmStat3', 'M164']\n"
     ]
    }
   ],
   "source": [
    "for key, value in cleaned_entity_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
