{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Extract Method Section"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from PubMed.PmcScraper import PmcScraper\n",
    "import os\n",
    "import ast\n",
    "import spacy\n",
    "import openai\n",
    "\n",
    "nlp = spacy.load(\"en_core_web_trf\")\n",
    "\n",
    "# need .env file to store ncbi api key\n",
    "pmc_scraper = PmcScraper(os.environ['NCBI_API_KEY'])\n",
    "\n",
    "# search based on keyword and retrieve the desired amount of articles\n",
    "search_results = pmc_scraper.search_pmc(search_term=\"gut microbiota\", number_of_articles=30)\n",
    "\n",
    "# extract the full text version of articles using pmcids\n",
    "full_text_articles = pmc_scraper.fetch_full_text(search_results['esearchresult']['idlist'])\n",
    "\n",
    "# extract the desired section of articles\n",
    "method_sections = pmc_scraper.extract_section(full_text_articles, 'method')\n",
    "\n",
    "# print as dictionary\n",
    "method_dict = dict(zip(search_results['esearchresult']['idlist'], method_sections))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'11198919': 'Methods:This was a cross-sectional multicenter study enrolling children aged 0–48 months, attending pediatric clinics. Questionnaires evaluated the clinical history, symptoms, and sociodemographic information. FGIDs were defined according to Rome IV criteria. PATIENTS AND METHODS Study design We performed a prospective, cross-sectional, multicenter study to assess the hospital-based prevalence of FGIDs in five regions of Saudi Arabia (Center, North, East, West, and South) including infants and toddlers, aged 0–4 years. Recruitment was conducted in general pediatric clinics located in six cities: Jeddah (King Abdulaziz Medical City and Soliman Fakeeh Hospital), Riyadh (King Abdulaziz Medical City and King Faisal Hospital and Research Center, and King Saud University Medical City), Tabouk (Tabuk Armed Forces Hospital), Al-Madinah Al-Munawarah (Maternal Children Hospital), Khamis Mushait (Maternal Children Hospital), and Dammam (Maternal Children Hospital). We recruited the patients who visited the well baby clinic, vaccination clinic, and primary care general pediatric clinics without organic diseases. These hospitals are government hospitals except Soliman Fakeeh Hospital where local infants and toddlers visit for routine medical check and/or vaccination. Participants Saudi Arabia’s 2020 population is estimated at 34,813,871 people, and children aged 0 to 14 years represent 24.5% of the population (https://www.stats.gov.sa/en). Data were collected from July 2022 to February 2023. Infants and toddlers were recruited if consent was provided and in the absence of any organic disease with these criteria: age: 0–4 years and fulfilled symptom-based Rome IV criteria for FGIDs. Exclusion criteria were age >4 years and any infants and children who have pathological disorders (neurological, metabolic, cardiac, immunology, and surgery). Those children without FIGD who did not fulfill symptom-based Rome IV criteria for FGIDs were considered to be normal children (controls). Sample size The sample size was calculated using an estimated 10% prevalence for infant colic and childhood constipation.[13] Assuming a confidence level of 95%, power of 80%, and a precision found to be the nearest 2%, a sample size of 1200 young children aged 1 month to 4 years was considered adequate. Survey design We designed a brief, user-friendly questionnaire that assesses FGIDs in Saudi infants and toddlers. This questionnaire was pilot tested by three pediatric gastroenterologists. The questionnaire was then revised based on reproducibility, validity, and question value. An initial pool of items was evaluated for content and understandability by three pediatric gastroenterologists and three pediatric practitioners (Cronbach’s alpha = 0.8). Changes and modifications were based on the pilot results. The survey was administered via two mechanisms: direct communication or through phone call. Most of the participants were interviewed face to face using an English-language questionnaire (translated to Arabic during interviews with parents), whereas few participants completed the questionnaire by phone. The participants’ identities were not part of the research record, and no information in the questionnaire was used to ascertain identity. The estimated time to complete the survey was approximately 25 minutes. Questionnaire The questionnaire had two sections: Section 1 contains questions on demographic characteristics, including age, gender, birth weight, growth pattern, and feeding practices. Section 2 contains explicit questions on FGID symptoms, including stool pattern, spitting, and crying habits, for infants and young children based on Rome IV criteria to diagnose FGIDs as follows: functional vomiting or regurgitation, infant colic, aerophagia, infant dyschezia, functional constipation, and functional diarrhea. The definition of each functional GI disease is based on its criteria.[14] All infants and children who fulfilled one to three symptom-based criteria for FGIDs were included in the study. Those children without FIGD were considered normal children and were included as controls. Statistical analysis The associations of sociodemographic characteristics and variables of potential risk factors with FGIDs of children were examined through univariate analysis. Backward stepwise procedures were used to build the multivariate analysis; the final model included only those variables that were found to be statistically significant in the univariate analysis. The associations were expressed as odds ratios (OR) with their confidence intervals (95% CIs). The data were analyzed with the Statistical Package for the Social Sciences (SPSS) software (SPSS for Windows, version 13). Demographic data were compared using Fisher’s exact test or the Wilcoxon rank-sum test when the epidemiology of FGIDs was examined through logistic regression analysis. A 2-tailed test indicated statistical significance at P < .05. Ethics approval This study was approved by the Ethics and Research Committee of the National Guard Health Affairs, King Abdullah International Medical Research Center (NRJ21J/147/05. 16/8/2021). The parents or legal guardians signed a consent for their children to participate in the study. All information collected was kept strictly confidential.',\n",
       " '11198832': 'MethodsAfter treatment with antibiotics and probiotics, plasma NBP concentrations in SD rats were determined by high-performance liquid chromatography-tandem mass spectrometry (HPLC-MS/MS). The effect of intestinal microbiota changes on NBP pharmacokinetics was compared. Intestinal microbiota changes after NBP treatment were analyzed by 16S rRNA sequencing. Expressions of CYP3A1 mRNA and protein in the liver and small intestine tissues under different intestinal flora conditions were determined by qRT-PCR and Western Blot. KEGG analysis was used to analyze the effect of intestinal microbiota changes on metabolic pathways. Materials and methods Instruments The following instruments were used: high-performance liquid chromatography (HPLC) (Shimadzu, LC-20AD), SCIEX mass spectrometer (MS) (AB SCIEX, API 4000+), PCR amplification instrument (ABI, 2720), enzyme labeling instrument (BioTek, FLX800T, electrophoresis apparatus (Beijing Liuyi Instrument Factory, DYY-6C), a gel imaging system (Beijing Bijing Biotechnology Co., LTD., BG-gdsAUTO130), Nanodrop UV quantitative system (Thermo Fisher Scientific, NC2000), Sequencer (Illumina, Novaseq6000), and fluorescence quantitative PCR instrument (Bio-rad Corporation, CFX). Drugs and reagents The following were used: vancomycin hydrochloride for injection (VIANEX S.A.), bifidobacterium quadruplex viable tablets (Hangzhou Yuanda Biological Pharmaceutical), NBP capsules (Shiyao Group Enbipu Pharmaceutical), NBP reference (Shiyao Group Enbipu Pharmaceutical.), glipizide reference (Sichuan Vicchi Biochemical Technology), Soil DNA Kit (Omega Bio-Tek), Quant-iT PicoGreen dsDNA Assay Kit (ABI), First Strand cDNA Synthesis Kit (Servicebio), and SYBR Green qPCR Master Mix (Servicebio). Vancomycin is not absorbed during oral administration and can maintain a large concentration in the intestine, the oral administration of vancomycin was selected in the study of SD rats [18]. The probiotics selected in this study are bifidobacteria tetrad, including Bifidobacteria infantilus, Lactobacillus acidophilus, Enterococcus faecalis and Bacillus cereus. The first three are normal bacteria in healthy gut, and Bacillus cereus can consume oxygen molecules in intestine to create an anaerobic environment and enhance therapeutic effect. The selection of such probiotics had little effect on microflora structure [19]. Animals Male Sprague Dawley (SD) rats (7 weeks old, 200 ± 20 g) were purchased from Beijing Huafukang Bioscience Co. Ltd. The rats were kept in a specific pathogen-free (SPF) facility. Animal experiments were approved by the Research Ethics Committee of the Second Hospital of Hebei Medical University. The experimental animals were euthanized through bloodletting following urethane anesthesia. Anesthesia was induced by intraperitoneal injection of 20% urethane (1mg/kg), administered 20 minutes prior to blood collection and dissection, ensuring deep anesthesia and minimizing animal discomfort during the experiment. Quantification of NBP in plasma by HPLC-MS/MS Chromatography separation was performed on a Symmetry C18 column (4.6 × 150 mm, 3.5 μm), and the column temperature was 40°C. The mobile phase consisted of water (A) and acetonitrile (B, containing 0.1% formic acid) with a flow rate of 0.8 mL·min-1. The elution procedure was as follows: 0–2.0 min, 75%-95% B; 2.0–6.5 min, 95% B; 6.5–7.0 min, 95%- 75% B; and 7.0–8.0 min, 75% B. The injection volume was 5 μL, and the internal standard (IS) was glipizide. Mass spectrum conditions Both NBP and glipizide were monitored in positive ESI mode. The scanning mode was multireaction monitoring (MRM), with the ion transitions of m/z 191.1→45.1 and 446.2→321.0, respectively. The declustering potential (DP) and collision energy (CE) of NBP were 60V and 22V, and those of the glipizide were 100V and 23V, respectively. Plasma sample processing method Acetonitrile (150 μL) containing glipizide (250 ng·mL-1) was added to plasma (50 μL) and centrifuged at 10,900 g for 5 min. The supernatant was transferred to the autosampler vial for HPLC-MS/MS detection. Standard curves A series of NBP solutions with a concentration of 20–2000 ng·mL-1 were added to the blank plasma to prepare the simulated plasma samples, which were tested according to the plasma sample processing method. The standard curve was drawn with the concentration of NBP as the abscissa (X) and the peak area ratio of NBP to the glipizide as the ordinate (Y). The standard curve equation was obtained through regression. Experimental design and sample collection Before the experiment, 30 SD male rats were randomly divided into three groups (n = 10): the antibiotic, probiotic, and control groups. The antibiotic group received 50 mg·kg-1 of vancomycin solution. The probiotic group received 600 mg·kg-1 of live bifidobacterium tetrad bacteria suspension. The control group received physiological saline. All groups received daily gavage for 7 days. Fecal samples were collected on day 0 and day 8 before NBP administration. On day 8, all rats were gavaged using 70 mg·kg-1 of NBP solution. Blood samples (about 300 μL) were collected in heparinized centrifuge tubes at 0.08, 0.16, 0.33, 0.5, 0.75, 1, 1.5, 2, 3, 4, 6, and 8 h after gavage. Plasma samples were centrifuged at 8000 RPM for 10 min at 4°C. The rats were sacrificed after collecting blood and samples from the liver and small intestine tissues. The samples were stored at -80°C. Effects of the intestinal microbiota on the pharmacokinetics of NBP The frozen plasma samples were thawed naturally, processed according to the “plasma sample processing method,” and injected for analysis. NBP concentrations at different time points were calculated according to the standard curve equation. The pharmacokinetic parameters were calculated using the DAS 2.0 software. Statistical analysis was performed with SPSS 20 software to investigate the effect of antibiotic or probiotic interventions on NBP pharmacokinetics. T-test was used for data consistent with normal distribution, and Wilcoxon rank sum test was used for data inconsistent with normal distribution. CYP3A1 mRNA and protein expressions in the liver and small intestinal Quantitative real-time PCR (qRT-PCR) to detect CYP3A1 mRNA expression The frozen small intestine and liver tissue samples were naturally thawed at room temperature. Total RNA was extracted and tested for RNA concentration and purity, followed by reverse transcription and primer amplification (primer sequence shown in Table 1), with three replicates per transcript. CYP3A1 was encoded by Cyp3a1, and its internal reference was β-actin. Data processing was performed using the 2-ΔΔCt method. T-test was used for data consistent with normal distribution, and Wilcoxon rank sum test was used for data inconsistent with normal distribution. 10.1371/journal.pone.0297713.t001 Table 1 The sequence of primers. Name Sequence (5’-3’) 16s RNA V3~V4 F: barcode + ACTCCTACGGGAGGCAGCA R: GGACTACHVGGGTWTCTAAT Cyp3a1 F: ACCATCCTCGTGCTCCTGTATC R: AAGCTCCGACGGTTTGTGAAG β-actin F: TGCTATGTTGCCCTAGACTTCG R: GTTGGCATAGAGGTCTTTACGG Western blot assay for CYP3A1 protein expression The frozen small intestine and liver tissue samples were thawed at room temperature. CYP3A1 total protein was extracted, added to 10% SDS-PAGE gel, and then transferred to the nitrocellulose membranes. Nitrocellulose membranes were incubated with the monoclonal CYP3A4 antibody (1: 2000 dilution) at 4°C for 12 h, followed by the incubation with a secondary antibody (1:5000 dilution) at room temperature for 30 min. The GAPDH was used as a loading control. The Odyssey CLx Imaging System was used to detect target proteins. The protein level was normalized to the control group and expressed as multiple changes relative to the control group. For statistical analysis, t-test was used for data consistent with normal distribution, and Wilcoxon rank sum test was used for data inconsistent with normal distribution. Effects of antibiotics and probiotics on the intestinal microbiota The frozen feces were naturally thawed at room temperature, and DNA was extracted. PCR amplification was performed on specific primers for the V3-V4 region of 16S rRNA bacterial genes according to the Illumina 16S metagenomic sequencing library (Table 1). The amplified products were quantified and mixed for library construction and quality inspection. After the individual quantification step, amplicons were pooled in equal amounts and subjected to paired-end 2 × 250 bp sequencing to understand the effects of antibiotics and probiotics on the intestinal microbiota. Data analysis and statistics 16S rRNA intestinal microbiota data were analyzed using the Genescloud cloud platform online. The relative abundance of the microbiota was plotted as a stacked-bar plot at the phylum and genus levels. α-diversity indices (diversity within a sample) were calculated using filtered data. Among these indices, the Chao1 diversity index estimated total richness, and the Shannon diversity index evaluated the richness and evenness of species at the genus level. β-diversity (sample dissimilarity, the difference between two samples) was calculated using the principal coordinate analysis (PCoA) based on the Bray-Curtis dissimilar matrix. Kruskal-Wallis rank sum test was used to analyze bacteria with significant abundance differences between groups, and linear discriminant analysis effect size (LEfSe) was used to analyze bacteria enrichment. Microbial functional abundances based on marker gene sequences were predicted by Phylogenetic Investigation of Communities by Reconstruction of Unobserved States (PICRUSt2) in the Kyoto Encyclopedia of Genes and Genomes (KEGG). Spearman correlation analysis was performed to explore the relationship between the intestinal microbiota and the metabolic pathways of NBP. P<0.05 was considered statistically significant. Plasma sample processing method Acetonitrile (150 μL) containing glipizide (250 ng·mL-1) was added to plasma (50 μL) and centrifuged at 10,900 g for 5 min. The supernatant was transferred to the autosampler vial for HPLC-MS/MS detection.',\n",
       " '11198784': 'Materials and methods Participant characteristics Participants diagnosed with GWI and HC participants were recruited between 06th June and 30th November 2023 to be included in this study. The GWI group was composed of six Australian males who engaged in combat at the GW, were diagnosed with GWI previously, and met CDC Case Definition [14] and Kansas criteria [15] for GWI. Six HC male participants were selected for this study, all those reported to be in good health, absence of illness and no fatigue episodes. All volunteers were between 18 and 65 years and non-smokers. Participants were excluded from this investigation if they were previously diagnosed with a chronic illness or reported alcohol abuse, use of opioids, medication, or supplements that interfere with TRPM3 ion channels or Ca2+ signalling. However, participants had the option to cease taking conflicting medications in accordance with the half-life of pharmacological compounds and if authorised by their physician. This investigation was approved by the Griffith University Human Research Ethics Committee (GU HREC 2022/666) and all participants provided their written consent. Participant symptoms and disability All participants completed a questionnaire created by the National Centre for Neuroimmunology and Emerging Diseases (NCNED) to provide their medical history, sociodemographic background, illness symptoms, and disability information. To assess disability data the questionnaire includes the World Health Organization Disability Assessment Schedule (WHODAS). In this study, symptoms from people with GWI were classified in ten subtypes: (1) cognitive difficulties (e.g. cognitive overload, confusion, disorientation, impaired concentration, forgetfulness and memory problems); (2) pain (e.g. headaches, muscle aches and multi-joint pain); (3) sleep disturbances (e.g. unrefreshing sleep, frequent awakenings, prolonged sleep, reversed sleep cycle); (4) cardiovascular symptoms (e.g. orthostatic intolerance, cardiac arrhythmias, heart palpitations, light headedness and dizziness); (5) respiratory symptoms (e.g. air hunger, difficulty breathing); (6) thermostatic intolerances (e.g. subnormal body temperature, abnormal sweating episodes, hot flushes and cold extremities); (7) neurosensory or perceptual symptoms (e.g. inability to focus vision, impaired depth perception, sensitivity to touch, light, odour, taste, sound, vibration and poor balance or coordination); (8) urinary changes (e.g. changes to urination frequency and urgency to urinate); (9) immune disturbances (e.g. sore throat, tender lymph nodes, new allergies/sensitivities); and (10) gastrointestinal disturbances (e.g. nausea, abdominal pain, bloating, diarrhoea and irritable bowel syndrome). The WHODAS indicates the level of disability from each participant and data is combined by groups. WHODAS is subdivided into seven domains of life: (1) Communication and understanding; (2) Mobility; (3) Self-care; (4) Interpersonal connections; (5) Life activities; (6) Work or School participation; and (7) Participation in society. WHODAS items were scored on a five-point scale (none, mild, moderate, severe, and extreme or cannot do). The subscale scores were determined in accordance with the WHODAS 2.0 manual, first converting each item score into the corresponding, predefined weighted values [46]. Scores converted from 0% to 100%, disability are inversely proportional to the scale, whereby lower scores indicate less disability and correspond 100% to full disability [47]. Peripheral blood mononuclear cell and natural killer cell isolation Each participant donated between 40 ml and 84 ml of whole blood. All blood collections were conducted by a qualified phlebotomist, via venepuncture, using ethylenediaminetetraacetic acid (EDTA) tubes. A sample of 4 ml of whole blood from each participant was sent to a pathology laboratory for full blood count (FBC). The remaining whole blood samples were used to isolate peripheral blood mononuclear cells (PBMCs) by centrifugation over a density gradient medium (Ficoll-Paque Premium, GE Healthcare, Uppsala, Sweden). PBMCs total cell count, live cell count and viability were assessed using trypan blue dye (Invitrogen, Carlsband, CA, USA) and automatic cell counter (TC20 Automated cell counter, Bio-Rad, Laboratories, Hercules, CA). For NK cell isolations, PBMCs were adjusted for a concentration of 5×107 cells/ml. NK cell isolations were conducted by immunomagnetic selection using an EasySep Negative Human NK Cell Isolation Kit (Stem Cell Technologies, Vancouver, BC, Canada). Flow cytometry was performed to identify the NK cell purification from each NK cell isolation. Immediately after NK cell isolation, cells were incubated with CD56 APC (0.25g/20l) and CD3 PE Cy7 (0.25g/5l) monoclonal antibodies (Becton Dickinson (BD) Bioscience, San Jose, CA, USA) for 20 minutes in the dark at room temperature. NK cells were washed and resuspended in 350 ml of stain buffer (BD Bioscience, New Jersey, USA) and acquired at 10,000 events using the BD LSR- FortessaTM X-20 flow cytometer (BD Biosciences, San Diego, CA, USA). The NK cell population was then identified using phenotypic surface expression as CD3-CD56+. For this study, acceptable NK cells purity was ≥ 90%. S1 Fig shows and compares purity results from HC and GWI groups. Importantly, there was no statistical difference between groups. Electrophysiological experiments The gold standard patch-clamp technique was conducted to determine TRPM3 ion channel activity in NK cells freshly isolated from HC and people with GWI. In this study, borosilicate glass capillaries (Harvard Apparatus, Holliston, MA, USA, GC150F-15, outside diameter = 1.5 mm, inside diameter = 0.86 mm) were pulled to obtain glass pipette (Sutter Instrumental, model P-97) and polished posteriorly (Narishige, Micro Forge MF-900). When filled with pipette solution, membrane resistance was 8 to 12 MΩ. A CV203BU head-stage (Molecular Devices, Sunnyvale, CA, USA) connected to a 3-way coarse manipulator and a micromanipulator (Narishige, Tokyo, Japan) were used in these experiments. To amplify and record electrical signals, an Axopatch 200B amplifier and pClamp 10.7 software (Molecular Devices, Sunnyvale, CA, USA) were used, with data filtered at 5 kHz and sampled digitally at 10 kHz via a Digidata 1440A analogue to digital converter (Molecular Devices, Sunnyvale, CA, USA). The voltage-ramp protocol was a step from a holding potential of +10 mV to -90 mV, followed by a 0.1 s ramp to +110 mV, before returning to +10 mV (repeated every 10 seconds). The liquid junction potential between the pipette and bath solutions (10 mV) was corrected and no leak current component was subtracted. The intracellular pipette solution contained: 30 mM CsCl, 2 mM MgCl2, 110 mM L-Aspartic acid, 1 mM EGTA, 10 mM HEPES, 4 mM ATP disodium hydrate, 0.1 mM GTP sodium salt hydrate (pH = 7.2, adjusted with CsOH; Osmolality = 290 mOsm/L, adjusted with D-mannitol), filtered with 0.22 m membrane filter (Sigma-Aldrich, St. Louise, MO, USA), aliquoted and stored at -20°C. The possibility of chloride current involvement in TRPM3 assessment was minimized by using L-Aspartic acid in the intracellular pipette solution. The extracellular solution contained: 130 mM NaCl, 10 mM CsCl, 1 mM MgCl2, 1.5 mM CaCl2 2H2O, 10 mM HEPES, (pH = 7.4, adjusted with NaOH; Osmolarity = 300 mOsm/L, adjusted with D-glucose) freshly prepared. As previously validated by NCNED, pharmacological agents were included in the extracellular solution to assess TRPM3 ionic currents [26]. Briefly, a gravity perfusion system was used to apply extracellular solution for 50 seconds to establish a baseline current. Subsequently, extracellular solution containing 100 μM of PregS was added to stimulate TRPM3 ion channels for 2.5 minutes. Following on from this addition extracellular solution with 10 μM Ononetin and 100 μM of PregS was applied for another 2.5 minutes to block TRPM3 ion channels. After the conclusion of the drugs application, another cycle of 100 seconds of only extracellular solution was applied to remove the drugs. ATP and GTP were purchased from Sapphire Bioscience Reagents, PregS and Ononetin were purchased from Tocris Bioscience, while all other reagents and chemicals were ordered from Sigma-Aldrich. PregS and Ononetin were resuspended and stored in accordance with the manufacturer’s instructions. Electrophysiological experiments were conducted at room temperature (22–24°C). All recordings were analysed individually by a blinded researcher and posteriorly data was reviewed one by one by another blinded researcher, as detailed in S2 Fig. Additionally, any unstable currents or chloride contamination was excluded from the analysis. Statistical analysis Questionnaire data were analysed through the Statistical Package for the Social Sciences (SPSS) software, version 27 (IBM Corp, Armonk, NY, USA) and purity results with GraphPad Prism v9 (GraphPad Software Inc., La Jolla, CA, USA). For electrophysiological data analysis and data presentation, pCLAMP 10.7 software (Molecular Devices, Sunnyvale, CA, USA), Origin 2021 (OriginLab Corporation, Northampton, MA, USA), and GraphPad Prism version 9 were used. Shapiro-Wilk normality test was performed to identify the distribution of data. ROUT method was conducted to determine outliers and they were removed from analysis. The independent nonparametric Mann-Whitney U test was performed to identify the statistical significance between GWI and HC groups in PregS and Ononetin amplitude. The Fisher’s exact test (applying Bonferroni method) was conducted to determine statistical significance regarding sensitivity to Ononetin in NK cells. Significance was set at p < 0.05 and the data are presented as mean ± standard error of the mean (SEM) unless otherwise stated.',\n",
       " '11198732': '',\n",
       " '11198029': 'Methods Literature Search For the purpose of evaluating the evidence regarding the efficacy of metformin therapy for AF, a narrative review of the literature published on Embase was conducted, facilitated by a librarian and co-author (RM). For this review, both peer reviewed original studies and reviews, along with non-peer reviewed (preprint) papers from bioRxiv and medRxiv etc., regarding the effects of metformin on AF were critically evaluated. Publications were selected based on evidence that the manuscripts support their claims with statistical evidence. Their scientific merit was also determined on the basis of their study design. Publications in Embase were identified by using the search strategy described in Table 2. Table 2Search Strategy Used on Embase. The Search Was Based on “Metformin and AF and Diabetes and (Clinical Trials or Reviews)”. Total Number of Studies Used Was = 196 After Removing Duplicates (n = 3)StepSearchResults1(metformin or dimethylbiguanidine or dimethylguanylguanidine or glucophage).ab,ti.47,4802Metformin emtree: exp metformin/86,0233(“Atrial fibrillation” OR AF OR Afib OR AFib OR A-fib OR A-Fib OR “Supraventricular tachycardia” OR SVT OR Arrhythmi* OR palpitat*).ab,ti.346,3314AF emtree: exp atrial fibrillation/126,43851 OR 289,22763 OR 4379,49975 AND 61,1378Diabetes (Emtree term): exp diabetes mellitus/1,224,3809Clinical trials (Emtree term): exp clinical trials/447,028109 OR (randomized controlled trial.ti,ab).574,55511Reviews (Emtree term): exp review/3,175,7991211 OR (literature review or literature).ti,ab.4,159,842137 AND 88621410 OR 124,515,7651513 AND 1420016Limit 15 to yr=”1998–2024”199 The data presented in Figure 1 was generated from the Scopus database and based on the following search: (TITLE-ABS-KEY (metformin OR dimethylbiguanidine OR dimethylguanylguanidine OR glucophage)) AND (TITLE-ABS-KEY (“atrial fibrillation” OR afib OR a-fib OR “Supraventricular tachycardia” OR svt OR arrhythmi* OR palpitat*))',\n",
       " '11197919': 'Materials and methods Study participants This study was conducted at Cornell University in Ithaca, New York, between October and December 2020. Participants were recruited via advertisements using flyers and e-mail listservs in and around the Cornell University campus. All human-related procedures and sample and data collection were approved by the Cornell University Institutional Review Board for Human Participant Research (Protocol Number: 1902008575) prior to recruitment and enrollment of participants. Study participants included healthy males and healthy, non-pregnant or lactating females 18–59 years old. Baseline participant characteristics can be found in Table 1. Exclusion criteria included a history of gastrointestinal diseases or surgeries; type 1 or type 2 diabetes, prediabetes or impaired glucose tolerance, self-reported untreated thyroid condition; use of antibiotics 6 months prior to the start of the study; and chronic alcohol intake (>5 drinks/day). The full list of inclusion and exclusion criteria is provided in Table S1. This trial was registered at clinicaltrials.gov as NCT05743790 on February 24, 2023.Table 1.Baseline characteristics for all participants assigned to a study arm. Group A (n = 35)Group B (n = 33)Totalp valueAge in years28 ± 927 ± 1127 ± 100.376 Sex, n (%) 0.542 Female22 (62.9%)24 (72.7%)46 (67.6%) Male13 (37.1%)9 (27.3%)22 (32.4%) Body fat percentage Female26.2 ± 7.924.9 ± 6.625.50 ± 7.20.531 Male18.0 ± 5.120.40 ± 7.019.00 ± 5.90.351Energy-adjusted dietary fiber intake at baseline in grams12.7 ± 5.511.0 ± 4.311.9 ± 5.00.235There were 35 participants in Group A and 33 in Group B, for a total of 68 who were assigned to a study arm. Data are presented as mean (standard deviation) or n (%). There were no significant differences between Group A and B at baseline for age, sex, body fat percentage for males, body fat percentage for females, and energy-adjusted dietary fiber intake at baseline. T-tests and Wilcoxon rank sum tests were used for numerical variables and chi-squared tests were used for binary variables. Study design We enrolled eligible participants into a 7-week open-label crossover dietary intervention study (CONSORT diagram: Figure S1). Each participant was assigned a study identification (ID) number based on the order in which they were enrolled in the study. All participants with an odd ID number were allocated to Group A, and all participants with an even ID number were allocated to Group B, achieving a 1:1 balanced allocation. A single participant with an odd ID number was allocated to Group B to balance the groups. Participants were instructed to consume crackers containing RS2, RS4, or a digestible control starch (Ctl), in their assigned treatment order, each for 10 d (Figure 1), in addition to their normal dietary intake. The groups received treatments in the following order. Group A: Treatment 1 = RS2, Treatment 2 = Ctl, Treatment 3 = RS4; Group B: Treatment 1 = RS4, Treatment 2 = Ctl, Treatment 3 = RS2. The three treatment periods were separated by 5-d washout periods when no study crackers were consumed. Participants collected fecal samples prior to (“PreRS2,” “PreCtl,” or “PreRS4”) and at the end of (“EndRS2,” “EndCtl,” or “EndRS4”) the treatment periods (Figure 1: see “Pre” and “End” time points). Figure 1.Study design.Participants were enrolled in one of the two arms (Groups A and B) of a 7-week crossover dietary intervention study during which we supplemented their diets with crackers containing one of the three starches: RS2 = resistant starch type 2, RS4 = resistant starch type 4, Control = digestible starch. Each treatment lasted for 10 d, and there were 5-d washouts between treatments. Stool samples were collected at baseline and at the beginning (Pre) and end (End) of each treatment. Please see CONSORT flow diagram (Figure S1). This figure was created using BioRender. Dietary supplementation Participants were provided with crackers that contained RS2 (HI-MAIZE® 260 starch, Ingredion), RS4 (VERSAFIBE™ 1490 starch, Ingredion), or a digestible control starch (AMIOCA™ TF starch, Ingredion). Study cracker formulations are listed in Table S2a. During Treatments 1 and 3, we aimed to provide 30 g of each type of RS in a daily serving of 120 g of crackers for 7 d after a 3-d ramp-up period to minimize gastrointestinal discomfort. The daily serving size for the control cracker after the ramp-up period of 3 d was also 120 g. Thus, during each 10-d treatment period, we followed a dose escalation of study crackers where participants gradually increased the dosage over 3 d to reach a final dose of 120 g of crackers per day for the last 7 d (Day 1: 25%, Day 2: 50%, Day 3: 75%, Day 4–Day 10: 100%). The RS2, RS4, and control crackers were approximately matched for total carbohydrate, fat, and protein content, but the control cracker contained more calories than the RS crackers (Table S2b). Following production, the crackers were analyzed for RS and total dietary fiber content by Medallion Laboratories (Minneapolis, MN) using AOAC 2002.02 and AOAC 991.43 methods, respectively (Table S3). The method AOAC 2002.02 accurately quantifies RS2 but not RS4, so it is not providing an accurate measurement of RS in the RS4 crackers.24,25 However, the method AOAC 991.43 does detect RS4, so by deduction, the AOAC 991.43 measurement is indicating the approximate amount of RS4 (less the 1.92 g of total dietary fiber from the other ingredients, which are also in the control cracker).26 After baking, the amount of RS in RS2 crackers was reduced from 30 g to 21.27 g per serving, possibly due to starch gelatinization, and this may have also occurred in the RS4 crackers. Participants were asked to maintain their habitual dietary intake and physical activity across interventions and to avoid taking prebiotic- or probiotic-added foods, drinks, or supplements throughout the study. Study crackers were provided in individual bags containing a preportioned daily supply by weight. At the end of each treatment period, participants were asked to complete a short questionnaire to indicate the percentage of crackers consumed per day as an indicator of adherence to the protocol. Data from participants who reported a low adherence to cracker consumption were excluded from analysis. Low adherence was defined as having consumed <75% of study crackers for 5 or more days during the treatment or <50% on the day of or day before collecting their End time point fecal sample. Anthropometric measurements At baseline, we measured the participants’ total body fat percentage with a Tanita SC-240 Total Body Composition Analyzer using Bioelectrical Impedance Analysis (BIA) as per the manufacturer’s instructions. Dietary intake data During the baseline and treatment periods, we instructed participants to complete two nonconsecutive 1-d food records of one weekday and one weekend day, excluding study cracker consumption, using the Automated Self-Administered 24-hour (ASA24) Dietary Assessment Tool 2020 developed by the National Cancer Institute, Bethesda, MD (https://epi.grants.cancer.gov/asa24.).27 Data were averaged from the two food records for each participant within the baseline week and each of the three treatments (Figure S2a). Data were excluded if only one of the two records was completed. Saliva collection We obtained 5 ml of saliva at baseline and during each treatment for a total of four saliva samples throughout the study for each participant (Figure 1). Participants were instructed to refrain from brushing their teeth for a minimum of 6 hours and to avoid consuming any food or beverages including water for a minimum of 30 minutes prior to sample collection. They were then instructed to accumulate saliva in their mouth and express it into a 50 ml sterile conical tube. Saliva samples were stored on ice immediately after collection, aliquoted within 3 hours, and stored at −80°C. AMY1 copy number determination by qPCR and ddPCR Genomic DNA was extracted from saliva samples using the QIAamp 96 DNA Blood Kit, QIAamp Blood Mini Kit, and the QIAamp Investigator Kit (Qiagen, cat # 51161, 51104, 56504). We performed qPCR using primers, previously described in Poole et al.,18 to amplify AMY1 paralogs and our reference gene, EIF2B2 (CN = 2). For each gene, each qPCR reaction consisted of 1 μl genomic DNA normalized to 5 ng/μl, 0.5 μl of each forward and reverse primer at 10 μM, 3 μl of PCR grade H2O, and 5 μl iTaq™ Universal SYBR® Green Supermix (BioRad, cat # 1725122), for a total volume of 10 μl per reaction. The qPCR conditions were as follows: initial denaturation at 95°C for 5 minutes and 40 cycles of 95°C for 10 seconds and 60°C for 30 seconds on a Roche LightCycler 480 Real-Time PCR Instrument. We made a standard curve using genomic DNA NA12286 (Coriell Institute; AMY1 CN = 2). The following genomic DNAs were used as positive controls on all qPCR plates, as their AMY1 copy numbers have previously been estimated: NA18972, NA12873, NA10472, NA12890, NA10852, NA12043, NA11992, NA12414, NA12340, NA06994, NA12342, NA12286, NA18522, and NA19138 (Coriell Institute). All reactions including standards and blanks were performed in quadruplicate, and results were averaged for technical replicates with a coefficient of variation <0.05. At least two qPCR runs were performed for all participants, and we calculated the median value of all qPCR results to determine the final qPCR AMY1 CN value. For digital PCR, genomic DNA was digested with the restriction enzyme HaeIII (New England Biolabs, cat # R0108S) and diluted to a final concentration of approximately 15 ng/µl. Digital PCR was performed using Life Technologies TaqMan Copy Number Assay Id Hs07226361_cn for the AMY1 locus and TaqMan Copy Number Reference Assay Hs06006763_cn for the AP3B1 gene to normalize for total DNA. The reactions were run on a QX100 Droplet Digital PCR System in duplicate. We averaged the median of all qPCR results and the two digital PCR results to determine the final AMY1 CN used in our analyses (Figure S2b). AMY1 CN group was defined as that used in a previous publication that inspired this work18 because even though categorization is often used, currently, there is no consensus in the field regarding the threshold for high and low categories.28,29 Perry et al. reported that low starch consuming populations have a median AMY1 CN of 5 and high starch consumers have a median of 7.30 Although the threshold for the effects of this gene on physiology or health is currently unknown, Perry et al.’s conclusion that positive selection occurred at this locus due to dietary adaptation suggests that the low end of phenotypic effect is CN of 5. In our cohort, salivary amylase activity (SAA) is significantly different between our AMY1 CN groups with the mean of the high group (SAA = 145 U/ml) being over twice that of the low group (SAA = 66 U/ml) over time (p = 4.85 × 10−4). Salivary amylase activity assay We measured SAA for each saliva sample in triplicate, from up to four saliva samples donated from each participant throughout the duration of the study, using the Salimetrics Salivary Alpha-Amylase Enzymatic Kit (SALIMETRICS, cat # 1–1902). The manufacturer’s protocol was followed except for the use of 300 µl amylase substrate per reaction instead of 320 µl. We averaged SAA determined for each participant to obtain the mean SAA used in our analyses (Figure S2c). Fecal sample collection Participants collected fecal samples at seven time points throughout the study (Figure 1). Samples were collected from a single bowel movement and stored at −80°C within 24 hours of collection. We lyophilized an aliquot from each sample for 16S rRNA gene sequencing. Samples from study participants who reported collection of fecal samples ≥2 d after completion of the treatment were excluded from analysis. 16S rRNA gene sequencing DNA extraction was performed on 0.030–0.045 g lyophilized fecal samples using the DNeasy PowerSoil 96 HTP Kit (Qiagen, cat # 12888–100), following the manufacturer’s instructions with the specifications: samples were loaded into PowerBead Plates and stored at −20°C in Bead Solution until extraction, and instead of vortexing to mechanically lyse samples, samples were placed in a BioSpec 1001 Mini-Beadbeater-96 for 3 minutes. We amplified the V4 region of the 16S rRNA gene using the universal primers 515F and barcoded 806 R31 and approximately 100 ng of genomic DNA from each sample in duplicate PCR reactions using 25 µl Classic++™ Hot Start Taq DNA Polymerase Master Mix (Tonbo Biosciences, cat # TB-31-5011–1000 R), 22 µl PCR water, 0.5 µl of each 10 nM primer, and 2 µl DNA. We used the PCR program previously described but with 25 cycles of amplification.31 We purified amplicons using Mag-Bind TotalPure (Omega Bio-tek, cat # M1378–01) using a 1.8X bead ratio, pooled 100 ng of amplicons from each sample, and performed 2 × 250 bp sequencing on an Illumina MiSeq instrument. We performed microbiome bioinformatics with QIIME 2.32 We demultiplexed and quality filtered raw sequence data via q2-demux. Then we resolved ASVs using DADA233 via q2-dada2. We aligned ASVs with mafft34 via q2-alignment and constructed a phylogenetic tree using fasttree35 via q2-phylogeny. We used q2-diversity to calculate alpha-diversity (Faith’s Phylogenetic Diversity, Faith’s PD)36 after samples were subsampled without replacement to 23,081 sequences per sample based on the sample with the lowest sequence count. These rarefied data were used exclusively for calculating Faith’s PD. For differential abundance analysis, we utilized the non-rarefied feature table to retain the full complexity of the microbial communities. We used the q2-feature-classifier37 classify-sklearn naive Bayes classifier against the Greengenes 13_8 99% OTUs reference sequences38 to assign taxonomy to ASVs. We identified 6,547 ASVs within our entire dataset (all time points from 59 participants). After we calculated the mean number of ASVs present (number of ASVs with a non-zero read count) in each individual across all time points, we observed that the mean of the means was 159 ASVs per person, the median of the means was 156 ASVs per person, and the standard deviation of the means was 45. Short-chain fatty acid measurements We measured SCFA concentrations in stool samples collected at baseline and at the time points Pre and End of each treatment. SCFAs including acetate, propionate, isobutyrate, butyrate, isovalerate, valeric acid, isocaproic acid, caproic acid, and heptanoic acid were quantified using ultra-performance liquid chromatography (Acquity UPLC system, Waters Corporation, Milford, MA) at the PennCHOP Microbiome Program Microbial Culture & Metabolomics Core. Total SCFA concentration was calculated as the sum of all nine SCFAs quantified. Statistical analysis We performed all statistical analyses using RStudio version 4.2.1.39 We adjusted dietary fiber intake by energy (g/1,000 kcal/day) to account for differences in participants’ overall energy intake. We considered p-values <0.05 to be statistically significant. When adjusting p-values we used the Benjamini–Hochberg false discovery rate correction and considered a q-value (adjusted p-value) of <0.05 to be statistically significant. To calculate fold changes of ASVs, we added a pseudocount of 1 to all ASV counts prior to calculating relative abundances to avoid division by zero. Since the limit of detection of the quantification method for SCFAs was 5.0 µmol/g of stool, a count of 2.5 µmol/g was added to all SCFA measurements in the linear regression and LASSO analyses, in which logarithm or fold change was calculated, to avoid an undefined term (logarithm of zero or division by zero). We used R package ComplexHeatmap40 and circlize41 to generate all heat maps. To assess the response of ASVs to each treatment, we used the R package MaAsLin2 (Microbiome Multivariable Association with Linear Models 2) because this software allows us to include repeat measures in longitudinal sampling (Pre and End) as well as covariates, takes into consideration the nature of ASV data (zero-inflated, high dimensional, non-normal distribution) and controls for the false discovery rate.42 Data from all participants were included, regardless of the order of treatment (Group). For each treatment (RS2, RS4, and Ctl), we used the min_prevalence default = 0.1 parameter to filter out the ASVs present in less than 10% of the samples collected. For example, ASVs had to be present in at least 10% of the combined samples from PreRS2 and EndRS2 to be included in the RS2 linear mixed model, leaving a total of 325 ASVs. After prevalence filtering, there were 330 ASVs included in the linear mixed model for RS4 and 365 ASVs for Ctl. Prior to fitting the models, the ASV data were normalized using total sum scaling. We used the log transformation setting within MaAsLin2’s transform parameter. For each treatment, we fit three types of models. First, we fit main effects models using time point, abbreviated TP (PreRS2 versus EndRS2, PreRS4 versus EndRS4, or PreCtl versus EndCtl), as a fixed effect and participant as a random effect in each model to identify ASVs that, on average, changed in relative abundance from Pre to End of each treatment period.(1)Relative abundance \\\\~ TP Pre, End + 1|Participant Our next goal was to identify variables that affected the change in relative abundances of ASVs during each treatment. Thus, we added interaction terms to each of these separate models (AMY1 CN × time point, AMY1 CN group × time point, mean SAA × time point, baseline dietary fiber intake × time point, dietary fiber intake during treatment × time point, and treatment order × time point). Because this was a screening step, we followed up on all interactions with q < 0.25. Next, we fit main effects models using End time points only, with each candidate predictor as an independent variable in a separate model, to detect whether the candidate predicted the relative abundance of any ASVs at the end of the treatment with q < 0.05. To determine the effect of each treatment on SCFA concentrations, we fit linear mixed models with log SCFA concentration as the response variable, time point (TP; Pre vs End) as a fixed effect, and participant as a random effect.(2)Log SCFA + 2.5 \\\\~ TPPre, End + 1|Participant We fit separate models for each treatment (RS2, RS4, Ctl) and SCFA (acetate, butyrate, propionate, and total). To determine whether treatment order (Group A versus B) affected changes in SCFA concentrations from Pre to End (time point), we used linear regression models with each SCFA as the response variable and included an interaction term between Group and time point. A random effect term was included for participant. Fixed effects were tested using F tests with the Kenward-Roger approximation for the degrees of freedom, and post-hoc pairwise comparisons were performed using Tukey’s HSD method.(3)Log SCFA + 2.5 \\\\~ TPPre, End x GroupA, B + TPPre, End + GroupA, B + 1|Participant To assess whether alpha diversity could predict changes in SCFA concentrations, we fit logistic regression models with SCFA change score as the response variable and Faith’s phylogenetic diversity (PD) as a fixed effect. We defined any increase in SCFA concentration from Pre to End (End – Pre > 0) to have a change score of 1, while no change or a decrease in SCFA concentration (End – Pre ≤ 0) was assigned a change score of 0.(4)SCFA change score \\\\~ Faith′s PD Pre Because our goal is to predict the SCFA change score, we used Faith’s PD at the beginning of the treatment (Pre) as the predictor in each model. We fit separate models for each treatment (RS2, RS4, Ctl). We used the R package glmnet43 to perform 10-fold cross-validated Least Absolute Shrinkage and Selection Operator (LASSO) linear regressions to model the log fold change of acetate, propionate, and butyrate concentrations between Pre and End of each treatment period (log2 [End/Pre]). For variable selection, we included the predictors: 10% prevalence-filtered ASVs at Pre, body fat percentage, sex, treatment order, AMY1 CN, AMY1 CN group, mean SAA, and self-reported physical activity (minutes/week of vigorous, moderate, and low-intensity activity) at baseline. We used a separate model for each SCFA and each treatment, and after 10% prevalence filtering for each treatment, there were 332 ASVs in the RS2 models, 339 ASVs in the RS4 models, and 349 ASVs in the Ctl models. To create the figure depicting the LASSO coefficients, we standardized the regression coefficients and excluded the predictor group, or treatment order, to show all the coefficients on a comparable scale. Standardization was performed by multiplying each β coefficient by the standard deviation of the relative abundance of the corresponding ASV at Pre.',\n",
       " '11197841': 'Materials and methods Detailed methods are provided in the ESI† of this paper.',\n",
       " '11197476': 'Materials and Methods Preparation and characterization of CXB@Lipo and C@ECN CXB@Lipo (C@L) was obtained by the thin-film hydration method. Briefly, CXB (1 mg), cholesterol (38 mg), and HSPC (200 mg) were dissolved in chloroform. Then, chloroform was removed from this hybrid solution by a rotary evaporator from the film on the round-bottom flask. Next, the film was retuned by adding 2 ml of H2O and sonicating for 10 min. Following this, the pre-C@L was further extruded with a liposome extrude (LiposoFast, Avestin, Canada) through 200-nm polycarbonate membrane filters (Merck Millipore Ltd., USA). Finally, the prepared C@L was concentrated in an ultrafiltration centrifuge tube (30 kDa at 2,000 rpm for 30 min). C@L was riveted on the surface of the ECN, ECN (1 × 107) was resuspended in PBS buffer, and 1 mg of glycol chitosan was added and stirred for 30 min. Then, 1 mL of C@L was added to this buffer (ECN-GC) and stirred for another 30 min. Finally, C@L was removed by washing the complex 3 times with H2O and the mixture was stored at 4 °C for 1 week. The morphology of the ECN or C@ECN was determined via transmission electron microscopy (TEM; JEM-1400, Japan). The average size and zeta potential of the various groups were measured by a Malvern Nano ZS ZEN3600 (Malvern, UK). The characteristic absorption of CXB, liposome, and C@ECN-PL was also determined by UV-VIS (Thermo Fisher Scientific, USA). ICG was used to replace CXB using the above method to obtain ICG@ECN (I@ECN). Then, the effect of mixed liposomes loaded with I@ECN was tested by flow cytometry (CytoFLEX, Beckman, USA). The viability of C@ECN was determined in LB media supplemented with UV‒VIS. Then, 10 μL of ECN, ECN-GC, or C@ECN was added to 5 mL of LB media and incubated at 37 ℃ and 200 rpm. The number of ECNs was determined by the absorption at OD600 nm using UV‒VIS. Preparation of C@ECN-PLs pGEX-Pvhb-Lysis was constructed through the Gibson self-assembly method, involving cleavage of the pGEX plasmid by Bam HI and Eco RI endonucleases, seamless ligation of the Pvhb-Lysis gene fragment into linearized pGEX using a cloning kit (Beyotime Biotechnology Co. Ltd., Shanghai, China), and subsequent introduction of the ligated plasmid into ECN competent cells. Cell uptake of C@ECN To determine the uptake effect of C@ECN or C@L, ICG was used to replace CXB with ICG@Liposome (I@L) or I@ECN [ICG concentration of 0.1 mg/ml and 108 cfu of ECN], respectively. CT26 tumor cells (1 × 106) were seeded into 30-mm glass plates. After incubating for 12 h, fresh DMEM containing I@ECN or I@L was added to the CT26 tumor cells and incubated for another 4 h. Then, the DMEM was removed, and the cells were washed with PBS 3 times. The sections were incubated with DAPI for another 10 min at room temperature, washed with PBS 3 times and stored at 4 °C. Finally, the cells were imaged by confocal laser scanning microscopy (CLSM) (Nikon, Japan). Cell cytotoxicity assay CT26 and MCF-7 tumor cells were seeded into 96-well plate at a starting concentration of 104 cells per well. After incubation for 12 h, free DMEM containing various concentrations of C@L, ECN, C@ECN-PL, C@L + DOX, ECN + DOX, or C@ECN-PL + DOX was added to the medium (DOX concentration of 1.5 μM) and the cells were incubated for another 24 h. Finally, cell viability was determined with a CCK-8 assay kit (Beyotime Biotechnology Co. Ltd., Shanghai, China). COX-2 expression analysis To determine the expression level of COX-2 after various treatments, CT26 tumor cells (106) were seeded into 6-well plates, respectively. After incubating for 12 h, the cell were incubated with fresh DMEM supplemented with 20 μM CXB for another 12 h. Then, RIPA lysis buffer was used to extract proteins from these cells. The expression levels of COX-2 and β-actin were determined by Western blot (WB) analysis. Additionally, all these images were further analyzed and quantified using ImageJ. CLSM was also used to further test COX-2 expression. Briefly, CT26 or MCF-7 tumor cells (106) were seeded into 3cm glass plates. After 12 h of incubation, 20 or 30 μM CXB was added to the plate, and the cells were cultured for another 6 h. Then, the DMEM was removed, and the cells were washed 3 times with PBS. The cells were stained with a FITC-conjugated anti-COX-2 antibody (Affinity, Jiangsu, China) for 2 h at 4 °C, followed by DAPI staining for another 10 min and 3 washes with PBS. Finally, CLSM was used to test the fluorescence signal. After CT26 cells were treated with the aforementioned method, they were collected in 1.5 mL EP tubes and subjected to flow cytometric analysis. Pharmacokinetics and biodistribution of C@ECN in vivo The biodistribution of C@ECN was investigated using an in vivo images system (IVIS). In total, 107 CT26 cells were subcutaneously injected into the left forelimb under the armpit of 6-week-old BALB/c mice. Once the tumor volume reached approximately 100 mm3, the mice were randomly divided into 2 groups and intravenously administered I@L or I@ECN (ECN, 106). The biodistribution of I@L or I@ECN was observed through imaging at 0, 12, 24, 48, and 72 h after injection. After the 72-h treatment period, the mice were euthanized, and major organs (heart, liver, spleen, lung, kidney, and tumor) were collected for in vitro fluorescence signal testing. Tumor tissues were harvested and subsequently embedded in optimal cutting temperature (OCT) compound. After treatment with I@L or I@ECN for 48 h, the specimens were sectioned into slices of 10 μm thick slices. After incubation with DAPI for a period of 10 min, the slices were imaged via CLSM to capture their fluorescence characteristics. Antitumor efficacy of C@ECN-PLs in CT26 tumors in vivo CT26 tumor model was established by subcutaneous transplantation. First, CT26 tumor cells (108) were injected into the left forelimb under the armpit of BALB/c mice to establish the main tumor model. When the tumor volume reached approximately 100 mm3, the mice were randomly divided into 4 groups: PBS, DOX, CXB + ECN-PL + DOX, and C@ECN-PL + DOX, respectively (DOX, 4 mg/kg; CXB, 5 mg/kg; ECN, 107 CFU). Tumor volume and body weight were measured by a Vernier caliper and electronic scale every 2 d for 14 d. The tumor volume was calculated by the following formula: (major axis) × (minor axis2)/2. After 14 d, the mice were sacrificed, and tumors were collected for weighing. Evaluation of tumor apoptosis and proliferation in vivo The CT26 tumor model was established by the above method. All the mice were sacrificed, and the tumor tissue was collected and stored in OCT gel and cut into 6-μm slices at −20 °C. First, this slices were blocked with PBS containing 5% FBS for 30 min. Then, FITC-conjugated anti-mouse CD3, FITC-conjugated anti-mouse CD4, and PE-conjugated anti-mouse CD8 were used to stain the slices for 4 h (dilution factor 1:500, Biolegend Inc., USA). After staining with DAPI for another 10 min and washing with PBS 3 times, the fluorescence signal was obtained by CLSM. In addition, TUNEL and H&E staining were used to analyze CT26 tumor apoptosis in vivo, and Ki67 was used to assess proliferation after various treatments. Biosafety evaluation of C@ECN-PL The biosafety of C@ECN-PL was evaluated by intravenously injected into BALB/c mice. Briefly, healthy mice were intravenously injected with C@ECN-PL at a concentration of (5 mg/kg) CXB, and the number of ECNs was (107 CFU). Seven days later, the mice were sacrificed and blood and major organs (heart, liver, spleen, lungs, and kidney) were collected. The serum levels of aspartate aminotransferase (AST), alanine aminotransferase (ALT), blood urea nitrogen (BUN), and creatinine (CRE) were evaluated by assay kits purchased from the Jiancheng Bioengineering Institute (Nanjing, China). The integrity of major organs was also evaluated by H&E staining analysis, and images were further quantitatively analyzed by ImageJ 6.0. Statistical analysis Statistical analysis was performed via a one-way analysis of variance (ANOVA) test. Moreover, post hoc analysis was performed using the Wilcoxon rank sum test with a Bonferroni correction when needed. *P < 0.05 was considered to indicate statistical significance; **P < 0.01 and ***P < 0.001 were considered to indicate extreme significance; NS, no significant difference.',\n",
       " '11197382': 'MethodsData of 121,583 participants were obtained from the TWB. Male participants with a serum uric acid level >7 mg/dl and female participants with a serum uric acid level >6 mg/dl were classified as having hyperuricemia. Details of self-reported PUD were obtained by questionnaire. The association between hyperuricemia and self-reported PUD in the male and female participants was examined using multivariable logistic regression analysis. Materials and methods TWB To enhance biomedical and epidemiological research and address the aging population in Taiwan, the TWB is an ongoing prospective study launched by the Ministry of Health and Welfare in 2012 of community-dwelling cancer-free women and men (27, 28). Ethical approval for the TWB was given by the Ethics and Governance Council of the TWB and Institutional Review Board on Biomedical Science Research, Academia Sinica, Taiwan. The TWB contains medical, genomic and lifestyle factor data, including age, weight, height, and diagnoses of hypertension and diabetes mellitus (DM). In addition, laboratory tests on fasting serum samples (Roche Diagnostics GmbH, D-68298 Mannheim COBAS Integra 400) are conducted to collect data on glucose, hemoglobin, triglycerides, total cholesterol, high- and low-density lipoprotein cholesterol (HDL-C/LDL-C), and uric acid. Estimated glomerular filtration rate (eGFR) and serum creatinine levels were calculated as reported in previous studies (29). The average of three blood pressure measurements was used for analysis, with each measurement being performed in the absence of caffeine, nicotine and exercise by a nurse using an electronic monitor. Regular exercise was defined according to the “Physical Fitness 333 Plan” in Taiwan as at least three sessions of exercise per week with each session lasting at least 30 min (30). This study complies with the Declaration of Helsinki and was performed according to institutional review board approval (KMUHIRB-E(I)-20210058). Sample population and sample size The TWB enrolls cancer-free members of the community aged 30–70 years, and includes data on medical, genetic, and lifestyle factors. We collected 121,583 enrollees in the TWB. These participants were then classified into those with and without hyperuricemia based on a serum uric acid concentration of >7.0 and >6.0 mg/dl in males and females, respectively (31) (Figure 1). FIGURE 1 Flowchart of study population. Definitions of self-reported PUD A history of self-reported PUD was recorded using self-reported questionnaires. The presence of self-reported PUD was defined by asking the participants whether they a history of PUD. Statistical analysis The statistical analyses in this study were performed using SPSS version 19.0 for Windows (IBM Inc., Armonk, NY, USA). Continuous variables were expressed as mean ± standard deviation, and between-group differences were analyzed using the independent t-test. Categorical variables were presented as frequencies and percentages, and between-group differences were analyzed using the Chi-square test. Associations between hyperuricemia and self-reported PUD in the male and female participants were examined using multivariable logistic regression analysis, which included significant variables in univariable analysis. An interaction p in logistic analysis: Model disease (y) = x1 + x2 + x1 × x2 + covariates. x1 × x2 was the interaction term, in which y = self-reported PUD; x1 = sex; x2 = hyperuricemia; covariates = age, sex, DM, hypertension, smoking and alcohol history, regular exercise habit, systolic blood pressure (SBP), body mass index (BMI), hyperuricemia, fasting glucose, hemoglobin, triglycerides, total cholesterol, LDL-cholesterol, and eGFR. A two-tailed p-value < 0.05 was considered statistically significant.',\n",
       " '11197378': \"MethodsDaily average mass concentration data of six air pollutants in the urban area of Yan'an City from January 1, 2017 to December 31, 2020 were collected from the monitoring station in Baota District, Yan'an City. Meteorological information was obtained from the Meteorological Bureau of Yan'an City, including temperature,relative humidity and wind speed for the time period. The mass concentration of air pollutants in each exposure window of pregnant women was assessed by the nearest monitoring station method, and conditional logistic regression was used to analyze the relationship between air pollutants and preterm births, as well as the lagged and cumulative effects of air pollutants. Multiple linear regression was used to explore the relationship between air pollutants and blood tests after stepwise linear regression was used to determine confounders for each blood test. Methods Research population In this study, the data of pregnant women in the Department of Obstetrics and Gynecology of two local hospitals in Bota District, Yan'an City, were collected from 2018 to 2020. After collecting data information, we confirmed the inclusion and exclusion criteria. Inclusion criteria: residents of Baota District, Yan'an City, who have lived in the district for one year or more; normal mental status; no communication barriers; no assisted conception; no missing information; no major diseases.Exclusion criteria: people with various missing information; people with abnormal mental status who cannot communicate; people with assisted conception or multiple pregnancies; people with hereditary diseases.All pregnant women have signed an informed consent form, and the study has been approved by the ethics committee of Medical School of Yan'an University (approval number: 2018051). After screening for inclusion criteria, 460 cases of preterm birth(PTB) with complete maternal data were collected as the case group. In order to reduce the influence of confounding factors on the results of the study, we took term births(TB) of the same age and the same gestation as preterm births as the control group and selected 1,840 cases of term births as the control group in a ratio of 1 to 4. The data collected from pregnant women included the following: general data of pregnant women (name, age, gestational address, occupation), pregnancy data (gestational age, Pregnancy times, number of births, number of caesarean sections, regularity of menstrual cycle, season of the last menstrual period, Complications diseases, Comorbidity diseases, and hypertension in pregnancy), neonatal data (birth weight, date of birth) and routine blood data:leukocyte count (WBC), percentage of neutrophils (NEUT), percentage of lymphocytes (LYM), percentage of monocytes (MONO), percentage of eosinophils (EOS), percentage of basophils (BAS), erythrocyte count (RBC), hs-CRP,CRP. Complications diseases: severe vomiting of pregnancy, ectopic pregnancy, placenta previa, placental abruption, excessive or low amniotic fluid, premature rupture of membranes, hyperemesis gravidarum, acute chorioamnionitis. Comorbidity diseases: combined cardiovascular diseases (congenital heart disease, rheumatic heart disease,etc.), combined hematological diseases (chronic aplastic anemia, idiopathic thrombocytopenic purpura,etc.), combined respiratory diseases (tuberculosis, bronchial asthma), combined gastrointestinal system diseases (viral hepatitis, acute appendicitis,etc.), combined urinary system diseases (acute pyelonephritis, chronic glomerulonephritis,etc.), combined endocrine system diseases (hyperthyroidism, hypothyroidism,etc.), combined dermatological disorders (scleroderma, hives, herpes), combined infectious diseases (cytomegalovirus infections, genital herpes,etc.), combined tumors (uterine fibroids, cervical cancer,etc.). Pollutant exposure assessment Data sources for air pollutants: All pollutants information was obtained from the Qingyue Open Environmental Data Centre (http://data.epmap.org/), and data were obtained from the daily average mass values of six pollutants (PM2.5, PM10, SO2, NO2, CO and O3) at four monitoring stations in Baota District, Yan'an City (the mass concentration unit of CO is mg/m3, and the mass concentration of the remaining air pollutants is μg/m3). Meteorological data were obtained from Yan'an Meteorological Monitoring Station. The time span of pollutant data and meteorological data is from 1 January 2017 to 31 December 2020. Exposure assessment method: Based on the address of the pregnant woman's place of residence during pregnancy, the latitude and longitude of her place of residence were obtained from Gaode Map. The distance from the latitude and longitude of the place of residence to the latitude and longitude of the monitoring station was calculated, and the pollutant mass concentration of the nearest monitoring station to the place of residence was selected as the individual's exposure mass concentration. After dividing each exposure window according to the date of the last menstrual period of pregnant women, the mass concentration of pollutants in each exposure window was calculated (Fig. 1).Fig. 1Location of Yan'an city in the surrounding area Calculation of pollutants The exposure window is divided according to the date of the mother's last menstrual period into:E: the entire pregnancy (last menstrual period to the date of birth of the births), T1:early pregnancy (last menstrual period to the twelfth week of gestation), T2:mid-pregnancy (the thirteenth week of gestation to the twenty-seventh week of gestation), and T3:late pregnancy (the twenty-eighth week of gestation to the date of birth of the births). Considering that air pollutants have short-term effects in addition to long-term effects on birth, lagged and cumulative effects were calculated based on the date of admission to hospital. Matching of same-day and lag-day mass concentrations: Short-term same-day lagged mass concentrations of each pollutant on the day of the admission date and before the admission date are matched based on the maternal admission date. This exposure mass concentration is the daily average of the day, where the short-term lagged exposure dates are the day of the admission date (Lag0) and the days 1 (Lag1), 2 (Lag2), 3 (Lag3), 4 (Lag4), 5 (Lag5), 6 (Lag6), and 7 (Lag7) before the admission date. Calculation of mass concentration values for cumulative exposure: Matching the mass concentration of each pollutant prior to the date of admission to the date of maternal admission, the cumulative exposure mass concentration value is the average of all daily averages over the exposure period. The cumulative exposure period is 1 day (Lag-1), 2 days (Lag-2), 3 days (Lag-3), 4 days (Lag-4), 5 days (Lag-5), 6 days (Lag-6), and 7 days (Lag-7) before the day of admission. Statistical methods A database was created by EXCEL and data were analyzed by IBM SPSS 20. Normality test for pollutants was performed and pollutants were described using mean, median, standard deviation(SD), and interquartile range(IQR). The categorical variable information in the general information of pregnant women was statistically described using frequency (n) and composition ratio (%), and the difference between the case and control groups was compared using the chi-square test to determine confounders in turn. The correlation between air pollutants and temperature and relative humidity was analyzed, and pollutant mass concentrations were calculated for each exposure window using the R language. The long-term effects and lagged and cumulative effects of air pollutants and preterm birth were analyzed using conditional logisitic models, and after adjusting for each confounding factor, pollutants were introduced into the conditional logistic model with as a continuous variable with in, and ORs and 95% confidence intervals were calculated. After identifying the association between air pollutants and preterm birth, we adjusted for confounders and investigated the relationship between air pollutants and blood counts using multiple linear regression. Logistic regression was then used to explore the relationship between air pollutants and C-reactive protein at a test level of 0.05 (two-sided test). Correlation analysis Spearman's correlation analysis was used to analyze the correlation between air pollutants, blood markers, and meteorological factors (mean temperature throughout the pregnancy, relative humidity throughout the pregnancy, and mean wind speed throughout the pregnancy). Sensitivity analysis In order to verify the stability of the main model, the effect of each pollutant on preterm birth was analyzed separately. Then other confounding factors were added one by one to analyze the relationship between each pollutant and preterm delivery. Three sensitivity analyses were performed with meteorological factors as confounding factors: (1) Model 1: correlation analysis between a single pollutant and each blood index; (2) Model 2: add occupation, birth order and last menstrual season to analyze the correlation between each pollutant and preterm delivery in Model 1; and (3) Model 3: add menstrual cycle and complications diseases to Model 2. Statistical methods A database was created by EXCEL and data were analyzed by IBM SPSS 20. Normality test for pollutants was performed and pollutants were described using mean, median, standard deviation(SD), and interquartile range(IQR). The categorical variable information in the general information of pregnant women was statistically described using frequency (n) and composition ratio (%), and the difference between the case and control groups was compared using the chi-square test to determine confounders in turn. The correlation between air pollutants and temperature and relative humidity was analyzed, and pollutant mass concentrations were calculated for each exposure window using the R language. The long-term effects and lagged and cumulative effects of air pollutants and preterm birth were analyzed using conditional logisitic models, and after adjusting for each confounding factor, pollutants were introduced into the conditional logistic model with as a continuous variable with in, and ORs and 95% confidence intervals were calculated. After identifying the association between air pollutants and preterm birth, we adjusted for confounders and investigated the relationship between air pollutants and blood counts using multiple linear regression. Logistic regression was then used to explore the relationship between air pollutants and C-reactive protein at a test level of 0.05 (two-sided test). Correlation analysis Spearman's correlation analysis was used to analyze the correlation between air pollutants, blood markers, and meteorological factors (mean temperature throughout the pregnancy, relative humidity throughout the pregnancy, and mean wind speed throughout the pregnancy). Sensitivity analysis In order to verify the stability of the main model, the effect of each pollutant on preterm birth was analyzed separately. Then other confounding factors were added one by one to analyze the relationship between each pollutant and preterm delivery. Three sensitivity analyses were performed with meteorological factors as confounding factors: (1) Model 1: correlation analysis between a single pollutant and each blood index; (2) Model 2: add occupation, birth order and last menstrual season to analyze the correlation between each pollutant and preterm delivery in Model 1; and (3) Model 3: add menstrual cycle and complications diseases to Model 2.\",\n",
       " '11197292': '',\n",
       " '11197282': 'MethodsWe included 623 matched case–control pairs from the Cancer Prevention Study (CPS) cohorts. Pre-diagnosis blood samples were collected between 1998 and 2001 in the CPS-II Nutrition cohort and 2006 and 2013 in the CPS-3 cohort and were sent for metabolomics profiling simultaneously. Cancer-free controls at the time of case diagnosis were 1:1 matched to cases on date of birth, blood draw date, sex, and race/ethnicity. Odds ratios (ORs) and 95% confidence intervals (CIs) were estimated using conditional logistic regression, controlling for confounders. The Benjamini–Hochberg method was used to correct for multiple comparisons. Methods Study design and population Lung cancer cases and matched controls included in this analysis are participants from the CPS-II Nutrition cohort and ongoing CPS-3 cohort. At enrollment of the CPS-II Nutrition cohort in 1992–1993, participants completed a self-administered questionnaire that included anthropometric, demographic, dietary, lifestyle, and medical information. Follow-up questionnaires were sent to the cohort participants in 1997 and every other year thereafter to update exposures and to ascertain newly diagnosed cancers. A subset of 39,371 CPS-II Nutrition cohort participants provided a non-fasting blood samples between 1998 and 2001, and the information on demographic characteristics and other covariates in the analysis was assessed from the survey collected at blood draw or the 1999 survey. At enrollment of the CPS-3 cohort between 2006 and 2013, participants provided informed consent, a non-fasting blood sample and completed a brief enrollment survey on demographic characteristics and other covariates. Follow-up questionnaires were sent to active participants in 2015 and every 3 years to update exposures and ascertain newly diagnosed cancer cases. Detailed descriptions of the two cohorts can be found elsewhere [22, 23]. All aspects of the CPS-II Nutrition cohort (IRB00045780) and CPS-3 cohort (IRB00059007) were reviewed and approved by the Emory University Institutional Review Board. A total of 1913 lung cancer cases were identified in the CPS-II Nutrition cohort through June 2015 and 176 lung cancer cases were identified in the CPS-3 Cohort through December 2015. Cases in the CPS-II Nutrition cohort were first identified through self-report and then were verified with medical records, state cancer registry linkage, or linkage with the National Death Index (defined by ICD-10 codes C33 and C34, excluding histology codes ≥ 9590). Cases in the CPS-3 cohort were identified primarily through linkage with state cancer registries, and a small proportion were identified by self-report that were verified by medical records during tumor collection. We applied a series of exclusion criteria to include participants (Additional file 1: Fig. S1). As a result, 500 and 123 lung cancer cases from the CPS-II Nutrition cohort and CPS-3 cohort were included in the analysis, respectively. Controls who were cancer-free at the time of case diagnosis were matched 1:1 to cases on age at blood draw (± 6 months), sex, race/ethnicity, and blood draw date (± 30 days). Metabolomics profiling The pre-diagnosis blood samples collected from both cohorts were sent to Metabolon, Inc. (Durham, NC, USA) for untargeted metabolomics profiling simultaneously, using ultrahigh-performance liquid chromatography-tandem mass spectrometry (UPLC-MS/MS) analysis techniques. A detailed process was described elsewhere [24, 25] and in supplemental materials. A total of 1,401 metabolites were detected. After filtering metabolites that were unknown (n = 238), were missing technical intraclass correlation coefficient (ICC) (n = 34), with ICC < 50% (n = 201), and were undetectable in > 90% of samples (n = 41), 887 known metabolites were included in the statistical analysis with an average ICC of 84% (interquartile range (IQR): 77–94%) and the coefficient of variation (CV)% of 24% (IQR: 12–30%). Statistical analysis As metabolomics assessments were conducted simultaneously for cases and controls in both cohorts, we performed a pooled analysis. Metabolites were naturally log-transformed and auto-scaled to approximate normal distribution before formal analysis. Covariate data obtained in each cohort were harmonized. The characteristics between lung cancer cases and matched controls were compared using Student’s t-test for continuous variables and Pearson’s chi-squared test for categorical variables. For the primary pooled analysis, we applied conditional logistic regression to estimate the odds ratio (OR) and 95% confidence interval (CI) per one standard deviation increase in the naturally log-transformed level of each known metabolite with lung cancer risk. The statistical models were conditioned on the matching variables and controlled for the body mass index (BMI) group (underweight: < 18.5 kg/m2, normal weight: 18.5–25 kg/m2, overweight: 25–30 kg/m2, obese: ≥ 30 kg/m2), hours since last meal (continuous; to account for length of fasting), physical activity (continuous; hours/week), fruits and vegetables consumption (continuous; servings/week), smoking status (categorical: never, former, current, and unknown), and hormone use (categorical: not a current user, current user, not applicable, unknown). Physical activity estimates the average total hours per week of walking or exercise in the CPS-II Nutrition cohort, while it estimates the average hours per day during the past 2 years in the CPS-3 cohort. We harmonized the variables and converted them into hours per week. The covariates were selected based on the literature review and a Directed Acyclic Graph. We removed the observations with any missing data for continuous covariates. We assigned an unknown category for missing data for categorical covariates. If a case was removed, its matching control was removed simultaneously, and vice versa. For the primary pooled analysis, 116 case–control pairs were removed due to missing values in hours since the last meal, physical activity, and fruit and vegetable consumption for either case or its matching control (Additional file 1: Fig. S1). Benjamini–Hochberg approach was used to calculate false discovery rates (FDRs) to correct for multiple comparisons. Metabolites associated with lung cancer risk at FDR < 0.2 were deemed statistically significant. To gain more biological responses of lung cancer, we focused on metabolites associated with lung cancer risk at P < 0.05 (P-value from statistical models before multiple comparison corrections) and further described and summarized the pathways in which these metabolites were involved. We conducted an agglomerative hierarchical clustering analysis to group the lung cancer-associated metabolites (P < 0.05) based on their similarities. Pearson correlation was calculated between each pair of metabolites and then used for distance measure. Euclidean distance was computed between each pair of metabolites and returned the distance matrix. We then used the Ward clustering method to compute the similarity of the two clusters for merging [26]. The R package “pheatmap” was used for this analysis and result visualization. We further examined the associations stratified by sex and by years between blood draw and lung cancer diagnosis (< 3 years, ≥ 3 years) using conditional logistic regression with the same set of covariates but excluding hormone use for males. The goal of the stratified analysis by years since the blood draw was to identify metabolites that may potentially serve as early detection biomarkers of lung cancer. Additionally, we stratified the analysis by smoking status (never, ever), by stage (localized, regional, distant), and by histological subtypes (squamous cell carcinoma, adenocarcinoma) using unconditional logistic regression, adjusting for matching variables as well as BMI group, hours since last meal, physical activity, fruits and vegetables consumption, hormone use, and smoking status (only for stage- and subtype-stratified analyses). For stratified analyses by smoking status, stage, and subtype where the unconditional logistic regression was applied, matching factors were adjusted as covariates in the model. If a case was removed due to missing covariates, its matching control would not be removed if the control has completed the covariates information, and vice versa. The lung cancer stage was examined according to the Surveillance, Epidemiology and End Results (SEER) stage at diagnosis: localized (invasive tumors confined to the lung); regional (tumors that extend to adjacent tissue or regional lymph nodes); distant (tumors are metastasized). The lung cancer histological subtype was categorized using ICD-O-3 morphology codes [27]. The morphology codes for each subtype can be found in supplementary materials. P for interaction was calculated using the likelihood ratio test. To test heterogeneity by stage and by subtype, we used the “eh_test_subtype” function in the R package “riskclustr” [28]. This function is designed for the test of etiologic heterogeneity across disease subtypes in the context of the case–control study. P-heterogeneity < 0.05 was considered statistically significant. To examine the robustness of the results, we conducted a series of sensitivity analyses: (1) we recategorized the smoking variables into 7 categories (never, current smoker for < 50 years, current smoker for ≥ 50 years, former smoker quit < 10 years ago, former smoker quit 10–20 years ago, former smoker quit ≥ 20 years ago, unknown) based on smoking status and duration of time and adjusted it in the main analysis; (2) we further adjusted for cohort (CPS-II Nutrition, CPS-3) in the analysis to evaluate if any differences between cohorts (e.g., age of blood samples) would affect the main analysis results; (3) we also examined the associations stratified by years between blood draw and lung cancer diagnosis (< 5 years, ≥ 5 years). All analyses were conducted using R (version 4.1.0.).',\n",
       " '11197204': 'MethodsThe study utilised 6 publicly available bulk and single-cell transcriptomic datasets from human and mice studies downloaded from Gene Expression Omnibus (GEO). Machine learning models were employed to model and statistically test datasets for conserved gene expression profiles. Identified genes were validated in OA tissues from obese and healthy weight individuals using quantitative PCR method (N = 38). Obese and healthy-weight patients were categorised by BMI > 30 and BMI between 18 and 24.9 respectively. Informed consent was obtained from all study participants who were scheduled to undergo elective arthroplasty. Methods Study recruitment UK National Research Ethics Committee (NRES 16/SS/0172) granted ethical approval for post-operative collection of joint tissue from consenting patients with OA. Study recruitment was coordinated by research nurses at the Royal Orthopaedic Hospital, Birmingham (United Kingdom) and Russell’s Hall Hospital, Dudley (United Kingdom). Differential gene expression analysis Genes are commonly assessed according to their fold change, which represents the extent of expression variations, and by their respective p-values, which measure the statistical significance of such variations. Genes with significant fold changes (fold changes ± 1.5) and low p-values (p < 0.05) are considered statistically significant differentially expressed. Multiple testing correction was performed to control for false positives that may arise due to testing multiple genes simultaneously, and multiple testing correction methods (BH false discovery rate correction) were applied to adjust p-values (adj p < 0.05). Datasets All the datasets were obtained from Gene Expression Omnibus (GEO) on 29th October 2022. Those data sets are from both human and mouse samples. A detailed summary of each dataset can be found in Table 1.Table 1List of the public data sets, organism and tissue sources used in this studyGEO dataOrganismType of datasetNumber of participantsTissue sourceLeanObeseTotalGSE24883HumanArray323264Subcutaneous and visceral adipose tissueGSE59034HumanArray161632Subcutaneous and visceral adipose tissueGSE49195MouseArray6612Lean and obese LiverGSE39375MouseArray5510Lean and obese LiverGSE152815HumanSingle Cell RNA sequencing448Hand, hip, knee, and foot JointsGSE219027HumanBulk RNA sequencing242448Hand, hip, knee, and foot Joints Patients and tissue samples Obese and lean-weight patients, categorised by BMI > 30 and BMI between 18 and 24.9 respectively, were recruited for the study. These patients were scheduled to undergo elective arthroplasty for OA and synovium was collected from hip or knee joints (N = 38). Ethics approval was provided by the UK National Research Ethics Committee (approval no. 14/ES/1044), and informed consent was obtained from all patients. The characteristics of study participants are summarised in Table 2.Table 2Summary of participant characteristics. Data displayed as mean ± standard deviationLeanN = 16ObeseN = 22P-valueAge in years69.1 ± 2.666.7 ± 1.80.4382Gender (Number of female participats, (%))12 (75%)12 (55%)0.2071Hip: Knee Ratio (Number of hip joints, (%))12 (75%)12 (55%)0.0717BMI22.8 ± 0.435.9 ± 1.3 < 0.0001Waist: Hip Ratio0.8 ± 0.020.9 ± 0.010.0007N Number of participants Quantitative real-time PCR validations of identified genes Total RNA was isolated from synovium using TRIzol (Life Technologies). Primers for individual transcripts of interest (see Supplementary Table 1) were either predesigned TaqMan™ Gene Expression Assay (FAM) from Life Technologies Ltd or designed using NCBI’s Primer-BLAST tool and Easy Oligos were ordered from Merck Life Science UK Ltd. PCR was performed from total RNA in a one-step reaction using either iTaq Universal One-Step or iTaq™ Universal SYBR® Green One-Step Kit from BioRad. Relative expression was determined using the ΔΔCt method, followed by normalization of values to those of 18S and GAPDH. Statistical and machine learning methods Multiple statistical and machine learning models were used to identify genes differentiating between obesity and lean samples (DEG summary in Supplementary Table 1). Unsupervised Principal Component Analysis (PCA), a dimensionality reduction technique, was used in machine learning and data analysis. PCA seeks to preserve the underlying structure and relationships in the data while reducing its dimensionality, making it easier to visualize and analyse. Logistic regression was used to evaluate how well different classes could be distinguished. To estimate sample size, an inhouse developed tool known as Tools was used [20]. qPCR data was plotted using GraphPad PRISM with unpaired student’s t-test. Pathway enrichment analysis Pathway enrichment analysis was performed using the Enrichr tool to determine functional mechanisms informed by differentially expressed genes derived from the datasets analysed. This approach identifies biological pathways that show enrichment in a gene list beyond what would be expected by random chance. We have used ShinyGO [21] and Enrichr [22] for this analysis. The analysis workflow is illustrated in Fig. 1. Fig. 1Overview of the gene target identification workflow using public and internal RNA sequencing data Statistical and machine learning methods Multiple statistical and machine learning models were used to identify genes differentiating between obesity and lean samples (DEG summary in Supplementary Table 1). Unsupervised Principal Component Analysis (PCA), a dimensionality reduction technique, was used in machine learning and data analysis. PCA seeks to preserve the underlying structure and relationships in the data while reducing its dimensionality, making it easier to visualize and analyse. Logistic regression was used to evaluate how well different classes could be distinguished. To estimate sample size, an inhouse developed tool known as Tools was used [20]. qPCR data was plotted using GraphPad PRISM with unpaired student’s t-test.',\n",
       " '11197185': 'MethodsThe current study aimed at analyzing and comparing the microbial profile of Aedes albopictus saliva and midgut as well as assessing the impact of Zika virus (ZIKV) infection on the midgut and saliva microbial composition. Colony-reared Ae. albopictus strains were either exposed to ZIKV infectious or noninfectious bloodmeal. At 14 ays postinfection, the 16S V3–V4 hypervariable rRNA region was amplified from midgut and saliva samples and sequenced on an Illumina MiSeq platform. The relative abundance and diversity of midgut and saliva microbial taxa were assessed. Methods Virus The ZIKV strain utilized in this study was HND (2016-19563, GenBank accession no. KX906952). The Zika virus isolate used in this study was obtained from an infected patient serum. In the lab, virus stocks were created by passaging three times on Vero cell culture (ATCC) and once on C636 cell culture (ATCC). Stocks were frozen at −70 °C before use. Insect sampling and sample preparation Ae. albopictus mosquitoes were collected from Suffolk County, New York State (NYS) in 2015 (kindly provided by I. Rochlin) and colonized at the New York State Department of Health (NYSDOH) Arbovirus Laboratory. The F15 generation Ae. albopictus eggs were vacuum hatched. The larvae that emerged from the hatched eggs were maintained in plastic rectangular flat containers (35.6 cm length × 27.9 cm width × 8.3 cm height; Sterilite, catalog no. 1963) at a density of one larva per 5 ml of dechlorinated water and reared at 60% relative humidity and a light–dark (LD) photoperiod of 14:10 h. The larvae were fed 1.25 mg/larvae of Tetra Pond Koi growth feed for first and second instar larvae and 2.5 mg/larvae for third and fourth instar larvae [37]. Upon emergence, the male and female adults were transferred to 3.8-l containers and housed together for 8 days at 28 °C at a relative humidity of 60% while being provided with sugar and water ad libitum [37]. To stimulate blood feeding, females were starved of water and sugar for 12 h before infectious blood meal according to our insectary laboratory standard operating procedures. The blood meal consisted of either 1:1 dilution of defibrinated sheep blood plus 2.5% sucrose, sodium bicarbonate (to adjust pH to 8.0) and virus, or a non-infectious blood meal containing a final concentration of 2.5% sucrose solution. Infectious blood meals contained 8.3 log10 PFU/ml ZIKV HND [38]. The female mosquitoes were exposed to blood meals in a 37 °C preheated Hemotek membrane feeding system (Discovery Workshops, Accrington, UK) with a porcine sausage casing membrane. After an hour, the mosquitoes were anesthetized with CO2 and immobilized on a prechilled tray connected to 100% CO2 Engorged females were separated and placed in separate 0.6 l cardboard cartons (30 individuals per carton). Blood-fed females were maintained on a 10% sucrose solution provided ad libitum. At 14 days postinfection (dpi), the female mosquitoes were immobilized using triethylamine (Sigma Aldrich, St. Louis, MO, USA). The mosquitoes were surface sterilized by dipping twice in 70% ethanol. To examine for ZIKV dissemination, the legs were removed from the mosquitoes and placed in individual tubes. Saliva was collected under sterile conditions by inserting the proboscis of the surface-sterilized female into a capillary tube containing ~ 20 µl filter sterilized fetal bovine serum plus 50% sucrose 1:1 for 30 min before subsequently ejecting the mixture into filter-sterilized 125 µl Mosquito Diluent [MD; 20% heat-inactivated fetal bovine serum in Dulbecco phosphate-buffered saline plus 50 µg/ml penicillin/streptomycin, 50 µg/ml gentamicin and 2 µg/ml Fungizone (Sigma Aldrich, St. Louis, MO, USA)]. As described in [39], briefly, to dissect the salivary glands, the tip of the abdomen was removed with a sharp cut using a dissecting probe. Placing a probe in the thorax, the head was removed gently by pulling away from the thorax. The salivary glands were then detached from the head using the probe, rinsed twice in sterilized phosphate-buffered Saline (PBS) (Sigma Aldrich, St. Louis, MO, USA), and transferred to a sterile microcentrifuge tube and stored at −80 °C. The midgut was pulled from the tip of the abdomen and rinsed twice in sterile PBS before being transferred to sterile microcentrifuge tubes and stored at −80 °C until tested. The individual mosquito carcasses were then transferred into individual tubes sterile microcentrifuge tubes. All samples were held at −80 °C until assayed. Sequencing and analysis of microbiome of Aedes mosquito saliva and midguts A ZIKV-specific quantitative PCR assay that targets the NS1 region [40] was utilized to obtain viral titer from the legs and saliva as described by [38]. The individuals whose midguts and saliva samples were infected with the virus were identified. ZIKV-infected saliva and midgut samples as well as those exposed to a naïve blood meal were used for downstream analysis. We experienced challenges in obtaining enough genomic DNA material from singleton saliva samples and hence to increase the amount of genomic material for microbiome study, we adopted a whole transcriptome amplification approach. RNA was isolated from the saliva and midgut samples using Trizol (Thermo Fisher Scientific, Waltham, MA, USA). The saliva and midgut RNA samples were reverse transcribed, followed by whole transcriptome amplification according to the REPLI-g whole transcriptome amplification (WTA) single-cell kit protocol (Qiagen, Hilden, Germany). The resulting amplified complementary DNA (cDNA) was diluted 1:100. Thereafter, the 16S V3–V4 hypervariable region was amplified using an Illumina barcoded 16S primer set [16S_341F (TCG TCG GCA GCG TCA GAT GTG TAT AAG AGA CAG CCT ACG GGN GGC WGC AG) and 16S_805R(GTC TCG TGG GCT CGG AGA TCT GTA TAA GAG ACA GGA CTA CHV GGG TAT CTA ATC C)] [36]. PCR reactions were carried out in a total volume of 50 µl, 5 µM each of 16S ribosomal RNA (rRNA) primer pair, 2 µl WTA cDNA, 8 µl deionized filter-sterilized water, and 36 µl AccuStart II PCR supermix (Quantabio, Beverly, MA, USA). We included two negative controls at the PCR amplification step: water and nontemplate control. In addition, a positive spike at the sequencing step. A fragment size of ~ 460 bp of each sample was submitted to the Wadsworth Center Applied Genomics Core for sequencing together with the negative controls. Automated cluster generation and paired-end sequencing (250-bp reads) was performed on the Illumina MiSeq 500 cycle. A total of 30 samples [five infected midguts (INF MG), six noninfected midguts (NINF MG), ten infected saliva (INF SAL), six noninfected saliva (NINF SAL), one negative control (NC), one nontemplate control (NTC), and one positive spike (PC)] were included in this study. Analysis of the data was carried out on QIITA (https://qiita.ucsd.edu/) and MicrobiomeAnalyst [41]. In summary, the command split libraries FASTQ was used to convert the de-multiplexed FASTQ files to the format used by QIITA. This was followed by clustering of the sequences into operational taxonomic units (OTUs), which was done by the utilization of a closed-reference OTU picking based on the GreenGenes 16S reference database. The choice of closed reference was due to its fully parallelizable process, which makes it a computationally efficient process. We neither elected to utilize open reference nor de novo OTU picking due to the computationally complex strategies that make them unsuitable for large data sets. Open-reference OTU picking reduces read loss by de novo clustering reads that do not match the reference sequences; some of the steps of the workflow run serially and hence slows down the process. De novo OTU picking, on the other hand, clusters reads against one another without an external reference collection, is computationally complex, results in large data set loss, and can reduce quantification accuracy while reducing the potential to detect rare taxa [42]. A 97% sequence identity threshold was utilized to assign taxonomy to the sequence. A BIOM-formatted OTU table was subsequently generated. To estimate alpha diversity metrics, we adjusted for differences in library sizes across samples using rarefaction and generated rarefaction curves to assess whether the sequencing depth was sufficient. Rarefaction involves picking a specified number of samples equal to or less than the number of samples in the smallest sample followed by a random removal of reads from larger samples until the number of remaining samples is equal to this threshold [43]. The reads have been deposited in NCBI GenBank Short Read Archives. We anticipated the presence of rare taxa in our microbial community. To accommodate the rare taxa while estimating diversity, we utilized Chao1 diversity measure at the family taxonomic level to assess species richness of INF and NINF saliva and INF and NINF MG. Statistical significance was tested using a t-test assuming a normal distribution of values. Chao1 index was used to estimate alpha diversity because in estimating the diversity it makes inferences about rare or difficult-to-detect species [44]. We compared these results with other ecological indices used to test for alpha diversity such as the Shannon diversity and Simpson index. The Shannon diversity index considers both the richness and evenness of the microbial species present. It calculates this by utilizing the proportion of species i relative to the total number of species (pi) and multiplying by the natural logarithm of the proportion (lnpi) [45]. The Simpson Index tests for species dominance by considering the total number of species present as well as the relative abundance of each species. In this diversity index, species richness and evenness are directly proportional to diversity [46]. To compare the microbial community composition between the global INF and NINF saliva and INF and NINF MG, beta diversity was measured utilizing the Bray–Curtis dissimilarity statistic at the family taxonomic level [47]. The beta diversity measurement utilized the permutational analysis of variance (PERMANOVA) test to estimate the differences in microbial communities. PERMANOVA is a nonparametric multivariate statistical permutational test measured through a geometric partitioning of multivariate variation of a dissimilarity measure according to a given ANOVA design [48]. The distance matrix generated after comparing each sample to the other was visualized for the dissimilarity between samples using the principal coordinate analysis ordination method. Biomarker analysis Random forest algorithm [49] within Microbiome Analyst [41] was utilized to identify taxa associated with the different infection statuses of saliva and midgut. Random forest algorithm [49] is a directed classification algorithm of trees generated by bootstrapping samples while training data and random feature selection in tree induction. It is an ensemble of unpruned classification or regression trees trained with the bagging method [50]. The default setting of the number of trees to grow and the number of predictors to try was applied (500 and 7, respectively) with the randomness setting left on. Further, we tried to adjust the parameters to different settings, but we did not observe any improvement in the model performance. We assessed the Out-of-Bag (OOB) error rate of the model.',\n",
       " '11197061': '',\n",
       " '11196966': 'MethodsCRC organoids were implanted into the cecal walls of wild type mice, and animals were screened for liver metastasis. At the time of metastasis, 1/3 partial hepatectomy was performed and the tumor burden was assessed longitudinally using MRI. After euthanasia, different tissues were analyzed for immunological and transcriptional changes using FACS, qPCR, RNA sequencing, and immunohistochemistry. 2 Methods 2.1 Organoid culture APTAK organoids were kindly gifted by Florian Greten, Georg Speyer Haus, Frankfurt. These cells are murine colorectal carcinoma cells holding the following mutational features: Deletions in the Apc-, Trp53- and Tgfbr2-gene, a myristoylated Akt-1 with constitutively activated signaling and a Kras G12D-mutation (16). The APTAK organoids were maintained in Advanced DMEM-F12 (Gibco/Thermo Fisher Scientific, Waltham, MA, USA) supplemented with penicillin (100 U/ml)/streptomycin (100 µg/ml) (Gibco/Thermo Fisher Scientific), HEPES 10 mmol/L (Invitrogen/Thermo Fisher Scientific), Glutamax 2 mM (Invitrogen/Thermo Fisher Scientific), N2 1× (Gibco/Thermo Fisher Scientific), B27 1× (Gibco/Thermo Fisher Scientific), and N-acetylcysteine 1 mmol/L (Sigma-Aldrich, St. Louis, MO, USA) and supplemented with hygromycin (200 µg/ml, Invitrogen/Thermo Fisher Scientific) and puromycin (2 µg/ml, InvivoGen, San Diego, CA, USA) (hereafter referred to as basal medium). The organoids were split every 3–5 days at a 1:10 ratio and cryoconserved for long-term storage. 2.2 Animal studies All animal studies were performed using 8–12 weeks old C57Bl6/J-mice (purchased from Charles River, Wilmington, MA, USA) at the pathogen-free barrier facility of the CEMT, University Medical Center Freiburg. The experiments were planned and carried out in accordance with the institutional guidelines and all experiments were approved by the regional board (Regierungspräsidium Freiburg, G21/001). 2.3 Orthotopic organoid transplantation mouse model We adapted the subserosal orthotopic organoid implantation technique described by Fumagalli et al. (17). APTAK organoids cultured in Cultrex RGF basement membrane extract (BME) Type 2 (R&D Systems, Minneapolis, MN, USA) were dissociated into single cells by mechanical disruption and washed with cold PBS. After determining the cell number with the cell counter TC20 (BioRad, Hercules, CA, USA), the appropriate cell number was resuspended in collagen I/5x neutralization buffer (vol/vol-ratio 1:4), and collagen domes of 100.000 cells/10 µl were plated on a 6-well plate. After 30 min polymerization at 37°C, basal medium was added and cells were incubated over night until implantation. Mice analgesia was performed with intraperitoneal (i.p.) injection of 0.1 mg/kg bodyweight (BW) buprenorphine (Temgesic, Schering-Plough, Kenilworth, NJ, USA). Anesthesia was induced by 2.5 Vol% Isoflurane (Isoflurane Piramal, Piramal, Mumbai, India) with the SomnoSuite® Low-Flow Anesthesia System (Kent Scientific Corporation, Torrington, CT, USA). The unconscious mice were positioned on a heating pad with all legs fixed and the abdomen was shaved while anesthesia was maintained by continuous application of isoflurane via mouthpiece. After disinfection of the skin, a median laparotomy of approximately 1 cm was performed on the lower abdomen. The cecum was mobilized using two cotton swabs and placed on a wet gauze. Under the microscope, a 2 mm incision of the serosal layer was made and a deep subserosal pocket was created in which the organoid-collagen-graft was fully placed. After covering the graft with the serosal layer, an anti-adhesion layer (Seprafilm, Baxter, Deerfield, IL, USA) was placed on top. The cecum was then replaced in the abdominal cavity and the abdominal wall and skin were closed in two layers using 6–0 Vicryl (Ethicon/Johnson&Johnson, New Brunswick, NJ, USA) suture. The duration of the surgical procedure was approximately 10–15 minutes. 2.4 Magnetic resonance imaging MR imaging was performed using a 9.4 tesla small bore animal scanner (BioSpec 94/21, Bruker Biospin, Karlsruhe Germany) and a dedicated mouse quadrature-resonator (Bruker). The mice were anesthetized under spontaneous breathing conditions using Isoflurane. Heart and respiration rates were continuously monitored and maintained at a constant level. Gating was used to reduce motion and blood flow artifacts during the scan. The MRI protocol consisted of a localizer and a T2-weighted spin echo RARE (Rapid Acquisition with Relaxation Enhancement) sequence. This sequence was performed to delineate the tumor and eventual metastasis from the surrounding healthy tissue. The RARE sequence in axial orientation featured a field-of-view (FOV) of 28 mm (1), a matrix size of 280 x 280 pixels, and an in-plane resolution of 100 x 100 µm (1). The slice thickness was 0.75 mm (TR/TEeff/FA: 300 ms/36 ms/180°). The number of slices was adjusted to the measured volume (on average 30) to ensure complete coverage of the abdomen. The Bruker ParaVision 6.0.1 software was used for imaging and images were analyzed using the NORA software supplied by the Medical Center of the University of Freiburg. The volumes of all hepatic lobes and metastases were assessed individually by manually marking the respective lobes/LMs as regions of interest (ROIs) in each section. 2.5 Ultrasound During isoflurane anesthesia mice were placed on a heating pad and the abdomen was shaved. Prewarmed ultrasound gel was applied to the abdomen and a VEVO 3100 (Fujifilm/VisualSonics, Toronto, Canada) ultrasound together with a small animal transducer was used for ultrasound imaging. 2.6 Partial hepatectomy For distribution among the control-, SHAM-, and pHx-groups, we assessed the preoperative hepatic tumor load using MRI. Animals displaying an approximately similar number and volume of LM in the imaging were allocated equally among the different protocols to reduce the influence of cage-specific phenotypes. After general analgesia and anesthesia with buprenorphine and isoflurane as described above, mice were shaved and placed on a heating pad. After disinfection, a median laparotomy of approximately 2 cm, starting at the xiphoid, was performed and retractors were placed on both sides of the abdominal wall and under the xiphoid. The falciforme ligament and the membrane between the medial and left hepatic lobes were dissected. A 3–0 Vicryl (Ethicon/Johnson&Johnson) suture was placed around the base of the medial lobe and tightened with 3 knots, after which the ischemic lobe was cut approximately 0.1 cm distal to the suture ( Supplementary Figure 1A ). The resected lobe was placed in cooled PBS Buffer on ice until further analysis. The abdominal cavity was flushed with prewarmed saline, and the abdominal wall and skin were closed using 6–0 Vicryl (Ethicon/Johnson&Johnson) stitches. The pHx procedure took approximately 15 min. SHAM mice underwent laparotomy, including dissection of the falciforme ligament and the membrane between the medial and left lobes without resection of the lobe. Control animals received general anesthesia without any surgical procedures. 2.7 Tumor dissociation Animals were euthanized by cervical dislocation. Subsequently, the abdomen was opened and each lobe of the liver was resected directly at the vena cava. The lobes were weighed and all visible liver metastases were dissected while the organ was sliced. The entire cecum was resected and the tumor tissue was separated from the healthy intestine. We collected samples for RNA isolation (stored in RNAprotect buffer (Qiagen, Netherlands)) and for protein isolation from all tissues mentioned. Healthy liver tissue was minced and dissociated using the Liver Dissociation Kit, mouse (Miltenyi Biotec, Bergisch Gladbach, Germany) according to the manufacturer’s protocol. The primary tumor and LM were minced into <1 mm pieces and dissected using the Tumor Dissociation Kit, mouse (Miltenyi Biotec) according to the manufacturer’s instructions. The single cell suspension was filtered through a 100 µm cell strainer and ACK lysis buffer (Gibco/Thermo Fisher Scientific) was added to lyse erythrocytes during a 1.5 minutes incubation. 2.8 Flow cytometry Cells were transferred to 100 µl FACS buffer (PBS + 0,5% BSA), incubated with TruStain FcX PLUS (anti-mouse CD16/32) antibody (BioLegend, San Diego, CA, USA) for 15 min and then stained with the respective fluorescent antibodies for 60 min. Cells were then washed with FACS buffer (for extracellular staining) and fixed with 1x Fixation/Permeabilization buffer for 60 min. For intracellular staining, we used the eBioscience™ Foxp3/Transcription Factor Staining Buffer Set (Invitrogen/Thermo Fisher Scientific) according to the manufacturer’s instructions. All FACS antibodies used were supplied by BioLegend and used undiluted with 1 µl/100 µl cell suspension. FACS was performed using Beckman Coulter Gallios (Beckman Coulter, Brea, CA, USA) or BD LSRFortessa (Becton Dickinson, Franklin Lakes, NJ, USA). Data were analyzed using Kaluza Software (Beckman Coulter). Cellular aggregates were excluded using SSC-W. 2.9 T cell stimulation Equal amounts of single cells from the tumor/liver dissociation were seeded in RPMI-Medium (+ 1% Pen/Strep + 1% HEPES + 5 µl beta-mercaptoethanol) on a 12-well plate overnight. The cells were then stimulated by adding Cell Activation Cocktail (BioLegend) for 3 hours, followed by addition of Brefeldin A (BioLegend) for 2 hours. The cells were then harvested and stained for FACS analysis, as described above. 2.10 RNA isolation and cDNA synthesis Tissue samples for RNA isolation were stored in RNAprotect Buffer. The tissue was shredded in 350 µl RLT buffer (Qiagen, Venlo, Netherlands) + 1% beta-mercaptoethanol. Further RNA isolation was carried out using the RNeasy Mini Kit (Qiagen) according to the manufacturer’s instructions. RNA concentrations were measured using a NanoDrop™ spectrophotometer (Thermo Fisher Scientific). For cDNA synthesis RNA was transcribed using ProtoScript II First Strand cDNA Synthesis Kit (New England Biolabs, Ipswich, MA USA) according to the manufacturer’s instructions. cDNA was finally diluted to 3 ng/µl for qPCR experiments. 2.11 DNA isolation For quantification of bacterial DNA, we isolated DNA from tissue samples using the DNeasy Blood & Tissue Kit (Qiagen) according to the manufacturer’s instructions, and concentrations were assessed using a NanoDrop™ spectrophotometer (Thermo Fisher Scientific). 2.12 Quantitative real-time PCR Quantitative Real-Time PCR was performed using QuantiTect SYBR Green RT PCR mastermix (Qiagen) with 7.5 ng template cDNA and 5 pmol of each primer (purchased from Eurofins Genomics, Ebersberg, Germany; primer sequences supplied the supplements). qPCR was carried out on a Roche LightCycler 480 (Roche, Basel, Switzerland) with each sample-gene combination in triplicates. The protocol consisted of a 10 min activation step at 95°C followed by 50 cycles of 45 s amplification at 60°C and inactivation at 95°C for 15 s. 18S was used as a housekeeping gene, Cp values were calculated using the LightCycler480 software (Roche), and the relative expression of target genes was calculated by comparative method after normalization to 18S-expression. 2.13 Bacterial 16S qPCR In order to quantify bacterial infiltration, we performed qPCR with amplification of the bacterial V6 region using five different primers (see Supplements). Each sample was measured in triplicates using the QuantiFast SYBR® Green PCR Kit (Qiagen). 100 ng DNA was mixed with 500 nM primers. By diluting the DNA of Bifidobacterium actinocoloniiforme (DSMZ-Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, DSM22766) in 10-fold series we generated a standard curve using the Threshold cycle (Ct) values. To quantify the sample bacterial DNA, we correlated the average Ct-values to the standard curve of log values from the Bifidobacterium actinocoloniforme. 2.14 Bulk RNA sequencing RNA sequencing was performed at Genewiz, Germany, using the standard Illumina protocol to create raw sequence files (.fastq files). Poly(A) mRNA was used to prepare cDNA libraries, which were then sequenced as 2x 150 bp paired end using the Illumina NovaSeq instrument (Illumina, San Diego, CA, USA). 2.15 RNA seq analysis Raw sequencing files were uploaded to the galaxy web platform (https://usegalaxy.eu) (18) and quality was assessed using FASTQC. Owing to good quality and no adaptor content, trimming was not necessary. Data were mapped to the murine reference genome (GRCm39) provided by GENCODE (www.gencodegenes.org, downloaded September 2021) using the STAR aligner (19) in standard settings for paired sequencing data. Reads were assigned to counts using featureCounts software (20) and the gene annotations also provided by GENCODEs (downloaded September 2021). Paired-end settings were used for featureCounts software, and a quality score of >10 was required for counting. Data were imported in R 4.0.2 (https://www.r-project.org), and differentially expressed gene analyses were conducted using DESeq2 (21). A batch effect was removed using the limma::removeBatchEffect() function (22). Differentially expressed genes were defined as padj <0.05 and abs(log2 FC) >1. The Ensembl gene ID was converted to MGI symbols using the biomaRt package (23). The fgsea package (24) conducted Gene Set Enrichment Analysis (GSEA) by ranking all genes with at least >10 counts according to the log2 FC, which were previously shrunken by the apeglm logarithm (25) before. Gene sets were accessed from MsigDB (26) and downloaded on the 26th of May 2023. For visualization, ggplot2 (27), ggraph and ComplexHeatmaps (28) were used. 2.16 Protein isolation Tissue samples were shredded in RIPA buffer containing phosphatase and proteinase inhibitors (Phosstop (Roche), protease inhibitor cocktail, sodium orthovanadate, and PMSF (Santa Cruz, Santa Cruz, CA, USA) on ice. Three cycles of freezing and thawing were applied using liquid nitrogen, and the suspension was centrifuged at maximum speed for 10 min. The supernatant containing the protein lysate was transferred to a new tube. Protein concentration was measured using a Pierce BCA-Kit (Thermo Fisher Scientific) and an Azure Ao Microplate Reader (Azure Biosystems, Dublin, CA; USA). 2.17 Western blot Protein samples were aliquoted with 4x Laemmli Buffer (BioRad) and denatured at 95°C for 3 min. An equal amount of protein was loaded on a 12–20% Mini-Protean TGX precast gel (BioRad) together with a color-coded prestained broad range protein ladder (Cell Signaling Technology, Danvers, MA, USA). After SDS-electrophoresis, the proteins were blotted onto a PVDF membrane using the trans-blot turbo transfer system (BioRad) according to the manufacturer’s instructions. The membrane was then blocked with 5% BSA/TBST for 60 min at RT and incubated with the respective primary (Anti-S100A8, Anti-MMP9, Anti-Claudin-2, Anti-Occludin, and Anti-ZO-1) and secondary (anti-rabbit IgG) antibodies according to the manufacturer’s instructions (a detailed description of the antibodies used is supplied in the Supplements). For detection, the Immobilon Crescendo Western HRP substrate (Merck Millipore, Burlington, MA, USA) was added to the membrane and imaging was carried out using a ChemiDoc BioRad imager (BioRad). 2.18 Immunohistochemistry Tissue samples were fixed in 4% PFA for 24 hours, dehydrated and embedded in paraffin. Sections of 3 µm were cut, rehydrated, and heat induced epitope retrieval was carried out with citrate or TE-Buffer (Zytomed Systems, Berlin, Germany). Sections were then blocked with 2.5% BSA and avidin-biotin-block (Zytomed Systems) according to the manufacturer’s instructions. Sections were then incubated with the primary antibody overnight and biotinylated secondary antibody for 1 h according to the manufacturer’s instructions (see a detailed description of the antibodies used in Supplements). Positive staining was visualized using DAB (Zytomed Systems), and the sections were counterstained with hematoxylin before dehydration and mounting with ROTI® Histokitt (Carl Roth, Karlsruhe, Germany). Slides were digitalized using a Zeiss Axio Scan Z.1 Microscope Slide Scanner (Zeiss, Jena, Germany). 2.19 Immunofluorescence Slides were treated until secondary antibody incubation as described in the Immunohistochemistry section, and then incubated with the Vector® TrueVIEW® Autofluorescence Quenching Kit (Biozol, Eching, Germany) to reduce autofluorescence. After washing, nuclear staining was performed using NucBlue™ Fixed Cell Stain ReadyProbes™ (Thermo Fisher Scientific). Sections were then mounted with VECTASHIELD Vibrance Antifade Mounting Medium (Biozol) and scanned using a Zeiss Axio Scan Z.1 Microscope Slide Scanner (Zeiss). 2.20 Statistics For statistical analysis, we used a one-way analysis of variance (ANOVA) test with Tukey’s post-hoc analysis when comparing multiple samples or a two-tailed Students-t-test for two variables. Statistical significance was set at p-values <0.05, and significance levels are represented in the graphic as follows: *** = p<0.001, ** = p<0.01, * = p<0.05, n.s. = not significant. All statistical analyses were performed using GraphPad Prism.',\n",
       " '11196847': '2 Materials and methods 2.1 Sample collection The macrofungal strain fruiting body was collected from the Neelkantha forest, Rishikesh, Uttarakhand, situated at an altitude of 6,596 meters, latitude 30°43′48′′N, and longitude 79°24′20′′E. 2.2 Establishment of tissue culture and submerged cultivation of Trametes sp. Fruiting bodies were cleaned, and the inner mass of the mushroom was removed with a sterilized knife and cultivated for 14 days at 20°C in a suitable potato dextrose agar (PDA) medium. A pure culture was obtained and sustained after additional tissue culture was established and multiple subcultures were carried out to purify the culture. After generating further sub-cultures from the pure culture medium, the biomass was grown in a liquid medium (potato dextrose broth; PDB) in a 250-mL volumetric flask and incubated for 14 days at 20°C in the dark. This step was done in batches. Following that, biomass was filtered and dried at 50°C for 2 h. Using a pestle and mortar, the dried biomass was ground into a fine powder. The resulting powder was then used and prepared for analysis. The pure culture was stored in a suitable potato dextrose agar medium at 4°C and was subcultured every 30 days onto the fresh medium. 2.3 Amplification of the LSU F/R-rDNA fragment and phylogenetic analysis Genomic DNA was extracted from cultured samples by a simple lysis protocol, as described in Karn et al. (2011). In total, 125 ηg of extracted DNA is used for amplification, along with 10 pM of each primer. LSU F/R-rDNA fragment was amplified using the ABI 3130 Genetic Analyzer with the following set of primers: forward primer: 5’-TCCTGAGGGAAACTTCG-3’ and reverse primer: 5’-ACCCGCTGAACTTAAGC-3’. The reaction mixture of PCR contained 10x Taq DNA polymerase Assay Buffer, 2.5 mM each dNTP, 3.2 mM MgCl2, and 3 U/ml Taq DNA Polymerase in a final volume of 30 ml sterile MQ water. The PCR was performed with an initial denaturation at 94°C for 3 min, followed by 30 cycles of 94°C for 1 min, 50°C for 1 min, and 72°C for 2 min, and a final extension of 72°C for 7 min. The amplification product was gel-purified using ABI 3130xl gel sequencing. The PCR product was sequenced bi-directionally. The sequence data were aligned and compared against the available DNA sequences from type strains in GenBank (http://www.ncbi.nlm.nih.gov/) using the BLAST sequence match tool and analyzed to identify the mushroom and its closest neighbors joining by making a phylogenetic tree using the software Phylogenetic Tree Builder (Bruno et al., 2000). 2.4 Preparation of mycelial biomass extract and broth extract Ethanol (80%, v/v) was used as an extracting solvent for obtaining MBE and BE. The dry biomass (10g) was ground and mixed with 100 ml of 80% ethanol, then left in a laboratory shaker at 25°C and 140 rpm for 24 h. After that, extracts were separated from the mycelium biomass by centrifugation. The process was then carried out once more after the solid had been redissolved in ethanol. Next, Whatman paper No. 4 was used to combine and filter the liquid. In a vacuum evaporator, the filtrate was concentrated, and the thick concentrated extract was collected and used for phytochemical and polysaccharide quantification research. 2.5 Phytochemical analysis 2.5.1 Total protein analysis The total protein content in MBE and BE was determined using Lowry’s assay. An aliquot of 1 mL of extract is treated with 5 mL of Reagent A and kept in the dark for 10 min. Then the mixture was loaded with 0.5 mL of Folin-Ciocalteu reagent and left in a dark place for 30 min. After incubation, the absorbance was measured at 630nm. Protein concentration (mg/mL) was determined using the Bovine Serum Albumin (BSA) standard curve in the extracted samples. Reagent A is made by combining 99 mL of reagent B (2% Na2CO3 and 0.1N NaOH), 0.5 mL of reagent C (2% of copper sulfate pentahydrate), and 0.5 mL of reagent D (2% of potassium sodium tartrate). 2.5.2 Total carbohydrate analysis The total carbohydrate content of extracted samples was determined using the Anthrone technique, with glucose (10–80 mg/L) used as the standard. For the test, 1 mL of extract is treated with 5 mL of the anthrone reagent (dissolve 2g of anthrone in 1000 mL of concentrated sulfuric acid) and incubated at 100°C for 10 min. After incubation, the absorbance was measured at 620nm. Carbohydrate concentration (mg/mL) was determined using the glucose standard curve in the extracted samples. 2.5.3 Total flavonoid analysis The total flavonoid content (TFC) of fungal MBE and BE extracts was evaluated according to the method described by Wandati et al. (2013). An aliquot of 1 mL of the sample was added to 0.3 mL of 5% NaNO2, and 4 mL of distilled water was kept for 5 min. Then the mixture was loaded with 0.3 mL of 10% AlCl3 and allowed to sit for 6 min. Finally, 2 mL of 1 M NaOH was added to these tubes, followed by 10 ml of distilled water. The absorbance was measured at 415nm. Quercetin was used as a standard in a concentration range of 20–100 µg/mL, and the result was expressed as µg QE/g DW. 2.5.4 Total phenol analysis The total phenol content (TPC) of the fungal MBE and BE extracts was analyzed using the Folin-Ciocalteu method of Wandati et al. (2013), with some modifications. Each sample (1 mL) was mixed with 0.5 mL of 10% (v/v) Folin-Ciocalteu phenol reagent and 2 mL of 700 mM Na2CO3 in distilled H2O. The mixture was vortexed well and left for 30 min. After incubation, the absorbance was measured at 765nm. The TPC in the extract was expressed as μg gallic acid equivalent (GAE) per g dry weight (μg GAE/g DW). 2.5.5 Total saponin analysis The total amount of saponin in fungal MBE and BE extracts was analyzed as described by Le et al. (2018). An aliquot of 1 mL of the sample was added to 1 mL of 8% vanillin and 2.5 mL of 72% sulfuric acid. The mixture was incubated for 15 min at 60°C. After incubation, the absorbance was measured at 560nm. Aescin was used as a standard in a range of 20–100 µg/mL, and the result was expressed as mg Aescin/g DW. 2.5.6 Total anthraquinone analysis The total anthraquinone content of fungal MBE and BE extracts was evaluated according to the method described by Sakulpanich and Gritsanapan (2008), with some modifications. In 1 mL of sample, 10 mL of distilled water was added. Then, 20 mL of 10.5% ferric chloride was added, and the mixture was mixed and refluxed for 10 min. Add 1 mL of sulfuric acid and reflux for another 10 min. Transfer the final content into a separating funnel, and extraction was done three times with 25 mL of ether. After that, the ether layer was evaporated to dryness to collect the residues. The residues were dissolved in 1 ml of 0.5% Mg(CH3COO)2 in CH3OH, and absorbance was measured at 515 nm. Rhein was used as a standard in a range of 20–100 µg/ml, and the results were expressed as µg Rhein/g DW. 2.5.7 Total alkaloid analysis The total alkaloid content in macrofungal MBE and BE extracts was determined as described by Nantongo et al. (2018). A total of 50 mL of 10% acetic acid in ethanol was added to 100 mg of the sample; the mixture was covered and incubated for 4 h. After that, the extract was concentrated in a water bath to one-quarter of the original volume, followed by the addition of five drops of concentrated NH4OH to precipitate alkaloids. The precipitate was allowed to rest for 3 h to sediment before being rinsed with 10 mL of 0.1M NH4OH and filtered. The residue on the paper was dried at 60°C for 30 min and then reweighed. Mathematically, the % of the alkaloid is calculated as mentioned in Equation 1: (1) Alkaloid percentage=weight of residue/weight of sample∗100 2.5.8 Terpenoids analysis The terpenoid content in macrofungal MBE and BE extracts was determined as described in Malik (2017). A total of 9 mL of ethanol was added to 100 mg of extract and incubated in the mixture for 24 h. Using the separating funnel, the extract was filtered with 10 mL of petroleum ether. A pre-weighed glass vial was used to collect the ether extract. At room temperature, the solvent was allowed to completely evaporate and then reweighed. The % terpenoids content was measured using the formula given in Equation 2: (2) Terpenoids Percentage= weight of residue/weight of sample ∗100 2.5.9 Total tannin analysis The total tannin content in macrofungal MBE and BE extracts was determined using the Folin-Denis reagent. 10 mL of distilled water was added to 1mg of extract, gently boiled for 1 h in a water bath, and filtered using Whatman filter paper in a 10 mL volumetric flask. To this content, 5 mL of Folin-Denis reagent and 10 mL of saturated Na2CO3 solution were added and incubated at 25°C in a water bath for 30 min. After incubation, the absorbance was measured at 700 nm. The amount of tannin in the extracts was calculated using the equation developed on the standard curve of tannic acid. 2.5.10 Sterols analysis The sterol content in macrofungal MBE and BE extracts was determined using the Liebermann-Burchard reagent, and cholesterol was used as the standard concentration range of 0.2–1 mg/mL. In 1 ml of acetic acid, 1 mg of the extract was dissolved. Then, it was treated with 5 mL of Liebermann-Burchard reagent and incubated at 37°C in a water bath for 10 min. After incubation, the absorbance was measured at 650 nm. 2.5.11 Antioxidant property of the 2,2-diphenyl-1-picrylhydrazyl assay The ability of the MBE and BE extracts to donate electrons and scavenge DPPH radicals was determined by the slightly modified method of Anjana et al. (2016). A freshly prepared 1 mM ethanol solution of DPPH was mixed with the mycelium biomass and mushroom extract at a ratio of 1:0.5 (v/v). The mixture was vortexed properly and incubated in the dark for 10 min. The light absorption was measured at 517 nm. The % scavenging value at the variable concentration was used to plot a regression curve, and the equation generated on the curve was used to estimate the IC50 value for both extract samples. The Equation 3 is used for % scavenging calculation: (3) Scavenging percentage = absorbance of control – absorbance of sample * 100 2.5.12 β-glucan analysis The amount of β-glucan in fungal MBE and BE was measured based on the enzymatic release of glucose, as described in Toledo et al. (2013). This procedure calls for a standard reactive glucose solution with a concentration of 100 mg dL-1, an enzyme reactive mixture with glucose oxidase and peroxidase, color reagent 1 (4-aminophenazone, 25 mM, 920 mM Tris), and color reagent 2 (55 mM phenol). The working reagent was prepared in a 250-mL volumetric flask using 225 mL of distilled water, 12.5 mL of color reagents 1 and 2, and 0.75 mL of enzyme reactive. In total, 20 mL of each sample extract was added to 2 mL of working reagent in the reaction mixture and incubated at 37°C for 15 min. After that, the absorbance at 505 nm was determined using a spectrophotometer. Using 2 mL of the working reagent and 20 mL of standard reactive glucose, the reaction was carried out under the same conditions as above. The results were examined using Equation 4 below: (4) β–glucan (mg/dL)=A * f * 0.9 Where A denotes the absorbance of the sample at 505 nm, which is equal to 100 mg/dL of normal glucose, and 0.9 denotes the proportion of free glucose conversion that was established for anhydrous glucose, which occurs in β-glucan. 2.5.13 Glucan content analysis using the Megazyme kit The contents of the total α- and β-glucans were determined in the MBE and BE using the mushroom and yeast β-glucan assay kit (Megazyme Int., Dublin, Ireland), following the instructions of the manufacturer. Briefly, to estimate the total glucan content in the samples, 2 mL of ice-cold 12M sulfuric acid was used for the hydrolysis of the polysaccharides in the samples at 100°C for 2 h. After neutralization, hydrolysis proceeded to glucose using a mixture of exo-β-(1,3)-D-glucanase + β-glucosidase in sodium acetate buffer (pH 4.5) at 40°C for 1 h. For α-glucan content estimation, enzymatic hydrolysis with amyloglucosidase and invertase was conducted. After that, a glucose oxidase/peroxidase reagent was added to estimate the total glucan and α-glucan content at 510nm. The β-glucan content was calculated by subtracting the α-glucan from the total glucan content. All values of total, α-, and β-glucans in biomass and mushrooms were expressed in g/100g of DW. Determination of β-glucan: The β-glucan content was determined by subtracting the α glucan content from the total glucan content. 2.5.14 Quantification of phytochemical content by HPLC The identification and quantification of phytochemicals from mycelium biomass and mushroom extracts were performed using the high-performance liquid chromatography (PDA) technique. Chromatography was performed on a 250mm x 4.5mm RP C-18 column. Preliminary experiments, in which various ratios of methanol and water were examined, helped to optimize the condition. Quercetin, sesquiterpene glycosides, ergosterol, and RidentinB were the standards employed for the identification and quantification of total polyphenols, terpenoids, sterols, and sesquiterpenes, respectively. The flow rate, column temperature, and detector wavelength were all adjusted to 1 mL/min, 25°C, and 210 to 370 nm, respectively. The formula used for calculation is mentioned in Equation 5: (5) The concentration of compound (mg/mL)=(sample absorbance/mean absorbance of standard) * (standard dilution/sample dilution) * (standard potency/100) * 100. 2.6 Statistical evaluation All the experiments were conducted in triplicate, and the values were expressed as mean ± SD. Statistical significance was detected by the Statistical Package for the Social Sciences (SPSS); a value of p<0.05 indicated a statistical difference.',\n",
       " '11196826': '',\n",
       " '11196812': '',\n",
       " '11196808': '',\n",
       " '11196703': 'Methods This trial was designed as a randomized, double-blind cross-over study. The effect of the sugar substitute ENSO 16 was compared with a 30 g glucose in water solution on glucose metabolism parameters in healthy male or female humans. The trial was approved by the Ethics Committee of the Medical University of Vienna (EK 1126/2022). The study was conducted in accordance with the protocol, the principles of the \"Declaration of Helsinki\" in its current version and the requirements of the Austrian law (AMG). According to the stipulations of the Austrian Data Protection Law, confidentiality and pseudonymity of the volunteers were assured. The informed consent was obtained from all the participants for study participation. The recruiting and data allocation took place between 11. July 2022 and 29. July 2022. Population 18 healthy participants (7 males, 8 females) were included in the study. Participants that dropped out were replaced according to protocol to generate data of 15 different individuals. Inclusion criteria were signed informed consent, age > 18 years, body mass index 18–25 kg*m−2 and a fasting capillary plasma glucose concentration ≤ 100 mg*dL−1. Exclusion criteria were presence of any chronic illness, smoking or drug abuse, food allergies, pregnancy or lactations or participation in another trial 3 weeks before the first study day. Study product The investigational product ENSO 16 (NEOH by Alpha Republic GmbH, Rooseveltplatz 11/1, 1090 Vienna, Austria) is a sugar substitute mixture containing 62% organic fibers like inulin and oligofructose and 38% of other sugar alcohols and sweetener substances. The composition is confidential and not fully disclosed. ENSO 16 was produced, packaged and labeled by STAMAG Stadlauer Malzfabrik GesmbH, Smolagasse 1, 1220 Vienna, Austria according to applicable food law provision and guidelines. The active comparator was glucose powder provided by Special Ingredients Ltd, 4, Foxwood Industrial Park, Chesterfield, S41 9RN, UK. 30 g of each substance were dissolved in 200 mL water for oral administration. A similar amount of sweetener was used in other investigations of sugar substitutes to describe endocrine responses14. 30 g of ENSO 16 contain 198 kJ (49.5 kcal) with an amount of 9.3 g carbohydrates of which 0.72 g is glucose. 30 g of glucose have an equivalent of 486 kJ (116 kcal). The FDA recommends a maximum daily intake of added sugar of 200 kcal (48 g). Study procedure Two study days with a washout period of at least 7 days were planned. Participants were randomized to receive ENSO 16 or glucose on the first study day. The sponsor conducted randomization using a predefined randomization list from the web application at www.randomizer.org. The study medication was provided by the sponsor in identical-looking bottles. Due to their completely identical appearance and smell (white, powdery substance, sweet smell), no difference could be perceived by the investigators or subjects. Preparation of the study product on the study days was performed by blinded investigators. Unblinding was performed after the database lock. Following the wash out period, subjects crossed over to the alternate treatment on the second study day. (Supplement Table 1) Plasma samples for analysis of glucose, C-peptide and insulin were collected under fasted conditions and at pre-defined timepoints (15, 30, 45, 60, 90, 120, 180 min) after single administration of ENSO 16 or glucose. (Supplement Fig. 1) Gastrointestinal symptoms (abdominal pain, nausea, vomiting, diarrhea, abdominal rumbling, bloating, belching and flatulence) were assessed using a standardized questionnaire. Participants were asked to choose between “no symptom” (0 points), “mild symptoms” (1 point) and “severe symptoms” (2 points) for each question at each time point. (Supplement Table 2). Table 1Baseline characteristics of the study population, n = 15.Baseline characteristicsp valuesFemalen, %853.3Malen, %746.7BMI[kg*m−2]Mean ± SD22 ± 2Age[yrs]Mean ± SD28.4 ± 6.1Fasting glucose D1[mg*dL−1]Mean ± SD87 ± 8Fasting glucose D2[mg*dL−1]Mean ± SD89 ± 70.3746 vs. Day 1Figure 1Study flow chart.Table 2Fasting plasma concentrations of glucose, insulin and C-peptide at baseline in participants receiving a single dose of glucose or ENSO 16.GlucoseENSO 16p valuesGlucose [mg*dL−1]79 ± 980 ± 60.5069Insulin [µIU*mL−1]6.36 ± 2.096.59 ± 2.080.676C-peptide [ng*dL−1]1.78 ± 0.691.67 ± 0.350.4593Results are mean ± SD, n = 15 per treatment. Differences in laboratory values were not significant between treatments at baseline (Student’s paired t-test). Methodology Plasma glucose, insulin and C-peptide concentrations were analyzed according to laboratory standards at the Department of Laboratory Medicine, Medical University of Vienna. Venous plasma sampling was performed from an indwelling cannula in an antecubital vein. To assess plasma glucose concentrations, blood was drawn into tubes containing sodium fluoride K3EDTA (Vacuette, Greiner Bio-One, Austria). Insulin and C-peptide concentrations were quantified from blood collected into serum tubes with clot activator (Vacuette, Greiner Bio-One, Austria). Study endpoints Changes in plasma glucose were compared from baseline-adjusted area under the plasma glucose versus time curve until 180 min (AUC0–180 min). Secondary endpoints included plasma glucose, plasma insulin and C-peptide, respectively. Further, the impact of ENSO 16 and glucose on gastrointestinal symptoms were assessed. Statistics Due to a lack of data for the impact of ENSO 16 compared to glucose on plasma, a difference of 10% between groups is assumed to be clinically relevant. With a sample size of 15 subjects, based on other trials and recommendations13 (cross over) a difference of at least 10% between AUCs (with a standard deviation of 10%) can be detected with 80% power using a paired t test at a significance level of 5% (two-sided). All statistical analyses were conducted using the commercially available GraphPad Prism 9 (GraphPad, 225 Franklin Street. Fl. 26 Boston, MA 02110, USA). Continuous variables were summarized as mean ± standard deviation (SD) if their distributions were consistent with normality. If the distributions were not consistent with normality, the variables were summarized by their median and the 25th to 75th percentile range. Between-group comparisons for primary and secondary outcome parameters were assessed using paired t-tests or Wilcoxon signed rank tests as appropriate. Differences were considered significant if the p-value was less than 0.05. For the evaluation of insulin, C-peptide, and glucose profiles, baseline-corrected values were used to calculate the area under the curve (AUC0–180 min) using the trapezoidal rule. Maximum plasma concentrations were used to calculate individual ratios of effects between treatments. Ethical approval The trial was approved by the Ethics Committee of the Medical University of Vienna (EK 1126/2022). Methodology Plasma glucose, insulin and C-peptide concentrations were analyzed according to laboratory standards at the Department of Laboratory Medicine, Medical University of Vienna. Venous plasma sampling was performed from an indwelling cannula in an antecubital vein. To assess plasma glucose concentrations, blood was drawn into tubes containing sodium fluoride K3EDTA (Vacuette, Greiner Bio-One, Austria). Insulin and C-peptide concentrations were quantified from blood collected into serum tubes with clot activator (Vacuette, Greiner Bio-One, Austria).',\n",
       " '11196646': '',\n",
       " '11196440': 'MethodPubMed, Embase, Web of Science and Cochrane Library databases were search up to March, 2023 to identify relevant studies, including studies researching beverage consumption, the incidence and mortality of T2DM and incidence of cardiovascular disease, a kind of complication of T2DM. The way to explore the source of heterogeneity is performing subgroup analyses and sensitivity analyses. Funnel plots and Egger’s regression test were performed to assess publication bias. The Hazard ratio (HR) and 95% confidence intervals (95% CIs) were used to analysis the results. Fifteen observational studies were included in our meta-analysis. Methods This systematic review and meta-analysis was written based on the Preferred Reporting Items of Systematic Reviews and Meta-Analysis (PRISMA) statement guidelines [8]. Search methods The databases searched for this article include PubMed, Embase, Web of science and Cochrane Library databases, and the search authors include Zhang Yuqing, Wang Xu and Liu Yuxiang. The search term is ((type 2 diabetes [Title/Abstract]) OR (type 2 diabetes mellitus[MeSH Terms])) AND ((beverage[MeSH Terms]) OR (beverage[Title/Abstract])), which is consist of main Mesh and non-Mesh terms. The search of relevant literature were searched for relevant studies from inception to March 2023. The MeSH/Emtree and abstract/title keyword were combined to identify the eligible English articles. In addition, a manual search was conducted for the references of the included articles (shown in Fig. 1).Fig. 1Flowchart of selecting studies for review Study eligibility Eligibility criteria and exclusion criteria should be identified, which were used to search and screen potential articles. Articles may be eligible, if they fulfill all the following conditions: (1) prospective population-based studies or patients diagnosed with T2DM; (2) the study compared with the use of beverages (including coffee, tea and SBs) with no use of beverages; (3) the study measured and reported the outcomes in terms of mortality of T2DM, incidence of T2DM or incidence of complications; The exclusion criteria were as follows: (1) all criteria were not fulfilled; (2) the study was a case report, an animal study, a review, a conference abstract or an abstract; (3) OR/RR/HR with 95% CIs were not provided or could not be calculated; Date extraction and study quality The selection and searching of potential articles were completed independently by two researchers, based on the inclusion and exclusion criteria, to identify eligible studies. Data extraction was conducted separately by another researchers. Moreover, any disagreements would be decided by the senior researcher to resolve this controversy. Relevant data were collected, which include the year of publication, first author, study title, study type, study period, type of beverage, the efficacy in control and treatment groups, T2DM definition, reported outcomes and so on. The characteristics of the studies included in this meta-analysis included beverage type, follow-up duration range, number of participants, age, sex distribution, beverage intake, and outcome measures (shown in Table 1). Beverage portion sizes and frequency of consumption were different among the included articles, some were not mentioned in the original study, the available data have been extracted into the Table 1. For every abstracted data of articles, two investigators independently assess the risk of bias for this meta-analysis based on The Newcastle–Ottawa Scale (NOS) [9] (shown in Table 2). Articles were thought high quality which received seven or more points. Covariates were composed of eight parts: age and sex, race and socioeconomic status, physical activities, smoking and drinking, body mass index, diet factors, clinical features and other types of beverages. The types of statistical data related to outcomes extracted in these 15 articles were all binary variables, HR values, and 95% confidence intervals. Table 1Characteristics of the included studies about the association between beverages and the risk of incidence or prognosis in T2DM patientsAbbreviationT2DM Type 2 of diabetes mellitus; SSB Sugar-sweetened beverages; ASB Artificially sweetened beverages; TSB Total sweetened beverage (combined ASB and SSB); NJs Natural juices; LCSBs Low-calorie sweetened beverageTable 2NOS scores of the included studies Statistical analysis and publication bias The aim of forest plots was to synthesize HRs and 95%CI across studies. The I2 statistics presented the potential heterogeneity; the pooling outcome was considered to represent non-significant, low, moderate and high heterogeneity respectively, when I2 was 0–25%, 26–50%, 51–75% or > 75%. The source of heterogeneity was explored by performing subgroup analysis. The potential source of heterogeneity includes the definition of exposure, grouping criteria, study sites, number of participants, sex ratio, follow-up period, quality of study, baseline age and adjustment of clinical features. Egger’s linearity regression test and Begg’s funnel plot was performed to evaluate the publication bias. For the asymmetry, funnel plots were visually assessed. The aim of Egger’s lire available to assess, sensitivity analysis was performed to evaluate the influence of a single study on the overall effect estimate by sequential stepwise omitnear regression is to confirm the presence of a small size. If sufficient data weting of one study. All analyses were completed by using Stata12.0(College Station, Texas 77845, USA), random effects model and inverted variance method. P < 0.05 was considered statistically significant. Search methods The databases searched for this article include PubMed, Embase, Web of science and Cochrane Library databases, and the search authors include Zhang Yuqing, Wang Xu and Liu Yuxiang. The search term is ((type 2 diabetes [Title/Abstract]) OR (type 2 diabetes mellitus[MeSH Terms])) AND ((beverage[MeSH Terms]) OR (beverage[Title/Abstract])), which is consist of main Mesh and non-Mesh terms. The search of relevant literature were searched for relevant studies from inception to March 2023. The MeSH/Emtree and abstract/title keyword were combined to identify the eligible English articles. In addition, a manual search was conducted for the references of the included articles (shown in Fig. 1).Fig. 1Flowchart of selecting studies for review',\n",
       " '11196417': 'MethodsThe study first accessed SD datasets from the GEO and identified DEGs. These DEGs were then subjected to gene set enrichment analysis. Several advanced techniques, including the RF, SVM-RFE, PPI networks, and LASSO methodologies, were utilized to identify hub genes closely associated with SD. Additionally, the ssGSEA approach was employed to analyze immune cell infiltration and functional gene set scores in SD. DEGs were also scrutinized in relation to miRNA, and the DGIdb database was used to explore potential pharmacological treatments for SD. Furthermore, in an SD murine model, the expression levels of these hub genes were confirmed through RT-qPCR and Western Blot analyses. 2 Methods 2.1 Data collection and normalization We retrieved datasets related to SD and their corresponding control groups from the Gene Expression Omnibus (GEO) database (http://www.ncbi.nlm.nih.gov/geo) (12). Human RNA expression data from 17 individuals with SD and 25 healthy controls were obtained from the GSE208668 dataset, derived from the GPL10904 platform. For the validation of the subsequently identified hub genes, we utilized the GSE240851, GSE56931, and GSE98582 datasets, which were sourced from the GPL24676, GPL10379, and GPL6244 platforms, respectively. These datasets included varying numbers of SD patient and control samples. Additionally, the GSE165041 dataset, generated on the GPL18573 platform, comprised microRNA expression data from 10 SD patients and an equal number of healthy controls. All datasets were normalized using the “limma” package within the R software environment, version 4.1.2 (13). 2.2 Identification of differentially expressed genes We utilized the limma package in R software to process the normalized datasets GSE240851 and GSE208668, aiming to identify DEGs. To ascertain statistical significance, DEGs were determined based on |log2fold change| ≥ 0.58 and a false discovery rate <0.05. Subsequently, we employed the ggplot2 package to create volcano plots for the DEGs. Moreover, we selected the top 20 genes, ranked by |log2fold change|, to build heatmaps as part of our analysis. 2.3 Gene function enrichment analysis The “c2.all.v2023.1.Hs.entrez” and “c5.all.v2023.1.Hs.entrez” datasets, which serve as reference gene sets, were downloaded from the GSEA official website originating from the MSigDB database (14). Gene set enrichment analysis (GSEA) was then performed using the “clusterProfiler” package in R software (15). Subsequently, the analysis results were visualized utilizing the “enrichplot” package in R software. 2.4 Immune infiltration analysis We employed the ImmuCellAI and ssGSEA methodologies to estimate the abundance of 24 and 28 distinct types of immune cells, respectively, in the tissues of SD patients and a normal population, thereby enabling precise delineation of immune cell profile differences between the two groups (16, 17). Furthermore, we conducted Spearman correlation analysis to elucidate the interrelationships among the distributions of various immune cells. 2.5 ssGSEA We downloaded the H: hallmark gene sets from the GSEA official website to probe the functional disparities between SD patients and the normal cohort. With these gene sets, we applied the ssGSEA algorithm to evaluate 50 gene sets, aiming to discern potential variances between the two groups. Subsequently, we utilized the Mantel algorithm to analyze the correlations among these gene sets (18). This approach allowed us to investigate the differences in gene expression profiles between the two cohorts and to identify potential molecular pathways associated with the disease. 2.6 Protein-protein interaction network construction For protein interaction analysis, we utilized the STRING 4 online platform and specifically selected PPI pairs with a confidence score greater than 0.40. Subsequently, we employed the Cytoscape V3.9.0 software for visualizing the PPI network (19). Within the network, the significance of each node was determined by calculating their Degree values using the CytoHubba plugin. This analysis allowed us to identify the top 20 pivotal genes based on their ranking of importance (20). 2.7 Random forest gene selection For gene selection, we utilized the Random Forest (RF) algorithm, a binary tree-based recursive partitioning method. The “randomForest” package in R was used, with parameters set to ntree=1000, mtry=3, and importance=true (21). Employing the Gini index as the primary assessment criterion, the Random Forest algorithm was used to rank the DEGs, and the top 20 genes with a significance value greater than 3 were earmarked for further analysis. 2.8 Support vector machine gene selection We utilized the SVM-RFE (Recursive Feature Elimination) approach to optimize the predictive model by minimizing the number of feature vectors produced by the SVM. This approach, being an effective binary classification tool, operates by constructing a classification hyperplane to delineate decision boundaries. In order to enhance the algorithm’s precision, we configured parameters to method=repeatedcv and repeats=10 in the R package “1071”, and employed ten-fold cross-validation. This approach aimed at augmenting the algorithm’s precision (22). 2.9 Model construction and evaluation To develop the LASSO model (23), we integrated genes identified through CytoHubba, RF, and SVM algorithms. This method effectively enabled the identification of critical hub genes for diagnosing SD. We then employed the Logistic regression approach to investigate pivotal factors associated with SD, ultimately constructing a simplified model. Subsequently, we evaluated the classification performance of the model using the Receiver Operating Characteristic (ROC) curve and the corresponding Area Under the Curve (AUC). 2.10 Validation of the diagnostic model In order to evaluate the robustness and general applicability of our developed diagnostic model, we computed the AUC of the ROC curve for the model using three distinct datasets: GSE240851, GSE56931, and GSE98582. This procedure aimed to ascertain the model’s performance across diverse datasets. It was essential to assess its efficacy in different contexts and ensure its capability to perform consistently across varied data sources. 2.11 Exploration microRNAs targeting the genes We utilized the Limma package in R to conduct a differential analysis of the expression matrix from GSE165041, aiming to identify miRNAs that were differentially expressed (DEmiRNAs). Our criteria for significance were miRNAs exhibiting a false discovery rate <0.05 and |log2fold change| >0.5. Moreover, we employed the miRNet database to investigate potential miRNAs associated with differentially expressed genes (DEGs), in order to gain further insight into their involvement in SD (24). 2.12 Drug and gene interaction scoring We obtained data on drugs related to core genes from the Drug-Gene Interaction database (DGIdb) (25). We used the “ggplot2” package in R to create bar charts showing interaction scores, visually indicating the intensity of interactions between various drugs and core genes. 2.13 Establishment of animal models The SD model, following the methodology outlined by Alkadhi and Alhaider (26), was established in 10-month-old male mice. The experimental mice were subjected to SD treatment for a duration of 8 weeks by being placed on a small fixed platform encircled by water, with access only to water and food. Meanwhile, for the control group, another set of mice was housed in a comfortable environment with a 12-hour light/dark cycle and unrestricted access to water and food. All mice received standard pellet feed, with the daily quantity of feed being maintained uniformly across all groups. This experimental design aimed to simulate the effects of SD on physiological functions, thus laying the groundwork for subsequent investigations into gene expression and drug treatment efficacy. 2.14 RT-qPCR The cortex in SD mice was used for total RNA extraction, employing the TransZol Up Plus RNA Kit (TransGEN, Beijing, China) (27). The RNA concentration and quality were then assessed using the Nanodrop Spectrophotometer (Termo Scientifc, Waltham, MA, USA). Subsequently, reverse transcription was carried out using the TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix (AT311, TransGEN, Beijing, China). Amplification was monitored with the ChamQ Universal SYBR qPCR Master Mix (Novozymes Q711) and a QuantStudio™ 5 Real-Time PCR System (Thermo Fisher Scientific). The internal reference was β-actin, and the relative gene expression was determined using the 2-ΔΔCT formula. Detailed primer sequences can be found in Table 1 . Table 1 Primer sequences of mRNA for RT-qPCR. Gene Primer sequence, 5’–3’ Forward Reverse IPO9 CAGTGACAGCCTTGGTGAAA TCTCCAGTAGGGCATGGACA RAP2A CAAACTGTACCACGCCCTCT GTTGCTAGGTGGATTGGGCT DDX17 TCTTCAGCCAACAATCCCAATC GGCTCTATCGGTTTCACTACG MBNL2 CCCAAAAGTTGCCAGGTTGAA CTGGGTTTTTAAGTGTGTCGGA PIK3AP1 CTGGACTCTGCTTCTAACCCC TGACACCATTCCTCCGCATC ZNF385A CAGAACCAAGGGAAGGGGAC GAAGGGCAGGATCTGCTTGA β-Actin GCAGGAGTACGATGAGTCCG ACGCAGCTCAGTAACAGTCC 2.15 Immunoblotting for protein evaluation Western blotting was performed for IPO9, RAP2A, DDX17, and GAPDH, as described previously ( Table 2 ). Enhanced chemiluminescence reagents were used to detect protein expression, and quantitative analysis was conducted using Image J software (28). Table 2 List of the primary antibodies. Antibody Catalogue number Brand Application Dilution Species MW(kDa) IPO9 abs134286 absin WB 1:1000 Rabbit 116 RAP2A abs105749 absin WB 1:1000 Rabbit 21 DDX17 abs111924 absin WB 1:1000 Rabbit 72 GAPDH GB15002-100 Servicebio WB 1:2000 Rabbit 36 2.16 Statistical analysis The normality of the data was assessed using the Shapiro–Wilk test. The t-test was utilized to compare the data between two groups, while comparisons among multiple groups were conducted using one-way ANOVA, followed by either the LSD post hoc test or Tukey’s post hoc test. The statistical analyses were conducted using R software version 4.2.1 and SPSS 25, with significance defined as P < 0.05. Furthermore, all experiments were independently repeated at least three times to ensure the validity of the results.',\n",
       " '11196401': '',\n",
       " '11196393': '2 Materials and methods 2.1 Human samples Samples of peripheral blood were collected from healthy adults, full-term infants, and preterm infants. Samples were collected from 32 subjects from Guangzhou Women and Children’s Medical Center and the Third Affiliated Hospital of Guangzhou Medical University. The basic characteristics of the samples are included in Supplementary Table S2 . All subjects were collected within 7 days after birth and screened for serum hepatitis B surface antigen (HBsAg), hepatitis C virus (HCV) antibody, hepatitis D virus (HDV) antigen, HDV antibody, and HIV antibody, and positive individuals were excluded from this study. Preterm infants were defined as those with a gestational age of less than 37 weeks. Full-term infants were defined as those with gestational age between 37 and 42 weeks. Informed consent was obtained from all participants. 2.2 Sample preparation for 10× genomic single-cell sequencing Peripheral blood samples were collected from three full-term, three preterm, and three adult controls. Single-cell suspensions of peripheral blood mononuclear cells (PBMCs) were obtained by centrifuging human blood samples on a Ficoll gradient. Briefly, blood was mixed with an equal volume of 2% FBS in PBS and gently layered on the Ficoll gradient. Cells were centrifuged at 1,000×g for 25 min without braking. The cells in the middle layer were then washed once with PBS, and the red cells were removed using ACK lysing buffer and resuspended in 2% FBS in PBS for use. The PBMC suspensions were stained with surface markers at 30 min at 4°C in the dark. The strategy for MDSCs was CD11b+HLA-DR−/lo. Live immature myeloid cells in RPMI1640 supplemented with 20% FBS were sorted in a BD FACSAria III cell sorter (BD Biosciences, USA). 2.3 Flow cytometric analysis Freshly prepared PBMCs were incubated with Fc-block (Miltenyi, Bergisch Gladbach, Nordrhein-Westfalen, Germany) for 10 min, and surface staining was performed at 4°C for 30 min. Cells were stained with surface antibodies against CD11b, HLA-DR, CD14, CD177, and CXCR2. For Arg1 staining, single-cell suspensions were stained with antibodies to surface antigen, then fixed and permeabilized, followed by staining with anti-Arg1 (A1exF5; Thermo Fisher, Waltham, Massachusetts, USA). For ROS staining, single-cell suspensions were stained with antibodies to surface antigen and were evaluated with 20 μM DCFDA (Abcam, Cambridge, UK) staining at 37°C for 30 min. The analysis was performed by BD FACSCanto™ II. The following antibodies from (BioLegend, San Diego, California, United States) were used: APC antihuman CD11b(ICRF44), Brilliant Violet 605™ antihuman HLA-DR(L243), APC/Cyanine7 antihuman CD14(63D3), PerCP/Cyanine5.5 antihuman CXCR2(5E8/CXCR2), and FITC antihuman CD177(MEM-166). 2.4 scRNA-seq data collection and analysis The data about newborns have been deposited in NCBI’s Gene Expression Omnibus and are accessible through GEO Series accession number GSE253963 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE253963). The raw data were preprocessed using Cell Ranger (version 7.0). The R toolkit Seurat (version 4.0.1) was used for downstream analysis (10). Cell subpopulations were annotated based on the expression of canonical marker genes. The function of AddModuleScore was performed to calculate the scores of molecular signatures in individual cells. The gene sets used for each molecular signature are listed in Supplementary Table S4 . Genes associated with the enriched pathways identified through GO analysis are listed in Supplementary Tables S3 , S4 . The scRNA-seq dataset of tumor MDSCs (GSE163834) was downloaded from the Gene Expression Omnibus GEO database (https://www.ncbi.nlm.nih.gov/gds/) (11). 2.5 Pseudo-time developmental trajectory analysis The R package monocle2 (version 2.26.0) was used to infer the differentiation path among distinct cell subpopulations (12). Briefly, a CellDataSet class was created by using a normalized scRNA-seq dataset. Subsequently, the differentialGeneTest function was performed to identify differentially expressed genes (DEGs) with each subpopulation. These DEGs were then used as input genes to perform dimension reduction using the DDRTree method. Next, each cell was assigned a pseudo-time order by executing the orderCell function. Finally, the pseudo-time trajectory was plotted by performing the function of plot_cell_trajectory. 2.6 Statistical analysis All data were derived from at least two independent experiments. Statistical analysis was performed with GraphPad Prism 8.0. Statistical significance was determined by a paired Student’s t-test or an unpaired Student’s t-test for comparing two groups, and a one-way ANOVA with a Tukey–Kramer multiple comparisons test was performed for comparing three groups within the same experiment. A p-value < 0.05 was considered significant, and the results showed the mean ± SEM.',\n",
       " '11196377': \"MethodsA total of 600 23‐day‐old broiler male chicks (Cobb‐500) were assigned to 10 treatments with six replicates and 10 birds each. Experimental treatments were included factorial arrangement of five increment (2.5%) levels of finisher diet ND (92.5%, 95%, 97.5%, 100% and 102.5% as strain recommendation) and slaughtered at 38 or 46 days of age. The relative difference in the energy level of experimental diets was used to increase ND levels at the same ratio. 2 MATERIALS AND METHODS 2.1 Feedstuff analysis Before to the trial, protein, amino acids and digestible amino acids of feed ingredients were measured by using near‐infrared analysis through Evonik Co. (Evonik Nutrition & Care GmbH) agent in Tehran, Iran. These values were used in a least‐cost equation for the formulation experimental diets. 2.2 Birds, housing and diets A total of 840 1‐day‐old male broiler chicks (Cobb‐500) were allocated into 60 pens (14 birds/pen, 100 × 100 cm2) and raised until 22 days on a common condition. The ambient temperature initially was maintained at 32 ± 2°C; after 3 days, it decreased by 0.5°C per day to reach 20–22°C and after that, it was kept constant, the relative humidity was 50%–60%; and a lighting schedule of 18L: 6D was imposed throughout the experimental period. At 23 days of age, with removing the highest and the lowest weight birds, 10 birds were fixed in each pen (average LBW was 695 ±11 g/bird). The pens were randomly assigned to a 5 × 2 factorial arrangement of feeding five levels of finisher diet ND 92.5%, 95%, 97.5%, 100% and 102.5% as strain recommendation (Cobb‐Vantress, 2019), and two SAs (38 vs. 46 days old) with six replicates per treatment. The experimental diets were formulated in a minimum cost equation by user‐friendly feed formulation done again (UFFDA, 1992) software, University of Georgia, Athens, GA, United States (Table 1). The relative difference in the experimental diets’ energy level was used to change protein, indispensable amino acids and minerals inclusion levels at the same ratio. All birds had free access to drinking water and feed throughout the experiment period. Birds and housing facilities were inspected three times daily. TABLE 1 Ingredients and nutrients composition of the experimental diets. Items Finisher diet nutrients density a 92.5% 95% 97.5% 100% 102.5% Ingredients, g/kg as‐fed basis Corn 652.8 636.2 619.6 603.0 586.4 Soya bean meal 236.0 253.5 271.1 288.8 306.3 Wheat bran 68.1 53.6 39.1 24.5 10.0 Soya bean oil 10.0 22.7 35.4 48.1 60.8 Limestone 7.9 8.0 8.0 8.1 8.2 Dicalcium phosphate 11.2 11.9 12.6 13.3 14.0 Common salt 3.9 4.0 4.1 4.1 4.2 Vitamin premix b 2.5 2.5 2.5 2.5 2.5 Mineral premix c 2.5 2.5 2.5 2.5 2.5 dl‐Methionine 2.3 2.4 2.5 2.6 2.7 l‐Lysine‐HCL 1.9 1.8 1.7 1.6 1.5 l‐Threonine 0.9 0.9 0.9 0.9 0.9 Determined nutrients composition d , as‐fed basis Metabolizable energy, kcal/kg 2928 3007 3087 3166 3246 Crude protein, g/kg 175.7 180.5 185.2 190.0 194.7 ME/CP ratio 16.6 16.6 16.6 16.6 16.6 Crude fibre, g/kg 35.2 34.2 33.2 32.2 31.2 Calcium, g/kg 7.0 7.2 7.4 7.6 7.8 Available phosphorus, g/kg 3.5 3.6 3.7 3.8 3.9 Sodium, g/kg 1.8 1.8 1.8 1.9 1.9 Digestible lysine, g/kg 9.2 9.5 9.7 10.0 10.3 Digestible methionine, g/kg 4.7 4.9 5.0 5.2 5.3 Digestible Sulphur amino acids, g/kg 7.2 7.4 7.6 7.8 8.0 Digestible threonine, g/kg 6.3 6.5 6.7 6.8 7.0 Abbreviations: CP, crude protein; ME, metabolizable energy. a Percentage of nutrient recommendations stated by the Cobb‐500 (Cobb‐Vantress, 2019) commercial management guide. b Vitamin premix supplied the following per kilogram of diet: vitamin A (all‐trans‐retinol), 12,000 IU; vitamin D3 (cholecalciferol), 5000 IU; vitamin E (α‐tocopherol), 18 IU; vitamin K3 (menadione), 2.65 mg; vitamin B1 (thiamin), 2.97 mg; vitamin B2 (riboflavin), 8.0 mg; vitamin B3 (niacin), 57.42 mg; vitamin B5 (pantothenic acid), 17.86 mg; vitamin B6 (pyridoxine), 4.45 mg; vitamin B9 (folic acid), 1.9 mg; vitamin B12 (cyanocobalamin), 0.02 mg; vitamin H2 (biotin), 0.18 mg; choline chloride and 487.5 mg, antioxidant 1.0 mg. c Mineral premix supplied the following per kilogram of diet. Zn (zinc sulphate), 110 mg; Mn (manganese sulphate), 120.6 mg; Fe (iron sulphate), 40.5 mg; Cu (copper sulphate), 16.1 mg; I (calcium iodate), 1.26 mg; Se (Sodium Selenite) and 0.31 mg; choline chloride, 474.0 mg. d The determined ingredient analysis data was used to calculate nutrients composition; crude protein, calcium and sodium were measured by the AOAC (2002) methods; metabolizable energy, digestible amino acids, and available phosphorus were measured by using the near‐infrared analysis. John Wiley & Sons, Ltd. 2.3 Data collection and sampling 2.3.1 Growth performance The chicks of each pen were weighed in groups at the beginning (23 days old) and at the end (38 and or 46 days old). In order to minimize the error resulting from the digestive tract contents, the birds were starved for 4 h before weighing. The feed consumption of each pen was calculated by subtracting the amount of feed remaining at the end from the total feed given during the experimental period and adjusted for mortality. The growth performance traits, such as LBW, FI, WG, feed conversion ratio (FCR), yield per unit area (YUA) (Abougabal & Taboosha, 2020), bio‐economical index (BEI) (Santos et al., 2008) and European production efficiency factor (EPEF) (Cobb‐Vantress, 2019), were calculated. The YUA, BEI and EPEF were calculated by the following equations: YUA=Penlivebodyweightkg×Livability%Penaream2 BEI=Weightgainkg−FeedcostProductioncost×Feedintakekg EPEF=Weightgainkg×Viability%Feedconversionratio×Ageday×100 2.3.2 Energy and protein efficiency The following equations were used to calculate fat content of gain (FCG), protein content of gain (PCG), energy and protein efficiency and energy requirements (maintenance + gain) by using the function of initial and final LBW, WG and gain composition (Carré & Méda, 2015). The expected feed intake (EFI) was estimated by dividing the total energy requirements to diet AME. Residual feed intake (RFI) was obtained by subtracting the EFI from the observed FI (Nikbakhtzade et al., 2020). FCG=12.30.025+0.137BWinitial0.418+BWfinal0.4182e−1.082CPdietAMEdiet−2.09PE PCG=0.179+0.00169Ageinitial+Agefinal2×1−FCG Energyefficiency=WG9.13FCG+5.64PCGFI×AMEdiet Proteinefficiency=WG×PCGFI×CPdiet Energyrequirementsformaintenance=0.680BWinitial0.643+BWfinal0.6432 Energyrequirementsforgain=WG0.03288PCG+0.04372FCG where FCG is the fat content of gain, PCG is the protein content of gain, BW initial is the live body weight at the start of the test, BW final is the live body weight at the end of the test, WG is the weight gain, CP diet is the diet crude protein, AME diet is the diet metabolizable energy and FI is the feed intake. 2.3.3 Carcass and cut‐up parts At 38 and 46 days old, 2 birds from each pen (12 birds/treatment), after 6 h of feed withdrawal but had free access to drinking water, were randomly selected and slaughtered by severing the jugular vein, allowed bleeding for the 90 s, scalded for 2 min at 60°C, and picked (Zarghi et al., 2020). The gastrointestinal tract (GIT) was emptied, and abdominal fat was collected carefully. After carcass chilling for 24 h at 4°C (Tabatabaei et al., 2017), carcasses were weighed to determine the carcass yield as a percentage of LBW. The carcass parts – legs (thigh + drumstick), breast and frame (carcass without breast and legs) – were cut according to a standardized procedure (Tabatabaei et al., 2017) and weighed by a digital weighing scale (0.001−g; Model GF 400, A&D Weighing). 2.3.4 Breast meat's physicochemical properties The right side of the breast muscle was used to assess physicochemical properties. Water holding capacity (WHC) was measured as (100% expressed juice), following a modification of the press method (Zarghi et al., 2020). Cooking loss (CL) was measured as proportionate weight loss of a sample after cooking in an open plastic bag in a water bath at 70°C for 40 min, followed by cooling in cold‐running tap water for 15 min (Honikel, 1998). The equation is shown as follows: Cl%=Weightbeforecooking−WeightaftercookingWeightbeforecooking×100 Colour intensity was measured on breast muscle samples with a Chroma‐Meter (CR‐410 Konica Minolta, INC). The optical density of meat, lightness (L*), redness (a*) and yellowness (b*) values. The hue angle (h*) and Chroma (c*) were calculated using the following formula (Zarghi et al., 2020). The equations are shown as follows: h∗=tan−1b∗a∗ c∗=a∗2+b∗20.5 The left side of skinless and boneless breast meat, after freezing, was homogenized in a cutter for approximately 3 min until a homogeneous paste‐like mixture was obtained, then about 100 g of them were removed and freeze‐dried, which was then used to determine the breast meat composition such as moisture, crude protein, fat and ash according to standardized methods (AOAC, 2002). 2.4 Statistical analysis The experimental data were analysed for normality via the univariate procedure and then analysed by ANOVA using general linear model procedure of SAS 9.4 software (SAS Institute, 2014) appropriate for treatments in a randomized completely design with factorial arrangement. Means were compared using the Tukey test at the level of 0.05 probability (Tukey, 1991). Orthogonal polynomials contrast coefficients were used to determine linear and quadratic effects of increasing finisher diet ND levels on all measurements. To estimate the optimized responses, liner broken‐line, quadratic broken‐line and quadratic polynomial equation models were used via the NLIN procedure Gauss–Newton method of SAS 9.4 software. The iterative procedure makes repeated estimates for coefficients and minimizes residual error until the best‐fit lines are achieved (Robbins et al., 2006). The equations are shown as follows: ForlinearbrokenlineY=L+U×R−X×I ForquadraticbrokenlineY=L+U×R−X2×I ForquadraticpolynomialequationY=a+bX+cX2 R2=correctedtotalsumofsquares−sumsofsquaresforerrorcorrectedtotalsumofsquares where Y is the dependent variable, L is the theoretical maximum, R is the optimized dietary ND, X is the independent variable, I is the 1 (if X < R) or I = 0 (if X > R) and U, a, b and c = rate constant.\",\n",
       " '11196265': \"Material and methods Chemicals Polyethylene microplastics (PE-MPs) powder was supplied by Toxemerge Pty Ltd. (Melbourne, Australia). Purified water (Milli-Q) was used to create the base solution from the powder in accordance with the manufacturer's instructions and kept at 4 °C in the dark until characterization15. AK Scientific, Inc., Union City, CA, USA, provided tadalafil (Cilais) substance. Justification for drug dosage selection The dosage of tadalafil employed in our study was chosen based on previous research21. However, a study by Bianco et al. [21] indicated that there is no human health risk for exposure to sildenafil (LC50 = 42.74 for B. calyciflorus) and tadalafil (LC50 = no effect up to 20 for B. calyciflorus). However, they require special care as this form of contamination is found in any aquatic system evaluated. Another study found that the process of chlorination of sildenafil (LC50 = 0.13–0.27) and tadalafil (LC50 = 0.15–0.29) can be hazardous to fish that are exposed to it34. Therefore, we chose to utilize the maximum dosage of 20 mg/l of tadalafil in our research. Conversely, the PEMPs used in our investigation were selected based on earlier research15. Experimental design Fish (Oreochromis niloticus) (mean body weighing 242.7 ± 22.4 g; mean body length 27.3 ± 1.4 cm) were originated from private fish farm, Assiut, Egypt. They were healthy and parasite-free according to35. Oreochromis niloticus were acclimatized for 4 weeks in 96 L glass aquaria (80 × 40 × 30 cm) filled with dechlorinated water in the laboratory conditions: conductivity (261 ± 5.2 mM cm−1),water temperature (26 ± 1 °C), dissolved oxygen (7 ± 0.2 mg L−1), pH (7.4 ± 0.4), and photoperiod 12:12 h. Oreochromis niloticus were fed with a commercial fish pellet feed (containing 30% protein and 12% lipid; El Nasr Company, Egypt) at rate of 3% body biomass daily. One hundred and twenty fish (O. niloticus) were selected and divided into four treatments, each treatment contains 30 fish in triplicates (as 10 fish for each) in 96L glass aquarium water. The fish were distributed into 1st group control group (fish exposed to only water); 2nd group exposed to 10 mg/LPE-MPs15; 3rd group exposed to 20 mg/l tadalafil (Cilais)21; 4th group exposed to 20 mg/l tadalafil (Cilais) + 10 mg/LPE-MPs. Twelve fish from each group were chosen at random after 15 days of exposure and put unconscious with crushed ice36. Blood samples (1.5 ml) were collected from the caudal vein, part for hematological indices, and the other part was centrifuged undercooling for biochemical and antioxidants parameters. Liver, gills, and kidney tissues were used for histological and immunohistochemical studies. Hemato-biochemical parameters Hematological parameters such as the number of red blood cells (RBCs), white blood cells (WBCs), differential WBCs, blood platelets, hematocrit level (Hct) and hemoglobin level (Hb), as well as erythrocyte indices such as mean corpuscular hemoglobin (MCH), mean corpuscular volume (MCV), and mean corpuscular hemoglobin concentration (MCHC) were all measured using an automated technical analyzer (BC-2800 from Mindray). Blood samples for biochemical parameters were obtained from fish without the use of an anticoagulant agent to acquire serum. Colorimetric determinations of the following important biochemical indices: alanine aminotransferase (ALT), aspartate aminotransferase (AST), creatinine, uric acid, albumin, globulin, and glucose were carried out according to Hamed, Soliman15 by using a spectrophotometer in a wavelength range of 340–546 nm (Biodiagonstic Company, Egypt). Antioxidant parameters and Lipid peroxidation Reduced glutathione (GSH) concentration was evaluated according to Beutler et al.37. TAC (total antioxidant capacity) was determined utilizing kits (Sigma-Aldrich, USA). The malondialdehyde (MDA) level was determined using a thiobarbituric acid reaction38. Histopathological analysis Tissue samples of the gills, liver, and kidney were collected from three fish from each group and fixed in neutral buffered formalin before being processed using a standard automated process (dehydrated through graded ethanol concentration, cleared with methyl benzoate), embedded in paraffin wax, and sectioned (5 microns)39. De-waxed slides are rehydrated before being stained with hematoxylin and eosin (H & E)40. Finally, an Olympus CH30 microscope was used to analyze and photograph the specimen. Pathology severity was graded as follows: (−), absent; (+), minor; (++), moderate; (+++), severe; and a maximum score (++++)41. Immunohistochemical analysis An immunohistochemistry recoloring method was utilized to look at Interleukin-6 (IL-6). Four- µm -thick tissue segments were deparaffinized in xylene and rehydrated. To halt endogenous peroxidase action, the areas were submerged in 3% H2O2 for 10 min. After that, they were PBS-washed for 2 min. three times. The segments were at that point treated with standard goat serum per the prescribed procedure for 30 min at 37 °C (Vector Research facilities, Burlingame, CA), and after that for 1 h at room temperature with essential polyclonal IL-6 antibody; 1/50 (E-AB-40021, Elabscience Biotechnology Inc, USA). The expansion of polyperoxidase-anti-Mouse/Rabbit IgG is taken after 20 min. With the help of a streptavidin–biotin-peroxidase pack and Mayer's hematoxylin as a counterstain, the antigen–antibody complex was found. For each explore, positive and negative control areas were utilized. Intense brown nuclear and cytoplasmic staining assisted in identifying active IL-6 immunostaining cells. A light microscope was used to examine the slices in the 20 randomly selected locations (Olympus BX41, New York) at a magnification (50 ×). A blind fashion evaluation was carried out, with comparisons performed using the average score across all groups. The photomicrographs were taken with digital camera (canon digital camera (PowershotA95)). Statistical analysis The SPSS 16.0 program for Windows (SPSS 2007, Inc., IL, USA) was used to analyze the data, and 0.05 was regarded as the point of significance. The Shapiro–Wilk test was used to determine whether the data was normal. ANOVA was used to determine the homogeneity of variances after the one-way analysis of variance (Levene’s test). Fisher’s LSD post hoc analysis was used to compare the treatment groups to the control group in the case of variance equality. Dunnett’s post hoc test was used to compare the treatment groups to the control group where there was variance inequality. Ethical statement The ethics committee of the Faculty of Veterinary Medicine, Assiut University, Assiut, Egypt, authorized the experimental setup and fish handling in accordance with the OIE criteria for the use of animals in research. All methods were performed in accordance with the relevant guidelines and regulations and in accordance with ARRIVE guidelines (No, 06\\\\2023\\\\0037).\",\n",
       " '11196252': 'Materials and methods ITIES studies All electroanalytical studies were conducted using an AUTOLAB-PGSTAT302N instrument (Metrohm Autolab B.V., The Netherlands) controlled via NOVA 1.11.1 software. Measurements were carried out with a four-electrode electrochemical system (Fig. 1B) comprising of two platinum (Pt) wires serving as the counter electrodes and two silver/silver chloride wires (Ag/AgCl) functioning as reference electrodes (as depicted in Scheme 1). The Galvani potential difference of the studied ions was calculated using the data from cyclic voltammograms whereas the potential axis was calibrated using the standard Galvani potential difference of the tetrapropylammonium cation ion transfer (TPrA+, \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{mathrsfs} \\\\usepackage{upgreek} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document}$${\\\\Delta }_{org}^{aq}{\\\\Phi }_{{TPrA}^{+}}^{0}=-0.091 V$$\\\\end{document}ΔorgaqΦTPrA+0=-0.091V)29. Cells from Schemes 1 and 2 were used during this study.Scheme 1Electrochemical cell employed for electroanalytical studies of DANO at ITIES.Scheme 2Electrochemical cell used to evaluate the effect of pH on the DANA behavior at ITIES. Measurements at the GCE This part of measurements were conducted using an EmStat3 potentiostat under the control of PSTrace software supplied by PalmSens B.V., the Netherlands. All experiments were carried out in conjunction with an automated electrode stand (M164, MTM Anko Instruments, Cracow, Poland). The electrochemical three-electrode setup (Fig. 1C) consists of glassy carbon electrode (GCE) with a 3 mm diameter and a geometric area of 7.1 mm2 (Basi®, USA) as the working electrode, silver/silver chloride electrode (Ag/AgCl/3.0 mol L−1 KCl, MTM Anko Instruments, Kraków, Poland) as a reference electrode and a platinum wire (Pt, 99.99%, The Mint of Poland, Warsaw, Poland) as an auxiliary electrode. Square-wave voltammetry (SWV) and cyclic voltammetry (CV) were employed as the electroanalytical techniques to investigate DANO behavior at the GCE. Reagents and solutions All chemicals utilized in this study were of analytical reagent grade. DANO with a purity of ≥ 98% was procured from Merck. Fresh stock solutions of DANO (10 mM) were meticulously prepared in graduated glass flasks by dissolving the calculated quantity of DANO in the appropriate volume of 10 mM hydrochloric acid (HCl) (studies at the ITIES) or deionized water (studies at the GCE). Potassium tetrakis(4-chlorophenyl)borate (KTPBCl, > 98%) and bis(triphenylphosphoranylidene) ammonium chloride (BTPPACl, 97%) were obtained from Merck and served as substrates for synthesizing bis(triphenylphosphoranylidene)ammonium tetrakis(4-chlorophenyl)borate (BTPPATPBCl) which was used as the background electrolyte in the organic phase. Tetrapropylammonium chloride (TPrACl, Alfa Aesar, > 99%,) served as the interfacially active internal reference probe used to calibrate the potential axis to the Galvani potential difference. The chemicals for the preparation of Britton–Robinson buffers (BRBs) were procured from POCH. A BRB matrix with a concentration of 40 mM was formulated by dissolving H3BO3, H3PO4, and CH3COOH in 10 mM NaCl (ITIES) or deionized water (GCE). The BRB matrix was titrated with 0.2 M NaOH to achieve the required pH values spanning through the range 2.0–12.0. 1,2–dichloroethane (1,2–DCE, POCH) and deionized water were employed as the solvents for the preparation of solutions in the organic and aqueous phases, respectively. Analytical standards for potentially interfering agents (citric acid, galactose, lactose, glucose, calcium chloride, potassium chloride, magnesium chloride, iron(III) chloride, sodium lactate and orthophosphoric acid (V)) were procured in their analytical reagent-grade from different suppliers (Sigma–Aldrich, Chempur, Fisher Chemical or Alfa Aesar). Fresh stock solutions of interfering species (10 mM) were prepared in a glass graduated flasks by dissolving the appropriate amount of chosen analyte in 10 mL of 10 mM HCl. Only in the case of orthophosphoric acid (V) deionized water was used as a solvent. Shortly after preparation all solutions were subjected to 10 min of sonification and were stored at 4.0 °C. The real samples were ultra high temperature (UHT) milk (1.5% fat content), which were obtained from a local supermarket. The milk samples were not subjected to any preparation process. The pH of the BRBs and the aqueous phase solutions was adjusted utilizing a combined pH electrode (Polilyte Lab, Hamilton, Switzerland).',\n",
       " '11196211': ''}"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "method_dict"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Identify Product Entities"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "entity_dict = {}\n",
    "for key, value in method_dict.items():\n",
    "    if value:\n",
    "        doc = nlp(value)\n",
    "        entities = []\n",
    "        for ent in doc.ents:\n",
    "            if ent.label_ == 'PRODUCT' and ent.text not in entities:\n",
    "                entities.append(ent.text)\n",
    "        entity_dict[key] = entities"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'11198919': ['Rome IV',\n",
       "  'the Statistical Package for the Social Sciences',\n",
       "  'SPSS',\n",
       "  'Windows'],\n",
       " '11198832': ['Western Blot',\n",
       "  'SCIEX',\n",
       "  '2720',\n",
       "  'FLX800',\n",
       "  'Nanodrop UV',\n",
       "  'NC2000',\n",
       "  'Sequencer',\n",
       "  'Novaseq6000',\n",
       "  'Quant-iT PicoGreen dsDNA Assay Kit',\n",
       "  'First Strand',\n",
       "  'SYBR Green',\n",
       "  'Servicebio',\n",
       "  'GAPDH',\n",
       "  'The Odyssey CLx Imaging System'],\n",
       " '11198784': ['Ficoll-Paque Premium',\n",
       "  'TC20 Automated cell counter',\n",
       "  'an EasySep Negative Human NK Cell Isolation Kit',\n",
       "  'Micro Forge MF-900',\n",
       "  'Axopatch 200B',\n",
       "  'PregS',\n",
       "  'Ononetin',\n",
       "  'GraphPad Prism v9',\n",
       "  'GraphPad Prism version'],\n",
       " '11198029': ['Embase'],\n",
       " '11197919': ['BioRender',\n",
       "  'RS4',\n",
       "  'a Tanita SC-240 Total Body Composition Analyzer',\n",
       "  'the Automated Self-Administered 24-hour',\n",
       "  'ASA24',\n",
       "  'the QIAamp 96 DNA Blood Kit',\n",
       "  'the QIAamp Investigator Kit',\n",
       "  'iTaq™ Universal SYBR® Green Supermix',\n",
       "  'NA12286',\n",
       "  'AMY1',\n",
       "  'a QX100 Droplet Digital PCR System',\n",
       "  'SALIMETRICS',\n",
       "  'the DNeasy PowerSoil 96 HTP Kit',\n",
       "  'Qiagen',\n",
       "  'a BioSpec 1001 Mini-Beadbeater-96 for 3 minutes',\n",
       "  'Mag-Bind TotalPure',\n",
       "  'Omega Bio',\n",
       "  'MiSeq',\n",
       "  'DADA233',\n",
       "  'RStudio',\n",
       "  'ComplexHeatmap40',\n",
       "  'circlize41',\n",
       "  'RS2',\n",
       "  'Ctl',\n",
       "  'PreRS2',\n",
       "  'EndRS2',\n",
       "  'MaAsLin2',\n",
       "  'PreRS4',\n",
       "  'EndRS4',\n",
       "  'glmnet43',\n",
       "  'ASVs'],\n",
       " '11197841': [],\n",
       " '11197476': ['a Malvern Nano ZS ZEN3600',\n",
       "  'Bam HI',\n",
       "  'Eco RI',\n",
       "  'CT26',\n",
       "  'Ki67',\n",
       "  'C@ECN-PL'],\n",
       " '11197382': [],\n",
       " '11197378': ['Gaode Map'],\n",
       " '11197282': ['pheatmap'],\n",
       " '11197204': ['TRIzol',\n",
       "  'Easy Oligos',\n",
       "  'iTaq Universal One-Step',\n",
       "  'iTaq™ Universal SYBR® Green One-Step',\n",
       "  'Tools',\n",
       "  'GraphPad PRISM',\n",
       "  'Enrichr',\n",
       "  'ShinyGO'],\n",
       " '11197185': ['MiSeq',\n",
       "  'F15',\n",
       "  'Hemotek',\n",
       "  'Mosquito Diluent',\n",
       "  'Trizol',\n",
       "  'AccuStart II',\n",
       "  'MiSeq 500',\n",
       "  'MicrobiomeAnalyst',\n",
       "  '16S'],\n",
       " '11196966': ['Advanced DMEM-F12',\n",
       "  'C57Bl6/J-mice',\n",
       "  'TC20',\n",
       "  'Isoflurane Piramal',\n",
       "  'the SomnoSuite® Low-Flow Anesthesia System',\n",
       "  'BioSpec 94/21',\n",
       "  'Bruker',\n",
       "  'Isoflurane',\n",
       "  'the Liver Dissociation Kit',\n",
       "  'Miltenyi Biotec',\n",
       "  'the Tumor Dissociation Kit',\n",
       "  'SSC-W.',\n",
       "  'Cell Activation Cocktail',\n",
       "  'BioLegend',\n",
       "  'Brefeldin A',\n",
       "  'the RNeasy Mini Kit',\n",
       "  'ProtoScript II First Strand cDNA Synthesis Kit',\n",
       "  'the DNeasy Blood & Tissue Kit',\n",
       "  'Qiagen',\n",
       "  'QuantiTect SYBR Green RT PCR mastermix',\n",
       "  'LightCycler480',\n",
       "  'Roche',\n",
       "  'the QuantiFast SYBR® Green PCR Kit',\n",
       "  'FASTQC',\n",
       "  'DESeq2',\n",
       "  'ggplot2',\n",
       "  'ggraph',\n",
       "  'ComplexHeatmaps',\n",
       "  'a Pierce BCA-Kit',\n",
       "  'an Azure Ao Microplate Reader',\n",
       "  'BioRad',\n",
       "  'Mini-Protean TGX',\n",
       "  'Supplements',\n",
       "  'Immobilon Crescendo Western HRP substrate',\n",
       "  'ROTI® Histokitt',\n",
       "  'a Zeiss Axio Scan Z.1 Microscope Slide Scanner',\n",
       "  'the Vector® TrueVIEW® Autofluorescence Quenching Kit',\n",
       "  'NucBlue™ Fixed Cell Stain ReadyProbes™',\n",
       "  'VECTASHIELD Vibrance Antifade Mounting Medium',\n",
       "  'GraphPad Prism'],\n",
       " '11196847': ['the ABI 3130 Genetic Analyzer',\n",
       "  'Phylogenetic Tree Builder',\n",
       "  '12M',\n",
       "  'RP C-18 column'],\n",
       " '11196703': ['ENSO 16', 'NEOH', 'GraphPad Prism 9'],\n",
       " '11196440': ['Embase'],\n",
       " '11196417': ['GSE208668',\n",
       "  'GPL10904',\n",
       "  'GSE240851',\n",
       "  'GSE56931',\n",
       "  'GSE98582',\n",
       "  'GPL24676',\n",
       "  'GPL10379',\n",
       "  'GPL6244',\n",
       "  'GSE165041',\n",
       "  'GPL18573',\n",
       "  'ggplot2',\n",
       "  'Cytoscape V3.9.0',\n",
       "  '1071',\n",
       "  'the TransZol Up Plus RNA Kit',\n",
       "  'the Nanodrop Spectrophotometer',\n",
       "  'the TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix',\n",
       "  'AT311',\n",
       "  'the ChamQ Universal SYBR qPCR Master Mix',\n",
       "  'Q711',\n",
       "  'a QuantStudio™ 5',\n",
       "  'Image J'],\n",
       " '11196393': ['RPMI1640',\n",
       "  'BD FACSCanto™ II',\n",
       "  'Brilliant Violet 605™',\n",
       "  'GEO Series',\n",
       "  'Cell Ranger',\n",
       "  'Seurat',\n",
       "  'GraphPad Prism 8.0'],\n",
       " '11196377': ['Model GF 400', 'a Chroma‐Meter (CR‐410 Konica Minolta, INC'],\n",
       " '11196265': ['Milli-Q', 'CH30', 'PowershotA95'],\n",
       " '11196252': ['AUTOLAB-PGSTAT302N',\n",
       "  'NOVA 1.11.1',\n",
       "  'EmStat3',\n",
       "  'PSTrace',\n",
       "  'M164']}"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "entity_dict"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Filtering Products with LLM"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "openai.api_key = os.environ['OPENAI_API_KEY']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "system_prompt = \"\"\"You are an expert Physical Commercial Product Recognition system.\n",
    "Your task is to accept a list of strings as input and extract the physical commercial products.\n",
    "Here is an example of the output format for a list of strings.\n",
    "\n",
    "List of strings: [\"Python\", \"Researchers used 3M Littmann Stethoscope\", \"Omron Blood Pressure Monitor is important\", \"Microsoft Office\", \"Braun ThermoScan\", \"SPSS\"]\n",
    "Answer: [\"3M Littmann Stethoscope\", \"Omron Blood Pressure Monitor\", \"Braun ThermoScan\"]\n",
    "\n",
    "Only use this output format. Do not return anything besides this output format.\n",
    "Output physical commercial products in the order they occur in the input list of strings.\n",
    "\"\"\"\n",
    "\n",
    "user_prompt = \"\"\"Q: Given the list of strings below, identify the physical commercial products.\n",
    "\n",
    "List of strings: {}\n",
    "Answer:\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "def product_filtering(list_of_products):\n",
    "    response = openai.chat.completions.create(\n",
    "        model=\"gpt-4-turbo\",\n",
    "        presence_penalty=0,\n",
    "        # top_p=1e-16,\n",
    "        temperature=1e-16,\n",
    "        messages=[\n",
    "            {\"role\": \"system\", \"content\": system_prompt},\n",
    "            {\"role\": \"user\", \"content\": user_prompt.format(list_of_products)}\n",
    "        ]\n",
    "    )\n",
    "    return response.choices[0].message.content"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198919: []\n",
      "11198832: [\"Nanodrop UV\", \"Novaseq6000\", \"Quant-iT PicoGreen dsDNA Assay Kit\", \"The Odyssey CLx Imaging System\"]\n",
      "11198784: [\"Ficoll-Paque Premium\", \"TC20 Automated cell counter\", \"EasySep Negative Human NK Cell Isolation Kit\", \"Micro Forge MF-900\", \"Axopatch 200B\"]\n",
      "11198029: []\n",
      "11197919: [\"Tanita SC-240 Total Body Composition Analyzer\", \"QIAamp 96 DNA Blood Kit\", \"QIAamp Investigator Kit\", \"QX100 Droplet Digital PCR System\", \"DNeasy PowerSoil 96 HTP Kit\", \"BioSpec 1001 Mini-Beadbeater-96\"]\n",
      "11197476: [\"Malvern Nano ZS ZEN3600\"]\n",
      "11197378: []\n",
      "11197282: []\n",
      "11197204: [\"TRIzol\", \"Easy Oligos\", \"iTaq Universal One-Step\", \"iTaq™ Universal SYBR® Green One-Step\"]\n",
      "11197185: [\"MiSeq\", \"Hemotek\", \"Trizol\", \"AccuStart II\", \"MiSeq 500\"]\n",
      "11196966: [\"Isoflurane Piramal\", \"the SomnoSuite® Low-Flow Anesthesia System\", \"the Liver Dissociation Kit\", \"the Tumor Dissociation Kit\", \"the RNeasy Mini Kit\", \"ProtoScript II First Strand cDNA Synthesis Kit\", \"the DNeasy Blood & Tissue Kit\", \"the QuantiFast SYBR® Green PCR Kit\", \"a Pierce BCA-Kit\", \"an Azure Ao Microplate Reader\", \"a Zeiss Axio Scan Z.1 Microscope Slide Scanner\", \"the Vector® TrueVIEW® Autofluorescence Quenching Kit\", \"NucBlue™ Fixed Cell Stain ReadyProbes™\", \"VECTASHIELD Vibrance Antifade Mounting Medium\"]\n",
      "11196847: [\"ABI 3130 Genetic Analyzer\", \"RP C-18 column\"]\n",
      "11196703: [\"ENSO 16\", \"NEOH\"]\n",
      "11196440: []\n",
      "11196417: [\"TransZol Up Plus RNA Kit\", \"Nanodrop Spectrophotometer\", \"TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix\", \"ChamQ Universal SYBR qPCR Master Mix\", \"QuantStudio™ 5\"]\n",
      "11196393: [\"BD FACSCanto™ II\", \"Brilliant Violet 605™\", \"GraphPad Prism 8.0\"]\n",
      "11196377: [\"Chroma‐Meter (CR‐410 Konica Minolta, INC\"]\n",
      "11196265: [\"PowershotA95\"]\n",
      "11196252: [\"AUTOLAB-PGSTAT302N\", \"EmStat3\", \"M164\"]\n"
     ]
    }
   ],
   "source": [
    "filtered_entity_dict = {}\n",
    "syntax_error = False\n",
    "for key, value in entity_dict.items():\n",
    "    if value:\n",
    "        filtered_products = product_filtering(value)\n",
    "        try:\n",
    "            filtered_products = ast.literal_eval(filtered_products)\n",
    "        except SyntaxError:\n",
    "            syntax_error = True\n",
    "            print(f\"Syntax Error with ID {key}\")\n",
    "        if not syntax_error:\n",
    "            filtered_entity_dict[key] = filtered_products"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198832: ['Nanodrop UV', 'Novaseq6000', 'Quant-iT PicoGreen dsDNA Assay Kit', 'The Odyssey CLx Imaging System']\n",
      "11198784: ['Ficoll-Paque Premium', 'TC20 Automated cell counter', 'EasySep Negative Human NK Cell Isolation Kit', 'Micro Forge MF-900', 'Axopatch 200B']\n",
      "11197919: ['Tanita SC-240 Total Body Composition Analyzer', 'QIAamp 96 DNA Blood Kit', 'QIAamp Investigator Kit', 'QX100 Droplet Digital PCR System', 'DNeasy PowerSoil 96 HTP Kit', 'BioSpec 1001 Mini-Beadbeater-96']\n",
      "11197476: ['Malvern Nano ZS ZEN3600']\n",
      "11197204: ['TRIzol', 'Easy Oligos', 'iTaq Universal One-Step', 'iTaq™ Universal SYBR® Green One-Step']\n",
      "11197185: ['MiSeq', 'Hemotek', 'Trizol', 'AccuStart II', 'MiSeq 500']\n",
      "11196966: ['Isoflurane Piramal', 'the SomnoSuite® Low-Flow Anesthesia System', 'the Liver Dissociation Kit', 'the Tumor Dissociation Kit', 'the RNeasy Mini Kit', 'ProtoScript II First Strand cDNA Synthesis Kit', 'the DNeasy Blood & Tissue Kit', 'the QuantiFast SYBR® Green PCR Kit', 'a Pierce BCA-Kit', 'an Azure Ao Microplate Reader', 'a Zeiss Axio Scan Z.1 Microscope Slide Scanner', 'the Vector® TrueVIEW® Autofluorescence Quenching Kit', 'NucBlue™ Fixed Cell Stain ReadyProbes™', 'VECTASHIELD Vibrance Antifade Mounting Medium']\n",
      "11196847: ['ABI 3130 Genetic Analyzer', 'RP C-18 column']\n",
      "11196703: ['ENSO 16', 'NEOH']\n",
      "11196417: ['TransZol Up Plus RNA Kit', 'Nanodrop Spectrophotometer', 'TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix', 'ChamQ Universal SYBR qPCR Master Mix', 'QuantStudio™ 5']\n",
      "11196393: ['BD FACSCanto™ II', 'Brilliant Violet 605™', 'GraphPad Prism 8.0']\n",
      "11196377: ['Chroma‐Meter (CR‐410 Konica Minolta, INC']\n",
      "11196265: ['PowershotA95']\n",
      "11196252: ['AUTOLAB-PGSTAT302N', 'EmStat3', 'M164']\n"
     ]
    }
   ],
   "source": [
    "for key, value in filtered_entity_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198919: ['Rome IV', 'the Statistical Package for the Social Sciences', 'SPSS', 'Windows']\n",
      "11198832: ['Western Blot', 'SCIEX', '2720', 'FLX800', 'Nanodrop UV', 'NC2000', 'Sequencer', 'Novaseq6000', 'Quant-iT PicoGreen dsDNA Assay Kit', 'First Strand', 'SYBR Green', 'Servicebio', 'GAPDH', 'The Odyssey CLx Imaging System']\n",
      "11198784: ['Ficoll-Paque Premium', 'TC20 Automated cell counter', 'an EasySep Negative Human NK Cell Isolation Kit', 'Micro Forge MF-900', 'Axopatch 200B', 'PregS', 'Ononetin', 'GraphPad Prism v9', 'GraphPad Prism version']\n",
      "11198029: ['Embase']\n",
      "11197919: ['BioRender', 'RS4', 'a Tanita SC-240 Total Body Composition Analyzer', 'the Automated Self-Administered 24-hour', 'ASA24', 'the QIAamp 96 DNA Blood Kit', 'the QIAamp Investigator Kit', 'iTaq™ Universal SYBR® Green Supermix', 'NA12286', 'AMY1', 'a QX100 Droplet Digital PCR System', 'SALIMETRICS', 'the DNeasy PowerSoil 96 HTP Kit', 'Qiagen', 'a BioSpec 1001 Mini-Beadbeater-96 for 3 minutes', 'Mag-Bind TotalPure', 'Omega Bio', 'MiSeq', 'DADA233', 'RStudio', 'ComplexHeatmap40', 'circlize41', 'RS2', 'Ctl', 'PreRS2', 'EndRS2', 'MaAsLin2', 'PreRS4', 'EndRS4', 'glmnet43', 'ASVs']\n",
      "11197476: ['a Malvern Nano ZS ZEN3600', 'Bam HI', 'Eco RI', 'CT26', 'Ki67', 'C@ECN-PL']\n",
      "11197378: ['Gaode Map']\n",
      "11197282: ['pheatmap']\n",
      "11197204: ['TRIzol', 'Easy Oligos', 'iTaq Universal One-Step', 'iTaq™ Universal SYBR® Green One-Step', 'Tools', 'GraphPad PRISM', 'Enrichr', 'ShinyGO']\n",
      "11197185: ['MiSeq', 'F15', 'Hemotek', 'Mosquito Diluent', 'Trizol', 'AccuStart II', 'MiSeq 500', 'MicrobiomeAnalyst', '16S']\n",
      "11196966: ['Advanced DMEM-F12', 'C57Bl6/J-mice', 'TC20', 'Isoflurane Piramal', 'the SomnoSuite® Low-Flow Anesthesia System', 'BioSpec 94/21', 'Bruker', 'Isoflurane', 'the Liver Dissociation Kit', 'Miltenyi Biotec', 'the Tumor Dissociation Kit', 'SSC-W.', 'Cell Activation Cocktail', 'BioLegend', 'Brefeldin A', 'the RNeasy Mini Kit', 'ProtoScript II First Strand cDNA Synthesis Kit', 'the DNeasy Blood & Tissue Kit', 'Qiagen', 'QuantiTect SYBR Green RT PCR mastermix', 'LightCycler480', 'Roche', 'the QuantiFast SYBR® Green PCR Kit', 'FASTQC', 'DESeq2', 'ggplot2', 'ggraph', 'ComplexHeatmaps', 'a Pierce BCA-Kit', 'an Azure Ao Microplate Reader', 'BioRad', 'Mini-Protean TGX', 'Supplements', 'Immobilon Crescendo Western HRP substrate', 'ROTI® Histokitt', 'a Zeiss Axio Scan Z.1 Microscope Slide Scanner', 'the Vector® TrueVIEW® Autofluorescence Quenching Kit', 'NucBlue™ Fixed Cell Stain ReadyProbes™', 'VECTASHIELD Vibrance Antifade Mounting Medium', 'GraphPad Prism']\n",
      "11196847: ['the ABI 3130 Genetic Analyzer', 'Phylogenetic Tree Builder', '12M', 'RP C-18 column']\n",
      "11196703: ['ENSO 16', 'NEOH', 'GraphPad Prism 9']\n",
      "11196440: ['Embase']\n",
      "11196417: ['GSE208668', 'GPL10904', 'GSE240851', 'GSE56931', 'GSE98582', 'GPL24676', 'GPL10379', 'GPL6244', 'GSE165041', 'GPL18573', 'ggplot2', 'Cytoscape V3.9.0', '1071', 'the TransZol Up Plus RNA Kit', 'the Nanodrop Spectrophotometer', 'the TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix', 'AT311', 'the ChamQ Universal SYBR qPCR Master Mix', 'Q711', 'a QuantStudio™ 5', 'Image J']\n",
      "11196393: ['RPMI1640', 'BD FACSCanto™ II', 'Brilliant Violet 605™', 'GEO Series', 'Cell Ranger', 'Seurat', 'GraphPad Prism 8.0']\n",
      "11196377: ['Model GF 400', 'a Chroma‐Meter (CR‐410 Konica Minolta, INC']\n",
      "11196265: ['Milli-Q', 'CH30', 'PowershotA95']\n",
      "11196252: ['AUTOLAB-PGSTAT302N', 'NOVA 1.11.1', 'EmStat3', 'PSTrace', 'M164']\n"
     ]
    }
   ],
   "source": [
    "for key, value in entity_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Cleaning Product Entities"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "def remove_stopwords(input_string):\n",
    "    # can add more stopwords here\n",
    "    # use NLTK if these stopwords are not enough\n",
    "    stopwords = ['a', 'an', 'the', 'of']\n",
    "    tokens = input_string.split()\n",
    "    token_list = []\n",
    "    for token in tokens:\n",
    "        if token not in stopwords:\n",
    "            token_list.append(token)\n",
    "    output_string = \" \".join(token_list)\n",
    "    return output_string"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "cleaned_entity_dict = {}\n",
    "for key, value in filtered_entity_dict.items():\n",
    "    if value:\n",
    "        cleaned_entities = [remove_stopwords(entity) for entity in value]\n",
    "        cleaned_entity_dict[key] = cleaned_entities"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198832: ['Nanodrop UV', 'Novaseq6000', 'Quant-iT PicoGreen dsDNA Assay Kit', 'The Odyssey CLx Imaging System']\n",
      "11198784: ['Ficoll-Paque Premium', 'TC20 Automated cell counter', 'EasySep Negative Human NK Cell Isolation Kit', 'Micro Forge MF-900', 'Axopatch 200B']\n",
      "11197919: ['Tanita SC-240 Total Body Composition Analyzer', 'QIAamp 96 DNA Blood Kit', 'QIAamp Investigator Kit', 'QX100 Droplet Digital PCR System', 'DNeasy PowerSoil 96 HTP Kit', 'BioSpec 1001 Mini-Beadbeater-96']\n",
      "11197476: ['Malvern Nano ZS ZEN3600']\n",
      "11197204: ['TRIzol', 'Easy Oligos', 'iTaq Universal One-Step', 'iTaq™ Universal SYBR® Green One-Step']\n",
      "11197185: ['MiSeq', 'Hemotek', 'Trizol', 'AccuStart II', 'MiSeq 500']\n",
      "11196966: ['Isoflurane Piramal', 'SomnoSuite® Low-Flow Anesthesia System', 'Liver Dissociation Kit', 'Tumor Dissociation Kit', 'RNeasy Mini Kit', 'ProtoScript II First Strand cDNA Synthesis Kit', 'DNeasy Blood & Tissue Kit', 'QuantiFast SYBR® Green PCR Kit', 'Pierce BCA-Kit', 'Azure Ao Microplate Reader', 'Zeiss Axio Scan Z.1 Microscope Slide Scanner', 'Vector® TrueVIEW® Autofluorescence Quenching Kit', 'NucBlue™ Fixed Cell Stain ReadyProbes™', 'VECTASHIELD Vibrance Antifade Mounting Medium']\n",
      "11196847: ['ABI 3130 Genetic Analyzer', 'RP C-18 column']\n",
      "11196703: ['ENSO 16', 'NEOH']\n",
      "11196417: ['TransZol Up Plus RNA Kit', 'Nanodrop Spectrophotometer', 'TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix', 'ChamQ Universal SYBR qPCR Master Mix', 'QuantStudio™ 5']\n",
      "11196393: ['BD FACSCanto™ II', 'Brilliant Violet 605™', 'GraphPad Prism 8.0']\n",
      "11196377: ['Chroma‐Meter (CR‐410 Konica Minolta, INC']\n",
      "11196265: ['PowershotA95']\n",
      "11196252: ['AUTOLAB-PGSTAT302N', 'EmStat3', 'M164']\n"
     ]
    }
   ],
   "source": [
    "for key, value in cleaned_entity_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Identify Products with LLM"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "system_prompt = \"\"\"You are an expert Physical Commercial Product Recognition system.\n",
    "Your task is to accept a text as input and extract the physical commercial products.\n",
    "Here is an example of the output format for a list of strings.\n",
    "\n",
    "Text: I started my day with a fresh cup of coffee brewed from my Keurig coffee maker, and then I prepared breakfast using my new KitchenAid stand mixer. After breakfast, I grabbed my Nike running shoes and went for a jog in the park. On my way back, I stopped by the store to pick up a pack of Tide laundry detergent and a bottle of Coca-Cola.\n",
    "Answer: [\"Keurig coffee maker\", \"KitchenAid stand mixer\", \"Nike running shoes\", \"Tide laundry detergent\", \"Coca-Cola\"]\n",
    "\n",
    "Only use this output format. Do not return anything besides this output format.\n",
    "Output physical commercial products in the order they occur in the input text.\n",
    "\"\"\"\n",
    "\n",
    "user_prompt = \"\"\"Q: Given the text below, identify the physical commercial products.\n",
    "\n",
    "Text: {}\n",
    "Answer:\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "def product_recognition(input_text):\n",
    "    response = openai.chat.completions.create(\n",
    "        model=\"gpt-4-turbo\",\n",
    "        presence_penalty=0,\n",
    "        # top_p=1e-16,\n",
    "        temperature=1e-16,\n",
    "        messages=[\n",
    "            {\"role\": \"system\", \"content\": system_prompt},\n",
    "            {\"role\": \"user\", \"content\": user_prompt.format(input_text)}\n",
    "        ]\n",
    "    )\n",
    "    return response.choices[0].message.content"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "product_dict = {}\n",
    "syntax_error = False\n",
    "for key, value in method_dict.items():\n",
    "    if value:\n",
    "        identified_products = product_recognition(value)\n",
    "        try:\n",
    "            identified_products = ast.literal_eval(identified_products)\n",
    "        except SyntaxError:\n",
    "            syntax_error = True\n",
    "            print(f\"Syntax Error with ID {key}\")\n",
    "        if not syntax_error:\n",
    "            product_dict[key] = identified_products"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198832: ['high-performance liquid chromatography (HPLC) (Shimadzu, LC-20AD)', 'SCIEX mass spectrometer (MS) (AB SCIEX, API 4000+)', 'PCR amplification instrument (ABI, 2720)', 'enzyme labeling instrument (BioTek, FLX800T)', 'electrophoresis apparatus (Beijing Liuyi Instrument Factory, DYY-6C)', 'gel imaging system (Beijing Bijing Biotechnology Co., LTD., BG-gdsAUTO130)', 'Nanodrop UV quantitative system (Thermo Fisher Scientific, NC2000)', 'Sequencer (Illumina, Novaseq6000)', 'fluorescence quantitative PCR instrument (Bio-rad Corporation, CFX)', 'vancomycin hydrochloride for injection (VIANEX S.A.)', 'bifidobacterium quadruplex viable tablets (Hangzhou Yuanda Biological Pharmaceutical)', 'NBP capsules (Shiyao Group Enbipu Pharmaceutical)', 'NBP reference (Shiyao Group Enbipu Pharmaceutical)', 'glipizide reference (Sichuan Vicchi Biochemical Technology)', 'Soil DNA Kit (Omega Bio-Tek)', 'Quant-iT PicoGreen dsDNA Assay Kit (ABI)', 'First Strand cDNA Synthesis Kit (Servicebio)', 'SYBR Green qPCR Master Mix (Servicebio)']\n",
      "11198784: ['Ficoll-Paque Premium', 'trypan blue dye', 'TC20 Automated cell counter', 'EasySep Negative Human NK Cell Isolation Kit', 'CD56 APC', 'CD3 PE Cy7', 'stain buffer', 'BD LSR- FortessaTM X-20 flow cytometer', 'borosilicate glass capillaries', 'Sutter Instrumental model P-97', 'Narishige Micro Forge MF-900', 'CV203BU head-stage', 'Axopatch 200B amplifier', 'pClamp 10.7 software', 'Digidata 1440A analogue to digital converter', '0.22 m membrane filter', 'Sapphire Bioscience Reagents', 'Tocris Bioscience']\n",
      "11197919: ['HI-MAIZE® 260 starch', 'VERSAFIBE™ 1490 starch', 'AMIOCA™ TF starch', 'Tanita SC-240 Total Body Composition Analyzer', 'Automated Self-Administered 24-hour (ASA24) Dietary Assessment Tool', 'Salimetrics Salivary Alpha-Amylase Enzymatic Kit', 'Acquity UPLC system', 'DNeasy PowerSoil 96 HTP Kit', 'Classic++™ Hot Start Taq DNA Polymerase Master Mix', 'Mag-Bind TotalPure', 'QIAamp 96 DNA Blood Kit', 'QIAamp Blood Mini Kit', 'QIAamp Investigator Kit', 'iTaq™ Universal SYBR® Green Supermix']\n",
      "11197476: ['LiposoFast', 'Merck Millipore Ltd.', 'Malvern Nano ZS ZEN3600', 'Thermo Fisher Scientific', 'CytoFLEX', 'JEM-1400', 'Nikon', 'CCK-8 assay kit', 'FITC-conjugated anti-COX-2 antibody', 'in vivo images system', 'Biolegend Inc.']\n",
      "11197382: ['COBAS Integra 400']\n",
      "11197204: ['TRIzol', 'TaqMan™ Gene Expression Assay', 'iTaq Universal One-Step', 'iTaq™ Universal SYBR® Green One-Step Kit', 'GraphPad PRISM']\n",
      "11197185: ['Sterilite container', 'Tetra Pond Koi growth feed', 'Hemotek membrane feeding system', 'Triethylamine', 'Mosquito Diluent', 'Trizol', 'REPLI-g whole transcriptome amplification single-cell kit', 'AccuStart II PCR supermix']\n",
      "11196966: ['Advanced DMEM-F12', 'N-acetylcysteine', 'SomnoSuite® Low-Flow Anesthesia System', 'Seprafilm', '6–0 Vicryl', 'VEVO 3100', 'BioSpec 94/21', 'Liver Dissociation Kit', 'Tumor Dissociation Kit', 'ACK lysis buffer', 'eBioscience™ Foxp3/Transcription Factor Staining Buffer Set', 'RNeasy Mini Kit', 'ProtoScript II First Strand cDNA Synthesis Kit', 'DNeasy Blood & Tissue Kit', 'QuantiFast SYBR® Green PCR Kit', 'Pierce BCA-Kit', 'Mini-Protean TGX precast gel', 'trans-blot turbo transfer system', 'Immobilon Crescendo Western HRP substrate', 'ChemiDoc BioRad imager', 'Vector® TrueVIEW® Autofluorescence Quenching Kit', 'NucBlue™ Fixed Cell Stain ReadyProbes™', 'VECTASHIELD Vibrance Antifade Mounting Medium']\n",
      "11196847: ['ABI 3130 Genetic Analyzer', 'Whatman paper No. 4', 'Megazyme kit']\n",
      "11196703: ['ENSO 16', 'glucose powder']\n",
      "11196393: ['BD FACSAria III cell sorter', 'Ficoll gradient', 'ACK lysing buffer', 'BD FACSCanto™ II', 'Cell Ranger', 'Seurat', 'monocle2', 'GraphPad Prism']\n",
      "11196377: ['Cobb-500', 'Evonik Nutrition & Care GmbH', 'UFFDA software', 'Model GF 400 A&D Weighing', 'CR-410 Konica Minolta']\n",
      "11196265: ['commercial fish pellet feed']\n",
      "11196252: ['AUTOLAB-PGSTAT302N instrument', 'EmStat3 potentiostat', 'automated electrode stand (M164)', 'glassy carbon electrode (GCE)', 'silver/silver chloride electrode (Ag/AgCl/3.0 mol L−1 KCl)', 'platinum wire (Pt, 99.99%)', 'combined pH electrode (Polilyte Lab)']\n"
     ]
    }
   ],
   "source": [
    "for key, value in product_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Extracting Company/Brand Names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "system_prompt = \"\"\"You are an expert Company Identification system. \n",
    "Your task is to take a list of strings as input and identify the company that manufactures each product mentioned.\n",
    "Strings provided in the list can be misleading; use your internal knowledge to identify company names.\n",
    "If the manufacturer is unclear for any product, return a null value for that string.\n",
    "Here is an example of the output format for a list of strings.\n",
    "\n",
    "List of strings: [\"PlayStation 5\", \"iPhone 15 Plus\", \"Pencil\", \"Stainless Steel Tumbler\", \"Whopper\", \"Amazon Speed ETD\"]\n",
    "Answer: [\"Sony\", \"Apple\", None, None, \"Burger King\", \"Bruker\"]\n",
    "\n",
    "Only use this output format. Do not output anything besides this output format.\n",
    "Output the manufacturer names in the order they occur in the input list of strings.\n",
    "\"\"\"\n",
    "\n",
    "user_prompt = \"\"\"Q: Given the list of strings below, identify the companies that manufactures each product.\n",
    "\n",
    "List of strings: {}\n",
    "Answer:\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "def company_identification(list_of_products):\n",
    "    response = openai.chat.completions.create(\n",
    "        model=\"gpt-4-turbo\",\n",
    "        presence_penalty=0,\n",
    "        # top_p=1e-16,\n",
    "        temperature=1e-16,\n",
    "        messages=[\n",
    "            {\"role\": \"system\", \"content\": system_prompt},\n",
    "            {\"role\": \"user\", \"content\": user_prompt.format(list_of_products)}\n",
    "        ]\n",
    "    )\n",
    "    return response.choices[0].message.content"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "company_dict = {}\n",
    "syntax_error = False\n",
    "for key, value in product_dict.items():\n",
    "    if value:\n",
    "        identified_companies = company_identification(value)\n",
    "        try:\n",
    "            identified_companies = ast.literal_eval(identified_companies)\n",
    "        except SyntaxError:\n",
    "            syntax_error = True\n",
    "            print(f\"Syntax Error with ID {key}\")\n",
    "        if not syntax_error:\n",
    "            company_dict[key] = identified_companies"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## LLM"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198832: ['Shimadzu', 'SCIEX', 'ABI', 'BioTek', 'Beijing Liuyi Instrument Factory', 'Beijing Bijing Biotechnology Co., LTD.', 'Thermo Fisher Scientific', 'Illumina', 'Bio-rad Corporation', 'VIANEX S.A.', 'Hangzhou Yuanda Biological Pharmaceutical', 'Shiyao Group Enbipu Pharmaceutical', 'Shiyao Group Enbipu Pharmaceutical', 'Sichuan Vicchi Biochemical Technology', 'Omega Bio-Tek', 'ABI', 'Servicebio', 'Servicebio']\n",
      "11198784: ['Cytiva', None, 'Bio-Rad', 'STEMCELL Technologies', None, None, None, 'BD Biosciences', None, 'Sutter Instrument', 'Narishige', None, 'Molecular Devices', 'Molecular Devices', 'Molecular Devices', None, None, None]\n",
      "11197919: ['Ingredion', 'Ingredion', 'Ingredion', 'Tanita', None, 'Salimetrics', 'Waters Corporation', 'Qiagen', None, None, 'Qiagen', 'Qiagen', 'Qiagen', None]\n",
      "11197476: ['Bruker', 'Merck', 'Malvern Instruments', 'Thermo Fisher Scientific', 'Beckman Coulter', 'JEOL', 'Nikon', None, None, None, 'BioLegend']\n",
      "11197382: ['Roche']\n",
      "11197204: ['Thermo Fisher Scientific', 'Thermo Fisher Scientific', 'Bio-Rad Laboratories', 'Bio-Rad Laboratories', 'GraphPad Software']\n",
      "11197185: ['Sterilite', 'Tetra', None, None, None, 'Thermo Fisher Scientific', 'Qiagen', 'QuantaBio']\n",
      "11196966: ['Thermo Fisher Scientific', None, 'Kent Scientific', 'Baxter', 'Ethicon', 'FUJIFILM VisualSonics', 'Bruker', 'Miltenyi Biotec', 'Miltenyi Biotec', None, 'Thermo Fisher Scientific', 'QIAGEN', 'New England Biolabs', 'QIAGEN', 'QIAGEN', 'Thermo Fisher Scientific', 'Bio-Rad', 'Bio-Rad', 'Merck Millipore', 'Bio-Rad', 'Vector Laboratories', 'Thermo Fisher Scientific', 'Vector Laboratories']\n",
      "11196847: ['Applied Biosystems', None, 'Megazyme International']\n",
      "11196703: [None, None]\n",
      "11196393: ['BD Biosciences', None, None, 'BD Biosciences', None, None, None, None]\n",
      "11196377: ['Tyson Foods', None, None, 'A&D Company', 'Konica Minolta']\n",
      "11196265: [None]\n",
      "11196252: ['Metrohm', 'PalmSens', None, None, None, None, 'Hamilton']\n"
     ]
    }
   ],
   "source": [
    "for key, value in company_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198832: ['high-performance liquid chromatography (HPLC) (Shimadzu, LC-20AD)', 'SCIEX mass spectrometer (MS) (AB SCIEX, API 4000+)', 'PCR amplification instrument (ABI, 2720)', 'enzyme labeling instrument (BioTek, FLX800T)', 'electrophoresis apparatus (Beijing Liuyi Instrument Factory, DYY-6C)', 'gel imaging system (Beijing Bijing Biotechnology Co., LTD., BG-gdsAUTO130)', 'Nanodrop UV quantitative system (Thermo Fisher Scientific, NC2000)', 'Sequencer (Illumina, Novaseq6000)', 'fluorescence quantitative PCR instrument (Bio-rad Corporation, CFX)', 'vancomycin hydrochloride for injection (VIANEX S.A.)', 'bifidobacterium quadruplex viable tablets (Hangzhou Yuanda Biological Pharmaceutical)', 'NBP capsules (Shiyao Group Enbipu Pharmaceutical)', 'NBP reference (Shiyao Group Enbipu Pharmaceutical)', 'glipizide reference (Sichuan Vicchi Biochemical Technology)', 'Soil DNA Kit (Omega Bio-Tek)', 'Quant-iT PicoGreen dsDNA Assay Kit (ABI)', 'First Strand cDNA Synthesis Kit (Servicebio)', 'SYBR Green qPCR Master Mix (Servicebio)']\n",
      "11198784: ['Ficoll-Paque Premium', 'trypan blue dye', 'TC20 Automated cell counter', 'EasySep Negative Human NK Cell Isolation Kit', 'CD56 APC', 'CD3 PE Cy7', 'stain buffer', 'BD LSR- FortessaTM X-20 flow cytometer', 'borosilicate glass capillaries', 'Sutter Instrumental model P-97', 'Narishige Micro Forge MF-900', 'CV203BU head-stage', 'Axopatch 200B amplifier', 'pClamp 10.7 software', 'Digidata 1440A analogue to digital converter', '0.22 m membrane filter', 'Sapphire Bioscience Reagents', 'Tocris Bioscience']\n",
      "11197919: ['HI-MAIZE® 260 starch', 'VERSAFIBE™ 1490 starch', 'AMIOCA™ TF starch', 'Tanita SC-240 Total Body Composition Analyzer', 'Automated Self-Administered 24-hour (ASA24) Dietary Assessment Tool', 'Salimetrics Salivary Alpha-Amylase Enzymatic Kit', 'Acquity UPLC system', 'DNeasy PowerSoil 96 HTP Kit', 'Classic++™ Hot Start Taq DNA Polymerase Master Mix', 'Mag-Bind TotalPure', 'QIAamp 96 DNA Blood Kit', 'QIAamp Blood Mini Kit', 'QIAamp Investigator Kit', 'iTaq™ Universal SYBR® Green Supermix']\n",
      "11197476: ['LiposoFast', 'Merck Millipore Ltd.', 'Malvern Nano ZS ZEN3600', 'Thermo Fisher Scientific', 'CytoFLEX', 'JEM-1400', 'Nikon', 'CCK-8 assay kit', 'FITC-conjugated anti-COX-2 antibody', 'in vivo images system', 'Biolegend Inc.']\n",
      "11197382: ['COBAS Integra 400']\n",
      "11197204: ['TRIzol', 'TaqMan™ Gene Expression Assay', 'iTaq Universal One-Step', 'iTaq™ Universal SYBR® Green One-Step Kit', 'GraphPad PRISM']\n",
      "11197185: ['Sterilite container', 'Tetra Pond Koi growth feed', 'Hemotek membrane feeding system', 'Triethylamine', 'Mosquito Diluent', 'Trizol', 'REPLI-g whole transcriptome amplification single-cell kit', 'AccuStart II PCR supermix']\n",
      "11196966: ['Advanced DMEM-F12', 'N-acetylcysteine', 'SomnoSuite® Low-Flow Anesthesia System', 'Seprafilm', '6–0 Vicryl', 'VEVO 3100', 'BioSpec 94/21', 'Liver Dissociation Kit', 'Tumor Dissociation Kit', 'ACK lysis buffer', 'eBioscience™ Foxp3/Transcription Factor Staining Buffer Set', 'RNeasy Mini Kit', 'ProtoScript II First Strand cDNA Synthesis Kit', 'DNeasy Blood & Tissue Kit', 'QuantiFast SYBR® Green PCR Kit', 'Pierce BCA-Kit', 'Mini-Protean TGX precast gel', 'trans-blot turbo transfer system', 'Immobilon Crescendo Western HRP substrate', 'ChemiDoc BioRad imager', 'Vector® TrueVIEW® Autofluorescence Quenching Kit', 'NucBlue™ Fixed Cell Stain ReadyProbes™', 'VECTASHIELD Vibrance Antifade Mounting Medium']\n",
      "11196847: ['ABI 3130 Genetic Analyzer', 'Whatman paper No. 4', 'Megazyme kit']\n",
      "11196703: ['ENSO 16', 'glucose powder']\n",
      "11196393: ['BD FACSAria III cell sorter', 'Ficoll gradient', 'ACK lysing buffer', 'BD FACSCanto™ II', 'Cell Ranger', 'Seurat', 'monocle2', 'GraphPad Prism']\n",
      "11196377: ['Cobb-500', 'Evonik Nutrition & Care GmbH', 'UFFDA software', 'Model GF 400 A&D Weighing', 'CR-410 Konica Minolta']\n",
      "11196265: ['commercial fish pellet feed']\n",
      "11196252: ['AUTOLAB-PGSTAT302N instrument', 'EmStat3 potentiostat', 'automated electrode stand (M164)', 'glassy carbon electrode (GCE)', 'silver/silver chloride electrode (Ag/AgCl/3.0 mol L−1 KCl)', 'platinum wire (Pt, 99.99%)', 'combined pH electrode (Polilyte Lab)']\n"
     ]
    }
   ],
   "source": [
    "for key, value in product_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Spacy"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198832: ['Thermo Fisher Scientific', 'Illumina', 'Thermo Fisher Scientific', 'LI-COR Biosciences']\n",
      "11198784: ['Cytiva', 'Bio-Rad', 'STEMCELL Technologies', 'Narishige', 'Molecular Devices']\n",
      "11197919: ['Tanita', 'Qiagen', 'Qiagen', 'Bio-Rad', 'Qiagen', 'BioSpec Products']\n",
      "11197476: ['Malvern Panalytical']\n",
      "11197204: ['Thermo Fisher Scientific', None, 'Bio-Rad', 'Bio-Rad']\n",
      "11197185: ['Illumina', None, 'Thermo Fisher Scientific', None, 'Illumina']\n",
      "11196966: ['Piramal', 'Kent Scientific', None, None, 'Qiagen', 'New England Biolabs', 'Qiagen', 'Qiagen', 'Thermo Fisher Scientific', 'Azure Biosystems', 'Zeiss', 'Vector Laboratories', 'Thermo Fisher Scientific', 'Vector Laboratories']\n",
      "11196847: ['Thermo Fisher Scientific', None]\n",
      "11196703: [None, None]\n",
      "11196417: ['TransGen Biotech', 'Thermo Fisher Scientific', 'TransGen Biotech', 'Vazyme', 'Thermo Fisher Scientific']\n",
      "11196393: ['Becton Dickinson', None, 'GraphPad Software']\n",
      "11196377: ['Konica Minolta']\n",
      "11196265: ['Canon']\n",
      "11196252: ['Metrohm', 'PalmSens', None]\n"
     ]
    }
   ],
   "source": [
    "for key, value in company_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11198832: ['Nanodrop UV', 'Novaseq6000', 'Quant-iT PicoGreen dsDNA Assay Kit', 'The Odyssey CLx Imaging System']\n",
      "11198784: ['Ficoll-Paque Premium', 'TC20 Automated cell counter', 'EasySep Negative Human NK Cell Isolation Kit', 'Micro Forge MF-900', 'Axopatch 200B']\n",
      "11197919: ['Tanita SC-240 Total Body Composition Analyzer', 'QIAamp 96 DNA Blood Kit', 'QIAamp Investigator Kit', 'QX100 Droplet Digital PCR System', 'DNeasy PowerSoil 96 HTP Kit', 'BioSpec 1001 Mini-Beadbeater-96']\n",
      "11197476: ['Malvern Nano ZS ZEN3600']\n",
      "11197204: ['TRIzol', 'Easy Oligos', 'iTaq Universal One-Step', 'iTaq™ Universal SYBR® Green One-Step']\n",
      "11197185: ['MiSeq', 'Hemotek', 'Trizol', 'AccuStart II', 'MiSeq 500']\n",
      "11196966: ['Isoflurane Piramal', 'SomnoSuite® Low-Flow Anesthesia System', 'Liver Dissociation Kit', 'Tumor Dissociation Kit', 'RNeasy Mini Kit', 'ProtoScript II First Strand cDNA Synthesis Kit', 'DNeasy Blood & Tissue Kit', 'QuantiFast SYBR® Green PCR Kit', 'Pierce BCA-Kit', 'Azure Ao Microplate Reader', 'Zeiss Axio Scan Z.1 Microscope Slide Scanner', 'Vector® TrueVIEW® Autofluorescence Quenching Kit', 'NucBlue™ Fixed Cell Stain ReadyProbes™', 'VECTASHIELD Vibrance Antifade Mounting Medium']\n",
      "11196847: ['ABI 3130 Genetic Analyzer', 'RP C-18 column']\n",
      "11196703: ['ENSO 16', 'NEOH']\n",
      "11196417: ['TransZol Up Plus RNA Kit', 'Nanodrop Spectrophotometer', 'TransScript® One-Step gDNA Removal and cDNA Synthesis SuperMix', 'ChamQ Universal SYBR qPCR Master Mix', 'QuantStudio™ 5']\n",
      "11196393: ['BD FACSCanto™ II', 'Brilliant Violet 605™', 'GraphPad Prism 8.0']\n",
      "11196377: ['Chroma‐Meter (CR‐410 Konica Minolta, INC']\n",
      "11196265: ['PowershotA95']\n",
      "11196252: ['AUTOLAB-PGSTAT302N', 'EmStat3', 'M164']\n"
     ]
    }
   ],
   "source": [
    "for key, value in cleaned_entity_dict.items():\n",
    "    if value:\n",
    "        print(f\"{key}: {value}\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
